{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ATENÇÃO À UTILIZAÇÃO DESTA FUNÇÃO\n",
    "# DEMASIADAS QUERIES SEGUIDAS PODEM LEVAR A BLOQUEIO POR PARTE DO SERVIDOR\n",
    "# EM PRINCÍPIO O MÓDULO ENTREZ ESTÁ PREPARADO PARA LIDAR COM ISSO MAS CONVÉM TER SEMPRE CUIDADO\n",
    "\n",
    "def lit_search(email, term,db = 'pubmed'):\n",
    "\n",
    "    # TODO Adicionar indicações para fazer o print e elencar a lista de resultados\n",
    "    # TODO Adicionar documentação\n",
    "\n",
    "\n",
    "    # Importamos o módulo \n",
    "    from Bio import Entrez\n",
    "\n",
    "    # Definimos o email para os fins de pesquisa\n",
    "    Entrez.email = email\n",
    "    \n",
    "    \n",
    "\n",
    "    # Secção egquery que pode ser skipable\n",
    "\n",
    "    handle = Entrez.egquery(term=term)\n",
    "    egq_res = Entrez.read(handle)\n",
    "\n",
    "    for _ in egq_res['eGQueryResult']:\n",
    "        if db in _.values():\n",
    "            print('Resultados em \"pubmed\":',_['Count'])\n",
    "\n",
    "    # Sacar as IDs\n",
    "    handle      = Entrez.esearch(db=db,term=term, retmax=10)\n",
    "    esearch_res = Entrez.read(handle)\n",
    "\n",
    "    lista_ids = esearch_res['IdList']\n",
    "    print('Top 10 artigos ->',lista_ids) # Converter para uma lista de títulos mais à frente\n",
    "    \n",
    "    \n",
    "\n",
    "    # Sacar os abstracts\n",
    "    handle = Entrez.efetch(db=db,id=','.join(lista_ids),rettype='abstract',retmode='text')\n",
    "    fetch_res = handle.read()\n",
    "    \n",
    "    # Usar com print para já, para não estar a bagunçar demasiado.\n",
    "    # Perceber como se pode implementar a escrita de ficheiros para ser mais fácil de mastigar os resultados.\n",
    "\n",
    "    return fetch_res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Resultados em \"pubmed\": 50\n",
      "Top 10 artigos -> ['37827449', '37679517', '36973646', '36269708', '35606880', '35568807', '35500836', '35436980', '34888651', '34095496']\n"
     ]
    }
   ],
   "source": [
    "termo = 'hla-dqa2 polymorphism'\n",
    "email = 'alterar-o-email@mail.com'\n",
    "teste = lit_search(email, termo)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\n",
      " 2023 Oct 10.\n",
      "\n",
      "Genome-wide identification of m(6)A-associated single nucleotide polymorphisms \n",
      "in complex diseases of nervous system.\n",
      "\n",
      "Guo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \n",
      "Guo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\n",
      "\n",
      "Author information:\n",
      "(1)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China.\n",
      "(2)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China.\n",
      "(3)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "jiangyongshuai@hrbmu.edu.cn.\n",
      "(4)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "zhangmingming@hrbmu.edu.cn.\n",
      "\n",
      "N6-methyladenosine (m6A) is one of the most abundant chemical modifications on \n",
      "RNA and can affect the occurrence and development of diseases. Some studies have \n",
      "shown that the expressions of some m6A-related genes are significantly regulated \n",
      "by single nucleotide variants (SNV). However, the function of m6A-associated \n",
      "single nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \n",
      "(MS), Alzheimer's disease (AD) and Parkinson's disease (PD). Here, we identified \n",
      "the disease-associated m6A-SNPs by integrating genome-wide association study \n",
      "(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \n",
      "between these identified m6A-SNPs and their target genes in eQTL analysis and \n",
      "gene differential expression analysis. Finally, 26 genes corresponding to 20 \n",
      "m6A-SNPs with eQTL signals were identified and differentially expressed \n",
      "(P < 0.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P < 1e-04) were \n",
      "differentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \n",
      "expression of 31 genes were identified. There were 5 HLA genes with eQTL signals \n",
      "(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \n",
      "three diseases. In summary, our study provided new insights into understanding \n",
      "the potential roles of these m6A-SNPs in disease pathogenesis as well as \n",
      "therapeutic target.\n",
      "\n",
      "Copyright © 2023 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.neulet.2023.137513\n",
      "PMID: 37827449 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "2. Br J Cancer. 2023 Oct;129(9):1510-1515. doi: 10.1038/s41416-023-02419-3. Epub \n",
      "2023 Sep 7.\n",
      "\n",
      "Integrating genomics and proteomics data to identify candidate plasma biomarkers \n",
      "for lung cancer risk among European descendants.\n",
      "\n",
      "Yang Y(#)(1), Xu S(#)(2), Jia G(2), Yuan F(2), Ping J(2), Guo X(2), Tao R(3)(4), \n",
      "Shu XO(2), Zheng W(2), Long J(2), Cai Q(5).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Public Health Genomics, Department of Public Health Sciences, UVA \n",
      "Comprehensive Cancer Center, School of Medicine, University of Virginia, \n",
      "Charlottesville, VA, USA. vta8we@virginia.edu.\n",
      "(2)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(3)Department of Biostatistics, Vanderbilt University Medical Center, Nashville, \n",
      "TN, USA.\n",
      "(4)Vanderbilt Genetics Institute, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(5)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA. qiuyin.cai@vanderbilt.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: Plasma proteins are potential biomarkers for complex diseases. We \n",
      "aimed to identify plasma protein biomarkers for lung cancer.\n",
      "METHODS: We investigated genetically predicted plasma levels of 1130 proteins in \n",
      "association with lung cancer risk among 29,266 cases and 56,450 controls of \n",
      "European descent. For proteins significantly associated with lung cancer risk, \n",
      "we evaluated associations of genetically predicted expression of their coding \n",
      "genes with the risk of lung cancer.\n",
      "RESULTS: Nine proteins were identified with genetically predicted plasma levels \n",
      "significantly associated with overall lung cancer risk at a false discovery rate \n",
      "(FDR) of <0.05. Proteins C2, MICA, AIF1, and CTSH were associated with increased \n",
      "lung cancer risk, while proteins SFTPB, HLA-DQA2, MICB, NRP1, and GMFG were \n",
      "associated with decreased lung cancer risk. Stratified analyses by histological \n",
      "types revealed the cross-subtype consistency of these nine associations and \n",
      "identified an additional protein, ICAM5, significantly associated with lung \n",
      "adenocarcinoma risk (FDR < 0.05). Coding genes of NRP1 and ICAM5 proteins are \n",
      "located at two loci that have never been reported by previous GWAS. Genetically \n",
      "predicted blood levels of genes C2, AIF1, and CTSH were associated with lung \n",
      "cancer risk, in directions consistent with those shown in protein-level \n",
      "analyses.\n",
      "CONCLUSION: Identification of novel plasma protein biomarkers provided new \n",
      "insights into the biology of lung cancer.\n",
      "\n",
      "© 2023. The Author(s), under exclusive licence to Springer Nature Limited.\n",
      "\n",
      "DOI: 10.1038/s41416-023-02419-3\n",
      "PMCID: PMC10628278\n",
      "PMID: 37679517 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "3. BMC Genomics. 2023 Mar 27;24(1):153. doi: 10.1186/s12864-023-09227-2.\n",
      "\n",
      "Genome-wide identification of RNA modification-related single nucleotide \n",
      "polymorphisms associated with rheumatoid arthritis.\n",
      "\n",
      "Wang M(#)(1)(2), Wu J(#)(1)(2), Lei S(1)(2), Mo X(3)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China.\n",
      "(2)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China.\n",
      "(3)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China. xbmo@suda.edu.cn.\n",
      "(4)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China. xbmo@suda.edu.cn.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: RNA modification plays important roles in many biological processes, \n",
      "such as gene expression control. The aim of this study was to identify single \n",
      "nucleotide polymorphisms related to RNA modification (RNAm-SNPs) for rheumatoid \n",
      "arthritis (RA) as putative functional variants.\n",
      "METHODS: We examined the association of RNAm-SNPs with RA in summary data from a \n",
      "genome-wide association study of 19,234 RA cases and 61,565 controls. We \n",
      "performed eQTL and pQTL analyses for the RNAm-SNPs to find associated gene \n",
      "expression and protein levels. Furthermore, we examined the associations of gene \n",
      "expression and circulating protein levels with RA using two-sample Mendelian \n",
      "randomization analysis methods.\n",
      "RESULTS: A total of 160 RNAm-SNPs related to m6A, m1A, A-to-I, m7G, m5C, m5U and \n",
      "m6Am modifications were identified to be significantly associated with RA. These \n",
      "RNAm-SNPs were located in 62 protein-coding genes, which were significantly \n",
      "enriched in immune-related pathways. RNAm-SNPs in important RA susceptibility \n",
      "genes, such as PADI2, SPRED2, PLCL2, HLA-A, HLA-B, HLA-DRB1, HLA-DPB1, TRAF1 and \n",
      "TXNDC11, were identified. Most of these RNAm-SNPs showed eQTL effects, and the \n",
      "expression levels of 26 of the modifiable genes (e.g., PADI2, TRAF1, HLA-A, \n",
      "HLA-DRB1, HLA-DPB1 and HLA-B) in blood cells were associated with RA. \n",
      "Circulating protein levels, such as CFB, GZMA, HLA-DQA2, IL21, LRPAP1 and TFF3, \n",
      "were affected by RNAm-SNPs and were associated with RA.\n",
      "CONCLUSION: The present study identified RNAm-SNPs in the reported RA \n",
      "susceptibility genes and suggested that RNAm-SNPs may affect RA risk by \n",
      "affecting the expression levels of corresponding genes and proteins.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12864-023-09227-2\n",
      "PMCID: PMC10045113\n",
      "PMID: 36973646 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "4. PLoS One. 2022 Oct 21;17(10):e0275934. doi: 10.1371/journal.pone.0275934. \n",
      "eCollection 2022.\n",
      "\n",
      "Genome-wide discovery for diabetes-dependent triglycerides-associated loci.\n",
      "\n",
      "Selvaraj MS(1)(2)(3), Paruchuri K(1)(2), Haidermota S(1)(2), Bernardo R(1)(2), \n",
      "Rich SS(4), Peloso GM(5), Natarajan P(1)(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Cardiovascular Research Center, Massachusetts General Hospital, Boston, MA, \n",
      "United States of America.\n",
      "(2)Program in Medical and Population Genetics, Broad Institute of Harvard and \n",
      "MIT, Cambridge, MA, United States of America.\n",
      "(3)Department of Medicine, Harvard Medical School, Boston, MA, United States of \n",
      "America.\n",
      "(4)Center for Public Health Genomics, University of Virginia, Charlottesville, \n",
      "VA, United States of America.\n",
      "(5)Department of Biostatistics, Boston University School of Public Health, \n",
      "Boston, MA, United States of America.\n",
      "\n",
      "PURPOSE: We aimed to discover loci associated with triglyceride (TG) levels in \n",
      "the context of type 2 diabetes (T2D). We conducted a genome-wide association \n",
      "study (GWAS) in 424,120 genotyped participants of the UK Biobank (UKB) with T2D \n",
      "status and TG levels.\n",
      "METHODS: We stratified the cohort based on T2D status and conducted association \n",
      "analyses of TG levels for genetic variants with minor allele count (MAC) at \n",
      "least 20 in each stratum. Effect differences of genetic variants by T2D status \n",
      "were determined by Cochran's Q-test and we validated the significantly \n",
      "associated variants in the Mass General Brigham Biobank (MGBB).\n",
      "RESULTS: Among 21,176 T2D and 402,944 non-T2D samples from UKB, stratified GWAS \n",
      "identified 19 and 315 genomic risk loci significantly associated with TG levels, \n",
      "respectively. Only chr6p21.32 exhibited genome-wide significant heterogeneity \n",
      "(I2 = 98.4%; pheterogeneity = 2.1x10-15), with log(TG) effect estimates of \n",
      "-0.066 (95%CI: -0.082, -0.050) and 0.002 (95%CI: -0.002, 0.006) for T2D and \n",
      "non-T2D, respectively. The lead variant rs9274619:A (allele frequency 0.095) is \n",
      "located 2Kb upstream of the HLA-DQB1 gene, between HLA-DQB1 and HLA-DQA2 genes. \n",
      "We replicated this finding among 25,137 participants (6,951 T2D cases) of MGBB \n",
      "(pheterogeneity = 9.5x10-3). Phenome-wide interaction association analyses \n",
      "showed that the lead variant was strongly associated with a concomitant \n",
      "diagnosis of type 1 diabetes (T1D) as well as diabetes-associated complications.\n",
      "CONCLUSION: In conclusion, we identified an intergenic variant near \n",
      "HLA-DQB1/DQA2 significantly associates with decreased triglycerides only among \n",
      "those with T2D and highlights an immune overlap with T1D.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0275934\n",
      "PMCID: PMC9586367\n",
      "PMID: 36269708 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: P.N. reports grants from Amgen, Apple, \n",
      "AstraZeneca, Boston Scientific, and Novartis, personal fees from Apple, \n",
      "AstraZeneca, Blackstone Life Sciences, Foresite Labs, Genentech / Roche, \n",
      "Novartis, and TenSixteen Bio, equity in geneXwell, and TenSixteen Bio, \n",
      "co-founder of TenSixteen Bio, and spousal employment at Vertex, all unrelated to \n",
      "the present work. This does not alter our adherence to PLOS ONE policies on \n",
      "sharing data and materials.\n",
      "\n",
      "\n",
      "5. Genome Med. 2022 May 24;14(1):55. doi: 10.1186/s13073-022-01058-2.\n",
      "\n",
      "Fine-mapping studies distinguish genetic risks for childhood- and adult-onset \n",
      "asthma in the HLA region.\n",
      "\n",
      "Clay SM(1), Schoettler N(2), Goldstein AM(3), Carbonetto P(4), Dapas M(4), \n",
      "Altman MC(5)(6), Rosasco MG(6), Gern JE(7), Jackson DJ(7), Im HK(8), Stephens \n",
      "M(3), Nicolae DL(4)(3), Ober C(9).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "seleneclay@uchicago.edu.\n",
      "(2)Section of Pulmonary and Critical Care, Department of Medicine, University of \n",
      "Chicago, Chicago, IL, 60637, USA.\n",
      "(3)Department of Statistics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(4)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(5)Division of Allergy and Infectious Diseases, Department of Medicine, \n",
      "University of Washington, Seattle, WA, 98109, USA.\n",
      "(6)Systems Immunology Program, Benaroya Research Institute, Seattle, WA, 98101, \n",
      "USA.\n",
      "(7)Department of Pediatrics, University of Wisconsin, School of Medicine and \n",
      "Public Health, Madison, WI, 53706, USA.\n",
      "(8)Section of Genetic Medicine, Department of Medicine, University of Chicago, \n",
      "Chicago, IL, 60637, USA.\n",
      "(9)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "c-ober@genetics.uchicago.edu.\n",
      "\n",
      "BACKGROUND: Genome-wide association studies of asthma have revealed robust \n",
      "associations with variation across the human leukocyte antigen (HLA) complex \n",
      "with independent associations in the HLA class I and class II regions for both \n",
      "childhood-onset asthma (COA) and adult-onset asthma (AOA). However, the specific \n",
      "variants and genes contributing to risk are unknown.\n",
      "METHODS: We used Bayesian approaches to perform genetic fine-mapping for COA and \n",
      "AOA (n=9432 and 21,556, respectively; n=318,167 shared controls) in White \n",
      "British individuals from the UK Biobank and to perform expression quantitative \n",
      "trait locus (eQTL) fine-mapping in immune (lymphoblastoid cell lines, n=398; \n",
      "peripheral blood mononuclear cells, n=132) and airway (nasal epithelial cells, \n",
      "n=188) cells from ethnically diverse individuals. We also examined putatively \n",
      "causal protein coding variation from protein crystal structures and conducted \n",
      "replication studies in independent multi-ethnic cohorts from the UK Biobank (COA \n",
      "n=1686; AOA n=3666; controls n=56,063).\n",
      "RESULTS: Genetic fine-mapping revealed both shared and distinct causal variation \n",
      "between COA and AOA in the class I region but only distinct causal variation in \n",
      "the class II region. Both gene expression levels and amino acid variation \n",
      "contributed to risk. Our results from eQTL fine-mapping and amino acid \n",
      "visualization suggested that the HLA-DQA1*03:01 allele and variation associated \n",
      "with expression of the nonclassical HLA-DQA2 and HLA-DQB2 genes accounted \n",
      "entirely for the most significant association with AOA in GWAS. Our studies also \n",
      "suggested a potentially prominent role for HLA-C protein coding variation in the \n",
      "class I region in COA. We replicated putatively causal variant associations in a \n",
      "multi-ethnic cohort.\n",
      "CONCLUSIONS: We highlight roles for both gene expression and protein coding \n",
      "variation in asthma risk and identified putatively causal variation and genes in \n",
      "the HLA region. A convergence of genomic, transcriptional, and protein coding \n",
      "evidence implicates the HLA-DQA2 and HLA-DQB2 genes and HLA-DQA1*03:01 allele in \n",
      "AOA.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s13073-022-01058-2\n",
      "PMCID: PMC9128203\n",
      "PMID: 35606880 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "6. BMC Genomics. 2022 May 14;23(1):368. doi: 10.1186/s12864-022-08587-5.\n",
      "\n",
      "Diabetes risk loci-associated pathways are shared across metabolic tissues.\n",
      "\n",
      "Bouland GA(1), Beulens JWJ(2)(3), Nap J(1), van der Slik AR(4), Zaldumbide A(1), \n",
      "'t Hart LM(1)(2)(5), Slieker RC(6)(7).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Cell and Chemical Biology, Leiden University Medical Center, \n",
      "Einthovenweg 20, 2333ZC, Leiden, the Netherlands.\n",
      "(2)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \n",
      "Amsterdam Public Health Institute, Amsterdam, the Netherlands.\n",
      "(3)Julius Center for Health Sciences and Primary Care, University Medical Center \n",
      "Utrecht, Utrecht, the Netherlands.\n",
      "(4)Department of Immunohematology and Blood Transfusion, Leiden University \n",
      "Medical Center, Leiden, The Netherlands.\n",
      "(5)Molecular Epidemiology Section, Department of Biomedical Data Sciences, \n",
      "Leiden University Medical Center, Leiden, the Netherlands.\n",
      "(6)Department of Cell and Chemical Biology, Leiden University Medical Center, \n",
      "Einthovenweg 20, 2333ZC, Leiden, the Netherlands. r.c.slieker@lumc.nl.\n",
      "(7)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \n",
      "Amsterdam Public Health Institute, Amsterdam, the Netherlands. \n",
      "r.c.slieker@lumc.nl.\n",
      "\n",
      "AIMS/HYPOTHESIS: Numerous genome-wide association studies have been performed to \n",
      "understand the influence of genetic variation on type 2 diabetes etiology. Many \n",
      "identified risk variants are located in non-coding and intergenic regions, which \n",
      "complicates understanding of how genes and their downstream pathways are \n",
      "influenced. An integrative data approach will help to understand the mechanism \n",
      "and consequences of identified risk variants.\n",
      "METHODS: In the current study we use our previously developed method CONQUER to \n",
      "overlap 403 type 2 diabetes risk variants with regulatory, expression and \n",
      "protein data to identify tissue-shared disease-relevant mechanisms.\n",
      "RESULTS: One SNP rs474513 was found to be an expression-, protein- and \n",
      "metabolite QTL. Rs474513 influenced LPA mRNA and protein levels in the pancreas \n",
      "and plasma, respectively. On the pathway level, in investigated tissues most \n",
      "SNPs linked to metabolism. However, in eleven of the twelve tissues investigated \n",
      "nine SNPs were linked to differential expression of the ribosome pathway. \n",
      "Furthermore, seven SNPs were linked to altered expression of genes linked to the \n",
      "immune system. Among them, rs601945 was found to influence multiple HLA genes, \n",
      "including HLA-DQA2, in all twelve tissues investigated.\n",
      "CONCLUSION: Our results show that in addition to the classical metabolism \n",
      "pathways, other pathways may be important to type 2 diabetes that show a \n",
      "potential overlap with type 1 diabetes.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12864-022-08587-5\n",
      "PMCID: PMC9107144\n",
      "PMID: 35568807 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "7. J Thorac Oncol. 2022 Aug;17(8):974-990. doi: 10.1016/j.jtho.2022.04.011. Epub \n",
      "2022 Apr 30.\n",
      "\n",
      "A Large-Scale Genome-Wide Gene-Gene Interaction Study of Lung Cancer \n",
      "Susceptibility in Europeans With a Trans-Ethnic Validation in Asians.\n",
      "\n",
      "Zhang R(1), Shen S(2), Wei Y(1), Zhu Y(3), Li Y(4), Chen J(3), Guan J(3), Pan \n",
      "Z(3), Wang Y(5), Zhu M(5), Xie J(6), Xiao X(7), Zhu D(7), Li Y(7), Albanes D(8), \n",
      "Landi MT(8), Caporaso NE(8), Lam S(9), Tardon A(10), Chen C(11), Bojesen SE(12), \n",
      "Johansson M(13), Risch A(14), Bickeböller H(15), Wichmann HE(16), Rennert G(17), \n",
      "Arnold S(18), Brennan P(13), McKay JD(13), Field JK(19), Shete SS(20), Le \n",
      "Marchand L(21), Liu G(22), Andrew AS(23), Kiemeney LA(24), Zienolddiny-Narui \n",
      "S(25), Behndig A(26), Johansson M(27), Cox A(28), Lazarus P(29), Schabath \n",
      "MB(30), Aldrich MC(31), Dai J(5), Ma H(5), Zhao Y(3), Hu Z(32), Hung RJ(33), \n",
      "Amos CI(7), Shen H(32), Chen F(34), Christiani DC(35).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Department of Environmental \n",
      "Health, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \n",
      "International Cooperation Center (CICC) for Environment and Human Health, \n",
      "Nanjing Medical University, Nanjing, People's Republic of China.\n",
      "(2)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Department of Environmental \n",
      "Health, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \n",
      "International Cooperation Center (CICC) for Environment and Human Health, \n",
      "Nanjing Medical University, Nanjing, People's Republic of China; State Key \n",
      "Laboratory of Reproductive Medicine, Nanjing Medical University, Nanjing, \n",
      "People's Republic of China.\n",
      "(3)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China.\n",
      "(4)Department of Biostatistics, University of Michigan, Ann Arbor, Michigan.\n",
      "(5)Department of Epidemiology, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer \n",
      "Biomarkers, Prevention and Treatment, Cancer Center, Collaborative Innovation \n",
      "Center for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, \n",
      "People's Republic of China.\n",
      "(6)Department of Epidemiology, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China.\n",
      "(7)The Institute for Clinical and Translational Research, Department of \n",
      "Medicine, Baylor College of Medicine, Houston, Texas.\n",
      "(8)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \n",
      "National Institutes of Health, Bethesda, Maryland.\n",
      "(9)Department of Medicine, British Columbia Cancer Agency, University of British \n",
      "Columbia, Vancouver, Canada.\n",
      "(10)Faculty of Medicine, University of Oviedo and CIBERESP, Oviedo, Spain.\n",
      "(11)Department of Epidemiology, University of Washington School of Public \n",
      "Health, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, \n",
      "Seattle, Washington.\n",
      "(12)Department of Clinical Biochemistry, Herlev and Gentofte Hospital, \n",
      "Copenhagen University Hospital, Copenhagen, Denmark.\n",
      "(13)Section of Genetics, International Agency for Research on Cancer, World \n",
      "Health Organization, Lyon, France.\n",
      "(14)Department of Biosciences and Cancer Cluster Salzburg, University of \n",
      "Salzburg, Salzburg, Austria.\n",
      "(15)Department of Genetic Epidemiology, University Medical Center, Georg August \n",
      "University Göttingen, Göttingen, Germany.\n",
      "(16)Institute of Medical Informatics, Biometry and Epidemiology, Ludwig \n",
      "Maximilians University, Munich, Germany.\n",
      "(17)Clalit National Cancer Control Center, Carmel Medical Center and Technion \n",
      "Faculty of Medicine, Carmel, Haifa, Israel.\n",
      "(18)Markey Cancer Center, University of Kentucky, Lexington, Kentucky.\n",
      "(19)Department of Molecular and Clinical Cancer Medicine, Institute of \n",
      "Translational Medicine, University of Liverpool, Liverpool, United Kingdom.\n",
      "(20)Department of Biostatistics, The University of Texas MD Anderson Cancer \n",
      "Center, Houston, Texas.\n",
      "(21)Epidemiology Program, University of Hawaii Cancer Center, Honolulu, Hawaii.\n",
      "(22)Princess Margaret Cancer Centre, Dalla Lana School of Public Health, \n",
      "University of Toronto, Toronto, Canada.\n",
      "(23)Department of Epidemiology, Department of Community and Family Medicine, \n",
      "Dartmouth Geisel School of Medicine, Hanover, New Hampshire.\n",
      "(24)Department for Health Evidence, Department of Urology, Radboud University \n",
      "Medical Center, Nijmegen, The Netherlands.\n",
      "(25)National Institute of Occupational Health, Oslo, Norway.\n",
      "(26)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(27)Department of Radiation Sciences, Umeå University, Umeå, Sweden.\n",
      "(28)Department of Oncology and Metabolism, The Medical School, University of \n",
      "Sheffield, Sheffield, United Kingdom.\n",
      "(29)Department of Pharmaceutical Sciences, College of Pharmacy, Washington State \n",
      "University, Spokane, Washington.\n",
      "(30)Department of Cancer Epidemiology, H. Lee Moffitt Cancer Center and Research \n",
      "Institute, Tampa, Florida.\n",
      "(31)Department of Thoracic Surgery and Division of Epidemiology, Vanderbilt \n",
      "University Medical Center, Nashville, Tennessee.\n",
      "(32)China International Cooperation Center (CICC) for Environment and Human \n",
      "Health, Nanjing Medical University, Nanjing, People's Republic of China; \n",
      "Department of Epidemiology, School of Public Health, Nanjing Medical University, \n",
      "Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \n",
      "Prevention and Treatment, Cancer Center, Collaborative Innovation Center for \n",
      "Cancer Personalized Medicine, Nanjing Medical University, Nanjing, People's \n",
      "Republic of China.\n",
      "(33)Lunenfeld-Tanenbaum Research Institute, Sinai Health System, Dalla Lana \n",
      "School of Public Health, University of Toronto, Toronto, Ontario, Canada.\n",
      "(34)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; China International Cooperation \n",
      "Center (CICC) for Environment and Human Health, Nanjing Medical University, \n",
      "Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \n",
      "Prevention and Treatment, Cancer Center, Collaborative Innovation Center for \n",
      "Cancer Personalized Medicine, Nanjing Medical University, Nanjing, People's \n",
      "Republic of China. Electronic address: fengchen@njmu.edu.cn.\n",
      "(35)Department of Environmental Health, Harvard T. H. Chan School of Public \n",
      "Health, Boston, Massachusetts; Pulmonary and Critical Care Division, Department \n",
      "of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, \n",
      "Massachusetts.\n",
      "\n",
      "Comment in\n",
      "    J Thorac Oncol. 2022 Aug;17(8):946-948.\n",
      "    J Thorac Oncol. 2023 Mar;18(3):e23-e24.\n",
      "    J Thorac Oncol. 2023 Mar;18(3):e24-e26.\n",
      "\n",
      "INTRODUCTION: Although genome-wide association studies have been conducted to \n",
      "investigate genetic variation of lung tumorigenesis, little is known about \n",
      "gene-gene (G × G) interactions that may influence the risk of non-small cell \n",
      "lung cancer (NSCLC).\n",
      "METHODS: Leveraging a total of 445,221 European-descent participants from the \n",
      "International Lung Cancer Consortium OncoArray project, Transdisciplinary \n",
      "Research in Cancer of the Lung and UK Biobank, we performed a large-scale \n",
      "genome-wide G × G interaction study on European NSCLC risk by a series of \n",
      "analyses. First, we used BiForce to evaluate and rank more than 58 billion G × G \n",
      "interactions from 340,958 single-nucleotide polymorphisms (SNPs). Then, the top \n",
      "interactions were further tested by demographically adjusted logistic regression \n",
      "models. Finally, we used the selected interactions to build lung cancer \n",
      "screening models of NSCLC, separately, for never and ever smokers.\n",
      "RESULTS: With the Bonferroni correction, we identified eight statistically \n",
      "significant pairs of SNPs, which predominantly appeared in the 6p21.32 and \n",
      "5p15.33 regions (e.g., rs521828C6orf10 and rs204999PRRT1, ORinteraction = 1.17, \n",
      "p = 6.57 × 10-13; rs3135369BTNL2 and rs2858859HLA-DQA1, ORinteraction = 1.17, \n",
      "p = 2.43 × 10-13; rs2858859HLA-DQA1 and rs9275572HLA-DQA2, ORinteraction = 1.15, \n",
      "p = 2.84 × 10-13; rs2853668TERT and rs62329694CLPTM1L, ORinteraction = 0.73, p = \n",
      "2.70 × 10-13). Notably, even with much genetic heterogeneity across ethnicities, \n",
      "three pairs of SNPs in the 6p21.32 region identified from the European-ancestry \n",
      "population remained significant among an Asian population from the Nanjing \n",
      "Medical University Global Screening Array project (rs521828C6orf10 and \n",
      "rs204999PRRT1, ORinteraction = 1.13, p = 0.008; rs3135369BTNL2 and \n",
      "rs2858859HLA-DQA1, ORinteraction = 1.11, p = 5.23 × 10-4; rs3135369BTNL2 and \n",
      "rs9271300HLA-DQA1, ORinteraction = 0.89, p = 0.006). The interaction-empowered \n",
      "polygenetic risk score that integrated classical polygenetic risk score and G × \n",
      "G information score was remarkable in lung cancer risk stratification.\n",
      "CONCLUSIONS: Important G × G interactions were identified and enriched in the \n",
      "5p15.33 and 6p21.32 regions, which may enhance lung cancer screening models.\n",
      "\n",
      "Copyright © 2022 International Association for the Study of Lung Cancer. \n",
      "Published by Elsevier Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jtho.2022.04.011\n",
      "PMCID: PMC9512697\n",
      "PMID: 35500836 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Disclosure: The authors declare no conflict of \n",
      "interest.\n",
      "\n",
      "\n",
      "8. Transl Psychiatry. 2022 Apr 18;12(1):163. doi: 10.1038/s41398-022-01913-1.\n",
      "\n",
      "Allele-specific analysis reveals exon- and cell-type-specific regulatory effects \n",
      "of Alzheimer's disease-associated genetic variants.\n",
      "\n",
      "He L(1), Loika Y(2), Kulminski AM(3).\n",
      "\n",
      "Author information:\n",
      "(1)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. lh235@duke.edu.\n",
      "(2)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA.\n",
      "(3)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. alexander.kulminski@duke.edu.\n",
      "\n",
      "Elucidating regulatory effects of Alzheimer's disease (AD)-associated genetic \n",
      "variants is critical for unraveling their causal pathways and understanding the \n",
      "pathology. However, their cell-type-specific regulatory mechanisms in the brain \n",
      "remain largely unclear. Here, we conducted an analysis of allele-specific \n",
      "expression quantitative trait loci (aseQTLs) for 33 AD-associated variants in \n",
      "four brain regions and seven cell types using ~3000 bulk RNA-seq samples and \n",
      ">0.25 million single nuclei. We first develop a flexible hierarchical Poisson \n",
      "mixed model (HPMM) and demonstrate its superior statistical power to a \n",
      "beta-binomial model achieved by unifying samples in both allelic and \n",
      "genotype-level expression data. Using the HPMM, we identified 24 (~73%) aseQTLs \n",
      "in at least one brain region, including three new eQTLs associated with CA12, \n",
      "CHRNE, and CASS4. Notably, the APOE ε4 variant reduces APOE expression across \n",
      "all regions, even in AD-unaffected controls. Our results reveal region-dependent \n",
      "and exon-specific effects of multiple aseQTLs, such as rs2093760 with CR1, \n",
      "rs7982 with CLU, and rs3865444 with CD33. In an attempt to pinpoint the cell \n",
      "types responsible for the observed tissue-level aseQTLs using the snRNA-seq \n",
      "data, we detected many aseQTLs in microglia or monocytes associated with \n",
      "immune-related genes, including HLA-DQB1, HLA-DQA2, CD33, FCER1G, MS4A6A, SPI1, \n",
      "and BIN1, highlighting the regulatory role of AD-associated variants in the \n",
      "immune response. These findings provide further insights into potential causal \n",
      "pathways and cell types mediating the effects of the AD-associated variants.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1038/s41398-022-01913-1\n",
      "PMCID: PMC9016079\n",
      "PMID: 35436980 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "9. Rheumatology (Oxford). 2022 Aug 3;61(8):3461-3470. doi: \n",
      "10.1093/rheumatology/keab912.\n",
      "\n",
      "Identification and functional characterization of a novel susceptibility locus \n",
      "for small vessel vasculitis with MPO-ANCA.\n",
      "\n",
      "Dahlqvist J(1)(2)(3), Ekman D(4), Sennblad B(5), Kozyrev SV(2), Nordin J(2), \n",
      "Karlsson Å(2), Meadows JRS(2), Hellbacher E(1), Rantapää-Dahlqvist S(6), Berglin \n",
      "E(6), Stegmayr B(6), Baslund B(7), Palm Ø(8), Haukeland H(9), Gunnarsson \n",
      "I(10)(11), Bruchfeld A(12)(13), Segelmark M(14), Ohlsson S(14), Mohammad \n",
      "AJ(15)(16), Svärd A(17), Pullerits R(18)(19), Herlitz H(20), Söderbergh A(21), \n",
      "Rosengren Pielberg G(2), Hultin Rosenberg L(2), Bianchi M(2), Murén E(2), Omdal \n",
      "R(22)(23), Jonsson R(24), Eloranta ML(1), Rönnblom L(1), Söderkvist P(25), \n",
      "Knight A(1), Eriksson P(26), Lindblad-Toh K(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medical Sciences.\n",
      "(2)Science for Life Laboratory, Department of Medical Biochemistry and \n",
      "Microbiology, Uppsala University, Uppsala, Sweden.\n",
      "(3)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\n",
      "(4)Department of Biochemistry and Biophysics, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Stockholm University, \n",
      "Stockholm.\n",
      "(5)Department of Cell and Molecular Biology, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala.\n",
      "(6)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(7)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \n",
      "Diseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\n",
      "(8)Department of Rheumatology, Oslo University Hospital.\n",
      "(9)Department of Rheumatology, Martina Hansens Hospital, Oslo, Norway.\n",
      "(10)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \n",
      "Stockholm.\n",
      "(11)Unit of Rheumatology, Karolinska University Hospital, Stockholm.\n",
      "(12)Department of Health, Medicine and Caring Sciences, Linköping University, \n",
      "Linköping.\n",
      "(13)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \n",
      "Karolinska Institutet, Stockholm.\n",
      "(14)Department of Clinical Sciences, Division of Nephrology, Lund University and \n",
      "Skåne University Hospital.\n",
      "(15)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \n",
      "University Hospital, Lund University, Lund, Sweden.\n",
      "(16)Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "(17)Center for Clinical Research Dalarna, Uppsala University, Uppsala.\n",
      "(18)Department of Rheumatology and Inflammation Research, Institution of \n",
      "Medicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg.\n",
      "(19)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \n",
      "University Hospital.\n",
      "(20)Department of Molecular and Clinical Medicine/Nephrology, Institute of \n",
      "Medicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg.\n",
      "(21)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\n",
      "(22)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \n",
      "University Hospital, Stavanger.\n",
      "(23)Department of Clinical Science.\n",
      "(24)Broegelmann Research Laboratory, Department of Clinical Science, University \n",
      "of Bergen, Bergen, Norway.\n",
      "(25)Department of Biomedical and Clinical Sciences, Division of Cell Biology.\n",
      "(26)Department of Biomedical and Clinical Sciences, Division of Inflammation and \n",
      "Infection, Linköping University, Linköping, Sweden.\n",
      "\n",
      "OBJECTIVE: To identify and characterize genetic loci associated with the risk of \n",
      "developing ANCA-associated vasculitides (AAV).\n",
      "METHODS: Genetic association analyses were performed after Illumina sequencing \n",
      "of 1853 genes and subsequent replication with genotyping of selected single \n",
      "nucleotide polymorphisms in a total cohort of 1110 Scandinavian cases with \n",
      "granulomatosis with polyangiitis or microscopic polyangiitis, and 1589 controls. \n",
      "A novel AAV-associated single nucleotide polymorphism was analysed for \n",
      "allele-specific effects on gene expression using luciferase reporter assay.\n",
      "RESULTS: PR3-ANCA+ AAV was significantly associated with two independent loci in \n",
      "the HLA-DPB1/HLA-DPA1 region [rs1042335, P = 6.3 × 10-61, odds ratio (OR) 0.10; \n",
      "rs9277341, P = 1.5 × 10-44, OR 0.22] and with rs28929474 in the SERPINA1 gene \n",
      "(P = 2.7 × 10-10, OR 2.9). MPO-ANCA+ AAV was significantly associated with the \n",
      "HLA-DQB1/HLA-DQA2 locus (rs9274619, P = 5.4 × 10-25, OR 3.7) and with a rare \n",
      "variant in the BACH2 gene (rs78275221, P = 7.9 × 10-7, OR 3.0), the latter a \n",
      "novel susceptibility locus for MPO-ANCA+ granulomatosis with \n",
      "polyangiitis/microscopic polyangiitis. The rs78275221-A risk allele reduced \n",
      "luciferase gene expression in endothelial cells, specifically, as compared with \n",
      "the non-risk allele.\n",
      "CONCLUSION: We identified a novel susceptibility locus for MPO-ANCA+ AAV and \n",
      "propose that the associated variant is of mechanistic importance, exerting a \n",
      "regulatory function on gene expression in specific cell types.\n",
      "\n",
      "© The Author(s) 2021. Published by Oxford University Press on behalf of the \n",
      "British Society for Rheumatology.\n",
      "\n",
      "DOI: 10.1093/rheumatology/keab912\n",
      "PMCID: PMC9348767\n",
      "PMID: 34888651 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "10. Neurol Genet. 2021 Jun 2;7(3):e592. doi: 10.1212/NXG.0000000000000592. \n",
      "eCollection 2021 Jun.\n",
      "\n",
      "Association of HLA-DQA2 and HLA-B With Moyamoya Disease in the Chinese Han \n",
      "Population.\n",
      "\n",
      "Wan J(1), Ling W(1), Zhengshan Z(1), Xianbo Z(1), Lian D(1), Kai W(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neurology (J.W.), the First Affiliated Hospital of Anhui \n",
      "Medical University, the School of Mental Health and Psychological Sciences, \n",
      "Anhui Medical University, Hefei, Anhui Province, Department of Neurology (J.W.), \n",
      "Drum Tower Hospital, Medical School and the State Key Laboratory of \n",
      "Pharmaceutical Biotechnology, Nanjing University; the School of Mental Health \n",
      "and Psychological Sciences (W.L.), Anhui Medical University, Anhui Province, \n",
      "Institute of Artificial Intelligence (W.L.), Hefei Comprehensive National \n",
      "Science Center. Anhui Province Key Laboratory of Cognition and Neuropsychiatric \n",
      "Disorders (W.L.), Hefei; Collaborative Innovation Center of Neuropsychiatric \n",
      "Disorders and Mental Health (W.L), Anhui Province; Department of Neurosurgery \n",
      "(Z.Z.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \n",
      "Hospital of PLA), Beijing; Department of Dermatology (Z.X.), the First \n",
      "Affiliated Hospital, Anhui Medical University, Hefei, Anhui Province; Key \n",
      "Laboratory of Dermatology (Z.X.), Anhui Medical University, Ministry of \n",
      "Education, Hefei, Anhui Province; State Key Lab of Dermatology Incubation Center \n",
      "(Z.X.), Anhui Medical University, Hefei, China; Department of Neurosurgery \n",
      "(D.L.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \n",
      "Hospital of PLA), Beijing; the School of Mental Health and Psychological \n",
      "Sciences (W.K.), Anhui Medical University, Anhui Province; Institute of \n",
      "Artificial Intelligence (W.K.), Hefei Comprehensive National Science Center; \n",
      "Anhui Province Key Laboratory of Cognition and Neuropsychiatric Disorders \n",
      "(W.K.), Hefei, Collaborative Innovation Center of Neuropsychiatric Disorders and \n",
      "Mental Health (W.K.), Anhui Province, China.\n",
      "\n",
      "OBJECTIVE: An HLA imputation was conducted to explore the relationship between \n",
      "HLA and patients with moyamoya disease (MMD) in the Chinese Han population.\n",
      "METHODS: In this study, we performed an association analysis of the major \n",
      "histocompatibility complex region in 2,786 individuals of Chinese Han ancestry \n",
      "(2,031 controls and 755 patients with MMD), through a widely used HLA imputation \n",
      "method.\n",
      "RESULTS: We identified that the variant rs3129731 (odds ratio [OR] = 1.79, p = \n",
      "3.69 × 10-16) located between the MTCO3P1 and HLA-DQA2 is a major genetic risk \n",
      "factor for MMD. In addition to this variant, found in the conditional \n",
      "association analysis, we also detected another independent signal, rs1071817 (OR \n",
      "= 0.62, p = 1.20 × 10-11), in HLA-B.\n",
      "CONCLUSIONS: Our research suggests that the genetic polymorphism of HLA-DQA2 and \n",
      "HLA-B could be a genetic predisposing factor for MMD in Chinese Han. This may \n",
      "provide some evidence for further HLA-related studies of patients with MMD of \n",
      "Chinese Han ethnicity and indicates that MMD is an immune-related disease.\n",
      "\n",
      "Copyright © 2021 The Author(s). Published by Wolters Kluwer Health, Inc. on \n",
      "behalf of the American Academy of Neurology.\n",
      "\n",
      "DOI: 10.1212/NXG.0000000000000592\n",
      "PMCID: PMC8176556\n",
      "PMID: 34095496\n"
     ]
    }
   ],
   "source": [
    "print(teste)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
