{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = 'pg21019@alunos.uminho.pt'\n",
    "\n",
    "from scripts import pesquisa_ncbi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 90 resultados em pubmed. Irão ser processados 90 resultados.\n",
      "\n",
      "1. Hepatol Int. 2023 Nov 11. doi: 10.1007/s12072-023-10608-8. Online ahead of \n",
      "print.\n",
      "\n",
      "Proteome-wide Mendelian randomization highlights AIF1 and HLA-DQA2 as targets \n",
      "for primary sclerosing cholangitis.\n",
      "\n",
      "Chen L(1), Zhao Y(1), Li M(1), Lv G(2).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \n",
      "First Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \n",
      "Jilin, China.\n",
      "(2)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \n",
      "First Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \n",
      "Jilin, China. lvgy@jlu.edu.cn.\n",
      "\n",
      "BACKGROUND: Primary sclerosing cholangitis (PSC) is a kind of cholestatic liver \n",
      "disease without effective therapies and its pathogenesis is largely unknown.\n",
      "METHODS: We performed the proteome-wide Mendelian randomization (MR) design to \n",
      "estimate the causal associations of protein levels with PSC risk. Therein, \n",
      "genetic associations with 4,907 plasma protein levels were extracted from a \n",
      "proteome-wide genome-wide association study (GWAS) with 35,559 individuals and \n",
      "those with PSC were obtained from the International PSC Study Group (2,871 cases \n",
      "and 12,019 controls) and the FinnGen study (1,491 cases and 301,383 controls). \n",
      "The colocalization analysis was performed to detect causal variants shared by \n",
      "proteins and PSC. The identified proteins were further enriched in pathways and \n",
      "diseases. A phenome-wide association screening was performed and potential drugs \n",
      "were assessed as well.\n",
      "RESULTS: The results indicated that genetically predicted plasma levels of 14 \n",
      "proteins were positively associated with an increased risk of PSC and 8 proteins \n",
      "were inversely associated with PSC risk in both PSC GWAS data sets, and they all \n",
      "survived in sensitivity analyses. The colocalization indicated that AIF1 \n",
      "(allograft inflammatory factor 1) and HLA-DQA2 (major histocompatibility \n",
      "complex, class II, DQ alpha 2) were shared proteins with PSC, and they should be \n",
      "direct targets for PSC. The phenome-wide screening suggested that variants \n",
      "located at AIF1 or HLA-DQA2 region were closely associated with several \n",
      "autoimmune diseases, such as rheumatoid arthritis, implicating the shared \n",
      "pathogenesis among them.\n",
      "CONCLUSIONS: Our study highly pinpointed two candidate targets (AIF1 and \n",
      "HLA-DQA2) for PSC.\n",
      "\n",
      "© 2023. Asian Pacific Association for the Study of the Liver.\n",
      "\n",
      "DOI: 10.1007/s12072-023-10608-8\n",
      "PMID: 37950809\n",
      "\n",
      "\n",
      "2. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\n",
      " 2023 Oct 10.\n",
      "\n",
      "Genome-wide identification of m(6)A-associated single nucleotide polymorphisms \n",
      "in complex diseases of nervous system.\n",
      "\n",
      "Guo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \n",
      "Guo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\n",
      "\n",
      "Author information:\n",
      "(1)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China.\n",
      "(2)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China.\n",
      "(3)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "jiangyongshuai@hrbmu.edu.cn.\n",
      "(4)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "zhangmingming@hrbmu.edu.cn.\n",
      "\n",
      "N6-methyladenosine (m6A) is one of the most abundant chemical modifications on \n",
      "RNA and can affect the occurrence and development of diseases. Some studies have \n",
      "shown that the expressions of some m6A-related genes are significantly regulated \n",
      "by single nucleotide variants (SNV). However, the function of m6A-associated \n",
      "single nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \n",
      "(MS), Alzheimer's disease (AD) and Parkinson's disease (PD). Here, we identified \n",
      "the disease-associated m6A-SNPs by integrating genome-wide association study \n",
      "(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \n",
      "between these identified m6A-SNPs and their target genes in eQTL analysis and \n",
      "gene differential expression analysis. Finally, 26 genes corresponding to 20 \n",
      "m6A-SNPs with eQTL signals were identified and differentially expressed \n",
      "(P < 0.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P < 1e-04) were \n",
      "differentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \n",
      "expression of 31 genes were identified. There were 5 HLA genes with eQTL signals \n",
      "(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \n",
      "three diseases. In summary, our study provided new insights into understanding \n",
      "the potential roles of these m6A-SNPs in disease pathogenesis as well as \n",
      "therapeutic target.\n",
      "\n",
      "Copyright © 2023 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.neulet.2023.137513\n",
      "PMID: 37827449 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "3. Br J Cancer. 2023 Oct;129(9):1510-1515. doi: 10.1038/s41416-023-02419-3. Epub \n",
      "2023 Sep 7.\n",
      "\n",
      "Integrating genomics and proteomics data to identify candidate plasma biomarkers \n",
      "for lung cancer risk among European descendants.\n",
      "\n",
      "Yang Y(#)(1), Xu S(#)(2), Jia G(2), Yuan F(2), Ping J(2), Guo X(2), Tao R(3)(4), \n",
      "Shu XO(2), Zheng W(2), Long J(2), Cai Q(5).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Public Health Genomics, Department of Public Health Sciences, UVA \n",
      "Comprehensive Cancer Center, School of Medicine, University of Virginia, \n",
      "Charlottesville, VA, USA. vta8we@virginia.edu.\n",
      "(2)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(3)Department of Biostatistics, Vanderbilt University Medical Center, Nashville, \n",
      "TN, USA.\n",
      "(4)Vanderbilt Genetics Institute, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(5)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA. qiuyin.cai@vanderbilt.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: Plasma proteins are potential biomarkers for complex diseases. We \n",
      "aimed to identify plasma protein biomarkers for lung cancer.\n",
      "METHODS: We investigated genetically predicted plasma levels of 1130 proteins in \n",
      "association with lung cancer risk among 29,266 cases and 56,450 controls of \n",
      "European descent. For proteins significantly associated with lung cancer risk, \n",
      "we evaluated associations of genetically predicted expression of their coding \n",
      "genes with the risk of lung cancer.\n",
      "RESULTS: Nine proteins were identified with genetically predicted plasma levels \n",
      "significantly associated with overall lung cancer risk at a false discovery rate \n",
      "(FDR) of <0.05. Proteins C2, MICA, AIF1, and CTSH were associated with increased \n",
      "lung cancer risk, while proteins SFTPB, HLA-DQA2, MICB, NRP1, and GMFG were \n",
      "associated with decreased lung cancer risk. Stratified analyses by histological \n",
      "types revealed the cross-subtype consistency of these nine associations and \n",
      "identified an additional protein, ICAM5, significantly associated with lung \n",
      "adenocarcinoma risk (FDR < 0.05). Coding genes of NRP1 and ICAM5 proteins are \n",
      "located at two loci that have never been reported by previous GWAS. Genetically \n",
      "predicted blood levels of genes C2, AIF1, and CTSH were associated with lung \n",
      "cancer risk, in directions consistent with those shown in protein-level \n",
      "analyses.\n",
      "CONCLUSION: Identification of novel plasma protein biomarkers provided new \n",
      "insights into the biology of lung cancer.\n",
      "\n",
      "© 2023. The Author(s), under exclusive licence to Springer Nature Limited.\n",
      "\n",
      "DOI: 10.1038/s41416-023-02419-3\n",
      "PMCID: PMC10628278\n",
      "PMID: 37679517 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "4. Front Oncol. 2023 Jun 23;13:1150098. doi: 10.3389/fonc.2023.1150098.\n",
      "eCollection  2023.\n",
      "\n",
      "Immune microenvironment analysis and novel biomarkers of early-stage lung \n",
      "adenocarcinoma evolution.\n",
      "\n",
      "Liu J(1), Ji Y(2), Weng X(3), Shao W(2), Zhao J(2), Chen H(4), Shen L(4), Wang \n",
      "F(4), Meng Q(4), Wu X(4), Wang X(4), Ou Q(4), Ke H(3).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Chemotherapy, Affiliated Hospital of Nantong University, \n",
      "Nantong, China.\n",
      "(2)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \n",
      "Medical School of Nantong University, Nantong, China.\n",
      "(3)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \n",
      "Nantong, China.\n",
      "(4)Geneseeq Research Institute, Nanjing Geneseeq Technology Inc., \n",
      "Nanjing, China.\n",
      "\n",
      "BACKGROUND: Lung cancer is the deadliest and most diagnosed type of cancer \n",
      "worldwide. The 5-year survival rate of lung adenocarcinoma (LUAD) dropped \n",
      "significantly when tumor stages advanced. Patients who received surgically \n",
      "resecting at the pre-invasive stage had a 5-year survival rate of nearly 100%. \n",
      "However, the study on the differences in gene expression profiles and immune \n",
      "microenvironment among pre-invasive LUAD patients is still lacking.\n",
      "METHODS: In this study, the gene expression profiles of three pre-invasive LUAD \n",
      "stages were compared using the RNA-sequencing data of 10 adenocarcinoma in situ \n",
      "(AIS) samples, 12 minimally invasive adenocarcinoma (MIA) samples, and 10 \n",
      "invasive adenocarcinoma (IAC) samples.\n",
      "RESULTS: The high expression levels of PTGFRN (Hazard Ratio [HR] = 1.45; 95% \n",
      "Confidence Interval [CI]: 1.08-1.94; log-rank P = 0.013) and SPP1 (HR = 1.44; \n",
      "95% CI: 1.07-1.93; log-rank P = 0.015) were identified to be associated with \n",
      "LUAD prognosis. Moreover, the early LUAD invasion was accompanied by the \n",
      "enhancement of antigen presentation ability, reflected by the increase of \n",
      "myeloid dendritic cells infiltration rate (Cuzick test P < 0.01) and the \n",
      "upregulation of seven important genes participating in the antigen presentation, \n",
      "including HLA-A (Cuzick test P = 0.03), MICA (Cuzick test P = 0.01), MICB \n",
      "(Cuzick test P = 0.01), HLA-DPA1 (Cuzick test P = 0.04), HLA-DQA2 (Cuzick test P \n",
      "< 0.01), HLA-DQB1 (Cuzick test P = 0.03), and HLA-DQB2 (Cuzick test P < 0.01). \n",
      "However, the tumor-killing ability of the immune system was inhibited during \n",
      "this process, as there were no rising cytotoxic T cell activity (Cuzick test P = \n",
      "0.20) and no increasing expression in genes encoding cytotoxic proteins.\n",
      "CONCLUSION: In all, our research elucidated the changes in the immune \n",
      "microenvironment during early-stage LUAD evolution and may provide a theoretical \n",
      "basis for developing novel early-stage lung cancer therapeutic targets.\n",
      "\n",
      "Copyright © 2023 Liu, Ji, Weng, Shao, Zhao, Chen, Shen, Wang, Meng, Wu, Wang, Ou \n",
      "and Ke.\n",
      "\n",
      "DOI: 10.3389/fonc.2023.1150098\n",
      "PMCID: PMC10328385\n",
      "PMID: 37427097\n",
      "\n",
      "Conflict of interest statement: HC, LS, FW, QM, XWU, XWA, and QO are employees \n",
      "of Nanjing Geneseeq Technology Inc., China. The remaining authors declare that \n",
      "the research was conducted in the absence of any commercial or financial \n",
      "relationships that could be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "5. Hereditas. 2023 May 31;160(1):27. doi: 10.1186/s41065-023-00289-6.\n",
      "\n",
      "NPM3 as a novel oncogenic factor and poor prognostic marker contributes to cell \n",
      "proliferation and migration in lung adenocarcinoma.\n",
      "\n",
      "Wei S(1)(2), Xing J(2)(3), Lu K(4), Wang K(1), Yu W(5).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Respiratory and Critical Care Medicine, The Fourth Affiliated \n",
      "Hospital, School of Medicine, Zhejiang University, Yiwu, 322000, Zhejiang, \n",
      "People's Republic of China.\n",
      "(2)Department of Respiratory and Critical Care Medicine, The Affiliated People's \n",
      "Hospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \n",
      "Baizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China.\n",
      "(3)Ningbo University School of Medicine, Zhejiang Province, Ningbo, People's \n",
      "Republic of China.\n",
      "(4)Department of Urology, The Affiliated First Hospital of Ningbo University \n",
      "(Ningbo First Hospital), No.59, Liuting Street, Ningbo, 315010, Zhejiang, \n",
      "People's Republic of China.\n",
      "(5)Department of Respiratory and Critical Care Medicine, The Affiliated People's \n",
      "Hospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \n",
      "Baizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China. \n",
      "nbyuwanjun@163.com.\n",
      "\n",
      "BACKGROUND: Lung cancer is the leading cause of cancer-related deaths worldwide, \n",
      "and despite recent advances in targeted therapies and immunotherapies, the \n",
      "clinical benefit remains limited. Therefore, there is an urgent need to further \n",
      "investigate the molecular mechanisms underlying lung cancer. The aim of this \n",
      "study was to investigate the expression and function of NPM3 in the tumor \n",
      "microenvironment of lung adenocarcinoma (LUAD).\n",
      "METHODS: We utilized bioinformatics tools and databases, including UALCAN, \n",
      "GEPIA2, HPA, and Sangerbox, to analyze NPM3 expression in LUAD samples and its \n",
      "association with prognosis and mutational landscape. NPM3 expression in various \n",
      "cell types was assessed at the single cell level using the TISCH database. We \n",
      "also used algorithms such as TIMER and EPIC to explore the crosstalk between \n",
      "NPM3 expression and immune features. KEGG enrichment analysis was performed to \n",
      "identify potential signaling pathways of NPM3. Finally, we employed siRNA \n",
      "knockdown strategy to investigate the effect of NPM3 on LUAD cell proliferation \n",
      "and migration in vitro.\n",
      "RESULTS: NPM3 was significantly upregulated in LUAD tissues and was strongly \n",
      "associated with poor prognosis and TP53 gene mutations. Single-cell sequencing \n",
      "analysis revealed that NPM3 was expressed in immune cells (dendritic cells and \n",
      "monocytes/macrophages) in the tumor microenvironment. Moreover, NPM3 expression \n",
      "was negatively associated with immune B cell and CD4 T cell infiltration, as \n",
      "well as with several immune-related genes (including CCL22, CXCR2, CX3CR1, CCR6, \n",
      "HLA-DOA, HLA-DQA2). KEGG enrichment analysis indicated that NPM3 expression was \n",
      "associated with cell cycle, CAMs, and NSCLC pathway genes. Finally, in vitro \n",
      "experiments showed that NPM3 knockdown inhibited LUAD cell proliferation and \n",
      "migration in NCI-H1299 and SPC-A1 cells, and suppressed the expression of CCNA2 \n",
      "and MAD2L1.\n",
      "CONCLUSION: Elevated NPM3 expression predicts poor clinical outcome and an \n",
      "immunosuppressive microenvironment in LUAD tissues. NPM3 promotes LUAD \n",
      "progression by promoting cell proliferation and migration, and targeting NPM3 \n",
      "may represent a novel therapeutic strategy for LUAD.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s41065-023-00289-6\n",
      "PMCID: PMC10230701\n",
      "PMID: 37254219 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors claim that there is no possible \n",
      "conflict of interest.\n",
      "\n",
      "\n",
      "6. Rheumatology (Oxford). 2023 Sep 1;62(9):3213-3218. doi: \n",
      "10.1093/rheumatology/kead152.\n",
      "\n",
      "Stratified genetic analysis reveals sex differences in MPO-ANCA-associated \n",
      "vasculitis.\n",
      "\n",
      "Ekman D(1), Sennblad B(2), Knight A(3), Karlsson Å(4), Rantapää-Dahlqvist S(5), \n",
      "Berglin E(5), Stegmayr B(5), Baslund B(6), Palm Ø(7), Haukeland H(8), Gunnarsson \n",
      "I(9)(10), Bruchfeld A(11)(12), Segelmark M(13), Ohlsson S(13), Mohammad \n",
      "AJ(14)(15), Svärd A(16), Pullerits R(17)(18), Herlitz H(19), Söderbergh A(20), \n",
      "Omdal R(21)(22), Jonsson R(23), Rönnblom L(3), Eriksson P(24), Lindblad-Toh \n",
      "K(4)(25), Dahlqvist J(3)(4)(25).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biochemistry and Biophysics, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Stockholm University, \n",
      "Sweden.\n",
      "(2)Department of Cell and Molecular Biology, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala, \n",
      "Sweden.\n",
      "(3)Department of Medical Sciences, Uppsala University, Uppsala, Sweden.\n",
      "(4)Science for Life Laboratory, Department of Medical Biochemistry and \n",
      "Microbiology, Uppsala University, Uppsala, Sweden.\n",
      "(5)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(6)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \n",
      "Diseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\n",
      "(7)Department of Rheumatology, Oslo University Hospital, Oslo, Norway.\n",
      "(8)Department of Rheumatology, Martina Hansens Hospital, Gjettum, Norway.\n",
      "(9)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \n",
      "Stockholm, Sweden.\n",
      "(10)Unit of Rheumatology, Karolinska University Hospital, Stockholm, Sweden.\n",
      "(11)Department of Health, Medicine and Caring Sciences, Linköping University, \n",
      "Linköping, Sweden.\n",
      "(12)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \n",
      "Karolinska Institutet, Stockholm, Sweden.\n",
      "(13)Department of Clinical Sciences, Division of Nephrology, Lund University and \n",
      "Skåne University Hospital, Lund, Sweden.\n",
      "(14)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \n",
      "University Hospital, Lund University, Lund, Sweden.\n",
      "(15)Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "(16)Center for Clinical Research Dalarna, Uppsala University, Uppsala, Sweden.\n",
      "(17)Department of Rheumatology and Inflammation Research, Institution of \n",
      "Medicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden.\n",
      "(18)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \n",
      "University Hospital, Gothenburg, Sweden.\n",
      "(19)Department of Molecular and Clinical Medicine/Nephrology, Institute of \n",
      "Medicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden.\n",
      "(20)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\n",
      "(21)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \n",
      "University Hospital, Stavanger, Norway.\n",
      "(22)Department of Clinical Science, University of Bergen, Bergen, Norway.\n",
      "(23)Broegelmann Research Laboratory, Department of Clinical Science, University \n",
      "of Bergen, Bergen, Norway.\n",
      "(24)Department of Biomedical and Clinical Sciences, Division of Inflammation and \n",
      "Infection, Linköping University, Linköping, Sweden.\n",
      "(25)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\n",
      "\n",
      "OBJECTIVE: To identify and genetically characterize subgroups of patients with \n",
      "ANCA-associated vasculitides (AAV) based on sex and ANCA subtype.\n",
      "METHODS: A previously established SNP dataset derived from DNA sequencing of \n",
      "1853 genes and genotyping of 1088 Scandinavian cases with AAV and 1589 controls \n",
      "was stratified for sex and ANCA subtype and analysed for association with five \n",
      "top AAV SNPs. rs9274619, a lead variant at the HLA-DQB1/HLA-DQA2 locus \n",
      "previously associated with AAV positive for myeloperoxidase (MPO)-ANCA, was \n",
      "analysed for association with the cumulative disease involvement of ten \n",
      "different organ systems.\n",
      "RESULTS: rs9274619 showed a significantly stronger association to \n",
      "MPO-ANCA-positive females than males [P = 2.0 × 10-4, OR = 2.3 (95% CI 1.5, \n",
      "3.5)], whereas proteinase 3 (PR3)-ANCA-associated variants rs1042335, rs9277341 \n",
      "(HLA-DPB1/A1) and rs28929474 (SERPINA1) were equally associated with females and \n",
      "males with PR3-ANCA. In MPO-ANCA-positive cases, carriers of the rs9274619 risk \n",
      "allele were more prone to disease engagement of eyes [P = 0.021, OR = 11 (95% CI \n",
      "2.2, 205)] but less prone to pulmonary involvement [P = 0.026, OR = 0.52 (95% CI \n",
      "0.30, 0.92)]. Moreover, AAV with both MPO-ANCA and PR3-ANCA was associated with \n",
      "the PR3-ANCA lead SNP rs1042335 [P = 0.0015, OR = 0.091 (95% CI 0.0022, 0.55)] \n",
      "but not with rs9274619.\n",
      "CONCLUSIONS: Females and males with MPO-ANCA-positive AAV differ in genetic \n",
      "predisposition to disease, suggesting at least partially distinct disease \n",
      "mechanisms between the sexes. Double ANCA-positive AAV cases are genetically \n",
      "similar to PR3-ANCA-positive cases, providing clues to the clinical follow-up \n",
      "and treatment of these patients.\n",
      "\n",
      "© The Author(s) 2023. Published by Oxford University Press on behalf of the \n",
      "British Society for Rheumatology.\n",
      "\n",
      "DOI: 10.1093/rheumatology/kead152\n",
      "PMCID: PMC10473270\n",
      "PMID: 37004177 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. BMC Genomics. 2023 Mar 27;24(1):153. doi: 10.1186/s12864-023-09227-2.\n",
      "\n",
      "Genome-wide identification of RNA modification-related single nucleotide \n",
      "polymorphisms associated with rheumatoid arthritis.\n",
      "\n",
      "Wang M(#)(1)(2), Wu J(#)(1)(2), Lei S(1)(2), Mo X(3)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China.\n",
      "(2)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China.\n",
      "(3)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China. xbmo@suda.edu.cn.\n",
      "(4)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China. xbmo@suda.edu.cn.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: RNA modification plays important roles in many biological processes, \n",
      "such as gene expression control. The aim of this study was to identify single \n",
      "nucleotide polymorphisms related to RNA modification (RNAm-SNPs) for rheumatoid \n",
      "arthritis (RA) as putative functional variants.\n",
      "METHODS: We examined the association of RNAm-SNPs with RA in summary data from a \n",
      "genome-wide association study of 19,234 RA cases and 61,565 controls. We \n",
      "performed eQTL and pQTL analyses for the RNAm-SNPs to find associated gene \n",
      "expression and protein levels. Furthermore, we examined the associations of gene \n",
      "expression and circulating protein levels with RA using two-sample Mendelian \n",
      "randomization analysis methods.\n",
      "RESULTS: A total of 160 RNAm-SNPs related to m6A, m1A, A-to-I, m7G, m5C, m5U and \n",
      "m6Am modifications were identified to be significantly associated with RA. These \n",
      "RNAm-SNPs were located in 62 protein-coding genes, which were significantly \n",
      "enriched in immune-related pathways. RNAm-SNPs in important RA susceptibility \n",
      "genes, such as PADI2, SPRED2, PLCL2, HLA-A, HLA-B, HLA-DRB1, HLA-DPB1, TRAF1 and \n",
      "TXNDC11, were identified. Most of these RNAm-SNPs showed eQTL effects, and the \n",
      "expression levels of 26 of the modifiable genes (e.g., PADI2, TRAF1, HLA-A, \n",
      "HLA-DRB1, HLA-DPB1 and HLA-B) in blood cells were associated with RA. \n",
      "Circulating protein levels, such as CFB, GZMA, HLA-DQA2, IL21, LRPAP1 and TFF3, \n",
      "were affected by RNAm-SNPs and were associated with RA.\n",
      "CONCLUSION: The present study identified RNAm-SNPs in the reported RA \n",
      "susceptibility genes and suggested that RNAm-SNPs may affect RA risk by \n",
      "affecting the expression levels of corresponding genes and proteins.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12864-023-09227-2\n",
      "PMCID: PMC10045113\n",
      "PMID: 36973646 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "8. Yi Chuan. 2023 Jan 20;45(1):52-66. doi: 10.16288/j.yczz.22-293.\n",
      "\n",
      "Application of transcriptome in time analysis and donor characterization in \n",
      "blood samples.\n",
      "\n",
      "Zhang J(1)(2), Liu KH(2), Zhang Y(2), Hao JP(2), Zhang GF(2), Xu XY(2), Chang \n",
      "J(2), Liu XP(3), Yang XY(2), Ye J(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)1. People's Public Security University of China, Beijing 100038, China.\n",
      "(2)2. Institute of Forensic Science Ministry of Public Security, Beijing 100038, \n",
      "China.\n",
      "(3)3. Ruijin Public Security Bureau, Ruijin 342500, China.\n",
      "\n",
      "As an effective supplement to the current forensic DNA typing and one of the \n",
      "research hotpots in forensic science, the in-depth mining and characterization \n",
      "of biological evidence can provide rich and reliable clues for case \n",
      "investigation. In this study, the time-dependent variations of transcriptome \n",
      "were confirmed in in vitro blood samples within 0-168 days and a random forest \n",
      "model was established to realize the classification of blood samples with \n",
      "different TSD (time since deposition). Meanwhile, significant differences were \n",
      "observed in the transcripts of blood samples with different smoking habits and \n",
      "genders within a certain time period. HLA-DRB1, HLA-DQB1 and HLA-DQA2 were \n",
      "identified as markers for smoking habit identification, while the transcripts \n",
      "for RPS4Y1 and EIF1AY from the non-recombining region of the Y chromosome (NRY) \n",
      "were identified as markers for male sex identification. Thus, this study \n",
      "provides a theoretical foundation and experimental strategy for establishing a \n",
      "transcriptome-based method for characterizing blood sample retention time and \n",
      "donor characteristics in the field of forensic investigation.\n",
      "\n",
      "Publisher: \n",
      "案件现场生物物证信息深度挖掘与刻画，可以为案件侦查、涉案人员查找提供丰富可靠线索，是当前法医DNA检验的有效补充，也是国内外法庭科学的研究热点之一。本研究以血液样本为研究对象，证实了0~168天内离体血液样本转录组变化的时间相关性，并建立随机森林模型实现不同离体时间血液样本分类。同时，进一步证实相同离体时间段内，不同吸烟习惯和不同性别供体来源的血液样本转录本具有显著差异，HLA-DRB1、HLA-DQB1和HLA-DQA2可以作为供体吸烟习惯判别标志，Y染色体非重组区(non-recombining \n",
      "Y, \n",
      "NRY)的转录本RPS4Y1和EIF1AY可以作为供体性别特征判别标志。本研究为法庭科学领域建立基于转录组分析的血液样本遗留时间和供体特征刻画方法提供了理论基础和实验依据。.\n",
      "\n",
      "DOI: 10.16288/j.yczz.22-293\n",
      "PMID: 36927638 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "9. Pediatr Rheumatol Online J. 2023 Feb 15;21(1):17. doi: \n",
      "10.1186/s12969-023-00787-x.\n",
      "\n",
      "Profiling the peripheral immune response to ex vivo TNF stimulation in untreated \n",
      "juvenile idiopathic arthritis using single cell RNA sequencing.\n",
      "\n",
      "Imbach KJ(#)(1), Treadway NJ(#)(2), Prahalad V(2), Kosters A(3), Arafat D(1), \n",
      "Duan M(1), Gergely T(2), Ponder LA(4), Chandrakasan S(4)(5), Ghosn EEB(2)(3)(4), \n",
      "Prahalad S(#)(6)(7)(8), Gibson G(#)(1)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Integrative Genomics, School of Biological Sciences, Georgia \n",
      "Institute of Technology, Atlanta, GA, 30332, USA.\n",
      "(2)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \n",
      "30223, USA.\n",
      "(3)Lowance Center for Human Immunology, Division of Immunology, Department of \n",
      "Medicine, Emory University School of Medicine, Atlanta, GA, 30223, USA.\n",
      "(4)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \n",
      "Atlanta, GA, 30223, USA.\n",
      "(5)Aflac Cancer and Blood Disorders Center, Department of Pediatrics, Children's \n",
      "Healthcare of Atlanta, Emory University School of Medicine, Atlanta, GA, 30223, \n",
      "USA.\n",
      "(6)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \n",
      "30223, USA. sprahal@emory.edu.\n",
      "(7)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \n",
      "Atlanta, GA, 30223, USA. sprahal@emory.edu.\n",
      "(8)Department of Human Genetics, Emory University School of Medicine, Atlanta, \n",
      "GA, 30223, USA. sprahal@emory.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: Juvenile Idiopathic Arthritis (JIA) is an autoimmune disease with a \n",
      "heterogenous clinical presentation and unpredictable response to available \n",
      "therapies. This personalized transcriptomics study sought proof-of-concept for \n",
      "single-cell RNA sequencing to characterize patient-specific immune profiles.\n",
      "METHODS: Whole blood samples from six untreated children, newly diagnosed with \n",
      "JIA, and two healthy controls were cultured for 24 h with or without ex vivo TNF \n",
      "stimulation and subjected to scRNAseq to examine cellular populations and \n",
      "transcript expression in PBMCs. A novel analytical pipeline, scPool, was \n",
      "developed wherein cells are first pooled into pseudocells prior to expression \n",
      "analysis, facilitating variance partitioning of the effects of TNF stimulus, JIA \n",
      "disease status, and individual donor.\n",
      "RESULTS: Seventeen robust immune cell-types were identified, the abundance of \n",
      "which was significantly affected by TNF stimulus, which resulted in notable \n",
      "elevation of memory CD8 + T-cells and NK56 cells, but down-regulation of naïve \n",
      "B-cell proportions. Memory CD8 + and CD4 + T-cells were also both reduced in the \n",
      "JIA cases relative to two controls. Significant differential expression \n",
      "responses to TNF stimulus were also characterized, with monocytes showing more \n",
      "transcriptional shifts than T-lymphocyte subsets, while the B-cell response was \n",
      "more limited. We also show that donor variability exceeds the small degree of \n",
      "possible intrinsic differentiation between JIA and control profiles. An \n",
      "incidental finding of interest was association of HLA-DQA2 and HLA-DRB5 \n",
      "expression with JIA status.\n",
      "CONCLUSIONS: These results support the development of personalized \n",
      "immune-profiling combined with ex-vivo immune stimulation for evaluation of \n",
      "patient-specific modes of immune cell activity in autoimmune rheumatic disease.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12969-023-00787-x\n",
      "PMCID: PMC9929251\n",
      "PMID: 36793127 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Dr. Prahalad has served on a macrophage \n",
      "activation syndrome adjudication committee for Novartis, but this had no bearing \n",
      "on the work described in this manuscript. None of the other authors have any \n",
      "competing interests in the manuscript.\n",
      "\n",
      "\n",
      "10. Epilepsia. 2023 Jan;64(1):218-230. doi: 10.1111/epi.17457. Epub 2022 Nov 16.\n",
      "\n",
      "Brain molecular mechanisms in Rasmussen encephalitis.\n",
      "\n",
      "Leitner DF(1)(2), Lin Z(3), Sawaged Z(3), Kanshin E(4), Friedman D(1), Devore \n",
      "S(1), Ueberheide B(2)(4)(5), Chang JW(6), Mathern GW(6), Anink JJ(7), Aronica \n",
      "E(7)(8), Wisniewski T(2)(9)(10), Devinsky O(1).\n",
      "\n",
      "Author information:\n",
      "(1)Comprehensive Epilepsy Center, NYU Grossman School of Medicine, New York, New \n",
      "York, USA.\n",
      "(2)Center for Cognitive Neurology, Department of Neurology, NYU Grossman School \n",
      "of Medicine, New York, New York, USA.\n",
      "(3)Applied Bioinformatics Laboratories, NYU Grossman School of Medicine, New \n",
      "York, New York, USA.\n",
      "(4)Proteomics Laboratory, Division of Advanced Research Technologies, NYU \n",
      "Grossman School of Medicine, New York, New York, USA.\n",
      "(5)Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of \n",
      "Medicine, New York, New York, USA.\n",
      "(6)Departments of Neurosurgery and Psychiatry, David Geffen School of Medicine, \n",
      "University of California, Los Angeles, Los Angeles, California, USA.\n",
      "(7)Department of (Neuro) Pathology, Amsterdam University Medical Center, \n",
      "University of Amsterdam, Amsterdam Neuroscience, Amsterdam, the Netherlands.\n",
      "(8)Epilepsy Institutes of the Netherlands Foundation, Heemstede, the \n",
      "Netherlands.\n",
      "(9)Department of Pathology, NYU Grossman School of Medicine, New York, New York, \n",
      "USA.\n",
      "(10)Department of Psychiatry, NYU Grossman School of Medicine, New York, New \n",
      "York, USA.\n",
      "\n",
      "OBJECTIVE: This study was undertaken to identify molecular mechanisms in brain \n",
      "tissue of Rasmussen encephalitis (RE) when compared to people with non-RE \n",
      "epilepsy (PWE) and control cases using whole exome sequencing (WES), RNAseq, and \n",
      "proteomics.\n",
      "METHODS: Frozen brain tissue (ages = 2-19 years) was obtained from control \n",
      "autopsy (n = 14), surgical PWE (n = 10), and surgical RE cases (n = 27). We \n",
      "evaluated WES variants in RE associated with epilepsy, seizures, RE, and human \n",
      "leukocyte antigens (HLAs). Differential expression was evaluated by RNAseq \n",
      "(adjusted p < .05) and label-free quantitative mass spectrometry (false \n",
      "discovery rate < 5%) in the three groups.\n",
      "RESULTS: WES revealed no common pathogenic variants in RE, but several rare and \n",
      "likely deleterious variants of unknown significance (VUS; ANGPTL7/MTOR, SCN1A, \n",
      "FCGR3B, MTOR) and more common HLA VUS in >25% of RE cases (HLA-DRB1, HLA-DQA2), \n",
      "all with allele frequency < 5% in the general population. RNAseq in RE versus \n",
      "PWE (1516 altered transcripts) revealed significant activation of crosstalk \n",
      "between dendritic and natural killer cells (p = 7.94 × 10-6 , z = 2.65), in RE \n",
      "versus control (7466 transcripts) neuroinflammation signaling activation \n",
      "(p = 6.31 × 10-13 , z = 5.07), and in PWE versus control (945 transcripts) \n",
      "phagosome formation activation (p = 2.00 × 10-13 , z = 5.61). Proteomics \n",
      "detected fewer altered targets.\n",
      "SIGNIFICANCE: In RE, we identified activated immune signaling pathways and \n",
      "immune cell type annotation enrichment that suggest roles of the innate and \n",
      "adaptive immune responses, as well as HLA variants that may increase \n",
      "vulnerability to RE. Follow-up studies could evaluate cell type density and \n",
      "subregional localization associated with top targets, clinical history \n",
      "(neuropathology, disease duration), and whether modulating crosstalk between \n",
      "dendritic and natural killer cells may limit disease progression.\n",
      "\n",
      "© 2022 International League Against Epilepsy.\n",
      "\n",
      "DOI: 10.1111/epi.17457\n",
      "PMCID: PMC9852002\n",
      "PMID: 36336987 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "11. PLoS One. 2022 Oct 21;17(10):e0275934. doi: 10.1371/journal.pone.0275934. \n",
      "eCollection 2022.\n",
      "\n",
      "Genome-wide discovery for diabetes-dependent triglycerides-associated loci.\n",
      "\n",
      "Selvaraj MS(1)(2)(3), Paruchuri K(1)(2), Haidermota S(1)(2), Bernardo R(1)(2), \n",
      "Rich SS(4), Peloso GM(5), Natarajan P(1)(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Cardiovascular Research Center, Massachusetts General Hospital, Boston, MA, \n",
      "United States of America.\n",
      "(2)Program in Medical and Population Genetics, Broad Institute of Harvard and \n",
      "MIT, Cambridge, MA, United States of America.\n",
      "(3)Department of Medicine, Harvard Medical School, Boston, MA, United States of \n",
      "America.\n",
      "(4)Center for Public Health Genomics, University of Virginia, Charlottesville, \n",
      "VA, United States of America.\n",
      "(5)Department of Biostatistics, Boston University School of Public Health, \n",
      "Boston, MA, United States of America.\n",
      "\n",
      "PURPOSE: We aimed to discover loci associated with triglyceride (TG) levels in \n",
      "the context of type 2 diabetes (T2D). We conducted a genome-wide association \n",
      "study (GWAS) in 424,120 genotyped participants of the UK Biobank (UKB) with T2D \n",
      "status and TG levels.\n",
      "METHODS: We stratified the cohort based on T2D status and conducted association \n",
      "analyses of TG levels for genetic variants with minor allele count (MAC) at \n",
      "least 20 in each stratum. Effect differences of genetic variants by T2D status \n",
      "were determined by Cochran's Q-test and we validated the significantly \n",
      "associated variants in the Mass General Brigham Biobank (MGBB).\n",
      "RESULTS: Among 21,176 T2D and 402,944 non-T2D samples from UKB, stratified GWAS \n",
      "identified 19 and 315 genomic risk loci significantly associated with TG levels, \n",
      "respectively. Only chr6p21.32 exhibited genome-wide significant heterogeneity \n",
      "(I2 = 98.4%; pheterogeneity = 2.1x10-15), with log(TG) effect estimates of \n",
      "-0.066 (95%CI: -0.082, -0.050) and 0.002 (95%CI: -0.002, 0.006) for T2D and \n",
      "non-T2D, respectively. The lead variant rs9274619:A (allele frequency 0.095) is \n",
      "located 2Kb upstream of the HLA-DQB1 gene, between HLA-DQB1 and HLA-DQA2 genes. \n",
      "We replicated this finding among 25,137 participants (6,951 T2D cases) of MGBB \n",
      "(pheterogeneity = 9.5x10-3). Phenome-wide interaction association analyses \n",
      "showed that the lead variant was strongly associated with a concomitant \n",
      "diagnosis of type 1 diabetes (T1D) as well as diabetes-associated complications.\n",
      "CONCLUSION: In conclusion, we identified an intergenic variant near \n",
      "HLA-DQB1/DQA2 significantly associates with decreased triglycerides only among \n",
      "those with T2D and highlights an immune overlap with T1D.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0275934\n",
      "PMCID: PMC9586367\n",
      "PMID: 36269708 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: P.N. reports grants from Amgen, Apple, \n",
      "AstraZeneca, Boston Scientific, and Novartis, personal fees from Apple, \n",
      "AstraZeneca, Blackstone Life Sciences, Foresite Labs, Genentech / Roche, \n",
      "Novartis, and TenSixteen Bio, equity in geneXwell, and TenSixteen Bio, \n",
      "co-founder of TenSixteen Bio, and spousal employment at Vertex, all unrelated to \n",
      "the present work. This does not alter our adherence to PLOS ONE policies on \n",
      "sharing data and materials.\n",
      "\n",
      "\n",
      "12. Medicine (Baltimore). 2022 Oct 7;101(40):e31071. doi: \n",
      "10.1097/MD.0000000000031071.\n",
      "\n",
      "Bioinformatics analysis of the clinical significance of HLA class II in breast \n",
      "cancer.\n",
      "\n",
      "Wu G(1), Xiao G(2), Yan Y(3), Guo C(4), Hu N(5), Shen S(3).\n",
      "\n",
      "Author information:\n",
      "(1)Finance Section, Yuebei People's Hospital, Shantou University, Shaoguan, \n",
      "China.\n",
      "(2)Department of Pathology, Yuebei People's Hospital, Shantou University, \n",
      "Shaoguan, China.\n",
      "(3)Breast Surgery, The Sixth Affiliated Hospital of Guangzhou Medical \n",
      "University, Qingyuan People's Hospital, Qingyuan, China.\n",
      "(4)Department of Radiology, 82 Group Hospital of PLA, Baoding, China.\n",
      "(5)Thoracic surgery, The Sixth Affiliated Hospital of Guangzhou Medical \n",
      "University, Qingyuan People's Hospital, Qingyuan, China.\n",
      "\n",
      "Human leukocyte antigen (HLA) class II plays critical roles in antigen \n",
      "presentation and the initiation of immune responses. However, the correlation \n",
      "between the HLA class II gene expression level and the survival of patients with \n",
      "breast cancer is still under investigation. We analyzed microarray and RNA-Seq \n",
      "data of breast cancer from the cancer genome atlas (TCGA), genotype-tissue \n",
      "expression (GTEx) and Oncomine databases by using bioinformatics tools. The \n",
      "expression of the HLA-DQA1, HLA-DQA2, and HLA-DQB2 genes was significantly \n",
      "upregulated in breast cancer. Higher expression levels of HLA class II genes in \n",
      "breast cancer, especially HLA-DOB and HLA-DQB2, were significantly associated \n",
      "with better overall survival. Furthermore, the expression of HLA class II genes \n",
      "was more closely associated with survival in breast cancer than in other cancer \n",
      "types. CD48 coexpressed with both HLA-DOB and HLA-DQB2 was also positively \n",
      "associated with the overall survival of breast cancer patients. The results \n",
      "indicated that HLA class II and CD48 may enhance antitumor immunity, and their \n",
      "expression patterns may serve as potential prognostic biomarkers and therapeutic \n",
      "targets in breast cancer.\n",
      "\n",
      "Copyright © 2022 the Author(s). Published by Wolters Kluwer Health, Inc.\n",
      "\n",
      "DOI: 10.1097/MD.0000000000031071\n",
      "PMCID: PMC9543021\n",
      "PMID: 36221383 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have no conflicts of interest to \n",
      "disclose.\n",
      "\n",
      "\n",
      "13. Front Immunol. 2022 Sep 9;13:1001381. doi: 10.3389/fimmu.2022.1001381. \n",
      "eCollection 2022.\n",
      "\n",
      "A specific immune signature for predicting the prognosis of glioma patients with \n",
      "IDH1-mutation and guiding immune checkpoint blockade therapy.\n",
      "\n",
      "Zeng Z(1)(2), Hu C(3)(4), Ruan W(5), Zhang J(1)(2), Lei S(1)(2), Yang Y(6), Peng \n",
      "P(5)(7), Pan F(8), Chen T(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Transformation Engineering Research Center of Chronic Disease Diagnosis and \n",
      "Treatment, Guizhou Medical University, Guiyang, China.\n",
      "(2)Department of Physiology, School of Basic Medicine, Guizhou Medical \n",
      "University, Guiyang, China.\n",
      "(3)State Key Laboratory of Functions and Applications of Medicinal Plants, \n",
      "Guizhou Medical University, Guiyang, China.\n",
      "(4)Guizhou Provincial Engineering Technology Research Center for Chemical Drug \n",
      "R&D, Guizhou Medical University, Guiyang, China.\n",
      "(5)School of Clinical Medicine, Guizhou Medical University, Guiyang, China.\n",
      "(6)Department of Pathology, Affiliated Hospital of Guizhou Medical University, \n",
      "Guiyang, China.\n",
      "(7)Department of Gastroenterology, Affiliated Hospital of Guizhou Medical \n",
      "University, Guiyang, China.\n",
      "(8)Department of Bone and Joint Surgery, Gui Zhou Orthopedic Hospital, Guiyang, \n",
      "China.\n",
      "\n",
      "Isocitrate dehydrogenase (IDH1) is frequently mutated in glioma tissues, and \n",
      "this mutation mediates specific tumor-promoting mechanisms in glioma cells. We \n",
      "aimed to identify specific immune biomarkers for IDH1-mutation (IDH1mt) glioma. \n",
      "The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) were used \n",
      "to obtain RNA sequencing data and clinical characteristics of glioma tissues, \n",
      "while the stromal and immune scores of TCGA glioma tissues were determined using \n",
      "the ESTIMATE algorithm. Differentially expressed genes (DEGs), the \n",
      "protein-protein interaction(PPI) network, and least absolute shrinkage and \n",
      "selection operator (LASSO) and Cox regression analyses were used to select hub \n",
      "genes associated with stroma and immune scores and the prognoses of patients and \n",
      "to construct the risk model. The practicability and specificity of the risk \n",
      "model in both IDH1mt and IDH1-wildtype (wtIDH1) gliomas in TCGA and CGGA were \n",
      "evaluated. Molecular mechanisms, immunological characteristics and benefits of \n",
      "immune checkpoint blockade therapy in glioma tissues with IDH1mt were analyzed \n",
      "using GSEA, immunohistochemical staining, CIBERSORT, and T-cell dysfunction and \n",
      "exclusion (TIDE) analysis. The overall survival rate for IDH1mt-glioma patients \n",
      "with high stroma/immune scores was lower than that for those with low \n",
      "stroma/immune scores. A total of 222 DEGs were identified in IDH1mt glioma \n",
      "tissues with high stroma/immune scores. Among them, 72 genes had interactions in \n",
      "the PPI network, while three genes, HLA-DQA2, HOXA3, and SAA2, were selected as \n",
      "hub genes and used to construct risk models classifying patients into high- and \n",
      "low-risk score groups, followed by LASSO and Cox regression analyses. This risk \n",
      "model showed prognostic value in IDH1mt glioma in both TCGA and CCGA; \n",
      "nevertheless, the model was not suitable for wtIDH1 glioma. The risk model may \n",
      "act as an independent prognostic factor for IDH1mt glioma. IDH1mt glioma tissues \n",
      "from patients with high-risk scores showed more infiltration of M1 and CD8 T \n",
      "cells than those from patients with low-risk scores. Moreover, TIDE analysis \n",
      "showed that immune checkpoint blockade(ICB) therapy was highly beneficial for \n",
      "IDH1mt patients with high-risk scores. The risk model showed specific potential \n",
      "to predict the prognosis of IDH1mt-glioma patients, as well as guide ICB, \n",
      "contributing to the diagnosis and therapy of IDH1mt-glioma patients.\n",
      "\n",
      "Copyright © 2022 Zeng, Hu, Ruan, Zhang, Lei, Yang, Peng, Pan and Chen.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.1001381\n",
      "PMCID: PMC9500319\n",
      "PMID: 36159801 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "14. J Immunol Res. 2022 Jun 17;2022:7212852. doi: 10.1155/2022/7212852.\n",
      "eCollection  2022.\n",
      "\n",
      "The Potential Diagnostic Value of Immune-Related Genes in Interstitial Fibrosis \n",
      "and Tubular Atrophy after Kidney Transplantation.\n",
      "\n",
      "Yang B(1), Shi D(2), Chen Y(1), Zhu Y(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Hepato-Pancreato-Biliary Surgery, The Second Affiliated \n",
      "Hospital, Zhejiang University School of Medicine, Zhejiang, China.\n",
      "(2)Department of Gastrointestinal Surgery, The Second Affiliated Hospital, \n",
      "Zhejiang University School of Medicine, Zhejiang, China.\n",
      "\n",
      "BACKGROUND: Inflammation within areas of interstitial fibrosis and tubular \n",
      "atrophy (IF/TA) is associated with kidney allograft failure. The aim of this \n",
      "study was to reveal new diagnostic markers of IF/TA based on bioinformatics \n",
      "analysis.\n",
      "METHODS: Raw data of IF/TA samples after kidney transplantation and control \n",
      "samples after kidney transplantation were extracted from the Gene Expression \n",
      "Omnibus (GEO) database (GSE76882 and GSE120495 datasets), and genes that were \n",
      "differentially expressed between the two groups (DEGs) were screened. Gene Set \n",
      "Enrichment Analysis (GSEA), ESTIMATE and single sample GSEA (ssGSEA), least \n",
      "absolute shrinkage and selection operator (LASSO) regression analysis, and \n",
      "competing endogenous RNA (ceRNA) network were used to analyze the data.\n",
      "RESULTS: The results of GSEA revealed that multiple immune-related pathways were \n",
      "enriched in the IF/TA group, and subsequent immune landscape analysis also \n",
      "showed that the IF/TA group had higher immune and stromal scores and up to 15 \n",
      "types of immune cells occupied them, such as B cells, cytotoxic cells, and T \n",
      "cells. LASSO regression analysis selected 6 (including ANGPTL3, APOH, LTF, \n",
      "FCGR2B, HLA-DQA2, and EGF) out of 14 DE-IRGs as diagnostic genes to construct a \n",
      "diagnostic model. Then, receiver operating characteristic (ROC) curve analysis \n",
      "showed the powerful diagnostic value of the model, and the area under the curve \n",
      "(AUC) of a single diagnostic gene was greater than 0.75. The results of \n",
      "ingenuity pathway analysis (IPA) also indicated that DEGs were involved in the \n",
      "immune system and kidney disease-related pathways. Finally, we found multiple \n",
      "miRNAs that could regulate diagnostic genes from the ceRNA network.\n",
      "CONCLUSION: This study identified 6 IF/TA-related genes, which might be used as \n",
      "a new diagnosis model in the clinical practice.\n",
      "\n",
      "Copyright © 2022 Bin Yang et al.\n",
      "\n",
      "DOI: 10.1155/2022/7212852\n",
      "PMCID: PMC9232312\n",
      "PMID: 35755170 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "15. Front Immunol. 2022 May 26;13:878226. doi: 10.3389/fimmu.2022.878226. \n",
      "eCollection 2022.\n",
      "\n",
      "Single-Cell RNA and ATAC Sequencing Reveal Hemodialysis-Related Immune \n",
      "Dysregulation of Circulating Immune Cell Subpopulations.\n",
      "\n",
      "Wu H(1)(2), Dong J(1)(2), Yu H(1), Wang K(1), Dai W(3), Zhang X(1), Hu N(1), Yin \n",
      "L(2), Tang D(1), Liu F(2), Dai Y(1).\n",
      "\n",
      "Author information:\n",
      "(1)The Second Clinical Medical College of Jinan University, Shenzhen People's \n",
      "Hospital, Shenzhen, China.\n",
      "(2)Institute of Nephrology and Blood Purification, The First Affiliated Hospital \n",
      "of Jinan University, Jinan University, Guangzhou, China.\n",
      "(3)College of Natural Science, University of Texas at Austin, Austin, TX, United \n",
      "States.\n",
      "\n",
      "BACKGROUND: An increased risk of infection, malignancy, and cardiovascular \n",
      "diseases in maintenance hemodialysis patients is associated with \n",
      "hemodialysis-related immunity disturbances. Although defects in \n",
      "T-lymphocyte-dependent immune responses and preactivation of antigen-presenting \n",
      "cells have been documented in hemodialysis patients, the effects of long-term \n",
      "hemodialysis on the transcriptional program and chromosomal accessibility of \n",
      "circulating immune cell subpopulations remain poorly defined.\n",
      "METHODS: We integrated single-cell RNA sequencing (scRNA-seq) and single-cell \n",
      "assay for transposase-accessible chromatin sequencing (scATAC-seq) to \n",
      "characterize the transcriptome profiles of peripheral mononuclear cells (PBMCs) \n",
      "from healthy controls and maintenance hemodialysis patients. Validation of \n",
      "differentially expressed genes in CD4+ T cells and monocytes were performed by \n",
      "magnetic bead separation and quantitative real-time PCR.\n",
      "RESULTS: We identified 16 and 15 PBMC subgroups in scRNA-seq and scATAC-seq \n",
      "datasets, respectively. Hemodialysis significantly suppressed the expression \n",
      "levels of T cell receptor (TCR) genes in CD4+ T cell subsets (e.g., TRAV4, CD45, \n",
      "CD3G, CD3D, CD3E) and major histocompatibility complex II (MHC-II) \n",
      "pathway-related genes in monocytes (HLA-DRB1, HLA-DQA2, HLA-DQA1, HLA-DPB1). \n",
      "Downstream pathways of TCR signaling, including PI3K-Akt-mTOR, MAPK, TNF, and \n",
      "NF-κB pathways, were also inhibited in CD4+ T cell subpopulations during the \n",
      "hemodialysis procedure. Hemodialysis altered cellular communication patterns \n",
      "between PBMC subgroups, particularly TGF-TGFBR, HVEM-BTLA, and IL16-CD4 \n",
      "signalings between CD4+ T cells and monocytes. Additionally, we found that \n",
      "hemodialysis inhibited the expression of AP-1 family transcription factors (JUN, \n",
      "JUND, FOS, FOSB) by interfering with the chromatin accessibility profile.\n",
      "CONCLUSIONS: Our study provides a valuable framework for future investigations \n",
      "of hemodialysis-related immune dysregulation and identifies potential \n",
      "therapeutic targets for reconstituting the circulating immune system in \n",
      "maintenance hemodialysis patients.\n",
      "\n",
      "Copyright © 2022 Wu, Dong, Yu, Wang, Dai, Zhang, Hu, Yin, Tang, Liu and Dai.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.878226\n",
      "PMCID: PMC9205630\n",
      "PMID: 35720370 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "16. Genome Med. 2022 May 24;14(1):55. doi: 10.1186/s13073-022-01058-2.\n",
      "\n",
      "Fine-mapping studies distinguish genetic risks for childhood- and adult-onset \n",
      "asthma in the HLA region.\n",
      "\n",
      "Clay SM(1), Schoettler N(2), Goldstein AM(3), Carbonetto P(4), Dapas M(4), \n",
      "Altman MC(5)(6), Rosasco MG(6), Gern JE(7), Jackson DJ(7), Im HK(8), Stephens \n",
      "M(3), Nicolae DL(4)(3), Ober C(9).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "seleneclay@uchicago.edu.\n",
      "(2)Section of Pulmonary and Critical Care, Department of Medicine, University of \n",
      "Chicago, Chicago, IL, 60637, USA.\n",
      "(3)Department of Statistics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(4)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(5)Division of Allergy and Infectious Diseases, Department of Medicine, \n",
      "University of Washington, Seattle, WA, 98109, USA.\n",
      "(6)Systems Immunology Program, Benaroya Research Institute, Seattle, WA, 98101, \n",
      "USA.\n",
      "(7)Department of Pediatrics, University of Wisconsin, School of Medicine and \n",
      "Public Health, Madison, WI, 53706, USA.\n",
      "(8)Section of Genetic Medicine, Department of Medicine, University of Chicago, \n",
      "Chicago, IL, 60637, USA.\n",
      "(9)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "c-ober@genetics.uchicago.edu.\n",
      "\n",
      "BACKGROUND: Genome-wide association studies of asthma have revealed robust \n",
      "associations with variation across the human leukocyte antigen (HLA) complex \n",
      "with independent associations in the HLA class I and class II regions for both \n",
      "childhood-onset asthma (COA) and adult-onset asthma (AOA). However, the specific \n",
      "variants and genes contributing to risk are unknown.\n",
      "METHODS: We used Bayesian approaches to perform genetic fine-mapping for COA and \n",
      "AOA (n=9432 and 21,556, respectively; n=318,167 shared controls) in White \n",
      "British individuals from the UK Biobank and to perform expression quantitative \n",
      "trait locus (eQTL) fine-mapping in immune (lymphoblastoid cell lines, n=398; \n",
      "peripheral blood mononuclear cells, n=132) and airway (nasal epithelial cells, \n",
      "n=188) cells from ethnically diverse individuals. We also examined putatively \n",
      "causal protein coding variation from protein crystal structures and conducted \n",
      "replication studies in independent multi-ethnic cohorts from the UK Biobank (COA \n",
      "n=1686; AOA n=3666; controls n=56,063).\n",
      "RESULTS: Genetic fine-mapping revealed both shared and distinct causal variation \n",
      "between COA and AOA in the class I region but only distinct causal variation in \n",
      "the class II region. Both gene expression levels and amino acid variation \n",
      "contributed to risk. Our results from eQTL fine-mapping and amino acid \n",
      "visualization suggested that the HLA-DQA1*03:01 allele and variation associated \n",
      "with expression of the nonclassical HLA-DQA2 and HLA-DQB2 genes accounted \n",
      "entirely for the most significant association with AOA in GWAS. Our studies also \n",
      "suggested a potentially prominent role for HLA-C protein coding variation in the \n",
      "class I region in COA. We replicated putatively causal variant associations in a \n",
      "multi-ethnic cohort.\n",
      "CONCLUSIONS: We highlight roles for both gene expression and protein coding \n",
      "variation in asthma risk and identified putatively causal variation and genes in \n",
      "the HLA region. A convergence of genomic, transcriptional, and protein coding \n",
      "evidence implicates the HLA-DQA2 and HLA-DQB2 genes and HLA-DQA1*03:01 allele in \n",
      "AOA.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s13073-022-01058-2\n",
      "PMCID: PMC9128203\n",
      "PMID: 35606880 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "17. BMC Genomics. 2022 May 14;23(1):368. doi: 10.1186/s12864-022-08587-5.\n",
      "\n",
      "Diabetes risk loci-associated pathways are shared across metabolic tissues.\n",
      "\n",
      "Bouland GA(1), Beulens JWJ(2)(3), Nap J(1), van der Slik AR(4), Zaldumbide A(1), \n",
      "'t Hart LM(1)(2)(5), Slieker RC(6)(7).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Cell and Chemical Biology, Leiden University Medical Center, \n",
      "Einthovenweg 20, 2333ZC, Leiden, the Netherlands.\n",
      "(2)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \n",
      "Amsterdam Public Health Institute, Amsterdam, the Netherlands.\n",
      "(3)Julius Center for Health Sciences and Primary Care, University Medical Center \n",
      "Utrecht, Utrecht, the Netherlands.\n",
      "(4)Department of Immunohematology and Blood Transfusion, Leiden University \n",
      "Medical Center, Leiden, The Netherlands.\n",
      "(5)Molecular Epidemiology Section, Department of Biomedical Data Sciences, \n",
      "Leiden University Medical Center, Leiden, the Netherlands.\n",
      "(6)Department of Cell and Chemical Biology, Leiden University Medical Center, \n",
      "Einthovenweg 20, 2333ZC, Leiden, the Netherlands. r.c.slieker@lumc.nl.\n",
      "(7)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \n",
      "Amsterdam Public Health Institute, Amsterdam, the Netherlands. \n",
      "r.c.slieker@lumc.nl.\n",
      "\n",
      "AIMS/HYPOTHESIS: Numerous genome-wide association studies have been performed to \n",
      "understand the influence of genetic variation on type 2 diabetes etiology. Many \n",
      "identified risk variants are located in non-coding and intergenic regions, which \n",
      "complicates understanding of how genes and their downstream pathways are \n",
      "influenced. An integrative data approach will help to understand the mechanism \n",
      "and consequences of identified risk variants.\n",
      "METHODS: In the current study we use our previously developed method CONQUER to \n",
      "overlap 403 type 2 diabetes risk variants with regulatory, expression and \n",
      "protein data to identify tissue-shared disease-relevant mechanisms.\n",
      "RESULTS: One SNP rs474513 was found to be an expression-, protein- and \n",
      "metabolite QTL. Rs474513 influenced LPA mRNA and protein levels in the pancreas \n",
      "and plasma, respectively. On the pathway level, in investigated tissues most \n",
      "SNPs linked to metabolism. However, in eleven of the twelve tissues investigated \n",
      "nine SNPs were linked to differential expression of the ribosome pathway. \n",
      "Furthermore, seven SNPs were linked to altered expression of genes linked to the \n",
      "immune system. Among them, rs601945 was found to influence multiple HLA genes, \n",
      "including HLA-DQA2, in all twelve tissues investigated.\n",
      "CONCLUSION: Our results show that in addition to the classical metabolism \n",
      "pathways, other pathways may be important to type 2 diabetes that show a \n",
      "potential overlap with type 1 diabetes.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12864-022-08587-5\n",
      "PMCID: PMC9107144\n",
      "PMID: 35568807 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "18. J Thorac Oncol. 2022 Aug;17(8):974-990. doi: 10.1016/j.jtho.2022.04.011. Epub\n",
      " 2022 Apr 30.\n",
      "\n",
      "A Large-Scale Genome-Wide Gene-Gene Interaction Study of Lung Cancer \n",
      "Susceptibility in Europeans With a Trans-Ethnic Validation in Asians.\n",
      "\n",
      "Zhang R(1), Shen S(2), Wei Y(1), Zhu Y(3), Li Y(4), Chen J(3), Guan J(3), Pan \n",
      "Z(3), Wang Y(5), Zhu M(5), Xie J(6), Xiao X(7), Zhu D(7), Li Y(7), Albanes D(8), \n",
      "Landi MT(8), Caporaso NE(8), Lam S(9), Tardon A(10), Chen C(11), Bojesen SE(12), \n",
      "Johansson M(13), Risch A(14), Bickeböller H(15), Wichmann HE(16), Rennert G(17), \n",
      "Arnold S(18), Brennan P(13), McKay JD(13), Field JK(19), Shete SS(20), Le \n",
      "Marchand L(21), Liu G(22), Andrew AS(23), Kiemeney LA(24), Zienolddiny-Narui \n",
      "S(25), Behndig A(26), Johansson M(27), Cox A(28), Lazarus P(29), Schabath \n",
      "MB(30), Aldrich MC(31), Dai J(5), Ma H(5), Zhao Y(3), Hu Z(32), Hung RJ(33), \n",
      "Amos CI(7), Shen H(32), Chen F(34), Christiani DC(35).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Department of Environmental \n",
      "Health, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \n",
      "International Cooperation Center (CICC) for Environment and Human Health, \n",
      "Nanjing Medical University, Nanjing, People's Republic of China.\n",
      "(2)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Department of Environmental \n",
      "Health, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \n",
      "International Cooperation Center (CICC) for Environment and Human Health, \n",
      "Nanjing Medical University, Nanjing, People's Republic of China; State Key \n",
      "Laboratory of Reproductive Medicine, Nanjing Medical University, Nanjing, \n",
      "People's Republic of China.\n",
      "(3)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China.\n",
      "(4)Department of Biostatistics, University of Michigan, Ann Arbor, Michigan.\n",
      "(5)Department of Epidemiology, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer \n",
      "Biomarkers, Prevention and Treatment, Cancer Center, Collaborative Innovation \n",
      "Center for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, \n",
      "People's Republic of China.\n",
      "(6)Department of Epidemiology, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China.\n",
      "(7)The Institute for Clinical and Translational Research, Department of \n",
      "Medicine, Baylor College of Medicine, Houston, Texas.\n",
      "(8)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \n",
      "National Institutes of Health, Bethesda, Maryland.\n",
      "(9)Department of Medicine, British Columbia Cancer Agency, University of British \n",
      "Columbia, Vancouver, Canada.\n",
      "(10)Faculty of Medicine, University of Oviedo and CIBERESP, Oviedo, Spain.\n",
      "(11)Department of Epidemiology, University of Washington School of Public \n",
      "Health, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, \n",
      "Seattle, Washington.\n",
      "(12)Department of Clinical Biochemistry, Herlev and Gentofte Hospital, \n",
      "Copenhagen University Hospital, Copenhagen, Denmark.\n",
      "(13)Section of Genetics, International Agency for Research on Cancer, World \n",
      "Health Organization, Lyon, France.\n",
      "(14)Department of Biosciences and Cancer Cluster Salzburg, University of \n",
      "Salzburg, Salzburg, Austria.\n",
      "(15)Department of Genetic Epidemiology, University Medical Center, Georg August \n",
      "University Göttingen, Göttingen, Germany.\n",
      "(16)Institute of Medical Informatics, Biometry and Epidemiology, Ludwig \n",
      "Maximilians University, Munich, Germany.\n",
      "(17)Clalit National Cancer Control Center, Carmel Medical Center and Technion \n",
      "Faculty of Medicine, Carmel, Haifa, Israel.\n",
      "(18)Markey Cancer Center, University of Kentucky, Lexington, Kentucky.\n",
      "(19)Department of Molecular and Clinical Cancer Medicine, Institute of \n",
      "Translational Medicine, University of Liverpool, Liverpool, United Kingdom.\n",
      "(20)Department of Biostatistics, The University of Texas MD Anderson Cancer \n",
      "Center, Houston, Texas.\n",
      "(21)Epidemiology Program, University of Hawaii Cancer Center, Honolulu, Hawaii.\n",
      "(22)Princess Margaret Cancer Centre, Dalla Lana School of Public Health, \n",
      "University of Toronto, Toronto, Canada.\n",
      "(23)Department of Epidemiology, Department of Community and Family Medicine, \n",
      "Dartmouth Geisel School of Medicine, Hanover, New Hampshire.\n",
      "(24)Department for Health Evidence, Department of Urology, Radboud University \n",
      "Medical Center, Nijmegen, The Netherlands.\n",
      "(25)National Institute of Occupational Health, Oslo, Norway.\n",
      "(26)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(27)Department of Radiation Sciences, Umeå University, Umeå, Sweden.\n",
      "(28)Department of Oncology and Metabolism, The Medical School, University of \n",
      "Sheffield, Sheffield, United Kingdom.\n",
      "(29)Department of Pharmaceutical Sciences, College of Pharmacy, Washington State \n",
      "University, Spokane, Washington.\n",
      "(30)Department of Cancer Epidemiology, H. Lee Moffitt Cancer Center and Research \n",
      "Institute, Tampa, Florida.\n",
      "(31)Department of Thoracic Surgery and Division of Epidemiology, Vanderbilt \n",
      "University Medical Center, Nashville, Tennessee.\n",
      "(32)China International Cooperation Center (CICC) for Environment and Human \n",
      "Health, Nanjing Medical University, Nanjing, People's Republic of China; \n",
      "Department of Epidemiology, School of Public Health, Nanjing Medical University, \n",
      "Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \n",
      "Prevention and Treatment, Cancer Center, Collaborative Innovation Center for \n",
      "Cancer Personalized Medicine, Nanjing Medical University, Nanjing, People's \n",
      "Republic of China.\n",
      "(33)Lunenfeld-Tanenbaum Research Institute, Sinai Health System, Dalla Lana \n",
      "School of Public Health, University of Toronto, Toronto, Ontario, Canada.\n",
      "(34)Department of Biostatistics, School of Public Health, Nanjing Medical \n",
      "University, Nanjing, People's Republic of China; China International Cooperation \n",
      "Center (CICC) for Environment and Human Health, Nanjing Medical University, \n",
      "Nanjing, People's Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \n",
      "Prevention and Treatment, Cancer Center, Collaborative Innovation Center for \n",
      "Cancer Personalized Medicine, Nanjing Medical University, Nanjing, People's \n",
      "Republic of China. Electronic address: fengchen@njmu.edu.cn.\n",
      "(35)Department of Environmental Health, Harvard T. H. Chan School of Public \n",
      "Health, Boston, Massachusetts; Pulmonary and Critical Care Division, Department \n",
      "of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, \n",
      "Massachusetts.\n",
      "\n",
      "Comment in\n",
      "    J Thorac Oncol. 2022 Aug;17(8):946-948.\n",
      "    J Thorac Oncol. 2023 Mar;18(3):e23-e24.\n",
      "    J Thorac Oncol. 2023 Mar;18(3):e24-e26.\n",
      "\n",
      "INTRODUCTION: Although genome-wide association studies have been conducted to \n",
      "investigate genetic variation of lung tumorigenesis, little is known about \n",
      "gene-gene (G × G) interactions that may influence the risk of non-small cell \n",
      "lung cancer (NSCLC).\n",
      "METHODS: Leveraging a total of 445,221 European-descent participants from the \n",
      "International Lung Cancer Consortium OncoArray project, Transdisciplinary \n",
      "Research in Cancer of the Lung and UK Biobank, we performed a large-scale \n",
      "genome-wide G × G interaction study on European NSCLC risk by a series of \n",
      "analyses. First, we used BiForce to evaluate and rank more than 58 billion G × G \n",
      "interactions from 340,958 single-nucleotide polymorphisms (SNPs). Then, the top \n",
      "interactions were further tested by demographically adjusted logistic regression \n",
      "models. Finally, we used the selected interactions to build lung cancer \n",
      "screening models of NSCLC, separately, for never and ever smokers.\n",
      "RESULTS: With the Bonferroni correction, we identified eight statistically \n",
      "significant pairs of SNPs, which predominantly appeared in the 6p21.32 and \n",
      "5p15.33 regions (e.g., rs521828C6orf10 and rs204999PRRT1, ORinteraction = 1.17, \n",
      "p = 6.57 × 10-13; rs3135369BTNL2 and rs2858859HLA-DQA1, ORinteraction = 1.17, \n",
      "p = 2.43 × 10-13; rs2858859HLA-DQA1 and rs9275572HLA-DQA2, ORinteraction = 1.15, \n",
      "p = 2.84 × 10-13; rs2853668TERT and rs62329694CLPTM1L, ORinteraction = 0.73, p = \n",
      "2.70 × 10-13). Notably, even with much genetic heterogeneity across ethnicities, \n",
      "three pairs of SNPs in the 6p21.32 region identified from the European-ancestry \n",
      "population remained significant among an Asian population from the Nanjing \n",
      "Medical University Global Screening Array project (rs521828C6orf10 and \n",
      "rs204999PRRT1, ORinteraction = 1.13, p = 0.008; rs3135369BTNL2 and \n",
      "rs2858859HLA-DQA1, ORinteraction = 1.11, p = 5.23 × 10-4; rs3135369BTNL2 and \n",
      "rs9271300HLA-DQA1, ORinteraction = 0.89, p = 0.006). The interaction-empowered \n",
      "polygenetic risk score that integrated classical polygenetic risk score and G × \n",
      "G information score was remarkable in lung cancer risk stratification.\n",
      "CONCLUSIONS: Important G × G interactions were identified and enriched in the \n",
      "5p15.33 and 6p21.32 regions, which may enhance lung cancer screening models.\n",
      "\n",
      "Copyright © 2022 International Association for the Study of Lung Cancer. \n",
      "Published by Elsevier Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jtho.2022.04.011\n",
      "PMCID: PMC9512697\n",
      "PMID: 35500836 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Disclosure: The authors declare no conflict of \n",
      "interest.\n",
      "\n",
      "\n",
      "19. Transl Psychiatry. 2022 Apr 18;12(1):163. doi: 10.1038/s41398-022-01913-1.\n",
      "\n",
      "Allele-specific analysis reveals exon- and cell-type-specific regulatory effects \n",
      "of Alzheimer's disease-associated genetic variants.\n",
      "\n",
      "He L(1), Loika Y(2), Kulminski AM(3).\n",
      "\n",
      "Author information:\n",
      "(1)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. lh235@duke.edu.\n",
      "(2)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA.\n",
      "(3)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. alexander.kulminski@duke.edu.\n",
      "\n",
      "Elucidating regulatory effects of Alzheimer's disease (AD)-associated genetic \n",
      "variants is critical for unraveling their causal pathways and understanding the \n",
      "pathology. However, their cell-type-specific regulatory mechanisms in the brain \n",
      "remain largely unclear. Here, we conducted an analysis of allele-specific \n",
      "expression quantitative trait loci (aseQTLs) for 33 AD-associated variants in \n",
      "four brain regions and seven cell types using ~3000 bulk RNA-seq samples and \n",
      ">0.25 million single nuclei. We first develop a flexible hierarchical Poisson \n",
      "mixed model (HPMM) and demonstrate its superior statistical power to a \n",
      "beta-binomial model achieved by unifying samples in both allelic and \n",
      "genotype-level expression data. Using the HPMM, we identified 24 (~73%) aseQTLs \n",
      "in at least one brain region, including three new eQTLs associated with CA12, \n",
      "CHRNE, and CASS4. Notably, the APOE ε4 variant reduces APOE expression across \n",
      "all regions, even in AD-unaffected controls. Our results reveal region-dependent \n",
      "and exon-specific effects of multiple aseQTLs, such as rs2093760 with CR1, \n",
      "rs7982 with CLU, and rs3865444 with CD33. In an attempt to pinpoint the cell \n",
      "types responsible for the observed tissue-level aseQTLs using the snRNA-seq \n",
      "data, we detected many aseQTLs in microglia or monocytes associated with \n",
      "immune-related genes, including HLA-DQB1, HLA-DQA2, CD33, FCER1G, MS4A6A, SPI1, \n",
      "and BIN1, highlighting the regulatory role of AD-associated variants in the \n",
      "immune response. These findings provide further insights into potential causal \n",
      "pathways and cell types mediating the effects of the AD-associated variants.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1038/s41398-022-01913-1\n",
      "PMCID: PMC9016079\n",
      "PMID: 35436980 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "20. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\n",
      "eCollection  2022.\n",
      "\n",
      "The Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren's \n",
      "Syndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\n",
      "\n",
      "Hong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \n",
      "Zhao T(2), Fu J(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "\n",
      "OBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \n",
      "of multiple sclerosis (MS) and Sjögren's syndrome (SS).\n",
      "METHODS: MS- and SS-related susceptibility genes and differentially expressed \n",
      "genes (DEGs) were identified by bioinformatics analysis based on genome-wide \n",
      "association studies (GWAS) and transcriptome data from GWAS catalog and Gene \n",
      "Expression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \n",
      "analysis, and protein-protein interaction analysis for susceptibility genes and \n",
      "DEGs were performed. The drugs targeting common pathways/genes were obtained \n",
      "through Comparative Toxicogenomics Database (CTD), DrugBank database, and \n",
      "Drug-Gene Interaction (DGI) Database. The target genes of \n",
      "approved/investigational drugs for MS and SS were obtained through DrugBank and \n",
      "compared with the common susceptibility genes.\n",
      "RESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \n",
      "(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \n",
      "HLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \n",
      "than two genes, and 28 common susceptibility pathways, with 15 drugs targeting \n",
      "three or more than three pathways. Based on transcriptome data, we found 3 hub \n",
      "common DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \n",
      "435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \n",
      "pathways and common risk pathways at the same time. There were 133 overlaps \n",
      "including JAK-STAT inhibitors between agents from GWAS and transcriptome data. \n",
      "Besides, we found that IL2RA and HLA-DRB1, identified as hub common \n",
      "susceptibility genes, were the targets of daclizumab and glatiramer that were \n",
      "used for MS, indicating that daclizumab and glatiramer may be therapeutic for \n",
      "SS.\n",
      "CONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \n",
      "signaling pathway played a vital role, which may be the genetic and molecular \n",
      "bases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \n",
      "therapies for MS and SS, especially for their comorbidity.\n",
      "\n",
      "Copyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.857014\n",
      "PMCID: PMC8959321\n",
      "PMID: 35356004 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "21. Front Immunol. 2022 Feb 16;13:814239. doi: 10.3389/fimmu.2022.814239. \n",
      "eCollection 2022.\n",
      "\n",
      "Global Characterization of Peripheral B Cells in Parkinson's Disease by \n",
      "Single-Cell RNA and BCR Sequencing.\n",
      "\n",
      "Wang P(1), Luo M(1), Zhou W(1), Jin X(1), Xu Z(1), Yan S(2), Li Y(1), Xu C(1), \n",
      "Cheng R(1), Huang Y(1), Lin X(1), Yao L(2), Nie H(1), Jiang Q(1)(3).\n",
      "\n",
      "Author information:\n",
      "(1)School of Life Science and Technology, Harbin Institute of Technology, \n",
      "Harbin, China.\n",
      "(2)Department of Neurology, First Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "(3)Key Laboratory of Biological Big Data (Harbin Institute of Technology), \n",
      "Ministry of Education, Harbin, China.\n",
      "\n",
      "Immune system plays important roles in the pathogenesis of Parkinson's disease \n",
      "(PD). However, the role of B cells in this complex disease are still not fully \n",
      "understood. B cells produce antibodies but can also regulate immune responses. \n",
      "In order to decode the relative contribution of peripheral B cell subtypes to \n",
      "the etiology of PD, we performed single cell RNA and BCR sequencing for 10,466 B \n",
      "cells from 8 PD patients and 6 age-matched healthy controls. We observed \n",
      "significant increased memory B cells and significant decreased naïve B cells in \n",
      "PD patients compared to healthy controls. Notably, we also discovered increased \n",
      "IgG and IgA isotypes and more frequent class switch recombination events in PD \n",
      "patients. Moreover, we identified preferential V and J gene segments of B cell \n",
      "receptors in PD patients as the evidence of convergent selection in PD. Finally, \n",
      "we found a marked clonal expanded memory B cell population in PD patients, \n",
      "up-regulating both MHC II genes (HLA-DRB5, HLA-DQA2 and HLA-DPB1) and \n",
      "transcription factor activator protein 1 (AP-1), suggesting that the antigen \n",
      "presentation capacity of B cells was enhanced and B cells were activated in PD \n",
      "patients. Overall, this study conducted a comprehensive analysis of peripheral B \n",
      "cell characteristics of PD patients, which provided novel insights into the \n",
      "humoral immune response in the pathogenesis of PD.\n",
      "\n",
      "Copyright © 2022 Wang, Luo, Zhou, Jin, Xu, Yan, Li, Xu, Cheng, Huang, Lin, Yao, \n",
      "Nie and Jiang.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.814239\n",
      "PMCID: PMC8888848\n",
      "PMID: 35250991 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "22. Rheumatology (Oxford). 2022 Aug 3;61(8):3461-3470. doi: \n",
      "10.1093/rheumatology/keab912.\n",
      "\n",
      "Identification and functional characterization of a novel susceptibility locus \n",
      "for small vessel vasculitis with MPO-ANCA.\n",
      "\n",
      "Dahlqvist J(1)(2)(3), Ekman D(4), Sennblad B(5), Kozyrev SV(2), Nordin J(2), \n",
      "Karlsson Å(2), Meadows JRS(2), Hellbacher E(1), Rantapää-Dahlqvist S(6), Berglin \n",
      "E(6), Stegmayr B(6), Baslund B(7), Palm Ø(8), Haukeland H(9), Gunnarsson \n",
      "I(10)(11), Bruchfeld A(12)(13), Segelmark M(14), Ohlsson S(14), Mohammad \n",
      "AJ(15)(16), Svärd A(17), Pullerits R(18)(19), Herlitz H(20), Söderbergh A(21), \n",
      "Rosengren Pielberg G(2), Hultin Rosenberg L(2), Bianchi M(2), Murén E(2), Omdal \n",
      "R(22)(23), Jonsson R(24), Eloranta ML(1), Rönnblom L(1), Söderkvist P(25), \n",
      "Knight A(1), Eriksson P(26), Lindblad-Toh K(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medical Sciences.\n",
      "(2)Science for Life Laboratory, Department of Medical Biochemistry and \n",
      "Microbiology, Uppsala University, Uppsala, Sweden.\n",
      "(3)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\n",
      "(4)Department of Biochemistry and Biophysics, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Stockholm University, \n",
      "Stockholm.\n",
      "(5)Department of Cell and Molecular Biology, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala.\n",
      "(6)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(7)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \n",
      "Diseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\n",
      "(8)Department of Rheumatology, Oslo University Hospital.\n",
      "(9)Department of Rheumatology, Martina Hansens Hospital, Oslo, Norway.\n",
      "(10)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \n",
      "Stockholm.\n",
      "(11)Unit of Rheumatology, Karolinska University Hospital, Stockholm.\n",
      "(12)Department of Health, Medicine and Caring Sciences, Linköping University, \n",
      "Linköping.\n",
      "(13)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \n",
      "Karolinska Institutet, Stockholm.\n",
      "(14)Department of Clinical Sciences, Division of Nephrology, Lund University and \n",
      "Skåne University Hospital.\n",
      "(15)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \n",
      "University Hospital, Lund University, Lund, Sweden.\n",
      "(16)Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "(17)Center for Clinical Research Dalarna, Uppsala University, Uppsala.\n",
      "(18)Department of Rheumatology and Inflammation Research, Institution of \n",
      "Medicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg.\n",
      "(19)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \n",
      "University Hospital.\n",
      "(20)Department of Molecular and Clinical Medicine/Nephrology, Institute of \n",
      "Medicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg.\n",
      "(21)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\n",
      "(22)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \n",
      "University Hospital, Stavanger.\n",
      "(23)Department of Clinical Science.\n",
      "(24)Broegelmann Research Laboratory, Department of Clinical Science, University \n",
      "of Bergen, Bergen, Norway.\n",
      "(25)Department of Biomedical and Clinical Sciences, Division of Cell Biology.\n",
      "(26)Department of Biomedical and Clinical Sciences, Division of Inflammation and \n",
      "Infection, Linköping University, Linköping, Sweden.\n",
      "\n",
      "OBJECTIVE: To identify and characterize genetic loci associated with the risk of \n",
      "developing ANCA-associated vasculitides (AAV).\n",
      "METHODS: Genetic association analyses were performed after Illumina sequencing \n",
      "of 1853 genes and subsequent replication with genotyping of selected single \n",
      "nucleotide polymorphisms in a total cohort of 1110 Scandinavian cases with \n",
      "granulomatosis with polyangiitis or microscopic polyangiitis, and 1589 controls. \n",
      "A novel AAV-associated single nucleotide polymorphism was analysed for \n",
      "allele-specific effects on gene expression using luciferase reporter assay.\n",
      "RESULTS: PR3-ANCA+ AAV was significantly associated with two independent loci in \n",
      "the HLA-DPB1/HLA-DPA1 region [rs1042335, P = 6.3 × 10-61, odds ratio (OR) 0.10; \n",
      "rs9277341, P = 1.5 × 10-44, OR 0.22] and with rs28929474 in the SERPINA1 gene \n",
      "(P = 2.7 × 10-10, OR 2.9). MPO-ANCA+ AAV was significantly associated with the \n",
      "HLA-DQB1/HLA-DQA2 locus (rs9274619, P = 5.4 × 10-25, OR 3.7) and with a rare \n",
      "variant in the BACH2 gene (rs78275221, P = 7.9 × 10-7, OR 3.0), the latter a \n",
      "novel susceptibility locus for MPO-ANCA+ granulomatosis with \n",
      "polyangiitis/microscopic polyangiitis. The rs78275221-A risk allele reduced \n",
      "luciferase gene expression in endothelial cells, specifically, as compared with \n",
      "the non-risk allele.\n",
      "CONCLUSION: We identified a novel susceptibility locus for MPO-ANCA+ AAV and \n",
      "propose that the associated variant is of mechanistic importance, exerting a \n",
      "regulatory function on gene expression in specific cell types.\n",
      "\n",
      "© The Author(s) 2021. Published by Oxford University Press on behalf of the \n",
      "British Society for Rheumatology.\n",
      "\n",
      "DOI: 10.1093/rheumatology/keab912\n",
      "PMCID: PMC9348767\n",
      "PMID: 34888651 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "23. J Radiat Res. 2022 Jan 20;63(1):8-18. doi: 10.1093/jrr/rrab091.\n",
      "\n",
      "Transcriptomic profiling reveals gene expression in human peripheral blood after \n",
      "exposure to low-dose ionizing radiation.\n",
      "\n",
      "Fang F, Yu X, Wang X, Zhu X, Liu L, Rong L, Niu D, Li J.\n",
      "\n",
      "Although the health effects of exposure to low-dose ionizing radiation have been \n",
      "the focus of many studies, the affected biological functions and underlying \n",
      "regulatory mechanisms are not well-understood. In particular, the influence of \n",
      "radiation exposure at doses of less than 200 mGy on the regulation of genes and \n",
      "pathways remains unclear. To investigate the molecular alterations induced by \n",
      "varying doses of low-dose radiation (LDR), transcriptomic analysis was conducted \n",
      "based on ribonucleic acid (RNA) sequencing following exposure to 50 and 150 mGy \n",
      "doses. Human peripheral blood was collected, and the samples were divided into \n",
      "three groups, including two treatments and one control (no radiation). A total \n",
      "of 876 (318 upregulated and 558 downregulated) and 486 (202 upregulated and 284 \n",
      "downregulated) differentially expressed genes (DEGs) were identified after \n",
      "exposure to 50 mGy and 150 mGy, respectively. Most upregulated genes in both the \n",
      "50 mGy and 150 mGy groups were associated with 'antigen processing and \n",
      "presentation,' which appeared to be the major targets affected by LDR exposure. \n",
      "Several interacting genes, including HLA-DQA1, HLA-DQA2, HLA-DQB2, HLA-DRB1, and \n",
      "HLA-DRB5 were mapped to 'antigen processing and presentation,' 'immune \n",
      "system-related diseases' and the 'cytokine-mediated signaling pathway,' \n",
      "suggesting that these genes might drive the downstream transmission of these \n",
      "signal transduction pathways. Our results suggest that exposure to LDR may \n",
      "elicit changes in key genes and associated pathways, probably helping further \n",
      "explore the biological processes and molecular mechanism responsible for \n",
      "low-dose occupational or environmental exposures in humans.\n",
      "\n",
      "© The Author(s) 2021. Published by Oxford University Press on behalf of The \n",
      "Japanese Radiation Research Society and Japanese Society for Radiation Oncology.\n",
      "\n",
      "DOI: 10.1093/jrr/rrab091\n",
      "PMCID: PMC8776696\n",
      "PMID: 34788452 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "24. Comput Math Methods Med. 2021 Sep 21;2021:6015473. doi: 10.1155/2021/6015473.\n",
      " eCollection 2021.\n",
      "\n",
      "Comprehensive Analysis of RNA Expression Profile Identifies Hub miRNA-circRNA \n",
      "Interaction Networks in the Hypoxic Ischemic Encephalopathy.\n",
      "\n",
      "Wei L(#)(1), Li X(#)(1), Wang L(2), Song Y(3), Dong H(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neonatology, Jinan Maternal and Child Health Care Hospital, \n",
      "Shandong, China.\n",
      "(2)Department of Pediatrics, Jinan Maternal and Child Health Care Hospital, \n",
      "Shandong, China.\n",
      "(3)Department of Pediatrics, Chinese People's Liberation Army of 960 Hospital, \n",
      "Shandong, China.\n",
      "(#)Contributed equally\n",
      "\n",
      "Hypoxic ischemic encephalopathy (HIE) is classified as a sort of serious nervous \n",
      "system syndrome that occurs in the early life period. Noncoding RNAs had been \n",
      "confirmed to have crucial roles in human diseases. So far, there were few \n",
      "systematical and comprehensive studies towards the expression profile of RNAs in \n",
      "the brain after hypoxia ischemia. In this study, 31 differentially expressed \n",
      "microRNAs (miRNAs) with upregulation were identified. In addition, 5512 \n",
      "differentially expressed mRNAs, long noncoding RNAs (lncRNAs), and circular RNAs \n",
      "(circRNAs) were identified in HIE groups. Bioinformatics analysis showed these \n",
      "circRNAs and mRNAs were significantly enriched in regulation of leukocyte \n",
      "activation, response to virus, and neutrophil degranulation. Pathway and its \n",
      "related gene network analysis indicated that HLA - DPA1, HLA - DQA2, HLA - DQB1, \n",
      "and HLA - DRB4 have a more crucial role in HIE. Finally, miRNA-circRNA-mRNA \n",
      "interaction network analysis was also performed to identify hub miRNAs and \n",
      "circRNAs. We found that miR-592 potentially targeting 5 circRNAs, thus affecting \n",
      "15 mRNA expressions in HIR. hsa_circ_0068397 and hsa_circ_0045698 were \n",
      "identified as hub circRNAs in HIE. Collectively, using RNA-seq, bioinformatics \n",
      "analysis, and circRNA/miRNA interaction prediction, we systematically \n",
      "investigated the differentially expressed RNAs in HIE, which could give a new \n",
      "hint of understanding the pathogenesis of HIE.\n",
      "\n",
      "Copyright © 2021 Lin Wei et al.\n",
      "\n",
      "DOI: 10.1155/2021/6015473\n",
      "PMCID: PMC8481051\n",
      "PMID: 34603484 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: No conflict of interest.\n",
      "\n",
      "\n",
      "25. Public Health Genomics. 2021;24(5-6):267-279. doi: 10.1159/000517308. Epub\n",
      "2021  Aug 2.\n",
      "\n",
      "Meta-Analyses of Multiple Gene Expression Profiles to Screen Hub Genes Related \n",
      "to Osteoarthritis.\n",
      "\n",
      "Zhu X(1), Guo W(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Orthopedics, Taizhou People's Hospital, Taizhou, China.\n",
      "\n",
      "BACKGROUND: This study aimed to screen and validate the crucial genes involved \n",
      "in osteoarthritis (OA) and explore its potential molecular mechanisms.\n",
      "METHODS: Four expression profile datasets related to OA were downloaded from the \n",
      "Gene Expression Omnibus (GEO). The differentially expressed genes (DEGs) from 4 \n",
      "microarray patterns were identified by the meta-analysis method. The weighted \n",
      "gene co-expression network analysis (WGCNA) method was used to investigate \n",
      "stable modules most related to OA. In addition, a protein-protein interaction \n",
      "(PPI) network was built to explore hub genes in OA. Moreover, OA-related genes \n",
      "and pathways were retrieved from Comparative Toxicogenomics Database (CTD).\n",
      "RESULTS: A total of 1,136 DEGs were identified from 4 datasets. Based on these \n",
      "DEGs, WGCNA further explored 370 genes included in the 3 OA-related stable \n",
      "modules. A total of 10 hub genes were identified in the PPI network, including \n",
      "AKT1, CDC42, HLA-DQA2, TUBB, TWISTNB, GSK3B, FZD2, KLC1, GUSB, and RHOG. \n",
      "Besides, 5 pathways including \"Lysosome,\" \"Pathways in cancer,\" \"Wnt signaling \n",
      "pathway,\" \"ECM-receptor interaction\" and \"Focal adhesion\" in CTD and enrichment \n",
      "analysis and 5 OA-related hub genes (including GSK3B, CDC42, AKT1, FZD2, and \n",
      "GUSB) were identified.\n",
      "CONCLUSION: In this study, the meta-analysis was used to screen the central \n",
      "genes associated with OA in a variety of gene expression profiles. Three \n",
      "OA-related modules (green, turquoise, and yellow) containing 370 genes were \n",
      "identified through WGCNA. It was discovered through the gene-pathway network \n",
      "that GSK3B, CDC42, AKT1, FZD2, and GUSB may be key genes related to the progress \n",
      "of OA and may become promising therapeutic targets.\n",
      "\n",
      "© 2021 The Author(s). Published by S. Karger AG, Basel.\n",
      "\n",
      "DOI: 10.1159/000517308\n",
      "PMID: 34340232 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "26. Restor Neurol Neurosci. 2021;39(4):247-266. doi: 10.3233/RNN-211161.\n",
      "\n",
      "Transcriptome analysis of human dorsal striatum implicates attenuated canonical \n",
      "WNT signaling in neuroinflammation and in age-related impairment of striatal \n",
      "neurogenesis and synaptic plasticity.\n",
      "\n",
      "Wu C(1), Bendriem RM(2), Freed WJ(3), Lee CT(4).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Molecular and Cellular Pharmacology, Miller School of Medicine, \n",
      "University of Miami, Miami, FL, USA.\n",
      "(2)Brain and Mind Research Institute, Weill Cornell Graduate School of Medical \n",
      "Sciences, New York, NY, USA.\n",
      "(3)Department of Biology, Lebanon Valley College, Annville, PA, USA.\n",
      "(4)Department of Neurology, Miller School of Medicine, University of Miami, \n",
      "Miami, FL, USA.\n",
      "\n",
      "BACKGROUND: Motor and cognitive decline as part of the normal aging process is \n",
      "linked to alterations in synaptic plasticity and reduction of adult neurogenesis \n",
      "in the dorsal striatum. Neuroinflammation, particularly in the form of \n",
      "microglial activation, is suggested to contribute to these age-associated \n",
      "changes.\n",
      "OBJECTIVE AND METHODS: To explore the molecular basis of alterations in striatal \n",
      "function during aging we analyzed RNA-Seq data for 117 postmortem human dorsal \n",
      "caudate samples and 97 putamen samples acquired through GTEx.\n",
      "RESULTS: Increased expression of neuroinflammatory transcripts including TREM2, \n",
      "MHC II molecules HLA-DMB, HLA-DQA2, HLA-DPA1, HLA-DPB1, HLA-DMA and HLA-DRA, \n",
      "complement genes C1QA, C1QB, CIQC and C3AR1, and MHCI molecules HLA-B and HLA-F \n",
      "was identified. We also identified down-regulation of transcripts involved in \n",
      "neurogenesis, synaptogenesis, and synaptic pruning, including DCX, CX3CL1, and \n",
      "CD200, and the canonical WNTs WNT7A, WNT7B, and WNT8A. The canonical WNT \n",
      "signaling pathway has previously been shown to mediate adult neurogenesis and \n",
      "synapse formation and growth. Recent findings also highlight the link between \n",
      "WNT/β-catenin signaling and inflammation pathways.\n",
      "CONCLUSIONS: These findings suggest that age-dependent attenuation of canonical \n",
      "WNT signaling plays a pivotal role in regulating striatal plasticity during \n",
      "aging. Dysregulation of WNT/β-catenin signaling via astrocyte-microglial \n",
      "interactions is suggested to be a novel mechanism that drives the decline of \n",
      "striatal neurogenesis and altered synaptic connectivity and plasticity, leading \n",
      "to a subsequent decrease in motor and cognitive performance with age. These \n",
      "findings may aid in the development of therapies targeting WNT/β-catenin \n",
      "signaling to combat cognitive and motor impairments associated with aging.\n",
      "\n",
      "DOI: 10.3233/RNN-211161\n",
      "PMID: 34275915 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "27. J Cell Mol Med. 2021 Jul;25(14):7001-7012. doi: 10.1111/jcmm.16714. Epub 2021\n",
      " Jun 16.\n",
      "\n",
      "Single-cell RNA sequencing identify SDCBP in ACE2-positive bronchial epithelial \n",
      "cells negatively correlates with COVID-19 severity.\n",
      "\n",
      "Ma D(1)(2), Liu S(1), Hu L(1)(3), He Q(1), Shi W(1)(2), Yan D(1)(3), Cao \n",
      "Y(1)(2)(3), Zhang G(1)(2)(3), Wang Z(1)(2)(3), Wu J(1), Jiang C(1)(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Jiangsu Key Laboratory of Molecular Medicine, National Institute of \n",
      "Healthcare Data Science at Nanjing University, Medical School of Nanjing \n",
      "University, Nanjing, China.\n",
      "(2)Department of Hepatobiliary Surgery, The Affiliated Drum Tower Hospital of \n",
      "Nanjing University Medical School, Nanjing, China.\n",
      "(3)Department of Hepatobiliary Surgery, Drum Tower Clinical College of Nanjing \n",
      "Medical University, Nanjing, China.\n",
      "\n",
      "Erratum in\n",
      "    J Cell Mol Med. 2023 Oct;27(19):3012-3015.\n",
      "\n",
      "The coronavirus disease 2019 (COVID-19), caused by the novel severe acute \n",
      "respiratory syndrome coronavirus 2 (SARS-CoV-2), has resulted in many deaths \n",
      "throughout the world. It is vital to identify the novel prognostic biomarkers \n",
      "and therapeutic targets to assist with the subsequent diagnosis and treatment \n",
      "plan to mitigate the expansion of COVID-19. Since angiotensin-converting enzyme \n",
      "2 (ACE2)-positive cells are hosts for COVID-19, we focussed on this cell type to \n",
      "explore the underlying mechanisms of COVID-19. In this study, we identified that \n",
      "ACE2-positive cells from the bronchoalveolar lavage fluid (BALF) of patients \n",
      "with COVID-19 belong to bronchial epithelial cells. Comparing with patients of \n",
      "COVID-19 showing severe symptoms, the antigen processing and presentation \n",
      "pathway was increased and 12 typical genes, HLA-DRB5, HLA-DRB1, CD74, HLA-DRA, \n",
      "HLA-DPA1, HLA-DQA1, HSP90AA1, HSP90AB1, HLA-DPB1, HLA-DQB1, HLA-DQA2, and \n",
      "HLA-DMA, particularly HLA-DPB1, were obviously up-regulated in ACE2-positive \n",
      "bronchial epithelial cells of patients with mild disease. We further discovered \n",
      "SDCBP was positively correlated with above 12 genes particularly with HLA-DPB1 \n",
      "in ACE2-positive bronchial epithelial cells of COVID-19 patients. Moreover, \n",
      "SDCBP may increase the immune infiltration of B cells, CD8+ T cells, CD4+ T \n",
      "cells, macrophages, neutrophils and dendritic cells in different lung carcinoma. \n",
      "Moreover, we found the expression of SDCBP was positively correlated with the \n",
      "expression of antigen processing and presentation genes in post-mortem lung \n",
      "biopsies tissues, which is consistent with previous discoveries. These results \n",
      "suggest that SDCBP has good potential to be further developed as a novel \n",
      "diagnostic and therapeutic target in the treatment of COVID-19.\n",
      "\n",
      "© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \n",
      "Foundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/jcmm.16714\n",
      "PMCID: PMC8278084\n",
      "PMID: 34137173 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that there are no conflicts \n",
      "of interest.\n",
      "\n",
      "\n",
      "28. Neurol Genet. 2021 Jun 2;7(3):e592. doi: 10.1212/NXG.0000000000000592. \n",
      "eCollection 2021 Jun.\n",
      "\n",
      "Association of HLA-DQA2 and HLA-B With Moyamoya Disease in the Chinese Han \n",
      "Population.\n",
      "\n",
      "Wan J(1), Ling W(1), Zhengshan Z(1), Xianbo Z(1), Lian D(1), Kai W(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neurology (J.W.), the First Affiliated Hospital of Anhui \n",
      "Medical University, the School of Mental Health and Psychological Sciences, \n",
      "Anhui Medical University, Hefei, Anhui Province, Department of Neurology (J.W.), \n",
      "Drum Tower Hospital, Medical School and the State Key Laboratory of \n",
      "Pharmaceutical Biotechnology, Nanjing University; the School of Mental Health \n",
      "and Psychological Sciences (W.L.), Anhui Medical University, Anhui Province, \n",
      "Institute of Artificial Intelligence (W.L.), Hefei Comprehensive National \n",
      "Science Center. Anhui Province Key Laboratory of Cognition and Neuropsychiatric \n",
      "Disorders (W.L.), Hefei; Collaborative Innovation Center of Neuropsychiatric \n",
      "Disorders and Mental Health (W.L), Anhui Province; Department of Neurosurgery \n",
      "(Z.Z.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \n",
      "Hospital of PLA), Beijing; Department of Dermatology (Z.X.), the First \n",
      "Affiliated Hospital, Anhui Medical University, Hefei, Anhui Province; Key \n",
      "Laboratory of Dermatology (Z.X.), Anhui Medical University, Ministry of \n",
      "Education, Hefei, Anhui Province; State Key Lab of Dermatology Incubation Center \n",
      "(Z.X.), Anhui Medical University, Hefei, China; Department of Neurosurgery \n",
      "(D.L.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \n",
      "Hospital of PLA), Beijing; the School of Mental Health and Psychological \n",
      "Sciences (W.K.), Anhui Medical University, Anhui Province; Institute of \n",
      "Artificial Intelligence (W.K.), Hefei Comprehensive National Science Center; \n",
      "Anhui Province Key Laboratory of Cognition and Neuropsychiatric Disorders \n",
      "(W.K.), Hefei, Collaborative Innovation Center of Neuropsychiatric Disorders and \n",
      "Mental Health (W.K.), Anhui Province, China.\n",
      "\n",
      "OBJECTIVE: An HLA imputation was conducted to explore the relationship between \n",
      "HLA and patients with moyamoya disease (MMD) in the Chinese Han population.\n",
      "METHODS: In this study, we performed an association analysis of the major \n",
      "histocompatibility complex region in 2,786 individuals of Chinese Han ancestry \n",
      "(2,031 controls and 755 patients with MMD), through a widely used HLA imputation \n",
      "method.\n",
      "RESULTS: We identified that the variant rs3129731 (odds ratio [OR] = 1.79, p = \n",
      "3.69 × 10-16) located between the MTCO3P1 and HLA-DQA2 is a major genetic risk \n",
      "factor for MMD. In addition to this variant, found in the conditional \n",
      "association analysis, we also detected another independent signal, rs1071817 (OR \n",
      "= 0.62, p = 1.20 × 10-11), in HLA-B.\n",
      "CONCLUSIONS: Our research suggests that the genetic polymorphism of HLA-DQA2 and \n",
      "HLA-B could be a genetic predisposing factor for MMD in Chinese Han. This may \n",
      "provide some evidence for further HLA-related studies of patients with MMD of \n",
      "Chinese Han ethnicity and indicates that MMD is an immune-related disease.\n",
      "\n",
      "Copyright © 2021 The Author(s). Published by Wolters Kluwer Health, Inc. on \n",
      "behalf of the American Academy of Neurology.\n",
      "\n",
      "DOI: 10.1212/NXG.0000000000000592\n",
      "PMCID: PMC8176556\n",
      "PMID: 34095496\n",
      "\n",
      "\n",
      "29. PLoS One. 2021 Apr 22;16(4):e0248429. doi: 10.1371/journal.pone.0248429. \n",
      "eCollection 2021.\n",
      "\n",
      "Hypomethylation mediates genetic association with the major histocompatibility \n",
      "complex genes in Sjögren's syndrome.\n",
      "\n",
      "Chi C(1)(2), Taylor KE(3), Quach H(2), Quach D(2), Criswell LA(3), Barcellos \n",
      "LF(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Computational Biology, College of Engineering, University of \n",
      "California, Berkeley, Berkeley, California, United States of America.\n",
      "(2)Genetic Epidemiology and Genomics Laboratory, School of Public Health, \n",
      "University of California, Berkeley, Berkeley, California, United States of \n",
      "America.\n",
      "(3)Department of Medicine, Russell/Engleman Rheumatology Research Center, \n",
      "University of California, San Francisco, San Francisco, California, United \n",
      "States of America.\n",
      "\n",
      "Erratum in\n",
      "    PLoS One. 2021 Jul 27;16(7):e0255549.\n",
      "\n",
      "Differential methylation of immune genes has been a consistent theme observed in \n",
      "Sjögren's syndrome (SS) in CD4+ T cells, CD19+ B cells, whole blood, and labial \n",
      "salivary glands (LSGs). Multiple studies have found associations supporting \n",
      "genetic control of DNA methylation in SS, which in the absence of reverse \n",
      "causation, has positive implications for the potential of epigenetic therapy. \n",
      "However, a formal study of the causal relationship between genetic variation, \n",
      "DNA methylation, and disease status is lacking. We performed a causal mediation \n",
      "analysis of DNA methylation as a mediator of nearby genetic association with SS \n",
      "using LSGs and genotype data collected from 131 female members of the Sjögren's \n",
      "International Collaborative Clinical Alliance registry, comprising of 64 SS \n",
      "cases and 67 non-cases. Bumphunter was used to first identify \n",
      "differentially-methylated regions (DMRs), then the causal inference test (CIT) \n",
      "was applied to identify DMRs mediating the association of nearby methylation \n",
      "quantitative trait loci (MeQTL) with SS. Bumphunter discovered 215 DMRs, with \n",
      "the majority located in the major histocompatibility complex (MHC) on chromosome \n",
      "6p21.3. Consistent with previous findings, regions hypomethylated in SS cases \n",
      "were enriched for gene sets associated with immune processes. Using the CIT, we \n",
      "observed a total of 19 DMR-MeQTL pairs that exhibited strong evidence for a \n",
      "causal mediation relationship. Close to half of these DMRs reside in the MHC and \n",
      "their corresponding meQTLs are in the region spanning the HLA-DQA1, HLA-DQB1, \n",
      "and HLA-DQA2 loci. The risk of SS conferred by these corresponding MeQTLs in the \n",
      "MHC was further substantiated by previous genome-wide association study results, \n",
      "with modest evidence for independent effects. By validating the presence of \n",
      "causal mediation, our findings suggest both genetic and epigenetic factors \n",
      "contribute to disease susceptibility, and inform the development of targeted \n",
      "epigenetic modification as a therapeutic approach for SS.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0248429\n",
      "PMCID: PMC8062105\n",
      "PMID: 33886574 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have declared that no competing \n",
      "interests exist.\n",
      "\n",
      "\n",
      "30. Arthritis Rheumatol. 2021 Oct;73(10):1921-1930. doi: 10.1002/art.41758. Epub \n",
      "2021 Aug 31.\n",
      "\n",
      "Transcriptomic Evaluation of Juvenile Localized Scleroderma Skin With Histologic \n",
      "and Clinical Correlation.\n",
      "\n",
      "Schutt C(1), Mirizio E(2), Salgado C(3), Reyes-Mugica M(3), Wang X(3), Chen \n",
      "W(4), Grunwaldt L(3), Schollaert KL(2), Torok KS(4).\n",
      "\n",
      "Author information:\n",
      "(1)University of Pittsburgh Medical Center, Children's Hospital of Pittsburgh, \n",
      "Pittsburgh, Pennsylvania, and University of Rochester MedicalCenter and Golisano \n",
      "Children's Hospital, Rochester, New York.\n",
      "(2)University of Pittsburgh, Pittsburgh, Pennsylvania.\n",
      "(3)University of Pittsburgh Medical Center, Children's Hospital of Pittsburgh, \n",
      "Pittsburgh, Pennsylvania.\n",
      "(4)University of Pittsburgh, University of Pittsburgh Medical Center, Children's \n",
      "Hospital of Pittsburgh, Pittsburgh, Pennsylvania.\n",
      "\n",
      "OBJECTIVE: Juvenile localized scleroderma (LS) is an autoimmune disease of the \n",
      "skin whose pathogenesis is not well understood due to the rarity of the disease. \n",
      "This study was undertaken to determine the skin transcriptome in skin biopsy \n",
      "tissue from children with juvenile LS compared to pediatric healthy controls, \n",
      "with identification of significant molecular targets using RNA sequencing \n",
      "(RNA-Seq). In this study, differentially expressed genes (DEGs) were assessed \n",
      "for correlations with histopathologic and clinical features in children with \n",
      "juvenile LS, and were used to group the children into distinct genetic clusters \n",
      "based on immunophenotype.\n",
      "METHODS: RNA-Seq was performed on sections of paraffin-embedded skin tissue \n",
      "obtained from 28 children with juvenile LS and 10 pediatric healthy controls. \n",
      "RNA-Seq was carried out using an Illumina HTS TruSeq RNA Access library prep \n",
      "kit, with data aligned using STAR and data analysis using a DESeq2 platform. A \n",
      "standardized histologic scoring system was used to score skin sections for the \n",
      "severity of inflammation and levels of collagen deposition. Histologic scoring \n",
      "was completed by 2 pathologists who were blinded with regard to the status of \n",
      "each sample. Spearman's rank correlation coefficients were used to assess \n",
      "significant correlations between DEG expression profiles and skin histologic \n",
      "findings in patients with juvenile LS.\n",
      "RESULTS: We identified 589 significant DEGs in children with juvenile LS as \n",
      "compared to healthy controls. Hierarchical clustering was used to demonstrate 3 \n",
      "distinct juvenile LS immunophenotype clusters. The histologic scores of skin \n",
      "inflammation (based on numbers and categories of inflammatory cell infiltrates) \n",
      "were significantly correlated with the expression levels of HLA-DPB1, HLA-DQA2, \n",
      "HLA-DRA, and STAT1 genes (rs > 0.5, P < 0.01). Collagen thickness correlated \n",
      "with the expression levels of collagen organization genes as well as with genes \n",
      "found to be correlated with the severity of inflammation, including genes for \n",
      "major histocompatibility complex (MHC) class I, MHC class II, and interferon-γ \n",
      "signaling.\n",
      "CONCLUSION: Among children with juvenile LS, 3 distinct genetic signatures, or \n",
      "clusters, were identified. In one cluster, inflammation-related pathways were \n",
      "up-regulated, corresponding to the histologic skin inflammation score. In the \n",
      "second cluster, fibrosis-related pathways were up-regulated. In the third \n",
      "cluster, gene expression in the skin corresponded to the patterns seen in \n",
      "healthy controls. Up-regulation of HLA class II genes was observed within the \n",
      "first cluster (characterized by predominant inflammation), a feature that has \n",
      "also been observed in the peripheral blood of patients with morphea and in the \n",
      "skin of patients with systemic sclerosis.\n",
      "\n",
      "© 2021, American College of Rheumatology.\n",
      "\n",
      "DOI: 10.1002/art.41758\n",
      "PMID: 33844442 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "31. J Cell Mol Med. 2021 May;25(10):4684-4695. doi: 10.1111/jcmm.16407. Epub 2021\n",
      " Mar 22.\n",
      "\n",
      "A single-cell survey of the human glomerulonephritis.\n",
      "\n",
      "Chen Z(1), Zhang T(2), Mao K(2), Shao X(1), Xu Y(1), Zhu M(1), Zhou H(1), Wang \n",
      "Q(1), Li Z(1), Xie Y(1), Yuan X(3), Ying L(3), Zhang M(3), Hu J(4), Mou S(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Nephrology, Molecular Cell Laboratory for Kidney Disease, Ren \n",
      "Ji Hospital, School of Medicine, Shanghai Jiao Tong University, Shanghai, China.\n",
      "(2)Shanghai Institute of Immunology, Department of Microbiology and Immunology, \n",
      "Shanghai Jiao Tong University School of Medicine (SJTU-SM), Shanghai, China.\n",
      "(3)Transplantation Center of Ren Ji Hospital, School of Medicine, Shanghai Jiao \n",
      "Tong University, Shanghai, China.\n",
      "(4)Department of Nuclear Medicine, Ruijin Hospital, Shanghai Jiao Tong \n",
      "University School of Medicine, Shanghai, China.\n",
      "\n",
      "Glomerulonephritis is the one of the major causes of the end-stage kidney \n",
      "disease, whereas the pathological process of glomerulonephritis is still not \n",
      "completely understood. Single-cell RNA sequencing (scRNA-seq) emerges to be a \n",
      "powerful tool to evaluate the full heterogeneity of kidney diseases. To reveal \n",
      "cellular gene expression profiles of glomerulonephritis, we performed scRNA-seq \n",
      "of 2 human kidney transplantation donor samples, 4 human glomerulonephritis \n",
      "samples, 1 human malignant hypertension (MH) sample and 1 human chronic \n",
      "interstitial nephritis (CIN) sample, all tissues were taken from the biopsy. \n",
      "After filtering the cells with < 200 genes and > 10% mitochondria (MT) genes, \n",
      "the resulting 14 932 cells can be divided into 20 cell clusters, consistently \n",
      "with the previous report, in disease samples dramatic immune cells infiltration \n",
      "was found, among which a proximal tubule (PT) subset characterized by wnt-β \n",
      "catenin activation and a natural killer T (NKT) subset high expressing LTB were \n",
      "found. Furthermore, in the cluster of the podocyte, three glomerulonephritis \n",
      "related genes named FXYD5, CD74 and B2M were found. Compared with the mesangial \n",
      "of donor, the gene CLIC1 and RPS26 were up-regulated in mesangial of IgA \n",
      "nephropathy(IgAN), whereas the gene JUNB was up-regulated in podocyte of IgAN in \n",
      "comparison with that of donor. Meanwhile, some membranous nephropathy (MN) high \n",
      "expressed genes such as HLA-DRB5, HLA-DQA2, IFNG, CCL2 and NR4A2, which involve \n",
      "in highest enrichment pathway, display the cellular-specific expression style, \n",
      "whereas monocyte marker of lupus nephritis (LN) named TNFSF13B was also found \n",
      "and interferon alpha/beta signalling pathway was enriched in B and NKT of LN \n",
      "comparing with donor. By scRNA-seq, we first defined the podocyte markers of \n",
      "glomerulonephritis and specific markers in IgA, MN and LN were found at cellular \n",
      "level. Furthermore, the critical role of interferon alpha/beta signalling \n",
      "pathway was enriched in B and NKT of LN was declared.\n",
      "\n",
      "© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \n",
      "Foundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/jcmm.16407\n",
      "PMCID: PMC8107090\n",
      "PMID: 33754492 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "32. Front Genet. 2020 Dec 10;11:604324. doi: 10.3389/fgene.2020.604324.\n",
      "eCollection  2020.\n",
      "\n",
      "Integrative Analysis of Transcriptome-Wide Association Study and mRNA Expression \n",
      "Profiles Identifies Candidate Genes Associated With Idiopathic Pulmonary \n",
      "Fibrosis.\n",
      "\n",
      "Gong W(1), Guo P(1), Liu L(1), Guan Q(2)(3)(4), Yuan Z(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biostatistics, School of Public Health, Cheeloo College of \n",
      "Medicine, Shandong University, Jinan, China.\n",
      "(2)Department of Endocrinology, Shandong Provincial Hospital Affiliated to \n",
      "Shandong First Medical University, Jinan, China.\n",
      "(3)Shandong Clinical Medical Center of Endocrinology and Metabolism, Jinan, \n",
      "China.\n",
      "(4)Shandong Institute of Endocrine and Metabolic Diseases, Jinan, China.\n",
      "\n",
      "Idiopathic pulmonary fibrosis (IPF) is a type of scarring lung disease \n",
      "characterized by a chronic, progressive, and irreversible decline in lung \n",
      "function. The genetic basis of IPF remains elusive. A transcriptome-wide \n",
      "association study (TWAS) of IPF was performed by FUSION using gene expression \n",
      "weights of three tissues combined with a large-scale genome-wide association \n",
      "study (GWAS) dataset, totally involving 2,668 IPF cases and 8,591 controls. \n",
      "Significant genes identified by TWAS were then subjected to gene ontology (GO) \n",
      "and pathway enrichment analysis. The overlapped GO terms and pathways between \n",
      "enrichment analysis of TWAS significant genes and differentially expressed genes \n",
      "(DEGs) from the genome-wide mRNA expression profiling of IPF were also \n",
      "identified. For TWAS significant genes, protein-protein interaction (PPI) \n",
      "network and clustering modules analyses were further conducted using STRING and \n",
      "Cytoscape. Overall, TWAS identified a group of candidate genes for IPF under the \n",
      "Bonferroni corrected P value threshold (0.05/14929 = 3.35 × 10-6), such as DSP \n",
      "(P TWAS = 1.35 × 10-29 for lung tissue), MUC5B (P TWAS = 1.09 × 10-28 for lung \n",
      "tissue), and TOLLIP (P TWAS = 1.41 × 10-15 for whole blood). Pathway enrichment \n",
      "analysis identified multiple candidate pathways, such as herpes simplex \n",
      "infection (P value = 7.93 × 10-5) and antigen processing and presentation (P \n",
      "value = 6.55 × 10-5). 38 common GO terms and 8 KEGG pathways shared by \n",
      "enrichment analysis of TWAS significant genes and DEGs were identified. In the \n",
      "PPI network, 14 genes (DYNLL1, DYNC1LI1, DYNLL2, HLA-DRB5, HLA-DPB1, HLA-DQB2, \n",
      "HLA-DQA2, HLA-DQB1, HLA-DRB1, POLR2L, CENPP, CENPK, NUP133, and NUP107) were \n",
      "simultaneously detected by hub gene and module analysis. In conclusion, through \n",
      "integrative analysis of TWAS and mRNA expression profiles, we identified \n",
      "multiple novel candidate genes, GO terms and pathways for IPF, which contributes \n",
      "to the understanding of the genetic mechanism of IPF.\n",
      "\n",
      "Copyright © 2020 Gong, Guo, Liu, Guan and Yuan.\n",
      "\n",
      "DOI: 10.3389/fgene.2020.604324\n",
      "PMCID: PMC7758323\n",
      "PMID: 33362862\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "33. Medicine (Baltimore). 2020 Nov 25;99(48):e23477. doi: \n",
      "10.1097/MD.0000000000023477.\n",
      "\n",
      "CD74, a novel predictor for bronchopulmonary dysplasia in preterm infants.\n",
      "\n",
      "Gao J(1), Wu M(1), Wang F(1), Jiang L(1), Tian R(2), Zhu X(3), He S(3)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Pediatrics, Affiliated Hospital of Yangzhou University, \n",
      "Yangzhou, Jiangsu.\n",
      "(2)Department of Pediatrics, The First People's Hospital of Kunming City, \n",
      "Kunming, Yunnan.\n",
      "(3)Department of Neonatology, Children's Hospital of Soochow University, Suzhou, \n",
      "Jiangsu.\n",
      "(4)Department of Pediatrics, The First People's Hospital of Yunnan Province, \n",
      "Kunming, Yunnan, China.\n",
      "\n",
      "Bronchopulmonary dysplasia (BPD) remains a major complication and accounts for \n",
      "high morbidity and mortality of preterm infants. The present study aimed to \n",
      "identify the key genes in the development of BPD and to provide some new \n",
      "insights into the pathogenesis of BPD. The GSE108754 dataset was downloaded from \n",
      "Gene Expression Omnibus database containing 5 samples of BPD patients and 6 of \n",
      "non-BPD infants. The differentially expressed genes (DEGs) between BPD and \n",
      "non-BPD patients were identified by R software. The pathway and function \n",
      "enrichment analyses were performed through Database for Annotation Visualization \n",
      "and Integrated Discovery website. The protein-protein interaction network for \n",
      "DEGs was established by Cytoscape software and the most highly connected module \n",
      "was selected through MCODE plugin. Furthermore, the clinical sample verification \n",
      "among 25 BPD patients and 10 non-BPD infants was carried out in our center. \n",
      "Finally, based on the results above, the gene set enrichment analysis focusing \n",
      "on CD74 upregulated status was employed. Totally, 189 DEGs including 147 \n",
      "upregulated genes and 42 downregulated genes between BPD and non-BPD patients \n",
      "were screened out. The pathway and function enrichments revealed these DEGs were \n",
      "mainly enriched in asthma, intestinal immune network for IgA production, antigen \n",
      "processing and presentation and immune response. Thirteen DEGs (CD74, HLA-DMA, \n",
      "HLA-DRA, HLA-DMB, HLA-DOB, HLA-DQA1, HLA-DRB5, HLA-DPA1, HLA-DOA, HLA-DPB1, \n",
      "HLA-DQB2, HLA-DQA2, and HLA-DQB1) were determined as hub genes. The mRNA \n",
      "expression levels of the 13 hub genes were tested by quantitative real-time \n",
      "polymerase chain reaction among our clinical samples. Eventually, CD74 was \n",
      "confirmed to be the most significant highly expressed in BPD samples (P < .001) \n",
      "and its expression level was negatively correlated with gestational age \n",
      "(r = -0.653) and birth weight (r = -0.675). The gene set enrichment analysis \n",
      "results showed the gene sets associated with lupus erythematosus, viral \n",
      "myocarditis, immune network for IgA production, graft versus host disease, cell \n",
      "adhesion molecules and so no were differentially enriched with the phenotype of \n",
      "high-expression CD74. In conclusion, CD74 may serve to predict the BPD \n",
      "development and provide a new therapeutic target for BPD.\n",
      "\n",
      "DOI: 10.1097/MD.0000000000023477\n",
      "PMCID: PMC7710202\n",
      "PMID: 33235138 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have no conflicts of interest to \n",
      "disclose.\n",
      "\n",
      "\n",
      "34. Clin Respir J. 2021 Apr;15(4):374-381. doi: 10.1111/crj.13309. Epub 2020 Nov\n",
      "30.\n",
      "\n",
      "Major histocompatibility complex class II polymorphic variants are associated \n",
      "with asthma predisposition in the Punjabi population of Lahore, Pakistan.\n",
      "\n",
      "Aslam R(1), Shahid M(1), Bano I(2), Ayoub M(1), Sabar MF(3), Altaf S(1), Kousar \n",
      "S(1), Ghani MU(3), Husnain T(1), Shahid AA(1).\n",
      "\n",
      "Author information:\n",
      "(1)Centre of Excellence in Molecular Biology, University of the Punjab, Lahore, \n",
      "Pakistan.\n",
      "(2)The Children's Hospital and The Institute of Child Health, Pediatric \n",
      "Pulmonology, Lahore, Pakistan.\n",
      "(3)Centre for Applied Molecular Biology, University of the Punjab, Lahore, \n",
      "Pakistan.\n",
      "\n",
      "INTRODUCTION: Various genome wide association studies have manifested that Major \n",
      "Histocompatibility Complex (MHC) region on chromosome 6p21 houses many potential \n",
      "candidate genes for asthma.\n",
      "OBJECTIVE: This Case-Control association study was planned to determine the \n",
      "association of 10 Single Nucleotide Polymorphisms (SNPs), residing within and \n",
      "around MHC genes' region on chromosome 6p21, with Asthma in Punjabi population \n",
      "of Lahore, Pakistan.\n",
      "METHODS: A total of 161 subjects, 61 physician-diagnosed asthma patients and 100 \n",
      "age-matched healthy controls, were recruited from Lahore, a city in Punjab. Ten \n",
      "single nucleotide polymorphisms (SNPs) (rs9378249, rs2070600, rs404860, rs6689, \n",
      "rs1049124, rs1063355, rs1049225, rs1049219, rs7773955 and rs928976) located \n",
      "within or near AGER, NOTCH and HLA genes in MHC region, were genotyped in both \n",
      "patients and controls using single base extension reaction and capillary \n",
      "electrophoresis-based genetic analyser. Statistical models were applied using \n",
      "SHEsis Plus. Results were adjusted for various cofactors (age, gender and \n",
      "environment) and by applying multiple corrections. Haplotype and linkage \n",
      "disequilibrium analyses were performed on Haploview software v4.1.\n",
      "RESULTS: Three of the studied SNPs rs1049124, rs1049219 and rs7773955 show \n",
      "independent significant association with asthma under allelic and genotypic \n",
      "models. Two of the haplotypes, H7 and H13, \"CTAATTT\" and \"CCACTAT\", \n",
      "respectively, for rs2070600, rs404860, rs6689, rs1049124, rs1063355, rs1049219 \n",
      "and rs7773955, are found to be significantly associated with the disease.\n",
      "CONCLUSION: This study reports association of SNP variants residing on HLA-DQB1 \n",
      "and HLA-DQA2 genes and haplotypes H7 and H13 on genomic region 6p21 with Asthma \n",
      "in the Punjabi population of Lahore, Pakistan.\n",
      "\n",
      "© 2020 John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/crj.13309\n",
      "PMID: 33211378 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "35. J Allergy Clin Immunol. 2021 Mar;147(3):933-940. doi: \n",
      "10.1016/j.jaci.2020.08.020. Epub 2020 Sep 2.\n",
      "\n",
      "A genome-wide association study of asthma hospitalizations in adults.\n",
      "\n",
      "Yan Q(1), Forno E(1), Herrera-Luis E(2), Pino-Yanes M(3), Yang G(1), Oh S(4), \n",
      "Acosta-Pérez E(5), Hu D(4), Eng C(4), Huntsman S(4), Rodriguez-Santana JR(6), \n",
      "Cloutier MM(7), Canino G(5), Burchard EG(4), Chen W(1), Celedón JC(8).\n",
      "\n",
      "Author information:\n",
      "(1)Division of Pediatric Pulmonary Medicine, UPMC Children's Hospital of \n",
      "Pittsburgh, University of Pittsburgh, Pittsburgh, Pa.\n",
      "(2)Genomics and Health Group, Department of Biochemistry, Microbiology, Cell \n",
      "Biology and Genetics, Universidad de La Laguna, La Laguna, Spain.\n",
      "(3)Genomics and Health Group, Department of Biochemistry, Microbiology, Cell \n",
      "Biology and Genetics, Universidad de La Laguna, La Laguna, Spain; CIBER de \n",
      "Enfermedades Respiratorias, Instituto de Salud Carlos III, Spain.\n",
      "(4)Department of Medicine, University of California San Francisco, San \n",
      "Francisco, Calif.\n",
      "(5)Behavioral Sciences Research Institute, University of Puerto Rico, San Juan, \n",
      "Puerto Rico.\n",
      "(6)Centro de Neumología Pediátrica, San Juan, Puerto Rico.\n",
      "(7)Department of Pediatrics, University of Connecticut, Farmington, Conn.\n",
      "(8)Division of Pediatric Pulmonary Medicine, UPMC Children's Hospital of \n",
      "Pittsburgh, University of Pittsburgh, Pittsburgh, Pa. Electronic address: \n",
      "juan.celedon@chp.edu.\n",
      "\n",
      "BACKGROUND: Little is known about the genetic determinants of severe asthma \n",
      "exacerbations.\n",
      "OBJECTIVES: We aimed to identify genetic variants associated with asthma \n",
      "hospitalizations.\n",
      "METHODS: We conducted a genome-wide association study of asthma hospitalizations \n",
      "in 34,167 white British adults with asthma, 1,658 of whom had at least 1 \n",
      "asthma-related hospitalization. This analysis was conducted by using logistic \n",
      "regression under an additive genetic model with adjustment for age, sex, body \n",
      "mass index, smoking status, and the first 5 principal components derived from \n",
      "genotypic data. We then analyzed data from 2 cohorts of Latino children and \n",
      "adolescents for replication and conducted quantitative trait locus and \n",
      "functional annotation analyses.\n",
      "RESULTS: At the chromosome 6p21.3 locus, the single-nucleotide polymorphism \n",
      "(SNP) rs56151658 (8 kb from the promoter of HLA-DQB1) was most significantly \n",
      "associated with asthma hospitalizations (for test allele A, odds ratio = 1.36 \n",
      "[95% CI = 1.22-1.52]; P = 3.11 × 10-8); 21 additional SNPs in this locus were \n",
      "associated with asthma hospitalizations at a P value less than 1 × 10-6. In the \n",
      "replication cohorts, multiple SNPs in strong linkage disequilibrium with \n",
      "rs56151658 were associated with severe asthma exacerbations at a P value of .01 \n",
      "or less in the same direction of association as in the discovery cohort. Three \n",
      "HLA genes (HLA-DQA2, HLA-DRB6, and HLA-DOB) were also shown to mediate the \n",
      "estimated effects of the SNPs associated with asthma hospitalizations through \n",
      "effects on gene expression in lung tissue.\n",
      "CONCLUSIONS: We identified strong candidate genes for asthma hospitalizations in \n",
      "adults in the region for class II HLA genes through genomic, quantitative trait \n",
      "locus, and summary data-based mendelian randomization analyses.\n",
      "\n",
      "Copyright © 2020 American Academy of Allergy, Asthma & Immunology. Published by \n",
      "Elsevier Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jaci.2020.08.020\n",
      "PMCID: PMC7921212\n",
      "PMID: 32890573 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "36. Kidney Int. 2021 Mar;99(3):671-685. doi: 10.1016/j.kint.2020.08.007. Epub\n",
      "2020  Sep 2.\n",
      "\n",
      "HLA-D and PLA2R1 risk alleles associate with recurrent primary membranous \n",
      "nephropathy in kidney transplant recipients.\n",
      "\n",
      "Berchtold L(1), Letouzé E(2), Alexander MP(3), Canaud G(4), Logt AV(5), Hamilton \n",
      "P(6), Mousson C(7), Vuiblet V(8), Moyer AM(9), Guibert S(10), Mrázová P(11), \n",
      "Levi C(12), Dubois V(13), Cruzado JM(14), Torres A(15), Gandhi MJ(16), Yousfi \n",
      "N(17), Tesar V(18), OndrejViklický(19), Hourmant M(20), Moulin B(21), Rieu \n",
      "P(22), Choukroun G(23), Legendre C(24), Wetzels J(5), Brenchley P(6), Ballarín \n",
      "Castan JA(25), Debiec H(17), Ronco P(26).\n",
      "\n",
      "Author information:\n",
      "(1)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France; Division of Nephrology, Department of \n",
      "Internal Medicine Specialties, University Hospital of Geneva, Geneva, \n",
      "Switzerland.\n",
      "(2)Centre de Recherche des Cordeliers, Sorbonne Université, INSERM, France; \n",
      "Functional Genomics of Solid Tumor, Labex Immuno-Oncology, Equipe Labellisée \n",
      "Ligue Contre le Cancer, Université Paris 13, Paris, France.\n",
      "(3)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, \n",
      "Minnesota, USA.\n",
      "(4)Inserm U1151, Necker-Enfants Malades Hospital, Paris, France; University \n",
      "Paris Descartes, Sorbonne Paris Cité, Paris, France; Department of Adult \n",
      "Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, France.\n",
      "(5)Department of Nephrology, Radboud University Medical Center, Radboud \n",
      "Institute for Health Sciences, Nijmegen, The Netherlands.\n",
      "(6)Division of Cardiovascular Sciences, School of Medical Sciences, Faculty of \n",
      "Biology, Medicine and Health, University of Manchester, Manchester, UK.\n",
      "(7)Department of Nephrology and Transplantation, University Hospital, Dijon, \n",
      "France.\n",
      "(8)BioSpec T Laboratory, EA 7506 URCA, Reims, France; Nephropathology Department \n",
      "of Biopathology Laboratory, Reims University Hospital, Reims, France; Division \n",
      "of Nephrology, Reims University Hospital, Reims, France.\n",
      "(9)Department of Laboratory Medicine and Pathology, Personalized Genomics \n",
      "Laboratory Mayo Clinic, Rochester, Minnesota, USA.\n",
      "(10)IntegraGen, Evry, France.\n",
      "(11)Transplant Laboratory, Institute for Clinical and Experimental Medicine, \n",
      "Prague, Czech Republic.\n",
      "(12)Department of Transplantation, Nephrology, and Clinical Immunology, Edouard \n",
      "Herriot Hospital, Hospices Civils de Lyon, Lyon, France.\n",
      "(13)Laboratoire HLA, Etablissement Français du Sang Auvergne Rhone-Alpes, Lyon, \n",
      "France.\n",
      "(14)Nephrology Department, Hospital Universitari Bellvitge, Bellvitge Research \n",
      "Institute (IDIBELL), Barcelona, Spain; RedInRen, RD16/0009/0031, University of \n",
      "Barcelona, L'Hospitalet de Llobregat, Barcelona, Spain.\n",
      "(15)Hospital Universitario de Canarias, Instituto de Tecnologías Biomédicas \n",
      "(ITB), Universidad de La Laguna, Tenerife, Spain.\n",
      "(16)Division of Transfusion Medicine, Mayo Clinic, Rochester, Minnesota, USA.\n",
      "(17)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France.\n",
      "(18)Department of Nephrology, First Faculty of Medicine, Charles University and \n",
      "General University Hospital, Prague, Czech Republic.\n",
      "(19)Department of Nephrology, Transplant Center, Institute for Clinical and \n",
      "Experimental Medicine, Prague, Czech Republic.\n",
      "(20)Department of Nephrology, University Hospital, Nantes, France.\n",
      "(21)Institut National de la Santé et de la Recherche Médicale, Unité Mixte de \n",
      "Recherche S 1109, Strasbourg University, Strasbourg, France; Fédération de \n",
      "Médecine Translationnelle de Strasbourg (FMTS), Strasbourg, France; Department \n",
      "of Nephrology, Strasbourg University Hospital, Strasbourg, France.\n",
      "(22)Division of Nephrology, Reims University Hospital, Reims, France; Université \n",
      "de Reims Champagne-Ardenne, CNRS UMR 7369 (Matrice Extracellulaire et Dynamique \n",
      "Cellulaire, MEDyC), Reims, France.\n",
      "(23)Department of Nephrology, University Hospital, Amiens, France.\n",
      "(24)University Paris Descartes, Sorbonne Paris Cité, Paris, France; Department \n",
      "of Adult Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, \n",
      "France.\n",
      "(25)Department of Nephrology, Fundación Puigvert, Barcelona, Spain.\n",
      "(26)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France; Department of Nephrology (Day Hospital), \n",
      "AP-HP, Tenon Hospital, Paris, France. Electronic address: pierre.ronco@upmc.fr.\n",
      "\n",
      "Erratum in\n",
      "    Kidney Int. 2021 Jul;100(1):243.\n",
      "\n",
      "Comment in\n",
      "    Kidney Int. 2021 Mar;99(3):545-548.\n",
      "\n",
      "Recurrence of primary membranous nephropathy after transplantation occurs in up \n",
      "to 44% of patients and is driven by PLA2R antibody. Here, we asked whether \n",
      "genetic determinants could improve risk prediction. First, we sequenced PLA2R1 \n",
      "and HLA-D loci in 248 patients with primary membranous nephropathy and \n",
      "identified two independent single nucleotide polymorphisms (SNPs) at risk for \n",
      "primary membranous nephropathy at each locus. These were rs9271188 (intergenic \n",
      "between HLA-DRB1 and HLA-DQA1,) and rs9275086 (intergenic between HLA-DQB1 and \n",
      "HLA-DQA2) at the HLA-D locus along with rs6726925 and rs13018963 at the PLA2R1 \n",
      "locus. Then we investigated whether primary membranous nephropathy at-risk \n",
      "variants were associated with recurrence in a retrospective cohort of 105 \n",
      "donor-recipient pairs and a replication cohort of 40 pairs. Seven SNPs located \n",
      "between HLA-DRB1 and HLA-DQA1 in linkage disequilibrium with rs9271188, and \n",
      "three SNPs in the PLA2R1 region predicted recurrence when presented by the \n",
      "donor, but not when presented by the recipient. The two SNPs in the HLA-D region \n",
      "most strongly associated with recurrence (rs9271705 and rs9271550) were \n",
      "confirmed in the replication cohort. A genetic risk score based on the two best \n",
      "predictors at each locus (rs9271705, rs9271550, rs17830558, and rs3828323) \n",
      "identified a group of patients with high risk of recurrence. Thus, our results \n",
      "suggest that the graft contributes to recurrence of primary membranous \n",
      "nephropathy through the disease susceptibility HLA-D and PLA2R1 SNPs in an \n",
      "autoimmune milieu. Further studies are needed before implementation of genetic \n",
      "testing for these in donor selection.\n",
      "\n",
      "Copyright © 2020. Published by Elsevier Inc.\n",
      "\n",
      "DOI: 10.1016/j.kint.2020.08.007\n",
      "PMID: 32889013 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "37. Front Pharmacol. 2020 Jul 24;11:1138. doi: 10.3389/fphar.2020.01138.\n",
      "eCollection  2020.\n",
      "\n",
      "Discovery of Selenocysteine as a Potential Nanomedicine Promotes Cartilage \n",
      "Regeneration With Enhanced Immune Response by Text Mining and Biomedical \n",
      "Databases.\n",
      "\n",
      "Ye J(1), Xu B(1), Fan B(2), Zhang J(1), Yuan F(1), Chen Y(1), Sun Z(1), Yan \n",
      "X(1), Song Y(1), Song S(1), Yang M(2), Yu JK(1).\n",
      "\n",
      "Author information:\n",
      "(1)Knee Surgery Department of the Institution of Sports Medicine, Peking \n",
      "University Third Hospital, Beijing Key Laboratory of Sports Injuries, Beijing, \n",
      "China.\n",
      "(2)School of Clinical Medicine, Weifang Medical University, Weifang, China.\n",
      "\n",
      "BACKGROUND: Unlike bone tissue, little progress has been made regarding \n",
      "cartilage regeneration, and many challenges remain. Furthermore, the key roles \n",
      "of cartilage lesion caused by traumas, focal lesion, or articular overstress \n",
      "remain unclear. Traumatic injuries to the meniscus as well as its degeneration \n",
      "are important risk factors for long-term joint dysfunction, degenerative joint \n",
      "lesions, and knee osteoarthritis (OA) a chronic joint disease characterized by \n",
      "degeneration of articular cartilage and hyperosteogeny. Nearly 50% of the \n",
      "individuals with meniscus injuries develop OA over time. Due to the limited \n",
      "inherent self-repair capacity of cartilage lesion, the Biomaterial \n",
      "drug-nanomedicine is considered to be a promising alternative. Therefore, it is \n",
      "important to elucidate the gene potential regeneration mechanisms and discover \n",
      "novel precise medication, which are identified through this study to investigate \n",
      "their function and role in pathogenesis.\n",
      "METHODS: We downloaded the mRNA microarray statistics GSE117999, involving \n",
      "paired cartilage lesion tissue samples from 12 OA patients and 12 patients from \n",
      "a control group. First, we analyzed these statistics to recognize the \n",
      "differentially expressed genes (DEGs). We then exposed the gene ontology (GO) \n",
      "annotation and the Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathway \n",
      "enrichment analyses for these DEGs. Protein-protein interaction (PPI) networks \n",
      "were then constructed, from which we attained eight significant genes after a \n",
      "functional interaction analysis. Finally, we identified a potential nanomedicine \n",
      "attained from this assay set, using a wide range of inhibitor information \n",
      "archived in the Search Tool for the Retrieval of Interacting Genes (STRING) \n",
      "database.\n",
      "RESULTS: Sixty-six DEGs were identified with our standards for meaning (adjusted \n",
      "P-value < 0.01, |log2 - FC| ≥1.2). Furthermore, we identified eight hub genes \n",
      "and one potential nanomedicine - Selenocysteine based on these integrative data.\n",
      "CONCLUSION: We identified eight hub genes that could work as prospective \n",
      "biomarkers for the diagnostic and biomaterial drug treatment of cartilage \n",
      "lesion, involving the novel genes CAMP, DEFA3, TOLLIP, HLA-DQA2, SLC38A6, \n",
      "SLC3A1, FAM20A, and ANO8. Meanwhile, these genes were mainly associated with \n",
      "immune response, immune mediator induction, and cell chemotaxis. Significant \n",
      "support is provided for obtaining a series of novel gene targets, and we \n",
      "identify potential mechanisms for cartilage regeneration and final nanomedicine \n",
      "immunotherapy in regenerative medicine.\n",
      "\n",
      "Copyright © 2020 Ye, Xu, Fan, Zhang, Yuan, Chen, Sun, Yan, Song, Song, Yang and \n",
      "Yu.\n",
      "\n",
      "DOI: 10.3389/fphar.2020.01138\n",
      "PMCID: PMC7394085\n",
      "PMID: 32792959\n",
      "\n",
      "\n",
      "38. Sci Rep. 2020 Jul 2;10(1):10868. doi: 10.1038/s41598-020-67730-y.\n",
      "\n",
      "Synovial cell cross-talk with cartilage plays a major role in the pathogenesis \n",
      "of osteoarthritis.\n",
      "\n",
      "Chou CH(#)(1), Jain V(#)(1), Gibson J(1), Attarian DE(2), Haraden CA(1), Yohn \n",
      "CB(3), Laberge RM(3), Gregory S(1), Kraus VB(4)(5).\n",
      "\n",
      "Author information:\n",
      "(1)Duke Molecular Physiology Institute, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA.\n",
      "(2)Department of Orthopaedic Surgery, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA.\n",
      "(3)Unity Biotechnology, South San Francisco, CA, USA.\n",
      "(4)Duke Molecular Physiology Institute, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA. vbk@duke.edu.\n",
      "(5)Division of Rheumatology, Department of Medicine, Duke University School of \n",
      "Medicine, Durham, NC, 27701, USA. vbk@duke.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "We elucidated the molecular cross-talk between cartilage and synovium in \n",
      "osteoarthritis, the most widespread arthritis in the world, using the powerful \n",
      "tool of single-cell RNA-sequencing. Multiple cell types were identified based on \n",
      "profiling of 10,640 synoviocytes and 26,192 chondrocytes: 12 distinct synovial \n",
      "cell types and 7 distinct articular chondrocyte phenotypes from matched tissues. \n",
      "Intact cartilage was enriched for homeostatic and hypertrophic chondrocytes, \n",
      "while damaged cartilage was enriched for prefibro- and fibro-, regulatory, \n",
      "reparative and prehypertrophic chondrocytes. A total of 61 cytokines and growth \n",
      "factors were predicted to regulate the 7 chondrocyte cell phenotypes. Based on \n",
      "production by > 1% of cells, 55% of the cytokines were produced by synovial \n",
      "cells (39% exclusive to synoviocytes and not expressed by chondrocytes) and \n",
      "their presence in osteoarthritic synovial fluid confirmed. The synoviocytes \n",
      "producing IL-1beta (a classic pathogenic cytokine in osteoarthritis), mainly \n",
      "inflammatory macrophages and dendritic cells, were characterized by \n",
      "co-expression of surface proteins corresponding to HLA-DQA1, HLA-DQA2, OLR1 or \n",
      "TLR2. Strategies to deplete these pathogenic intra-articular cell subpopulations \n",
      "could be a therapeutic option for human osteoarthritis.\n",
      "\n",
      "DOI: 10.1038/s41598-020-67730-y\n",
      "PMCID: PMC7331607\n",
      "PMID: 32616761 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: C.H.C., V.J., J.G., D.E.A., C.A.H. and S.G. \n",
      "declare no competing interests. C.B.Y. and R.M.L. are employees of Unity \n",
      "Biotechnology and played a role in the conceptualization of the study and in \n",
      "preparation of the manuscript. This work was funded in part by Unity \n",
      "Biotechnology, however, Unity Biotechnology had no role in the preparation of \n",
      "the manuscript, data collection or decision to publish this study. V.B.K. has \n",
      "been a periodic consultant for Unity Biotechnology providing periodic advice on \n",
      "clinical study design, unrelated to this study and has no competing interests to \n",
      "declare.\n",
      "\n",
      "\n",
      "39. Medicina (Kaunas). 2020 Mar 31;56(4):154. doi: 10.3390/medicina56040154.\n",
      "\n",
      "An Intergenic rs9275596 Polymorphism on Chr. 6p21 Is Associated with Multiple \n",
      "Sclerosis in Latvians.\n",
      "\n",
      "Paramonova N(1), Trapina I(1), Dokane K(1), Kalnina J(1), Sjakste T(1), Sjakste \n",
      "N(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Genomics and Bioinformatics, Institute of Biology of the University of \n",
      "Latvia, LV-1004 Riga, Latvia.\n",
      "(2)Department of Medical Biochemistry of the University of Latvia, LV-1004 Riga, \n",
      "Latvia.\n",
      "\n",
      "Background and objectives: Multiple sclerosis (MS) is a chronic inflammatory \n",
      "disease of the central nervous system, leading to demyelination of neurons and \n",
      "potentially debilitating physical and mental symptoms. The disease is more \n",
      "prevalent in women than in men. The major histocompatibility complex (MHC) \n",
      "region has been identified as a major genetic determinant for autoimmune \n",
      "diseases, and its role in some neurological disorders including MS was \n",
      "evaluated. An intergenic single-nucleotide polymorphism (SNP), rs9275596, \n",
      "located between the HLA-DQB1 and HLA-DQA2 genes, is in significant association \n",
      "with various autoimmune diseases according to genome-wide association studies \n",
      "(GWASs). A cumulative effect of this SNP with other polymorphisms from this \n",
      "region was revealed. The aim of the study was to verify the data on rs9275596 \n",
      "association in multiple sclerosis in a case/control study of the Latvian \n",
      "population and to evaluate eventual functional significance of allele \n",
      "substitutions. Materials and Methods: rs9275596 (chr6:32713854; GRCh38.p12) was \n",
      "genotyped in 273 MS patients and 208 controls on main and sex-specific \n",
      "associations. Eventual functional significance of allele substitutions was \n",
      "evaluated in silico using publicly available tools. Results: The rs9275596 rare \n",
      "alleles were identified as a disease susceptibility factor in association with \n",
      "the MS main group and in affected females (p < 0.001 and p < 0.01, \n",
      "respectively). Risk factor genotypes with rare alleles included were associated \n",
      "with the MS common cohort (p < 0.002) and female cohort (odds ratio, OR = 2.24) \n",
      "and were identified as disease susceptible in males (OR = 2.41). It was shown \n",
      "that structural changes of rs9275596 affect the secondary structure of DNA. \n",
      "Functional significance of allele substitutions was evaluated on the eventual \n",
      "sequence affinity to transcription factors (TFs) and splicing signals \n",
      "similarity. A possible impact of the particular polymorphisms on the \n",
      "transcription and splicing efficiency is discussed. Conclusions: Our results \n",
      "suggest susceptibility of rs9275596 to multiple sclerosis in Latvians.\n",
      "\n",
      "DOI: 10.3390/medicina56040154\n",
      "PMCID: PMC7230508\n",
      "PMID: 32244438 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no conflict \n",
      "of interest.\n",
      "\n",
      "\n",
      "40. Hum Immunol. 2020 May;81(5):202-205. doi: 10.1016/j.humimm.2020.02.003. Epub \n",
      "2020 Feb 28.\n",
      "\n",
      "A new set of reagents and related software used for NGS based classical and \n",
      "non-classical HLA typing showing evidence for a greater HLA haplotype diversity.\n",
      "\n",
      "Alizadeh M(1), Picard C(2), Frassati C(3), Walencik A(4), Gauthier AC(4), \n",
      "Bennasar F(5), Verite F(5), Semana G(5).\n",
      "\n",
      "Author information:\n",
      "(1)National Reagent Production Laboratory - Établissement Français du Sang \n",
      "Bretagne, Rennes, France. Electronic address: mehdi.alizadeh@efs.sante.fr.\n",
      "(2)Department of Histocompatibility - Établissement Français du Sang Alpes \n",
      "Méditerranée, Marseille, France; UMR 7268 ADÉS Aix-Marseille Université \n",
      "/EFS/CNRS, Marseille, France.\n",
      "(3)Department of Histocompatibility - Établissement Français du Sang Alpes \n",
      "Méditerranée, Marseille, France.\n",
      "(4)Department of Histocompatibility - Établissement Français du Sang Pays de la \n",
      "Loire, Nantes, France.\n",
      "(5)National Reagent Production Laboratory - Établissement Français du Sang \n",
      "Bretagne, Rennes, France.\n",
      "\n",
      "Erratum in\n",
      "    Hum Immunol. 2020 Aug;81(8):460.\n",
      "\n",
      "To evaluate the HLA typing performance of a new Long-Range PCR NGS set of \n",
      "reagents and its dedicated software, a panel of 41 reference homozygous cell \n",
      "lines from the International Histocompatibility Working Group (IHWG) and a panel \n",
      "of 376 volunteer bone marrow donors were analyzed for classical and \n",
      "non-classical HLA class I and class II genes. All results, except HLA-DPB1, were \n",
      "obtained without any ambiguities at the 3rd field level. Based on the high \n",
      "resolution performance of the reagents, a number of new alleles have been \n",
      "described not only for classical but also for non-classical HLA class I genes, \n",
      "leading to a more accurate haplotype definition. Linkage disequilibrium between \n",
      "HLA-A and HLA-G genes has been defined at 4th field level of resolution. \n",
      "Moreover, for the first time, HLA-DQA2 and DQB2 polymorphisms and their linkage \n",
      "disequilibrium with DQB1 were described.\n",
      "\n",
      "Copyright © 2020 American Society for Histocompatibility and Immunogenetics. \n",
      "Published by Elsevier Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.humimm.2020.02.003\n",
      "PMID: 32122686 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "41. J Clin Lab Anal. 2020 Feb;34(2):e23066. doi: 10.1002/jcla.23066. Epub 2019\n",
      "Oct  12.\n",
      "\n",
      "Association of GWAS-supported noncoding area loci rs404860, rs3117098, and \n",
      "rs7775228 with asthma in Chinese Zhuang population.\n",
      "\n",
      "Liang SQ(1), Deng JM(1), Wei X(1), Chen ZR(1), Yang ML(1), Qin HJ(1), Zhang \n",
      "JQ(1), He ZY(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Respiratory and Critical Care Medicine, First Affiliated \n",
      "Hospital of Guangxi Medical University, Nanning, China.\n",
      "\n",
      "BACKGROUND: Asthma is a complicated and polygenic inheritance disease, and its \n",
      "prevalence increases worldwide. Recent genome-wide association studies (GWASs) \n",
      "identified a significant association of single nucleotide polymorphism with \n",
      "asthma in the Japanese population. This study aimed to examine the association \n",
      "of GWAS-supported noncoding area loci, namely rs404860, rs3117098, and \n",
      "rs7775228, with asthma in Chinese Zhuang population.\n",
      "METHODS: A case-control study involving 223 individuals, comprising 123 patients \n",
      "with asthma and 100 healthy controls, was conducted. Genotypes were determined \n",
      "by polymerase chain reaction (PCR)/ligase detection reaction assay. The \n",
      "association between gene polymorphisms and asthma risk was calculated by \n",
      "logistic regression analysis using different genetic models through comparisons \n",
      "of alleles (A vs a), homozygote genotypes (AA vs aa), heterozygote genotypes (Aa \n",
      "vs aa), dominant models (AA+Aa vs aa), and recessive models (AA vs. Aa+aa).\n",
      "RESULTS: The distribution of the genotype frequency of rs3117098 was \n",
      "statistically different between the case and control groups. For rs3117098, \n",
      "significant associations were observed through comparisons of alleles (OR: \n",
      "1.832, 95% CI: 1.048-3.204, P = .034) and dominant models (OR: 2.065, 95% CI: \n",
      "1.001-4.260, P = .050). The statistical analysis showed no significant \n",
      "difference for loci rs404860 and rs7775228 between patients with asthma and \n",
      "controls.\n",
      "CONCLUSION: rs3117098 may be the risk factor for asthma in Chinese Zhuang \n",
      "population.\n",
      "\n",
      "© 2019 The Authors. Journal of Clinical Laboratory Analysis published by Wiley \n",
      "Periodicals, Inc.\n",
      "\n",
      "DOI: 10.1002/jcla.23066\n",
      "PMCID: PMC7031573\n",
      "PMID: 31605414 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: All authors report no conflicts of interest.\n",
      "\n",
      "\n",
      "42. Clin Exp Immunol. 2019 Dec;198(3):367-380. doi: 10.1111/cei.13367. Epub 2019\n",
      "Oct  1.\n",
      "\n",
      "Pneumonia: host susceptibility and shared genetics with pulmonary function and \n",
      "other traits.\n",
      "\n",
      "Khadzhieva MB(1)(2)(3), Kuzovlev AN(1), Salnikova LE(1)(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Federal Research and Clinical Center of Intensive Care Medicine and \n",
      "Rehabilitology, Moscow, Russia.\n",
      "(2)N. I. Vavilov Institute of General Genetics, Russian Academy of Sciences, \n",
      "Moscow, Russia.\n",
      "(3)Dmitry Rogachev National Research Center of Pediatric Hematology, Oncology \n",
      "and Immunology, Moscow, Russia.\n",
      "\n",
      "Pneumonia is a common and severe infectious lung disease. Host genetics, \n",
      "together with underlying medical and lifestyle conditions, determine pneumonia \n",
      "susceptibility. We performed a secondary analysis of the results of two \n",
      "genome-wide studies for pneumonia in 23andMe participants (40 600 cases/90 039 \n",
      "controls) (Tian et al., 2017) and UK Biobank (BB) participants (12 614 \n",
      "cases/324 585 controls) (via the Global Biobank Engine) and used the GTEx \n",
      "database to correlate the results with expression quantitative trait loci \n",
      "(eQTLs) data in lung and whole blood. In the 23andMe pneumonia single nucleotide \n",
      "polymorphism (SNP) set, 177 genotyped SNPs in the human leukocyte antigen (HLA) \n",
      "region satisfied the genome-wide significance level, P ≤ 5·0E-08. Several target \n",
      "genes (e.g. C4A, VARS2, SFTA2, HLA-C, HLA-DQA2) were unidirectionally regulated \n",
      "by many HLA eSNPs associated with a higher risk of pneumonia. In lung, C4A \n",
      "transcript was up-regulated by 291 pneumonia risk alleles spanning the half the \n",
      "HLA region. Among SNPs correlated with the expression levels of SFTA2 and VARS2, \n",
      "approximately 75% overlapped: all risk alleles were associated with VARS2 \n",
      "up-regulation and SFTA2 down-regulation. To find shared gene loci between \n",
      "pneumonia and pulmonary function (PF), we used data from the Global Biobank \n",
      "Engine and literature on genome-wide association studies (GWAS) of PF in general \n",
      "populations. Numerous gene loci overlapped between pneumonia and PF: 28·8% in \n",
      "the BB data set and 49·2% in the 23andMe data set. Enrichment analysis within \n",
      "the database of Genotypes and Phenotypes (dbGaP) and National Human Genome \n",
      "Research Institute-European Bioinformatics Institute (NHGRI-EBI) Catalog of \n",
      "pneumonia and pneumonia/PF gene sets identified significant overlap between \n",
      "these gene sets and genes related to inflammatory, developmental, \n",
      "neuropsychiatric and cardiovascular and obesity-related traits.\n",
      "\n",
      "© 2019 British Society for Immunology.\n",
      "\n",
      "DOI: 10.1111/cei.13367\n",
      "PMCID: PMC6857187\n",
      "PMID: 31487037 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing financial \n",
      "interests.\n",
      "\n",
      "\n",
      "43. Acta Neuropathol. 2019 Jun;137(6):879-899. doi: 10.1007/s00401-019-01962-9.\n",
      "Epub  2019 Feb 9.\n",
      "\n",
      "Genome-wide analyses as part of the international FTLD-TDP whole-genome \n",
      "sequencing consortium reveals novel disease risk factors and increases support \n",
      "for immune dysfunction in FTLD.\n",
      "\n",
      "Pottier C(1), Ren Y(2), Perkerson RB 3rd(1), Baker M(1), Jenkins GD(3), van \n",
      "Blitterswijk M(1), DeJesus-Hernandez M(1), van Rooij JGJ(4), Murray ME(1), \n",
      "Christopher E(1), McDonnell SK(3), Fogarty Z(3), Batzler A(3), Tian S(3), \n",
      "Vicente CT(1), Matchett B(1), Karydas AM(5), Hsiung GR(6), Seelaar H(4), Mol \n",
      "MO(4), Finger EC(7), Graff C(8)(9), Öijerstedt L(8)(9), Neumann M(10)(11), \n",
      "Heutink P(10)(12), Synofzik M(10)(12), Wilke C(10)(12), Prudlo J(10)(13), Rizzu \n",
      "P(10), Simon-Sanchez J(10)(12), Edbauer D(14)(15), Roeber S(16), Diehl-Schmid \n",
      "J(17), Evers BM(18), King A(19)(20), Mesulam MM(21), Weintraub S(21)(22), Geula \n",
      "C(21), Bieniek KF(1)(23), Petrucelli L(1), Ahern GL(24), Reiman EM(25), Woodruff \n",
      "BK(26), Caselli RJ(26), Huey ED(27), Farlow MR(28), Grafman J(29), Mead S(30), \n",
      "Grinberg LT(5)(31), Spina S(5), Grossman M(32), Irwin DJ(32), Lee EB(33), Suh \n",
      "E(33), Snowden J(34), Mann D(35), Ertekin-Taner N(1)(36), Uitti RJ(36), Wszolek \n",
      "ZK(36), Josephs KA(37), Parisi JE(37), Knopman DS(37), Petersen RC(37), Hodges \n",
      "JR(38), Piguet O(39), Geier EG(5), Yokoyama JS(5), Rissman RA(40)(41), Rogaeva \n",
      "E(42), Keith J(43)(44), Zinman L(43), Tartaglia MC(42)(45), Cairns NJ(46), \n",
      "Cruchaga C(47), Ghetti B(48), Kofler J(49), Lopez OL(24)(50), Beach TG(51), \n",
      "Arzberger T(14)(16)(52), Herms J(14)(16), Honig LS(53), Vonsattel JP(54), \n",
      "Halliday GM(38)(55), Kwok JB(38)(55), White CL 3rd(18), Gearing M(56), Glass \n",
      "J(56), Rollinson S(57), Pickering-Brown S(57), Rohrer JD(58), Trojanowski \n",
      "JQ(33), Van Deerlin V(33), Bigio EH(21), Troakes C(19), Al-Sarraj S(19)(20), \n",
      "Asmann Y(2), Miller BL(5), Graff-Radford NR(36), Boeve BF(37), Seeley WW(5)(31), \n",
      "Mackenzie IRA(59), van Swieten JC(4), Dickson DW(1), Biernacka JM(3), Rademakers \n",
      "R(60).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neuroscience, Mayo Clinic, 4500 San Pablo Road, Jacksonville, \n",
      "FL, 32224, USA.\n",
      "(2)Department of Health Sciences Research, Mayo Clinic, Jacksonville, FL, USA.\n",
      "(3)Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA.\n",
      "(4)Department of Neurology, Erasmus Medical Center, Wytemaweg 80, 3015 CN, \n",
      "Rotterdam, The Netherlands.\n",
      "(5)Department of Neurology, Memory and Aging Center, University of California, \n",
      "San Francisco, CA, USA.\n",
      "(6)Division of Neurology, Department of Medicine, University of British \n",
      "Columbia, Vancouver, BC, V6T 2B5, Canada.\n",
      "(7)Department of Clinical Neurological Sciences, Schulich School of Medicine and \n",
      "Dentistry, University of Western Ontario, London, ON, N6A 2E2, Canada.\n",
      "(8)Division of Neurogeriatrics, Department NVS, Karolinska Institutet, \n",
      "Visionsgatan 4, J10:20, 171 64, Solna, Sweden.\n",
      "(9)Theme Aging, Unit for Hereditary Dementias, Karolinska University Hospital, \n",
      "Solna, Sweden.\n",
      "(10)German Center for Neurodegenerative Diseases (DZNE), 18147, Rostock, \n",
      "Germany.\n",
      "(11)Department of Neuropathology, University of Tübingen, 72076, Tübingen, \n",
      "Germany.\n",
      "(12)Hertie Institute for Clinical Brain Research, University of Tübingen, 72076, \n",
      "Tübingen, Germany.\n",
      "(13)Department of Neurology, Rostock University Medical Center, 18147, Rostock, \n",
      "Germany.\n",
      "(14)German Center for Neurodegenerative Diseases (DZNE), Feodor-Lynen-Str 17, \n",
      "81377, Munich, Germany.\n",
      "(15)Munich Cluster of Systems Neurology (SyNergy), Feodor-Lynen-Str 17, 81377, \n",
      "Munich, Germany.\n",
      "(16)Center for Neuropathology and Prion Research, Ludwig-Maximilians-University \n",
      "of Munich, Feodor-Lynen-Straße 23, 81377, Munich, Germany.\n",
      "(17)Department of Psychiatry and Psychotherapy, Technische Universität München, \n",
      "Munich, Germany.\n",
      "(18)Division of Neuropathology, University of Texas Southwestern Medical Center, \n",
      "5323 Harry Hines Blvd, Dallas, TX, 75390-9073, USA.\n",
      "(19)London Neurodegenerative Diseases Brain Bank, Department of Basic and \n",
      "Clinical Neuroscience, Institute of Psychiatry, Psychology and Neuroscience, \n",
      "King's College London, London, SE5 8AF, UK.\n",
      "(20)Department of Clinical Neuropathology, King's College Hospital NHS \n",
      "Foundation Trust, London, SE5 9RS, UK.\n",
      "(21)Mesulam Center for Cognitive Neurology and Alzheimer's Disease, Northwestern \n",
      "University, Chicago, IL, 60611, USA.\n",
      "(22)Department of Psychiatry and Behavioral Sciences and Department of \n",
      "Neurology, Northwestern University Feinberg School of Medicine, Chicago, IL, \n",
      "60611, USA.\n",
      "(23)Glenn Biggs Institute for Alzheimer's and Neurodegenerative Diseases, \n",
      "University of Texas Health Science Center San Antonio, San Antonio, TX, 78229, \n",
      "USA.\n",
      "(24)Department of Neurology, University of Arizona Health Sciences Center, 1501 \n",
      "North Campbell Avenue, Tucson, AZ, 85724-5023, USA.\n",
      "(25)Banner Alzheimer's Institute, Phoenix, AZ, 85006, USA.\n",
      "(26)Department of Neurology, Mayo Clinic Arizona, Scottsdale, AZ, 85259, USA.\n",
      "(27)Departments of Psychiatry and Neurology, Taub Institute for Research on \n",
      "Alzheimer's Disease and the Aging Brain, Columbia University, 630 West 168th St \n",
      "P&S Box 16, New York, NY, 10032, USA.\n",
      "(28)Indiana University School of Medicine, 355 West 16th Street, GH 4700 \n",
      "Neurology, Indianapolis, IN, 46202, USA.\n",
      "(29)Department of Physical Medicine and Rehabilitation, Neurology, Cognitive \n",
      "Neurology and Alzheimer's Center, Department of Psychiatry, Feinberg School of \n",
      "Medicine, Northwestern University, 355 E Erie Street, Chicago, IL, 60611-5146, \n",
      "USA.\n",
      "(30)MRC Prion Unit at University College London, Institute of Prion Diseases, \n",
      "London, UK.\n",
      "(31)Department of Pathology, Memory and Aging Center, University of California, \n",
      "San Francisco, CA, USA.\n",
      "(32)Penn Frontotemporal Degeneration Center, Department of Neurology, Perelman \n",
      "School of Medicine at the University of Pennsylvania, Philadelphia, PA, 19104, \n",
      "USA.\n",
      "(33)Center for Neurodegenerative Disease Research, Department of Pathology and \n",
      "Laboratory Medicine, Perelman School of Medicine at the University of \n",
      "Pennsylvania, Philadelphia, PA, 19104, USA.\n",
      "(34)Cerebral Function Unit, Greater Manchester Neurosciences Centre, Salford \n",
      "Royal Hospital, Salford, UK.\n",
      "(35)Division of Neuroscience and Experimental Psychology, School of Biological \n",
      "Sciences, Faculty of Biology, Medicine and Health, University of Manchester, \n",
      "Salford Royal Hospital, Salford, UK.\n",
      "(36)Department of Neurology, Mayo Clinic, Jacksonville, FL, USA.\n",
      "(37)Department of Neurology, Mayo Clinic, Rochester, MN, USA.\n",
      "(38)Central Clinical School and Brain and Mind Centre, The University of Sydney, \n",
      "Sydney, 2050, Australia.\n",
      "(39)School of Psychology and Brain and Mind Centre, The University of Sydney, \n",
      "Sydney, 2050, Australia.\n",
      "(40)Department of Neurosciences, University of California, San Diego, La Jolla, \n",
      "CA, 92093, USA.\n",
      "(41)Veterans Affairs San Diego Healthcare System, San Diego, CA, 92161, USA.\n",
      "(42)Krembil Discovery Tower, Tanz Centre for Research in Neurodegenerative \n",
      "Disease, University of Toronto, 60 Leonard Av, 4th Floor - 4KD481, Toronto, ON, \n",
      "M5T 0S8, Canada.\n",
      "(43)Sunnybrook Health Sciences Centre, Toronto, ON, M4N 3M5, Canada.\n",
      "(44)Department of Laboratory Medicine and Pathobiology, University of Toronto, \n",
      "Toronto, ON, M5S 1A1, Canada.\n",
      "(45)Krembil Neuroscience Center, Movement Disorder's Clinic, Toronto Western \n",
      "Hospital, 399 Bathurst Street, Toronto, ON, M5T 2S8, Canada.\n",
      "(46)Department of Neurology, Knight Alzheimer Disease Research Center, \n",
      "Washington University School of Medicine, Saint Louis, MO, 63108, USA.\n",
      "(47)Department of Psychiatry, Knight Alzheimer Disease Research Center, \n",
      "Washington University School of Medicine, Saint Louis, MO, 63108, USA.\n",
      "(48)Department of Pathology and Laboratory Medicine, Indiana University School \n",
      "of Medicine, 635 Barnhill Drive, MS A138, Indianapolis, IN, 46202, USA.\n",
      "(49)Department of Pathology, University of Pittsburgh, Pittsburgh, PA, 15213, \n",
      "USA.\n",
      "(50)Department of Neurology, University of Pittsburgh, Pittsburgh, PA, 15213, \n",
      "USA.\n",
      "(51)Civin Laboratory for Neuropathology, Banner Sun Health Research Institute, \n",
      "Sun City, AZ, 85351, USA.\n",
      "(52)Department of Psychiatry and Psychotherapy, University Hospital, \n",
      "Ludwig-Maximilians-University of Munich, Nussbaumstraße 7, 80336, Munich, \n",
      "Germany.\n",
      "(53)Department of Neurology, Taub Institute, and GH Sergievsky Center, Columbia \n",
      "University Irving Medical Center, 630 West 168th St (P&S Unit 16), New York, NY, \n",
      "10032, USA.\n",
      "(54)Department of Pathology and Taub Institute, Columbia University Irving \n",
      "Medical Center, 630 West 168th St, New York, NY, 10032, USA.\n",
      "(55)UNSW Medicine and NeuRA, Randwick, 2031, Australia.\n",
      "(56)Department of Pathology and Laboratory Medicine and Department of Neurology, \n",
      "Emory University, Atlanta, GA, 30322, USA.\n",
      "(57)Division of Neuroscience and Experimental Psychology, School of Biological \n",
      "Sciences, Faculty of Biology, Medicine and Health, University of Manchester, \n",
      "Manchester, UK.\n",
      "(58)Dementia Research Centre, Department of Neurodegenerative Disease, UCL Queen \n",
      "Square Institute of Neurology, London, UK.\n",
      "(59)Department of Pathology and Laboratory Medicine, University of British \n",
      "Columbia, Vancouver, BC, V5Z 1M9, Canada.\n",
      "(60)Department of Neuroscience, Mayo Clinic, 4500 San Pablo Road, Jacksonville, \n",
      "FL, 32224, USA. Rademakers.rosa@mayo.edu.\n",
      "\n",
      "Comment in\n",
      "    Nat Rev Neurol. 2019 May;15(5):250-251.\n",
      "\n",
      "Frontotemporal lobar degeneration with neuronal inclusions of the TAR \n",
      "DNA-binding protein 43 (FTLD-TDP) represents the most common pathological \n",
      "subtype of FTLD. We established the international FTLD-TDP whole-genome \n",
      "sequencing consortium to thoroughly characterize the known genetic causes of \n",
      "FTLD-TDP and identify novel genetic risk factors. Through the study of 1131 \n",
      "unrelated Caucasian patients, we estimated that C9orf72 repeat expansions and \n",
      "GRN loss-of-function mutations account for 25.5% and 13.9% of FTLD-TDP patients, \n",
      "respectively. Mutations in TBK1 (1.5%) and other known FTLD genes (1.4%) were \n",
      "rare, and the disease in 57.7% of FTLD-TDP patients was unexplained by the known \n",
      "FTLD genes. To unravel the contribution of common genetic factors to the \n",
      "FTLD-TDP etiology in these patients, we conducted a two-stage association study \n",
      "comprising the analysis of whole-genome sequencing data from 517 FTLD-TDP \n",
      "patients and 838 controls, followed by targeted genotyping of the most \n",
      "associated genomic loci in 119 additional FTLD-TDP patients and 1653 controls. \n",
      "We identified three genome-wide significant FTLD-TDP risk loci: one new locus at \n",
      "chromosome 7q36 within the DPP6 gene led by rs118113626 (p value = 4.82e - 08, \n",
      "OR = 2.12), and two known loci: UNC13A, led by rs1297319 (p value = 1.27e - 08, \n",
      "OR = 1.50) and HLA-DQA2 led by rs17219281 (p value = 3.22e - 08, OR = 1.98). \n",
      "While HLA represents a locus previously implicated in clinical FTLD and related \n",
      "neurodegenerative disorders, the association signal in our study is independent \n",
      "from previously reported associations. Through inspection of our whole-genome \n",
      "sequence data for genes with an excess of rare loss-of-function variants in \n",
      "FTLD-TDP patients (n ≥ 3) as compared to controls (n = 0), we further discovered \n",
      "a possible role for genes functioning within the TBK1-related immune pathway \n",
      "(e.g., DHX58, TRIM21, IRF7) in the genetic etiology of FTLD-TDP. Together, our \n",
      "study based on the largest cohort of unrelated FTLD-TDP patients assembled to \n",
      "date provides a comprehensive view of the genetic landscape of FTLD-TDP, \n",
      "nominates novel FTLD-TDP risk loci, and strongly implicates the immune pathway \n",
      "in FTLD-TDP pathogenesis.\n",
      "\n",
      "DOI: 10.1007/s00401-019-01962-9\n",
      "PMCID: PMC6533145\n",
      "PMID: 30739198 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "44. Mol Cell Probes. 2019 Feb;43:64-71. doi: 10.1016/j.mcp.2018.11.003. Epub 2018\n",
      " Nov 14.\n",
      "\n",
      "Genes and transcription factors related to the adverse effects of maternal type \n",
      "I diabetes mellitus on fetal development.\n",
      "\n",
      "Han S(1), Yang H(2), Han Y(3), Zhang H(4).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Clinical Laboratory, Ji'nan Central Hospital Affiliated to \n",
      "Shandong University, Ji'nan, 250013, China.\n",
      "(2)Department of Obstetrics, Ji'nan Central Hospital Affiliated to Shandong \n",
      "University, Ji'nan, 250013, China. Electronic address: hsy1101@zxyy.cn.\n",
      "(3)Department of Obstetrics, Ji'nan Central Hospital Affiliated to Shandong \n",
      "University, Ji'nan, 250013, China.\n",
      "(4)Department of Gynecology, Ji'nan Central Hospital Affiliated to Shandong \n",
      "University, Ji'nan, 250013, China.\n",
      "\n",
      "PURPOSE: Maternal type I diabetes mellitus (T1DM) increases the risk of adverse \n",
      "pregnancy outcomes, but the corresponding mechanism is unclear. This study aims \n",
      "to investigate the mechanism underlying the adverse pregnancy outcomes of \n",
      "maternal T1DM.\n",
      "METHODS: Gene expression microarray (GSE51546) was down-loaded from the Gene \n",
      "Expression Omnibus. This dataset included 12 umbilical cord samples from the \n",
      "newborns of T1DM mothers (T1DM group, N = six) and non-diabetic mothers (control \n",
      "group, N = six).\n",
      "RESULTS: Consequently, 1051 differentially expressed genes (DEGs) were found \n",
      "between the two groups. The up-regulated DEGs enriched in 30 KEGG pathways. \n",
      "HLA-DPA1, HLA-DMA, HLA-DMB, HLA-DQA1, HLA-DQA2 and HLA-DRA enriched in \"Type I \n",
      "diabetes mellitus\". This pathway was strongly related to 14 pathways, most of \n",
      "which were associated with diseases. Then, a protein-protein interaction network \n",
      "was constructed, and 45 potential key DEGs were identified. The 45 DEGs enriched \n",
      "in pathways such as \"Rheumatoid arthritis\", \"Chemokine signaling pathway\" and \n",
      "\"Cytokine-cytokine receptor interaction\" (e.g. CXCL12 and CCL5). Transcription \n",
      "factors (TFs) of key DEGs were predicted, and a TF-DEG regulatory network was \n",
      "constructed.\n",
      "CONCLUSIONS: Some genes (e.g. CXCL12 and CCL5) and their TFs were significantly \n",
      "and abnormally regulated in the umbilical cord tissue from the pregnancies of \n",
      "T1DM mothers compared to that from non-T1DM mothers.\n",
      "\n",
      "Copyright © 2018. Published by Elsevier Ltd.\n",
      "\n",
      "DOI: 10.1016/j.mcp.2018.11.003\n",
      "PMID: 30447278 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "45. Ann Rheum Dis. 2018 Apr;77(4):589-595. doi: 10.1136/annrheumdis-2017-212372. \n",
      "Epub 2018 Jan 27.\n",
      "\n",
      "Cross-phenotype analysis of Immunochip data identifies KDM4C as a relevant locus \n",
      "for the development of systemic vasculitis.\n",
      "\n",
      "Ortiz-Fernández L(1), Carmona FD(2), López-Mejías R(3), González-Escribano \n",
      "MF(4), Lyons PA(5), Morgan AW(6), Sawalha AH(7), Merkel PA, Smith KGC(5), \n",
      "González-Gay MA(3)(8), Martín J(1); Spanish GCA Study Group, UK GCA Consortium, \n",
      "Turkish Takayasu Study Group, Vasculitis Clinical Research Consortium, IgAV \n",
      "Study Group, AAV Study group.\n",
      "\n",
      "Collaborators: Alegre-Sancho JJ, Blanco R, Callejas JL, Caminal-Montero L, \n",
      "Castañeda S, Cid MC, Corberabellalta M, Miguel E, Díaz-López JB, Escalante B, \n",
      "Mateo PF, Fernández-Gutiérrez B, Fernández-Nebro A, García-Villanueva MJ, \n",
      "Gómez-Vaquero C, Grau E, Rojas MG, Hernández-Rodríguez J, Hidalgo-Conde A, \n",
      "López-Longo FJ, Madroñero Vuelta AB, Marí-Alfonso B, Martínez L, \n",
      "Martínezberriochoa A, Martínez-Taboada VM, Martínez-Zapico A, Miranda-Filloy JA, \n",
      "Monfort J, Morado IC, Narváez J, Ordóñez-Cañizares MC, Ortego-Centeno N, \n",
      "Pérez-Conesa M, Prieto-González S, Ramentol-Sintas M, Raya E, Fernández RR, \n",
      "Rodríguez-Rodríguez L, Román JA, Sáez-Comet L, Sánchez-Martín J, \n",
      "Sánchez-Pernaute O, Solans R, Sopeña B, Tío L, Unzurrunzaga A, Vicente Bhaskar \n",
      "Dasgupta EF, Eyre S, Green M, Gough A, Hordon L, Jarret S, Kamath S, Levy S, \n",
      "Mackie SL, Patel Y, Pease CT, Taylor JC, Watts R, Worthington J, Alibaz-Oner F, \n",
      "Akar S, Akkoc N, Aksu K, Ates A, Aydin SZ, Bıcakcıgil M, Cefle A, Cobankara V, \n",
      "Coit P, Dalkilic E, Direskeneli H, Duzgun N, Erken E, Fresko I, Inanc M, Kamali \n",
      "S, Karaaslan Y, Karadag O, Kaşifoğlu T, Keser G, Kısacık B, Kiraz S, Onat AM, \n",
      "Onen F, Ozbalkan Z, Ozer HT, Ozturk MA, Pamuk ÖN, Saruhan-Direskeneli G, Seyahi \n",
      "E, Tunc E, Yazici A, Yentür Maximiliano Aragües SP, Blanco R, Blanco-Madrigal \n",
      "JM, Castañeda S, Argila D, Galíndez-Aguirregoikoa E, Genre F, Luque ML, Llorca \n",
      "J, Mijares V, Miranda-Filloy JA, Parejo AN, Ortego-Centeno N, Pina T, \n",
      "Remuzgo-Martínez S, Rubio E, Sevilla-Perez B, Ubilla B, Brenchley P, Chaundrhy \n",
      "AN, Clayton Panos Deloukas DG, Harper L, Jayne DR, Little MA, Padmanabhan S, \n",
      "Pusey CD, Rayner TF, Salamana AD, Savage CO, Trivedi S, Watts RA.\n",
      "\n",
      "Author information:\n",
      "(1)Instituto de Parasitologia y Biomedicina Lopez-Neyra, Granada, Spain.\n",
      "(2)Departamento de Genética e Instituto de Biotecnología, Universidad de \n",
      "Granada, Granada, Spain.\n",
      "(3)Epidemiology, Genetics and Atherosclerosis Research Group on Systemic \n",
      "Inflammatory Diseases, Rheumatology Division, Hospital Universitario Marqués de \n",
      "Valdecilla, IDIVAL, Santander, Spain.\n",
      "(4)Department of Immunology, Hospital Universitario Virgen del Rocío (IBiS,CSIC, \n",
      "US), Sevilla, Spain.\n",
      "(5)Department of Medicine, University of Cambridge School of Clinical Medicine, \n",
      "Cambridge Biomedical Campus, Cambridge, UK.\n",
      "(6)Leeds Institute of Rheumatic and Musculoskeletal Medicine, University of \n",
      "Leeds, and NIHR Leeds Biomedical Research Centre, Leeds Teaching Hospitals NHS \n",
      "Trust, Leeds, UK.\n",
      "(7)Division of Rheumatology, Department of Internal Medicine, University of \n",
      "Michigan, Ann Arbor, Michigan, USA.\n",
      "(8)School of Medicine, University of Cantabria, Santander, Spain.\n",
      "\n",
      "Erratum in\n",
      "    Ann Rheum Dis. 2018 Jun;77(6):950.\n",
      "\n",
      "OBJETIVE: Systemic vasculitides represent a heterogeneous group of rare complex \n",
      "diseases of the blood vessels with a poorly understood aetiology. To investigate \n",
      "the shared genetic component underlying their predisposition, we performed the \n",
      "first cross-phenotype meta-analysis of genetic data from different clinically \n",
      "distinct patterns of vasculitis.\n",
      "METHODS: Immunochip genotyping data from 2465 patients diagnosed with giant cell \n",
      "arteritis, Takayasu's arteritis, antineutrophil cytoplasmic antibody-associated \n",
      "vasculitis or IgA vasculitis as well as 4632 unaffected controls were analysed \n",
      "to identify common susceptibility loci for vasculitis development. The possible \n",
      "functional consequences of the associated variants were interrogated using \n",
      "publicly available annotation data.\n",
      "RESULTS: The strongest association signal corresponded with an intergenic \n",
      "polymorphism located between HLA-DQB1 and HLA-DQA2 (rs6932517, P=4.16E-14, \n",
      "OR=0.74). This single nucleotide polymorphism is in moderate linkage \n",
      "disequilibrium with the disease-specific human leucocyte antigen (HLA) class II \n",
      "associations of each type of vasculitis and could mark them. Outside the HLA \n",
      "region, we identified the KDM4C gene as a common risk locus for vasculitides \n",
      "(highest peak rs16925200, P=6.23E-07, OR=1.75). This gene encodes a histone \n",
      "demethylase involved in the epigenetic control of gene expression.\n",
      "CONCLUSIONS: Through a combined analysis of Immunochip data, we have identified \n",
      "KDM4C as a new risk gene shared between systemic vasculitides, consistent with \n",
      "the increasing evidences of the crucial role that the epigenetic mechanisms have \n",
      "in the development of complex immune-mediated conditions.\n",
      "\n",
      "© Article author(s) (or their employer(s) unless otherwise stated in the text of \n",
      "the article) 2018. All rights reserved. No commercial use is permitted unless \n",
      "otherwise expressly granted.\n",
      "\n",
      "DOI: 10.1136/annrheumdis-2017-212372\n",
      "PMCID: PMC5849568\n",
      "PMID: 29374629 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing interests: None declared.\n",
      "\n",
      "\n",
      "46. Sci Rep. 2018 Jan 18;8(1):1069. doi: 10.1038/s41598-017-18241-w.\n",
      "\n",
      "Genome-wide association study of self-reported food reactions in Japanese \n",
      "identifies shrimp and peach specific loci in the HLA-DR/DQ gene region.\n",
      "\n",
      "Khor SS(1), Morino R(2), Nakazono K(3), Kamitsuji S(3), Akita M(2), Kawajiri \n",
      "M(2), Yamasaki T(4), Kami A(2), Hoshi Y(4), Tada A(2), Ishikawa K(2), Hine M(5), \n",
      "Kobayashi M(5), Kurume N(5), Kamatani N(3), Tokunaga K(1), Johnson TA(6).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Human Genetics, Graduate School of Medicine, The University of \n",
      "Tokyo, Tokyo, 113-0033, Japan.\n",
      "(2)EverGene Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\n",
      "(3)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan.\n",
      "(4)Life Science Group, Healthcare Division, Department of Healthcare Business, \n",
      "MTI Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\n",
      "(5)LunaLuna Division, Department of Healthcare Business, MTI Ltd., Shinjuku-ku, \n",
      "Tokyo, 163-1435, Japan.\n",
      "(6)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan. \n",
      "todd.johnson@stagen.co.jp.\n",
      "\n",
      "Food allergy is an increasingly important health problem in the world. Several \n",
      "genome-wide association studies (GWAS) focused on European ancestry samples have \n",
      "identified food allergy-specific loci in the HLA class II region. We conducted \n",
      "GWAS of self-reported reactivity with common foods using the data from 11011 \n",
      "Japanese women and identified shrimp and peach allergy-specific loci in the \n",
      "HLA-DR/DQ gene region tagged by rs74995702 (P = 6.30 × 10-17, OR = 1.91) and \n",
      "rs28359884 (P = 2.3 × 10-12, OR = 1.80), respectively. After HLA imputation \n",
      "using a Japanese population-specific reference, the most strongly associated \n",
      "haplotype was HLA-DRB1*04:05-HLA-DQB1*04:01 for shrimp allergy \n",
      "(P = 3.92 × 10-19, OR = 1.99) and HLA-DRB1*09:01-HLA-DQB1*03:03 for peach \n",
      "allergy (P = 1.15 × 10-7, OR = 1.68). Additionally, both allergies' associated \n",
      "variants were eQTLs for several HLA genes, with HLA-DQA2 the single eQTL gene \n",
      "shared between the two traits. Our study suggests that allergy to certain foods \n",
      "may be related to genetic differences that tag both HLA alleles having \n",
      "particular epitope binding specificities as well as variants modulating \n",
      "expression of particular HLA genes. Investigating this further could increase \n",
      "our understanding of food allergy aetiology and potentially lead to better \n",
      "therapeutic strategies for allergen immunotherapies.\n",
      "\n",
      "DOI: 10.1038/s41598-017-18241-w\n",
      "PMCID: PMC5773682\n",
      "PMID: 29348432 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: T.A.J., S.K., N.Ka., S.K. are employees of \n",
      "StaGen Co. Ltd. R.M., M.A., M.Ka., T.Y., A.K., M.T., Y.H., A.T., K.I., M.H., \n",
      "M.Ko., and N.Ku. are employees of EverGene/MTI. EverGene/MTI has applied for \n",
      "patents related to the food allergy associations described in this report. N.Ka. \n",
      "is also Chairman of StaGen Co. Ltd. and a Director of the Tsukuba International \n",
      "Clinical Pharmacology Clinic. S.S.K. and K.T. declare no competing financial \n",
      "interests.\n",
      "\n",
      "\n",
      "47. Trop Anim Health Prod. 2017 Apr;49(4):867-873. doi:\n",
      "10.1007/s11250-017-1277-9.  Epub 2017 Mar 23.\n",
      "\n",
      "Genetic appraisal of serological response post vaccination against \n",
      "enterotoxaemia (ET) in Malpura and Avikalin sheep.\n",
      "\n",
      "Gowane GR(1), Akram N(2), Prince LL(2), Prakash V(2), Kumar A(2).\n",
      "\n",
      "Author information:\n",
      "(1)ICAR-Central Sheep & Wool Research Institute, Avikanagar via, Jaipur, \n",
      "Rajasthan, 304501, India. gopalgowane@gmail.com.\n",
      "(2)ICAR-Central Sheep & Wool Research Institute, Avikanagar via, Jaipur, \n",
      "Rajasthan, 304501, India.\n",
      "\n",
      "Enterotoxaemia (ET) is a fatal enteric disease of small ruminants attributable \n",
      "to a toxigenic type of Clostridium perfringens. The key strategy for prevention \n",
      "of ET is the management and vaccination. Present study aimed at identifying the \n",
      "sources of variation for ET vaccine response especially against epsilon toxin in \n",
      "173 sheep that included 83 Avikalin and 90 Malpura lambs raised at the institute \n",
      "flock in the semi-arid region of India. The mean age at vaccination was 90 days. \n",
      "Sera were tested by blocking ELISA. Study showed significant variability for \n",
      "response to ET vaccine. 5.2% animals had + positivity, 20.8% animals had ++ \n",
      "positivity, 51.4% animals had +++ positivity and 22.5% animals had ++++ \n",
      "positivity. Amongst environmental determinants, breed, season, sex and age at \n",
      "vaccination proved to be non-significant sources of variation (P > 0.05). MHC \n",
      "genotypes with DRB1 gene and DQA2 genes also revealed non-significant \n",
      "association with ET vaccine response; however, a trend of decreasing PI values \n",
      "with increasing ranks was observed. Study revealed strong response of epsilon \n",
      "toxin along with complexity of the ET vaccine response as phenotype to be \n",
      "explained by genetic and non-genetic factors. The importance of better \n",
      "management practices and vaccination is suggested for preventive measures.\n",
      "\n",
      "DOI: 10.1007/s11250-017-1277-9\n",
      "PMID: 28337572 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "48. Sci Rep. 2015 Dec 21;5:18855. doi: 10.1038/srep18855.\n",
      "\n",
      "Network-assisted analysis of primary Sjögren's syndrome GWAS data in Han \n",
      "Chinese.\n",
      "\n",
      "Fang K(1), Zhang K(1), Wang J(1).\n",
      "\n",
      "Author information:\n",
      "(1)Key Laboratory of Mental Health, Institute of Psychology, Chinese Academy of \n",
      "Sciences, Beijing 100101, China.\n",
      "\n",
      "Primary Sjögren's syndrome (pSS) is a complex autoimmune disorder. So far, \n",
      "genetic research in pSS has lagged far behind and the underlying biological \n",
      "mechanism is unclear. Further exploring existing genome-wide association study \n",
      "(GWAS) data is urgently expected to uncover disease-related gene combination \n",
      "patterns. Herein, we conducted a network-based analysis by integrating pSS GWAS \n",
      "in Han Chinese with a protein-protein interactions network to identify pSS \n",
      "candidate genes. After module detection and evaluation, 8 dense modules covering \n",
      "40 genes were obtained for further functional annotation. Additional 31 MHC \n",
      "genes with significant gene-level P-values (sigMHC-gene) were also remained. The \n",
      "combined module genes and sigMHC-genes, a total of 71 genes, were denoted as pSS \n",
      "candidate genes. Of these pSS candidates, 14 genes had been reported to be \n",
      "associated with any of pSS, RA, and SLE, including STAT4, GTF2I, HLA-DPB1, \n",
      "HLA-DRB1, PTTG1, HLA-DQB1, MBL2, TAP2, CFLAR, NFKBIE, HLA-DRA, APOM, HLA-DQA2 \n",
      "and NOTCH4. This is the first report of the network-assisted analysis for pSS \n",
      "GWAS data to explore combined gene patterns associated with pSS. Our study \n",
      "suggests that network-assisted analysis is a useful approach to gaining further \n",
      "insights into the biology of associated genes and providing important clues for \n",
      "future research into pSS etiology.\n",
      "\n",
      "DOI: 10.1038/srep18855\n",
      "PMCID: PMC4685393\n",
      "PMID: 26686423 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "49. PLoS One. 2015 Dec 16;10(12):e0144719. doi: 10.1371/journal.pone.0144719. \n",
      "eCollection 2015.\n",
      "\n",
      "Evidence for Association of Cell Adhesion Molecules Pathway and NLGN1 \n",
      "Polymorphisms with Schizophrenia in Chinese Han Population.\n",
      "\n",
      "Zhang Z(1)(2), Yu H(3)(4), Jiang S(1)(2), Liao J(1)(2), Lu T(1)(2), Wang \n",
      "L(1)(2), Zhang D(1)(2)(4)(5), Yue W(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Institute of Mental Health, The Sixth Hospital, Peking University, Beijing, \n",
      "China.\n",
      "(2)Key Laboratory of Mental Health, Ministry of Health & National Clinical \n",
      "Research Center for Mental Disorders (Peking University), Beijing, China.\n",
      "(3)School of Life Sciences, Tsinghua University, Beijing, China.\n",
      "(4)Tsinghua University-Peking University Joint Center for Life Sciences, \n",
      "Beijing, China.\n",
      "(5)PKU-IDG/McGovern Institute for Brain Research, Peking University, Beijing, \n",
      "China.\n",
      "\n",
      "Multiple risk variants of schizophrenia have been identified by Genome-wide \n",
      "association studies (GWAS). As a complement for GWAS, previous pathway-based \n",
      "analysis has indicated that cell adhesion molecules (CAMs) pathway might be \n",
      "involved in the pathogenesis of schizophrenia. However, less replication studies \n",
      "have been reported. Our objective was to investigate the association between \n",
      "CAMs pathway and schizophrenia in the Chinese Han population. We first performed \n",
      "a pathway analysis utilizing our previous GWAS data. The CAMs pathway (hsa04514) \n",
      "was significantly associated with schizophrenia using hybrid gene set-based test \n",
      "(P = 1.03×10-10) and hypergeometric test (P = 5.04×10-6). Moreover, 12 genes \n",
      "(HLA-A, HLA-C, HLA-DOB, HLA-DPB1, HLA-DQA2, HLA-DRB1, MPZ, CD276, NLGN1, NRCAM, \n",
      "CLDN1 and ICAM3) were modestly significantly associated with schizophrenia \n",
      "(P<0.01). Then, we selected one promising gene neuroligin 1 (NLGN1) to further \n",
      "investigate the association between eight significant SNPs and schizophrenia in \n",
      "an independent sample (1814 schizophrenia cases and 1487 healthy controls). Our \n",
      "study showed that seven SNPs of NLGN1 and two haplotype blocks were \n",
      "significantly associated with schizophrenia. This association was confirmed by \n",
      "the results of combined analysis. Among them, SNP rs9835385 had the most \n",
      "significant association with schizophrenia (P = 2.83×10-7). Furthermore, in \n",
      "silico analysis we demonstrated that NLGN1 is preferentially expressed in human \n",
      "brain and SNP rs1488547 was related to the expression level. We validated the \n",
      "association of CAMs pathway with schizophrenia in pathway-level and identified \n",
      "one susceptibility gene NLGN1. Further investigation of the roles of CAMs \n",
      "pathway in the pathogenesis of schizophrenia is warranted.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0144719\n",
      "PMCID: PMC4682938\n",
      "PMID: 26674772 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "50. Arthritis Rheumatol. 2016 Apr;68(4):932-43. doi: 10.1002/art.39504.\n",
      "\n",
      "Genome-Wide Association Study in an Amerindian Ancestry Population Reveals Novel \n",
      "Systemic Lupus Erythematosus Risk Loci and the Role of European Admixture.\n",
      "\n",
      "Alarcón-Riquelme ME(1), Ziegler JT(2), Molineros J(1), Howard TD(2), \n",
      "Moreno-Estrada A(3), Sánchez-Rodríguez E(4), Ainsworth HC(2), Ortiz-Tello P(3), \n",
      "Comeau ME(2), Rasmussen A(1), Kelly JA(1), Adler A(1), Acevedo-Vázquez EM(5), \n",
      "Cucho-Venegas JM(5), García-De la Torre I(6), Cardiel MH(7), Miranda P(8), \n",
      "Catoggio LJ(9), Maradiaga-Ceceña M(10), Gaffney PM(1), Vyse TJ(11), Criswell \n",
      "LA(12), Tsao BP(13), Sivils KL(1), Bae SC(14), James JA(15), Kimberly RP(16), \n",
      "Kaufman KM(17), Harley JB(17), Esquivel-Valerio JA(18), Moctezuma JF(19), García \n",
      "MA(20), Berbotto GA(21), Babini AM(22), Scherbarth H(23), Toloza S(24), Baca \n",
      "V(25), Nath SK(1), Aguilar Salinas C(26), Orozco L(27), Tusié-Luna T(28), \n",
      "Zidovetzki R(29), Pons-Estel BA(30), Langefeld CD(2), Jacob CO(31).\n",
      "\n",
      "Author information:\n",
      "(1)Oklahoma Medical Research Foundation, Oklahoma City.\n",
      "(2)Wake Forest School of Medicine, Winston-Salem, North Carolina.\n",
      "(3)Stanford University School of Medicine, Stanford, California, and Laboratorio \n",
      "Nacional de Genómica para la Biodiversidad, Centro de Investigación y de \n",
      "Estudios Avanzados del Instituto Politécnico Nacional, Irapuato, Mexico.\n",
      "(4)Icahn School of Medicine at Mount Sinai, New York, New York.\n",
      "(5)Hospital Nacional Guillermo Almenara Irigoyen, Lima, Peru.\n",
      "(6)Hospital General de Occidente, Zapopan, Mexico.\n",
      "(7)Centro de Investigación Clínica de Morelia, Morelia, Mexico.\n",
      "(8)Centro de Estudios Reumatológicos, Santiago, Chile.\n",
      "(9)Hospital Italiano de Buenos Aires, Buenos Aires, Argentina.\n",
      "(10)Hospital General de Culiacán, Culiacán, Mexico.\n",
      "(11)King's College London, London, UK.\n",
      "(12)University of California, San Francisco.\n",
      "(13)University of California, Los Angeles.\n",
      "(14)Hanyang University Hospital for Rheumatic Diseases, Seoul, Republic of \n",
      "Korea.\n",
      "(15)Oklahoma Medical Research Foundation and University of Oklahoma Health \n",
      "Sciences Center, Oklahoma City.\n",
      "(16)University of Alabama at Birmingham.\n",
      "(17)Cincinnati Children's Hospital Medical Center, Cincinnati, Ohio.\n",
      "(18)Hospital Universitario Dr. José Eleuterio González Universidad Autonoma de \n",
      "Nuevo León, Monterrey, Mexico.\n",
      "(19)Hospital General de México, Mexico City, Mexico.\n",
      "(20)Hospital Interzonal General de Agudos General San Martin, La Plata, \n",
      "Argentina.\n",
      "(21)Hospital Escuela Eva Perón, Granadero Baigorria, Argentina.\n",
      "(22)Hospital Italiano de Córdoba, Córdoba, Argentina.\n",
      "(23)Hospital Interzonal General de Agudos Oscar E. Alende, Mar del Plata, \n",
      "Argentina.\n",
      "(24)Hospital Interzonal San Juan Bautista, San Fernando del Valle de Catamarca, \n",
      "Argentina.\n",
      "(25)Hospital de Peditaria, Centro Médico Nacional Siglo XXI, Instituto Mexicano \n",
      "del Seguro Social, Mexico City, Mexico.\n",
      "(26)Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico \n",
      "City, Mexico.\n",
      "(27)Instituto Nacional de Medicina Genómica, Mexico City, Mexico.\n",
      "(28)Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán and \n",
      "Instituto de Investigaciones Biomédicas de la Universidad Nacional Autónoma de \n",
      "México, Mexico City, Mexico.\n",
      "(29)University of California, Riverside.\n",
      "(30)Sanatorio Parque, Rosario, Argentina.\n",
      "(31)University of Southern California School of Medicine, Los Angeles.\n",
      "\n",
      "OBJECTIVE: Systemic lupus erythematosus (SLE) is a chronic autoimmune disease \n",
      "with a strong genetic component. We undertook the present work to perform the \n",
      "first genome-wide association study on individuals from the Americas who are \n",
      "enriched for Native American heritage.\n",
      "METHODS: We analyzed 3,710 individuals from the US and 4 countries of Latin \n",
      "America who were diagnosed as having SLE, and healthy controls. Samples were \n",
      "genotyped with HumanOmni1 BeadChip. Data on out-of-study controls genotyped with \n",
      "HumanOmni2.5 were also included. Statistical analyses were performed using \n",
      "SNPtest and SNPGWA. Data were adjusted for genomic control and false discovery \n",
      "rate. Imputation was performed using Impute2 and, for classic HLA alleles, \n",
      "HiBag. Odds ratios (ORs) and 95% confidence intervals (95% CIs) were calculated.\n",
      "RESULTS: The IRF5-TNPO3 region showed the strongest association and largest OR \n",
      "for SLE (rs10488631: genomic control-adjusted P [Pgcadj ] = 2.61 × 10(-29), OR \n",
      "2.12 [95% CI 1.88-2.39]), followed by HLA class II on the DQA2-DQB1 loci \n",
      "(rs9275572: Pgcadj  = 1.11 × 10(-16), OR 1.62 [95% CI 1.46-1.80] and rs9271366: \n",
      "Pgcadj  = 6.46 × 10(-12), OR 2.06 [95% CI 1.71-2.50]). Other known SLE loci \n",
      "found to be associated in this population were ITGAM, STAT4, TNIP1, NCF2, and \n",
      "IRAK1. We identified a novel locus on 10q24.33 (rs4917385: Pgcadj  = 1.39 × \n",
      "10(-8)) with an expression quantitative trait locus (eQTL) effect (Peqtl  = 8.0 \n",
      "× 10(-37) at USMG5/miR1307), and several new suggestive loci. SLE risk loci \n",
      "previously identified in Europeans and Asians were corroborated. Local ancestry \n",
      "estimation showed that the HLA allele risk contribution is of European ancestral \n",
      "origin. Imputation of HLA alleles suggested that autochthonous Native American \n",
      "haplotypes provide protection against development of SLE.\n",
      "CONCLUSION: Our results demonstrate that studying admixed populations provides \n",
      "new insights in the delineation of the genetic architecture that underlies \n",
      "autoimmune and complex diseases.\n",
      "\n",
      "© 2016, American College of Rheumatology.\n",
      "\n",
      "DOI: 10.1002/art.39504\n",
      "PMCID: PMC4829354\n",
      "PMID: 26606652 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Conflicts of Interest The authors declare no \n",
      "conflicts of interest\n",
      "\n",
      "\n",
      "51. Lancet Respir Med. 2015 Oct;3(10):769-81. doi: 10.1016/S2213-2600(15)00283-0.\n",
      " Epub 2015 Sep 27.\n",
      "\n",
      "Novel insights into the genetics of smoking behaviour, lung function, and \n",
      "chronic obstructive pulmonary disease (UK BiLEVE): a genetic association study \n",
      "in UK Biobank.\n",
      "\n",
      "Wain LV(1), Shrine N(1), Miller S(2), Jackson VE(1), Ntalla I(1), Soler Artigas \n",
      "M(1), Billington CK(2), Kheirallah AK(2), Allen R(1), Cook JP(1), Probert K(2), \n",
      "Obeidat M(3), Bossé Y(4), Hao K(5), Postma DS(6), Paré PD(3), Ramasamy A(7); UK \n",
      "Brain Expression Consortium (UKBEC); Mägi R(8), Mihailov E(8), Reinmaa E(8), \n",
      "Melén E(9), O'Connell J(10), Frangou E(11), Delaneau O(12); OxGSK Consortium; \n",
      "Freeman C(13), Petkova D(13), McCarthy M(14), Sayers I(2), Deloukas P(15), \n",
      "Hubbard R(16), Pavord I(17), Hansell AL(18), Thomson NC(19), Zeggini E(20), \n",
      "Morris AP(21), Marchini J(10), Strachan DP(22), Tobin MD(23), Hall IP(24).\n",
      "\n",
      "Collaborators: Hardy JA, Weale ME, Ryten M, Smith C, Walker R, Botía J, \n",
      "Vandrocova J, Guelfi S, D'Sa K, Matarin M, Varghese V, Trabzuni D, Ramasamy A, \n",
      "Forabosco P, Liu JZ, Tozzi F, Waterworth DM, Pillai SG, Muglia P, Middleton L, \n",
      "Berrettini W, Knouff CW, Yuan X, Waeber G, Vollenweider P, Preisig M, Wareham \n",
      "NJ, Zhao JH, Loos RJ, Barroso I, Khaw KT, Grundy S, Barter P, Mahley R, \n",
      "Kesaniemi A, McPherson R, Vincent JB, Strauss J, Kennedy JL, Farmer A, McGuffin \n",
      "P, Day R, Matthews K, Bakke P, Gulsvik A, Lucae S, Ising M, Brueckl T, Horstmann \n",
      "S, Wichmann HE, Rawal R, Dahmen N, Lamina C, Polasek O, Zgaga L, Huffman J, \n",
      "Campbell S, Kooner J, Chambers JC, Burnett MS, Devaney JM, Pichard AD, Kent KM, \n",
      "Satler L, Lindsay JM, Waksman R, Epstein S, Wilson JF, Wild SH, Campbell H, \n",
      "Vitart V, Reilly MP, Li M, Qu L, Wilensky R, Matthai W, Hakonarson HH, Rader DJ, \n",
      "Ellinghaus D, Lieb W, Franke A, Uda M, Terracciano A, Xiao X, Busonero F, Scheet \n",
      "P, Schlessinger D, St Clair D, Rujescu D, Abecasis GR, Grabe HJ, Teumer A, \n",
      "Völzke H, Petersmann A, John U, Rudan I, Hayward C, Wright AF, Kolcic I, Wright \n",
      "BJ, Thompson JR, Balmforth AJ, Hall AS, Samani NJ, Anderson CA, Ahmad T, Mathew \n",
      "CG, Parkes M, Satsangi J, Caulfield M, Munroe PB, Farrall M, Dominiczak A, \n",
      "Worthington J, Thomson W, Eyre S, Barton A, Mooser V, Francks C, Marchini J.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Health Sciences, University of Leicester, Leicester, UK.\n",
      "(2)Division of Respiratory Medicine, Queen's Medical Centre, University of \n",
      "Nottingham, Nottingham, UK.\n",
      "(3)University of British Columbia Centre for Heart Lung Innovation, St Paul's \n",
      "Hospital, Vancouver, BC, Canada.\n",
      "(4)Institut universitaire de cardiologie et de pneumologie de Québec, Department \n",
      "of Molecular Medicine, Laval University, Québec, QC, Canada.\n",
      "(5)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \n",
      "Mount Sinai, New York, NY, USA; Icahn Institute of Genomics and Multiscale \n",
      "Biology, Icahn School of Medicine at Mount Sinai, New York, NY, USA; Department \n",
      "of Respiratory Medicine, Shanghai Tenth People's Hospital, Tongji University, \n",
      "Shanghai, China.\n",
      "(6)Department of Pulmonary Medicine and Tuberculosis, University of Groningen, \n",
      "University Medical Center Groningen, Groningen, Netherlands.\n",
      "(7)Department of Molecular Neuroscience, UCL Institute of Neurology, London, UK; \n",
      "Department of Medical and Molecular Genetics, King's College London, London, UK; \n",
      "Jenner Institute, University of Oxford, Oxford, UK.\n",
      "(8)Estonian Genome Center, University of Tartu, Tartu, Estonia.\n",
      "(9)Institute of Environmental Medicine, Karolinska Institutet and Sachs' \n",
      "Children's Hospital, Stockholm, Sweden.\n",
      "(10)Department of Statistics, University of Oxford, Oxford, UK; Wellcome Trust \n",
      "Centre for Human Genetics, University of Oxford, Oxford, UK.\n",
      "(11)Department of Statistics, University of Oxford, Oxford, UK; Centre for \n",
      "Statistics in Medicine, Nuffield Department of Orthopaedics, Rheumatology and \n",
      "Musculoskeletal Sciences, University of Oxford, Oxford, UK.\n",
      "(12)Department of Statistics, University of Oxford, Oxford, UK; Department of \n",
      "Genetic Medicine and Development, University of Geneva Medical School, Geneva, \n",
      "Switzerland.\n",
      "(13)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.\n",
      "(14)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK; \n",
      "Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, \n",
      "Oxford, UK.\n",
      "(15)William Harvey Research Institute, Barts and The London School of Medicine \n",
      "and Dentistry, Queen Mary University London, London, UK; Princess Al-Jawhara \n",
      "Al-Brahim Centre of Excellence in Research of Hereditary Disorders (PACER-HD), \n",
      "King Abdulaziz University, Jeddah, Saudi Arabia.\n",
      "(16)Faculty of Medicine and Health Sciences, School of Medicine, University of \n",
      "Nottingham, Nottingham, UK.\n",
      "(17)Respiratory Medicine, University of Oxford, Oxford, UK.\n",
      "(18)UK Small Area Health Statistics Unit, MRC-PHE Centre for Environment and \n",
      "Health, School of Public Health, Imperial College London, London, UK; Imperial \n",
      "College Healthcare NHS Trust, St Mary's Hospital, Paddington, London, UK.\n",
      "(19)Institute of Infection, Immunity and Inflammation, University of Glasgow, \n",
      "Glasgow, UK.\n",
      "(20)Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.\n",
      "(21)Department of Biostatistics, University of Liverpool, Liverpool, UK.\n",
      "(22)Population Health Research Institute, St George's, University of London, \n",
      "London, UK.\n",
      "(23)Department of Health Sciences, University of Leicester, Leicester, UK; \n",
      "National Institute for Health Research, Leicester Respiratory Biomedical \n",
      "Research Unit, Glenfield Hospital, Leicester, UK. Electronic address: \n",
      "mt47@leicester.ac.uk.\n",
      "(24)Division of Respiratory Medicine, Queen's Medical Centre, University of \n",
      "Nottingham, Nottingham, UK. Electronic address: ian.hall@nottingham.ac.uk.\n",
      "\n",
      "Erratum in\n",
      "    Lancet Respir Med. 2016 Jan;4(1):e4.\n",
      "    Lancet Respir Med. 2016 Jan;4(1):e4.\n",
      "\n",
      "Comment in\n",
      "    Lancet Respir Med. 2015 Oct;3(10):737-9.\n",
      "    Lancet Respir Med. 2015 Dec;3(12):e42.\n",
      "    Lancet Respir Med. 2015 Dec;3(12):e42-3.\n",
      "\n",
      "BACKGROUND: Understanding the genetic basis of airflow obstruction and smoking \n",
      "behaviour is key to determining the pathophysiology of chronic obstructive \n",
      "pulmonary disease (COPD). We used UK Biobank data to study the genetic causes of \n",
      "smoking behaviour and lung health.\n",
      "METHODS: We sampled individuals of European ancestry from UK Biobank, from the \n",
      "middle and extremes of the forced expiratory volume in 1 s (FEV1) distribution \n",
      "among heavy smokers (mean 35 pack-years) and never smokers. We developed a \n",
      "custom array for UK Biobank to provide optimum genome-wide coverage of common \n",
      "and low-frequency variants, dense coverage of genomic regions already implicated \n",
      "in lung health and disease, and to assay rare coding variants relevant to the UK \n",
      "population. We investigated whether there were shared genetic causes between \n",
      "different phenotypes defined by extremes of FEV1. We also looked for novel \n",
      "variants associated with extremes of FEV1 and smoking behaviour and assessed \n",
      "regions of the genome that had already shown evidence for a role in lung health \n",
      "and disease. We set genome-wide significance at p<5 × 10(-8).\n",
      "FINDINGS: UK Biobank participants were recruited from March 15, 2006, to July 7, \n",
      "2010. Sample selection for the UK BiLEVE study started on Nov 22, 2012, and was \n",
      "completed on Dec 20, 2012. We selected 50,008 unique samples: 10,002 individuals \n",
      "with low FEV1, 10,000 with average FEV1, and 5002 with high FEV1 from each of \n",
      "the heavy smoker and never smoker groups. We noted a substantial sharing of \n",
      "genetic causes of low FEV1 between heavy smokers and never smokers \n",
      "(p=2.29 × 10(-16)) and between individuals with and without doctor-diagnosed \n",
      "asthma (p=6.06 × 10(-11)). We discovered six novel genome-wide significant \n",
      "signals of association with extremes of FEV1, including signals at four novel \n",
      "loci (KANSL1, TSEN54, TET2, and RBM19/TBX5) and independent signals at two \n",
      "previously reported loci (NPNT and HLA-DQB1/HLA-DQA2). These variants also \n",
      "showed association with COPD, including in individuals with no history of \n",
      "smoking. The number of copies of a 150 kb region containing the 5' end of \n",
      "KANSL1, a gene that is important for epigenetic gene regulation, was associated \n",
      "with extremes of FEV1. We also discovered five new genome-wide significant \n",
      "signals for smoking behaviour, including a variant in NCAM1 (chromosome 11) and \n",
      "a variant on chromosome 2 (between TEX41 and PABPC1P2) that has a trans effect \n",
      "on expression of NCAM1 in brain tissue.\n",
      "INTERPRETATION: By sampling from the extremes of the lung function distribution \n",
      "in UK Biobank, we identified novel genetic causes of lung function and smoking \n",
      "behaviour. These results provide new insight into the specific mechanisms \n",
      "underlying airflow obstruction, COPD, and tobacco addiction, and show \n",
      "substantial shared genetic architecture underlying airflow obstruction across \n",
      "individuals, irrespective of smoking behaviour and other airway disease.\n",
      "FUNDING: Medical Research Council.\n",
      "\n",
      "Copyright © 2015 Wain et al. Open Access article distributed under the terms of \n",
      "CC BY. Published by Elsevier Ltd.. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/S2213-2600(15)00283-0\n",
      "PMCID: PMC4593935\n",
      "PMID: 26423011 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "52. Gut. 2016 May;65(5):788-96. doi: 10.1136/gutjnl-2014-308617. Epub 2015 Mar 2.\n",
      "\n",
      "Deep resequencing of 131 Crohn's disease associated genes in pooled DNA \n",
      "confirmed three reported variants and identified eight novel variants.\n",
      "\n",
      "Hong SN(1), Park C(2), Park SJ(3), Lee CK(4), Ye BD(5), Kim YS(6), Lee S(7), \n",
      "Chae J(2), Kim JI(8), Kim YH(1); IBD Study Group of the Korean Association for \n",
      "the Study of Intestinal Diseases (KASID).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medicine, Samsung Medical Center, Sungkyunkwan University \n",
      "School of Medicine, Seoul, Korea.\n",
      "(2)Department of Biomedical Sciences, Seoul National University Graduate School, \n",
      "Seoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \n",
      "University College of Medicine, Seoul, Korea.\n",
      "(3)Department of Internal Medicine and Institute of Gastroenterology, Yonsei \n",
      "University College of Medicine, Seoul, Korea.\n",
      "(4)Department of Internal Medicine, Kyung Hee University School of Medicine, \n",
      "Seoul, Korea.\n",
      "(5)Department of Gastroenterology and Inflammatory Bowel Disease Center, Asan \n",
      "Medical Centre, University of Ulsan College of Medicine.\n",
      "(6)Department of Internal Medicine, Seoul Paik Hospital, Inje University College \n",
      "of Medicine, Seoul, Korea.\n",
      "(7)Department of Biomedical Sciences, Seoul National University Graduate School, \n",
      "Seoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \n",
      "University College of Medicine, Seoul, Korea Medical Research Center, Genomic \n",
      "Medicine Institute (GMI), Seoul National University, Seoul, Korea.\n",
      "(8)Department of Biomedical Sciences, Seoul National University Graduate School, \n",
      "Seoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \n",
      "University College of Medicine, Seoul, Korea Medical Research Center, Genomic \n",
      "Medicine Institute (GMI), Seoul National University, Seoul, Korea Cancer \n",
      "Research Institute, Seoul National University College of Medicine, Seoul, Korea.\n",
      "\n",
      "OBJECTIVE: Genome wide association studies (GWAS) and meta-analyses for Crohn's \n",
      "disease (CD) have not fully explained the heritability of CD, suggesting that \n",
      "additional loci are yet to be found and that the known loci may contain high \n",
      "effect rare risk variants that have thus far gone undetected by GWAS. While the \n",
      "cost of deep sequencing remains too high to analyse many samples, targeted \n",
      "sequencing of pooled DNA samples allows the efficient and cost effective capture \n",
      "of all variations in a target region.\n",
      "DESIGN: We performed pooled sequencing in 500 Korean CD cases and 1000 controls \n",
      "to evaluate the coding exon and 5' and 3' untranslated regions of 131 CD \n",
      "associated genes. The identified genetic variants were validated using \n",
      "genotyping in an independent set of 500 CD cases and 1000 controls.\n",
      "RESULTS: Pooled sequencing identified 30 common/low single nucleotide variants \n",
      "(SNVs) in 12 genes and 3 rare SNVs in 3 genes. Our results confirmed a \n",
      "significant association of CD with the following previously reported risk loci: \n",
      "rs3810936 in TNFSF15 (OR=1.83, p<2.2×10(-16)), rs76418789 in IL23R (OR=0.47, \n",
      "p=1.14×10(-8)) and rs2241880 in ATG16L1 (OR=1.30, p=5.28×10(-6)). In addition, \n",
      "novel loci were identified in TNFSF8 (rs3181374, OR=1.53, p=1.03×10(-14)), BTNL2 \n",
      "(rs28362680, OR=1.47, p=9.67×10(-11)), HLA-DQA2 (rs3208181, OR=1.36, \n",
      "p=4.66×10(-6)), STAT3 (rs1053004, OR=1.29, p=2.07×10(-5)), NFKBIA (rs2273650, \n",
      "OR=0.80, p=3.93×10(-4)), NKX2-3 (rs888208, OR=0.82, p=6.37×10(-4)) and DNAH12 \n",
      "(rs4462937, OR=1.13, p=3.17×10(-2)). A novel rare SNV, rs200735402 in CARD9, was \n",
      "shown to have a protective effect (OR=0.09, p=5.28×10(-5)).\n",
      "CONCLUSIONS: Our deep resequencing of 131 CD associated genes confirmed 3 \n",
      "reported risk loci and identified 8 novel risk loci for CD in Koreans, providing \n",
      "new insights into the genetic architecture of CD.\n",
      "\n",
      "Published by the BMJ Publishing Group Limited. For permission to use (where not \n",
      "already granted under a licence) please go to \n",
      "http://www.bmj.com/company/products-services/rights-and-licensing/\n",
      "\n",
      "DOI: 10.1136/gutjnl-2014-308617\n",
      "PMID: 25731871 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "53. PLoS Genet. 2014 Oct 9;10(10):e1004637. doi: 10.1371/journal.pgen.1004637. \n",
      "eCollection 2014 Oct.\n",
      "\n",
      "Dominant sequences of human major histocompatibility complex conserved extended \n",
      "haplotypes from HLA-DQA2 to DAXX.\n",
      "\n",
      "Larsen CE(1), Alford DR(2), Trautwein MR(2), Jalloh YK(2), Tarnacki JL(2), \n",
      "Kunnenkeri SK(2), Fici DA(3), Yunis EJ(4), Awdeh ZL(3), Alper CA(5).\n",
      "\n",
      "Author information:\n",
      "(1)Program in Cellular and Molecular Medicine, Boston Children's Hospital, \n",
      "Boston, Massachusetts, United States of America; Department of Medicine, Harvard \n",
      "Medical School, Boston, Massachusetts, United States of America.\n",
      "(2)Program in Cellular and Molecular Medicine, Boston Children's Hospital, \n",
      "Boston, Massachusetts, United States of America.\n",
      "(3)Program in Cellular and Molecular Medicine, Boston Children's Hospital, \n",
      "Boston, Massachusetts, United States of America; Pulsar Clinical Technologies, \n",
      "Cambridge, Massachusetts, United States of America.\n",
      "(4)Department of Cancer Immunology and AIDS, Dana-Farber Cancer Institute, \n",
      "Boston, Massachusetts, United States of America; Department of Pathology, \n",
      "Harvard Medical School, Boston, Massachusetts, United States of America.\n",
      "(5)Program in Cellular and Molecular Medicine, Boston Children's Hospital, \n",
      "Boston, Massachusetts, United States of America; Department of Pediatrics, \n",
      "Harvard Medical School, Boston, Massachusetts, United States of America.\n",
      "\n",
      "We resequenced and phased 27 kb of DNA within 580 kb of the MHC class II region \n",
      "in 158 population chromosomes, most of which were conserved extended haplotypes \n",
      "(CEHs) of European descent or contained their centromeric fragments. We \n",
      "determined the single nucleotide polymorphism and deletion-insertion \n",
      "polymorphism alleles of the dominant sequences from HLA-DQA2 to DAXX for these \n",
      "CEHs. Nine of 13 CEHs remained sufficiently intact to possess a dominant \n",
      "sequence extending at least to DAXX, 230 kb centromeric to HLA-DPB1. We \n",
      "identified the regions centromeric to HLA-DQB1 within which single instances of \n",
      "eight \"common\" European MHC haplotypes previously sequenced by the MHC Haplotype \n",
      "Project (MHP) were representative of those dominant CEH sequences. Only two MHP \n",
      "haplotypes had a dominant CEH sequence throughout the centromeric and extended \n",
      "class II region and one MHP haplotype did not represent a known European CEH \n",
      "anywhere in the region. We identified the centromeric recombination transition \n",
      "points of other MHP sequences from CEH representation to non-representation. \n",
      "Several CEH pairs or groups shared sequence identity in small blocks but had \n",
      "significantly different (although still conserved for each separate CEH) \n",
      "sequences in surrounding regions. These patterns partly explain strong \n",
      "calculated linkage disequilibrium over only short (tens to hundreds of \n",
      "kilobases) distances in the context of a finite number of observed \n",
      "megabase-length CEHs comprising half a population's haplotypes. Our results \n",
      "provide a clearer picture of European CEH class II allelic structure and \n",
      "population haplotype architecture, improved regional CEH markers, and raise \n",
      "questions concerning regional recombination hotspots.\n",
      "\n",
      "DOI: 10.1371/journal.pgen.1004637\n",
      "PMCID: PMC4191933\n",
      "PMID: 25299700 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have declared that no competing \n",
      "interests exist.\n",
      "\n",
      "\n",
      "54. PLoS One. 2014 Jun 18;9(6):e99724. doi: 10.1371/journal.pone.0099724. \n",
      "eCollection 2014.\n",
      "\n",
      "A genome-wide association study on chronic HBV infection and its clinical \n",
      "progression in male Han-Taiwanese.\n",
      "\n",
      "Chang SW(1), Fann CS(2), Su WH(3), Wang YC(4), Weng CC(4), Yu CJ(5), Hsu CL(2), \n",
      "Hsieh AR(6), Chien RN(4), Chu CM(4), Tai DI(4).\n",
      "\n",
      "Author information:\n",
      "(1)Clinical Informatics and Medical Statistics Research Center, Chang Gung \n",
      "University College of Medicine, Taoyuan, Taiwan.\n",
      "(2)Institute of Biomedical Sciences, Academia Sinica, Taipei, Taiwan.\n",
      "(3)Department of Biomedical Sciences, Graduate Institute of Biomedical Sciences, \n",
      "Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, \n",
      "Taiwan.\n",
      "(4)Division of Hepatology, Department of Gastroenterology and Hepatology, Chang \n",
      "Gung Memorial Hospital, Linkou, Taiwan.\n",
      "(5)Department of Cell and Molecular Biology, College of Medicine and Molecular \n",
      "Medicine Research Center, Chang Gung University, Tao-Yuan, Taiwan.\n",
      "(6)Graduate Institute of Biostatistics, China Medical University, Taichung, \n",
      "Taiwan.\n",
      "\n",
      "It is common to observe the clustering of chronic hepatitis B surface antigen \n",
      "(HBsAg) carriers in families. Intra-familial transmission of hepatitis B virus \n",
      "(HBV) could be the reason for the familial clustering of HBsAg carriers. \n",
      "Additionally, genetic and gender factors have been reported to be involved. We \n",
      "conducted a three-stage genome-wide association study to identify genetic \n",
      "factors associated with chronic HBV susceptibility. A total of 1,065 male \n",
      "controls and 1,623 male HBsAg carriers were included. The whole-genome \n",
      "genotyping was done on Illumina HumanHap550 beadchips in 304 healthy controls \n",
      "and HumanHap610 beadchips in 321 cases. We found that rs9277535 (HLA-DPB1, \n",
      "P = 4.87×10(-14)), rs9276370 (HLA-DQA2, P = 1.9×10(-12)), rs7756516 and \n",
      "rs7453920 (HLA-DQB2, P = 1.48×10(-11) and P = 6.66×10(-15) respectively) were \n",
      "significantly associated with persistent HBV infection. A novel SNP rs9366816 \n",
      "near HLA-DPA3 also showed significant association (P = 2.58×10(-10)). The \n",
      "\"T-T-G-G-T\" haplotype of the five SNPs further signified their association with \n",
      "the disease (P = 1.48×10(-12); OR = 1.49). The \"T-T\" haplotype composed of \n",
      "rs7756516 and rs9276370 was more prevalent in severe disease subgroups and \n",
      "associated with non-sustained therapeutic response (P = 0.0262). The \"G-C\" \n",
      "haplotype was associated with sustained therapeutic response (P = 0.0132; \n",
      "OR = 2.49). We confirmed that HLA-DPB1, HLA-DQA2 and HLA-DQB2 loci were \n",
      "associated with persistent HBV infection in male Taiwan Han-Chinese. In \n",
      "addition, the HLA-DQA2 and -DQB2 complex was associated with clinical \n",
      "progression and therapeutic response.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0099724\n",
      "PMCID: PMC4062466\n",
      "PMID: 24940741 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "55. J Occup Environ Med. 2014 Apr;56(4):382-7. doi: 10.1097/JOM.0000000000000138.\n",
      "\n",
      "Genetic variants in the major histocompatibility complex class I and class II \n",
      "genes are associated with diisocyanate-induced Asthma.\n",
      "\n",
      "Yucesoy B(1), Johnson VJ, Lummus ZL, Kashon ML, Rao M, Bannerman-Thompson H, \n",
      "Frye B, Wang W, Gautrin D, Cartier A, Boulet LP, Sastre J, Quirce S, Tarlo SM, \n",
      "Germolec DR, Luster MI, Bernstein DI.\n",
      "\n",
      "Author information:\n",
      "(1)From the Health Effects Laboratory Division (Drs Yucesoy and Kashon, Ms Frye, \n",
      "and Ms Wang) NIOSH/CDC, Morgantown, WVa; BRT-Burleson Research Technologies (Dr \n",
      "Johnson), Morrisville, NC; Division of Immunology, Allergy and Rheumatology, \n",
      "Department of Medicine (Drs Yucesoy, Lummus, and Bernstein) and Department of \n",
      "Environmental Health (Drs Rao and Bannerman-Thompson), University of Cincinnati, \n",
      "Ohio; Hôpital du Sacré-Cœur de Montréal (Drs Gautrin and Cartier), Université de \n",
      "Montréal; Hôpital Laval (Dr Boulet), Université Laval, Sainte-Foy, Québec, \n",
      "Canada; Department of Allergy (Dr Sastre), Fundación Jiménez Díaz and CIBER de \n",
      "Enfermedades Respiratorias CIBERES; Department of Allergy (Dr Quirce), Hospital \n",
      "La Paz-IdiPAZ and CIBER de Enfermedades Respiratorias CIBERES, Madrid, Spain; \n",
      "Department of Medicine (Dr Tarlo), University of Toronto, Ontario, Canada; \n",
      "Toxicology Branch (Dr Germolec), DNTP/NIEHS, Research Triangle Park, NC; and \n",
      "School of Public Health (Dr Luster), West Virginia University, Morgantown.\n",
      "\n",
      "OBJECTIVE: To investigate the association between single nucleotide \n",
      "polymorphisms (SNPs) located across the major histocompatibility complex and \n",
      "susceptibility to diisocyanate-induced asthma (DA).\n",
      "METHODS: The study population consisted of 140 diisocyanate-exposed workers. \n",
      "Genotyping was performed using the Illumina GoldenGate major histocompatibility \n",
      "complex panels.\n",
      "RESULTS: The HLA-E rs1573294 and HLA-DPB1 rs928976 SNPs were associated with an \n",
      "increased risk of DA under dominant (odds ratio [OR], 6.27; 95% confidence \n",
      "interval [CI], 2.37 to 16.6; OR, 2.79, 95% CI, 0.99 to 7.81, respectively) and \n",
      "recessive genetic models (OR, 6.27, 95% CI, 1.63 to 24.13; OR, 10.10, 95% CI, \n",
      "3.16 to 32.33, respectively). The HLA-B rs1811197, HLA-DOA rs3128935, and \n",
      "HLA-DQA2 rs7773955 SNPs conferred an increased risk of DA in a dominant model \n",
      "(OR, 7.64, 95% CI, 2.25 to 26.00; OR, 19.69, 95% CI, 2.89 to 135.25; OR, 8.43, \n",
      "95% CI, 3.03 to 23.48, respectively).\n",
      "CONCLUSION: These results suggest that genetic variations within HLA genes play \n",
      "a role in DA risk.\n",
      "\n",
      "DOI: 10.1097/JOM.0000000000000138\n",
      "PMCID: PMC4572490\n",
      "PMID: 24709764 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no conflict of interest. \n",
      "Authors Yucesoy, Johnson, Lummus, Kashon, Rao, Bannerman-Thompson, Frye, Wang, \n",
      "Gautrin, Cartier, Boulet, Sastre, Quirce, Tarlo, Germolec, Luster, and Bernstein \n",
      "have no relationships/conditions/circumstances that present potential conflict \n",
      "of interest. The JOEM editorial board and planners have no financial interest \n",
      "related to this research.\n",
      "\n",
      "\n",
      "56. Eur Rev Med Pharmacol Sci. 2013 Apr;17(8):1044-50.\n",
      "\n",
      "DNA microarray-based screening of differentially expressed genes related to \n",
      "acute lung injury and functional analysis.\n",
      "\n",
      "Chen Y(1), Shi JX, Pan XF, Feng J, Zhao H.\n",
      "\n",
      "Author information:\n",
      "(1)Thoracic Department, Shanghai Chest Hospital, Shanghai, China.\n",
      "\n",
      "OBJECTIVES: The purpose of this study was to identify differentially expressed \n",
      "genes (DEGs) related to acute lung injury (ALI) induced by sepsis with DNA \n",
      "microarray.\n",
      "MATERIALS AND METHODS: Gene expression profile GSE10474 was downloaded from Gene \n",
      "Expression Omnibus (GEO) database which includes 34 samples, among which 13 \n",
      "patients with ALI + sepsis and 21 patients with sepsis alone. The DEGs were \n",
      "identified between ALI + sepsis and sepsis alone samples using R, which were \n",
      "further analyzed using bioinformatics methods. Firstly, HitPredict was used to \n",
      "search protein-protein interactions of the DEGs. Secondly, WebGestalt was \n",
      "adopted for functional enrichment analysis of genes in the interaction networks. \n",
      "Finally, DNA methylation was analyzed to explain the differential expression.\n",
      "RESULTS: A total of 12 genes were identified as DEGs by comparing chip data from \n",
      "ALI + sepsis samples and those from sepsis alone samples, among which occludin \n",
      "(OCLN) and major histocompatibility complex, class II, DQ beta 1 (HLA-DQB1) had \n",
      "21 and 6 interactors, respectively. Functional enrichment analysis revealed \n",
      "several significantly over-represented terms: cellular component organization, \n",
      "macromolecular organization and biosynthesis, and response to stimulus. In \n",
      "addition, methylation was found in the promoters of OCLN and HLA-DQB1.\n",
      "CONCLUSIONS: We screened DEGs in septic ALI samples, and several interesting \n",
      "genes were obtained, especially OCLN and HLA-DQB1. They may be developed into \n",
      "marker genes for diagnosis or treatment of ALI.\n",
      "\n",
      "PMID: 23661517 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "57. PLoS Genet. 2012;8(12):e1003098. doi: 10.1371/journal.pgen.1003098. Epub 2012\n",
      " Dec 20.\n",
      "\n",
      "Genome-wide joint meta-analysis of SNP and SNP-by-smoking interaction identifies \n",
      "novel loci for pulmonary function.\n",
      "\n",
      "Hancock DB(1), Soler Artigas M, Gharib SA, Henry A, Manichaikul A, Ramasamy A, \n",
      "Loth DW, Imboden M, Koch B, McArdle WL, Smith AV, Smolonska J, Sood A, Tang W, \n",
      "Wilk JB, Zhai G, Zhao JH, Aschard H, Burkart KM, Curjuric I, Eijgelsheim M, \n",
      "Elliott P, Gu X, Harris TB, Janson C, Homuth G, Hysi PG, Liu JZ, Loehr LR, \n",
      "Lohman K, Loos RJ, Manning AK, Marciante KD, Obeidat M, Postma DS, Aldrich MC, \n",
      "Brusselle GG, Chen TH, Eiriksdottir G, Franceschini N, Heinrich J, Rotter JI, \n",
      "Wijmenga C, Williams OD, Bentley AR, Hofman A, Laurie CC, Lumley T, Morrison AC, \n",
      "Joubert BR, Rivadeneira F, Couper DJ, Kritchevsky SB, Liu Y, Wjst M, Wain LV, \n",
      "Vonk JM, Uitterlinden AG, Rochat T, Rich SS, Psaty BM, O'Connor GT, North KE, \n",
      "Mirel DB, Meibohm B, Launer LJ, Khaw KT, Hartikainen AL, Hammond CJ, Gläser S, \n",
      "Marchini J, Kraft P, Wareham NJ, Völzke H, Stricker BH, Spector TD, \n",
      "Probst-Hensch NM, Jarvis D, Jarvelin MR, Heckbert SR, Gudnason V, Boezen HM, \n",
      "Barr RG, Cassano PA, Strachan DP, Fornage M, Hall IP, Dupuis J, Tobin MD, London \n",
      "SJ.\n",
      "\n",
      "Author information:\n",
      "(1)Behavioral Health Epidemiology Program, Research Triangle Institute \n",
      "International, Research Triangle Park, North Carolina, USA.\n",
      "\n",
      "Comment in\n",
      "    On Lung Function and Interactions Using Genome-Wide Data.\n",
      "\n",
      "Genome-wide association studies have identified numerous genetic loci for \n",
      "spirometic measures of pulmonary function, forced expiratory volume in one \n",
      "second (FEV(1)), and its ratio to forced vital capacity (FEV(1)/FVC). Given that \n",
      "cigarette smoking adversely affects pulmonary function, we conducted genome-wide \n",
      "joint meta-analyses (JMA) of single nucleotide polymorphism (SNP) and \n",
      "SNP-by-smoking (ever-smoking or pack-years) associations on FEV(1) and \n",
      "FEV(1)/FVC across 19 studies (total N = 50,047). We identified three novel loci \n",
      "not previously associated with pulmonary function. SNPs in or near DNER \n",
      "(smallest P(JMA = )5.00×10(-11)), HLA-DQB1 and HLA-DQA2 (smallest \n",
      "P(JMA = )4.35×10(-9)), and KCNJ2 and SOX9 (smallest P(JMA = )1.28×10(-8)) were \n",
      "associated with FEV(1)/FVC or FEV(1) in meta-analysis models including SNP main \n",
      "effects, smoking main effects, and SNP-by-smoking (ever-smoking or pack-years) \n",
      "interaction. The HLA region has been widely implicated for autoimmune and lung \n",
      "phenotypes, unlike the other novel loci, which have not been widely implicated. \n",
      "We evaluated DNER, KCNJ2, and SOX9 and found them to be expressed in human lung \n",
      "tissue. DNER and SOX9 further showed evidence of differential expression in \n",
      "human airway epithelium in smokers compared to non-smokers. Our findings \n",
      "demonstrated that joint testing of SNP and SNP-by-environment interaction \n",
      "identified novel loci associated with complex traits that are missed when \n",
      "considering only the genetic main effects.\n",
      "\n",
      "DOI: 10.1371/journal.pgen.1003098\n",
      "PMCID: PMC3527213\n",
      "PMID: 23284291 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have declared that no competing \n",
      "interests exist.\n",
      "\n",
      "\n",
      "58. Lupus. 2013 Feb;22(2):198-204. doi: 10.1177/0961203312470183. Epub 2012 Dec\n",
      "20.\n",
      "\n",
      "HLA variants rs9271366 and rs9275328 are associated with systemic lupus \n",
      "erythematosus susceptibility in Malays and Chinese.\n",
      "\n",
      "Chai HC(1), Phipps ME, Othman I, Tan LP, Chua KH.\n",
      "\n",
      "Author information:\n",
      "(1)Jeffrey Cheah School of Medicine and Health Sciences, Monash University \n",
      "(Sunway Campus), Malaysia.\n",
      "\n",
      "BACKGROUND: Human leukocyte antigen (HLA) antigens and genes have long been \n",
      "reported associated with systemic lupus erythematosus (SLE) susceptibility in \n",
      "many populations. With the advance in technologies such as genome-wide \n",
      "association studies, many newly discovered SLE-associated single-nucleotide \n",
      "polymorphisms (SNPs) have been reported in recent years. These include \n",
      "HLA-DRB1/HLA-DQA1 rs9271366 and HLA-DQB1/HLA-DQA2 rs9275328. Our aim was to \n",
      "investigate these SNPs in a Malaysian SLE cohort.\n",
      "MATERIALS AND METHODS: SNPs rs9271366 and rs9275328 were screened across 790 \n",
      "Malaysian citizens from three ethnic groups (360 patients and 430 healthy \n",
      "volunteers) by Taqman SNP genotyping assays. Allele and genotyping frequencies, \n",
      "Hardy-Weinberg equilibrium, Fisher's exact test and odds ratio were calculated \n",
      "for each SNP and ethnic group. Linkage disequilibrium and interaction between \n",
      "the two SNPs were also evaluated.\n",
      "RESULTS: The minor allele G and its homozygous genotype GG of HLA-DRB1/HLA-DQA1 \n",
      "rs9271366 significantly increased the SLE susceptibility in Malaysian patients, \n",
      "including those of Malay and Chinese ethnicity (odds ratio (OR) > 1, p < 0.05). \n",
      "As for HLA-DQB1/HLA-DQA2 rs9275328, the minor allele T and the heterozygous \n",
      "genotype CT conferred protective effect to SLE in Malaysians, as well as in \n",
      "Malays and Chinese, by having OR < 1 and p value <0.05. Both SNPs did not show \n",
      "associations to SLE in Indians. D' and r (2) values for the two SNPs in LD \n",
      "analysis were 0.941 and 0.065, respectively, with haplotype GC and AT being \n",
      "significantly associated with SLE (p < 5.0 × 10(-4)) after 10,000 permutations \n",
      "were performed. The MDR test clustered the genotype combinations of GG and CC, \n",
      "and AG and CC of rs9271366 and rs9275328, accordingly, as high-risk group, and \n",
      "the two SNPs interacted redundantly by removing 1.96% of the entropy.\n",
      "CONCLUSIONS: Our findings suggest that in addition to some classical HLA \n",
      "variants, rs9271366 and rs9275328 are additional polymorphisms worth considering \n",
      "in the Malaysian and possibly in a larger Asian SLE scenario.\n",
      "\n",
      "DOI: 10.1177/0961203312470183\n",
      "PMID: 23257407 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "59. Clin Exp Allergy. 2012 Dec;42(12):1724-33. doi: 10.1111/cea.12000.\n",
      "\n",
      "HLA-DQ strikes again: genome-wide association study further confirms HLA-DQ in \n",
      "the diagnosis of asthma among adults.\n",
      "\n",
      "Lasky-Su J(1), Himes BE, Raby BA, Klanderman BJ, Sylvia JS, Lange C, Melen E, \n",
      "Martinez FD, Israel E, Gauderman J, Gilliland F, Sleiman P, Hakonarson H, \n",
      "Celedón JC, Soto-Quiros M, Avila L, Lima JJ, Irvin CG, Peters SP, Boushey H, \n",
      "Chinchilli VM, Mauger D, Tantisira K, Weiss ST; SHARP investigators.\n",
      "\n",
      "Author information:\n",
      "(1)Channing Laboratory, Department of Medicine, Brigham and Women's Hospital, \n",
      "and Harvard Medical School, Boston, MA, USA.\n",
      "\n",
      "BACKGROUND: Asthma is a common chronic respiratory disease in children and \n",
      "adults. An important genetic component to asthma susceptibility has long been \n",
      "recognized, most recently through the identification of several genes (e.g., \n",
      "ORMDL3, PDE4D, HLA-DQ, and TLE4) via genome-wide association studies.\n",
      "OBJECTIVE: To identify genetic variants associated with asthma affection status \n",
      "using genome-wide association data.\n",
      "METHODS: We describe results from a genome-wide association study on asthma \n",
      "performed in 3855 subjects using a panel of 455 089 single nucleotide \n",
      "polymorphisms (SNPs).\n",
      "RESULT: The genome-wide association study resulted in the prioritization of 33 \n",
      "variants for immediate follow-up in a multi-staged replication effort. Of these, \n",
      "a common polymorphism (rs9272346) localizing to within 1 Kb of HLA-DQA1 \n",
      "(chromosome 6p21.3) was associated with asthma in adults (P-value = 2.2E-08) \n",
      "with consistent evidence in the more heterogeneous group of adults and children \n",
      "(P-value = 1.0E-04). Moreover, some genes identified in prior asthma GWAS were \n",
      "nominally associated with asthma in our populations.\n",
      "CONCLUSION: Overall, our findings further replicate the HLA-DQ region in the \n",
      "pathogenesis of asthma. HLA-DQA1 is the fourth member of the HLA family found to \n",
      "be associated with asthma, in addition to the previously identified HLA-DRA, \n",
      "HLA-DQB1 and HLA-DQA2.\n",
      "\n",
      "© 2012 Blackwell Publishing Ltd.\n",
      "\n",
      "DOI: 10.1111/cea.12000\n",
      "PMCID: PMC6343489\n",
      "PMID: 23181788 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Conflict of Interests: Charles Irvin’s work was \n",
      "funded by a grant from the American Lung Association. Stephen Peters has been a \n",
      "member of the NHLBI’s ACRN since its creation in 1993. Homer Boushey received \n",
      "funding from the NIH for his participation in the research of the Inner City \n",
      "Asthma Consortium, from Genentech for research on the airway microbiome of \n",
      "severe asthma, and from GlaxoSmithKline for research on bronchial epithelial \n",
      "cell susceptibility to viral infection. He has served as an ad-hoc consultant or \n",
      "advisory board member for Genentech, GlaxoSmithKline, Kalobios, Pharmaxis, Merck \n",
      "and Johnson & Johnson. Vernon Chinchilli, Blanca Himes, Scott Weiss, Juan \n",
      "Celedon, Barbara Klanderman, Erik Melen, Hakon Hakonarson, Lydiana Avila, \n",
      "Benjamin Raby, Kelan Tantisira, David Mauger, Patrick Sleiman, Fernando \n",
      "Martinez, W. James Gauderman, Frank Gilliland, John Lima, Elliot Israel, \n",
      "Christoph Lange, Jody Senter Sylvia, Manuel Soto Quiros and Jessica Lasky-Su \n",
      "have no conflict of interests to declare\n",
      "\n",
      "\n",
      "60. J Immunol. 2012 Apr 15;188(8):3903-11. doi: 10.4049/jimmunol.1103048. Epub\n",
      "2012  Mar 9.\n",
      "\n",
      "HLA-DQA2 and HLA-DQB2 genes are specifically expressed in human Langerhans cells \n",
      "and encode a new HLA class II molecule.\n",
      "\n",
      "Lenormand C(1), Bausinger H, Gross F, Signorino-Gelo F, Koch S, Peressin M, \n",
      "Fricker D, Cazenave JP, Bieber T, Hanau D, de la Salle H, Tourne S.\n",
      "\n",
      "Author information:\n",
      "(1)INSERM, Unité Mixte de Recherche S725, Biologie des Cellules Dendritiques \n",
      "Humaines, Strasbourg, 67065 France.\n",
      "\n",
      "The precise role of human epidermal Langerhans cells (LCs) in immune response is \n",
      "highly controversial. While studying the gene expression profile of these cells, \n",
      "we were intrigued to identify the HLA-DQB2 gene as potentially expressed in LCs. \n",
      "Despite a strong evolutionary conservation of their sequences, the concomitant \n",
      "expression of the poorly polymorphic HLA-DQA2/HLA-DQB2 genes, paralogous to the \n",
      "HLA-DQA1/HLA-DQB1 genes, has never been detected in any cell type. We confirmed \n",
      "by RT-PCR that the HLA-DQA2 and -DQB2 genes are both expressed in LCs, but not \n",
      "in monocyte-derived dendritic cells, or in blood CD1c(+) or plasmacytoid \n",
      "dendritic cells. The presence of the HLA-DQβ2 chain in LCs could be demonstrated \n",
      "by Western blotting, whereas immunofluorescence revealed its localization in \n",
      "early endosomes. As in the case of other HLA class II molecules, the HLA-DQα2 \n",
      "and -DQβ2 chains formed heterodimers that had to associate with the invariant \n",
      "chain to reach endosomal compartments. HLA-DQα2/β2 heterodimers were expressed \n",
      "at the cell surface, where they could mediate staphylococcal superantigen \n",
      "stimulation of T cells. Interestingly, HLA-DQα2 and HLA-DQβ1 chains formed mixed \n",
      "heterodimers which efficiently left the endoplasmic reticulum. These \n",
      "observations strongly suggest that the poorly polymorphic HLA-DQA2 and -DQB2 \n",
      "genes should be considered to be of immunological importance. The HLA-DQα2/β2 \n",
      "molecules could influence the complexity of the repertoire of Ags presented by \n",
      "LCs.\n",
      "\n",
      "DOI: 10.4049/jimmunol.1103048\n",
      "PMID: 22407913 [Indexed for MEDLINE]61. PLoS One. 2011;6(11):e23089. doi: 10.1371/journal.pone.0023089. Epub 2011 Nov\n",
      " 18.\n",
      "\n",
      "Psoriasis regression analysis of MHC loci identifies shared genetic variants \n",
      "with vitiligo.\n",
      "\n",
      "Zhu KJ(1), Lv YM, Yin XY, Wang ZX, Sun LD, He SM, Cheng H, Hu DY, Zhang Z, Li Y, \n",
      "Zuo XB, Zhou YW, Yang S, Fan X, Zhang XJ, Zhang FY.\n",
      "\n",
      "Author information:\n",
      "(1)Institute of Dermatology and Department of Dermatology at No.1 Hospital, \n",
      "Anhui Medical University, Hefei, Anhui, China.\n",
      "\n",
      "Psoriasis is a common inflammatory skin disease with genetic components of both \n",
      "immune system and the epidermis. PSOR1 locus (6q21) has been strongly associated \n",
      "with psoriasis; however, it is difficult to identify additional independent \n",
      "association due to strong linkage disequilibrium in the MHC region. We performed \n",
      "stepwise regression analyses of more than 3,000 SNPs in the MHC region genotyped \n",
      "using Human 610-Quad (Illumina) in 1,139 cases with psoriasis and 1,132 controls \n",
      "of Han Chinese population to search for additional independent association. With \n",
      "four regression models obtained, two SNPs rs9468925 in HLA-C/HLA-B and rs2858881 \n",
      "in HLA-DQA2 were repeatedly selected in all models, suggesting that multiple \n",
      "loci outside PSOR1 locus were associated with psoriasis. More importantly we \n",
      "find that rs9468925 in HLA-C/HLA-B is associated with both psoriasis and \n",
      "vitiligo, providing first important evidence that two major skin diseases share \n",
      "a common genetic locus in the MHC, and a basis for elucidating the molecular \n",
      "mechanism of skin disorders.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0023089\n",
      "PMCID: PMC3220662\n",
      "PMID: 22125590 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "62. PLoS One. 2011;6(11):e27109. doi: 10.1371/journal.pone.0027109. Epub 2011 Nov\n",
      "9.\n",
      "\n",
      "Evidence for more than one Parkinson's disease-associated variant within the HLA \n",
      "region.\n",
      "\n",
      "Hill-Burns EM(1), Factor SA, Zabetian CP, Thomson G, Payami H.\n",
      "\n",
      "Author information:\n",
      "(1)New York State Department of Health Wadsworth Center, Albany, New York, \n",
      "United States of America.\n",
      "\n",
      "Parkinson's disease (PD) was recently found to be associated with HLA in a \n",
      "genome-wide association study (GWAS). Follow-up GWAS's replicated the PD-HLA \n",
      "association but their top hits differ. Do the different hits tag the same locus \n",
      "or is there more than one PD-associated variant within HLA? We show that the top \n",
      "GWAS hits are not correlated with each other (0.00≤r(2)≤0.15). Using our GWAS \n",
      "(2000 cases, 1986 controls) we conducted step-wise conditional analysis on 107 \n",
      "SNPs with P<10(-3) for PD-association; 103 dropped-out, four remained \n",
      "significant. Each SNP, when conditioned on the other three, yielded \n",
      "P(SNP1) = 5×10(-4), P(SNP2) = 5×10(-4), P(SNP3) = 4×10(-3) and P(SNP4) = 0.025. \n",
      "The four SNPs were not correlated (0.01≤r(2)≤0.20). Haplotype analysis \n",
      "(excluding rare SNP2) revealed increasing PD risk with increasing risk alleles \n",
      "from OR = 1.27, P = 5×10(-3) for one risk allele to OR = 1.65, P = 4×10(-8) for \n",
      "three. Using additional 843 cases and 856 controls we replicated the independent \n",
      "effects of SNP1 (P(conditioned-on-SNP4) = 0.04) and SNP4 \n",
      "(P(conditioned-on-SNP1) = 0.04); SNP2 and SNP3 could not be replicated. In \n",
      "pooled GWAS and replication, SNP1 had OR(conditioned-on-SNP4) = 1.23, \n",
      "P(conditioned-on-SNP4) = 6×10(-7); SNP4 had OR(conditioned-on-SNP1) = 1.18, \n",
      "P(conditioned-on-SNP1) = 3×10(-3); and the haplotype with both risk alleles had \n",
      "OR = 1.48, P = 2×10(-12). Genotypic OR increased with the number of risk alleles \n",
      "an individual possessed up to OR = 1.94, P = 2×10(-11) for individuals who were \n",
      "homozygous for the risk allele at both SNP1 and SNP4. SNP1 is a variant in \n",
      "HLA-DRA and is associated with HLA-DRA, DRB5 and DQA2 gene expression. SNP4 is \n",
      "correlated (r(2) = 0.95) with variants that are associated with HLA-DQA2 \n",
      "expression, and with the top HLA SNP from the IPDGC GWAS (r(2) = 0.60). Our \n",
      "findings suggest more than one PD-HLA association; either different alleles of \n",
      "the same gene, or separate loci.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0027109\n",
      "PMCID: PMC3212531\n",
      "PMID: 22096524 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "63. J Allergy Clin Immunol. 2012 Mar;129(3):840-845.e21. doi: \n",
      "10.1016/j.jaci.2011.09.029. Epub 2011 Nov 9.\n",
      "\n",
      "A genome-wide association study of plasma total IgE concentrations in the \n",
      "Framingham Heart Study.\n",
      "\n",
      "Granada M(1), Wilk JB, Tuzova M, Strachan DP, Weidinger S, Albrecht E, Gieger C, \n",
      "Heinrich J, Himes BE, Hunninghake GM, Celedón JC, Weiss ST, Cruikshank WW, \n",
      "Farrer LA, Center DM, O'Connor GT.\n",
      "\n",
      "Author information:\n",
      "(1)Pulmonary Center, Boston University School of Medicine, Boston, MA 02118, \n",
      "USA.\n",
      "\n",
      "Comment in\n",
      "    J Allergy Clin Immunol. 2012 Mar;129(3):846-7.\n",
      "\n",
      "BACKGROUND: Atopy and plasma IgE concentration are genetically complex traits, \n",
      "and the specific genetic risk factors that lead to IgE dysregulation and \n",
      "clinical atopy are an area of active investigation.\n",
      "OBJECTIVE: We sought to ascertain the genetic risk factors that lead to IgE \n",
      "dysregulation.\n",
      "METHODS: A genome-wide association study (GWAS) was performed in 6819 \n",
      "participants from the Framingham Heart Study (FHS). Seventy of the top single \n",
      "nucleotide polymorphisms (SNPs) were selected based on P values and linkage \n",
      "disequilibrium among neighboring SNPs and evaluated in a meta-analysis with 5 \n",
      "independent populations from the Cooperative Health Research in the Region of \n",
      "Augsburg cohort, the British 1958 Birth Cohort, and the Childhood Asthma \n",
      "Management Program cohort.\n",
      "RESULTS: Thirteen SNPs located in the region of 3 genes, FCER1A, signal \n",
      "transducer and activator of transcription 6 (STAT6), and IL13, were found to \n",
      "have genome-wide significance in the FHS cohort GWAS. The most significant SNPs \n",
      "from the 3 regions were rs2251746 (FCER1A, P = 2.11 × 10(-12)), rs1059513 \n",
      "(STAT6, P = 2.87 × 10(-8)), and rs1295686 (IL13, P = 3.55 × 10(-8)). Four \n",
      "additional gene regions, HLA-G, HLA-DQA2, HLA-A, and Duffy blood group, \n",
      "chemokine receptor (DARC), reached genome-wide statistical significance in a \n",
      "meta-analysis combining the FHS and replication cohorts, although the DARC \n",
      "association did not appear independent of SNPs in the nearby FCER1A gene.\n",
      "CONCLUSION: This GWAS of the FHS cohort has identified genetic loci in HLA genes \n",
      "that might have a role in the pathogenesis of IgE dysregulation and atopy. It \n",
      "also confirmed the association of the known susceptibility loci FCER1A, STAT6, \n",
      "and IL13 for the dysregulation of total IgE.\n",
      "\n",
      "Copyright © 2011 American Academy of Allergy, Asthma & Immunology. Published by \n",
      "Mosby, Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jaci.2011.09.029\n",
      "PMCID: PMC3293994\n",
      "PMID: 22075330 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "64. Vet Immunol Immunopathol. 2011 Dec 15;144(3-4):312-20. doi: \n",
      "10.1016/j.vetimm.2011.08.014. Epub 2011 Aug 26.\n",
      "\n",
      "Association between variation in faecal egg count for a mixed field-challenge of \n",
      "nematode parasites and ovine MHC-DQA2 polymorphism.\n",
      "\n",
      "Hickford JG(1), Forrest RH, Zhou H, Fang Q, Frampton CM.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Agricultural Sciences, Faculty of Agriculture and Life \n",
      "Sciences, Lincoln University, Lincoln 7647, Canterbury, New Zealand. \n",
      "jon.hickford@lincoln.ac.nz\n",
      "\n",
      "The selection of sheep that are resistant to gastrointestinal parasites and have \n",
      "lower faecal egg counts (FECs) has been the subject of extensive research. This \n",
      "has led to the speculation that the Major Histocompatibility Complex (MHC) genes \n",
      "could be used as markers to reduce FEC. In this study, associations between \n",
      "variation in ovine MHC-DQA2 and various measures of FEC recorded at two times \n",
      "(approximately 4 and 9 months of age) were investigated in a large group of New \n",
      "Zealand lambs (n=4676), derived from 185 different sire-lines, of a variety of \n",
      "breeds and raised on 25 separate farms. Pair-sample t-tests revealed that FEC \n",
      "for Nematodirus spp., Strongyle spp. and total FEC differed significantly \n",
      "between the two assessments. A total of twenty one DQA2 alleles or \n",
      "DQA2-DQA2-like haplotypes were identified, with allele/haplotype presence and \n",
      "frequency varying significantly between farms. For example, allele *0103 was \n",
      "observed on all farms, ranging in frequency from 0.2 to 60.9%, while haplotype \n",
      "*0101-*1601 was only present on one farm, in two lambs. A number of associations \n",
      "between the presence/absence of these alleles and egg counts were observed, but \n",
      "nearly all the allelic/haplotypic associations were age and parasite specific, \n",
      "suggesting that immune response is both age and challenge (parasite species mix) \n",
      "dependent. The exception was allele *1201 which was associated with increased \n",
      "total FECs at both 4 and 9 months of age; with it either being, or tending \n",
      "toward being, significantly associated with both increased Strongyle spp. and \n",
      "Nematodirus spp. counts as well. However, the observed increases in egg counts \n",
      "were small and ranged between 5 and 32 eggs per gram. In conclusion, we believe \n",
      "that the MHC plays an important role in parasite resistance, but that the \n",
      "MHC-nematode interaction is complex and thus the development of a single \n",
      "gene-marker based on the \"MHC effect\" is unlikely.\n",
      "\n",
      "Copyright © 2011 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.vetimm.2011.08.014\n",
      "PMID: 21945177 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "65. Genet Sel Evol. 2010 Dec 21;42(1):43. doi: 10.1186/1297-9686-42-43.\n",
      "\n",
      "Genetic diversity of selected genes that are potentially economically important \n",
      "in feral sheep of New Zealand.\n",
      "\n",
      "McKenzie GW(1), Abbott J, Zhou H, Fang Q, Merrick N, Forrest RH, Sedcole JR, \n",
      "Hickford JG.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Agricultural Science, Faculty of Agriculture and Life Sciences, \n",
      "PO Box 84, Lincoln University, Lincoln 7647, New Zealand.\n",
      "\n",
      "BACKGROUND: Feral sheep are considered to be a source of genetic variation that \n",
      "has been lost from their domestic counterparts through selection.\n",
      "METHODS: This study investigates variation in the genes KRTAP1-1, KRT33, ADRB3 \n",
      "and DQA2 in Merino-like feral sheep populations from New Zealand and its \n",
      "offshore islands. These genes have previously been shown to influence wool, lamb \n",
      "survival and animal health.\n",
      "RESULTS: All the genes were polymorphic, but no new allele was identified in the \n",
      "feral populations. In some of these populations, allele frequencies differed \n",
      "from those observed in commercial Merino sheep and other breeds found in New \n",
      "Zealand. Heterozygosity levels were comparable to those observed in other \n",
      "studies on feral sheep. Our results suggest that some of the feral populations \n",
      "may have been either inbred or outbred over the duration of their apparent \n",
      "isolation.\n",
      "CONCLUSION: The variation described here allows us to draw some conclusions \n",
      "about the likely genetic origin of the populations and selective pressures that \n",
      "may have acted upon them, but they do not appear to be a source of new genetic \n",
      "material, at least for these four genes.\n",
      "\n",
      "DOI: 10.1186/1297-9686-42-43\n",
      "PMCID: PMC3025881\n",
      "PMID: 21176141 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "66. Nat Genet. 2010 Sep;42(9):786-9. doi: 10.1038/ng.647. Epub 2010 Aug 15.\n",
      "\n",
      "Genome-wide association study identifies new HLA class II haplotypes strongly \n",
      "protective against narcolepsy.\n",
      "\n",
      "Hor H(1), Kutalik Z, Dauvilliers Y, Valsesia A, Lammers GJ, Donjacour CE, Iranzo \n",
      "A, Santamaria J, Peraita Adrados R, Vicario JL, Overeem S, Arnulf I, Theodorou \n",
      "I, Jennum P, Knudsen S, Bassetti C, Mathis J, Lecendreux M, Mayer G, Geisler P, \n",
      "Benetó A, Petit B, Pfister C, Bürki JV, Didelot G, Billiard M, Ercilla G, \n",
      "Verduijn W, Claas FH, Vollenweider P, Waeber G, Waterworth DM, Mooser V, Heinzer \n",
      "R, Beckmann JS, Bergmann S, Tafti M.\n",
      "\n",
      "Author information:\n",
      "(1)Center for Integrative Genomics, University of Lausanne, Lausanne, \n",
      "Switzerland.\n",
      "\n",
      "Erratum in\n",
      "    Nat Genet. 2011 Apr;43(4):388. Vollenwider, Peter [corrected to \n",
      "Vollenweider, Peter].\n",
      "\n",
      "Narcolepsy is a rare sleep disorder with the strongest human leukocyte antigen \n",
      "(HLA) association ever reported. Since the associated HLA-DRB1*1501-DQB1*0602 \n",
      "haplotype is common in the general population (15-25%), it has been suggested \n",
      "that it is almost necessary but not sufficient for developing narcolepsy. To \n",
      "further define the genetic basis of narcolepsy risk, we performed a genome-wide \n",
      "association study (GWAS) in 562 European individuals with narcolepsy (cases) and \n",
      "702 ethnically matched controls, with independent replication in 370 cases and \n",
      "495 controls, all heterozygous for DRB1*1501-DQB1*0602. We found association \n",
      "with a protective variant near HLA-DQA2 (rs2858884; P < 3 x 10(-8)). Further \n",
      "analysis revealed that rs2858884 is strongly linked to DRB1*03-DQB1*02 (P < 4 x \n",
      "10(-43)) and DRB1*1301-DQB1*0603 (P < 3 x 10(-7)). Cases almost never carried a \n",
      "trans DRB1*1301-DQB1*0603 haplotype (odds ratio = 0.02; P < 6 x 10(-14)). This \n",
      "unexpected protective HLA haplotype suggests a virtually causal involvement of \n",
      "the HLA region in narcolepsy susceptibility.\n",
      "\n",
      "DOI: 10.1038/ng.647\n",
      "PMID: 20711174 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "67. PLoS One. 2010 Apr 13;5(4):e10142. doi: 10.1371/journal.pone.0010142.\n",
      "\n",
      "The effect of single nucleotide polymorphisms from genome wide association \n",
      "studies in multiple sclerosis on gene expression.\n",
      "\n",
      "Handel AE(1), Handunnetthi L, Berlanga AJ, Watson CT, Morahan JM, Ramagopalan \n",
      "SV.\n",
      "\n",
      "Author information:\n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, \n",
      "United Kingdom.\n",
      "\n",
      "BACKGROUND: Multiple sclerosis (MS) is a complex neurological disorder. Its \n",
      "aetiology involves both environmental and genetic factors. Recent genome-wide \n",
      "association studies have identified a number of single nucleotide polymorphisms \n",
      "(SNPs) associated with susceptibility to (MS). We investigated whether these \n",
      "genetic variations were associated with alteration in gene expression.\n",
      "METHODS/PRINCIPAL FINDINGS: We used a database of mRNA expression and genetic \n",
      "variation derived from immortalised peripheral lymphocytes to investigate \n",
      "polymorphisms associated with MS for correlation with gene expression. Several \n",
      "SNPs were found to be associated with changes in expression: in particular two \n",
      "with HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB1, HLA-DRB4 and HLA-DRB5, one with \n",
      "ZFP57, one with CD58, two with IL7 and FAM164A, and one with FAM119B, TSFM and \n",
      "KUB3. We found minimal cross-over with a recent whole genome expression study in \n",
      "MS patients.\n",
      "DISCUSSION: We have shown that many susceptibility loci in MS are associated \n",
      "with changes in gene expression using an unbiased expression database. Several \n",
      "of these findings suggest novel gene candidates underlying the effects of \n",
      "MS-associated genetic variation.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0010142\n",
      "PMCID: PMC2854120\n",
      "PMID: 20405052 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "68. J Allergy Clin Immunol. 2010 Feb;125(2):328-335.e11. doi: \n",
      "10.1016/j.jaci.2009.11.018.\n",
      "\n",
      "Genome-wide association study of asthma identifies RAD50-IL13 and HLA-DR/DQ \n",
      "regions.\n",
      "\n",
      "Li X(1), Howard TD, Zheng SL, Haselkorn T, Peters SP, Meyers DA, Bleecker ER.\n",
      "\n",
      "Author information:\n",
      "(1)Center for Human Genomics, Wake Forest University School of Medicine, \n",
      "Winston-Salem, NC 27157, USA.\n",
      "\n",
      "BACKGROUND: Asthma is a heterogeneous disease that is caused by the interaction \n",
      "of genetic susceptibility with environmental influences. Genome-wide association \n",
      "studies (GWASs) represent a powerful approach to investigate the association of \n",
      "DNA variants with disease susceptibility. To date, few GWASs for asthma have \n",
      "been reported.\n",
      "OBJECTIVES: A GWAS was performed on a population of patients with severe or \n",
      "difficult-to-treat asthma to identify genes that are involved in the \n",
      "pathogenesis of asthma.\n",
      "METHODS: A total of 292,443 single nucleotide polymorphisms (SNPs) were tested \n",
      "for association with asthma in 473 The Epidemiology and Natural History of \n",
      "Asthma: Outcomes and Treatment Regimens (TENOR) cases and 1892 Illumina general \n",
      "population controls. Asthma-related quantitative traits (total serum IgE, \n",
      "FEV(1), forced vital capacity, and FEV(1)/forced vital capacity) were also \n",
      "tested in identified candidate regions in 473 TENOR cases and 363 phenotyped \n",
      "controls without a history of asthma to analyze GWAS results further. Imputation \n",
      "was performed in identified candidate regions for analysis with denser SNP \n",
      "coverage.\n",
      "RESULTS: Multiple SNPs in the RAD50-IL13 region on chromosome 5q31.1 were \n",
      "associated with asthma: rs2244012 in intron 2 of RAD50 (P = 3.04E-07). The \n",
      "HLA-DR/DQ region on chromosome 6p21.3 was also associated with asthma: rs1063355 \n",
      "in the 3' untranslated region of HLA-DQB1 (P = 9.55E-06). Imputation identified \n",
      "several significant SNPs in the T(H)2 locus control region 3' of RAD50. \n",
      "Imputation also identified a more significant SNP, rs3998159 (P = 1.45E-06), \n",
      "between HLA-DQB1 and HLA-DQA2.\n",
      "CONCLUSION: This GWAS confirmed the important role of T(H)2 cytokine and antigen \n",
      "presentation genes in asthma at a genome-wide level and the importance of \n",
      "additional investigation of these 2 regions to delineate their structural \n",
      "complexity and biologic function in the development of asthma.\n",
      "\n",
      "Copyright 2010 American Academy of Allergy, Asthma & Immunology. Published by \n",
      "Mosby, Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jaci.2009.11.018\n",
      "PMCID: PMC2824608\n",
      "PMID: 20159242 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "69. Immunogenetics. 2008 Apr;60(3-4):167-76. doi: 10.1007/s00251-008-0275-5. Epub\n",
      " 2008 Mar 6.\n",
      "\n",
      "Genomic analysis of Ovis aries (Ovar) MHC class IIa loci.\n",
      "\n",
      "Herrmann-Hoesing LM(1), White SN, Kappmeyer LS, Herndon DR, Knowles DP.\n",
      "\n",
      "Author information:\n",
      "(1)US Department of Agriculture, Animal Disease Research Unit, Agricultural \n",
      "Research Service, Pullman, WA 99164-6630, USA. lherrman@vetmed.wsu.edu\n",
      "\n",
      "Determining the genomic organization of the Ovis aries (Ovar) major \n",
      "histocompatibility complex class IIa region is essential for future functional \n",
      "studies related to antigen presentation. In this study, a bacterial artificial \n",
      "chromosome (BAC) library of genomic DNA from peripheral blood leukocytes (PBL) \n",
      "of a Rambouillet ram was constructed, and BAC clone consisting of the major \n",
      "histocompatibility complex (MHC) class II DQB2, DQA2, DQB1, DQA1, and DRB1 loci \n",
      "was identified and completely sequenced. The BAC clone consists of 160,889 bp of \n",
      "finished sequence with the loci arranged in the following order: DQB2, DQA2, \n",
      "DQB1, DQA1, and DRB1 with 14.3, 25, 6.6, and 40.9 Kb spanning between the loci, \n",
      "respectively. All five of these loci were transcribed in the animal used to \n",
      "generate the MHC class II BAC clone. Repeat or retrotransposable elements along \n",
      "with MHC class II cis promoter elements consisting of S, X, and Y boxes were \n",
      "identified in the sequence. In addition, 16 non-coding conserved sequences \n",
      "amongst primates, carnivores, and ruminants were identified (p < 0.001). These \n",
      "conserved sequences include binding sites for transcription factors with known \n",
      "roles in immune cells, and they provide a basis for further functional \n",
      "investigation of the genes in this region. This is the first ruminant finished \n",
      "sequence of the DQB2-DRB1 region, and this sequence information will aid in \n",
      "whole genome and transcriptome analyses of MHC class II.\n",
      "\n",
      "DOI: 10.1007/s00251-008-0275-5\n",
      "PMID: 18322680 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "70. J Autoimmun. 2008 Jun;30(4):266-72. doi: 10.1016/j.jaut.2007.10.006. Epub\n",
      "2007  Dec 11.\n",
      "\n",
      "The MHC type 1 diabetes susceptibility gene is centromeric to HLA-DQB1.\n",
      "\n",
      "Husain Z(1), Kelly MA, Eisenbarth GS, Pugliese A, Awdeh ZL, Larsen CE, Alper CA.\n",
      "\n",
      "Author information:\n",
      "(1)Immune Disease Institute, 800 Huntington Avenue, Boston, MA 02115, USA.\n",
      "\n",
      "HLA-DQB1 is widely considered to be the major histocompatibility complex (MHC) \n",
      "susceptibility gene for type 1 diabetes (T1D). However, since inheritance of the \n",
      "gene in T1D is recessive, the presence of the protective HLA-DQB1 0602 allele \n",
      "with normal nucleotide sequence in some patients raises the question of whether \n",
      "HLA-DQB1 is not the susceptibility locus itself but merely a good marker. \n",
      "HLA-DQB1 0602 is part of a conserved extended haplotype (CEH) [HLA-B7, SC31, \n",
      "DR2] (B7, DR2) with fixed DNA over more than 1Mb of genomic DNA that normally \n",
      "carries a protective allele at the true susceptibility locus. We postulated \n",
      "that, in patients with HLA-DQB1 0602, the protective allele at the \n",
      "susceptibility locus has been replaced by a susceptibility allele through an \n",
      "ancient crossover at meiosis centromeric to HLA-DQB1. We analyzed single \n",
      "nucleotide polymorphisms (SNPs) distinguishing the HLA-DQA2 (the first expressed \n",
      "gene centromeric to HLA-DQB1) allele on the normal HLA-B7, DR2 CEH from those on \n",
      "susceptibility CEHs in T1D patients and controls with HLA-DQB1 0602. All but 1 \n",
      "of 20 healthy control HLA-DQB1 0602 haplotypes had identical (consensus) first \n",
      "intron HLA-DQA2 5-SNP haplotypes. Fifteen of 19 patients with HLA-DQB1 0602 were \n",
      "homozygous for 1 or more HLA-DQA2 SNPs differing from consensus HLA-DQA2 SNPs, \n",
      "providing evidence of crossover involving the HLA-DQA2 locus. The remaining 4 \n",
      "patients were heterozygous at all positions and therefore uninformative. The \n",
      "loss of dominant protection usually associated with HLA-DQB1 0602 haplotypes is \n",
      "consistent with a locus centromeric to HLA-DQB1 being a major determinant of \n",
      "MHC-associated susceptibility, and perhaps the true T1D susceptibility locus.\n",
      "\n",
      "DOI: 10.1016/j.jaut.2007.10.006\n",
      "PMID: 18065200 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "71. Genes Immun. 2007 Sep;8(6):480-91. doi: 10.1038/sj.gene.6364400. Epub 2007\n",
      "Jun  14.\n",
      "\n",
      "Distinctive gene expression signatures in rheumatoid arthritis synovial tissue \n",
      "fibroblast cells: correlates with disease activity.\n",
      "\n",
      "Galligan CL(1), Baig E, Bykerk V, Keystone EC, Fish EN.\n",
      "\n",
      "Author information:\n",
      "(1)Toronto General Research Institute, University Health Network, Toronto, \n",
      "Ontario, Canada.\n",
      "\n",
      "Gene expression profiling of rheumatoid arthritis (RA) and osteoarthritis (OA) \n",
      "joint tissue samples provides a unique insight into the gene signatures involved \n",
      "in disease development and progression. Fibroblast-like synovial (FLS) cells \n",
      "were obtained from RA, OA and control trauma joint tissues (non-RA, non-OA) and \n",
      "RNA was analyzed by Affymetrix microarray. Thirty-four genes specific to RA and \n",
      "OA FLS cells were identified (P<0.05). HOXD10, HOXD11, HOXD13, CCL8 and LIM \n",
      "homeobox 2 were highly and exclusively expressed in RA and CLU, \n",
      "sarcoglycan-gamma, GPR64, POU3F3, peroxisome proliferative activated \n",
      "receptor-gamma and tripartite motif-containing 2 were expressed only in OA. The \n",
      "data also revealed expression heterogeneity for patients with the same disease. \n",
      "To address disease heterogeneity in RA FLS cells, we examined the effects of \n",
      "clinical disease parameters (Health Assessment Questionnaire (HAQ) score, \n",
      "C-reactive protein (CRP), erythrocyte sedimentation rate (ESR), rheumatoid \n",
      "factor (RF)) and drug therapies (methotrexate/prednisone) on RA FLS cell gene \n",
      "expression. Eight specific and unique correlations were identified: human \n",
      "leukocyte antigen (HLA)-DQA2 with HAQ score; Clec12A with RF; MAB21L2, SIAT7E, \n",
      "HAPLN1 and BAIAP2L1 with CRP level; RGMB and OSAP with ESR. Signature RA FLS \n",
      "cell gene expression profiles may provide insights into disease pathogenesis and \n",
      "have utility in diagnosis, prognosis and drug responsiveness.\n",
      "\n",
      "DOI: 10.1038/sj.gene.6364400\n",
      "PMID: 17568789 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "72. Tissue Antigens. 2005 May;65(5):448-58. doi:\n",
      "10.1111/j.1399-0039.2005.00389.x.\n",
      "\n",
      "High-resolution genotyping of HLA-DQA1 in the GoKinD study and identification of \n",
      "novel alleles HLA-DQA1*040102, HLA-DQA1*0402 and HLA-DQA1*0404.\n",
      "\n",
      "Cordovado SK(1), Hancock LN, Simone AE, Hendrix M, Mueller PW.\n",
      "\n",
      "Author information:\n",
      "(1)Division of Laboratory Sciences, Molecular Biology Branch, National Center \n",
      "for Environmental Health, Centers for Disease Control and Prevention, 4770 \n",
      "Buford Highway, Atlanta, GA 30341, USA. snc4@cdc.gov\n",
      "\n",
      "In order to achieve high-resolution HLA-DQA1 genotyping, it is necessary to \n",
      "identify polymorphisms in exons 1, 2 and 3. We present a high-resolution \n",
      "sequence-based typing (SBT) strategy for genotyping exons 1, 2 and 3 of the \n",
      "polymorphic HLA-DQA1 locus. This method is an improvement upon previously \n",
      "presented methods, because it utilizes the minimum number of SSP-PCR assays to \n",
      "obtain clear DNA sequence in both the forward and reverse directions of all \n",
      "three exons. All known HLA-DQA1 alleles are resolved with the exception of \n",
      "HLA-DQA1*010101 and HLA-DQA1*010102 for which the distinguishing polymorphism is \n",
      "located in exon 4 and does not result in an amino acid change. This method has \n",
      "enabled our laboratory to identify three new HLA-DQA1 alleles - HLA-DQA1*040102, \n",
      "HLA- DQA1*0402 and HLA-DQA1*0404 - in the Genetics of Kidneys in Diabetes \n",
      "(GoKinD) study population. Additionally, we present single-allele amplification \n",
      "methods, which identify the coding sequences of HLA-DQA1 exons 1, 2, 3, intron 2 \n",
      "and 300 bp of the HLA-DQA1 promoter (QAP). This study, also describes the QAP \n",
      "for most of the known HLA-DQA1 alleles, three HLA-DQA2 promoter sequences and \n",
      "the intron 2 sequences for HLA-DQA1*040101, HLA-DQA1*040102, HLA-DQA1*0402 and \n",
      "HLA-DQA1*0404.\n",
      "\n",
      "DOI: 10.1111/j.1399-0039.2005.00389.x\n",
      "PMID: 15853899 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "73. Immunol Rev. 2001 Oct;183:76-85. doi: 10.1034/j.1600-065x.2001.1830106.x.\n",
      "\n",
      "Differential evolutionary MHC class II strategies in humans and rhesus macaques: \n",
      "relevance for biomedical studies.\n",
      "\n",
      "Doxiadis GG(1), Otting N, de Groot NG, Bontrop RE.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Immunobiology, Biomedical Primate Research Centre, Rijswijk, \n",
      "The Netherlands. doxiadis@bprc.nl\n",
      "\n",
      "The rhesus macaque is an important preclinical model in transplantation research \n",
      "and in investigations of chronic and infectious diseases that need a \n",
      "well-characterised major histocompatibility complex (MHC-Mamu). In a large \n",
      "population of pedigreed rhesus macaques, 70 Mamu-DRB, 18 -DQA1, 24 -DQB1, and 14 \n",
      "-DPB1 alleles were detected. In humans, five HLA-DRB region configurations are \n",
      "present, displaying diversity with regard to number and combinations of loci. \n",
      "The HLA-DRB1 gene of each of these configurations is highly polymorphic. For \n",
      "rhesus monkeys, at least 31 Mamu-DRB region configurations have been determined. \n",
      "In contrast to humans, most Mamu-DRB region configurations display no or only \n",
      "limited allelic polymorphism. Segregation analyses revealed 28 Mamu-DQA1/DQB1 \n",
      "pairs, each pair linked to a limited number of Mamu-DRB region configurations \n",
      "and vice versa. In comparison with humans, the degree of freedom of \n",
      "recombination between Mamu-DQA1 and -DQB1 is extremely low and equivalents of \n",
      "HLA-DQA2/DQB2 are absent. The Mamu-DPA1 gene is invariant and -DPB1 manifests \n",
      "only moderate allelic variation, whereas the HLA-DPA1 gene is oligomorphic and \n",
      "HLA-DPB1 highly polymorphic. Thus, both species used different evolutionary \n",
      "strategies to create polymorphism and diversity at the MHC class II loci in \n",
      "order to cope with pathogens.\n",
      "\n",
      "DOI: 10.1034/j.1600-065x.2001.1830106.x\n",
      "PMID: 11782248 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "74. Hum Immunol. 2001 May;62(5):504-8. doi: 10.1016/s0198-8859(01)00236-1.\n",
      "\n",
      "Absence of in vivo DNA-protein interactions in the DQA2 and DQB2 promoter \n",
      "regions.\n",
      "\n",
      "Indovina P(1), Megiorni F, Fontemaggi G, Coni P, Mora B, Mazzilli MC.\n",
      "\n",
      "Author information:\n",
      "(1)Dipartimento di Medicina Sperimentale e Patologia, Università \"La Sapienza,\", \n",
      "Rome, Italy.\n",
      "\n",
      "The DQ subregion of the human major histocompatibility complex (HLA) contains \n",
      "two pairs of loci: the DQA1/B1 genes (hereafter called DQ1), coding for the DQ \n",
      "molecules, and the DQA2/B2 pseudogenes (hereafter called DQ2). These pseudogenes \n",
      "are highly homologous to the functional DQ1 genes and they have no apparent \n",
      "abnormal features in their sequences that could prevent their activity. Only \n",
      "recently a low expression of the DQA2 gene has been observed whereas the DQB2 \n",
      "transcript was never found. The comparison between the DQ1 and DQ2 regulatory \n",
      "sequences revealed several differences in their W, X, and Y cis-acting elements. \n",
      "To examine the DNA/protein interactions in the DQ promoter regions, we performed \n",
      "in vivo footprinting experiments. Whereas the functional DQ1 loci showed a \n",
      "series of DNA-protein contact points in the X and Y boxes, the promoters of the \n",
      "DQ2 pseudogenes displayed an unoccupied phenotype. These findings suggest that \n",
      "the very low level of DQA2 expression and the apparent lack of DQB2 activity are \n",
      "caused by the reduced binding affinity of specific transcription factors.\n",
      "\n",
      "DOI: 10.1016/s0198-8859(01)00236-1\n",
      "PMID: 11334674 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "75. Ann N Y Acad Sci. 1998 May 13;841:355-9. doi: \n",
      "10.1111/j.1749-6632.1998.tb10946.x.\n",
      "\n",
      "Identification of a novel HLA class II association with DQB1*0502 in an Italian \n",
      "myasthenic population.\n",
      "\n",
      "Baggi F(1), Antozzi C, Andreetta F, Confalonieri P, Ciusani E, Begovich AB, \n",
      "Erlich HA, Cornelio F, Mantegazza R.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neuromuscular Diseases, National Neurological Institute C. \n",
      "Besta, Milan, Italy.\n",
      "\n",
      "DOI: 10.1111/j.1749-6632.1998.tb10946.x\n",
      "PMID: 9668258 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "76. Yi Chuan Xue Bao. 1997 Oct;24(5):394-402.\n",
      "\n",
      "[Molecular evolution of MHC DQA genes. II. Phylogenetic analysis based on \n",
      "nucleotide substitution and SCU bias].\n",
      "\n",
      "[Article in Chinese]\n",
      "\n",
      "Pan X(1), Fu J.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biology, Second Military Medical University, Shanghai.\n",
      "\n",
      "Phylogenetics of 23 alleles at MHC DQA loci in 7 mammalian species was studied \n",
      "based on their nucleotide (NT) substitution and synonymous codon usage (SCU) \n",
      "bias. (1) It was demonstrated that the NT substitution rates are 1.0 x 10(-9) \n",
      "NT/site/yr for exon2 and 1.3 x 10(-9) NT/site/yr for exon2-4 in a large time \n",
      "scale, which is similar to other nuclear genes, while for mouse and rat the \n",
      "rates are nearly twice as high as above mentioned. (2) The DQA locus diversity \n",
      "and their interallelic diversity developed long after the radiation of mammalian \n",
      "80Mya (million years ago). The bovine counterpart, of, and with the same recent \n",
      "ancestor of ovine DQA2, remains to be discovered. HLA-DQA2 locus split from \n",
      "HLA-DQA1 ancestor at the time between 12 approximately 20 Mya while allele \n",
      "diversity of HLA-DQA1 emerged and developed from 24 Mya to less than 1 Mya. (3) \n",
      "The phylogenetic trees based on SCU divergence reflect the phylogenetics of MHC \n",
      "DQA genes quite well generally in a new respect and reveal that HLA-DQA2 has a \n",
      "distinctive SCU bias different from all other MHC DQA locianalyzed. It indicates \n",
      "that SCU statistics plays an important and unique role in phylogenetic analysis \n",
      "of orthologous genes. The method to estimate the SCU divergence and SCU \n",
      "similarity was improved in this research.\n",
      "\n",
      "PMID: 9494291 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "77. J Immunol. 1997 Mar 1;158(5):2116-25.\n",
      "\n",
      "The nonpolymorphic MHC class II isotype, HLA-DQA2, is expressed on the surface \n",
      "of B lymphoblastoid cells.\n",
      "\n",
      "Rudy GB(1), Lew AM.\n",
      "\n",
      "Author information:\n",
      "(1)Walter and Eliza Hall Institute of Medical Research, Royal Melbourne \n",
      "Hospital, Parkville, Australia.\n",
      "\n",
      "The more centromeric of the two human MHC class II DQA genes, DQA2, has not \n",
      "previously been shown to express a protein product. However, its high degree of \n",
      "sequence similarity to DQA1, the fact that it is highly evolutionarily \n",
      "conserved, and the recent demonstration of its transcription in some cell lines \n",
      "suggest that a DQA2 alpha-chain may be expressed. Polyclonal anti-peptide \n",
      "antisera were generated and shown to be specific for DQA2 in Western blotting of \n",
      "Triton X-114 preparations of B lymphoblastoid cell lines. DQA2 expression is \n",
      "variable, although always at least threefold less than that of DQA1, and is \n",
      "found in all haplotypes examined. Immunoprecipitations of biotin-labeled cell \n",
      "surface proteins reveal that DQA2 appears at the cell surface. In keeping with \n",
      "others' findings, no evidence for an expressed DQB2 beta-chain was found, \n",
      "prompting investigation of the means by which DQA2 gains access to the cell \n",
      "surface. Antisera specific for allotypic and isotypic MHC class II beta-chains \n",
      "were used in sequential immunoprecipitation experiments to search for a \n",
      "beta-chain partner for DQA2, and a transfectant system was established in an \n",
      "attempt to promote pairing of DQA2 with a selected DQB1 (*0501) chain. In \n",
      "neither case was DQA2 found to dimerize with an MHC class II beta-chain. Despite \n",
      "this, DQA2 is capable of forming a complex with invariant chain.\n",
      "\n",
      "PMID: 9036956 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "78. Anim Genet. 1996 Oct;27(5):305-12. doi: 10.1111/j.1365-2052.1996.tb00974.x.\n",
      "\n",
      "Polymorphism at the ovine major histocompatibility complex class II loci.\n",
      "\n",
      "Escayg AP(1), Hickford JG, Montgomery GW, Dodds KG, Bullock DW.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Animal and Veterinary Sciences, Lincoln University, Canterbury, \n",
      "New Zealand.\n",
      "\n",
      "Southern hybridization analysis of the ovine major histocompatibility complex \n",
      "(MHC) (MhcOvar) class II region, using sheep-specific probes for the DQA1, DQA2, \n",
      "DQB and DRA loci, has revealed extensive polymorphism. DQA1 and DQA2 had eight \n",
      "and 16 alleles respectively, DQB had six and DRA had three alleles. Little \n",
      "information was derived from the DRB locus owing to extensive \n",
      "cross-hybridization between the DRB probe and the DQB locus. Differences in \n",
      "allele frequency between breeds were revealed. At the DQA1 locus a null allele \n",
      "(DQA1-N) was observed with a frequency of between 27% and 45%, making this the \n",
      "most common DQA1 allele in all breeds examined. The frequency of DQA1-N \n",
      "homozygotes was between 11% and 18%, raising questions as to the functional \n",
      "significance of the DQA1 gene. Linkage analysis between the DQA1, DQA2, DQB and \n",
      "DRA loci did not reveal any recombination.\n",
      "\n",
      "DOI: 10.1111/j.1365-2052.1996.tb00974.x\n",
      "PMID: 8930070 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "79. Curr Opin Rheumatol. 1994 Nov;6(6):612-5. doi:\n",
      "10.1097/00002281-199411000-00011.\n",
      "\n",
      "Immunologic aspects of scleroderma.\n",
      "\n",
      "White B(1).\n",
      "\n",
      "Author information:\n",
      "(1)University of Maryland School of Medicine, Baltimore.\n",
      "\n",
      "Many distinct areas of investigation contribute to the understanding of immune \n",
      "abnormalities in systemic sclerosis. Recent immunohistochemical studies question \n",
      "the causal relationship of increased transforming growth factor-beta expression \n",
      "to dermal fibrosis. RNA polymerase I, II, and III have been identified as \n",
      "autoantigens specific for systemic sclerosis. Anti-RNA polymerase antibodies are \n",
      "directed against both unique and shared subunits of the multiprotein complexes. \n",
      "The targeting of several subunits suggests that the entire complexes are \n",
      "processed and presented by antigen-presenting cells. Genetic studies show that \n",
      "both HLA-DR and HLA-DQ genes control the antitopoisomerase response in Japanese \n",
      "patients. The null allele of the complement component C4 and HLA-DQA2 have been \n",
      "identified as two independent disease susceptibility genes.\n",
      "\n",
      "DOI: 10.1097/00002281-199411000-00011\n",
      "PMID: 7865382 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "80. Q J Med. 1994 Jun;87(6):337-41.\n",
      "\n",
      "Treatment of Graves' disease with the block-replace regimen of antithyroid \n",
      "drugs: the effect of treatment duration and immunogenetic susceptibility on \n",
      "relapse.\n",
      "\n",
      "Weetman AP(1), Pickerill AP, Watson P, Chatterjee VK, Edwards OM.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medicine, University of Sheffield Clinical Sciences Centre, \n",
      "Northern General Hospital, UK.\n",
      "\n",
      "Antithyroid drugs are commonly used as first-line treatment for Graves' disease, \n",
      "but the optimum regimen for inducing remission remains unclear. We gave the \n",
      "block-replace regimen of carbimazole plus thyroxine to 100 patients for 6 or 12 \n",
      "months, to determine whether prolonged treatment is associated with fewer \n",
      "relapses. The remission rate one year after cessation of treatment was 59% with \n",
      "the 6 month course and 65% with 12 months; this was not significantly different. \n",
      "We also analysed HLA markers identified by restriction fragment length \n",
      "polymorphisms and could not confirm the recently reported associations of \n",
      "outcome with HLA-DR4 or with an HLA-DQA2 allele. These results show that six \n",
      "months treatment with a block-replace regimen of antithyroid drugs is probably \n",
      "sufficient, in the UK, to achieve maximum remission of Graves' disease and that \n",
      "there are no HLA markers which clearly predict outcome.\n",
      "\n",
      "PMID: 7913766 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "81. Hum Immunol. 1994 Mar;39(3):225-9. doi: 10.1016/0198-8859(94)90264-x.\n",
      "\n",
      "Limited polymorphism of the HLA-DQA2 promoter and identification of a variant \n",
      "octamer.\n",
      "\n",
      "Rudy G(1), Lew AM.\n",
      "\n",
      "Author information:\n",
      "(1)Walter and Eliza Hall Institute of Medical Research, Parkville, Australia.\n",
      "\n",
      "Previous studies have suggested that the HLA-DQA2 gene may be associated with \n",
      "IDDM. The apparently limited allelism at this locus prompted us to investigate \n",
      "whether this association might be with the level of gene expression rather than \n",
      "with specific alleles. The proximal promoter region of HLA-DQA2 was sequenced in \n",
      "three homozygous DR4;DQ8 subjects with IDDM, six homozygous DR3;DQ2 subjects \n",
      "(three healthy controls and three with IDDM), and selected DR4 and DR6 cell \n",
      "lines. This 388-bp region encompassed the known control W/Z/H/S, X, and Y boxes \n",
      "and included a previously unremarked variant octamer sequence 40 bp upstream of \n",
      "the transcription start site. Only one polymorphic site was present among these \n",
      "15 sequences, found in one DR3;DQ2 subject and a DR6;DQ6 cell line. This \n",
      "indicates that any disease association with HLA-DQA2, at least among DR3;DQ2 \n",
      "individuals, cannot be accounted for solely by polymorphism of the proximal \n",
      "promoter region.\n",
      "\n",
      "DOI: 10.1016/0198-8859(94)90264-x\n",
      "PMID: 8026991 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "82. Anim Genet. 1993 Aug;24(4):249-55. doi: 10.1111/j.1365-2052.1993.tb00307.x.\n",
      "\n",
      "Isolation, characterization and evolution of ovine major histocompatibility \n",
      "complex class II DRA and DQA genes.\n",
      "\n",
      "Fabb SA(1), Maddox JF, Gogolin-Ewens KJ, Baker L, Wu MJ, Brandon MR.\n",
      "\n",
      "Author information:\n",
      "(1)Centre for Animal Biotechnology, School of Veterinary Science, University of \n",
      "Melbourne, Parkville, Victoria, Australia.\n",
      "\n",
      "Four full-length ovine major histocompatibility complex (MHC) class II A cDNA \n",
      "clones coding for new alleles of DRA, DQA1 and DQA2 genes were isolated from two \n",
      "ovine lambda gt10 cDNA libraries. The derived amino acid sequences of these \n",
      "clones resemble class II A molecules from other species in both size and \n",
      "structure. Restriction fragment length polymorphism analysis, using an Ovar-DRA \n",
      "probe on DNA from Merino and Romney sheep revealed only limited polymorphism in \n",
      "contrast to the high levels of polymorphism revealed by Ovar-DQA probes. \n",
      "Comparison of the predicted amino acid sequences for the three ovine A genes \n",
      "with class II A genes from five other species revealed that the most variable \n",
      "region of the molecule is the signal peptide. Although virtually every amino \n",
      "acid site shows variation, within or between species, there are some blocks of \n",
      "highly conserved residues. Within gene comparisons of nucleotide differences \n",
      "reveal that the greatest number of changes is found between the alleles of \n",
      "Ovar-DQA1 and -DQA2 genes and the least between Ovar-DRA1 alleles. Phylogenetic \n",
      "analysis of class II A sequences from several species place DRA and DQA genes on \n",
      "two distinct branches, with Ovar-DRA1 and BOLA-DRA, and Ovar-DQA1 and BOLA-DQA \n",
      "being most similar on their respective branches.\n",
      "\n",
      "DOI: 10.1111/j.1365-2052.1993.tb00307.x\n",
      "PMID: 7902039 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "83. Neurology. 1993 Jan;43(1):163-8. doi: 10.1212/wnl.43.1_part_1.163.\n",
      "\n",
      "Multiple sclerosis is associated with genes within or close to the HLA-DR-DQ \n",
      "subregion on a normal DR15,DQ6,Dw2 haplotype.\n",
      "\n",
      "Hillert J(1), Olerup O.\n",
      "\n",
      "Author information:\n",
      "(1)Center for BioTechnology, Karolinska Institute, NOVUM, Huddinge, Sweden.\n",
      "\n",
      "The HLA class II haplotype DR15,DQ6,Dw2 is associated with an increased risk for \n",
      "multiple sclerosis (MS). It has previously been shown that this haplotype \n",
      "extends to the HLA-DQA2 locus but not to the HLA-DP subregion. We report the \n",
      "distribution in 148 MS patients and 158 healthy controls of Taq I restriction \n",
      "fragments of the C4, cytochrome P-21 (CYP21), and HLA-B genes. Approximately 90% \n",
      "of the DR15,DQ6,Dw2 haplotypes extend to the C4/CYP21 loci and 50% to the HLA-B \n",
      "locus, indicated by a significant association of the haplotype with restriction \n",
      "fragments of these loci. However, there was no independent association in MS \n",
      "patients with these genes. This indicates that the susceptibility to MS is coded \n",
      "for by genes within or close to the HLA-DR-DQ subregion. Furthermore, the \n",
      "delineation of the DR15,DQ6,Dw2 haplotype is very similar in MS patients and \n",
      "control subjects. This observation supports the hypothesis that \"normal\" genes, \n",
      "common to all carriers of the DR15,DQ6,Dw2 haplotype, confer the increased \n",
      "genetic susceptibility to MS associated with this haplotype.\n",
      "\n",
      "DOI: 10.1212/wnl.43.1_part_1.163\n",
      "PMID: 8093808 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "84. J Immunol. 1992 Feb 15;148(4):1265-73.\n",
      "\n",
      "Differential expression of related HLA class II DQ beta genes caused by \n",
      "nucleotide variation in transcriptional regulatory elements.\n",
      "\n",
      "Shewey LM(1), Beaty JS, Andersen LC, Nepom GT.\n",
      "\n",
      "Author information:\n",
      "(1)Virginia Mason Research Center, Seattle, WA 98101.\n",
      "\n",
      "HLA class II genes comprise a large multigene family with intra- and interlocus \n",
      "variation in structure and expression. Within this family of related genes, the \n",
      "HLA-DX alpha and beta loci (HLA DQA2 and DQB2) are highly homologous to \n",
      "functional HLA-DQ loci (HLA DQA1 and DQB1) but are frequently termed pseudogenes \n",
      "because DX gene transcription has not been observed, even in cells expressing \n",
      "HLA-DQ. Analysis of upstream transcriptional regulatory elements for the DX beta \n",
      "and DQ beta genes identified a high degree of nucleotide homology, consistent \n",
      "with their derivation from a common ancestral class II gene. However, transient \n",
      "expression assays with plasmids utilizing promoter elements from the DX beta \n",
      "gene had no activity in transfected human B cells, in contrast to homologous DQ \n",
      "beta sequences. Reciprocal exchange of specific sequences from the DQ beta gene \n",
      "with those of the DX beta gene restored expression to wild-type DQ beta levels, \n",
      "as did mutagenesis of only three DX-specific nucleotides in the upstream \n",
      "regulatory region. These three nucleotides mark two binding sites for distinct \n",
      "nuclear DNA-binding proteins that differentially recognize DQ beta and DX beta \n",
      "sequences. Transcription of these genes is critically dependent on interactions \n",
      "between these two upstream regulatory region sites which distinguish DX beta \n",
      "from its closely related homologue, DQ beta.\n",
      "\n",
      "PMID: 1737939 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "85. J Immunol. 1991 Dec 15;147(12):4393-7.\n",
      "\n",
      "The cryptic HLA-DQA2 (\"DX alpha\") gene is expressed in human B cell lines.\n",
      "\n",
      "Yu LP(1), Sheehy MJ.\n",
      "\n",
      "Author information:\n",
      "(1)Research Department, American Red Cross Blood Services, Madison, WI.\n",
      "\n",
      "Although there are two gene pairs in the HLA-DQ region, DQA1/B1 and DQA2/B2, the \n",
      "latter pair have never been found to be expressed. The DQA2 and DQB2 genes \n",
      "appear to be normal at the DNA sequence level, there being no indication that \n",
      "they should be pseudogenes, and normal splicing occurs when DQA2 is included in \n",
      "a retroviral vector. The putative amino acid sequences of DQA2 and DQB2 have \n",
      "been nearly invariant, raising the questions of how these genes have been \n",
      "maintained normal-appearing and nearly monomorphic if they are not expressed \n",
      "and, if they are expressed, in what cell types are they expressed. We present \n",
      "Northern blot, nuclease protection assay, and mRNA sequence data that \n",
      "demonstrate the existence of spliced DQA2 mRNA in human B lymphoblastoid cell \n",
      "lines. In contrast, we found no evidence of DQB2 mRNA in the same cell lines, \n",
      "nor of DQA2 mRNA in the monocyte line U937 or in activated T cells.\n",
      "\n",
      "PMID: 1753107 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "86. Hum Immunol. 1991 Feb;30(2):136-9. doi: 10.1016/0198-8859(91)90082-k.\n",
      "\n",
      "Linkage disequilibrium between the HLA-DQA2 alleles and the HLA/DR/DQ complex.\n",
      "\n",
      "Ratanachaiyavong S(1), Bidwell JL, Bidwell EA, McGregor AM.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medicine, King's College School of Medicine and Dentistry, \n",
      "London, United Kingdom.\n",
      "\n",
      "A statistically significant association was observed between alleles of the \n",
      "HLA-DQA2 and of the DR/DQ complex in a DNA-restriction fragment length \n",
      "polymorphism study of 219 members from 21 multiplex families of patients with \n",
      "hyperthyroid Graves' disease and 773 unrelated individuals selected for \n",
      "homozygosity of the HLA-DQA2 alleles. This data provides evidence for linkage \n",
      "disequilibrium rather than for a hot spot of recombination within the HLA-DQ \n",
      "subregion.\n",
      "\n",
      "DOI: 10.1016/0198-8859(91)90082-k\n",
      "PMID: 1673673 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "87. Immunogenetics. 1991;34(2):69-79. doi: 10.1007/BF00211419.\n",
      "\n",
      "Nucleotide sequence, polymorphism, and evolution of ovine MHC class II DQA \n",
      "genes.\n",
      "\n",
      "Scott PC(1), Gogolin-Ewens KJ, Adams TE, Brandon MR.\n",
      "\n",
      "Author information:\n",
      "(1)Centre for Animal Biotechnology, School of Veterinary Science, University of \n",
      "Melbourne, Parkville, Victoria, Australia.\n",
      "\n",
      "The nucleotide sequence of all exons and introns, excluding exon 1, of the ovine \n",
      "major histocompatibility complex (MhcOvar) genes analogous to the HLA-DQA1 and \n",
      "-DQA2 genes has been determined and the gene structure found to be similar to \n",
      "that reported for other species. The predicted amino acid sequences of the \n",
      "Ovar-DQA genes have been compared with the equivalent DQA genes in man, mouse, \n",
      "rat, rabbit, and cattle and used to determine the evolutionary relationships of \n",
      "the sheep class II genes to these other species. Northern blot analysis of sheep \n",
      "mRNA using exon specific probes for each of the two Ovar-DQA genes show that \n",
      "both genes are transcribed, whereas in humans there is no evidence that HLA-DQA2 \n",
      "is transcriptionally active. Restriction fragment length polymorphisms (RFLPs) \n",
      "have been used to define a polymorphic series of alleles in both Ovar-DQA genes \n",
      "and have indicated that the number of DQA genes is not constant in sheep as it \n",
      "is in humans, but varies with the haplotype.\n",
      "\n",
      "DOI: 10.1007/BF00211419\n",
      "PMID: 1678367 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "88. Hum Immunol. 1990 Dec;29(4):256-62. doi: 10.1016/0198-8859(90)90039-r.\n",
      "\n",
      "HLA-DQA2 (DX alpha) polymorphism and insulin dependent diabetes.\n",
      "\n",
      "Rowe JR(1), Neme de Gimenez MH, Emler CA, Sheehy MJ.\n",
      "\n",
      "Author information:\n",
      "(1)Research Department, American Red Cross Blood Services, Madison, WI 53705.\n",
      "\n",
      "A certain HLA-DQA2 locus TaqI fragment, DX alpha\"U\", has been reported to be \n",
      "associated with insulin-dependent diabetes mellitus (IDDM). Reports of various \n",
      "studies in this vein have ranged from stating that the association of DQA2\"U\" \n",
      "with IDDM exists even among subjects positive for HLA-DR3 and -DR4 to stating \n",
      "that the association of DQA2\"U\" with diabetes can be attributed to linkage \n",
      "disequilibrium between the DQA2\"U\" and some component(s) on the affected \n",
      "haplotypes. Using a synthetic 97-base probe corresponding to a portion of an \n",
      "intron of DQA2, in a Southern blot analysis of IDDM and control subjects from \n",
      "Wisconsin, we were able to confirm the association of DQA2\"U\" with diabetes. \n",
      "However, among DR3 subjects there was no significant association between DQA2\"U\" \n",
      "and diabetes (p = 0.26). Although there was a (nonsignificant) association of \n",
      "IDDM with DQA2\"U\" among DR4-positive subjects (p = 0.14), this can be completely \n",
      "attributed to linkage disequilibrium between DQA2\"U\" and DQw8. We also sequenced \n",
      "most of the second exon (corresponding to the alpha 1 domain of the DQA2 \n",
      "glycoprotein) from five individuals that were homozygous for either DQA2\"U\" or \n",
      "DQA2\"L.\" The only polymorphisms observed were a \"silent\" mutation at position 36 \n",
      "and one example of a difference that would result in a change of amino acid at \n",
      "position 41.\n",
      "\n",
      "DOI: 10.1016/0198-8859(90)90039-r\n",
      "PMID: 1981060 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "89. Immunogenetics. 1990;31(4):229-32. doi: 10.1007/BF00204892.\n",
      "\n",
      "A SINE insertion provides information on the divergence of the HLA-DQA1 and \n",
      "HLA-DQA2 genes.\n",
      "\n",
      "Del Pozzo G(1), Guardiola J.\n",
      "\n",
      "Author information:\n",
      "(1)International Institute of Genetics and Biophysics, CNR, Napoli, Italy.\n",
      "\n",
      "The class II region of the human major histocompatibility complex (MHC) contains \n",
      "a cluster of highly polymorphic genes organized into at least three subloci (DR, \n",
      "DQ, and DP), each encoding a subset of surface antigens participating in the \n",
      "modulation of the immune response. Genetic diversity in this system is brought \n",
      "about by two major mechanisms, hypermutation and trans-species evolution. The DQ \n",
      "subregion contains a pair of closely related A genes, HLA-DQA1 and HLA-DQA2, \n",
      "whose phylogenetic relationship is uncertain, although their generation by \n",
      "duplication of an ancestral A gene before or after speciation can be implied. We \n",
      "report here the presence of a member of the Alu repetitive family immediately 5' \n",
      "to the HLA-DQA1 gene. The sequence of this element indicates that it may have \n",
      "integrated by transposition at the time of divergence of hominoids from Old \n",
      "World monkeys. HLA-DQA2 carries an empty integration target site in place of the \n",
      "Alu, thereby suggesting that the insertion of Alu near HLA-DQA1 was preceded by \n",
      "the separation of the two genes.\n",
      "\n",
      "DOI: 10.1007/BF00204892\n",
      "PMID: 2158485 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "90. Immunogenetics. 1989;30(6):432-9. doi: 10.1007/BF02421175.\n",
      "\n",
      "RFLP analysis of the HLA-, ChLA-, and RhLA-DQ alpha chain gene regions: \n",
      "conservation of restriction sites during evolution.\n",
      "\n",
      "Bontrop RE(1), Otting N, Broos LA, Noort MC, Kenter M, Jonker M.\n",
      "\n",
      "Author information:\n",
      "(1)TNO Primate Center, Rijswijk, The Netherlands.\n",
      "\n",
      "Genomic DNA samples, derived from a panel of 60 chimpanzees and 45 rhesus \n",
      "monkeys, were digested with the restriction enzymes Taq I and Bgl II and \n",
      "hybridized with an HLA-DQ alpha chain cDNA probe. The results were compared with \n",
      "the data available on a human reference panel. Use of the restriction enzyme Taq \n",
      "I and the DQ alpha chain probe allows the detection of five HLA-DQA1 and two \n",
      "HLA-DQA2 gene-associated fragments within the human population. For the ChLA and \n",
      "RhLA systems, 3 and 7 different DQA1-associated restriction patterns were \n",
      "detected, respectively, while for the chimpanzee a nonpolymorphic DQA2 (DX \n",
      "alpha) gene-associated fragment was also observed. The equivalent of the HLA- \n",
      "and ChLA-DQA2 genes appears to be absent in the rhesus monkey. The ChLA-DQA1 and \n",
      "-DQA2 gene-associated RFLP patterns are identical in man and chimpanzee, whereas \n",
      "such restriction site conservation is not seen in the rhesus monkey. The \n",
      "conclusion drawn is that the genetic organization of the HLA-DQA and ChLA-DQA \n",
      "gene regions, and possibly some of their allelic variabilities, already existed \n",
      "before man and chimpanzee separated in evolution. Moreover, the particular \n",
      "duplication which led to the generation of the HLA- and ChLA-DQA2 genes must \n",
      "have happened before speciation of members belonging to the superfamily \n",
      "Hominoidea (man, chimpanzee, etc), but probably after the separation of \n",
      "superfamily Cercopitecoidea (rhesus monkeys, baboons, etc.) from Hominoidea.\n",
      "\n",
      "DOI: 10.1007/BF02421175\n",
      "PMID: 2574158 [Indexed for MEDLINE]\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1. Hepatol Int. 2023 Nov 11. doi: 10.1007/s12072-023-10608-8. Online ahead of \\nprint.\\n\\nProteome-wide Mendelian randomization highlights AIF1 and HLA-DQA2 as targets \\nfor primary sclerosing cholangitis.\\n\\nChen L(1), Zhao Y(1), Li M(1), Lv G(2).\\n\\nAuthor information:\\n(1)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \\nFirst Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \\nJilin, China.\\n(2)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \\nFirst Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \\nJilin, China. lvgy@jlu.edu.cn.\\n\\nBACKGROUND: Primary sclerosing cholangitis (PSC) is a kind of cholestatic liver \\ndisease without effective therapies and its pathogenesis is largely unknown.\\nMETHODS: We performed the proteome-wide Mendelian randomization (MR) design to \\nestimate the causal associations of protein levels with PSC risk. Therein, \\ngenetic associations with 4,907 plasma protein levels were extracted from a \\nproteome-wide genome-wide association study (GWAS) with 35,559 individuals and \\nthose with PSC were obtained from the International PSC Study Group (2,871 cases \\nand 12,019 controls) and the FinnGen study (1,491 cases and 301,383 controls). \\nThe colocalization analysis was performed to detect causal variants shared by \\nproteins and PSC. The identified proteins were further enriched in pathways and \\ndiseases. A phenome-wide association screening was performed and potential drugs \\nwere assessed as well.\\nRESULTS: The results indicated that genetically predicted plasma levels of 14 \\nproteins were positively associated with an increased risk of PSC and 8 proteins \\nwere inversely associated with PSC risk in both PSC GWAS data sets, and they all \\nsurvived in sensitivity analyses. The colocalization indicated that AIF1 \\n(allograft inflammatory factor 1) and HLA-DQA2 (major histocompatibility \\ncomplex, class II, DQ alpha 2) were shared proteins with PSC, and they should be \\ndirect targets for PSC. The phenome-wide screening suggested that variants \\nlocated at AIF1 or HLA-DQA2 region were closely associated with several \\nautoimmune diseases, such as rheumatoid arthritis, implicating the shared \\npathogenesis among them.\\nCONCLUSIONS: Our study highly pinpointed two candidate targets (AIF1 and \\nHLA-DQA2) for PSC.\\n\\n© 2023. Asian Pacific Association for the Study of the Liver.\\n\\nDOI: 10.1007/s12072-023-10608-8\\nPMID: 37950809\\n\\n\\n2. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\\n 2023 Oct 10.\\n\\nGenome-wide identification of m(6)A-associated single nucleotide polymorphisms \\nin complex diseases of nervous system.\\n\\nGuo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \\nGuo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\\n\\nAuthor information:\\n(1)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China.\\n(2)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China.\\n(3)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\njiangyongshuai@hrbmu.edu.cn.\\n(4)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\nzhangmingming@hrbmu.edu.cn.\\n\\nN6-methyladenosine (m6A) is one of the most abundant chemical modifications on \\nRNA and can affect the occurrence and development of diseases. Some studies have \\nshown that the expressions of some m6A-related genes are significantly regulated \\nby single nucleotide variants (SNV). However, the function of m6A-associated \\nsingle nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \\n(MS), Alzheimer\\'s disease (AD) and Parkinson\\'s disease (PD). Here, we identified \\nthe disease-associated m6A-SNPs by integrating genome-wide association study \\n(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \\nbetween these identified m6A-SNPs and their target genes in eQTL analysis and \\ngene differential expression analysis. Finally, 26 genes corresponding to 20 \\nm6A-SNPs with eQTL signals were identified and differentially expressed \\n(P\\xa0<\\xa00.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P\\xa0<\\xa01e-04) were \\ndifferentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \\nexpression of 31 genes were identified. There were 5 HLA genes with eQTL signals \\n(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \\nthree diseases. In summary, our study provided new insights into understanding \\nthe potential roles of these m6A-SNPs in disease pathogenesis as well as \\ntherapeutic target.\\n\\nCopyright © 2023 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.neulet.2023.137513\\nPMID: 37827449 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n3. Br J Cancer. 2023 Oct;129(9):1510-1515. doi: 10.1038/s41416-023-02419-3. Epub \\n2023 Sep 7.\\n\\nIntegrating genomics and proteomics data to identify candidate plasma biomarkers \\nfor lung cancer risk among European descendants.\\n\\nYang Y(#)(1), Xu S(#)(2), Jia G(2), Yuan F(2), Ping J(2), Guo X(2), Tao R(3)(4), \\nShu XO(2), Zheng W(2), Long J(2), Cai Q(5).\\n\\nAuthor information:\\n(1)Center for Public Health Genomics, Department of Public Health Sciences, UVA \\nComprehensive Cancer Center, School of Medicine, University of Virginia, \\nCharlottesville, VA, USA. vta8we@virginia.edu.\\n(2)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \\nCenter, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \\nNashville, TN, USA.\\n(3)Department of Biostatistics, Vanderbilt University Medical Center, Nashville, \\nTN, USA.\\n(4)Vanderbilt Genetics Institute, Vanderbilt University Medical Center, \\nNashville, TN, USA.\\n(5)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \\nCenter, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \\nNashville, TN, USA. qiuyin.cai@vanderbilt.edu.\\n(#)Contributed equally\\n\\nBACKGROUND: Plasma proteins are potential biomarkers for complex diseases. We \\naimed to identify plasma protein biomarkers for lung cancer.\\nMETHODS: We investigated genetically predicted plasma levels of 1130 proteins in \\nassociation with lung cancer risk among 29,266 cases and 56,450 controls of \\nEuropean descent. For proteins significantly associated with lung cancer risk, \\nwe evaluated associations of genetically predicted expression of their coding \\ngenes with the risk of lung cancer.\\nRESULTS: Nine proteins were identified with genetically predicted plasma levels \\nsignificantly associated with overall lung cancer risk at a\\xa0false discovery rate \\n(FDR) of <0.05. Proteins C2, MICA, AIF1, and CTSH were associated with increased \\nlung cancer risk, while proteins SFTPB, HLA-DQA2, MICB, NRP1, and GMFG were \\nassociated with decreased lung cancer risk. Stratified analyses by histological \\ntypes revealed the cross-subtype consistency of these nine associations and \\nidentified an additional protein, ICAM5, significantly associated with lung \\nadenocarcinoma risk (FDR\\u2009<\\u20090.05). Coding genes of NRP1 and ICAM5 proteins\\xa0are \\nlocated at two loci that have never been reported by previous GWAS. Genetically \\npredicted blood levels of genes C2, AIF1, and CTSH were associated with lung \\ncancer risk, in directions consistent with those shown in protein-level \\nanalyses.\\nCONCLUSION: Identification of novel plasma protein biomarkers provided new \\ninsights into the biology of lung cancer.\\n\\n© 2023. The Author(s), under exclusive licence to Springer Nature Limited.\\n\\nDOI: 10.1038/s41416-023-02419-3\\nPMCID: PMC10628278\\nPMID: 37679517 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n4. Front Oncol. 2023 Jun 23;13:1150098. doi: 10.3389/fonc.2023.1150098.\\neCollection  2023.\\n\\nImmune microenvironment analysis and novel biomarkers of early-stage lung \\nadenocarcinoma evolution.\\n\\nLiu J(1), Ji Y(2), Weng X(3), Shao W(2), Zhao J(2), Chen H(4), Shen L(4), Wang \\nF(4), Meng Q(4), Wu X(4), Wang X(4), Ou Q(4), Ke H(3).\\n\\nAuthor information:\\n(1)Department of Chemotherapy, Affiliated Hospital of Nantong University, \\nNantong,\\xa0China.\\n(2)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \\nMedical School of Nantong University, Nantong,\\xa0China.\\n(3)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \\nNantong,\\xa0China.\\n(4)Geneseeq Research Institute, Nanjing Geneseeq Technology Inc., \\nNanjing,\\xa0China.\\n\\nBACKGROUND: Lung cancer is the deadliest and most diagnosed type of cancer \\nworldwide. The 5-year survival rate of lung adenocarcinoma (LUAD) dropped \\nsignificantly when tumor stages advanced. Patients who received surgically \\nresecting at the pre-invasive stage had a 5-year survival rate of nearly 100%. \\nHowever, the study on the differences in gene expression profiles and immune \\nmicroenvironment among pre-invasive LUAD patients is still lacking.\\nMETHODS: In this study, the gene expression profiles of three pre-invasive LUAD \\nstages were compared using the RNA-sequencing data of 10 adenocarcinoma in situ \\n(AIS) samples, 12 minimally invasive adenocarcinoma (MIA) samples, and 10 \\ninvasive adenocarcinoma (IAC) samples.\\nRESULTS: The high expression levels of PTGFRN (Hazard Ratio [HR] = 1.45; 95% \\nConfidence Interval [CI]: 1.08-1.94; log-rank P = 0.013) and SPP1 (HR = 1.44; \\n95% CI: 1.07-1.93; log-rank P = 0.015) were identified to be associated with \\nLUAD prognosis. Moreover, the early LUAD invasion was accompanied by the \\nenhancement of antigen presentation ability, reflected by the increase of \\nmyeloid dendritic cells infiltration rate (Cuzick test P < 0.01) and the \\nupregulation of seven important genes participating in the antigen presentation, \\nincluding HLA-A (Cuzick test P = 0.03), MICA (Cuzick test P = 0.01), MICB \\n(Cuzick test P = 0.01), HLA-DPA1 (Cuzick test P = 0.04), HLA-DQA2 (Cuzick test P \\n< 0.01), HLA-DQB1 (Cuzick test P = 0.03), and HLA-DQB2 (Cuzick test P < 0.01). \\nHowever, the tumor-killing ability of the immune system was inhibited during \\nthis process, as there were no rising cytotoxic T cell activity (Cuzick test P = \\n0.20) and no increasing expression in genes encoding cytotoxic proteins.\\nCONCLUSION: In all, our research elucidated the changes in the immune \\nmicroenvironment during early-stage LUAD evolution and may provide a theoretical \\nbasis for developing novel early-stage lung cancer therapeutic targets.\\n\\nCopyright © 2023 Liu, Ji, Weng, Shao, Zhao, Chen, Shen, Wang, Meng, Wu, Wang, Ou \\nand Ke.\\n\\nDOI: 10.3389/fonc.2023.1150098\\nPMCID: PMC10328385\\nPMID: 37427097\\n\\nConflict of interest statement: HC, LS, FW, QM, XWU, XWA, and QO are employees \\nof Nanjing Geneseeq Technology Inc., China. The remaining authors declare that \\nthe research was conducted in the absence of any commercial or financial \\nrelationships that could be construed as a potential conflict of interest.\\n\\n\\n5. Hereditas. 2023 May 31;160(1):27. doi: 10.1186/s41065-023-00289-6.\\n\\nNPM3 as a novel oncogenic factor and poor prognostic marker contributes to cell \\nproliferation and migration in lung adenocarcinoma.\\n\\nWei S(1)(2), Xing J(2)(3), Lu K(4), Wang K(1), Yu W(5).\\n\\nAuthor information:\\n(1)Department of Respiratory and Critical Care Medicine, The Fourth Affiliated \\nHospital, School of Medicine, Zhejiang University, Yiwu, 322000, Zhejiang, \\nPeople\\'s Republic of China.\\n(2)Department of Respiratory and Critical Care Medicine, The Affiliated People\\'s \\nHospital of Ningbo University (Ningbo Yinzhou People\\'s Hospital), No.251, \\nBaizhang Road, Ningbo, 315040, Zhejiang, People\\'s Republic of China.\\n(3)Ningbo University School of Medicine, Zhejiang Province, Ningbo, People\\'s \\nRepublic of China.\\n(4)Department of Urology, The Affiliated First Hospital of Ningbo University \\n(Ningbo First Hospital), No.59, Liuting Street, Ningbo, 315010, Zhejiang, \\nPeople\\'s Republic of China.\\n(5)Department of Respiratory and Critical Care Medicine, The Affiliated People\\'s \\nHospital of Ningbo University (Ningbo Yinzhou People\\'s Hospital), No.251, \\nBaizhang Road, Ningbo, 315040, Zhejiang, People\\'s Republic of China. \\nnbyuwanjun@163.com.\\n\\nBACKGROUND: Lung cancer is the leading cause of cancer-related deaths worldwide, \\nand despite recent advances in targeted therapies and immunotherapies, the \\nclinical benefit remains limited. Therefore, there is an urgent need to further \\ninvestigate the molecular mechanisms underlying lung cancer. The aim of this \\nstudy was to investigate the expression and function of NPM3 in the tumor \\nmicroenvironment of lung adenocarcinoma (LUAD).\\nMETHODS: We utilized bioinformatics tools and databases, including UALCAN, \\nGEPIA2, HPA, and Sangerbox, to analyze NPM3 expression in LUAD samples and its \\nassociation with prognosis and mutational landscape. NPM3 expression in various \\ncell types was assessed at the single cell level using the TISCH database. We \\nalso used algorithms such as TIMER and EPIC to explore the crosstalk between \\nNPM3 expression and immune features. KEGG enrichment analysis was performed to \\nidentify potential signaling pathways of NPM3. Finally, we employed siRNA \\nknockdown strategy to investigate the effect of NPM3 on LUAD cell proliferation \\nand migration in vitro.\\nRESULTS: NPM3 was significantly upregulated in LUAD tissues and was strongly \\nassociated with poor prognosis and TP53 gene mutations. Single-cell sequencing \\nanalysis revealed that NPM3 was expressed in immune cells (dendritic cells and \\nmonocytes/macrophages) in the tumor microenvironment. Moreover, NPM3 expression \\nwas negatively associated with immune B cell and CD4 T cell infiltration, as \\nwell as with several immune-related genes (including CCL22, CXCR2, CX3CR1, CCR6, \\nHLA-DOA, HLA-DQA2). KEGG enrichment analysis indicated that NPM3 expression was \\nassociated with cell cycle, CAMs, and NSCLC pathway genes. Finally, in vitro \\nexperiments showed that NPM3 knockdown inhibited LUAD cell proliferation and \\nmigration in NCI-H1299 and SPC-A1 cells, and suppressed the expression of CCNA2 \\nand MAD2L1.\\nCONCLUSION: Elevated NPM3 expression predicts poor clinical outcome and an \\nimmunosuppressive microenvironment in LUAD tissues. NPM3 promotes LUAD \\nprogression by promoting cell proliferation and migration, and targeting NPM3 \\nmay represent a novel therapeutic strategy for LUAD.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s41065-023-00289-6\\nPMCID: PMC10230701\\nPMID: 37254219 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors claim that there is no possible \\nconflict of interest.\\n\\n\\n6. Rheumatology (Oxford). 2023 Sep 1;62(9):3213-3218. doi: \\n10.1093/rheumatology/kead152.\\n\\nStratified genetic analysis reveals sex differences in MPO-ANCA-associated \\nvasculitis.\\n\\nEkman D(1), Sennblad B(2), Knight A(3), Karlsson Å(4), Rantapää-Dahlqvist S(5), \\nBerglin E(5), Stegmayr B(5), Baslund B(6), Palm Ø(7), Haukeland H(8), Gunnarsson \\nI(9)(10), Bruchfeld A(11)(12), Segelmark M(13), Ohlsson S(13), Mohammad \\nAJ(14)(15), Svärd A(16), Pullerits R(17)(18), Herlitz H(19), Söderbergh A(20), \\nOmdal R(21)(22), Jonsson R(23), Rönnblom L(3), Eriksson P(24), Lindblad-Toh \\nK(4)(25), Dahlqvist J(3)(4)(25).\\n\\nAuthor information:\\n(1)Department of Biochemistry and Biophysics, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Stockholm University, \\nSweden.\\n(2)Department of Cell and Molecular Biology, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala, \\nSweden.\\n(3)Department of Medical Sciences, Uppsala University, Uppsala, Sweden.\\n(4)Science for Life Laboratory, Department of Medical Biochemistry and \\nMicrobiology, Uppsala University, Uppsala, Sweden.\\n(5)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \\nSweden.\\n(6)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \\nDiseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\\n(7)Department of Rheumatology, Oslo University Hospital, Oslo, Norway.\\n(8)Department of Rheumatology, Martina Hansens Hospital, Gjettum, Norway.\\n(9)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \\nStockholm, Sweden.\\n(10)Unit of Rheumatology, Karolinska University Hospital, Stockholm, Sweden.\\n(11)Department of Health, Medicine and Caring Sciences, Linköping University, \\nLinköping, Sweden.\\n(12)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \\nKarolinska Institutet, Stockholm, Sweden.\\n(13)Department of Clinical Sciences, Division of Nephrology, Lund University and \\nSkåne University Hospital, Lund, Sweden.\\n(14)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \\nUniversity Hospital, Lund University, Lund, Sweden.\\n(15)Department of Medicine, University of Cambridge, Cambridge, UK.\\n(16)Center for Clinical Research Dalarna, Uppsala University, Uppsala, Sweden.\\n(17)Department of Rheumatology and Inflammation Research, Institution of \\nMedicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden.\\n(18)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \\nUniversity Hospital, Gothenburg, Sweden.\\n(19)Department of Molecular and Clinical Medicine/Nephrology, Institute of \\nMedicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden.\\n(20)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\\n(21)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \\nUniversity Hospital, Stavanger, Norway.\\n(22)Department of Clinical Science, University of Bergen, Bergen, Norway.\\n(23)Broegelmann Research Laboratory, Department of Clinical Science, University \\nof Bergen, Bergen, Norway.\\n(24)Department of Biomedical and Clinical Sciences, Division of Inflammation and \\nInfection, Linköping University, Linköping, Sweden.\\n(25)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\\n\\nOBJECTIVE: To identify and genetically characterize subgroups of patients with \\nANCA-associated vasculitides (AAV) based on sex and ANCA subtype.\\nMETHODS: A previously established SNP dataset derived from DNA sequencing of \\n1853 genes and genotyping of 1088 Scandinavian cases with AAV and 1589 controls \\nwas stratified for sex and ANCA subtype and analysed for association with five \\ntop AAV SNPs. rs9274619, a lead variant at the HLA-DQB1/HLA-DQA2 locus \\npreviously associated with AAV positive for myeloperoxidase (MPO)-ANCA, was \\nanalysed for association with the cumulative disease involvement of ten \\ndifferent organ systems.\\nRESULTS: rs9274619 showed a significantly stronger association to \\nMPO-ANCA-positive females than males [P\\u2009=\\u20092.0\\u2009×\\u200910-4, OR\\u2009=\\u20092.3 (95% CI 1.5, \\n3.5)], whereas proteinase 3 (PR3)-ANCA-associated variants rs1042335, rs9277341 \\n(HLA-DPB1/A1) and rs28929474 (SERPINA1) were equally associated with females and \\nmales with PR3-ANCA. In MPO-ANCA-positive cases, carriers of the rs9274619 risk \\nallele were more prone to disease engagement of eyes [P\\u2009=\\u20090.021, OR\\u2009=\\u200911 (95% CI \\n2.2, 205)] but less prone to pulmonary involvement [P\\u2009=\\u20090.026, OR\\u2009=\\u20090.52 (95% CI \\n0.30, 0.92)]. Moreover, AAV with both MPO-ANCA and PR3-ANCA was associated with \\nthe PR3-ANCA lead SNP rs1042335 [P\\u2009=\\u20090.0015, OR\\u2009=\\u20090.091 (95% CI 0.0022, 0.55)] \\nbut not with rs9274619.\\nCONCLUSIONS: Females and males with MPO-ANCA-positive AAV differ in genetic \\npredisposition to disease, suggesting at least partially distinct disease \\nmechanisms between the sexes. Double ANCA-positive AAV cases are genetically \\nsimilar to PR3-ANCA-positive cases, providing clues to the clinical follow-up \\nand treatment of these patients.\\n\\n© The Author(s) 2023. Published by Oxford University Press on behalf of the \\nBritish Society for Rheumatology.\\n\\nDOI: 10.1093/rheumatology/kead152\\nPMCID: PMC10473270\\nPMID: 37004177 [Indexed for MEDLINE]\\n\\n\\n7. BMC Genomics. 2023 Mar 27;24(1):153. doi: 10.1186/s12864-023-09227-2.\\n\\nGenome-wide identification of RNA modification-related single nucleotide \\npolymorphisms associated with rheumatoid arthritis.\\n\\nWang M(#)(1)(2), Wu J(#)(1)(2), Lei S(1)(2), Mo X(3)(4).\\n\\nAuthor information:\\n(1)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \\nSchool of Public Health, Medical College of Soochow University, 199 Renai Road, \\nSuzhou, Jiangsu, 215123, People\\'s Republic of China.\\n(2)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \\nDiseases, School of Public Health, Medical College of Soochow University, \\nSuzhou, China.\\n(3)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \\nSchool of Public Health, Medical College of Soochow University, 199 Renai Road, \\nSuzhou, Jiangsu, 215123, People\\'s Republic of China. xbmo@suda.edu.cn.\\n(4)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \\nDiseases, School of Public Health, Medical College of Soochow University, \\nSuzhou, China. xbmo@suda.edu.cn.\\n(#)Contributed equally\\n\\nBACKGROUND: RNA modification plays important roles in many biological processes, \\nsuch as gene expression control. The aim of this study was to identify single \\nnucleotide polymorphisms related to RNA modification (RNAm-SNPs) for rheumatoid \\narthritis (RA) as putative functional variants.\\nMETHODS: We examined the association of RNAm-SNPs with RA in summary data from a \\ngenome-wide association study of 19,234 RA cases and 61,565 controls. We \\nperformed eQTL and pQTL analyses for the RNAm-SNPs to find associated gene \\nexpression and protein levels. Furthermore, we examined the associations of gene \\nexpression and circulating protein levels with RA using two-sample Mendelian \\nrandomization analysis methods.\\nRESULTS: A total of 160 RNAm-SNPs related to m6A, m1A, A-to-I, m7G, m5C, m5U and \\nm6Am modifications were identified to be significantly associated with RA. These \\nRNAm-SNPs were located in 62 protein-coding genes, which were significantly \\nenriched in immune-related pathways. RNAm-SNPs in important RA susceptibility \\ngenes, such as PADI2, SPRED2, PLCL2, HLA-A, HLA-B, HLA-DRB1, HLA-DPB1, TRAF1 and \\nTXNDC11, were identified. Most of these RNAm-SNPs showed eQTL effects, and the \\nexpression levels of 26 of the modifiable genes (e.g., PADI2, TRAF1, HLA-A, \\nHLA-DRB1, HLA-DPB1 and HLA-B) in blood cells were associated with RA. \\nCirculating protein levels, such as CFB, GZMA, HLA-DQA2, IL21, LRPAP1 and TFF3, \\nwere affected by RNAm-SNPs and were associated with RA.\\nCONCLUSION: The present study identified RNAm-SNPs in the reported RA \\nsusceptibility genes and suggested that RNAm-SNPs may affect RA risk by \\naffecting the expression levels of corresponding genes and proteins.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s12864-023-09227-2\\nPMCID: PMC10045113\\nPMID: 36973646 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n8. Yi Chuan. 2023 Jan 20;45(1):52-66. doi: 10.16288/j.yczz.22-293.\\n\\nApplication of transcriptome in time analysis and donor characterization in \\nblood samples.\\n\\nZhang J(1)(2), Liu KH(2), Zhang Y(2), Hao JP(2), Zhang GF(2), Xu XY(2), Chang \\nJ(2), Liu XP(3), Yang XY(2), Ye J(1)(2).\\n\\nAuthor information:\\n(1)1. People\\'s Public Security University of China, Beijing 100038, China.\\n(2)2. Institute of Forensic Science Ministry of Public Security, Beijing 100038, \\nChina.\\n(3)3. Ruijin Public Security Bureau, Ruijin 342500, China.\\n\\nAs an effective supplement to the current forensic DNA typing and one of the \\nresearch hotpots in forensic science, the in-depth mining and characterization \\nof biological evidence can provide rich and reliable clues for case \\ninvestigation. In this study, the time-dependent variations of transcriptome \\nwere confirmed in in vitro blood samples within 0-168 days and a random forest \\nmodel was established to realize the classification of blood samples with \\ndifferent TSD (time since deposition). Meanwhile, significant differences were \\nobserved in the transcripts of blood samples with different smoking habits and \\ngenders within a certain time period. HLA-DRB1, HLA-DQB1 and HLA-DQA2 were \\nidentified as markers for smoking habit identification, while the transcripts \\nfor RPS4Y1 and EIF1AY from the non-recombining region of the Y chromosome (NRY) \\nwere identified as markers for male sex identification. Thus, this study \\nprovides a theoretical foundation and experimental strategy for establishing a \\ntranscriptome-based method for characterizing blood sample retention time and \\ndonor characteristics in the field of forensic investigation.\\n\\nPublisher: \\n案件现场生物物证信息深度挖掘与刻画，可以为案件侦查、涉案人员查找提供丰富可靠线索，是当前法医DNA检验的有效补充，也是国内外法庭科学的研究热点之一。本研究以血液样本为研究对象，证实了0~168天内离体血液样本转录组变化的时间相关性，并建立随机森林模型实现不同离体时间血液样本分类。同时，进一步证实相同离体时间段内，不同吸烟习惯和不同性别供体来源的血液样本转录本具有显著差异，HLA-DRB1、HLA-DQB1和HLA-DQA2可以作为供体吸烟习惯判别标志，Y染色体非重组区(non-recombining \\nY, \\nNRY)的转录本RPS4Y1和EIF1AY可以作为供体性别特征判别标志。本研究为法庭科学领域建立基于转录组分析的血液样本遗留时间和供体特征刻画方法提供了理论基础和实验依据。.\\n\\nDOI: 10.16288/j.yczz.22-293\\nPMID: 36927638 [Indexed for MEDLINE]\\n\\n\\n9. Pediatr Rheumatol Online J. 2023 Feb 15;21(1):17. doi: \\n10.1186/s12969-023-00787-x.\\n\\nProfiling the peripheral immune response to ex vivo TNF stimulation in untreated \\njuvenile idiopathic arthritis using single cell RNA sequencing.\\n\\nImbach KJ(#)(1), Treadway NJ(#)(2), Prahalad V(2), Kosters A(3), Arafat D(1), \\nDuan M(1), Gergely T(2), Ponder LA(4), Chandrakasan S(4)(5), Ghosn EEB(2)(3)(4), \\nPrahalad S(#)(6)(7)(8), Gibson G(#)(1)(4).\\n\\nAuthor information:\\n(1)Center for Integrative Genomics, School of Biological Sciences, Georgia \\nInstitute of Technology, Atlanta, GA, 30332, USA.\\n(2)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \\n30223, USA.\\n(3)Lowance Center for Human Immunology, Division of Immunology, Department of \\nMedicine, Emory University School of Medicine, Atlanta, GA, 30223, USA.\\n(4)Center for Immunity and Applied Genomics, Children\\'s Healthcare of Atlanta, \\nAtlanta, GA, 30223, USA.\\n(5)Aflac Cancer and Blood Disorders Center, Department of Pediatrics, Children\\'s \\nHealthcare of Atlanta, Emory University School of Medicine, Atlanta, GA, 30223, \\nUSA.\\n(6)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \\n30223, USA. sprahal@emory.edu.\\n(7)Center for Immunity and Applied Genomics, Children\\'s Healthcare of Atlanta, \\nAtlanta, GA, 30223, USA. sprahal@emory.edu.\\n(8)Department of Human Genetics, Emory University School of Medicine, Atlanta, \\nGA, 30223, USA. sprahal@emory.edu.\\n(#)Contributed equally\\n\\nBACKGROUND: Juvenile Idiopathic Arthritis (JIA) is an autoimmune disease with a \\nheterogenous clinical presentation and unpredictable response to available \\ntherapies. This personalized transcriptomics study sought proof-of-concept for \\nsingle-cell RNA sequencing to characterize patient-specific immune profiles.\\nMETHODS: Whole blood samples from six untreated children, newly diagnosed with \\nJIA, and two healthy controls were cultured for 24\\xa0h with or without ex vivo TNF \\nstimulation and subjected to scRNAseq to examine cellular populations and \\ntranscript expression in PBMCs. A novel analytical pipeline, scPool, was \\ndeveloped wherein cells are first pooled into pseudocells prior to expression \\nanalysis, facilitating variance partitioning of the effects of TNF stimulus, JIA \\ndisease status, and individual donor.\\nRESULTS: Seventeen robust immune cell-types were identified, the abundance of \\nwhich was significantly affected by TNF stimulus, which resulted in notable \\nelevation of memory CD8\\u2009+\\u2009T-cells and NK56 cells, but down-regulation of naïve \\nB-cell proportions. Memory CD8\\u2009+\\u2009and CD4\\u2009+\\u2009T-cells were also both reduced in the \\nJIA cases relative to two controls. Significant differential expression \\nresponses to TNF stimulus were also characterized, with monocytes showing more \\ntranscriptional shifts than T-lymphocyte subsets, while the B-cell response was \\nmore limited. We also show that donor variability exceeds the small degree of \\npossible intrinsic differentiation between JIA and control profiles. An \\nincidental finding of interest was association of HLA-DQA2 and HLA-DRB5 \\nexpression with JIA status.\\nCONCLUSIONS: These results support the development of personalized \\nimmune-profiling combined with ex-vivo immune stimulation for evaluation of \\npatient-specific modes of immune cell activity in autoimmune rheumatic disease.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s12969-023-00787-x\\nPMCID: PMC9929251\\nPMID: 36793127 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Dr. Prahalad has served on a macrophage \\nactivation syndrome adjudication committee for Novartis, but this had no bearing \\non the work described in this manuscript. None of the other authors have any \\ncompeting interests in the manuscript.\\n\\n\\n10. Epilepsia. 2023 Jan;64(1):218-230. doi: 10.1111/epi.17457. Epub 2022 Nov 16.\\n\\nBrain molecular mechanisms in Rasmussen encephalitis.\\n\\nLeitner DF(1)(2), Lin Z(3), Sawaged Z(3), Kanshin E(4), Friedman D(1), Devore \\nS(1), Ueberheide B(2)(4)(5), Chang JW(6), Mathern GW(6), Anink JJ(7), Aronica \\nE(7)(8), Wisniewski T(2)(9)(10), Devinsky O(1).\\n\\nAuthor information:\\n(1)Comprehensive Epilepsy Center, NYU Grossman School of Medicine, New York, New \\nYork, USA.\\n(2)Center for Cognitive Neurology, Department of Neurology, NYU Grossman School \\nof Medicine, New York, New York, USA.\\n(3)Applied Bioinformatics Laboratories, NYU Grossman School of Medicine, New \\nYork, New York, USA.\\n(4)Proteomics Laboratory, Division of Advanced Research Technologies, NYU \\nGrossman School of Medicine, New York, New York, USA.\\n(5)Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of \\nMedicine, New York, New York, USA.\\n(6)Departments of Neurosurgery and Psychiatry, David Geffen School of Medicine, \\nUniversity of California, Los Angeles, Los Angeles, California, USA.\\n(7)Department of (Neuro) Pathology, Amsterdam University Medical Center, \\nUniversity of Amsterdam, Amsterdam Neuroscience, Amsterdam, the Netherlands.\\n(8)Epilepsy Institutes of the Netherlands Foundation, Heemstede, the \\nNetherlands.\\n(9)Department of Pathology, NYU Grossman School of Medicine, New York, New York, \\nUSA.\\n(10)Department of Psychiatry, NYU Grossman School of Medicine, New York, New \\nYork, USA.\\n\\nOBJECTIVE: This study was undertaken to identify molecular mechanisms in brain \\ntissue of Rasmussen encephalitis (RE) when compared to people with non-RE \\nepilepsy (PWE) and control cases using whole exome sequencing (WES), RNAseq, and \\nproteomics.\\nMETHODS: Frozen brain tissue (ages\\xa0=\\xa02-19\\u2009years) was obtained from control \\nautopsy (n\\xa0=\\xa014), surgical PWE (n\\xa0=\\xa010), and surgical RE cases (n\\xa0=\\xa027). We \\nevaluated WES variants in RE associated with epilepsy, seizures, RE, and human \\nleukocyte antigens (HLAs). Differential expression was evaluated by RNAseq \\n(adjusted p\\u2009<\\u2009.05) and label-free quantitative mass spectrometry (false \\ndiscovery rate\\u2009<\\u20095%) in the three groups.\\nRESULTS: WES revealed no common pathogenic variants in RE, but several rare and \\nlikely deleterious variants of unknown significance (VUS; ANGPTL7/MTOR, SCN1A, \\nFCGR3B, MTOR) and more common HLA VUS in >25% of RE cases (HLA-DRB1, HLA-DQA2), \\nall with allele frequency\\u2009<\\u20095% in the general population. RNAseq in RE versus \\nPWE (1516 altered transcripts) revealed significant activation of crosstalk \\nbetween dendritic and natural killer cells (p\\xa0=\\xa07.94\\u2009×\\u200910-6 , z\\xa0=\\xa02.65), in RE \\nversus control (7466 transcripts) neuroinflammation signaling activation \\n(p\\xa0=\\xa06.31\\u2009×\\u200910-13 , z\\xa0=\\xa05.07), and in PWE versus control (945 transcripts) \\nphagosome formation activation (p\\xa0=\\xa02.00\\u2009×\\u200910-13 , z\\xa0=\\xa05.61). Proteomics \\ndetected fewer altered targets.\\nSIGNIFICANCE: In RE, we identified activated immune signaling pathways and \\nimmune cell type annotation enrichment that suggest roles of the innate and \\nadaptive immune responses, as well as HLA variants that may increase \\nvulnerability to RE. Follow-up studies could evaluate cell type density and \\nsubregional localization associated with top targets, clinical history \\n(neuropathology, disease duration), and whether modulating crosstalk between \\ndendritic and natural killer cells may limit disease progression.\\n\\n© 2022 International League Against Epilepsy.\\n\\nDOI: 10.1111/epi.17457\\nPMCID: PMC9852002\\nPMID: 36336987 [Indexed for MEDLINE]\\n\\n\\n11. PLoS One. 2022 Oct 21;17(10):e0275934. doi: 10.1371/journal.pone.0275934. \\neCollection 2022.\\n\\nGenome-wide discovery for diabetes-dependent triglycerides-associated loci.\\n\\nSelvaraj MS(1)(2)(3), Paruchuri K(1)(2), Haidermota S(1)(2), Bernardo R(1)(2), \\nRich SS(4), Peloso GM(5), Natarajan P(1)(2)(3).\\n\\nAuthor information:\\n(1)Cardiovascular Research Center, Massachusetts General Hospital, Boston, MA, \\nUnited States of America.\\n(2)Program in Medical and Population Genetics, Broad Institute of Harvard and \\nMIT, Cambridge, MA, United States of America.\\n(3)Department of Medicine, Harvard Medical School, Boston, MA, United States of \\nAmerica.\\n(4)Center for Public Health Genomics, University of Virginia, Charlottesville, \\nVA, United States of America.\\n(5)Department of Biostatistics, Boston University School of Public Health, \\nBoston, MA, United States of America.\\n\\nPURPOSE: We aimed to discover loci associated with triglyceride (TG) levels in \\nthe context of type 2 diabetes (T2D). We conducted a genome-wide association \\nstudy (GWAS) in 424,120 genotyped participants of the UK Biobank (UKB) with T2D \\nstatus and TG levels.\\nMETHODS: We stratified the cohort based on T2D status and conducted association \\nanalyses of TG levels for genetic variants with minor allele count (MAC) at \\nleast 20 in each stratum. Effect differences of genetic variants by T2D status \\nwere determined by Cochran\\'s Q-test and we validated the significantly \\nassociated variants in the Mass General Brigham Biobank (MGBB).\\nRESULTS: Among 21,176 T2D and 402,944 non-T2D samples from UKB, stratified GWAS \\nidentified 19 and 315 genomic risk loci significantly associated with TG levels, \\nrespectively. Only chr6p21.32 exhibited genome-wide significant heterogeneity \\n(I2 = 98.4%; pheterogeneity = 2.1x10-15), with log(TG) effect estimates of \\n-0.066 (95%CI: -0.082, -0.050) and 0.002 (95%CI: -0.002, 0.006) for T2D and \\nnon-T2D, respectively. The lead variant rs9274619:A (allele frequency 0.095) is \\nlocated 2Kb upstream of the HLA-DQB1 gene, between HLA-DQB1 and HLA-DQA2 genes. \\nWe replicated this finding among 25,137 participants (6,951 T2D cases) of MGBB \\n(pheterogeneity = 9.5x10-3). Phenome-wide interaction association analyses \\nshowed that the lead variant was strongly associated with a concomitant \\ndiagnosis of type 1 diabetes (T1D) as well as diabetes-associated complications.\\nCONCLUSION: In conclusion, we identified an intergenic variant near \\nHLA-DQB1/DQA2 significantly associates with decreased triglycerides only among \\nthose with T2D and highlights an immune overlap with T1D.\\n\\nDOI: 10.1371/journal.pone.0275934\\nPMCID: PMC9586367\\nPMID: 36269708 [Indexed for MEDLINE]\\n\\nConflict of interest statement: P.N. reports grants from Amgen, Apple, \\nAstraZeneca, Boston Scientific, and Novartis, personal fees from Apple, \\nAstraZeneca, Blackstone Life Sciences, Foresite Labs, Genentech / Roche, \\nNovartis, and TenSixteen Bio, equity in geneXwell, and TenSixteen Bio, \\nco-founder of TenSixteen Bio, and spousal employment at Vertex, all unrelated to \\nthe present work. This does not alter our adherence to PLOS ONE policies on \\nsharing data and materials.\\n\\n\\n12. Medicine (Baltimore). 2022 Oct 7;101(40):e31071. doi: \\n10.1097/MD.0000000000031071.\\n\\nBioinformatics analysis of the clinical significance of HLA class II in breast \\ncancer.\\n\\nWu G(1), Xiao G(2), Yan Y(3), Guo C(4), Hu N(5), Shen S(3).\\n\\nAuthor information:\\n(1)Finance Section, Yuebei People\\'s Hospital, Shantou University, Shaoguan, \\nChina.\\n(2)Department of Pathology, Yuebei People\\'s Hospital, Shantou University, \\nShaoguan, China.\\n(3)Breast Surgery, The Sixth Affiliated Hospital of Guangzhou Medical \\nUniversity, Qingyuan People\\'s Hospital, Qingyuan, China.\\n(4)Department of Radiology, 82 Group Hospital of PLA, Baoding, China.\\n(5)Thoracic surgery, The Sixth Affiliated Hospital of Guangzhou Medical \\nUniversity, Qingyuan People\\'s Hospital, Qingyuan, China.\\n\\nHuman leukocyte antigen (HLA) class II plays critical roles in antigen \\npresentation and the initiation of immune responses. However, the correlation \\nbetween the HLA class II gene expression level and the survival of patients with \\nbreast cancer is still under investigation. We analyzed microarray and RNA-Seq \\ndata of breast cancer from the cancer genome atlas (TCGA), genotype-tissue \\nexpression (GTEx) and Oncomine databases by using bioinformatics tools. The \\nexpression of the HLA-DQA1, HLA-DQA2, and HLA-DQB2 genes was significantly \\nupregulated in breast cancer. Higher expression levels of HLA class II genes in \\nbreast cancer, especially HLA-DOB and HLA-DQB2, were significantly associated \\nwith better overall survival. Furthermore, the expression of HLA class II genes \\nwas more closely associated with survival in breast cancer than in other cancer \\ntypes. CD48 coexpressed with both HLA-DOB and HLA-DQB2 was also positively \\nassociated with the overall survival of breast cancer patients. The results \\nindicated that HLA class II and CD48 may enhance antitumor immunity, and their \\nexpression patterns may serve as potential prognostic biomarkers and therapeutic \\ntargets in breast cancer.\\n\\nCopyright © 2022 the Author(s). Published by Wolters Kluwer Health, Inc.\\n\\nDOI: 10.1097/MD.0000000000031071\\nPMCID: PMC9543021\\nPMID: 36221383 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have no conflicts of interest to \\ndisclose.\\n\\n\\n13. Front Immunol. 2022 Sep 9;13:1001381. doi: 10.3389/fimmu.2022.1001381. \\neCollection 2022.\\n\\nA specific immune signature for predicting the prognosis of glioma patients with \\nIDH1-mutation and guiding immune checkpoint blockade therapy.\\n\\nZeng Z(1)(2), Hu C(3)(4), Ruan W(5), Zhang J(1)(2), Lei S(1)(2), Yang Y(6), Peng \\nP(5)(7), Pan F(8), Chen T(1)(2).\\n\\nAuthor information:\\n(1)Transformation Engineering Research Center of Chronic Disease Diagnosis and \\nTreatment, Guizhou Medical University, Guiyang, China.\\n(2)Department of Physiology, School of Basic Medicine, Guizhou Medical \\nUniversity, Guiyang, China.\\n(3)State Key Laboratory of Functions and Applications of Medicinal Plants, \\nGuizhou Medical University, Guiyang, China.\\n(4)Guizhou Provincial Engineering Technology Research Center for Chemical Drug \\nR&D, Guizhou Medical University, Guiyang, China.\\n(5)School of Clinical Medicine, Guizhou Medical University, Guiyang, China.\\n(6)Department of Pathology, Affiliated Hospital of Guizhou Medical University, \\nGuiyang, China.\\n(7)Department of Gastroenterology, Affiliated Hospital of Guizhou Medical \\nUniversity, Guiyang, China.\\n(8)Department of Bone and Joint Surgery, Gui Zhou Orthopedic Hospital, Guiyang, \\nChina.\\n\\nIsocitrate dehydrogenase (IDH1) is frequently mutated in glioma tissues, and \\nthis mutation mediates specific tumor-promoting mechanisms in glioma cells. We \\naimed to identify specific immune biomarkers for IDH1-mutation (IDH1mt) glioma. \\nThe Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) were used \\nto obtain RNA sequencing data and clinical characteristics of glioma tissues, \\nwhile the stromal and immune scores of TCGA glioma tissues were determined using \\nthe ESTIMATE algorithm. Differentially expressed genes (DEGs), the \\nprotein-protein interaction(PPI) network, and least absolute shrinkage and \\nselection operator (LASSO) and Cox regression analyses were used to select hub \\ngenes associated with stroma and immune scores and the prognoses of patients and \\nto construct the risk model. The practicability and specificity of the risk \\nmodel in both IDH1mt and IDH1-wildtype (wtIDH1) gliomas in TCGA and CGGA were \\nevaluated. Molecular mechanisms, immunological characteristics and benefits of \\nimmune checkpoint blockade therapy in glioma tissues with IDH1mt were analyzed \\nusing GSEA, immunohistochemical staining, CIBERSORT, and T-cell dysfunction and \\nexclusion (TIDE) analysis. The overall survival rate for IDH1mt-glioma patients \\nwith high stroma/immune scores was lower than that for those with low \\nstroma/immune scores. A total of 222 DEGs were identified in IDH1mt glioma \\ntissues with high stroma/immune scores. Among them, 72 genes had interactions in \\nthe PPI network, while three genes, HLA-DQA2, HOXA3, and SAA2, were selected as \\nhub genes and used to construct risk models classifying patients into high- and \\nlow-risk score groups, followed by LASSO and Cox regression analyses. This risk \\nmodel showed prognostic value in IDH1mt glioma in both TCGA and CCGA; \\nnevertheless, the model was not suitable for wtIDH1 glioma. The risk model may \\nact as an independent prognostic factor for IDH1mt glioma. IDH1mt glioma tissues \\nfrom patients with high-risk scores showed more infiltration of M1 and CD8 T \\ncells than those from patients with low-risk scores. Moreover, TIDE analysis \\nshowed that immune checkpoint blockade(ICB) therapy was highly beneficial for \\nIDH1mt patients with high-risk scores. The risk model showed specific potential \\nto predict the prognosis of IDH1mt-glioma patients, as well as guide ICB, \\ncontributing to the diagnosis and therapy of IDH1mt-glioma patients.\\n\\nCopyright © 2022 Zeng, Hu, Ruan, Zhang, Lei, Yang, Peng, Pan and Chen.\\n\\nDOI: 10.3389/fimmu.2022.1001381\\nPMCID: PMC9500319\\nPMID: 36159801 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n14. J Immunol Res. 2022 Jun 17;2022:7212852. doi: 10.1155/2022/7212852.\\neCollection  2022.\\n\\nThe Potential Diagnostic Value of Immune-Related Genes in Interstitial Fibrosis \\nand Tubular Atrophy after Kidney Transplantation.\\n\\nYang B(1), Shi D(2), Chen Y(1), Zhu Y(1).\\n\\nAuthor information:\\n(1)Department of Hepato-Pancreato-Biliary Surgery, The Second Affiliated \\nHospital, Zhejiang University School of Medicine, Zhejiang, China.\\n(2)Department of Gastrointestinal Surgery, The Second Affiliated Hospital, \\nZhejiang University School of Medicine, Zhejiang, China.\\n\\nBACKGROUND: Inflammation within areas of interstitial fibrosis and tubular \\natrophy (IF/TA) is associated with kidney allograft failure. The aim of this \\nstudy was to reveal new diagnostic markers of IF/TA based on bioinformatics \\nanalysis.\\nMETHODS: Raw data of IF/TA samples after kidney transplantation and control \\nsamples after kidney transplantation were extracted from the Gene Expression \\nOmnibus (GEO) database (GSE76882 and GSE120495 datasets), and genes that were \\ndifferentially expressed between the two groups (DEGs) were screened. Gene Set \\nEnrichment Analysis (GSEA), ESTIMATE and single sample GSEA (ssGSEA), least \\nabsolute shrinkage and selection operator (LASSO) regression analysis, and \\ncompeting endogenous RNA (ceRNA) network were used to analyze the data.\\nRESULTS: The results of GSEA revealed that multiple immune-related pathways were \\nenriched in the IF/TA group, and subsequent immune landscape analysis also \\nshowed that the IF/TA group had higher immune and stromal scores and up to 15 \\ntypes of immune cells occupied them, such as B cells, cytotoxic cells, and T \\ncells. LASSO regression analysis selected 6 (including ANGPTL3, APOH, LTF, \\nFCGR2B, HLA-DQA2, and EGF) out of 14 DE-IRGs as diagnostic genes to construct a \\ndiagnostic model. Then, receiver operating characteristic (ROC) curve analysis \\nshowed the powerful diagnostic value of the model, and the area under the curve \\n(AUC) of a single diagnostic gene was greater than 0.75. The results of \\ningenuity pathway analysis (IPA) also indicated that DEGs were involved in the \\nimmune system and kidney disease-related pathways. Finally, we found multiple \\nmiRNAs that could regulate diagnostic genes from the ceRNA network.\\nCONCLUSION: This study identified 6 IF/TA-related genes, which might be used as \\na new diagnosis model in the clinical practice.\\n\\nCopyright © 2022 Bin Yang et al.\\n\\nDOI: 10.1155/2022/7212852\\nPMCID: PMC9232312\\nPMID: 35755170 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n15. Front Immunol. 2022 May 26;13:878226. doi: 10.3389/fimmu.2022.878226. \\neCollection 2022.\\n\\nSingle-Cell RNA and ATAC Sequencing Reveal Hemodialysis-Related Immune \\nDysregulation of Circulating Immune Cell Subpopulations.\\n\\nWu H(1)(2), Dong J(1)(2), Yu H(1), Wang K(1), Dai W(3), Zhang X(1), Hu N(1), Yin \\nL(2), Tang D(1), Liu F(2), Dai Y(1).\\n\\nAuthor information:\\n(1)The Second Clinical Medical College of Jinan University, Shenzhen People\\'s \\nHospital, Shenzhen, China.\\n(2)Institute of Nephrology and Blood Purification, The First Affiliated Hospital \\nof Jinan University, Jinan University, Guangzhou, China.\\n(3)College of Natural Science, University of Texas at Austin, Austin, TX, United \\nStates.\\n\\nBACKGROUND: An increased risk of infection, malignancy, and cardiovascular \\ndiseases in maintenance hemodialysis patients is associated with \\nhemodialysis-related immunity disturbances. Although defects in \\nT-lymphocyte-dependent immune responses and preactivation of antigen-presenting \\ncells have been documented in hemodialysis patients, the effects of long-term \\nhemodialysis on the transcriptional program and chromosomal accessibility of \\ncirculating immune cell subpopulations remain poorly defined.\\nMETHODS: We integrated single-cell RNA sequencing (scRNA-seq) and single-cell \\nassay for transposase-accessible chromatin sequencing (scATAC-seq) to \\ncharacterize the transcriptome profiles of peripheral mononuclear cells (PBMCs) \\nfrom healthy controls and maintenance hemodialysis patients. Validation of \\ndifferentially expressed genes in CD4+ T cells and monocytes were performed by \\nmagnetic bead separation and quantitative real-time PCR.\\nRESULTS: We identified 16 and 15 PBMC subgroups in scRNA-seq and scATAC-seq \\ndatasets, respectively. Hemodialysis significantly suppressed the expression \\nlevels of T cell receptor (TCR) genes in CD4+ T cell subsets (e.g., TRAV4, CD45, \\nCD3G, CD3D, CD3E) and major histocompatibility complex II (MHC-II) \\npathway-related genes in monocytes (HLA-DRB1, HLA-DQA2, HLA-DQA1, HLA-DPB1). \\nDownstream pathways of TCR signaling, including PI3K-Akt-mTOR, MAPK, TNF, and \\nNF-κB pathways, were also inhibited in CD4+ T cell subpopulations during the \\nhemodialysis procedure. Hemodialysis altered cellular communication patterns \\nbetween PBMC subgroups, particularly TGF-TGFBR, HVEM-BTLA, and IL16-CD4 \\nsignalings between CD4+ T cells and monocytes. Additionally, we found that \\nhemodialysis inhibited the expression of AP-1 family transcription factors (JUN, \\nJUND, FOS, FOSB) by interfering with the chromatin accessibility profile.\\nCONCLUSIONS: Our study provides a valuable framework for future investigations \\nof hemodialysis-related immune dysregulation and identifies potential \\ntherapeutic targets for reconstituting the circulating immune system in \\nmaintenance hemodialysis patients.\\n\\nCopyright © 2022 Wu, Dong, Yu, Wang, Dai, Zhang, Hu, Yin, Tang, Liu and Dai.\\n\\nDOI: 10.3389/fimmu.2022.878226\\nPMCID: PMC9205630\\nPMID: 35720370 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n16. Genome Med. 2022 May 24;14(1):55. doi: 10.1186/s13073-022-01058-2.\\n\\nFine-mapping studies distinguish genetic risks for childhood- and adult-onset \\nasthma in the HLA region.\\n\\nClay SM(1), Schoettler N(2), Goldstein AM(3), Carbonetto P(4), Dapas M(4), \\nAltman MC(5)(6), Rosasco MG(6), Gern JE(7), Jackson DJ(7), Im HK(8), Stephens \\nM(3), Nicolae DL(4)(3), Ober C(9).\\n\\nAuthor information:\\n(1)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \\nseleneclay@uchicago.edu.\\n(2)Section of Pulmonary and Critical Care, Department of Medicine, University of \\nChicago, Chicago, IL, 60637, USA.\\n(3)Department of Statistics, University of Chicago, Chicago, IL, 60637, USA.\\n(4)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA.\\n(5)Division of Allergy and Infectious Diseases, Department of Medicine, \\nUniversity of Washington, Seattle, WA, 98109, USA.\\n(6)Systems Immunology Program, Benaroya Research Institute, Seattle, WA, 98101, \\nUSA.\\n(7)Department of Pediatrics, University of Wisconsin, School of Medicine and \\nPublic Health, Madison, WI, 53706, USA.\\n(8)Section of Genetic Medicine, Department of Medicine, University of Chicago, \\nChicago, IL, 60637, USA.\\n(9)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \\nc-ober@genetics.uchicago.edu.\\n\\nBACKGROUND: Genome-wide association studies of asthma have revealed robust \\nassociations with variation across the human leukocyte antigen (HLA) complex \\nwith independent associations in the HLA class I and class II regions for both \\nchildhood-onset asthma (COA) and adult-onset asthma (AOA). However, the specific \\nvariants and genes contributing to risk are unknown.\\nMETHODS: We used Bayesian approaches to perform genetic fine-mapping for COA and \\nAOA (n=9432 and 21,556, respectively; n=318,167 shared controls) in White \\nBritish individuals from the UK Biobank and to perform expression quantitative \\ntrait locus (eQTL) fine-mapping in immune (lymphoblastoid cell lines, n=398; \\nperipheral blood mononuclear cells, n=132) and airway (nasal epithelial cells, \\nn=188) cells\\xa0from ethnically diverse individuals. We also examined putatively \\ncausal protein coding variation from protein crystal structures and conducted \\nreplication studies in independent multi-ethnic cohorts from the UK Biobank (COA \\nn=1686; AOA n=3666; controls n=56,063).\\nRESULTS: Genetic fine-mapping revealed both shared and distinct causal variation \\nbetween COA and AOA in the class I region but only distinct causal variation in \\nthe class II region. Both gene expression levels and amino acid variation \\ncontributed to risk. Our results from eQTL fine-mapping and amino acid \\nvisualization suggested that the HLA-DQA1*03:01 allele and variation associated \\nwith expression of the nonclassical HLA-DQA2 and HLA-DQB2 genes accounted \\nentirely for the most significant association with AOA in GWAS. Our studies also \\nsuggested a potentially prominent role for HLA-C protein coding variation in the \\nclass I region in COA. We replicated putatively causal variant associations in a \\nmulti-ethnic cohort.\\nCONCLUSIONS: We highlight roles for both gene expression and protein coding \\nvariation in asthma risk and identified putatively causal variation and genes in \\nthe HLA region. A convergence of genomic, transcriptional, and protein coding \\nevidence implicates the HLA-DQA2 and HLA-DQB2 genes and HLA-DQA1*03:01 allele in \\nAOA.\\n\\n© 2022. The Author(s).\\n\\nDOI: 10.1186/s13073-022-01058-2\\nPMCID: PMC9128203\\nPMID: 35606880 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n17. BMC Genomics. 2022 May 14;23(1):368. doi: 10.1186/s12864-022-08587-5.\\n\\nDiabetes risk loci-associated pathways are shared across metabolic tissues.\\n\\nBouland GA(1), Beulens JWJ(2)(3), Nap J(1), van der Slik AR(4), Zaldumbide A(1), \\n\\'t Hart LM(1)(2)(5), Slieker RC(6)(7).\\n\\nAuthor information:\\n(1)Department of Cell and Chemical Biology, Leiden University Medical Center, \\nEinthovenweg 20, 2333ZC, Leiden, the Netherlands.\\n(2)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \\nAmsterdam Public Health Institute, Amsterdam, the Netherlands.\\n(3)Julius Center for Health Sciences and Primary Care, University Medical Center \\nUtrecht, Utrecht, the Netherlands.\\n(4)Department of Immunohematology and Blood Transfusion, Leiden University \\nMedical Center, Leiden, The Netherlands.\\n(5)Molecular Epidemiology Section, Department of Biomedical Data Sciences, \\nLeiden University Medical Center, Leiden, the Netherlands.\\n(6)Department of Cell and Chemical Biology, Leiden University Medical Center, \\nEinthovenweg 20, 2333ZC, Leiden, the Netherlands. r.c.slieker@lumc.nl.\\n(7)Department of Epidemiology and Data Science, Amsterdam UMC, Location VUMC, \\nAmsterdam Public Health Institute, Amsterdam, the Netherlands. \\nr.c.slieker@lumc.nl.\\n\\nAIMS/HYPOTHESIS: Numerous genome-wide association studies have been performed to \\nunderstand the influence of genetic variation on type 2 diabetes etiology. Many \\nidentified risk\\xa0variants are located in non-coding and intergenic regions, which \\ncomplicates understanding of how genes and their downstream pathways are \\ninfluenced. An integrative data approach will help to understand the mechanism \\nand consequences of identified risk\\xa0variants.\\nMETHODS: In the current study we use our previously developed method CONQUER to \\noverlap 403 type 2 diabetes risk\\xa0variants with regulatory, expression and \\nprotein data to identify tissue-shared disease-relevant mechanisms.\\nRESULTS: One SNP rs474513 was found to be an expression-, protein- and \\nmetabolite QTL. Rs474513 influenced LPA mRNA and protein levels in the pancreas \\nand plasma, respectively. On the pathway level, in investigated tissues most \\nSNPs linked to metabolism. However, in eleven of the twelve tissues investigated \\nnine SNPs were linked to differential expression of the ribosome pathway. \\nFurthermore, seven SNPs were linked to altered expression of genes linked to the \\nimmune system. Among them, rs601945 was found to influence multiple HLA genes, \\nincluding HLA-DQA2, in all twelve tissues investigated.\\nCONCLUSION: Our results show that in addition to the classical metabolism \\npathways, other pathways may be important to type 2 diabetes that show a \\npotential overlap with type 1 diabetes.\\n\\n© 2022. The Author(s).\\n\\nDOI: 10.1186/s12864-022-08587-5\\nPMCID: PMC9107144\\nPMID: 35568807 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n18. J Thorac Oncol. 2022 Aug;17(8):974-990. doi: 10.1016/j.jtho.2022.04.011. Epub\\n 2022 Apr 30.\\n\\nA Large-Scale Genome-Wide Gene-Gene Interaction Study of Lung Cancer \\nSusceptibility in Europeans With a Trans-Ethnic Validation in Asians.\\n\\nZhang R(1), Shen S(2), Wei Y(1), Zhu Y(3), Li Y(4), Chen J(3), Guan J(3), Pan \\nZ(3), Wang Y(5), Zhu M(5), Xie J(6), Xiao X(7), Zhu D(7), Li Y(7), Albanes D(8), \\nLandi MT(8), Caporaso NE(8), Lam S(9), Tardon A(10), Chen C(11), Bojesen SE(12), \\nJohansson M(13), Risch A(14), Bickeböller H(15), Wichmann HE(16), Rennert G(17), \\nArnold S(18), Brennan P(13), McKay JD(13), Field JK(19), Shete SS(20), Le \\nMarchand L(21), Liu G(22), Andrew AS(23), Kiemeney LA(24), Zienolddiny-Narui \\nS(25), Behndig A(26), Johansson M(27), Cox A(28), Lazarus P(29), Schabath \\nMB(30), Aldrich MC(31), Dai J(5), Ma H(5), Zhao Y(3), Hu Z(32), Hung RJ(33), \\nAmos CI(7), Shen H(32), Chen F(34), Christiani DC(35).\\n\\nAuthor information:\\n(1)Department of Biostatistics, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China; Department of Environmental \\nHealth, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \\nInternational Cooperation Center (CICC) for Environment and Human Health, \\nNanjing Medical University, Nanjing, People\\'s Republic of China.\\n(2)Department of Biostatistics, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China; Department of Environmental \\nHealth, Harvard T. H. Chan School of Public Health, Boston, Massachusetts; China \\nInternational Cooperation Center (CICC) for Environment and Human Health, \\nNanjing Medical University, Nanjing, People\\'s Republic of China; State Key \\nLaboratory of Reproductive Medicine, Nanjing Medical University, Nanjing, \\nPeople\\'s Republic of China.\\n(3)Department of Biostatistics, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China.\\n(4)Department of Biostatistics, University of Michigan, Ann Arbor, Michigan.\\n(5)Department of Epidemiology, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China; Jiangsu Key Lab of Cancer \\nBiomarkers, Prevention and Treatment, Cancer Center, Collaborative Innovation \\nCenter for Cancer Personalized Medicine, Nanjing Medical University, Nanjing, \\nPeople\\'s Republic of China.\\n(6)Department of Epidemiology, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China.\\n(7)The Institute for Clinical and Translational Research, Department of \\nMedicine, Baylor College of Medicine, Houston, Texas.\\n(8)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \\nNational Institutes of Health, Bethesda, Maryland.\\n(9)Department of Medicine, British Columbia Cancer Agency, University of British \\nColumbia, Vancouver, Canada.\\n(10)Faculty of Medicine, University of Oviedo and CIBERESP, Oviedo, Spain.\\n(11)Department of Epidemiology, University of Washington School of Public \\nHealth, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, \\nSeattle, Washington.\\n(12)Department of Clinical Biochemistry, Herlev and Gentofte Hospital, \\nCopenhagen University Hospital, Copenhagen, Denmark.\\n(13)Section of Genetics, International Agency for Research on Cancer, World \\nHealth Organization, Lyon, France.\\n(14)Department of Biosciences and Cancer Cluster Salzburg, University of \\nSalzburg, Salzburg, Austria.\\n(15)Department of Genetic Epidemiology, University Medical Center, Georg August \\nUniversity Göttingen, Göttingen, Germany.\\n(16)Institute of Medical Informatics, Biometry and Epidemiology, Ludwig \\nMaximilians University, Munich, Germany.\\n(17)Clalit National Cancer Control Center, Carmel Medical Center and Technion \\nFaculty of Medicine, Carmel, Haifa, Israel.\\n(18)Markey Cancer Center, University of Kentucky, Lexington, Kentucky.\\n(19)Department of Molecular and Clinical Cancer Medicine, Institute of \\nTranslational Medicine, University of Liverpool, Liverpool, United Kingdom.\\n(20)Department of Biostatistics, The University of Texas MD Anderson Cancer \\nCenter, Houston, Texas.\\n(21)Epidemiology Program, University of Hawaii Cancer Center, Honolulu, Hawaii.\\n(22)Princess Margaret Cancer Centre, Dalla Lana School of Public Health, \\nUniversity of Toronto, Toronto, Canada.\\n(23)Department of Epidemiology, Department of Community and Family Medicine, \\nDartmouth Geisel School of Medicine, Hanover, New Hampshire.\\n(24)Department for Health Evidence, Department of Urology, Radboud University \\nMedical Center, Nijmegen, The Netherlands.\\n(25)National Institute of Occupational Health, Oslo, Norway.\\n(26)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \\nSweden.\\n(27)Department of Radiation Sciences, Umeå University, Umeå, Sweden.\\n(28)Department of Oncology and Metabolism, The Medical School, University of \\nSheffield, Sheffield, United Kingdom.\\n(29)Department of Pharmaceutical Sciences, College of Pharmacy, Washington State \\nUniversity, Spokane, Washington.\\n(30)Department of Cancer Epidemiology, H. Lee Moffitt Cancer Center and Research \\nInstitute, Tampa, Florida.\\n(31)Department of Thoracic Surgery and Division of Epidemiology, Vanderbilt \\nUniversity Medical Center, Nashville, Tennessee.\\n(32)China International Cooperation Center (CICC) for Environment and Human \\nHealth, Nanjing Medical University, Nanjing, People\\'s Republic of China; \\nDepartment of Epidemiology, School of Public Health, Nanjing Medical University, \\nNanjing, People\\'s Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \\nPrevention and Treatment, Cancer Center, Collaborative Innovation Center for \\nCancer Personalized Medicine, Nanjing Medical University, Nanjing, People\\'s \\nRepublic of China.\\n(33)Lunenfeld-Tanenbaum Research Institute, Sinai Health System, Dalla Lana \\nSchool of Public Health, University of Toronto, Toronto, Ontario, Canada.\\n(34)Department of Biostatistics, School of Public Health, Nanjing Medical \\nUniversity, Nanjing, People\\'s Republic of China; China International Cooperation \\nCenter (CICC) for Environment and Human Health, Nanjing Medical University, \\nNanjing, People\\'s Republic of China; Jiangsu Key Lab of Cancer Biomarkers, \\nPrevention and Treatment, Cancer Center, Collaborative Innovation Center for \\nCancer Personalized Medicine, Nanjing Medical University, Nanjing, People\\'s \\nRepublic of China. Electronic address: fengchen@njmu.edu.cn.\\n(35)Department of Environmental Health, Harvard T. H. Chan School of Public \\nHealth, Boston, Massachusetts; Pulmonary and Critical Care Division, Department \\nof Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, \\nMassachusetts.\\n\\nComment in\\n    J Thorac Oncol. 2022 Aug;17(8):946-948.\\n    J Thorac Oncol. 2023 Mar;18(3):e23-e24.\\n    J Thorac Oncol. 2023 Mar;18(3):e24-e26.\\n\\nINTRODUCTION: Although genome-wide association studies have been conducted to \\ninvestigate genetic variation of lung tumorigenesis, little is known about \\ngene-gene (G\\xa0× G) interactions that may influence the risk of non-small cell \\nlung cancer (NSCLC).\\nMETHODS: Leveraging a total of 445,221 European-descent participants from the \\nInternational Lung Cancer Consortium OncoArray project, Transdisciplinary \\nResearch in Cancer of the Lung and UK Biobank, we performed a large-scale \\ngenome-wide G\\xa0× G interaction study on European NSCLC risk by a series of \\nanalyses. First, we used BiForce to evaluate and rank more than 58 billion G\\xa0× G \\ninteractions from 340,958 single-nucleotide polymorphisms (SNPs). Then, the top \\ninteractions were further tested by demographically adjusted logistic regression \\nmodels. Finally, we used the selected interactions to build lung cancer \\nscreening models of NSCLC, separately, for never and ever smokers.\\nRESULTS: With the Bonferroni correction, we identified eight statistically \\nsignificant pairs of SNPs, which predominantly appeared in the 6p21.32 and \\n5p15.33 regions (e.g., rs521828C6orf10 and rs204999PRRT1, ORinteraction\\xa0= 1.17, \\np\\xa0= 6.57\\xa0× 10-13; rs3135369BTNL2 and rs2858859HLA-DQA1, ORinteraction\\xa0= 1.17, \\np\\xa0= 2.43\\xa0× 10-13; rs2858859HLA-DQA1 and rs9275572HLA-DQA2, ORinteraction\\xa0= 1.15, \\np\\xa0= 2.84\\xa0× 10-13; rs2853668TERT and rs62329694CLPTM1L, ORinteraction\\xa0= 0.73, p\\xa0= \\n2.70\\xa0× 10-13). Notably, even with much genetic heterogeneity across ethnicities, \\nthree pairs of SNPs in the 6p21.32 region identified from the European-ancestry \\npopulation remained significant among an Asian population from the Nanjing \\nMedical University Global Screening Array project (rs521828C6orf10 and \\nrs204999PRRT1, ORinteraction\\xa0= 1.13, p\\xa0= 0.008; rs3135369BTNL2 and \\nrs2858859HLA-DQA1, ORinteraction\\xa0= 1.11, p\\xa0= 5.23\\xa0× 10-4; rs3135369BTNL2 and \\nrs9271300HLA-DQA1, ORinteraction\\xa0= 0.89, p\\xa0= 0.006). The interaction-empowered \\npolygenetic risk score that integrated classical polygenetic risk score and G\\xa0× \\nG information score was remarkable in lung cancer risk stratification.\\nCONCLUSIONS: Important G\\xa0× G interactions were identified and enriched in the \\n5p15.33 and 6p21.32 regions, which may enhance lung cancer screening models.\\n\\nCopyright © 2022 International Association for the Study of Lung Cancer. \\nPublished by Elsevier Inc. All rights reserved.\\n\\nDOI: 10.1016/j.jtho.2022.04.011\\nPMCID: PMC9512697\\nPMID: 35500836 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Disclosure: The authors declare no conflict of \\ninterest.\\n\\n\\n19. Transl Psychiatry. 2022 Apr 18;12(1):163. doi: 10.1038/s41398-022-01913-1.\\n\\nAllele-specific analysis reveals exon- and cell-type-specific regulatory effects \\nof Alzheimer\\'s disease-associated genetic variants.\\n\\nHe L(1), Loika Y(2), Kulminski AM(3).\\n\\nAuthor information:\\n(1)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA. lh235@duke.edu.\\n(2)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA.\\n(3)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA. alexander.kulminski@duke.edu.\\n\\nElucidating regulatory effects of Alzheimer\\'s disease (AD)-associated genetic \\nvariants is critical for unraveling their causal pathways and understanding the \\npathology. However, their cell-type-specific regulatory mechanisms in the brain \\nremain largely unclear. Here, we conducted an analysis of allele-specific \\nexpression quantitative trait loci (aseQTLs) for 33 AD-associated variants in \\nfour brain regions and seven cell types using ~3000 bulk RNA-seq samples and \\n>0.25 million single nuclei. We first develop a flexible hierarchical Poisson \\nmixed model (HPMM) and demonstrate its superior statistical power to a \\nbeta-binomial model achieved by unifying samples in both allelic and \\ngenotype-level expression data. Using the HPMM, we identified 24 (~73%) aseQTLs \\nin at least one brain region, including three new eQTLs associated with CA12, \\nCHRNE, and CASS4. Notably, the APOE ε4 variant reduces APOE expression across \\nall regions, even in AD-unaffected controls. Our results reveal region-dependent \\nand exon-specific effects of multiple aseQTLs, such as rs2093760 with CR1, \\nrs7982 with CLU, and rs3865444 with CD33. In an attempt to pinpoint the cell \\ntypes responsible for the observed tissue-level aseQTLs using the snRNA-seq \\ndata, we detected many aseQTLs in microglia or monocytes associated with \\nimmune-related genes, including HLA-DQB1, HLA-DQA2, CD33, FCER1G, MS4A6A, SPI1, \\nand BIN1, highlighting the regulatory role of AD-associated variants in the \\nimmune response. These findings provide further insights into potential causal \\npathways and cell types mediating the effects of the AD-associated variants.\\n\\n© 2022. The Author(s).\\n\\nDOI: 10.1038/s41398-022-01913-1\\nPMCID: PMC9016079\\nPMID: 35436980 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n20. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\\neCollection  2022.\\n\\nThe Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren\\'s \\nSyndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\\n\\nHong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \\nZhao T(2), Fu J(1).\\n\\nAuthor information:\\n(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n\\nOBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \\nof multiple sclerosis (MS) and Sjögren\\'s syndrome (SS).\\nMETHODS: MS- and SS-related susceptibility genes and differentially expressed \\ngenes (DEGs) were identified by bioinformatics analysis based on genome-wide \\nassociation studies (GWAS) and transcriptome data from GWAS catalog and Gene \\nExpression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \\nanalysis, and protein-protein interaction analysis for susceptibility genes and \\nDEGs were performed. The drugs targeting common pathways/genes were obtained \\nthrough Comparative Toxicogenomics Database (CTD), DrugBank database, and \\nDrug-Gene Interaction (DGI) Database. The target genes of \\napproved/investigational drugs for MS and SS were obtained through DrugBank and \\ncompared with the common susceptibility genes.\\nRESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \\n(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \\nHLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \\nthan two genes, and 28 common susceptibility pathways, with 15 drugs targeting \\nthree or more than three pathways. Based on transcriptome data, we found 3 hub \\ncommon DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \\n435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \\npathways and common risk pathways at the same time. There were 133 overlaps \\nincluding JAK-STAT inhibitors between agents from GWAS and transcriptome data. \\nBesides, we found that IL2RA and HLA-DRB1, identified as hub common \\nsusceptibility genes, were the targets of daclizumab and glatiramer that were \\nused for MS, indicating that daclizumab and glatiramer may be therapeutic for \\nSS.\\nCONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \\nsignaling pathway played a vital role, which may be the genetic and molecular \\nbases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \\ntherapies for MS and SS, especially for their comorbidity.\\n\\nCopyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\\n\\nDOI: 10.3389/fimmu.2022.857014\\nPMCID: PMC8959321\\nPMID: 35356004 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n21. Front Immunol. 2022 Feb 16;13:814239. doi: 10.3389/fimmu.2022.814239. \\neCollection 2022.\\n\\nGlobal Characterization of Peripheral B Cells in Parkinson\\'s Disease by \\nSingle-Cell RNA and BCR Sequencing.\\n\\nWang P(1), Luo M(1), Zhou W(1), Jin X(1), Xu Z(1), Yan S(2), Li Y(1), Xu C(1), \\nCheng R(1), Huang Y(1), Lin X(1), Yao L(2), Nie H(1), Jiang Q(1)(3).\\n\\nAuthor information:\\n(1)School of Life Science and Technology, Harbin Institute of Technology, \\nHarbin, China.\\n(2)Department of Neurology, First Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n(3)Key Laboratory of Biological Big Data (Harbin Institute of Technology), \\nMinistry of Education, Harbin, China.\\n\\nImmune system plays important roles in the pathogenesis of Parkinson\\'s disease \\n(PD). However, the role of B cells in this complex disease are still not fully \\nunderstood. B cells produce antibodies but can also regulate immune responses. \\nIn order to decode the relative contribution of peripheral B cell subtypes to \\nthe etiology of PD, we performed single cell RNA and BCR sequencing for 10,466 B \\ncells from 8 PD patients and 6 age-matched healthy controls. We observed \\nsignificant increased memory B cells and significant decreased naïve B cells in \\nPD patients compared to healthy controls. Notably, we also discovered increased \\nIgG and IgA isotypes and more frequent class switch recombination events in PD \\npatients. Moreover, we identified preferential V and J gene segments of B cell \\nreceptors in PD patients as the evidence of convergent selection in PD. Finally, \\nwe found a marked clonal expanded memory B cell population in PD patients, \\nup-regulating both MHC II genes (HLA-DRB5, HLA-DQA2 and HLA-DPB1) and \\ntranscription factor activator protein 1 (AP-1), suggesting that the antigen \\npresentation capacity of B cells was enhanced and B cells were activated in PD \\npatients. Overall, this study conducted a comprehensive analysis of peripheral B \\ncell characteristics of PD patients, which provided novel insights into the \\nhumoral immune response in the pathogenesis of PD.\\n\\nCopyright © 2022 Wang, Luo, Zhou, Jin, Xu, Yan, Li, Xu, Cheng, Huang, Lin, Yao, \\nNie and Jiang.\\n\\nDOI: 10.3389/fimmu.2022.814239\\nPMCID: PMC8888848\\nPMID: 35250991 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n22. Rheumatology (Oxford). 2022 Aug 3;61(8):3461-3470. doi: \\n10.1093/rheumatology/keab912.\\n\\nIdentification and functional characterization of a novel susceptibility locus \\nfor small vessel vasculitis with MPO-ANCA.\\n\\nDahlqvist J(1)(2)(3), Ekman D(4), Sennblad B(5), Kozyrev SV(2), Nordin J(2), \\nKarlsson Å(2), Meadows JRS(2), Hellbacher E(1), Rantapää-Dahlqvist S(6), Berglin \\nE(6), Stegmayr B(6), Baslund B(7), Palm Ø(8), Haukeland H(9), Gunnarsson \\nI(10)(11), Bruchfeld A(12)(13), Segelmark M(14), Ohlsson S(14), Mohammad \\nAJ(15)(16), Svärd A(17), Pullerits R(18)(19), Herlitz H(20), Söderbergh A(21), \\nRosengren Pielberg G(2), Hultin Rosenberg L(2), Bianchi M(2), Murén E(2), Omdal \\nR(22)(23), Jonsson R(24), Eloranta ML(1), Rönnblom L(1), Söderkvist P(25), \\nKnight A(1), Eriksson P(26), Lindblad-Toh K(2)(3).\\n\\nAuthor information:\\n(1)Department of Medical Sciences.\\n(2)Science for Life Laboratory, Department of Medical Biochemistry and \\nMicrobiology, Uppsala University, Uppsala, Sweden.\\n(3)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\\n(4)Department of Biochemistry and Biophysics, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Stockholm University, \\nStockholm.\\n(5)Department of Cell and Molecular Biology, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala.\\n(6)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \\nSweden.\\n(7)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \\nDiseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\\n(8)Department of Rheumatology, Oslo University Hospital.\\n(9)Department of Rheumatology, Martina Hansens Hospital, Oslo, Norway.\\n(10)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \\nStockholm.\\n(11)Unit of Rheumatology, Karolinska University Hospital, Stockholm.\\n(12)Department of Health, Medicine and Caring Sciences, Linköping University, \\nLinköping.\\n(13)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \\nKarolinska Institutet, Stockholm.\\n(14)Department of Clinical Sciences, Division of Nephrology, Lund University and \\nSkåne University Hospital.\\n(15)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \\nUniversity Hospital, Lund University, Lund, Sweden.\\n(16)Department of Medicine, University of Cambridge, Cambridge, UK.\\n(17)Center for Clinical Research Dalarna, Uppsala University, Uppsala.\\n(18)Department of Rheumatology and Inflammation Research, Institution of \\nMedicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg.\\n(19)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \\nUniversity Hospital.\\n(20)Department of Molecular and Clinical Medicine/Nephrology, Institute of \\nMedicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg.\\n(21)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\\n(22)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \\nUniversity Hospital, Stavanger.\\n(23)Department of Clinical Science.\\n(24)Broegelmann Research Laboratory, Department of Clinical Science, University \\nof Bergen, Bergen, Norway.\\n(25)Department of Biomedical and Clinical Sciences, Division of Cell Biology.\\n(26)Department of Biomedical and Clinical Sciences, Division of Inflammation and \\nInfection, Linköping University, Linköping, Sweden.\\n\\nOBJECTIVE: To identify and characterize genetic loci associated with the risk of \\ndeveloping ANCA-associated vasculitides (AAV).\\nMETHODS: Genetic association analyses were performed after Illumina sequencing \\nof 1853 genes and subsequent replication with genotyping of selected single \\nnucleotide polymorphisms in a total cohort of 1110 Scandinavian cases with \\ngranulomatosis with polyangiitis or microscopic polyangiitis, and 1589 controls. \\nA novel AAV-associated single nucleotide polymorphism was analysed for \\nallele-specific effects on gene expression using luciferase reporter assay.\\nRESULTS: PR3-ANCA+ AAV was significantly associated with two independent loci in \\nthe HLA-DPB1/HLA-DPA1 region [rs1042335, P\\u2009=\\u20096.3\\u2009×\\u200910-61, odds ratio (OR) 0.10; \\nrs9277341, P\\u2009=\\u20091.5\\u2009×\\u200910-44, OR 0.22] and with rs28929474 in the SERPINA1 gene \\n(P\\u2009=\\u20092.7\\u2009×\\u200910-10, OR 2.9). MPO-ANCA+ AAV was significantly associated with the \\nHLA-DQB1/HLA-DQA2 locus (rs9274619, P\\u2009=\\u20095.4\\u2009×\\u200910-25, OR 3.7) and with a rare \\nvariant in the BACH2 gene (rs78275221, P\\u2009=\\u20097.9\\u2009×\\u200910-7, OR 3.0), the latter a \\nnovel susceptibility locus for MPO-ANCA+ granulomatosis with \\npolyangiitis/microscopic polyangiitis. The rs78275221-A risk allele reduced \\nluciferase gene expression in endothelial cells, specifically, as compared with \\nthe non-risk allele.\\nCONCLUSION: We identified a novel susceptibility locus for MPO-ANCA+ AAV and \\npropose that the associated variant is of mechanistic importance, exerting a \\nregulatory function on gene expression in specific cell types.\\n\\n© The Author(s) 2021. Published by Oxford University Press on behalf of the \\nBritish Society for Rheumatology.\\n\\nDOI: 10.1093/rheumatology/keab912\\nPMCID: PMC9348767\\nPMID: 34888651 [Indexed for MEDLINE]\\n\\n\\n23. J Radiat Res. 2022 Jan 20;63(1):8-18. doi: 10.1093/jrr/rrab091.\\n\\nTranscriptomic profiling reveals gene expression in human peripheral blood after \\nexposure to low-dose ionizing radiation.\\n\\nFang F, Yu X, Wang X, Zhu X, Liu L, Rong L, Niu D, Li J.\\n\\nAlthough the health effects of exposure to low-dose ionizing radiation have been \\nthe focus of many studies, the affected biological functions and underlying \\nregulatory mechanisms are not well-understood. In particular, the influence of \\nradiation exposure at doses of less than 200\\xa0mGy on the regulation of genes and \\npathways remains unclear. To investigate the molecular alterations induced by \\nvarying doses of low-dose radiation (LDR), transcriptomic analysis was conducted \\nbased on ribonucleic acid (RNA) sequencing following exposure to 50 and 150\\xa0mGy \\ndoses. Human peripheral blood was collected, and the samples were divided into \\nthree groups, including two treatments and one control (no radiation). A total \\nof 876 (318 upregulated and 558 downregulated) and 486 (202 upregulated and 284 \\ndownregulated) differentially expressed genes (DEGs) were identified after \\nexposure to 50\\xa0mGy and 150\\xa0mGy, respectively. Most upregulated genes in both the \\n50\\xa0mGy and 150\\xa0mGy groups were associated with \\'antigen processing and \\npresentation,\\' which appeared to be the major targets affected by LDR exposure. \\nSeveral interacting genes, including HLA-DQA1, HLA-DQA2, HLA-DQB2, HLA-DRB1, and \\nHLA-DRB5 were mapped to \\'antigen processing and presentation,\\' \\'immune \\nsystem-related diseases\\' and the \\'cytokine-mediated signaling pathway,\\' \\nsuggesting that these genes might drive the downstream transmission of these \\nsignal transduction pathways. Our results suggest that exposure to LDR may \\nelicit changes in key genes and associated pathways, probably helping further \\nexplore the biological processes and molecular mechanism responsible for \\nlow-dose occupational or environmental exposures in humans.\\n\\n© The Author(s) 2021. Published by Oxford University Press on behalf of The \\nJapanese Radiation Research Society and Japanese Society for Radiation Oncology.\\n\\nDOI: 10.1093/jrr/rrab091\\nPMCID: PMC8776696\\nPMID: 34788452 [Indexed for MEDLINE]\\n\\n\\n24. Comput Math Methods Med. 2021 Sep 21;2021:6015473. doi: 10.1155/2021/6015473.\\n eCollection 2021.\\n\\nComprehensive Analysis of RNA Expression Profile Identifies Hub miRNA-circRNA \\nInteraction Networks in the Hypoxic Ischemic Encephalopathy.\\n\\nWei L(#)(1), Li X(#)(1), Wang L(2), Song Y(3), Dong H(1).\\n\\nAuthor information:\\n(1)Department of Neonatology, Jinan Maternal and Child Health Care Hospital, \\nShandong, China.\\n(2)Department of Pediatrics, Jinan Maternal and Child Health Care Hospital, \\nShandong, China.\\n(3)Department of Pediatrics, Chinese People\\'s Liberation Army of 960 Hospital, \\nShandong, China.\\n(#)Contributed equally\\n\\nHypoxic ischemic encephalopathy (HIE) is classified as a sort of serious nervous \\nsystem syndrome that occurs in the early life period. Noncoding RNAs had been \\nconfirmed to have crucial roles in human diseases. So far, there were few \\nsystematical and comprehensive studies towards the expression profile of RNAs in \\nthe brain after hypoxia ischemia. In this study, 31 differentially expressed \\nmicroRNAs (miRNAs) with upregulation were identified. In addition, 5512 \\ndifferentially expressed mRNAs, long noncoding RNAs (lncRNAs), and circular RNAs \\n(circRNAs) were identified in HIE groups. Bioinformatics analysis showed these \\ncircRNAs and mRNAs were significantly enriched in regulation of leukocyte \\nactivation, response to virus, and neutrophil degranulation. Pathway and its \\nrelated gene network analysis indicated that HLA\\u2009-\\u2009DPA1, HLA\\u2009-\\u2009DQA2, HLA\\u2009-\\u2009DQB1, \\nand HLA\\u2009-\\u2009DRB4 have a more crucial role in HIE. Finally, miRNA-circRNA-mRNA \\ninteraction network analysis was also performed to identify hub miRNAs and \\ncircRNAs. We found that miR-592 potentially targeting 5 circRNAs, thus affecting \\n15 mRNA expressions in HIR. hsa_circ_0068397 and hsa_circ_0045698 were \\nidentified as hub circRNAs in HIE. Collectively, using RNA-seq, bioinformatics \\nanalysis, and circRNA/miRNA interaction prediction, we systematically \\ninvestigated the differentially expressed RNAs in HIE, which could give a new \\nhint of understanding the pathogenesis of HIE.\\n\\nCopyright © 2021 Lin Wei et al.\\n\\nDOI: 10.1155/2021/6015473\\nPMCID: PMC8481051\\nPMID: 34603484 [Indexed for MEDLINE]\\n\\nConflict of interest statement: No conflict of interest.\\n\\n\\n25. Public Health Genomics. 2021;24(5-6):267-279. doi: 10.1159/000517308. Epub\\n2021  Aug 2.\\n\\nMeta-Analyses of Multiple Gene Expression Profiles to Screen Hub Genes Related \\nto Osteoarthritis.\\n\\nZhu X(1), Guo W(1).\\n\\nAuthor information:\\n(1)Department of Orthopedics, Taizhou People\\'s Hospital, Taizhou, China.\\n\\nBACKGROUND: This study aimed to screen and validate the crucial genes involved \\nin osteoarthritis (OA) and explore its potential molecular mechanisms.\\nMETHODS: Four expression profile datasets related to OA were downloaded from the \\nGene Expression Omnibus (GEO). The differentially expressed genes (DEGs) from 4 \\nmicroarray patterns were identified by the meta-analysis method. The weighted \\ngene co-expression network analysis (WGCNA) method was used to investigate \\nstable modules most related to OA. In addition, a protein-protein interaction \\n(PPI) network was built to explore hub genes in OA. Moreover, OA-related genes \\nand pathways were retrieved from Comparative Toxicogenomics Database (CTD).\\nRESULTS: A total of 1,136 DEGs were identified from 4 datasets. Based on these \\nDEGs, WGCNA further explored 370 genes included in the 3 OA-related stable \\nmodules. A total of 10 hub genes were identified in the PPI network, including \\nAKT1, CDC42, HLA-DQA2, TUBB, TWISTNB, GSK3B, FZD2, KLC1, GUSB, and RHOG. \\nBesides, 5 pathways including \"Lysosome,\" \"Pathways in cancer,\" \"Wnt signaling \\npathway,\" \"ECM-receptor interaction\" and \"Focal adhesion\" in CTD and enrichment \\nanalysis and 5 OA-related hub genes (including GSK3B, CDC42, AKT1, FZD2, and \\nGUSB) were identified.\\nCONCLUSION: In this study, the meta-analysis was used to screen the central \\ngenes associated with OA in a variety of gene expression profiles. Three \\nOA-related modules (green, turquoise, and yellow) containing 370 genes were \\nidentified through WGCNA. It was discovered through the gene-pathway network \\nthat GSK3B, CDC42, AKT1, FZD2, and GUSB may be key genes related to the progress \\nof OA and may become promising therapeutic targets.\\n\\n© 2021 The Author(s). Published by S. Karger AG, Basel.\\n\\nDOI: 10.1159/000517308\\nPMID: 34340232 [Indexed for MEDLINE]\\n\\n\\n26. Restor Neurol Neurosci. 2021;39(4):247-266. doi: 10.3233/RNN-211161.\\n\\nTranscriptome analysis of human dorsal striatum implicates attenuated canonical \\nWNT signaling in neuroinflammation and in age-related impairment of striatal \\nneurogenesis and synaptic plasticity.\\n\\nWu C(1), Bendriem RM(2), Freed WJ(3), Lee CT(4).\\n\\nAuthor information:\\n(1)Department of Molecular and Cellular Pharmacology, Miller School of Medicine, \\nUniversity of Miami, Miami, FL, USA.\\n(2)Brain and Mind Research Institute, Weill Cornell Graduate School of Medical \\nSciences, New York, NY, USA.\\n(3)Department of Biology, Lebanon Valley College, Annville, PA, USA.\\n(4)Department of Neurology, Miller School of Medicine, University of Miami, \\nMiami, FL, USA.\\n\\nBACKGROUND: Motor and cognitive decline as part of the normal aging process is \\nlinked to alterations in synaptic plasticity and reduction of adult neurogenesis \\nin the dorsal striatum. Neuroinflammation, particularly in the form of \\nmicroglial activation, is suggested to contribute to these age-associated \\nchanges.\\nOBJECTIVE AND METHODS: To explore the molecular basis of alterations in striatal \\nfunction during aging we analyzed RNA-Seq data for 117 postmortem human dorsal \\ncaudate samples and 97 putamen samples acquired through GTEx.\\nRESULTS: Increased expression of neuroinflammatory transcripts including TREM2, \\nMHC II molecules HLA-DMB, HLA-DQA2, HLA-DPA1, HLA-DPB1, HLA-DMA and HLA-DRA, \\ncomplement genes C1QA, C1QB, CIQC and C3AR1, and MHCI molecules HLA-B and HLA-F \\nwas identified. We also identified down-regulation of transcripts involved in \\nneurogenesis, synaptogenesis, and synaptic pruning, including DCX, CX3CL1, and \\nCD200, and the canonical WNTs WNT7A, WNT7B, and WNT8A. The canonical WNT \\nsignaling pathway has previously been shown to mediate adult neurogenesis and \\nsynapse formation and growth. Recent findings also highlight the link between \\nWNT/β-catenin signaling and inflammation pathways.\\nCONCLUSIONS: These findings suggest that age-dependent attenuation of canonical \\nWNT signaling plays a pivotal role in regulating striatal plasticity during \\naging. Dysregulation of WNT/β-catenin signaling via astrocyte-microglial \\ninteractions is suggested to be a novel mechanism that drives the decline of \\nstriatal neurogenesis and altered synaptic connectivity and plasticity, leading \\nto a subsequent decrease in motor and cognitive performance with age. These \\nfindings may aid in the development of therapies targeting WNT/β-catenin \\nsignaling to combat cognitive and motor impairments associated with aging.\\n\\nDOI: 10.3233/RNN-211161\\nPMID: 34275915 [Indexed for MEDLINE]\\n\\n\\n27. J Cell Mol Med. 2021 Jul;25(14):7001-7012. doi: 10.1111/jcmm.16714. Epub 2021\\n Jun 16.\\n\\nSingle-cell RNA sequencing identify SDCBP in ACE2-positive bronchial epithelial \\ncells negatively correlates with COVID-19 severity.\\n\\nMa D(1)(2), Liu S(1), Hu L(1)(3), He Q(1), Shi W(1)(2), Yan D(1)(3), Cao \\nY(1)(2)(3), Zhang G(1)(2)(3), Wang Z(1)(2)(3), Wu J(1), Jiang C(1)(2)(3).\\n\\nAuthor information:\\n(1)Jiangsu Key Laboratory of Molecular Medicine, National Institute of \\nHealthcare Data Science at Nanjing University, Medical School of Nanjing \\nUniversity, Nanjing, China.\\n(2)Department of Hepatobiliary Surgery, The Affiliated Drum Tower Hospital of \\nNanjing University Medical School, Nanjing, China.\\n(3)Department of Hepatobiliary Surgery, Drum Tower Clinical College of Nanjing \\nMedical University, Nanjing, China.\\n\\nErratum in\\n    J Cell Mol Med. 2023 Oct;27(19):3012-3015.\\n\\nThe coronavirus disease 2019 (COVID-19), caused by the novel severe acute \\nrespiratory syndrome coronavirus 2 (SARS-CoV-2), has resulted in many deaths \\nthroughout the world. It is vital to identify the novel prognostic biomarkers \\nand therapeutic targets to assist with the subsequent diagnosis and treatment \\nplan to mitigate the expansion of COVID-19. Since angiotensin-converting enzyme \\n2 (ACE2)-positive cells are hosts for COVID-19, we focussed on this cell type to \\nexplore the underlying mechanisms of COVID-19. In this study, we identified that \\nACE2-positive cells from the bronchoalveolar lavage fluid (BALF) of patients \\nwith COVID-19 belong to bronchial epithelial cells. Comparing with patients of \\nCOVID-19 showing severe symptoms, the antigen processing and presentation \\npathway was increased and 12 typical genes, HLA-DRB5, HLA-DRB1, CD74, HLA-DRA, \\nHLA-DPA1, HLA-DQA1, HSP90AA1, HSP90AB1, HLA-DPB1, HLA-DQB1, HLA-DQA2, and \\nHLA-DMA, particularly HLA-DPB1, were obviously up-regulated in ACE2-positive \\nbronchial epithelial cells of patients with mild disease. We further discovered \\nSDCBP was positively correlated with above 12 genes particularly with HLA-DPB1 \\nin ACE2-positive bronchial epithelial cells of COVID-19 patients. Moreover, \\nSDCBP may increase the immune infiltration of B cells, CD8+ T cells, CD4+ T \\ncells, macrophages, neutrophils and dendritic cells in different lung carcinoma. \\nMoreover, we found the expression of SDCBP was positively correlated with the \\nexpression of antigen processing and presentation genes in post-mortem lung \\nbiopsies tissues, which is consistent with previous discoveries. These results \\nsuggest that SDCBP has good potential to be further developed as a novel \\ndiagnostic and therapeutic target in the treatment of COVID-19.\\n\\n© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \\nFoundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/jcmm.16714\\nPMCID: PMC8278084\\nPMID: 34137173 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that there are no conflicts \\nof interest.\\n\\n\\n28. Neurol Genet. 2021 Jun 2;7(3):e592. doi: 10.1212/NXG.0000000000000592. \\neCollection 2021 Jun.\\n\\nAssociation of HLA-DQA2 and HLA-B With Moyamoya Disease in the Chinese Han \\nPopulation.\\n\\nWan J(1), Ling W(1), Zhengshan Z(1), Xianbo Z(1), Lian D(1), Kai W(1).\\n\\nAuthor information:\\n(1)Department of Neurology (J.W.), the First Affiliated Hospital of Anhui \\nMedical University, the School of Mental Health and Psychological Sciences, \\nAnhui Medical University, Hefei, Anhui Province, Department of Neurology (J.W.), \\nDrum Tower Hospital, Medical School and the State Key Laboratory of \\nPharmaceutical Biotechnology, Nanjing University; the School of Mental Health \\nand Psychological Sciences (W.L.), Anhui Medical University, Anhui Province, \\nInstitute of Artificial Intelligence (W.L.), Hefei Comprehensive National \\nScience Center. Anhui Province Key Laboratory of Cognition and Neuropsychiatric \\nDisorders (W.L.), Hefei; Collaborative Innovation Center of Neuropsychiatric \\nDisorders and Mental Health (W.L), Anhui Province; Department of Neurosurgery \\n(Z.Z.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \\nHospital of PLA), Beijing; Department of Dermatology (Z.X.), the First \\nAffiliated Hospital, Anhui Medical University, Hefei, Anhui Province; Key \\nLaboratory of Dermatology (Z.X.), Anhui Medical University, Ministry of \\nEducation, Hefei, Anhui Province; State Key Lab of Dermatology Incubation Center \\n(Z.X.), Anhui Medical University, Hefei, China; Department of Neurosurgery \\n(D.L.), the Fifth Medical Centre, Chinese PLA General Hospital (Former 307th \\nHospital of PLA), Beijing; the School of Mental Health and Psychological \\nSciences (W.K.), Anhui Medical University, Anhui Province; Institute of \\nArtificial Intelligence (W.K.), Hefei Comprehensive National Science Center; \\nAnhui Province Key Laboratory of Cognition and Neuropsychiatric Disorders \\n(W.K.), Hefei, Collaborative Innovation Center of Neuropsychiatric Disorders and \\nMental Health (W.K.), Anhui Province, China.\\n\\nOBJECTIVE: An HLA imputation was conducted to explore the relationship between \\nHLA and patients with moyamoya disease (MMD) in the Chinese Han population.\\nMETHODS: In this study, we performed an association analysis of the major \\nhistocompatibility complex region in 2,786 individuals of Chinese Han ancestry \\n(2,031 controls and 755 patients with MMD), through a widely used HLA imputation \\nmethod.\\nRESULTS: We identified that the variant rs3129731 (odds ratio [OR] = 1.79, p = \\n3.69 × 10-16) located between the MTCO3P1 and HLA-DQA2 is a major genetic risk \\nfactor for MMD. In addition to this variant, found in the conditional \\nassociation analysis, we also detected another independent signal, rs1071817 (OR \\n= 0.62, p = 1.20 × 10-11), in HLA-B.\\nCONCLUSIONS: Our research suggests that the genetic polymorphism of HLA-DQA2 and \\nHLA-B could be a genetic predisposing factor for MMD in Chinese Han. This may \\nprovide some evidence for further HLA-related studies of patients with MMD of \\nChinese Han ethnicity and indicates that MMD is an immune-related disease.\\n\\nCopyright © 2021 The Author(s). Published by Wolters Kluwer Health, Inc. on \\nbehalf of the American Academy of Neurology.\\n\\nDOI: 10.1212/NXG.0000000000000592\\nPMCID: PMC8176556\\nPMID: 34095496\\n\\n\\n29. PLoS One. 2021 Apr 22;16(4):e0248429. doi: 10.1371/journal.pone.0248429. \\neCollection 2021.\\n\\nHypomethylation mediates genetic association with the major histocompatibility \\ncomplex genes in Sjögren\\'s syndrome.\\n\\nChi C(1)(2), Taylor KE(3), Quach H(2), Quach D(2), Criswell LA(3), Barcellos \\nLF(1)(2).\\n\\nAuthor information:\\n(1)Center for Computational Biology, College of Engineering, University of \\nCalifornia, Berkeley, Berkeley, California, United States of America.\\n(2)Genetic Epidemiology and Genomics Laboratory, School of Public Health, \\nUniversity of California, Berkeley, Berkeley, California, United States of \\nAmerica.\\n(3)Department of Medicine, Russell/Engleman Rheumatology Research Center, \\nUniversity of California, San Francisco, San Francisco, California, United \\nStates of America.\\n\\nErratum in\\n    PLoS One. 2021 Jul 27;16(7):e0255549.\\n\\nDifferential methylation of immune genes has been a consistent theme observed in \\nSjögren\\'s syndrome (SS) in CD4+ T cells, CD19+ B cells, whole blood, and labial \\nsalivary glands (LSGs). Multiple studies have found associations supporting \\ngenetic control of DNA methylation in SS, which in the absence of reverse \\ncausation, has positive implications for the potential of epigenetic therapy. \\nHowever, a formal study of the causal relationship between genetic variation, \\nDNA methylation, and disease status is lacking. We performed a causal mediation \\nanalysis of DNA methylation as a mediator of nearby genetic association with SS \\nusing LSGs and genotype data collected from 131 female members of the Sjögren\\'s \\nInternational Collaborative Clinical Alliance registry, comprising of 64 SS \\ncases and 67 non-cases. Bumphunter was used to first identify \\ndifferentially-methylated regions (DMRs), then the causal inference test (CIT) \\nwas applied to identify DMRs mediating the association of nearby methylation \\nquantitative trait loci (MeQTL) with SS. Bumphunter discovered 215 DMRs, with \\nthe majority located in the major histocompatibility complex (MHC) on chromosome \\n6p21.3. Consistent with previous findings, regions hypomethylated in SS cases \\nwere enriched for gene sets associated with immune processes. Using the CIT, we \\nobserved a total of 19 DMR-MeQTL pairs that exhibited strong evidence for a \\ncausal mediation relationship. Close to half of these DMRs reside in the MHC and \\ntheir corresponding meQTLs are in the region spanning the HLA-DQA1, HLA-DQB1, \\nand HLA-DQA2 loci. The risk of SS conferred by these corresponding MeQTLs in the \\nMHC was further substantiated by previous genome-wide association study results, \\nwith modest evidence for independent effects. By validating the presence of \\ncausal mediation, our findings suggest both genetic and epigenetic factors \\ncontribute to disease susceptibility, and inform the development of targeted \\nepigenetic modification as a therapeutic approach for SS.\\n\\nDOI: 10.1371/journal.pone.0248429\\nPMCID: PMC8062105\\nPMID: 33886574 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have declared that no competing \\ninterests exist.\\n\\n\\n30. Arthritis Rheumatol. 2021 Oct;73(10):1921-1930. doi: 10.1002/art.41758. Epub \\n2021 Aug 31.\\n\\nTranscriptomic Evaluation of Juvenile Localized Scleroderma Skin With Histologic \\nand Clinical Correlation.\\n\\nSchutt C(1), Mirizio E(2), Salgado C(3), Reyes-Mugica M(3), Wang X(3), Chen \\nW(4), Grunwaldt L(3), Schollaert KL(2), Torok KS(4).\\n\\nAuthor information:\\n(1)University of Pittsburgh Medical Center, Children\\'s Hospital of Pittsburgh, \\nPittsburgh, Pennsylvania, and University of Rochester MedicalCenter and Golisano \\nChildren\\'s Hospital, Rochester, New York.\\n(2)University of Pittsburgh, Pittsburgh, Pennsylvania.\\n(3)University of Pittsburgh Medical Center, Children\\'s Hospital of Pittsburgh, \\nPittsburgh, Pennsylvania.\\n(4)University of Pittsburgh, University of Pittsburgh Medical Center, Children\\'s \\nHospital of Pittsburgh, Pittsburgh, Pennsylvania.\\n\\nOBJECTIVE: Juvenile localized scleroderma (LS) is an autoimmune disease of the \\nskin whose pathogenesis is not well understood due to the rarity of the disease. \\nThis study was undertaken to determine the skin transcriptome in skin biopsy \\ntissue from children with juvenile LS compared to pediatric healthy controls, \\nwith identification of significant molecular targets using RNA sequencing \\n(RNA-Seq). In this study, differentially expressed genes (DEGs) were assessed \\nfor correlations with histopathologic and clinical features in children with \\njuvenile LS, and were used to group the children into distinct genetic clusters \\nbased on immunophenotype.\\nMETHODS: RNA-Seq was performed on sections of paraffin-embedded skin tissue \\nobtained from 28 children with juvenile LS and 10 pediatric healthy controls. \\nRNA-Seq was carried out using an Illumina HTS TruSeq RNA Access library prep \\nkit, with data aligned using STAR and data analysis using a DESeq2 platform. A \\nstandardized histologic scoring system was used to score skin sections for the \\nseverity of inflammation and levels of collagen deposition. Histologic scoring \\nwas completed by 2 pathologists who were blinded with regard to the status of \\neach sample. Spearman\\'s rank correlation coefficients were used to assess \\nsignificant correlations between DEG expression profiles and skin histologic \\nfindings in patients with juvenile LS.\\nRESULTS: We identified 589 significant DEGs in children with juvenile LS as \\ncompared to healthy controls. Hierarchical clustering was used to demonstrate 3 \\ndistinct juvenile LS immunophenotype clusters. The histologic scores of skin \\ninflammation (based on numbers and categories of inflammatory cell infiltrates) \\nwere significantly correlated with the expression levels of HLA-DPB1, HLA-DQA2, \\nHLA-DRA, and STAT1 genes (rs > 0.5, P < 0.01). Collagen thickness correlated \\nwith the expression levels of collagen organization genes as well as with genes \\nfound to be correlated with the severity of inflammation, including genes for \\nmajor histocompatibility complex (MHC) class I, MHC class II, and interferon-γ \\nsignaling.\\nCONCLUSION: Among children with juvenile LS, 3 distinct genetic signatures, or \\nclusters, were identified. In one cluster, inflammation-related pathways were \\nup-regulated, corresponding to the histologic skin inflammation score. In the \\nsecond cluster, fibrosis-related pathways were up-regulated. In the third \\ncluster, gene expression in the skin corresponded to the patterns seen in \\nhealthy controls. Up-regulation of HLA class II genes was observed within the \\nfirst cluster (characterized by predominant inflammation), a feature that has \\nalso been observed in the peripheral blood of patients with morphea and in the \\nskin of patients with systemic sclerosis.\\n\\n© 2021, American College of Rheumatology.\\n\\nDOI: 10.1002/art.41758\\nPMID: 33844442 [Indexed for MEDLINE]\\n\\n\\n31. J Cell Mol Med. 2021 May;25(10):4684-4695. doi: 10.1111/jcmm.16407. Epub 2021\\n Mar 22.\\n\\nA single-cell survey of the human glomerulonephritis.\\n\\nChen Z(1), Zhang T(2), Mao K(2), Shao X(1), Xu Y(1), Zhu M(1), Zhou H(1), Wang \\nQ(1), Li Z(1), Xie Y(1), Yuan X(3), Ying L(3), Zhang M(3), Hu J(4), Mou S(1).\\n\\nAuthor information:\\n(1)Department of Nephrology, Molecular Cell Laboratory for Kidney Disease, Ren \\nJi Hospital, School of Medicine, Shanghai Jiao Tong University, Shanghai, China.\\n(2)Shanghai Institute of Immunology, Department of Microbiology and Immunology, \\nShanghai Jiao Tong University School of Medicine (SJTU-SM), Shanghai, China.\\n(3)Transplantation Center of Ren Ji Hospital, School of Medicine, Shanghai Jiao \\nTong University, Shanghai, China.\\n(4)Department of Nuclear Medicine, Ruijin Hospital, Shanghai Jiao Tong \\nUniversity School of Medicine, Shanghai, China.\\n\\nGlomerulonephritis is the one of the major causes of the end-stage kidney \\ndisease, whereas the pathological process of glomerulonephritis is still not \\ncompletely understood. Single-cell RNA sequencing (scRNA-seq) emerges to be a \\npowerful tool to evaluate the full heterogeneity of kidney diseases. To reveal \\ncellular gene expression profiles of glomerulonephritis, we performed scRNA-seq \\nof 2 human kidney transplantation donor samples, 4 human glomerulonephritis \\nsamples, 1 human malignant hypertension (MH) sample and 1 human chronic \\ninterstitial nephritis (CIN) sample, all tissues were taken from the biopsy. \\nAfter filtering the cells with\\xa0<\\xa0200 genes and\\xa0>\\xa010% mitochondria (MT) genes, \\nthe resulting 14\\xa0932 cells can be divided into 20 cell clusters, consistently \\nwith the previous report, in disease samples dramatic immune cells infiltration \\nwas found, among which a proximal tubule (PT) subset characterized by wnt-β \\ncatenin activation and a natural killer T (NKT) subset high expressing LTB were \\nfound. Furthermore, in the cluster of the podocyte, three glomerulonephritis \\nrelated genes named FXYD5, CD74 and B2M were found. Compared with the mesangial \\nof donor, the gene CLIC1 and RPS26 were up-regulated in mesangial of IgA \\nnephropathy(IgAN), whereas the gene JUNB was up-regulated in podocyte of IgAN in \\ncomparison with that of donor. Meanwhile, some membranous nephropathy (MN) high \\nexpressed genes such as HLA-DRB5, HLA-DQA2, IFNG, CCL2 and NR4A2, which involve \\nin highest enrichment pathway, display the cellular-specific expression style, \\nwhereas monocyte marker of lupus nephritis (LN) named TNFSF13B was also found \\nand interferon alpha/beta signalling pathway was enriched in B and NKT of LN \\ncomparing with donor. By scRNA-seq, we first defined the podocyte markers of \\nglomerulonephritis and specific markers in IgA, MN and LN were found at cellular \\nlevel. Furthermore, the critical role of interferon alpha/beta signalling \\npathway was enriched in B and NKT of LN was declared.\\n\\n© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \\nFoundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/jcmm.16407\\nPMCID: PMC8107090\\nPMID: 33754492 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n32. Front Genet. 2020 Dec 10;11:604324. doi: 10.3389/fgene.2020.604324.\\neCollection  2020.\\n\\nIntegrative Analysis of Transcriptome-Wide Association Study and mRNA Expression \\nProfiles Identifies Candidate Genes Associated With Idiopathic Pulmonary \\nFibrosis.\\n\\nGong W(1), Guo P(1), Liu L(1), Guan Q(2)(3)(4), Yuan Z(1).\\n\\nAuthor information:\\n(1)Department of Biostatistics, School of Public Health, Cheeloo College of \\nMedicine, Shandong University, Jinan, China.\\n(2)Department of Endocrinology, Shandong Provincial Hospital Affiliated to \\nShandong First Medical University, Jinan, China.\\n(3)Shandong Clinical Medical Center of Endocrinology and Metabolism, Jinan, \\nChina.\\n(4)Shandong Institute of Endocrine and Metabolic Diseases, Jinan, China.\\n\\nIdiopathic pulmonary fibrosis (IPF) is a type of scarring lung disease \\ncharacterized by a chronic, progressive, and irreversible decline in lung \\nfunction. The genetic basis of IPF remains elusive. A transcriptome-wide \\nassociation study (TWAS) of IPF was performed by FUSION using gene expression \\nweights of three tissues combined with a large-scale genome-wide association \\nstudy (GWAS) dataset, totally involving 2,668 IPF cases and 8,591 controls. \\nSignificant genes identified by TWAS were then subjected to gene ontology (GO) \\nand pathway enrichment analysis. The overlapped GO terms and pathways between \\nenrichment analysis of TWAS significant genes and differentially expressed genes \\n(DEGs) from the genome-wide mRNA expression profiling of IPF were also \\nidentified. For TWAS significant genes, protein-protein interaction (PPI) \\nnetwork and clustering modules analyses were further conducted using STRING and \\nCytoscape. Overall, TWAS identified a group of candidate genes for IPF under the \\nBonferroni corrected P value threshold (0.05/14929 = 3.35 × 10-6), such as DSP \\n(P TWAS = 1.35 × 10-29 for lung tissue), MUC5B (P TWAS = 1.09 × 10-28 for lung \\ntissue), and TOLLIP (P TWAS = 1.41 × 10-15 for whole blood). Pathway enrichment \\nanalysis identified multiple candidate pathways, such as herpes simplex \\ninfection (P value = 7.93 × 10-5) and antigen processing and presentation (P \\nvalue = 6.55 × 10-5). 38 common GO terms and 8 KEGG pathways shared by \\nenrichment analysis of TWAS significant genes and DEGs were identified. In the \\nPPI network, 14 genes (DYNLL1, DYNC1LI1, DYNLL2, HLA-DRB5, HLA-DPB1, HLA-DQB2, \\nHLA-DQA2, HLA-DQB1, HLA-DRB1, POLR2L, CENPP, CENPK, NUP133, and NUP107) were \\nsimultaneously detected by hub gene and module analysis. In conclusion, through \\nintegrative analysis of TWAS and mRNA expression profiles, we identified \\nmultiple novel candidate genes, GO terms and pathways for IPF, which contributes \\nto the understanding of the genetic mechanism of IPF.\\n\\nCopyright © 2020 Gong, Guo, Liu, Guan and Yuan.\\n\\nDOI: 10.3389/fgene.2020.604324\\nPMCID: PMC7758323\\nPMID: 33362862\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n33. Medicine (Baltimore). 2020 Nov 25;99(48):e23477. doi: \\n10.1097/MD.0000000000023477.\\n\\nCD74, a novel predictor for bronchopulmonary dysplasia in preterm infants.\\n\\nGao J(1), Wu M(1), Wang F(1), Jiang L(1), Tian R(2), Zhu X(3), He S(3)(4).\\n\\nAuthor information:\\n(1)Department of Pediatrics, Affiliated Hospital of Yangzhou University, \\nYangzhou, Jiangsu.\\n(2)Department of Pediatrics, The First People\\'s Hospital of Kunming City, \\nKunming, Yunnan.\\n(3)Department of Neonatology, Children\\'s Hospital of Soochow University, Suzhou, \\nJiangsu.\\n(4)Department of Pediatrics, The First People\\'s Hospital of Yunnan Province, \\nKunming, Yunnan, China.\\n\\nBronchopulmonary dysplasia (BPD) remains a major complication and accounts for \\nhigh morbidity and mortality of preterm infants. The present study aimed to \\nidentify the key genes in the development of BPD and to provide some new \\ninsights into the pathogenesis of BPD. The GSE108754 dataset was downloaded from \\nGene Expression Omnibus database containing 5 samples of BPD patients and 6 of \\nnon-BPD infants. The differentially expressed genes (DEGs) between BPD and \\nnon-BPD patients were identified by R software. The pathway and function \\nenrichment analyses were performed through Database for Annotation Visualization \\nand Integrated Discovery website. The protein-protein interaction network for \\nDEGs was established by Cytoscape software and the most highly connected module \\nwas selected through MCODE plugin. Furthermore, the clinical sample verification \\namong 25 BPD patients and 10 non-BPD infants was carried out in our center. \\nFinally, based on the results above, the gene set enrichment analysis focusing \\non CD74 upregulated status was employed. Totally, 189 DEGs including 147 \\nupregulated genes and 42 downregulated genes between BPD and non-BPD patients \\nwere screened out. The pathway and function enrichments revealed these DEGs were \\nmainly enriched in asthma, intestinal immune network for IgA production, antigen \\nprocessing and presentation and immune response. Thirteen DEGs (CD74, HLA-DMA, \\nHLA-DRA, HLA-DMB, HLA-DOB, HLA-DQA1, HLA-DRB5, HLA-DPA1, HLA-DOA, HLA-DPB1, \\nHLA-DQB2, HLA-DQA2, and HLA-DQB1) were determined as hub genes. The mRNA \\nexpression levels of the 13 hub genes were tested by quantitative real-time \\npolymerase chain reaction among our clinical samples. Eventually, CD74 was \\nconfirmed to be the most significant highly expressed in BPD samples (P\\u200a<\\u200a.001) \\nand its expression level was negatively correlated with gestational age \\n(r\\u200a=\\u200a-0.653) and birth weight (r\\u200a=\\u200a-0.675). The gene set enrichment analysis \\nresults showed the gene sets associated with lupus erythematosus, viral \\nmyocarditis, immune network for IgA production, graft versus host disease, cell \\nadhesion molecules and so no were differentially enriched with the phenotype of \\nhigh-expression CD74. In conclusion, CD74 may serve to predict the BPD \\ndevelopment and provide a new therapeutic target for BPD.\\n\\nDOI: 10.1097/MD.0000000000023477\\nPMCID: PMC7710202\\nPMID: 33235138 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have no conflicts of interest to \\ndisclose.\\n\\n\\n34. Clin Respir J. 2021 Apr;15(4):374-381. doi: 10.1111/crj.13309. Epub 2020 Nov\\n30.\\n\\nMajor histocompatibility complex class II polymorphic variants are associated \\nwith asthma predisposition in the Punjabi population of Lahore, Pakistan.\\n\\nAslam R(1), Shahid M(1), Bano I(2), Ayoub M(1), Sabar MF(3), Altaf S(1), Kousar \\nS(1), Ghani MU(3), Husnain T(1), Shahid AA(1).\\n\\nAuthor information:\\n(1)Centre of Excellence in Molecular Biology, University of the Punjab, Lahore, \\nPakistan.\\n(2)The Children\\'s Hospital and The Institute of Child Health, Pediatric \\nPulmonology, Lahore, Pakistan.\\n(3)Centre for Applied Molecular Biology, University of the Punjab, Lahore, \\nPakistan.\\n\\nINTRODUCTION: Various genome wide association studies have manifested that Major \\nHistocompatibility Complex (MHC) region on chromosome 6p21 houses many potential \\ncandidate genes for asthma.\\nOBJECTIVE: This Case-Control association study was planned to determine the \\nassociation of 10 Single Nucleotide Polymorphisms (SNPs), residing within and \\naround MHC genes\\' region on chromosome 6p21, with Asthma in Punjabi population \\nof Lahore, Pakistan.\\nMETHODS: A total of 161 subjects, 61 physician-diagnosed asthma patients and 100 \\nage-matched healthy controls, were recruited from Lahore, a city in Punjab. Ten \\nsingle nucleotide polymorphisms (SNPs) (rs9378249, rs2070600, rs404860, rs6689, \\nrs1049124, rs1063355, rs1049225, rs1049219, rs7773955 and rs928976) located \\nwithin or near AGER, NOTCH and HLA genes in MHC region, were genotyped in both \\npatients and controls using single base extension reaction and capillary \\nelectrophoresis-based genetic analyser. Statistical models were applied using \\nSHEsis Plus. Results were adjusted for various cofactors (age, gender and \\nenvironment) and by applying multiple corrections. Haplotype and linkage \\ndisequilibrium analyses were performed on Haploview software v4.1.\\nRESULTS: Three of the studied SNPs rs1049124, rs1049219 and rs7773955 show \\nindependent significant association with asthma under allelic and genotypic \\nmodels. Two of the haplotypes, H7 and H13, \"CTAATTT\" and \"CCACTAT\", \\nrespectively, for rs2070600, rs404860, rs6689, rs1049124, rs1063355, rs1049219 \\nand rs7773955, are found to be significantly associated with the disease.\\nCONCLUSION: This study reports association of SNP variants residing on HLA-DQB1 \\nand HLA-DQA2 genes and haplotypes H7 and H13 on genomic region 6p21 with Asthma \\nin the Punjabi population of Lahore, Pakistan.\\n\\n© 2020 John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/crj.13309\\nPMID: 33211378 [Indexed for MEDLINE]\\n\\n\\n35. J Allergy Clin Immunol. 2021 Mar;147(3):933-940. doi: \\n10.1016/j.jaci.2020.08.020. Epub 2020 Sep 2.\\n\\nA genome-wide association study of asthma hospitalizations in adults.\\n\\nYan Q(1), Forno E(1), Herrera-Luis E(2), Pino-Yanes M(3), Yang G(1), Oh S(4), \\nAcosta-Pérez E(5), Hu D(4), Eng C(4), Huntsman S(4), Rodriguez-Santana JR(6), \\nCloutier MM(7), Canino G(5), Burchard EG(4), Chen W(1), Celedón JC(8).\\n\\nAuthor information:\\n(1)Division of Pediatric Pulmonary Medicine, UPMC Children\\'s Hospital of \\nPittsburgh, University of Pittsburgh, Pittsburgh, Pa.\\n(2)Genomics and Health Group, Department of Biochemistry, Microbiology, Cell \\nBiology and Genetics, Universidad de La Laguna, La Laguna, Spain.\\n(3)Genomics and Health Group, Department of Biochemistry, Microbiology, Cell \\nBiology and Genetics, Universidad de La Laguna, La Laguna, Spain; CIBER de \\nEnfermedades Respiratorias, Instituto de Salud Carlos III, Spain.\\n(4)Department of Medicine, University of California San Francisco, San \\nFrancisco, Calif.\\n(5)Behavioral Sciences Research Institute, University of Puerto Rico, San Juan, \\nPuerto Rico.\\n(6)Centro de Neumología Pediátrica, San Juan, Puerto Rico.\\n(7)Department of Pediatrics, University of Connecticut, Farmington, Conn.\\n(8)Division of Pediatric Pulmonary Medicine, UPMC Children\\'s Hospital of \\nPittsburgh, University of Pittsburgh, Pittsburgh, Pa. Electronic address: \\njuan.celedon@chp.edu.\\n\\nBACKGROUND: Little is known about the genetic determinants of severe asthma \\nexacerbations.\\nOBJECTIVES: We aimed to identify genetic variants associated with asthma \\nhospitalizations.\\nMETHODS: We conducted a genome-wide association study of asthma hospitalizations \\nin 34,167 white British adults with asthma, 1,658 of whom had at least 1 \\nasthma-related hospitalization. This analysis was conducted by using logistic \\nregression under an additive genetic model with adjustment for age, sex, body \\nmass index, smoking status, and the first 5 principal components derived from \\ngenotypic data. We then analyzed data from 2 cohorts of Latino children and \\nadolescents for replication and conducted quantitative trait locus and \\nfunctional annotation analyses.\\nRESULTS: At the chromosome 6p21.3 locus, the single-nucleotide polymorphism \\n(SNP) rs56151658 (8 kb from the promoter of HLA-DQB1) was most significantly \\nassociated with asthma hospitalizations (for test allele A, odds ratio\\xa0= 1.36 \\n[95% CI\\xa0= 1.22-1.52]; P\\xa0= 3.11\\xa0×\\xa010-8); 21 additional SNPs in this locus were \\nassociated with asthma hospitalizations at a P value less than 1\\xa0×\\xa010-6. In the \\nreplication cohorts, multiple SNPs in strong linkage disequilibrium with \\nrs56151658 were associated with severe asthma exacerbations at a P value of .01 \\nor less in the same direction of association as in the discovery cohort. Three \\nHLA genes (HLA-DQA2, HLA-DRB6, and HLA-DOB) were also shown to mediate the \\nestimated effects of the SNPs associated with asthma hospitalizations through \\neffects on gene expression in lung tissue.\\nCONCLUSIONS: We identified strong candidate genes for asthma hospitalizations in \\nadults in the region for class II HLA genes through genomic, quantitative trait \\nlocus, and summary data-based mendelian randomization analyses.\\n\\nCopyright © 2020 American Academy of Allergy, Asthma & Immunology. Published by \\nElsevier Inc. All rights reserved.\\n\\nDOI: 10.1016/j.jaci.2020.08.020\\nPMCID: PMC7921212\\nPMID: 32890573 [Indexed for MEDLINE]\\n\\n\\n36. Kidney Int. 2021 Mar;99(3):671-685. doi: 10.1016/j.kint.2020.08.007. Epub\\n2020  Sep 2.\\n\\nHLA-D and PLA2R1 risk alleles associate with recurrent primary membranous \\nnephropathy in kidney transplant recipients.\\n\\nBerchtold L(1), Letouzé E(2), Alexander MP(3), Canaud G(4), Logt AV(5), Hamilton \\nP(6), Mousson C(7), Vuiblet V(8), Moyer AM(9), Guibert S(10), Mrázová P(11), \\nLevi C(12), Dubois V(13), Cruzado JM(14), Torres A(15), Gandhi MJ(16), Yousfi \\nN(17), Tesar V(18), OndrejViklický(19), Hourmant M(20), Moulin B(21), Rieu \\nP(22), Choukroun G(23), Legendre C(24), Wetzels J(5), Brenchley P(6), Ballarín \\nCastan JA(25), Debiec H(17), Ronco P(26).\\n\\nAuthor information:\\n(1)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France; Division of Nephrology, Department of \\nInternal Medicine Specialties, University Hospital of Geneva, Geneva, \\nSwitzerland.\\n(2)Centre de Recherche des Cordeliers, Sorbonne Université, INSERM, France; \\nFunctional Genomics of Solid Tumor, Labex Immuno-Oncology, Equipe Labellisée \\nLigue Contre le Cancer, Université Paris 13, Paris, France.\\n(3)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, \\nMinnesota, USA.\\n(4)Inserm U1151, Necker-Enfants Malades Hospital, Paris, France; University \\nParis Descartes, Sorbonne Paris Cité, Paris, France; Department of Adult \\nNephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, France.\\n(5)Department of Nephrology, Radboud University Medical Center, Radboud \\nInstitute for Health Sciences, Nijmegen, The Netherlands.\\n(6)Division of Cardiovascular Sciences, School of Medical Sciences, Faculty of \\nBiology, Medicine and Health, University of Manchester, Manchester, UK.\\n(7)Department of Nephrology and Transplantation, University Hospital, Dijon, \\nFrance.\\n(8)BioSpec T Laboratory, EA 7506 URCA, Reims, France; Nephropathology Department \\nof Biopathology Laboratory, Reims University Hospital, Reims, France; Division \\nof Nephrology, Reims University Hospital, Reims, France.\\n(9)Department of Laboratory Medicine and Pathology, Personalized Genomics \\nLaboratory Mayo Clinic, Rochester, Minnesota, USA.\\n(10)IntegraGen, Evry, France.\\n(11)Transplant Laboratory, Institute for Clinical and Experimental Medicine, \\nPrague, Czech Republic.\\n(12)Department of Transplantation, Nephrology, and Clinical Immunology, Edouard \\nHerriot Hospital, Hospices Civils de Lyon, Lyon, France.\\n(13)Laboratoire HLA, Etablissement Français du Sang Auvergne Rhone-Alpes, Lyon, \\nFrance.\\n(14)Nephrology Department, Hospital Universitari Bellvitge, Bellvitge Research \\nInstitute (IDIBELL), Barcelona, Spain; RedInRen, RD16/0009/0031, University of \\nBarcelona, L\\'Hospitalet de Llobregat, Barcelona, Spain.\\n(15)Hospital Universitario de Canarias, Instituto de Tecnologías Biomédicas \\n(ITB), Universidad de La Laguna, Tenerife, Spain.\\n(16)Division of Transfusion Medicine, Mayo Clinic, Rochester, Minnesota, USA.\\n(17)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France.\\n(18)Department of Nephrology, First Faculty of Medicine, Charles University and \\nGeneral University Hospital, Prague, Czech Republic.\\n(19)Department of Nephrology, Transplant Center, Institute for Clinical and \\nExperimental Medicine, Prague, Czech Republic.\\n(20)Department of Nephrology, University Hospital, Nantes, France.\\n(21)Institut National de la Santé et de la Recherche Médicale, Unité Mixte de \\nRecherche S 1109, Strasbourg University, Strasbourg, France; Fédération de \\nMédecine Translationnelle de Strasbourg (FMTS), Strasbourg, France; Department \\nof Nephrology, Strasbourg University Hospital, Strasbourg, France.\\n(22)Division of Nephrology, Reims University Hospital, Reims, France; Université \\nde Reims Champagne-Ardenne, CNRS UMR 7369 (Matrice Extracellulaire et Dynamique \\nCellulaire, MEDyC), Reims, France.\\n(23)Department of Nephrology, University Hospital, Amiens, France.\\n(24)University Paris Descartes, Sorbonne Paris Cité, Paris, France; Department \\nof Adult Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, \\nFrance.\\n(25)Department of Nephrology, Fundación Puigvert, Barcelona, Spain.\\n(26)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France; Department of Nephrology (Day Hospital), \\nAP-HP, Tenon Hospital, Paris, France. Electronic address: pierre.ronco@upmc.fr.\\n\\nErratum in\\n    Kidney Int. 2021 Jul;100(1):243.\\n\\nComment in\\n    Kidney Int. 2021 Mar;99(3):545-548.\\n\\nRecurrence of primary membranous nephropathy after transplantation occurs in up \\nto 44% of patients and is driven by PLA2R antibody. Here, we asked whether \\ngenetic determinants could improve risk prediction. First, we sequenced PLA2R1 \\nand HLA-D loci in 248 patients with primary membranous nephropathy and \\nidentified two independent single nucleotide polymorphisms (SNPs) at risk for \\nprimary membranous nephropathy at each locus. These were rs9271188 (intergenic \\nbetween HLA-DRB1 and HLA-DQA1,) and rs9275086 (intergenic between HLA-DQB1 and \\nHLA-DQA2) at the HLA-D locus along with rs6726925 and rs13018963 at the PLA2R1 \\nlocus. Then we investigated whether primary membranous nephropathy at-risk \\nvariants were associated with recurrence in a retrospective cohort of 105 \\ndonor-recipient pairs and a replication cohort of 40 pairs. Seven SNPs located \\nbetween HLA-DRB1 and HLA-DQA1 in linkage disequilibrium with rs9271188, and \\nthree SNPs in the PLA2R1 region predicted recurrence when presented by the \\ndonor, but not when presented by the recipient. The two SNPs in the HLA-D region \\nmost strongly associated with recurrence (rs9271705 and rs9271550) were \\nconfirmed in the replication cohort. A genetic risk score based on the two best \\npredictors at each locus (rs9271705, rs9271550, rs17830558, and rs3828323) \\nidentified a group of patients with high risk of recurrence. Thus, our results \\nsuggest that the graft contributes to recurrence of primary membranous \\nnephropathy through the disease susceptibility HLA-D and PLA2R1 SNPs in an \\nautoimmune milieu. Further studies are needed before implementation of genetic \\ntesting for these in donor selection.\\n\\nCopyright © 2020. Published by Elsevier Inc.\\n\\nDOI: 10.1016/j.kint.2020.08.007\\nPMID: 32889013 [Indexed for MEDLINE]\\n\\n\\n37. Front Pharmacol. 2020 Jul 24;11:1138. doi: 10.3389/fphar.2020.01138.\\neCollection  2020.\\n\\nDiscovery of Selenocysteine as a\\xa0Potential Nanomedicine Promotes Cartilage \\nRegeneration With Enhanced Immune Response by Text Mining and Biomedical \\nDatabases.\\n\\nYe J(1), Xu B(1), Fan B(2), Zhang J(1), Yuan F(1), Chen Y(1), Sun Z(1), Yan \\nX(1), Song Y(1), Song S(1), Yang M(2), Yu JK(1).\\n\\nAuthor information:\\n(1)Knee Surgery Department of the Institution of Sports Medicine, Peking \\nUniversity Third Hospital, Beijing Key Laboratory of Sports Injuries, Beijing, \\nChina.\\n(2)School of Clinical Medicine, Weifang Medical University, Weifang, China.\\n\\nBACKGROUND: Unlike bone tissue, little progress has been made regarding \\ncartilage regeneration, and many challenges remain. Furthermore, the key roles \\nof cartilage lesion caused by traumas, focal lesion, or articular overstress \\nremain unclear. Traumatic injuries to the meniscus as well as its degeneration \\nare important risk factors for long-term joint dysfunction, degenerative joint \\nlesions, and knee osteoarthritis (OA) a chronic joint disease characterized by \\ndegeneration of articular cartilage and hyperosteogeny. Nearly 50% of the \\nindividuals with meniscus injuries develop OA over time. Due to the limited \\ninherent self-repair capacity of cartilage lesion, the Biomaterial \\ndrug-nanomedicine is considered to be a promising alternative. Therefore, it is \\nimportant to elucidate the gene potential regeneration mechanisms and discover \\nnovel precise medication, which are identified through this study to investigate \\ntheir function and role in pathogenesis.\\nMETHODS: We downloaded the mRNA microarray statistics GSE117999, involving \\npaired cartilage lesion tissue samples from 12 OA patients and 12 patients from \\na control group. First, we analyzed these statistics to recognize the \\ndifferentially expressed genes (DEGs). We then exposed the gene ontology (GO) \\nannotation and the Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathway \\nenrichment analyses for these DEGs. Protein-protein interaction (PPI) networks \\nwere then constructed, from which we attained eight significant genes after a \\nfunctional interaction analysis. Finally, we identified a potential nanomedicine \\nattained from this assay set, using a wide range of inhibitor information \\narchived in the Search Tool for the Retrieval of Interacting Genes (STRING) \\ndatabase.\\nRESULTS: Sixty-six DEGs were identified with our standards for meaning (adjusted \\nP-value < 0.01, |log2 - FC| ≥1.2). Furthermore, we identified eight hub genes \\nand one potential nanomedicine - Selenocysteine based on these integrative data.\\nCONCLUSION: We identified eight hub genes that could work as prospective \\nbiomarkers for the diagnostic and biomaterial drug treatment of cartilage \\nlesion, involving the novel genes CAMP, DEFA3, TOLLIP, HLA-DQA2, SLC38A6, \\nSLC3A1, FAM20A, and ANO8. Meanwhile, these genes were mainly associated with \\nimmune response, immune mediator induction, and cell chemotaxis. Significant \\nsupport is provided for obtaining a series of novel gene targets, and we \\nidentify potential mechanisms for cartilage regeneration and final nanomedicine \\nimmunotherapy in regenerative medicine.\\n\\nCopyright © 2020 Ye, Xu, Fan, Zhang, Yuan, Chen, Sun, Yan, Song, Song, Yang and \\nYu.\\n\\nDOI: 10.3389/fphar.2020.01138\\nPMCID: PMC7394085\\nPMID: 32792959\\n\\n\\n38. Sci Rep. 2020 Jul 2;10(1):10868. doi: 10.1038/s41598-020-67730-y.\\n\\nSynovial cell cross-talk with cartilage plays a major role in the pathogenesis \\nof osteoarthritis.\\n\\nChou CH(#)(1), Jain V(#)(1), Gibson J(1), Attarian DE(2), Haraden CA(1), Yohn \\nCB(3), Laberge RM(3), Gregory S(1), Kraus VB(4)(5).\\n\\nAuthor information:\\n(1)Duke Molecular Physiology Institute, Duke University School of Medicine, \\nDurham, NC, 27701, USA.\\n(2)Department of Orthopaedic Surgery, Duke University School of Medicine, \\nDurham, NC, 27701, USA.\\n(3)Unity Biotechnology, South San Francisco, CA, USA.\\n(4)Duke Molecular Physiology Institute, Duke University School of Medicine, \\nDurham, NC, 27701, USA. vbk@duke.edu.\\n(5)Division of Rheumatology, Department of Medicine, Duke University School of \\nMedicine, Durham, NC, 27701, USA. vbk@duke.edu.\\n(#)Contributed equally\\n\\nWe elucidated the molecular cross-talk between cartilage and synovium in \\nosteoarthritis, the most widespread arthritis in the world, using the powerful \\ntool of single-cell RNA-sequencing. Multiple cell types were identified based on \\nprofiling of 10,640 synoviocytes and 26,192 chondrocytes: 12 distinct synovial \\ncell types and 7 distinct articular chondrocyte phenotypes from matched tissues. \\nIntact cartilage was enriched for homeostatic and hypertrophic chondrocytes, \\nwhile damaged cartilage was enriched for prefibro- and fibro-, regulatory, \\nreparative and prehypertrophic chondrocytes. A total of 61 cytokines and growth \\nfactors were predicted to regulate the 7 chondrocyte cell phenotypes. Based on \\nproduction by >\\u20091% of cells, 55% of the cytokines were produced by synovial \\ncells (39% exclusive to synoviocytes and not expressed by chondrocytes) and \\ntheir presence in osteoarthritic synovial fluid confirmed. The synoviocytes \\nproducing IL-1beta (a classic pathogenic cytokine in osteoarthritis), mainly \\ninflammatory macrophages and dendritic cells, were characterized by \\nco-expression of surface proteins corresponding to HLA-DQA1, HLA-DQA2, OLR1 or \\nTLR2. Strategies to deplete these pathogenic intra-articular cell subpopulations \\ncould be a therapeutic option for human osteoarthritis.\\n\\nDOI: 10.1038/s41598-020-67730-y\\nPMCID: PMC7331607\\nPMID: 32616761 [Indexed for MEDLINE]\\n\\nConflict of interest statement: C.H.C., V.J., J.G., D.E.A., C.A.H. and S.G. \\ndeclare no competing interests. C.B.Y. and R.M.L. are employees of Unity \\nBiotechnology and played a role in the conceptualization of the study and in \\npreparation of the manuscript. This work was funded in part by Unity \\nBiotechnology, however, Unity Biotechnology had no role in the preparation of \\nthe manuscript, data collection or decision to publish this study. V.B.K. has \\nbeen a periodic consultant for Unity Biotechnology providing periodic advice on \\nclinical study design, unrelated to this study and has no competing interests to \\ndeclare.\\n\\n\\n39. Medicina (Kaunas). 2020 Mar 31;56(4):154. doi: 10.3390/medicina56040154.\\n\\nAn Intergenic rs9275596 Polymorphism on Chr. 6p21 Is Associated with Multiple \\nSclerosis in Latvians.\\n\\nParamonova N(1), Trapina I(1), Dokane K(1), Kalnina J(1), Sjakste T(1), Sjakste \\nN(1)(2).\\n\\nAuthor information:\\n(1)Genomics and Bioinformatics, Institute of Biology of the University of \\nLatvia, LV-1004 Riga, Latvia.\\n(2)Department of Medical Biochemistry of the University of Latvia, LV-1004 Riga, \\nLatvia.\\n\\nBackground and objectives: Multiple sclerosis (MS) is a chronic inflammatory \\ndisease of the central nervous system, leading to demyelination of neurons and \\npotentially debilitating physical and mental symptoms. The disease is more \\nprevalent in women than in men. The major histocompatibility complex (MHC) \\nregion has been identified as a major genetic determinant for autoimmune \\ndiseases, and its role in some neurological disorders including MS was \\nevaluated. An intergenic single-nucleotide polymorphism (SNP), rs9275596, \\nlocated between the HLA-DQB1 and HLA-DQA2 genes, is in significant association \\nwith various autoimmune diseases according to genome-wide association studies \\n(GWASs). A cumulative effect of this SNP with other polymorphisms from this \\nregion was revealed. The aim of the study was to verify the data on rs9275596 \\nassociation in multiple sclerosis in a case/control study of the Latvian \\npopulation and to evaluate eventual functional significance of allele \\nsubstitutions. Materials and Methods: rs9275596 (chr6:32713854; GRCh38.p12) was \\ngenotyped in 273 MS patients and 208 controls on main and sex-specific \\nassociations. Eventual functional significance of allele substitutions was \\nevaluated in silico using publicly available tools. Results: The rs9275596 rare \\nalleles were identified as a disease susceptibility factor in association with \\nthe MS main group and in affected females (p < 0.001 and p < 0.01, \\nrespectively). Risk factor genotypes with rare alleles included were associated \\nwith the MS common cohort (p < 0.002) and female cohort (odds ratio, OR = 2.24) \\nand were identified as disease susceptible in males (OR = 2.41). It was shown \\nthat structural changes of rs9275596 affect the secondary structure of DNA. \\nFunctional significance of allele substitutions was evaluated on the eventual \\nsequence affinity to transcription factors (TFs) and splicing signals \\nsimilarity. A possible impact of the particular polymorphisms on the \\ntranscription and splicing efficiency is discussed. Conclusions: Our results \\nsuggest susceptibility of rs9275596 to multiple sclerosis in Latvians.\\n\\nDOI: 10.3390/medicina56040154\\nPMCID: PMC7230508\\nPMID: 32244438 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no conflict \\nof interest.\\n\\n\\n40. Hum Immunol. 2020 May;81(5):202-205. doi: 10.1016/j.humimm.2020.02.003. Epub \\n2020 Feb 28.\\n\\nA new set of reagents and related software used for NGS based classical and \\nnon-classical HLA typing showing evidence for a greater HLA haplotype diversity.\\n\\nAlizadeh M(1), Picard C(2), Frassati C(3), Walencik A(4), Gauthier AC(4), \\nBennasar F(5), Verite F(5), Semana G(5).\\n\\nAuthor information:\\n(1)National Reagent Production Laboratory - Établissement Français du Sang \\nBretagne, Rennes, France. Electronic address: mehdi.alizadeh@efs.sante.fr.\\n(2)Department of Histocompatibility - Établissement Français du Sang Alpes \\nMéditerranée, Marseille, France; UMR 7268 ADÉS Aix-Marseille Université \\n/EFS/CNRS, Marseille, France.\\n(3)Department of Histocompatibility - Établissement Français du Sang Alpes \\nMéditerranée, Marseille, France.\\n(4)Department of Histocompatibility - Établissement Français du Sang Pays de la \\nLoire, Nantes, France.\\n(5)National Reagent Production Laboratory - Établissement Français du Sang \\nBretagne, Rennes, France.\\n\\nErratum in\\n    Hum Immunol. 2020 Aug;81(8):460.\\n\\nTo evaluate the HLA typing performance of a new Long-Range PCR NGS set of \\nreagents and its dedicated software, a panel of 41 reference homozygous cell \\nlines from the International Histocompatibility Working Group (IHWG) and a panel \\nof 376 volunteer bone marrow donors were analyzed for classical and \\nnon-classical HLA class I and class II genes. All results, except HLA-DPB1, were \\nobtained without any ambiguities at the 3rd field level. Based on the high \\nresolution performance of the reagents, a number of new alleles have been \\ndescribed not only for classical but also for non-classical HLA class I genes, \\nleading to a more accurate haplotype definition. Linkage disequilibrium between \\nHLA-A and HLA-G genes has been defined at 4th field level of resolution. \\nMoreover, for the first time, HLA-DQA2 and DQB2 polymorphisms and their linkage \\ndisequilibrium with DQB1 were described.\\n\\nCopyright © 2020 American Society for Histocompatibility and Immunogenetics. \\nPublished by Elsevier Inc. All rights reserved.\\n\\nDOI: 10.1016/j.humimm.2020.02.003\\nPMID: 32122686 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n41. J Clin Lab Anal. 2020 Feb;34(2):e23066. doi: 10.1002/jcla.23066. Epub 2019\\nOct  12.\\n\\nAssociation of GWAS-supported noncoding area loci rs404860, rs3117098, and \\nrs7775228 with asthma in Chinese Zhuang population.\\n\\nLiang SQ(1), Deng JM(1), Wei X(1), Chen ZR(1), Yang ML(1), Qin HJ(1), Zhang \\nJQ(1), He ZY(1).\\n\\nAuthor information:\\n(1)Department of Respiratory and Critical Care Medicine, First Affiliated \\nHospital of Guangxi Medical University, Nanning, China.\\n\\nBACKGROUND: Asthma is a complicated and polygenic inheritance disease, and its \\nprevalence increases worldwide. Recent genome-wide association studies (GWASs) \\nidentified a significant association of single nucleotide polymorphism with \\nasthma in the Japanese population. This study aimed to examine the association \\nof GWAS-supported noncoding area loci, namely rs404860, rs3117098, and \\nrs7775228, with asthma in Chinese Zhuang population.\\nMETHODS: A case-control study involving 223 individuals, comprising 123 patients \\nwith asthma and 100 healthy controls, was conducted. Genotypes were determined \\nby polymerase chain reaction (PCR)/ligase detection reaction assay. The \\nassociation between gene polymorphisms and asthma risk was calculated by \\nlogistic regression analysis using different genetic models through comparisons \\nof alleles (A vs a), homozygote genotypes (AA vs aa), heterozygote genotypes (Aa \\nvs aa), dominant models (AA+Aa vs aa), and recessive models (AA vs. Aa+aa).\\nRESULTS: The distribution of the genotype frequency of rs3117098 was \\nstatistically different between the case and control groups. For rs3117098, \\nsignificant associations were observed through comparisons of alleles (OR: \\n1.832, 95% CI: 1.048-3.204, P\\xa0=\\xa0.034) and dominant models (OR: 2.065, 95% CI: \\n1.001-4.260, P\\xa0=\\xa0.050). The statistical analysis showed no significant \\ndifference for loci rs404860 and rs7775228 between patients with asthma and \\ncontrols.\\nCONCLUSION: rs3117098 may be the risk factor for asthma in Chinese Zhuang \\npopulation.\\n\\n© 2019 The Authors. Journal of Clinical Laboratory Analysis published by Wiley \\nPeriodicals, Inc.\\n\\nDOI: 10.1002/jcla.23066\\nPMCID: PMC7031573\\nPMID: 31605414 [Indexed for MEDLINE]\\n\\nConflict of interest statement: All authors report no conflicts of interest.\\n\\n\\n42. Clin Exp Immunol. 2019 Dec;198(3):367-380. doi: 10.1111/cei.13367. Epub 2019\\nOct  1.\\n\\nPneumonia: host susceptibility and shared genetics with pulmonary function and \\nother traits.\\n\\nKhadzhieva MB(1)(2)(3), Kuzovlev AN(1), Salnikova LE(1)(2)(3).\\n\\nAuthor information:\\n(1)Federal Research and Clinical Center of Intensive Care Medicine and \\nRehabilitology, Moscow, Russia.\\n(2)N. I. Vavilov Institute of General Genetics, Russian Academy of Sciences, \\nMoscow, Russia.\\n(3)Dmitry Rogachev National Research Center of Pediatric Hematology, Oncology \\nand Immunology, Moscow, Russia.\\n\\nPneumonia is a common and severe infectious lung disease. Host genetics, \\ntogether with underlying medical and lifestyle conditions, determine pneumonia \\nsusceptibility. We performed a secondary analysis of the results of two \\ngenome-wide studies for pneumonia in 23andMe participants (40\\xa0600 cases/90\\xa0039 \\ncontrols) (Tian et al., 2017) and UK Biobank (BB) participants (12\\xa0614 \\ncases/324\\xa0585 controls) (via the Global Biobank Engine) and used the GTEx \\ndatabase to correlate the results with expression quantitative trait loci \\n(eQTLs) data in lung and whole blood. In the 23andMe pneumonia single nucleotide \\npolymorphism (SNP) set, 177 genotyped SNPs in the human leukocyte antigen (HLA) \\nregion satisfied the genome-wide significance level, P\\xa0≤ 5·0E-08. Several target \\ngenes (e.g. C4A, VARS2, SFTA2, HLA-C, HLA-DQA2) were unidirectionally regulated \\nby many HLA eSNPs associated with a higher risk of pneumonia. In lung, C4A \\ntranscript was up-regulated by 291 pneumonia risk alleles spanning the half the \\nHLA region. Among SNPs correlated with the expression levels of SFTA2 and VARS2, \\napproximately 75% overlapped: all risk alleles were associated with VARS2 \\nup-regulation and SFTA2 down-regulation. To find shared gene loci between \\npneumonia and pulmonary function (PF), we used data from the Global Biobank \\nEngine and literature on genome-wide association studies (GWAS) of PF in general \\npopulations. Numerous gene loci overlapped between pneumonia and PF: 28·8% in \\nthe BB data set and 49·2% in the 23andMe data set. Enrichment analysis within \\nthe database of Genotypes and Phenotypes (dbGaP) and National Human Genome \\nResearch Institute-European Bioinformatics Institute (NHGRI-EBI) Catalog of \\npneumonia and pneumonia/PF gene sets identified significant overlap between \\nthese gene sets and genes related to inflammatory, developmental, \\nneuropsychiatric and cardiovascular and obesity-related traits.\\n\\n© 2019 British Society for Immunology.\\n\\nDOI: 10.1111/cei.13367\\nPMCID: PMC6857187\\nPMID: 31487037 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing financial \\ninterests.\\n\\n\\n43. Acta Neuropathol. 2019 Jun;137(6):879-899. doi: 10.1007/s00401-019-01962-9.\\nEpub  2019 Feb 9.\\n\\nGenome-wide analyses as part of the international FTLD-TDP whole-genome \\nsequencing consortium reveals novel disease risk factors and increases support \\nfor immune dysfunction in FTLD.\\n\\nPottier C(1), Ren Y(2), Perkerson RB 3rd(1), Baker M(1), Jenkins GD(3), van \\nBlitterswijk M(1), DeJesus-Hernandez M(1), van Rooij JGJ(4), Murray ME(1), \\nChristopher E(1), McDonnell SK(3), Fogarty Z(3), Batzler A(3), Tian S(3), \\nVicente CT(1), Matchett B(1), Karydas AM(5), Hsiung GR(6), Seelaar H(4), Mol \\nMO(4), Finger EC(7), Graff C(8)(9), Öijerstedt L(8)(9), Neumann M(10)(11), \\nHeutink P(10)(12), Synofzik M(10)(12), Wilke C(10)(12), Prudlo J(10)(13), Rizzu \\nP(10), Simon-Sanchez J(10)(12), Edbauer D(14)(15), Roeber S(16), Diehl-Schmid \\nJ(17), Evers BM(18), King A(19)(20), Mesulam MM(21), Weintraub S(21)(22), Geula \\nC(21), Bieniek KF(1)(23), Petrucelli L(1), Ahern GL(24), Reiman EM(25), Woodruff \\nBK(26), Caselli RJ(26), Huey ED(27), Farlow MR(28), Grafman J(29), Mead S(30), \\nGrinberg LT(5)(31), Spina S(5), Grossman M(32), Irwin DJ(32), Lee EB(33), Suh \\nE(33), Snowden J(34), Mann D(35), Ertekin-Taner N(1)(36), Uitti RJ(36), Wszolek \\nZK(36), Josephs KA(37), Parisi JE(37), Knopman DS(37), Petersen RC(37), Hodges \\nJR(38), Piguet O(39), Geier EG(5), Yokoyama JS(5), Rissman RA(40)(41), Rogaeva \\nE(42), Keith J(43)(44), Zinman L(43), Tartaglia MC(42)(45), Cairns NJ(46), \\nCruchaga C(47), Ghetti B(48), Kofler J(49), Lopez OL(24)(50), Beach TG(51), \\nArzberger T(14)(16)(52), Herms J(14)(16), Honig LS(53), Vonsattel JP(54), \\nHalliday GM(38)(55), Kwok JB(38)(55), White CL 3rd(18), Gearing M(56), Glass \\nJ(56), Rollinson S(57), Pickering-Brown S(57), Rohrer JD(58), Trojanowski \\nJQ(33), Van Deerlin V(33), Bigio EH(21), Troakes C(19), Al-Sarraj S(19)(20), \\nAsmann Y(2), Miller BL(5), Graff-Radford NR(36), Boeve BF(37), Seeley WW(5)(31), \\nMackenzie IRA(59), van Swieten JC(4), Dickson DW(1), Biernacka JM(3), Rademakers \\nR(60).\\n\\nAuthor information:\\n(1)Department of Neuroscience, Mayo Clinic, 4500 San Pablo Road, Jacksonville, \\nFL, 32224, USA.\\n(2)Department of Health Sciences Research, Mayo Clinic, Jacksonville, FL, USA.\\n(3)Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA.\\n(4)Department of Neurology, Erasmus Medical Center, Wytemaweg 80, 3015 CN, \\nRotterdam, The Netherlands.\\n(5)Department of Neurology, Memory and Aging Center, University of California, \\nSan Francisco, CA, USA.\\n(6)Division of Neurology, Department of Medicine, University of British \\nColumbia, Vancouver, BC, V6T 2B5, Canada.\\n(7)Department of Clinical Neurological Sciences, Schulich School of Medicine and \\nDentistry, University of Western Ontario, London, ON, N6A 2E2, Canada.\\n(8)Division of Neurogeriatrics, Department NVS, Karolinska Institutet, \\nVisionsgatan 4, J10:20, 171 64, Solna, Sweden.\\n(9)Theme Aging, Unit for Hereditary Dementias, Karolinska University Hospital, \\nSolna, Sweden.\\n(10)German Center for Neurodegenerative Diseases (DZNE), 18147, Rostock, \\nGermany.\\n(11)Department of Neuropathology, University of Tübingen, 72076, Tübingen, \\nGermany.\\n(12)Hertie Institute for Clinical Brain Research, University of Tübingen, 72076, \\nTübingen, Germany.\\n(13)Department of Neurology, Rostock University Medical Center, 18147, Rostock, \\nGermany.\\n(14)German Center for Neurodegenerative Diseases (DZNE), Feodor-Lynen-Str 17, \\n81377, Munich, Germany.\\n(15)Munich Cluster of Systems Neurology (SyNergy), Feodor-Lynen-Str 17, 81377, \\nMunich, Germany.\\n(16)Center for Neuropathology and Prion Research, Ludwig-Maximilians-University \\nof Munich, Feodor-Lynen-Straße 23, 81377, Munich, Germany.\\n(17)Department of Psychiatry and Psychotherapy, Technische Universität München, \\nMunich, Germany.\\n(18)Division of Neuropathology, University of Texas Southwestern Medical Center, \\n5323 Harry Hines Blvd, Dallas, TX, 75390-9073, USA.\\n(19)London Neurodegenerative Diseases Brain Bank, Department of Basic and \\nClinical Neuroscience, Institute of Psychiatry, Psychology and Neuroscience, \\nKing\\'s College London, London, SE5 8AF, UK.\\n(20)Department of Clinical Neuropathology, King\\'s College Hospital NHS \\nFoundation Trust, London, SE5 9RS, UK.\\n(21)Mesulam Center for Cognitive Neurology and Alzheimer\\'s Disease, Northwestern \\nUniversity, Chicago, IL, 60611, USA.\\n(22)Department of Psychiatry and Behavioral Sciences and Department of \\nNeurology, Northwestern University Feinberg School of Medicine, Chicago, IL, \\n60611, USA.\\n(23)Glenn Biggs Institute for Alzheimer\\'s and Neurodegenerative Diseases, \\nUniversity of Texas Health Science Center San Antonio, San Antonio, TX, 78229, \\nUSA.\\n(24)Department of Neurology, University of Arizona Health Sciences Center, 1501 \\nNorth Campbell Avenue, Tucson, AZ, 85724-5023, USA.\\n(25)Banner Alzheimer\\'s Institute, Phoenix, AZ, 85006, USA.\\n(26)Department of Neurology, Mayo Clinic Arizona, Scottsdale, AZ, 85259, USA.\\n(27)Departments of Psychiatry and Neurology, Taub Institute for Research on \\nAlzheimer\\'s Disease and the Aging Brain, Columbia University, 630 West 168th St \\nP&S Box 16, New York, NY, 10032, USA.\\n(28)Indiana University School of Medicine, 355 West 16th Street, GH 4700 \\nNeurology, Indianapolis, IN, 46202, USA.\\n(29)Department of Physical Medicine and Rehabilitation, Neurology, Cognitive \\nNeurology and Alzheimer\\'s Center, Department of Psychiatry, Feinberg School of \\nMedicine, Northwestern University, 355 E Erie Street, Chicago, IL, 60611-5146, \\nUSA.\\n(30)MRC Prion Unit at University College London, Institute of Prion Diseases, \\nLondon, UK.\\n(31)Department of Pathology, Memory and Aging Center, University of California, \\nSan Francisco, CA, USA.\\n(32)Penn Frontotemporal Degeneration Center, Department of Neurology, Perelman \\nSchool of Medicine at the University of Pennsylvania, Philadelphia, PA, 19104, \\nUSA.\\n(33)Center for Neurodegenerative Disease Research, Department of Pathology and \\nLaboratory Medicine, Perelman School of Medicine at the University of \\nPennsylvania, Philadelphia, PA, 19104, USA.\\n(34)Cerebral Function Unit, Greater Manchester Neurosciences Centre, Salford \\nRoyal Hospital, Salford, UK.\\n(35)Division of Neuroscience and Experimental Psychology, School of Biological \\nSciences, Faculty of Biology, Medicine and Health, University of Manchester, \\nSalford Royal Hospital, Salford, UK.\\n(36)Department of Neurology, Mayo Clinic, Jacksonville, FL, USA.\\n(37)Department of Neurology, Mayo Clinic, Rochester, MN, USA.\\n(38)Central Clinical School and Brain and Mind Centre, The University of Sydney, \\nSydney, 2050, Australia.\\n(39)School of Psychology and Brain and Mind Centre, The University of Sydney, \\nSydney, 2050, Australia.\\n(40)Department of Neurosciences, University of California, San Diego, La Jolla, \\nCA, 92093, USA.\\n(41)Veterans Affairs San Diego Healthcare System, San Diego, CA, 92161, USA.\\n(42)Krembil Discovery Tower, Tanz Centre for Research in Neurodegenerative \\nDisease, University of Toronto, 60 Leonard Av, 4th Floor - 4KD481, Toronto, ON, \\nM5T 0S8, Canada.\\n(43)Sunnybrook Health Sciences Centre, Toronto, ON, M4N 3M5, Canada.\\n(44)Department of Laboratory Medicine and Pathobiology, University of Toronto, \\nToronto, ON, M5S 1A1, Canada.\\n(45)Krembil Neuroscience Center, Movement Disorder\\'s Clinic, Toronto Western \\nHospital, 399 Bathurst Street, Toronto, ON, M5T 2S8, Canada.\\n(46)Department of Neurology, Knight Alzheimer Disease Research Center, \\nWashington University School of Medicine, Saint Louis, MO, 63108, USA.\\n(47)Department of Psychiatry, Knight Alzheimer Disease Research Center, \\nWashington University School of Medicine, Saint Louis, MO, 63108, USA.\\n(48)Department of Pathology and Laboratory Medicine, Indiana University School \\nof Medicine, 635 Barnhill Drive, MS A138, Indianapolis, IN, 46202, USA.\\n(49)Department of Pathology, University of Pittsburgh, Pittsburgh, PA, 15213, \\nUSA.\\n(50)Department of Neurology, University of Pittsburgh, Pittsburgh, PA, 15213, \\nUSA.\\n(51)Civin Laboratory for Neuropathology, Banner Sun Health Research Institute, \\nSun City, AZ, 85351, USA.\\n(52)Department of Psychiatry and Psychotherapy, University Hospital, \\nLudwig-Maximilians-University of Munich, Nussbaumstraße 7, 80336, Munich, \\nGermany.\\n(53)Department of Neurology, Taub Institute, and GH Sergievsky Center, Columbia \\nUniversity Irving Medical Center, 630 West 168th St (P&S Unit 16), New York, NY, \\n10032, USA.\\n(54)Department of Pathology and Taub Institute, Columbia University Irving \\nMedical Center, 630 West 168th St, New York, NY, 10032, USA.\\n(55)UNSW Medicine and NeuRA, Randwick, 2031, Australia.\\n(56)Department of Pathology and Laboratory Medicine and Department of Neurology, \\nEmory University, Atlanta, GA, 30322, USA.\\n(57)Division of Neuroscience and Experimental Psychology, School of Biological \\nSciences, Faculty of Biology, Medicine and Health, University of Manchester, \\nManchester, UK.\\n(58)Dementia Research Centre, Department of Neurodegenerative Disease, UCL Queen \\nSquare Institute of Neurology, London, UK.\\n(59)Department of Pathology and Laboratory Medicine, University of British \\nColumbia, Vancouver, BC, V5Z 1M9, Canada.\\n(60)Department of Neuroscience, Mayo Clinic, 4500 San Pablo Road, Jacksonville, \\nFL, 32224, USA. Rademakers.rosa@mayo.edu.\\n\\nComment in\\n    Nat Rev Neurol. 2019 May;15(5):250-251.\\n\\nFrontotemporal lobar degeneration with neuronal inclusions of the TAR \\nDNA-binding protein 43 (FTLD-TDP) represents the most common pathological \\nsubtype of FTLD. We established the international FTLD-TDP whole-genome \\nsequencing consortium to thoroughly characterize the known genetic causes of \\nFTLD-TDP and identify novel genetic risk factors. Through the study of 1131 \\nunrelated Caucasian patients, we estimated that C9orf72 repeat expansions and \\nGRN loss-of-function mutations account for 25.5% and 13.9% of FTLD-TDP patients, \\nrespectively. Mutations in TBK1 (1.5%) and other known FTLD genes (1.4%) were \\nrare, and the disease in 57.7% of FTLD-TDP patients was unexplained by the known \\nFTLD genes. To unravel the contribution of common genetic factors to the \\nFTLD-TDP etiology in these patients, we conducted a two-stage association study \\ncomprising the analysis of whole-genome sequencing data from 517 FTLD-TDP \\npatients and 838 controls, followed by targeted genotyping of the most \\nassociated genomic loci in 119 additional FTLD-TDP patients and 1653 controls. \\nWe identified three genome-wide significant FTLD-TDP risk loci: one new locus at \\nchromosome 7q36 within the DPP6 gene led by rs118113626 (p value\\u2009=\\u20094.82e\\xa0-\\xa008, \\nOR\\u2009=\\u20092.12), and two known loci: UNC13A, led by rs1297319 (p value\\u2009=\\u20091.27e\\xa0-\\xa008, \\nOR\\u2009=\\u20091.50) and HLA-DQA2 led by rs17219281 (p value\\u2009=\\u20093.22e\\xa0-\\xa008, OR\\u2009=\\u20091.98). \\nWhile HLA represents a locus previously implicated in clinical FTLD and related \\nneurodegenerative disorders, the association signal in our study is independent \\nfrom previously reported associations. Through inspection of our whole-genome \\nsequence data for genes with an excess of rare loss-of-function variants in \\nFTLD-TDP patients (n\\u2009≥\\u20093) as compared to controls (n\\u2009=\\u20090), we further discovered \\na possible role for genes functioning within the TBK1-related immune pathway \\n(e.g., DHX58, TRIM21, IRF7) in the genetic etiology of FTLD-TDP. Together, our \\nstudy based on the largest cohort of unrelated FTLD-TDP patients assembled to \\ndate provides a comprehensive view of the genetic landscape of FTLD-TDP, \\nnominates novel FTLD-TDP risk loci, and strongly implicates the immune pathway \\nin FTLD-TDP pathogenesis.\\n\\nDOI: 10.1007/s00401-019-01962-9\\nPMCID: PMC6533145\\nPMID: 30739198 [Indexed for MEDLINE]\\n\\n\\n44. Mol Cell Probes. 2019 Feb;43:64-71. doi: 10.1016/j.mcp.2018.11.003. Epub 2018\\n Nov 14.\\n\\nGenes and transcription factors related to the adverse effects of maternal type \\nI diabetes mellitus on fetal development.\\n\\nHan S(1), Yang H(2), Han Y(3), Zhang H(4).\\n\\nAuthor information:\\n(1)Department of Clinical Laboratory, Ji\\'nan Central Hospital Affiliated to \\nShandong University, Ji\\'nan, 250013, China.\\n(2)Department of Obstetrics, Ji\\'nan Central Hospital Affiliated to Shandong \\nUniversity, Ji\\'nan, 250013, China. Electronic address: hsy1101@zxyy.cn.\\n(3)Department of Obstetrics, Ji\\'nan Central Hospital Affiliated to Shandong \\nUniversity, Ji\\'nan, 250013, China.\\n(4)Department of Gynecology, Ji\\'nan Central Hospital Affiliated to Shandong \\nUniversity, Ji\\'nan, 250013, China.\\n\\nPURPOSE: Maternal type I diabetes mellitus (T1DM) increases the risk of adverse \\npregnancy outcomes, but the corresponding mechanism is unclear. This study aims \\nto investigate the mechanism underlying the adverse pregnancy outcomes of \\nmaternal T1DM.\\nMETHODS: Gene expression microarray (GSE51546) was down-loaded from the Gene \\nExpression Omnibus. This dataset included 12 umbilical cord samples from the \\nnewborns of T1DM mothers (T1DM group, N\\u202f=\\u202fsix) and non-diabetic mothers (control \\ngroup, N\\u202f=\\u202fsix).\\nRESULTS: Consequently, 1051 differentially expressed genes (DEGs) were found \\nbetween the two groups. The up-regulated DEGs enriched in 30 KEGG pathways. \\nHLA-DPA1, HLA-DMA, HLA-DMB, HLA-DQA1, HLA-DQA2 and HLA-DRA enriched in \"Type I \\ndiabetes mellitus\". This pathway was strongly related to 14 pathways, most of \\nwhich were associated with diseases. Then, a protein-protein interaction network \\nwas constructed, and 45 potential key DEGs were identified. The 45 DEGs enriched \\nin pathways such as \"Rheumatoid arthritis\", \"Chemokine signaling pathway\" and \\n\"Cytokine-cytokine receptor interaction\" (e.g. CXCL12 and CCL5). Transcription \\nfactors (TFs) of key DEGs were predicted, and a TF-DEG regulatory network was \\nconstructed.\\nCONCLUSIONS: Some genes (e.g. CXCL12 and CCL5) and their TFs were significantly \\nand abnormally regulated in the umbilical cord tissue from the pregnancies of \\nT1DM mothers compared to that from non-T1DM mothers.\\n\\nCopyright © 2018. Published by Elsevier Ltd.\\n\\nDOI: 10.1016/j.mcp.2018.11.003\\nPMID: 30447278 [Indexed for MEDLINE]\\n\\n\\n45. Ann Rheum Dis. 2018 Apr;77(4):589-595. doi: 10.1136/annrheumdis-2017-212372. \\nEpub 2018 Jan 27.\\n\\nCross-phenotype analysis of Immunochip data identifies KDM4C as a relevant locus \\nfor the development of systemic vasculitis.\\n\\nOrtiz-Fernández L(1), Carmona FD(2), López-Mejías R(3), González-Escribano \\nMF(4), Lyons PA(5), Morgan AW(6), Sawalha AH(7), Merkel PA, Smith KGC(5), \\nGonzález-Gay MA(3)(8), Martín J(1); Spanish GCA Study Group, UK GCA Consortium, \\nTurkish Takayasu Study Group, Vasculitis Clinical Research Consortium, IgAV \\nStudy Group, AAV Study group.\\n\\nCollaborators: Alegre-Sancho JJ, Blanco R, Callejas JL, Caminal-Montero L, \\nCastañeda S, Cid MC, Corberabellalta M, Miguel E, Díaz-López JB, Escalante B, \\nMateo PF, Fernández-Gutiérrez B, Fernández-Nebro A, García-Villanueva MJ, \\nGómez-Vaquero C, Grau E, Rojas MG, Hernández-Rodríguez J, Hidalgo-Conde A, \\nLópez-Longo FJ, Madroñero Vuelta AB, Marí-Alfonso B, Martínez L, \\nMartínezberriochoa A, Martínez-Taboada VM, Martínez-Zapico A, Miranda-Filloy JA, \\nMonfort J, Morado IC, Narváez J, Ordóñez-Cañizares MC, Ortego-Centeno N, \\nPérez-Conesa M, Prieto-González S, Ramentol-Sintas M, Raya E, Fernández RR, \\nRodríguez-Rodríguez L, Román JA, Sáez-Comet L, Sánchez-Martín J, \\nSánchez-Pernaute O, Solans R, Sopeña B, Tío L, Unzurrunzaga A, Vicente Bhaskar \\nDasgupta EF, Eyre S, Green M, Gough A, Hordon L, Jarret S, Kamath S, Levy S, \\nMackie SL, Patel Y, Pease CT, Taylor JC, Watts R, Worthington J, Alibaz-Oner F, \\nAkar S, Akkoc N, Aksu K, Ates A, Aydin SZ, Bıcakcıgil M, Cefle A, Cobankara V, \\nCoit P, Dalkilic E, Direskeneli H, Duzgun N, Erken E, Fresko I, Inanc M, Kamali \\nS, Karaaslan Y, Karadag O, Kaşifoğlu T, Keser G, Kısacık B, Kiraz S, Onat AM, \\nOnen F, Ozbalkan Z, Ozer HT, Ozturk MA, Pamuk ÖN, Saruhan-Direskeneli G, Seyahi \\nE, Tunc E, Yazici A, Yentür Maximiliano Aragües SP, Blanco R, Blanco-Madrigal \\nJM, Castañeda S, Argila D, Galíndez-Aguirregoikoa E, Genre F, Luque ML, Llorca \\nJ, Mijares V, Miranda-Filloy JA, Parejo AN, Ortego-Centeno N, Pina T, \\nRemuzgo-Martínez S, Rubio E, Sevilla-Perez B, Ubilla B, Brenchley P, Chaundrhy \\nAN, Clayton Panos Deloukas DG, Harper L, Jayne DR, Little MA, Padmanabhan S, \\nPusey CD, Rayner TF, Salamana AD, Savage CO, Trivedi S, Watts RA.\\n\\nAuthor information:\\n(1)Instituto de Parasitologia y Biomedicina Lopez-Neyra, Granada, Spain.\\n(2)Departamento de Genética e Instituto de Biotecnología, Universidad de \\nGranada, Granada, Spain.\\n(3)Epidemiology, Genetics and Atherosclerosis Research Group on Systemic \\nInflammatory Diseases, Rheumatology Division, Hospital Universitario Marqués de \\nValdecilla, IDIVAL, Santander, Spain.\\n(4)Department of Immunology, Hospital Universitario Virgen del Rocío (IBiS,CSIC, \\nUS), Sevilla, Spain.\\n(5)Department of Medicine, University of Cambridge School of Clinical Medicine, \\nCambridge Biomedical Campus, Cambridge, UK.\\n(6)Leeds Institute of Rheumatic and Musculoskeletal Medicine, University of \\nLeeds, and NIHR Leeds Biomedical Research Centre, Leeds Teaching Hospitals NHS \\nTrust, Leeds, UK.\\n(7)Division of Rheumatology, Department of Internal Medicine, University of \\nMichigan, Ann Arbor, Michigan, USA.\\n(8)School of Medicine, University of Cantabria, Santander, Spain.\\n\\nErratum in\\n    Ann Rheum Dis. 2018 Jun;77(6):950.\\n\\nOBJETIVE: Systemic vasculitides represent a heterogeneous group of rare complex \\ndiseases of the blood vessels with a poorly understood aetiology. To investigate \\nthe shared genetic component underlying their predisposition, we performed the \\nfirst cross-phenotype meta-analysis of genetic data from different clinically \\ndistinct patterns of vasculitis.\\nMETHODS: Immunochip genotyping data from 2465 patients diagnosed with giant cell \\narteritis, Takayasu\\'s arteritis, antineutrophil cytoplasmic antibody-associated \\nvasculitis or IgA vasculitis as well as 4632 unaffected controls were analysed \\nto identify common susceptibility loci for vasculitis development. The possible \\nfunctional consequences of the associated variants were interrogated using \\npublicly available annotation data.\\nRESULTS: The strongest association signal corresponded with an intergenic \\npolymorphism located between HLA-DQB1 and HLA-DQA2 (rs6932517, P=4.16E-14, \\nOR=0.74). This single nucleotide polymorphism is in moderate linkage \\ndisequilibrium with the disease-specific human leucocyte antigen (HLA) class II \\nassociations of each type of vasculitis and could mark them. Outside the HLA \\nregion, we identified the KDM4C gene as a common risk locus for vasculitides \\n(highest peak rs16925200, P=6.23E-07, OR=1.75). This gene encodes a histone \\ndemethylase involved in the epigenetic control of gene expression.\\nCONCLUSIONS: Through a combined analysis of Immunochip data, we have identified \\nKDM4C as a new risk gene shared between systemic vasculitides, consistent with \\nthe increasing evidences of the crucial role that the epigenetic mechanisms have \\nin the development of complex immune-mediated conditions.\\n\\n© Article author(s) (or their employer(s) unless otherwise stated in the text of \\nthe article) 2018. All rights reserved. No commercial use is permitted unless \\notherwise expressly granted.\\n\\nDOI: 10.1136/annrheumdis-2017-212372\\nPMCID: PMC5849568\\nPMID: 29374629 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing interests: None declared.\\n\\n\\n46. Sci Rep. 2018 Jan 18;8(1):1069. doi: 10.1038/s41598-017-18241-w.\\n\\nGenome-wide association study of self-reported food reactions in Japanese \\nidentifies shrimp and peach specific loci in the HLA-DR/DQ gene region.\\n\\nKhor SS(1), Morino R(2), Nakazono K(3), Kamitsuji S(3), Akita M(2), Kawajiri \\nM(2), Yamasaki T(4), Kami A(2), Hoshi Y(4), Tada A(2), Ishikawa K(2), Hine M(5), \\nKobayashi M(5), Kurume N(5), Kamatani N(3), Tokunaga K(1), Johnson TA(6).\\n\\nAuthor information:\\n(1)Department of Human Genetics, Graduate School of Medicine, The University of \\nTokyo, Tokyo, 113-0033, Japan.\\n(2)EverGene Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\\n(3)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan.\\n(4)Life Science Group, Healthcare Division, Department of Healthcare Business, \\nMTI Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\\n(5)LunaLuna Division, Department of Healthcare Business, MTI Ltd., Shinjuku-ku, \\nTokyo, 163-1435, Japan.\\n(6)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan. \\ntodd.johnson@stagen.co.jp.\\n\\nFood allergy is an increasingly important health problem in the world. Several \\ngenome-wide association studies (GWAS) focused on European ancestry samples have \\nidentified food allergy-specific loci in the HLA class II region. We conducted \\nGWAS of self-reported reactivity with common foods using the data from 11011 \\nJapanese women and identified shrimp and peach allergy-specific loci in the \\nHLA-DR/DQ gene region tagged by rs74995702 (P\\u2009=\\u20096.30\\u2009×\\u200910-17, OR\\u2009=\\u20091.91) and \\nrs28359884 (P\\u2009=\\u20092.3\\u2009×\\u200910-12, OR\\u2009=\\u20091.80), respectively. After HLA imputation \\nusing a Japanese population-specific reference, the most strongly associated \\nhaplotype was HLA-DRB1*04:05-HLA-DQB1*04:01 for shrimp allergy \\n(P\\u2009=\\u20093.92\\u2009×\\u200910-19, OR\\u2009=\\u20091.99) and HLA-DRB1*09:01-HLA-DQB1*03:03 for peach \\nallergy (P\\u2009=\\u20091.15\\u2009×\\u200910-7, OR\\u2009=\\u20091.68). Additionally, both allergies\\' associated \\nvariants were eQTLs for several HLA genes, with HLA-DQA2 the single eQTL gene \\nshared between the two traits. Our study suggests that allergy to certain foods \\nmay be related to genetic differences that tag both HLA alleles having \\nparticular epitope binding specificities as well as variants modulating \\nexpression of particular HLA genes. Investigating this further could increase \\nour understanding of food allergy aetiology and potentially lead to better \\ntherapeutic strategies for allergen immunotherapies.\\n\\nDOI: 10.1038/s41598-017-18241-w\\nPMCID: PMC5773682\\nPMID: 29348432 [Indexed for MEDLINE]\\n\\nConflict of interest statement: T.A.J., S.K., N.Ka., S.K. are employees of \\nStaGen Co. Ltd. R.M., M.A., M.Ka., T.Y., A.K., M.T., Y.H., A.T., K.I., M.H., \\nM.Ko., and N.Ku. are employees of EverGene/MTI. EverGene/MTI has applied for \\npatents related to the food allergy associations described in this report. N.Ka. \\nis also Chairman of StaGen Co. Ltd. and a Director of the Tsukuba International \\nClinical Pharmacology Clinic. S.S.K. and K.T. declare no competing financial \\ninterests.\\n\\n\\n47. Trop Anim Health Prod. 2017 Apr;49(4):867-873. doi:\\n10.1007/s11250-017-1277-9.  Epub 2017 Mar 23.\\n\\nGenetic appraisal of serological response post vaccination against \\nenterotoxaemia (ET) in Malpura and Avikalin sheep.\\n\\nGowane GR(1), Akram N(2), Prince LL(2), Prakash V(2), Kumar A(2).\\n\\nAuthor information:\\n(1)ICAR-Central Sheep & Wool Research Institute, Avikanagar via, Jaipur, \\nRajasthan, 304501, India. gopalgowane@gmail.com.\\n(2)ICAR-Central Sheep & Wool Research Institute, Avikanagar via, Jaipur, \\nRajasthan, 304501, India.\\n\\nEnterotoxaemia (ET) is a fatal enteric disease of small ruminants attributable \\nto a toxigenic type of Clostridium perfringens. The key strategy for prevention \\nof ET is the management and vaccination. Present study aimed at identifying the \\nsources of variation for ET vaccine response especially against epsilon toxin in \\n173 sheep that included 83 Avikalin and 90 Malpura lambs raised at the institute \\nflock in the semi-arid region of India. The mean age at vaccination was 90\\xa0days. \\nSera were tested by blocking ELISA. Study showed significant variability for \\nresponse to ET vaccine. 5.2% animals had + positivity, 20.8% animals had ++ \\npositivity, 51.4% animals had +++ positivity and 22.5% animals had ++++ \\npositivity. Amongst environmental determinants, breed, season, sex and age at \\nvaccination proved to be non-significant sources of variation (P\\xa0>\\xa00.05). MHC \\ngenotypes with DRB1 gene and DQA2 genes also revealed non-significant \\nassociation with ET vaccine response; however, a trend of decreasing PI values \\nwith increasing ranks was observed. Study revealed strong response of epsilon \\ntoxin along with complexity of the ET vaccine response as phenotype to be \\nexplained by genetic and non-genetic factors. The importance of better \\nmanagement practices and vaccination is suggested for preventive measures.\\n\\nDOI: 10.1007/s11250-017-1277-9\\nPMID: 28337572 [Indexed for MEDLINE]\\n\\n\\n48. Sci Rep. 2015 Dec 21;5:18855. doi: 10.1038/srep18855.\\n\\nNetwork-assisted analysis of primary Sjögren\\'s syndrome GWAS data in Han \\nChinese.\\n\\nFang K(1), Zhang K(1), Wang J(1).\\n\\nAuthor information:\\n(1)Key Laboratory of Mental Health, Institute of Psychology, Chinese Academy of \\nSciences, Beijing 100101, China.\\n\\nPrimary Sjögren\\'s syndrome (pSS) is a complex autoimmune disorder. So far, \\ngenetic research in pSS has lagged far behind and the underlying biological \\nmechanism is unclear. Further exploring existing genome-wide association study \\n(GWAS) data is urgently expected to uncover disease-related gene combination \\npatterns. Herein, we conducted a network-based analysis by integrating pSS GWAS \\nin Han Chinese with a protein-protein interactions network to identify pSS \\ncandidate genes. After module detection and evaluation, 8 dense modules covering \\n40 genes were obtained for further functional annotation. Additional 31 MHC \\ngenes with significant gene-level P-values (sigMHC-gene) were also remained. The \\ncombined module genes and sigMHC-genes, a total of 71 genes, were denoted as pSS \\ncandidate genes. Of these pSS candidates, 14 genes had been reported to be \\nassociated with any of pSS, RA, and SLE, including STAT4, GTF2I, HLA-DPB1, \\nHLA-DRB1, PTTG1, HLA-DQB1, MBL2, TAP2, CFLAR, NFKBIE, HLA-DRA, APOM, HLA-DQA2 \\nand NOTCH4. This is the first report of the network-assisted analysis for pSS \\nGWAS data to explore combined gene patterns associated with pSS. Our study \\nsuggests that network-assisted analysis is a useful approach to gaining further \\ninsights into the biology of associated genes and providing important clues for \\nfuture research into pSS etiology.\\n\\nDOI: 10.1038/srep18855\\nPMCID: PMC4685393\\nPMID: 26686423 [Indexed for MEDLINE]\\n\\n\\n49. PLoS One. 2015 Dec 16;10(12):e0144719. doi: 10.1371/journal.pone.0144719. \\neCollection 2015.\\n\\nEvidence for Association of Cell Adhesion Molecules Pathway and NLGN1 \\nPolymorphisms with Schizophrenia in Chinese Han Population.\\n\\nZhang Z(1)(2), Yu H(3)(4), Jiang S(1)(2), Liao J(1)(2), Lu T(1)(2), Wang \\nL(1)(2), Zhang D(1)(2)(4)(5), Yue W(1)(2).\\n\\nAuthor information:\\n(1)Institute of Mental Health, The Sixth Hospital, Peking University, Beijing, \\nChina.\\n(2)Key Laboratory of Mental Health, Ministry of Health & National Clinical \\nResearch Center for Mental Disorders (Peking University), Beijing, China.\\n(3)School of Life Sciences, Tsinghua University, Beijing, China.\\n(4)Tsinghua University-Peking University Joint Center for Life Sciences, \\nBeijing, China.\\n(5)PKU-IDG/McGovern Institute for Brain Research, Peking University, Beijing, \\nChina.\\n\\nMultiple risk variants of schizophrenia have been identified by Genome-wide \\nassociation studies (GWAS). As a complement for GWAS, previous pathway-based \\nanalysis has indicated that cell adhesion molecules (CAMs) pathway might be \\ninvolved in the pathogenesis of schizophrenia. However, less replication studies \\nhave been reported. Our objective was to investigate the association between \\nCAMs pathway and schizophrenia in the Chinese Han population. We first performed \\na pathway analysis utilizing our previous GWAS data. The CAMs pathway (hsa04514) \\nwas significantly associated with schizophrenia using hybrid gene set-based test \\n(P = 1.03×10-10) and hypergeometric test (P = 5.04×10-6). Moreover, 12 genes \\n(HLA-A, HLA-C, HLA-DOB, HLA-DPB1, HLA-DQA2, HLA-DRB1, MPZ, CD276, NLGN1, NRCAM, \\nCLDN1 and ICAM3) were modestly significantly associated with schizophrenia \\n(P<0.01). Then, we selected one promising gene neuroligin 1 (NLGN1) to further \\ninvestigate the association between eight significant SNPs and schizophrenia in \\nan independent sample (1814 schizophrenia cases and 1487 healthy controls). Our \\nstudy showed that seven SNPs of NLGN1 and two haplotype blocks were \\nsignificantly associated with schizophrenia. This association was confirmed by \\nthe results of combined analysis. Among them, SNP rs9835385 had the most \\nsignificant association with schizophrenia (P = 2.83×10-7). Furthermore, in \\nsilico analysis we demonstrated that NLGN1 is preferentially expressed in human \\nbrain and SNP rs1488547 was related to the expression level. We validated the \\nassociation of CAMs pathway with schizophrenia in pathway-level and identified \\none susceptibility gene NLGN1. Further investigation of the roles of CAMs \\npathway in the pathogenesis of schizophrenia is warranted.\\n\\nDOI: 10.1371/journal.pone.0144719\\nPMCID: PMC4682938\\nPMID: 26674772 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n50. Arthritis Rheumatol. 2016 Apr;68(4):932-43. doi: 10.1002/art.39504.\\n\\nGenome-Wide Association Study in an Amerindian Ancestry Population Reveals Novel \\nSystemic Lupus Erythematosus Risk Loci and the Role of European Admixture.\\n\\nAlarcón-Riquelme ME(1), Ziegler JT(2), Molineros J(1), Howard TD(2), \\nMoreno-Estrada A(3), Sánchez-Rodríguez E(4), Ainsworth HC(2), Ortiz-Tello P(3), \\nComeau ME(2), Rasmussen A(1), Kelly JA(1), Adler A(1), Acevedo-Vázquez EM(5), \\nCucho-Venegas JM(5), García-De la Torre I(6), Cardiel MH(7), Miranda P(8), \\nCatoggio LJ(9), Maradiaga-Ceceña M(10), Gaffney PM(1), Vyse TJ(11), Criswell \\nLA(12), Tsao BP(13), Sivils KL(1), Bae SC(14), James JA(15), Kimberly RP(16), \\nKaufman KM(17), Harley JB(17), Esquivel-Valerio JA(18), Moctezuma JF(19), García \\nMA(20), Berbotto GA(21), Babini AM(22), Scherbarth H(23), Toloza S(24), Baca \\nV(25), Nath SK(1), Aguilar Salinas C(26), Orozco L(27), Tusié-Luna T(28), \\nZidovetzki R(29), Pons-Estel BA(30), Langefeld CD(2), Jacob CO(31).\\n\\nAuthor information:\\n(1)Oklahoma Medical Research Foundation, Oklahoma City.\\n(2)Wake Forest School of Medicine, Winston-Salem, North Carolina.\\n(3)Stanford University School of Medicine, Stanford, California, and Laboratorio \\nNacional de Genómica para la Biodiversidad, Centro de Investigación y de \\nEstudios Avanzados del Instituto Politécnico Nacional, Irapuato, Mexico.\\n(4)Icahn School of Medicine at Mount Sinai, New York, New York.\\n(5)Hospital Nacional Guillermo Almenara Irigoyen, Lima, Peru.\\n(6)Hospital General de Occidente, Zapopan, Mexico.\\n(7)Centro de Investigación Clínica de Morelia, Morelia, Mexico.\\n(8)Centro de Estudios Reumatológicos, Santiago, Chile.\\n(9)Hospital Italiano de Buenos Aires, Buenos Aires, Argentina.\\n(10)Hospital General de Culiacán, Culiacán, Mexico.\\n(11)King\\'s College London, London, UK.\\n(12)University of California, San Francisco.\\n(13)University of California, Los Angeles.\\n(14)Hanyang University Hospital for Rheumatic Diseases, Seoul, Republic of \\nKorea.\\n(15)Oklahoma Medical Research Foundation and University of Oklahoma Health \\nSciences Center, Oklahoma City.\\n(16)University of Alabama at Birmingham.\\n(17)Cincinnati Children\\'s Hospital Medical Center, Cincinnati, Ohio.\\n(18)Hospital Universitario Dr. José Eleuterio González Universidad Autonoma de \\nNuevo León, Monterrey, Mexico.\\n(19)Hospital General de México, Mexico City, Mexico.\\n(20)Hospital Interzonal General de Agudos General San Martin, La Plata, \\nArgentina.\\n(21)Hospital Escuela Eva Perón, Granadero Baigorria, Argentina.\\n(22)Hospital Italiano de Córdoba, Córdoba, Argentina.\\n(23)Hospital Interzonal General de Agudos Oscar E. Alende, Mar del Plata, \\nArgentina.\\n(24)Hospital Interzonal San Juan Bautista, San Fernando del Valle de Catamarca, \\nArgentina.\\n(25)Hospital de Peditaria, Centro Médico Nacional Siglo XXI, Instituto Mexicano \\ndel Seguro Social, Mexico City, Mexico.\\n(26)Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico \\nCity, Mexico.\\n(27)Instituto Nacional de Medicina Genómica, Mexico City, Mexico.\\n(28)Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán and \\nInstituto de Investigaciones Biomédicas de la Universidad Nacional Autónoma de \\nMéxico, Mexico City, Mexico.\\n(29)University of California, Riverside.\\n(30)Sanatorio Parque, Rosario, Argentina.\\n(31)University of Southern California School of Medicine, Los Angeles.\\n\\nOBJECTIVE: Systemic lupus erythematosus (SLE) is a chronic autoimmune disease \\nwith a strong genetic component. We undertook the present work to perform the \\nfirst genome-wide association study on individuals from the Americas who are \\nenriched for Native American heritage.\\nMETHODS: We analyzed 3,710 individuals from the US and 4 countries of Latin \\nAmerica who were diagnosed as having SLE, and healthy controls. Samples were \\ngenotyped with HumanOmni1 BeadChip. Data on out-of-study controls genotyped with \\nHumanOmni2.5 were also included. Statistical analyses were performed using \\nSNPtest and SNPGWA. Data were adjusted for genomic control and false discovery \\nrate. Imputation was performed using Impute2 and, for classic HLA alleles, \\nHiBag. Odds ratios (ORs) and 95% confidence intervals (95% CIs) were calculated.\\nRESULTS: The IRF5-TNPO3 region showed the strongest association and largest OR \\nfor SLE (rs10488631: genomic control-adjusted P [Pgcadj ] = 2.61 × 10(-29), OR \\n2.12 [95% CI 1.88-2.39]), followed by HLA class II on the DQA2-DQB1 loci \\n(rs9275572: Pgcadj \\u2009=\\u20091.11 × 10(-16), OR 1.62 [95% CI 1.46-1.80] and rs9271366: \\nPgcadj \\u2009=\\u20096.46 × 10(-12), OR 2.06 [95% CI 1.71-2.50]). Other known SLE loci \\nfound to be associated in this population were ITGAM, STAT4, TNIP1, NCF2, and \\nIRAK1. We identified a novel locus on 10q24.33 (rs4917385: Pgcadj \\u2009=\\u20091.39 × \\n10(-8)) with an expression quantitative trait locus (eQTL) effect (Peqtl \\u2009=\\u20098.0 \\n× 10(-37) at USMG5/miR1307), and several new suggestive loci. SLE risk loci \\npreviously identified in Europeans and Asians were corroborated. Local ancestry \\nestimation showed that the HLA allele risk contribution is of European ancestral \\norigin. Imputation of HLA alleles suggested that autochthonous Native American \\nhaplotypes provide protection against development of SLE.\\nCONCLUSION: Our results demonstrate that studying admixed populations provides \\nnew insights in the delineation of the genetic architecture that underlies \\nautoimmune and complex diseases.\\n\\n© 2016, American College of Rheumatology.\\n\\nDOI: 10.1002/art.39504\\nPMCID: PMC4829354\\nPMID: 26606652 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Conflicts of Interest The authors declare no \\nconflicts of interest\\n\\n\\n51. Lancet Respir Med. 2015 Oct;3(10):769-81. doi: 10.1016/S2213-2600(15)00283-0.\\n Epub 2015 Sep 27.\\n\\nNovel insights into the genetics of smoking behaviour, lung function, and \\nchronic obstructive pulmonary disease (UK BiLEVE): a genetic association study \\nin UK Biobank.\\n\\nWain LV(1), Shrine N(1), Miller S(2), Jackson VE(1), Ntalla I(1), Soler Artigas \\nM(1), Billington CK(2), Kheirallah AK(2), Allen R(1), Cook JP(1), Probert K(2), \\nObeidat M(3), Bossé Y(4), Hao K(5), Postma DS(6), Paré PD(3), Ramasamy A(7); UK \\nBrain Expression Consortium (UKBEC); Mägi R(8), Mihailov E(8), Reinmaa E(8), \\nMelén E(9), O\\'Connell J(10), Frangou E(11), Delaneau O(12); OxGSK Consortium; \\nFreeman C(13), Petkova D(13), McCarthy M(14), Sayers I(2), Deloukas P(15), \\nHubbard R(16), Pavord I(17), Hansell AL(18), Thomson NC(19), Zeggini E(20), \\nMorris AP(21), Marchini J(10), Strachan DP(22), Tobin MD(23), Hall IP(24).\\n\\nCollaborators: Hardy JA, Weale ME, Ryten M, Smith C, Walker R, Botía J, \\nVandrocova J, Guelfi S, D\\'Sa K, Matarin M, Varghese V, Trabzuni D, Ramasamy A, \\nForabosco P, Liu JZ, Tozzi F, Waterworth DM, Pillai SG, Muglia P, Middleton L, \\nBerrettini W, Knouff CW, Yuan X, Waeber G, Vollenweider P, Preisig M, Wareham \\nNJ, Zhao JH, Loos RJ, Barroso I, Khaw KT, Grundy S, Barter P, Mahley R, \\nKesaniemi A, McPherson R, Vincent JB, Strauss J, Kennedy JL, Farmer A, McGuffin \\nP, Day R, Matthews K, Bakke P, Gulsvik A, Lucae S, Ising M, Brueckl T, Horstmann \\nS, Wichmann HE, Rawal R, Dahmen N, Lamina C, Polasek O, Zgaga L, Huffman J, \\nCampbell S, Kooner J, Chambers JC, Burnett MS, Devaney JM, Pichard AD, Kent KM, \\nSatler L, Lindsay JM, Waksman R, Epstein S, Wilson JF, Wild SH, Campbell H, \\nVitart V, Reilly MP, Li M, Qu L, Wilensky R, Matthai W, Hakonarson HH, Rader DJ, \\nEllinghaus D, Lieb W, Franke A, Uda M, Terracciano A, Xiao X, Busonero F, Scheet \\nP, Schlessinger D, St Clair D, Rujescu D, Abecasis GR, Grabe HJ, Teumer A, \\nVölzke H, Petersmann A, John U, Rudan I, Hayward C, Wright AF, Kolcic I, Wright \\nBJ, Thompson JR, Balmforth AJ, Hall AS, Samani NJ, Anderson CA, Ahmad T, Mathew \\nCG, Parkes M, Satsangi J, Caulfield M, Munroe PB, Farrall M, Dominiczak A, \\nWorthington J, Thomson W, Eyre S, Barton A, Mooser V, Francks C, Marchini J.\\n\\nAuthor information:\\n(1)Department of Health Sciences, University of Leicester, Leicester, UK.\\n(2)Division of Respiratory Medicine, Queen\\'s Medical Centre, University of \\nNottingham, Nottingham, UK.\\n(3)University of British Columbia Centre for Heart Lung Innovation, St Paul\\'s \\nHospital, Vancouver, BC, Canada.\\n(4)Institut universitaire de cardiologie et de pneumologie de Québec, Department \\nof Molecular Medicine, Laval University, Québec, QC, Canada.\\n(5)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \\nMount Sinai, New York, NY, USA; Icahn Institute of Genomics and Multiscale \\nBiology, Icahn School of Medicine at Mount Sinai, New York, NY, USA; Department \\nof Respiratory Medicine, Shanghai Tenth People\\'s Hospital, Tongji University, \\nShanghai, China.\\n(6)Department of Pulmonary Medicine and Tuberculosis, University of Groningen, \\nUniversity Medical Center Groningen, Groningen, Netherlands.\\n(7)Department of Molecular Neuroscience, UCL Institute of Neurology, London, UK; \\nDepartment of Medical and Molecular Genetics, King\\'s College London, London, UK; \\nJenner Institute, University of Oxford, Oxford, UK.\\n(8)Estonian Genome Center, University of Tartu, Tartu, Estonia.\\n(9)Institute of Environmental Medicine, Karolinska Institutet and Sachs\\' \\nChildren\\'s Hospital, Stockholm, Sweden.\\n(10)Department of Statistics, University of Oxford, Oxford, UK; Wellcome Trust \\nCentre for Human Genetics, University of Oxford, Oxford, UK.\\n(11)Department of Statistics, University of Oxford, Oxford, UK; Centre for \\nStatistics in Medicine, Nuffield Department of Orthopaedics, Rheumatology and \\nMusculoskeletal Sciences, University of Oxford, Oxford, UK.\\n(12)Department of Statistics, University of Oxford, Oxford, UK; Department of \\nGenetic Medicine and Development, University of Geneva Medical School, Geneva, \\nSwitzerland.\\n(13)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.\\n(14)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK; \\nOxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, \\nOxford, UK.\\n(15)William Harvey Research Institute, Barts and The London School of Medicine \\nand Dentistry, Queen Mary University London, London, UK; Princess Al-Jawhara \\nAl-Brahim Centre of Excellence in Research of Hereditary Disorders (PACER-HD), \\nKing Abdulaziz University, Jeddah, Saudi Arabia.\\n(16)Faculty of Medicine and Health Sciences, School of Medicine, University of \\nNottingham, Nottingham, UK.\\n(17)Respiratory Medicine, University of Oxford, Oxford, UK.\\n(18)UK Small Area Health Statistics Unit, MRC-PHE Centre for Environment and \\nHealth, School of Public Health, Imperial College London, London, UK; Imperial \\nCollege Healthcare NHS Trust, St Mary\\'s Hospital, Paddington, London, UK.\\n(19)Institute of Infection, Immunity and Inflammation, University of Glasgow, \\nGlasgow, UK.\\n(20)Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.\\n(21)Department of Biostatistics, University of Liverpool, Liverpool, UK.\\n(22)Population Health Research Institute, St George\\'s, University of London, \\nLondon, UK.\\n(23)Department of Health Sciences, University of Leicester, Leicester, UK; \\nNational Institute for Health Research, Leicester Respiratory Biomedical \\nResearch Unit, Glenfield Hospital, Leicester, UK. Electronic address: \\nmt47@leicester.ac.uk.\\n(24)Division of Respiratory Medicine, Queen\\'s Medical Centre, University of \\nNottingham, Nottingham, UK. Electronic address: ian.hall@nottingham.ac.uk.\\n\\nErratum in\\n    Lancet Respir Med. 2016 Jan;4(1):e4.\\n    Lancet Respir Med. 2016 Jan;4(1):e4.\\n\\nComment in\\n    Lancet Respir Med. 2015 Oct;3(10):737-9.\\n    Lancet Respir Med. 2015 Dec;3(12):e42.\\n    Lancet Respir Med. 2015 Dec;3(12):e42-3.\\n\\nBACKGROUND: Understanding the genetic basis of airflow obstruction and smoking \\nbehaviour is key to determining the pathophysiology of chronic obstructive \\npulmonary disease (COPD). We used UK Biobank data to study the genetic causes of \\nsmoking behaviour and lung health.\\nMETHODS: We sampled individuals of European ancestry from UK Biobank, from the \\nmiddle and extremes of the forced expiratory volume in 1 s (FEV1) distribution \\namong heavy smokers (mean 35 pack-years) and never smokers. We developed a \\ncustom array for UK Biobank to provide optimum genome-wide coverage of common \\nand low-frequency variants, dense coverage of genomic regions already implicated \\nin lung health and disease, and to assay rare coding variants relevant to the UK \\npopulation. We investigated whether there were shared genetic causes between \\ndifferent phenotypes defined by extremes of FEV1. We also looked for novel \\nvariants associated with extremes of FEV1 and smoking behaviour and assessed \\nregions of the genome that had already shown evidence for a role in lung health \\nand disease. We set genome-wide significance at p<5\\u2008×\\u200810(-8).\\nFINDINGS: UK Biobank participants were recruited from March 15, 2006, to July 7, \\n2010. Sample selection for the UK BiLEVE study started on Nov 22, 2012, and was \\ncompleted on Dec 20, 2012. We selected 50,008 unique samples: 10,002 individuals \\nwith low FEV1, 10,000 with average FEV1, and 5002 with high FEV1 from each of \\nthe heavy smoker and never smoker groups. We noted a substantial sharing of \\ngenetic causes of low FEV1 between heavy smokers and never smokers \\n(p=2.29\\u2008×\\u200810(-16)) and between individuals with and without doctor-diagnosed \\nasthma (p=6.06\\u2008×\\u200810(-11)). We discovered six novel genome-wide significant \\nsignals of association with extremes of FEV1, including signals at four novel \\nloci (KANSL1, TSEN54, TET2, and RBM19/TBX5) and independent signals at two \\npreviously reported loci (NPNT and HLA-DQB1/HLA-DQA2). These variants also \\nshowed association with COPD, including in individuals with no history of \\nsmoking. The number of copies of a 150 kb region containing the 5\\' end of \\nKANSL1, a gene that is important for epigenetic gene regulation, was associated \\nwith extremes of FEV1. We also discovered five new genome-wide significant \\nsignals for smoking behaviour, including a variant in NCAM1 (chromosome 11) and \\na variant on chromosome 2 (between TEX41 and PABPC1P2) that has a trans effect \\non expression of NCAM1 in brain tissue.\\nINTERPRETATION: By sampling from the extremes of the lung function distribution \\nin UK Biobank, we identified novel genetic causes of lung function and smoking \\nbehaviour. These results provide new insight into the specific mechanisms \\nunderlying airflow obstruction, COPD, and tobacco addiction, and show \\nsubstantial shared genetic architecture underlying airflow obstruction across \\nindividuals, irrespective of smoking behaviour and other airway disease.\\nFUNDING: Medical Research Council.\\n\\nCopyright © 2015 Wain et al. Open Access article distributed under the terms of \\nCC BY. Published by Elsevier Ltd.. All rights reserved.\\n\\nDOI: 10.1016/S2213-2600(15)00283-0\\nPMCID: PMC4593935\\nPMID: 26423011 [Indexed for MEDLINE]\\n\\n\\n52. Gut. 2016 May;65(5):788-96. doi: 10.1136/gutjnl-2014-308617. Epub 2015 Mar 2.\\n\\nDeep resequencing of 131 Crohn\\'s disease associated genes in pooled DNA \\nconfirmed three reported variants and identified eight novel variants.\\n\\nHong SN(1), Park C(2), Park SJ(3), Lee CK(4), Ye BD(5), Kim YS(6), Lee S(7), \\nChae J(2), Kim JI(8), Kim YH(1); IBD Study Group of the Korean Association for \\nthe Study of Intestinal Diseases (KASID).\\n\\nAuthor information:\\n(1)Department of Medicine, Samsung Medical Center, Sungkyunkwan University \\nSchool of Medicine, Seoul, Korea.\\n(2)Department of Biomedical Sciences, Seoul National University Graduate School, \\nSeoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \\nUniversity College of Medicine, Seoul, Korea.\\n(3)Department of Internal Medicine and Institute of Gastroenterology, Yonsei \\nUniversity College of Medicine, Seoul, Korea.\\n(4)Department of Internal Medicine, Kyung Hee University School of Medicine, \\nSeoul, Korea.\\n(5)Department of Gastroenterology and Inflammatory Bowel Disease Center, Asan \\nMedical Centre, University of Ulsan College of Medicine.\\n(6)Department of Internal Medicine, Seoul Paik Hospital, Inje University College \\nof Medicine, Seoul, Korea.\\n(7)Department of Biomedical Sciences, Seoul National University Graduate School, \\nSeoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \\nUniversity College of Medicine, Seoul, Korea Medical Research Center, Genomic \\nMedicine Institute (GMI), Seoul National University, Seoul, Korea.\\n(8)Department of Biomedical Sciences, Seoul National University Graduate School, \\nSeoul, Korea Department of Biochemistry and Molecular Biology, Seoul National \\nUniversity College of Medicine, Seoul, Korea Medical Research Center, Genomic \\nMedicine Institute (GMI), Seoul National University, Seoul, Korea Cancer \\nResearch Institute, Seoul National University College of Medicine, Seoul, Korea.\\n\\nOBJECTIVE: Genome wide association studies (GWAS) and meta-analyses for Crohn\\'s \\ndisease (CD) have not fully explained the heritability of CD, suggesting that \\nadditional loci are yet to be found and that the known loci may contain high \\neffect rare risk variants that have thus far gone undetected by GWAS. While the \\ncost of deep sequencing remains too high to analyse many samples, targeted \\nsequencing of pooled DNA samples allows the efficient and cost effective capture \\nof all variations in a target region.\\nDESIGN: We performed pooled sequencing in 500 Korean CD cases and 1000 controls \\nto evaluate the coding exon and 5\\' and 3\\' untranslated regions of 131 CD \\nassociated genes. The identified genetic variants were validated using \\ngenotyping in an independent set of 500 CD cases and 1000 controls.\\nRESULTS: Pooled sequencing identified 30 common/low single nucleotide variants \\n(SNVs) in 12 genes and 3 rare SNVs in 3 genes. Our results confirmed a \\nsignificant association of CD with the following previously reported risk loci: \\nrs3810936 in TNFSF15 (OR=1.83, p<2.2×10(-16)), rs76418789 in IL23R (OR=0.47, \\np=1.14×10(-8)) and rs2241880 in ATG16L1 (OR=1.30, p=5.28×10(-6)). In addition, \\nnovel loci were identified in TNFSF8 (rs3181374, OR=1.53, p=1.03×10(-14)), BTNL2 \\n(rs28362680, OR=1.47, p=9.67×10(-11)), HLA-DQA2 (rs3208181, OR=1.36, \\np=4.66×10(-6)), STAT3 (rs1053004, OR=1.29, p=2.07×10(-5)), NFKBIA (rs2273650, \\nOR=0.80, p=3.93×10(-4)), NKX2-3 (rs888208, OR=0.82, p=6.37×10(-4)) and DNAH12 \\n(rs4462937, OR=1.13, p=3.17×10(-2)). A novel rare SNV, rs200735402 in CARD9, was \\nshown to have a protective effect (OR=0.09, p=5.28×10(-5)).\\nCONCLUSIONS: Our deep resequencing of 131 CD associated genes confirmed 3 \\nreported risk loci and identified 8 novel risk loci for CD in Koreans, providing \\nnew insights into the genetic architecture of CD.\\n\\nPublished by the BMJ Publishing Group Limited. For permission to use (where not \\nalready granted under a licence) please go to \\nhttp://www.bmj.com/company/products-services/rights-and-licensing/\\n\\nDOI: 10.1136/gutjnl-2014-308617\\nPMID: 25731871 [Indexed for MEDLINE]\\n\\n\\n53. PLoS Genet. 2014 Oct 9;10(10):e1004637. doi: 10.1371/journal.pgen.1004637. \\neCollection 2014 Oct.\\n\\nDominant sequences of human major histocompatibility complex conserved extended \\nhaplotypes from HLA-DQA2 to DAXX.\\n\\nLarsen CE(1), Alford DR(2), Trautwein MR(2), Jalloh YK(2), Tarnacki JL(2), \\nKunnenkeri SK(2), Fici DA(3), Yunis EJ(4), Awdeh ZL(3), Alper CA(5).\\n\\nAuthor information:\\n(1)Program in Cellular and Molecular Medicine, Boston Children\\'s Hospital, \\nBoston, Massachusetts, United States of America; Department of Medicine, Harvard \\nMedical School, Boston, Massachusetts, United States of America.\\n(2)Program in Cellular and Molecular Medicine, Boston Children\\'s Hospital, \\nBoston, Massachusetts, United States of America.\\n(3)Program in Cellular and Molecular Medicine, Boston Children\\'s Hospital, \\nBoston, Massachusetts, United States of America; Pulsar Clinical Technologies, \\nCambridge, Massachusetts, United States of America.\\n(4)Department of Cancer Immunology and AIDS, Dana-Farber Cancer Institute, \\nBoston, Massachusetts, United States of America; Department of Pathology, \\nHarvard Medical School, Boston, Massachusetts, United States of America.\\n(5)Program in Cellular and Molecular Medicine, Boston Children\\'s Hospital, \\nBoston, Massachusetts, United States of America; Department of Pediatrics, \\nHarvard Medical School, Boston, Massachusetts, United States of America.\\n\\nWe resequenced and phased 27 kb of DNA within 580 kb of the MHC class II region \\nin 158 population chromosomes, most of which were conserved extended haplotypes \\n(CEHs) of European descent or contained their centromeric fragments. We \\ndetermined the single nucleotide polymorphism and deletion-insertion \\npolymorphism alleles of the dominant sequences from HLA-DQA2 to DAXX for these \\nCEHs. Nine of 13 CEHs remained sufficiently intact to possess a dominant \\nsequence extending at least to DAXX, 230 kb centromeric to HLA-DPB1. We \\nidentified the regions centromeric to HLA-DQB1 within which single instances of \\neight \"common\" European MHC haplotypes previously sequenced by the MHC Haplotype \\nProject (MHP) were representative of those dominant CEH sequences. Only two MHP \\nhaplotypes had a dominant CEH sequence throughout the centromeric and extended \\nclass II region and one MHP haplotype did not represent a known European CEH \\nanywhere in the region. We identified the centromeric recombination transition \\npoints of other MHP sequences from CEH representation to non-representation. \\nSeveral CEH pairs or groups shared sequence identity in small blocks but had \\nsignificantly different (although still conserved for each separate CEH) \\nsequences in surrounding regions. These patterns partly explain strong \\ncalculated linkage disequilibrium over only short (tens to hundreds of \\nkilobases) distances in the context of a finite number of observed \\nmegabase-length CEHs comprising half a population\\'s haplotypes. Our results \\nprovide a clearer picture of European CEH class II allelic structure and \\npopulation haplotype architecture, improved regional CEH markers, and raise \\nquestions concerning regional recombination hotspots.\\n\\nDOI: 10.1371/journal.pgen.1004637\\nPMCID: PMC4191933\\nPMID: 25299700 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have declared that no competing \\ninterests exist.\\n\\n\\n54. PLoS One. 2014 Jun 18;9(6):e99724. doi: 10.1371/journal.pone.0099724. \\neCollection 2014.\\n\\nA genome-wide association study on chronic HBV infection and its clinical \\nprogression in male Han-Taiwanese.\\n\\nChang SW(1), Fann CS(2), Su WH(3), Wang YC(4), Weng CC(4), Yu CJ(5), Hsu CL(2), \\nHsieh AR(6), Chien RN(4), Chu CM(4), Tai DI(4).\\n\\nAuthor information:\\n(1)Clinical Informatics and Medical Statistics Research Center, Chang Gung \\nUniversity College of Medicine, Taoyuan, Taiwan.\\n(2)Institute of Biomedical Sciences, Academia Sinica, Taipei, Taiwan.\\n(3)Department of Biomedical Sciences, Graduate Institute of Biomedical Sciences, \\nChang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, \\nTaiwan.\\n(4)Division of Hepatology, Department of Gastroenterology and Hepatology, Chang \\nGung Memorial Hospital, Linkou, Taiwan.\\n(5)Department of Cell and Molecular Biology, College of Medicine and Molecular \\nMedicine Research Center, Chang Gung University, Tao-Yuan, Taiwan.\\n(6)Graduate Institute of Biostatistics, China Medical University, Taichung, \\nTaiwan.\\n\\nIt is common to observe the clustering of chronic hepatitis B surface antigen \\n(HBsAg) carriers in families. Intra-familial transmission of hepatitis B virus \\n(HBV) could be the reason for the familial clustering of HBsAg carriers. \\nAdditionally, genetic and gender factors have been reported to be involved. We \\nconducted a three-stage genome-wide association study to identify genetic \\nfactors associated with chronic HBV susceptibility. A total of 1,065 male \\ncontrols and 1,623 male HBsAg carriers were included. The whole-genome \\ngenotyping was done on Illumina HumanHap550 beadchips in 304 healthy controls \\nand HumanHap610 beadchips in 321 cases. We found that rs9277535 (HLA-DPB1, \\nP\\u200a=\\u200a4.87×10(-14)), rs9276370 (HLA-DQA2, P\\u200a=\\u200a1.9×10(-12)), rs7756516 and \\nrs7453920 (HLA-DQB2, P\\u200a=\\u200a1.48×10(-11) and P\\u200a=\\u200a6.66×10(-15) respectively) were \\nsignificantly associated with persistent HBV infection. A novel SNP rs9366816 \\nnear HLA-DPA3 also showed significant association (P\\u200a=\\u200a2.58×10(-10)). The \\n\"T-T-G-G-T\" haplotype of the five SNPs further signified their association with \\nthe disease (P\\u200a=\\u200a1.48×10(-12); OR\\u200a=\\u200a1.49). The \"T-T\" haplotype composed of \\nrs7756516 and rs9276370 was more prevalent in severe disease subgroups and \\nassociated with non-sustained therapeutic response (P\\u200a=\\u200a0.0262). The \"G-C\" \\nhaplotype was associated with sustained therapeutic response (P\\u200a=\\u200a0.0132; \\nOR\\u200a=\\u200a2.49). We confirmed that HLA-DPB1, HLA-DQA2 and HLA-DQB2 loci were \\nassociated with persistent HBV infection in male Taiwan Han-Chinese. In \\naddition, the HLA-DQA2 and -DQB2 complex was associated with clinical \\nprogression and therapeutic response.\\n\\nDOI: 10.1371/journal.pone.0099724\\nPMCID: PMC4062466\\nPMID: 24940741 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n55. J Occup Environ Med. 2014 Apr;56(4):382-7. doi: 10.1097/JOM.0000000000000138.\\n\\nGenetic variants in the major histocompatibility complex class I and class II \\ngenes are associated with diisocyanate-induced Asthma.\\n\\nYucesoy B(1), Johnson VJ, Lummus ZL, Kashon ML, Rao M, Bannerman-Thompson H, \\nFrye B, Wang W, Gautrin D, Cartier A, Boulet LP, Sastre J, Quirce S, Tarlo SM, \\nGermolec DR, Luster MI, Bernstein DI.\\n\\nAuthor information:\\n(1)From the Health Effects Laboratory Division (Drs Yucesoy and Kashon, Ms Frye, \\nand Ms Wang) NIOSH/CDC, Morgantown, WVa; BRT-Burleson Research Technologies (Dr \\nJohnson), Morrisville, NC; Division of Immunology, Allergy and Rheumatology, \\nDepartment of Medicine (Drs Yucesoy, Lummus, and Bernstein) and Department of \\nEnvironmental Health (Drs Rao and Bannerman-Thompson), University of Cincinnati, \\nOhio; Hôpital du Sacré-Cœur de Montréal (Drs Gautrin and Cartier), Université de \\nMontréal; Hôpital Laval (Dr Boulet), Université Laval, Sainte-Foy, Québec, \\nCanada; Department of Allergy (Dr Sastre), Fundación Jiménez Díaz and CIBER de \\nEnfermedades Respiratorias CIBERES; Department of Allergy (Dr Quirce), Hospital \\nLa Paz-IdiPAZ and CIBER de Enfermedades Respiratorias CIBERES, Madrid, Spain; \\nDepartment of Medicine (Dr Tarlo), University of Toronto, Ontario, Canada; \\nToxicology Branch (Dr Germolec), DNTP/NIEHS, Research Triangle Park, NC; and \\nSchool of Public Health (Dr Luster), West Virginia University, Morgantown.\\n\\nOBJECTIVE: To investigate the association between single nucleotide \\npolymorphisms (SNPs) located across the major histocompatibility complex and \\nsusceptibility to diisocyanate-induced asthma (DA).\\nMETHODS: The study population consisted of 140 diisocyanate-exposed workers. \\nGenotyping was performed using the Illumina GoldenGate major histocompatibility \\ncomplex panels.\\nRESULTS: The HLA-E rs1573294 and HLA-DPB1 rs928976 SNPs were associated with an \\nincreased risk of DA under dominant (odds ratio [OR], 6.27; 95% confidence \\ninterval [CI], 2.37 to 16.6; OR, 2.79, 95% CI, 0.99 to 7.81, respectively) and \\nrecessive genetic models (OR, 6.27, 95% CI, 1.63 to 24.13; OR, 10.10, 95% CI, \\n3.16 to 32.33, respectively). The HLA-B rs1811197, HLA-DOA rs3128935, and \\nHLA-DQA2 rs7773955 SNPs conferred an increased risk of DA in a dominant model \\n(OR, 7.64, 95% CI, 2.25 to 26.00; OR, 19.69, 95% CI, 2.89 to 135.25; OR, 8.43, \\n95% CI, 3.03 to 23.48, respectively).\\nCONCLUSION: These results suggest that genetic variations within HLA genes play \\na role in DA risk.\\n\\nDOI: 10.1097/JOM.0000000000000138\\nPMCID: PMC4572490\\nPMID: 24709764 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no conflict of interest. \\nAuthors Yucesoy, Johnson, Lummus, Kashon, Rao, Bannerman-Thompson, Frye, Wang, \\nGautrin, Cartier, Boulet, Sastre, Quirce, Tarlo, Germolec, Luster, and Bernstein \\nhave no relationships/conditions/circumstances that present potential conflict \\nof interest. The JOEM editorial board and planners have no financial interest \\nrelated to this research.\\n\\n\\n56. Eur Rev Med Pharmacol Sci. 2013 Apr;17(8):1044-50.\\n\\nDNA microarray-based screening of differentially expressed genes related to \\nacute lung injury and functional analysis.\\n\\nChen Y(1), Shi JX, Pan XF, Feng J, Zhao H.\\n\\nAuthor information:\\n(1)Thoracic Department, Shanghai Chest Hospital, Shanghai, China.\\n\\nOBJECTIVES: The purpose of this study was to identify differentially expressed \\ngenes (DEGs) related to acute lung injury (ALI) induced by sepsis with DNA \\nmicroarray.\\nMATERIALS AND METHODS: Gene expression profile GSE10474 was downloaded from Gene \\nExpression Omnibus (GEO) database which includes 34 samples, among which 13 \\npatients with ALI + sepsis and 21 patients with sepsis alone. The DEGs were \\nidentified between ALI + sepsis and sepsis alone samples using R, which were \\nfurther analyzed using bioinformatics methods. Firstly, HitPredict was used to \\nsearch protein-protein interactions of the DEGs. Secondly, WebGestalt was \\nadopted for functional enrichment analysis of genes in the interaction networks. \\nFinally, DNA methylation was analyzed to explain the differential expression.\\nRESULTS: A total of 12 genes were identified as DEGs by comparing chip data from \\nALI + sepsis samples and those from sepsis alone samples, among which occludin \\n(OCLN) and major histocompatibility complex, class II, DQ beta 1 (HLA-DQB1) had \\n21 and 6 interactors, respectively. Functional enrichment analysis revealed \\nseveral significantly over-represented terms: cellular component organization, \\nmacromolecular organization and biosynthesis, and response to stimulus. In \\naddition, methylation was found in the promoters of OCLN and HLA-DQB1.\\nCONCLUSIONS: We screened DEGs in septic ALI samples, and several interesting \\ngenes were obtained, especially OCLN and HLA-DQB1. They may be developed into \\nmarker genes for diagnosis or treatment of ALI.\\n\\nPMID: 23661517 [Indexed for MEDLINE]\\n\\n\\n57. PLoS Genet. 2012;8(12):e1003098. doi: 10.1371/journal.pgen.1003098. Epub 2012\\n Dec 20.\\n\\nGenome-wide joint meta-analysis of SNP and SNP-by-smoking interaction identifies \\nnovel loci for pulmonary function.\\n\\nHancock DB(1), Soler Artigas M, Gharib SA, Henry A, Manichaikul A, Ramasamy A, \\nLoth DW, Imboden M, Koch B, McArdle WL, Smith AV, Smolonska J, Sood A, Tang W, \\nWilk JB, Zhai G, Zhao JH, Aschard H, Burkart KM, Curjuric I, Eijgelsheim M, \\nElliott P, Gu X, Harris TB, Janson C, Homuth G, Hysi PG, Liu JZ, Loehr LR, \\nLohman K, Loos RJ, Manning AK, Marciante KD, Obeidat M, Postma DS, Aldrich MC, \\nBrusselle GG, Chen TH, Eiriksdottir G, Franceschini N, Heinrich J, Rotter JI, \\nWijmenga C, Williams OD, Bentley AR, Hofman A, Laurie CC, Lumley T, Morrison AC, \\nJoubert BR, Rivadeneira F, Couper DJ, Kritchevsky SB, Liu Y, Wjst M, Wain LV, \\nVonk JM, Uitterlinden AG, Rochat T, Rich SS, Psaty BM, O\\'Connor GT, North KE, \\nMirel DB, Meibohm B, Launer LJ, Khaw KT, Hartikainen AL, Hammond CJ, Gläser S, \\nMarchini J, Kraft P, Wareham NJ, Völzke H, Stricker BH, Spector TD, \\nProbst-Hensch NM, Jarvis D, Jarvelin MR, Heckbert SR, Gudnason V, Boezen HM, \\nBarr RG, Cassano PA, Strachan DP, Fornage M, Hall IP, Dupuis J, Tobin MD, London \\nSJ.\\n\\nAuthor information:\\n(1)Behavioral Health Epidemiology Program, Research Triangle Institute \\nInternational, Research Triangle Park, North Carolina, USA.\\n\\nComment in\\n    On Lung Function and Interactions Using Genome-Wide Data.\\n\\nGenome-wide association studies have identified numerous genetic loci for \\nspirometic measures of pulmonary function, forced expiratory volume in one \\nsecond (FEV(1)), and its ratio to forced vital capacity (FEV(1)/FVC). Given that \\ncigarette smoking adversely affects pulmonary function, we conducted genome-wide \\njoint meta-analyses (JMA) of single nucleotide polymorphism (SNP) and \\nSNP-by-smoking (ever-smoking or pack-years) associations on FEV(1) and \\nFEV(1)/FVC across 19 studies (total N\\u200a=\\u200a50,047). We identified three novel loci \\nnot previously associated with pulmonary function. SNPs in or near DNER \\n(smallest P(JMA\\u200a=\\u200a)5.00×10(-11)), HLA-DQB1 and HLA-DQA2 (smallest \\nP(JMA\\u200a=\\u200a)4.35×10(-9)), and KCNJ2 and SOX9 (smallest P(JMA\\u200a=\\u200a)1.28×10(-8)) were \\nassociated with FEV(1)/FVC or FEV(1) in meta-analysis models including SNP main \\neffects, smoking main effects, and SNP-by-smoking (ever-smoking or pack-years) \\ninteraction. The HLA region has been widely implicated for autoimmune and lung \\nphenotypes, unlike the other novel loci, which have not been widely implicated. \\nWe evaluated DNER, KCNJ2, and SOX9 and found them to be expressed in human lung \\ntissue. DNER and SOX9 further showed evidence of differential expression in \\nhuman airway epithelium in smokers compared to non-smokers. Our findings \\ndemonstrated that joint testing of SNP and SNP-by-environment interaction \\nidentified novel loci associated with complex traits that are missed when \\nconsidering only the genetic main effects.\\n\\nDOI: 10.1371/journal.pgen.1003098\\nPMCID: PMC3527213\\nPMID: 23284291 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have declared that no competing \\ninterests exist.\\n\\n\\n58. Lupus. 2013 Feb;22(2):198-204. doi: 10.1177/0961203312470183. Epub 2012 Dec\\n20.\\n\\nHLA variants rs9271366 and rs9275328 are associated with systemic lupus \\nerythematosus susceptibility in Malays and Chinese.\\n\\nChai HC(1), Phipps ME, Othman I, Tan LP, Chua KH.\\n\\nAuthor information:\\n(1)Jeffrey Cheah School of Medicine and Health Sciences, Monash University \\n(Sunway Campus), Malaysia.\\n\\nBACKGROUND: Human leukocyte antigen (HLA) antigens and genes have long been \\nreported associated with systemic lupus erythematosus (SLE) susceptibility in \\nmany populations. With the advance in technologies such as genome-wide \\nassociation studies, many newly discovered SLE-associated single-nucleotide \\npolymorphisms (SNPs) have been reported in recent years. These include \\nHLA-DRB1/HLA-DQA1 rs9271366 and HLA-DQB1/HLA-DQA2 rs9275328. Our aim was to \\ninvestigate these SNPs in a Malaysian SLE cohort.\\nMATERIALS AND METHODS: SNPs rs9271366 and rs9275328 were screened across 790 \\nMalaysian citizens from three ethnic groups (360 patients and 430 healthy \\nvolunteers) by Taqman SNP genotyping assays. Allele and genotyping frequencies, \\nHardy-Weinberg equilibrium, Fisher\\'s exact test and odds ratio were calculated \\nfor each SNP and ethnic group. Linkage disequilibrium and interaction between \\nthe two SNPs were also evaluated.\\nRESULTS: The minor allele G and its homozygous genotype GG of HLA-DRB1/HLA-DQA1 \\nrs9271366 significantly increased the SLE susceptibility in Malaysian patients, \\nincluding those of Malay and Chinese ethnicity (odds ratio (OR)\\u2009>\\u20091, p\\u2009<\\u20090.05). \\nAs for HLA-DQB1/HLA-DQA2 rs9275328, the minor allele T and the heterozygous \\ngenotype CT conferred protective effect to SLE in Malaysians, as well as in \\nMalays and Chinese, by having OR\\u2009<\\u20091 and p value <0.05. Both SNPs did not show \\nassociations to SLE in Indians. D\\' and r (2) values for the two SNPs in LD \\nanalysis were 0.941 and 0.065, respectively, with haplotype GC and AT being \\nsignificantly associated with SLE (p\\u2009<\\u20095.0\\u2009×\\u200910(-4)) after 10,000 permutations \\nwere performed. The MDR test clustered the genotype combinations of GG and CC, \\nand AG and CC of rs9271366 and rs9275328, accordingly, as high-risk group, and \\nthe two SNPs interacted redundantly by removing 1.96% of the entropy.\\nCONCLUSIONS: Our findings suggest that in addition to some classical HLA \\nvariants, rs9271366 and rs9275328 are additional polymorphisms worth considering \\nin the Malaysian and possibly in a larger Asian SLE scenario.\\n\\nDOI: 10.1177/0961203312470183\\nPMID: 23257407 [Indexed for MEDLINE]\\n\\n\\n59. Clin Exp Allergy. 2012 Dec;42(12):1724-33. doi: 10.1111/cea.12000.\\n\\nHLA-DQ strikes again: genome-wide association study further confirms HLA-DQ in \\nthe diagnosis of asthma among adults.\\n\\nLasky-Su J(1), Himes BE, Raby BA, Klanderman BJ, Sylvia JS, Lange C, Melen E, \\nMartinez FD, Israel E, Gauderman J, Gilliland F, Sleiman P, Hakonarson H, \\nCeledón JC, Soto-Quiros M, Avila L, Lima JJ, Irvin CG, Peters SP, Boushey H, \\nChinchilli VM, Mauger D, Tantisira K, Weiss ST; SHARP investigators.\\n\\nAuthor information:\\n(1)Channing Laboratory, Department of Medicine, Brigham and Women\\'s Hospital, \\nand Harvard Medical School, Boston, MA, USA.\\n\\nBACKGROUND: Asthma is a common chronic respiratory disease in children and \\nadults. An important genetic component to asthma susceptibility has long been \\nrecognized, most recently through the identification of several genes (e.g., \\nORMDL3, PDE4D, HLA-DQ, and TLE4) via genome-wide association studies.\\nOBJECTIVE: To identify genetic variants associated with asthma affection status \\nusing genome-wide association data.\\nMETHODS: We describe results from a genome-wide association study on asthma \\nperformed in 3855 subjects using a panel of 455 089 single nucleotide \\npolymorphisms (SNPs).\\nRESULT: The genome-wide association study resulted in the prioritization of 33 \\nvariants for immediate follow-up in a multi-staged replication effort. Of these, \\na common polymorphism (rs9272346) localizing to within 1 Kb of HLA-DQA1 \\n(chromosome 6p21.3) was associated with asthma in adults (P-value = 2.2E-08) \\nwith consistent evidence in the more heterogeneous group of adults and children \\n(P-value = 1.0E-04). Moreover, some genes identified in prior asthma GWAS were \\nnominally associated with asthma in our populations.\\nCONCLUSION: Overall, our findings further replicate the HLA-DQ region in the \\npathogenesis of asthma. HLA-DQA1 is the fourth member of the HLA family found to \\nbe associated with asthma, in addition to the previously identified HLA-DRA, \\nHLA-DQB1 and HLA-DQA2.\\n\\n© 2012 Blackwell Publishing Ltd.\\n\\nDOI: 10.1111/cea.12000\\nPMCID: PMC6343489\\nPMID: 23181788 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Conflict of Interests: Charles Irvin’s work was \\nfunded by a grant from the American Lung Association. Stephen Peters has been a \\nmember of the NHLBI’s ACRN since its creation in 1993. Homer Boushey received \\nfunding from the NIH for his participation in the research of the Inner City \\nAsthma Consortium, from Genentech for research on the airway microbiome of \\nsevere asthma, and from GlaxoSmithKline for research on bronchial epithelial \\ncell susceptibility to viral infection. He has served as an ad-hoc consultant or \\nadvisory board member for Genentech, GlaxoSmithKline, Kalobios, Pharmaxis, Merck \\nand Johnson & Johnson. Vernon Chinchilli, Blanca Himes, Scott Weiss, Juan \\nCeledon, Barbara Klanderman, Erik Melen, Hakon Hakonarson, Lydiana Avila, \\nBenjamin Raby, Kelan Tantisira, David Mauger, Patrick Sleiman, Fernando \\nMartinez, W. James Gauderman, Frank Gilliland, John Lima, Elliot Israel, \\nChristoph Lange, Jody Senter Sylvia, Manuel Soto Quiros and Jessica Lasky-Su \\nhave no conflict of interests to declare\\n\\n\\n60. J Immunol. 2012 Apr 15;188(8):3903-11. doi: 10.4049/jimmunol.1103048. Epub\\n2012  Mar 9.\\n\\nHLA-DQA2 and HLA-DQB2 genes are specifically expressed in human Langerhans cells \\nand encode a new HLA class II molecule.\\n\\nLenormand C(1), Bausinger H, Gross F, Signorino-Gelo F, Koch S, Peressin M, \\nFricker D, Cazenave JP, Bieber T, Hanau D, de la Salle H, Tourne S.\\n\\nAuthor information:\\n(1)INSERM, Unité Mixte de Recherche S725, Biologie des Cellules Dendritiques \\nHumaines, Strasbourg, 67065 France.\\n\\nThe precise role of human epidermal Langerhans cells (LCs) in immune response is \\nhighly controversial. While studying the gene expression profile of these cells, \\nwe were intrigued to identify the HLA-DQB2 gene as potentially expressed in LCs. \\nDespite a strong evolutionary conservation of their sequences, the concomitant \\nexpression of the poorly polymorphic HLA-DQA2/HLA-DQB2 genes, paralogous to the \\nHLA-DQA1/HLA-DQB1 genes, has never been detected in any cell type. We confirmed \\nby RT-PCR that the HLA-DQA2 and -DQB2 genes are both expressed in LCs, but not \\nin monocyte-derived dendritic cells, or in blood CD1c(+) or plasmacytoid \\ndendritic cells. The presence of the HLA-DQβ2 chain in LCs could be demonstrated \\nby Western blotting, whereas immunofluorescence revealed its localization in \\nearly endosomes. As in the case of other HLA class II molecules, the HLA-DQα2 \\nand -DQβ2 chains formed heterodimers that had to associate with the invariant \\nchain to reach endosomal compartments. HLA-DQα2/β2 heterodimers were expressed \\nat the cell surface, where they could mediate staphylococcal superantigen \\nstimulation of T cells. Interestingly, HLA-DQα2 and HLA-DQβ1 chains formed mixed \\nheterodimers which efficiently left the endoplasmic reticulum. These \\nobservations strongly suggest that the poorly polymorphic HLA-DQA2 and -DQB2 \\ngenes should be considered to be of immunological importance. The HLA-DQα2/β2 \\nmolecules could influence the complexity of the repertoire of Ags presented by \\nLCs.\\n\\nDOI: 10.4049/jimmunol.1103048\\nPMID: 22407913 [Indexed for MEDLINE]61. PLoS One. 2011;6(11):e23089. doi: 10.1371/journal.pone.0023089. Epub 2011 Nov\\n 18.\\n\\nPsoriasis regression analysis of MHC loci identifies shared genetic variants \\nwith vitiligo.\\n\\nZhu KJ(1), Lv YM, Yin XY, Wang ZX, Sun LD, He SM, Cheng H, Hu DY, Zhang Z, Li Y, \\nZuo XB, Zhou YW, Yang S, Fan X, Zhang XJ, Zhang FY.\\n\\nAuthor information:\\n(1)Institute of Dermatology and Department of Dermatology at No.1 Hospital, \\nAnhui Medical University, Hefei, Anhui, China.\\n\\nPsoriasis is a common inflammatory skin disease with genetic components of both \\nimmune system and the epidermis. PSOR1 locus (6q21) has been strongly associated \\nwith psoriasis; however, it is difficult to identify additional independent \\nassociation due to strong linkage disequilibrium in the MHC region. We performed \\nstepwise regression analyses of more than 3,000 SNPs in the MHC region genotyped \\nusing Human 610-Quad (Illumina) in 1,139 cases with psoriasis and 1,132 controls \\nof Han Chinese population to search for additional independent association. With \\nfour regression models obtained, two SNPs rs9468925 in HLA-C/HLA-B and rs2858881 \\nin HLA-DQA2 were repeatedly selected in all models, suggesting that multiple \\nloci outside PSOR1 locus were associated with psoriasis. More importantly we \\nfind that rs9468925 in HLA-C/HLA-B is associated with both psoriasis and \\nvitiligo, providing first important evidence that two major skin diseases share \\na common genetic locus in the MHC, and a basis for elucidating the molecular \\nmechanism of skin disorders.\\n\\nDOI: 10.1371/journal.pone.0023089\\nPMCID: PMC3220662\\nPMID: 22125590 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n62. PLoS One. 2011;6(11):e27109. doi: 10.1371/journal.pone.0027109. Epub 2011 Nov\\n9.\\n\\nEvidence for more than one Parkinson\\'s disease-associated variant within the HLA \\nregion.\\n\\nHill-Burns EM(1), Factor SA, Zabetian CP, Thomson G, Payami H.\\n\\nAuthor information:\\n(1)New York State Department of Health Wadsworth Center, Albany, New York, \\nUnited States of America.\\n\\nParkinson\\'s disease (PD) was recently found to be associated with HLA in a \\ngenome-wide association study (GWAS). Follow-up GWAS\\'s replicated the PD-HLA \\nassociation but their top hits differ. Do the different hits tag the same locus \\nor is there more than one PD-associated variant within HLA? We show that the top \\nGWAS hits are not correlated with each other (0.00≤r(2)≤0.15). Using our GWAS \\n(2000 cases, 1986 controls) we conducted step-wise conditional analysis on 107 \\nSNPs with P<10(-3) for PD-association; 103 dropped-out, four remained \\nsignificant. Each SNP, when conditioned on the other three, yielded \\nP(SNP1)\\u200a=\\u200a5×10(-4), P(SNP2)\\u200a=\\u200a5×10(-4), P(SNP3)\\u200a=\\u200a4×10(-3) and P(SNP4)\\u200a=\\u200a0.025. \\nThe four SNPs were not correlated (0.01≤r(2)≤0.20). Haplotype analysis \\n(excluding rare SNP2) revealed increasing PD risk with increasing risk alleles \\nfrom OR\\u200a=\\u200a1.27, P\\u200a=\\u200a5×10(-3) for one risk allele to OR\\u200a=\\u200a1.65, P\\u200a=\\u200a4×10(-8) for \\nthree. Using additional 843 cases and 856 controls we replicated the independent \\neffects of SNP1 (P(conditioned-on-SNP4)\\u200a=\\u200a0.04) and SNP4 \\n(P(conditioned-on-SNP1)\\u200a=\\u200a0.04); SNP2 and SNP3 could not be replicated. In \\npooled GWAS and replication, SNP1 had OR(conditioned-on-SNP4)\\u200a=\\u200a1.23, \\nP(conditioned-on-SNP4)\\u200a=\\u200a6×10(-7); SNP4 had OR(conditioned-on-SNP1)\\u200a=\\u200a1.18, \\nP(conditioned-on-SNP1)\\u200a=\\u200a3×10(-3); and the haplotype with both risk alleles had \\nOR\\u200a=\\u200a1.48, P\\u200a=\\u200a2×10(-12). Genotypic OR increased with the number of risk alleles \\nan individual possessed up to OR\\u200a=\\u200a1.94, P\\u200a=\\u200a2×10(-11) for individuals who were \\nhomozygous for the risk allele at both SNP1 and SNP4. SNP1 is a variant in \\nHLA-DRA and is associated with HLA-DRA, DRB5 and DQA2 gene expression. SNP4 is \\ncorrelated (r(2)\\u200a=\\u200a0.95) with variants that are associated with HLA-DQA2 \\nexpression, and with the top HLA SNP from the IPDGC GWAS (r(2)\\u200a=\\u200a0.60). Our \\nfindings suggest more than one PD-HLA association; either different alleles of \\nthe same gene, or separate loci.\\n\\nDOI: 10.1371/journal.pone.0027109\\nPMCID: PMC3212531\\nPMID: 22096524 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n63. J Allergy Clin Immunol. 2012 Mar;129(3):840-845.e21. doi: \\n10.1016/j.jaci.2011.09.029. Epub 2011 Nov 9.\\n\\nA genome-wide association study of plasma total IgE concentrations in the \\nFramingham Heart Study.\\n\\nGranada M(1), Wilk JB, Tuzova M, Strachan DP, Weidinger S, Albrecht E, Gieger C, \\nHeinrich J, Himes BE, Hunninghake GM, Celedón JC, Weiss ST, Cruikshank WW, \\nFarrer LA, Center DM, O\\'Connor GT.\\n\\nAuthor information:\\n(1)Pulmonary Center, Boston University School of Medicine, Boston, MA 02118, \\nUSA.\\n\\nComment in\\n    J Allergy Clin Immunol. 2012 Mar;129(3):846-7.\\n\\nBACKGROUND: Atopy and plasma IgE concentration are genetically complex traits, \\nand the specific genetic risk factors that lead to IgE dysregulation and \\nclinical atopy are an area of active investigation.\\nOBJECTIVE: We sought to ascertain the genetic risk factors that lead to IgE \\ndysregulation.\\nMETHODS: A genome-wide association study (GWAS) was performed in 6819 \\nparticipants from the Framingham Heart Study (FHS). Seventy of the top single \\nnucleotide polymorphisms (SNPs) were selected based on P values and linkage \\ndisequilibrium among neighboring SNPs and evaluated in a meta-analysis with 5 \\nindependent populations from the Cooperative Health Research in the Region of \\nAugsburg cohort, the British 1958 Birth Cohort, and the Childhood Asthma \\nManagement Program cohort.\\nRESULTS: Thirteen SNPs located in the region of 3 genes, FCER1A, signal \\ntransducer and activator of transcription 6 (STAT6), and IL13, were found to \\nhave genome-wide significance in the FHS cohort GWAS. The most significant SNPs \\nfrom the 3 regions were rs2251746 (FCER1A, P\\xa0= 2.11 × 10(-12)), rs1059513 \\n(STAT6, P\\xa0= 2.87 × 10(-8)), and rs1295686 (IL13, P\\xa0= 3.55 × 10(-8)). Four \\nadditional gene regions, HLA-G, HLA-DQA2, HLA-A, and Duffy blood group, \\nchemokine receptor (DARC), reached genome-wide statistical significance in a \\nmeta-analysis combining the FHS and replication cohorts, although the DARC \\nassociation did not appear independent of SNPs in the nearby FCER1A gene.\\nCONCLUSION: This GWAS of the FHS cohort has identified genetic loci in HLA genes \\nthat might have a role in the pathogenesis of IgE dysregulation and atopy. It \\nalso confirmed the association of the known susceptibility loci FCER1A, STAT6, \\nand IL13 for the dysregulation of total IgE.\\n\\nCopyright © 2011 American Academy of Allergy, Asthma & Immunology. Published by \\nMosby, Inc. All rights reserved.\\n\\nDOI: 10.1016/j.jaci.2011.09.029\\nPMCID: PMC3293994\\nPMID: 22075330 [Indexed for MEDLINE]\\n\\n\\n64. Vet Immunol Immunopathol. 2011 Dec 15;144(3-4):312-20. doi: \\n10.1016/j.vetimm.2011.08.014. Epub 2011 Aug 26.\\n\\nAssociation between variation in faecal egg count for a mixed field-challenge of \\nnematode parasites and ovine MHC-DQA2 polymorphism.\\n\\nHickford JG(1), Forrest RH, Zhou H, Fang Q, Frampton CM.\\n\\nAuthor information:\\n(1)Department of Agricultural Sciences, Faculty of Agriculture and Life \\nSciences, Lincoln University, Lincoln 7647, Canterbury, New Zealand. \\njon.hickford@lincoln.ac.nz\\n\\nThe selection of sheep that are resistant to gastrointestinal parasites and have \\nlower faecal egg counts (FECs) has been the subject of extensive research. This \\nhas led to the speculation that the Major Histocompatibility Complex (MHC) genes \\ncould be used as markers to reduce FEC. In this study, associations between \\nvariation in ovine MHC-DQA2 and various measures of FEC recorded at two times \\n(approximately 4 and 9 months of age) were investigated in a large group of New \\nZealand lambs (n=4676), derived from 185 different sire-lines, of a variety of \\nbreeds and raised on 25 separate farms. Pair-sample t-tests revealed that FEC \\nfor Nematodirus spp., Strongyle spp. and total FEC differed significantly \\nbetween the two assessments. A total of twenty one DQA2 alleles or \\nDQA2-DQA2-like haplotypes were identified, with allele/haplotype presence and \\nfrequency varying significantly between farms. For example, allele *0103 was \\nobserved on all farms, ranging in frequency from 0.2 to 60.9%, while haplotype \\n*0101-*1601 was only present on one farm, in two lambs. A number of associations \\nbetween the presence/absence of these alleles and egg counts were observed, but \\nnearly all the allelic/haplotypic associations were age and parasite specific, \\nsuggesting that immune response is both age and challenge (parasite species mix) \\ndependent. The exception was allele *1201 which was associated with increased \\ntotal FECs at both 4 and 9 months of age; with it either being, or tending \\ntoward being, significantly associated with both increased Strongyle spp. and \\nNematodirus spp. counts as well. However, the observed increases in egg counts \\nwere small and ranged between 5 and 32 eggs per gram. In conclusion, we believe \\nthat the MHC plays an important role in parasite resistance, but that the \\nMHC-nematode interaction is complex and thus the development of a single \\ngene-marker based on the \"MHC effect\" is unlikely.\\n\\nCopyright © 2011 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.vetimm.2011.08.014\\nPMID: 21945177 [Indexed for MEDLINE]\\n\\n\\n65. Genet Sel Evol. 2010 Dec 21;42(1):43. doi: 10.1186/1297-9686-42-43.\\n\\nGenetic diversity of selected genes that are potentially economically important \\nin feral sheep of New Zealand.\\n\\nMcKenzie GW(1), Abbott J, Zhou H, Fang Q, Merrick N, Forrest RH, Sedcole JR, \\nHickford JG.\\n\\nAuthor information:\\n(1)Department of Agricultural Science, Faculty of Agriculture and Life Sciences, \\nPO Box 84, Lincoln University, Lincoln 7647, New Zealand.\\n\\nBACKGROUND: Feral sheep are considered to be a source of genetic variation that \\nhas been lost from their domestic counterparts through selection.\\nMETHODS: This study investigates variation in the genes KRTAP1-1, KRT33, ADRB3 \\nand DQA2 in Merino-like feral sheep populations from New Zealand and its \\noffshore islands. These genes have previously been shown to influence wool, lamb \\nsurvival and animal health.\\nRESULTS: All the genes were polymorphic, but no new allele was identified in the \\nferal populations. In some of these populations, allele frequencies differed \\nfrom those observed in commercial Merino sheep and other breeds found in New \\nZealand. Heterozygosity levels were comparable to those observed in other \\nstudies on feral sheep. Our results suggest that some of the feral populations \\nmay have been either inbred or outbred over the duration of their apparent \\nisolation.\\nCONCLUSION: The variation described here allows us to draw some conclusions \\nabout the likely genetic origin of the populations and selective pressures that \\nmay have acted upon them, but they do not appear to be a source of new genetic \\nmaterial, at least for these four genes.\\n\\nDOI: 10.1186/1297-9686-42-43\\nPMCID: PMC3025881\\nPMID: 21176141 [Indexed for MEDLINE]\\n\\n\\n66. Nat Genet. 2010 Sep;42(9):786-9. doi: 10.1038/ng.647. Epub 2010 Aug 15.\\n\\nGenome-wide association study identifies new HLA class II haplotypes strongly \\nprotective against narcolepsy.\\n\\nHor H(1), Kutalik Z, Dauvilliers Y, Valsesia A, Lammers GJ, Donjacour CE, Iranzo \\nA, Santamaria J, Peraita Adrados R, Vicario JL, Overeem S, Arnulf I, Theodorou \\nI, Jennum P, Knudsen S, Bassetti C, Mathis J, Lecendreux M, Mayer G, Geisler P, \\nBenetó A, Petit B, Pfister C, Bürki JV, Didelot G, Billiard M, Ercilla G, \\nVerduijn W, Claas FH, Vollenweider P, Waeber G, Waterworth DM, Mooser V, Heinzer \\nR, Beckmann JS, Bergmann S, Tafti M.\\n\\nAuthor information:\\n(1)Center for Integrative Genomics, University of Lausanne, Lausanne, \\nSwitzerland.\\n\\nErratum in\\n    Nat Genet. 2011 Apr;43(4):388. Vollenwider, Peter [corrected to \\nVollenweider, Peter].\\n\\nNarcolepsy is a rare sleep disorder with the strongest human leukocyte antigen \\n(HLA) association ever reported. Since the associated HLA-DRB1*1501-DQB1*0602 \\nhaplotype is common in the general population (15-25%), it has been suggested \\nthat it is almost necessary but not sufficient for developing narcolepsy. To \\nfurther define the genetic basis of narcolepsy risk, we performed a genome-wide \\nassociation study (GWAS) in 562 European individuals with narcolepsy (cases) and \\n702 ethnically matched controls, with independent replication in 370 cases and \\n495 controls, all heterozygous for DRB1*1501-DQB1*0602. We found association \\nwith a protective variant near HLA-DQA2 (rs2858884; P < 3 x 10(-8)). Further \\nanalysis revealed that rs2858884 is strongly linked to DRB1*03-DQB1*02 (P < 4 x \\n10(-43)) and DRB1*1301-DQB1*0603 (P < 3 x 10(-7)). Cases almost never carried a \\ntrans DRB1*1301-DQB1*0603 haplotype (odds ratio = 0.02; P < 6 x 10(-14)). This \\nunexpected protective HLA haplotype suggests a virtually causal involvement of \\nthe HLA region in narcolepsy susceptibility.\\n\\nDOI: 10.1038/ng.647\\nPMID: 20711174 [Indexed for MEDLINE]\\n\\n\\n67. PLoS One. 2010 Apr 13;5(4):e10142. doi: 10.1371/journal.pone.0010142.\\n\\nThe effect of single nucleotide polymorphisms from genome wide association \\nstudies in multiple sclerosis on gene expression.\\n\\nHandel AE(1), Handunnetthi L, Berlanga AJ, Watson CT, Morahan JM, Ramagopalan \\nSV.\\n\\nAuthor information:\\n(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, \\nUnited Kingdom.\\n\\nBACKGROUND: Multiple sclerosis (MS) is a complex neurological disorder. Its \\naetiology involves both environmental and genetic factors. Recent genome-wide \\nassociation studies have identified a number of single nucleotide polymorphisms \\n(SNPs) associated with susceptibility to (MS). We investigated whether these \\ngenetic variations were associated with alteration in gene expression.\\nMETHODS/PRINCIPAL FINDINGS: We used a database of mRNA expression and genetic \\nvariation derived from immortalised peripheral lymphocytes to investigate \\npolymorphisms associated with MS for correlation with gene expression. Several \\nSNPs were found to be associated with changes in expression: in particular two \\nwith HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB1, HLA-DRB4 and HLA-DRB5, one with \\nZFP57, one with CD58, two with IL7 and FAM164A, and one with FAM119B, TSFM and \\nKUB3. We found minimal cross-over with a recent whole genome expression study in \\nMS patients.\\nDISCUSSION: We have shown that many susceptibility loci in MS are associated \\nwith changes in gene expression using an unbiased expression database. Several \\nof these findings suggest novel gene candidates underlying the effects of \\nMS-associated genetic variation.\\n\\nDOI: 10.1371/journal.pone.0010142\\nPMCID: PMC2854120\\nPMID: 20405052 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n68. J Allergy Clin Immunol. 2010 Feb;125(2):328-335.e11. doi: \\n10.1016/j.jaci.2009.11.018.\\n\\nGenome-wide association study of asthma identifies RAD50-IL13 and HLA-DR/DQ \\nregions.\\n\\nLi X(1), Howard TD, Zheng SL, Haselkorn T, Peters SP, Meyers DA, Bleecker ER.\\n\\nAuthor information:\\n(1)Center for Human Genomics, Wake Forest University School of Medicine, \\nWinston-Salem, NC 27157, USA.\\n\\nBACKGROUND: Asthma is a heterogeneous disease that is caused by the interaction \\nof genetic susceptibility with environmental influences. Genome-wide association \\nstudies (GWASs) represent a powerful approach to investigate the association of \\nDNA variants with disease susceptibility. To date, few GWASs for asthma have \\nbeen reported.\\nOBJECTIVES: A GWAS was performed on a population of patients with severe or \\ndifficult-to-treat asthma to identify genes that are involved in the \\npathogenesis of asthma.\\nMETHODS: A total of 292,443 single nucleotide polymorphisms (SNPs) were tested \\nfor association with asthma in 473 The Epidemiology and Natural History of \\nAsthma: Outcomes and Treatment Regimens (TENOR) cases and 1892 Illumina general \\npopulation controls. Asthma-related quantitative traits (total serum IgE, \\nFEV(1), forced vital capacity, and FEV(1)/forced vital capacity) were also \\ntested in identified candidate regions in 473 TENOR cases and 363 phenotyped \\ncontrols without a history of asthma to analyze GWAS results further. Imputation \\nwas performed in identified candidate regions for analysis with denser SNP \\ncoverage.\\nRESULTS: Multiple SNPs in the RAD50-IL13 region on chromosome 5q31.1 were \\nassociated with asthma: rs2244012 in intron 2 of RAD50 (P = 3.04E-07). The \\nHLA-DR/DQ region on chromosome 6p21.3 was also associated with asthma: rs1063355 \\nin the 3\\' untranslated region of HLA-DQB1 (P = 9.55E-06). Imputation identified \\nseveral significant SNPs in the T(H)2 locus control region 3\\' of RAD50. \\nImputation also identified a more significant SNP, rs3998159 (P = 1.45E-06), \\nbetween HLA-DQB1 and HLA-DQA2.\\nCONCLUSION: This GWAS confirmed the important role of T(H)2 cytokine and antigen \\npresentation genes in asthma at a genome-wide level and the importance of \\nadditional investigation of these 2 regions to delineate their structural \\ncomplexity and biologic function in the development of asthma.\\n\\nCopyright 2010 American Academy of Allergy, Asthma & Immunology. Published by \\nMosby, Inc. All rights reserved.\\n\\nDOI: 10.1016/j.jaci.2009.11.018\\nPMCID: PMC2824608\\nPMID: 20159242 [Indexed for MEDLINE]\\n\\n\\n69. Immunogenetics. 2008 Apr;60(3-4):167-76. doi: 10.1007/s00251-008-0275-5. Epub\\n 2008 Mar 6.\\n\\nGenomic analysis of Ovis aries (Ovar) MHC class IIa loci.\\n\\nHerrmann-Hoesing LM(1), White SN, Kappmeyer LS, Herndon DR, Knowles DP.\\n\\nAuthor information:\\n(1)US Department of Agriculture, Animal Disease Research Unit, Agricultural \\nResearch Service, Pullman, WA 99164-6630, USA. lherrman@vetmed.wsu.edu\\n\\nDetermining the genomic organization of the Ovis aries (Ovar) major \\nhistocompatibility complex class IIa region is essential for future functional \\nstudies related to antigen presentation. In this study, a bacterial artificial \\nchromosome (BAC) library of genomic DNA from peripheral blood leukocytes (PBL) \\nof a Rambouillet ram was constructed, and BAC clone consisting of the major \\nhistocompatibility complex (MHC) class II DQB2, DQA2, DQB1, DQA1, and DRB1 loci \\nwas identified and completely sequenced. The BAC clone consists of 160,889 bp of \\nfinished sequence with the loci arranged in the following order: DQB2, DQA2, \\nDQB1, DQA1, and DRB1 with 14.3, 25, 6.6, and 40.9 Kb spanning between the loci, \\nrespectively. All five of these loci were transcribed in the animal used to \\ngenerate the MHC class II BAC clone. Repeat or retrotransposable elements along \\nwith MHC class II cis promoter elements consisting of S, X, and Y boxes were \\nidentified in the sequence. In addition, 16 non-coding conserved sequences \\namongst primates, carnivores, and ruminants were identified (p < 0.001). These \\nconserved sequences include binding sites for transcription factors with known \\nroles in immune cells, and they provide a basis for further functional \\ninvestigation of the genes in this region. This is the first ruminant finished \\nsequence of the DQB2-DRB1 region, and this sequence information will aid in \\nwhole genome and transcriptome analyses of MHC class II.\\n\\nDOI: 10.1007/s00251-008-0275-5\\nPMID: 18322680 [Indexed for MEDLINE]\\n\\n\\n70. J Autoimmun. 2008 Jun;30(4):266-72. doi: 10.1016/j.jaut.2007.10.006. Epub\\n2007  Dec 11.\\n\\nThe MHC type 1 diabetes susceptibility gene is centromeric to HLA-DQB1.\\n\\nHusain Z(1), Kelly MA, Eisenbarth GS, Pugliese A, Awdeh ZL, Larsen CE, Alper CA.\\n\\nAuthor information:\\n(1)Immune Disease Institute, 800 Huntington Avenue, Boston, MA 02115, USA.\\n\\nHLA-DQB1 is widely considered to be the major histocompatibility complex (MHC) \\nsusceptibility gene for type 1 diabetes (T1D). However, since inheritance of the \\ngene in T1D is recessive, the presence of the protective HLA-DQB1 0602 allele \\nwith normal nucleotide sequence in some patients raises the question of whether \\nHLA-DQB1 is not the susceptibility locus itself but merely a good marker. \\nHLA-DQB1 0602 is part of a conserved extended haplotype (CEH) [HLA-B7, SC31, \\nDR2] (B7, DR2) with fixed DNA over more than 1Mb of genomic DNA that normally \\ncarries a protective allele at the true susceptibility locus. We postulated \\nthat, in patients with HLA-DQB1 0602, the protective allele at the \\nsusceptibility locus has been replaced by a susceptibility allele through an \\nancient crossover at meiosis centromeric to HLA-DQB1. We analyzed single \\nnucleotide polymorphisms (SNPs) distinguishing the HLA-DQA2 (the first expressed \\ngene centromeric to HLA-DQB1) allele on the normal HLA-B7, DR2 CEH from those on \\nsusceptibility CEHs in T1D patients and controls with HLA-DQB1 0602. All but 1 \\nof 20 healthy control HLA-DQB1 0602 haplotypes had identical (consensus) first \\nintron HLA-DQA2 5-SNP haplotypes. Fifteen of 19 patients with HLA-DQB1 0602 were \\nhomozygous for 1 or more HLA-DQA2 SNPs differing from consensus HLA-DQA2 SNPs, \\nproviding evidence of crossover involving the HLA-DQA2 locus. The remaining 4 \\npatients were heterozygous at all positions and therefore uninformative. The \\nloss of dominant protection usually associated with HLA-DQB1 0602 haplotypes is \\nconsistent with a locus centromeric to HLA-DQB1 being a major determinant of \\nMHC-associated susceptibility, and perhaps the true T1D susceptibility locus.\\n\\nDOI: 10.1016/j.jaut.2007.10.006\\nPMID: 18065200 [Indexed for MEDLINE]\\n\\n\\n71. Genes Immun. 2007 Sep;8(6):480-91. doi: 10.1038/sj.gene.6364400. Epub 2007\\nJun  14.\\n\\nDistinctive gene expression signatures in rheumatoid arthritis synovial tissue \\nfibroblast cells: correlates with disease activity.\\n\\nGalligan CL(1), Baig E, Bykerk V, Keystone EC, Fish EN.\\n\\nAuthor information:\\n(1)Toronto General Research Institute, University Health Network, Toronto, \\nOntario, Canada.\\n\\nGene expression profiling of rheumatoid arthritis (RA) and osteoarthritis (OA) \\njoint tissue samples provides a unique insight into the gene signatures involved \\nin disease development and progression. Fibroblast-like synovial (FLS) cells \\nwere obtained from RA, OA and control trauma joint tissues (non-RA, non-OA) and \\nRNA was analyzed by Affymetrix microarray. Thirty-four genes specific to RA and \\nOA FLS cells were identified (P<0.05). HOXD10, HOXD11, HOXD13, CCL8 and LIM \\nhomeobox 2 were highly and exclusively expressed in RA and CLU, \\nsarcoglycan-gamma, GPR64, POU3F3, peroxisome proliferative activated \\nreceptor-gamma and tripartite motif-containing 2 were expressed only in OA. The \\ndata also revealed expression heterogeneity for patients with the same disease. \\nTo address disease heterogeneity in RA FLS cells, we examined the effects of \\nclinical disease parameters (Health Assessment Questionnaire (HAQ) score, \\nC-reactive protein (CRP), erythrocyte sedimentation rate (ESR), rheumatoid \\nfactor (RF)) and drug therapies (methotrexate/prednisone) on RA FLS cell gene \\nexpression. Eight specific and unique correlations were identified: human \\nleukocyte antigen (HLA)-DQA2 with HAQ score; Clec12A with RF; MAB21L2, SIAT7E, \\nHAPLN1 and BAIAP2L1 with CRP level; RGMB and OSAP with ESR. Signature RA FLS \\ncell gene expression profiles may provide insights into disease pathogenesis and \\nhave utility in diagnosis, prognosis and drug responsiveness.\\n\\nDOI: 10.1038/sj.gene.6364400\\nPMID: 17568789 [Indexed for MEDLINE]\\n\\n\\n72. Tissue Antigens. 2005 May;65(5):448-58. doi:\\n10.1111/j.1399-0039.2005.00389.x.\\n\\nHigh-resolution genotyping of HLA-DQA1 in the GoKinD study and identification of \\nnovel alleles HLA-DQA1*040102, HLA-DQA1*0402 and HLA-DQA1*0404.\\n\\nCordovado SK(1), Hancock LN, Simone AE, Hendrix M, Mueller PW.\\n\\nAuthor information:\\n(1)Division of Laboratory Sciences, Molecular Biology Branch, National Center \\nfor Environmental Health, Centers for Disease Control and Prevention, 4770 \\nBuford Highway, Atlanta, GA 30341, USA. snc4@cdc.gov\\n\\nIn order to achieve high-resolution HLA-DQA1 genotyping, it is necessary to \\nidentify polymorphisms in exons 1, 2 and 3. We present a high-resolution \\nsequence-based typing (SBT) strategy for genotyping exons 1, 2 and 3 of the \\npolymorphic HLA-DQA1 locus. This method is an improvement upon previously \\npresented methods, because it utilizes the minimum number of SSP-PCR assays to \\nobtain clear DNA sequence in both the forward and reverse directions of all \\nthree exons. All known HLA-DQA1 alleles are resolved with the exception of \\nHLA-DQA1*010101 and HLA-DQA1*010102 for which the distinguishing polymorphism is \\nlocated in exon 4 and does not result in an amino acid change. This method has \\nenabled our laboratory to identify three new HLA-DQA1 alleles - HLA-DQA1*040102, \\nHLA- DQA1*0402 and HLA-DQA1*0404 - in the Genetics of Kidneys in Diabetes \\n(GoKinD) study population. Additionally, we present single-allele amplification \\nmethods, which identify the coding sequences of HLA-DQA1 exons 1, 2, 3, intron 2 \\nand 300 bp of the HLA-DQA1 promoter (QAP). This study, also describes the QAP \\nfor most of the known HLA-DQA1 alleles, three HLA-DQA2 promoter sequences and \\nthe intron 2 sequences for HLA-DQA1*040101, HLA-DQA1*040102, HLA-DQA1*0402 and \\nHLA-DQA1*0404.\\n\\nDOI: 10.1111/j.1399-0039.2005.00389.x\\nPMID: 15853899 [Indexed for MEDLINE]\\n\\n\\n73. Immunol Rev. 2001 Oct;183:76-85. doi: 10.1034/j.1600-065x.2001.1830106.x.\\n\\nDifferential evolutionary MHC class II strategies in humans and rhesus macaques: \\nrelevance for biomedical studies.\\n\\nDoxiadis GG(1), Otting N, de Groot NG, Bontrop RE.\\n\\nAuthor information:\\n(1)Department of Immunobiology, Biomedical Primate Research Centre, Rijswijk, \\nThe Netherlands. doxiadis@bprc.nl\\n\\nThe rhesus macaque is an important preclinical model in transplantation research \\nand in investigations of chronic and infectious diseases that need a \\nwell-characterised major histocompatibility complex (MHC-Mamu). In a large \\npopulation of pedigreed rhesus macaques, 70 Mamu-DRB, 18 -DQA1, 24 -DQB1, and 14 \\n-DPB1 alleles were detected. In humans, five HLA-DRB region configurations are \\npresent, displaying diversity with regard to number and combinations of loci. \\nThe HLA-DRB1 gene of each of these configurations is highly polymorphic. For \\nrhesus monkeys, at least 31 Mamu-DRB region configurations have been determined. \\nIn contrast to humans, most Mamu-DRB region configurations display no or only \\nlimited allelic polymorphism. Segregation analyses revealed 28 Mamu-DQA1/DQB1 \\npairs, each pair linked to a limited number of Mamu-DRB region configurations \\nand vice versa. In comparison with humans, the degree of freedom of \\nrecombination between Mamu-DQA1 and -DQB1 is extremely low and equivalents of \\nHLA-DQA2/DQB2 are absent. The Mamu-DPA1 gene is invariant and -DPB1 manifests \\nonly moderate allelic variation, whereas the HLA-DPA1 gene is oligomorphic and \\nHLA-DPB1 highly polymorphic. Thus, both species used different evolutionary \\nstrategies to create polymorphism and diversity at the MHC class II loci in \\norder to cope with pathogens.\\n\\nDOI: 10.1034/j.1600-065x.2001.1830106.x\\nPMID: 11782248 [Indexed for MEDLINE]\\n\\n\\n74. Hum Immunol. 2001 May;62(5):504-8. doi: 10.1016/s0198-8859(01)00236-1.\\n\\nAbsence of in vivo DNA-protein interactions in the DQA2 and DQB2 promoter \\nregions.\\n\\nIndovina P(1), Megiorni F, Fontemaggi G, Coni P, Mora B, Mazzilli MC.\\n\\nAuthor information:\\n(1)Dipartimento di Medicina Sperimentale e Patologia, Università \"La Sapienza,\", \\nRome, Italy.\\n\\nThe DQ subregion of the human major histocompatibility complex (HLA) contains \\ntwo pairs of loci: the DQA1/B1 genes (hereafter called DQ1), coding for the DQ \\nmolecules, and the DQA2/B2 pseudogenes (hereafter called DQ2). These pseudogenes \\nare highly homologous to the functional DQ1 genes and they have no apparent \\nabnormal features in their sequences that could prevent their activity. Only \\nrecently a low expression of the DQA2 gene has been observed whereas the DQB2 \\ntranscript was never found. The comparison between the DQ1 and DQ2 regulatory \\nsequences revealed several differences in their W, X, and Y cis-acting elements. \\nTo examine the DNA/protein interactions in the DQ promoter regions, we performed \\nin vivo footprinting experiments. Whereas the functional DQ1 loci showed a \\nseries of DNA-protein contact points in the X and Y boxes, the promoters of the \\nDQ2 pseudogenes displayed an unoccupied phenotype. These findings suggest that \\nthe very low level of DQA2 expression and the apparent lack of DQB2 activity are \\ncaused by the reduced binding affinity of specific transcription factors.\\n\\nDOI: 10.1016/s0198-8859(01)00236-1\\nPMID: 11334674 [Indexed for MEDLINE]\\n\\n\\n75. Ann N Y Acad Sci. 1998 May 13;841:355-9. doi: \\n10.1111/j.1749-6632.1998.tb10946.x.\\n\\nIdentification of a novel HLA class II association with DQB1*0502 in an Italian \\nmyasthenic population.\\n\\nBaggi F(1), Antozzi C, Andreetta F, Confalonieri P, Ciusani E, Begovich AB, \\nErlich HA, Cornelio F, Mantegazza R.\\n\\nAuthor information:\\n(1)Department of Neuromuscular Diseases, National Neurological Institute C. \\nBesta, Milan, Italy.\\n\\nDOI: 10.1111/j.1749-6632.1998.tb10946.x\\nPMID: 9668258 [Indexed for MEDLINE]\\n\\n\\n76. Yi Chuan Xue Bao. 1997 Oct;24(5):394-402.\\n\\n[Molecular evolution of MHC DQA genes. II. Phylogenetic analysis based on \\nnucleotide substitution and SCU bias].\\n\\n[Article in Chinese]\\n\\nPan X(1), Fu J.\\n\\nAuthor information:\\n(1)Department of Biology, Second Military Medical University, Shanghai.\\n\\nPhylogenetics of 23 alleles at MHC DQA loci in 7 mammalian species was studied \\nbased on their nucleotide (NT) substitution and synonymous codon usage (SCU) \\nbias. (1) It was demonstrated that the NT substitution rates are 1.0 x 10(-9) \\nNT/site/yr for exon2 and 1.3 x 10(-9) NT/site/yr for exon2-4 in a large time \\nscale, which is similar to other nuclear genes, while for mouse and rat the \\nrates are nearly twice as high as above mentioned. (2) The DQA locus diversity \\nand their interallelic diversity developed long after the radiation of mammalian \\n80Mya (million years ago). The bovine counterpart, of, and with the same recent \\nancestor of ovine DQA2, remains to be discovered. HLA-DQA2 locus split from \\nHLA-DQA1 ancestor at the time between 12 approximately 20 Mya while allele \\ndiversity of HLA-DQA1 emerged and developed from 24 Mya to less than 1 Mya. (3) \\nThe phylogenetic trees based on SCU divergence reflect the phylogenetics of MHC \\nDQA genes quite well generally in a new respect and reveal that HLA-DQA2 has a \\ndistinctive SCU bias different from all other MHC DQA locianalyzed. It indicates \\nthat SCU statistics plays an important and unique role in phylogenetic analysis \\nof orthologous genes. The method to estimate the SCU divergence and SCU \\nsimilarity was improved in this research.\\n\\nPMID: 9494291 [Indexed for MEDLINE]\\n\\n\\n77. J Immunol. 1997 Mar 1;158(5):2116-25.\\n\\nThe nonpolymorphic MHC class II isotype, HLA-DQA2, is expressed on the surface \\nof B lymphoblastoid cells.\\n\\nRudy GB(1), Lew AM.\\n\\nAuthor information:\\n(1)Walter and Eliza Hall Institute of Medical Research, Royal Melbourne \\nHospital, Parkville, Australia.\\n\\nThe more centromeric of the two human MHC class II DQA genes, DQA2, has not \\npreviously been shown to express a protein product. However, its high degree of \\nsequence similarity to DQA1, the fact that it is highly evolutionarily \\nconserved, and the recent demonstration of its transcription in some cell lines \\nsuggest that a DQA2 alpha-chain may be expressed. Polyclonal anti-peptide \\nantisera were generated and shown to be specific for DQA2 in Western blotting of \\nTriton X-114 preparations of B lymphoblastoid cell lines. DQA2 expression is \\nvariable, although always at least threefold less than that of DQA1, and is \\nfound in all haplotypes examined. Immunoprecipitations of biotin-labeled cell \\nsurface proteins reveal that DQA2 appears at the cell surface. In keeping with \\nothers\\' findings, no evidence for an expressed DQB2 beta-chain was found, \\nprompting investigation of the means by which DQA2 gains access to the cell \\nsurface. Antisera specific for allotypic and isotypic MHC class II beta-chains \\nwere used in sequential immunoprecipitation experiments to search for a \\nbeta-chain partner for DQA2, and a transfectant system was established in an \\nattempt to promote pairing of DQA2 with a selected DQB1 (*0501) chain. In \\nneither case was DQA2 found to dimerize with an MHC class II beta-chain. Despite \\nthis, DQA2 is capable of forming a complex with invariant chain.\\n\\nPMID: 9036956 [Indexed for MEDLINE]\\n\\n\\n78. Anim Genet. 1996 Oct;27(5):305-12. doi: 10.1111/j.1365-2052.1996.tb00974.x.\\n\\nPolymorphism at the ovine major histocompatibility complex class II loci.\\n\\nEscayg AP(1), Hickford JG, Montgomery GW, Dodds KG, Bullock DW.\\n\\nAuthor information:\\n(1)Department of Animal and Veterinary Sciences, Lincoln University, Canterbury, \\nNew Zealand.\\n\\nSouthern hybridization analysis of the ovine major histocompatibility complex \\n(MHC) (MhcOvar) class II region, using sheep-specific probes for the DQA1, DQA2, \\nDQB and DRA loci, has revealed extensive polymorphism. DQA1 and DQA2 had eight \\nand 16 alleles respectively, DQB had six and DRA had three alleles. Little \\ninformation was derived from the DRB locus owing to extensive \\ncross-hybridization between the DRB probe and the DQB locus. Differences in \\nallele frequency between breeds were revealed. At the DQA1 locus a null allele \\n(DQA1-N) was observed with a frequency of between 27% and 45%, making this the \\nmost common DQA1 allele in all breeds examined. The frequency of DQA1-N \\nhomozygotes was between 11% and 18%, raising questions as to the functional \\nsignificance of the DQA1 gene. Linkage analysis between the DQA1, DQA2, DQB and \\nDRA loci did not reveal any recombination.\\n\\nDOI: 10.1111/j.1365-2052.1996.tb00974.x\\nPMID: 8930070 [Indexed for MEDLINE]\\n\\n\\n79. Curr Opin Rheumatol. 1994 Nov;6(6):612-5. doi:\\n10.1097/00002281-199411000-00011.\\n\\nImmunologic aspects of scleroderma.\\n\\nWhite B(1).\\n\\nAuthor information:\\n(1)University of Maryland School of Medicine, Baltimore.\\n\\nMany distinct areas of investigation contribute to the understanding of immune \\nabnormalities in systemic sclerosis. Recent immunohistochemical studies question \\nthe causal relationship of increased transforming growth factor-beta expression \\nto dermal fibrosis. RNA polymerase I, II, and III have been identified as \\nautoantigens specific for systemic sclerosis. Anti-RNA polymerase antibodies are \\ndirected against both unique and shared subunits of the multiprotein complexes. \\nThe targeting of several subunits suggests that the entire complexes are \\nprocessed and presented by antigen-presenting cells. Genetic studies show that \\nboth HLA-DR and HLA-DQ genes control the antitopoisomerase response in Japanese \\npatients. The null allele of the complement component C4 and HLA-DQA2 have been \\nidentified as two independent disease susceptibility genes.\\n\\nDOI: 10.1097/00002281-199411000-00011\\nPMID: 7865382 [Indexed for MEDLINE]\\n\\n\\n80. Q J Med. 1994 Jun;87(6):337-41.\\n\\nTreatment of Graves\\' disease with the block-replace regimen of antithyroid \\ndrugs: the effect of treatment duration and immunogenetic susceptibility on \\nrelapse.\\n\\nWeetman AP(1), Pickerill AP, Watson P, Chatterjee VK, Edwards OM.\\n\\nAuthor information:\\n(1)Department of Medicine, University of Sheffield Clinical Sciences Centre, \\nNorthern General Hospital, UK.\\n\\nAntithyroid drugs are commonly used as first-line treatment for Graves\\' disease, \\nbut the optimum regimen for inducing remission remains unclear. We gave the \\nblock-replace regimen of carbimazole plus thyroxine to 100 patients for 6 or 12 \\nmonths, to determine whether prolonged treatment is associated with fewer \\nrelapses. The remission rate one year after cessation of treatment was 59% with \\nthe 6 month course and 65% with 12 months; this was not significantly different. \\nWe also analysed HLA markers identified by restriction fragment length \\npolymorphisms and could not confirm the recently reported associations of \\noutcome with HLA-DR4 or with an HLA-DQA2 allele. These results show that six \\nmonths treatment with a block-replace regimen of antithyroid drugs is probably \\nsufficient, in the UK, to achieve maximum remission of Graves\\' disease and that \\nthere are no HLA markers which clearly predict outcome.\\n\\nPMID: 7913766 [Indexed for MEDLINE]\\n\\n\\n81. Hum Immunol. 1994 Mar;39(3):225-9. doi: 10.1016/0198-8859(94)90264-x.\\n\\nLimited polymorphism of the HLA-DQA2 promoter and identification of a variant \\noctamer.\\n\\nRudy G(1), Lew AM.\\n\\nAuthor information:\\n(1)Walter and Eliza Hall Institute of Medical Research, Parkville, Australia.\\n\\nPrevious studies have suggested that the HLA-DQA2 gene may be associated with \\nIDDM. The apparently limited allelism at this locus prompted us to investigate \\nwhether this association might be with the level of gene expression rather than \\nwith specific alleles. The proximal promoter region of HLA-DQA2 was sequenced in \\nthree homozygous DR4;DQ8 subjects with IDDM, six homozygous DR3;DQ2 subjects \\n(three healthy controls and three with IDDM), and selected DR4 and DR6 cell \\nlines. This 388-bp region encompassed the known control W/Z/H/S, X, and Y boxes \\nand included a previously unremarked variant octamer sequence 40 bp upstream of \\nthe transcription start site. Only one polymorphic site was present among these \\n15 sequences, found in one DR3;DQ2 subject and a DR6;DQ6 cell line. This \\nindicates that any disease association with HLA-DQA2, at least among DR3;DQ2 \\nindividuals, cannot be accounted for solely by polymorphism of the proximal \\npromoter region.\\n\\nDOI: 10.1016/0198-8859(94)90264-x\\nPMID: 8026991 [Indexed for MEDLINE]\\n\\n\\n82. Anim Genet. 1993 Aug;24(4):249-55. doi: 10.1111/j.1365-2052.1993.tb00307.x.\\n\\nIsolation, characterization and evolution of ovine major histocompatibility \\ncomplex class II DRA and DQA genes.\\n\\nFabb SA(1), Maddox JF, Gogolin-Ewens KJ, Baker L, Wu MJ, Brandon MR.\\n\\nAuthor information:\\n(1)Centre for Animal Biotechnology, School of Veterinary Science, University of \\nMelbourne, Parkville, Victoria, Australia.\\n\\nFour full-length ovine major histocompatibility complex (MHC) class II A cDNA \\nclones coding for new alleles of DRA, DQA1 and DQA2 genes were isolated from two \\novine lambda gt10 cDNA libraries. The derived amino acid sequences of these \\nclones resemble class II A molecules from other species in both size and \\nstructure. Restriction fragment length polymorphism analysis, using an Ovar-DRA \\nprobe on DNA from Merino and Romney sheep revealed only limited polymorphism in \\ncontrast to the high levels of polymorphism revealed by Ovar-DQA probes. \\nComparison of the predicted amino acid sequences for the three ovine A genes \\nwith class II A genes from five other species revealed that the most variable \\nregion of the molecule is the signal peptide. Although virtually every amino \\nacid site shows variation, within or between species, there are some blocks of \\nhighly conserved residues. Within gene comparisons of nucleotide differences \\nreveal that the greatest number of changes is found between the alleles of \\nOvar-DQA1 and -DQA2 genes and the least between Ovar-DRA1 alleles. Phylogenetic \\nanalysis of class II A sequences from several species place DRA and DQA genes on \\ntwo distinct branches, with Ovar-DRA1 and BOLA-DRA, and Ovar-DQA1 and BOLA-DQA \\nbeing most similar on their respective branches.\\n\\nDOI: 10.1111/j.1365-2052.1993.tb00307.x\\nPMID: 7902039 [Indexed for MEDLINE]\\n\\n\\n83. Neurology. 1993 Jan;43(1):163-8. doi: 10.1212/wnl.43.1_part_1.163.\\n\\nMultiple sclerosis is associated with genes within or close to the HLA-DR-DQ \\nsubregion on a normal DR15,DQ6,Dw2 haplotype.\\n\\nHillert J(1), Olerup O.\\n\\nAuthor information:\\n(1)Center for BioTechnology, Karolinska Institute, NOVUM, Huddinge, Sweden.\\n\\nThe HLA class II haplotype DR15,DQ6,Dw2 is associated with an increased risk for \\nmultiple sclerosis (MS). It has previously been shown that this haplotype \\nextends to the HLA-DQA2 locus but not to the HLA-DP subregion. We report the \\ndistribution in 148 MS patients and 158 healthy controls of Taq I restriction \\nfragments of the C4, cytochrome P-21 (CYP21), and HLA-B genes. Approximately 90% \\nof the DR15,DQ6,Dw2 haplotypes extend to the C4/CYP21 loci and 50% to the HLA-B \\nlocus, indicated by a significant association of the haplotype with restriction \\nfragments of these loci. However, there was no independent association in MS \\npatients with these genes. This indicates that the susceptibility to MS is coded \\nfor by genes within or close to the HLA-DR-DQ subregion. Furthermore, the \\ndelineation of the DR15,DQ6,Dw2 haplotype is very similar in MS patients and \\ncontrol subjects. This observation supports the hypothesis that \"normal\" genes, \\ncommon to all carriers of the DR15,DQ6,Dw2 haplotype, confer the increased \\ngenetic susceptibility to MS associated with this haplotype.\\n\\nDOI: 10.1212/wnl.43.1_part_1.163\\nPMID: 8093808 [Indexed for MEDLINE]\\n\\n\\n84. J Immunol. 1992 Feb 15;148(4):1265-73.\\n\\nDifferential expression of related HLA class II DQ beta genes caused by \\nnucleotide variation in transcriptional regulatory elements.\\n\\nShewey LM(1), Beaty JS, Andersen LC, Nepom GT.\\n\\nAuthor information:\\n(1)Virginia Mason Research Center, Seattle, WA 98101.\\n\\nHLA class II genes comprise a large multigene family with intra- and interlocus \\nvariation in structure and expression. Within this family of related genes, the \\nHLA-DX alpha and beta loci (HLA DQA2 and DQB2) are highly homologous to \\nfunctional HLA-DQ loci (HLA DQA1 and DQB1) but are frequently termed pseudogenes \\nbecause DX gene transcription has not been observed, even in cells expressing \\nHLA-DQ. Analysis of upstream transcriptional regulatory elements for the DX beta \\nand DQ beta genes identified a high degree of nucleotide homology, consistent \\nwith their derivation from a common ancestral class II gene. However, transient \\nexpression assays with plasmids utilizing promoter elements from the DX beta \\ngene had no activity in transfected human B cells, in contrast to homologous DQ \\nbeta sequences. Reciprocal exchange of specific sequences from the DQ beta gene \\nwith those of the DX beta gene restored expression to wild-type DQ beta levels, \\nas did mutagenesis of only three DX-specific nucleotides in the upstream \\nregulatory region. These three nucleotides mark two binding sites for distinct \\nnuclear DNA-binding proteins that differentially recognize DQ beta and DX beta \\nsequences. Transcription of these genes is critically dependent on interactions \\nbetween these two upstream regulatory region sites which distinguish DX beta \\nfrom its closely related homologue, DQ beta.\\n\\nPMID: 1737939 [Indexed for MEDLINE]\\n\\n\\n85. J Immunol. 1991 Dec 15;147(12):4393-7.\\n\\nThe cryptic HLA-DQA2 (\"DX alpha\") gene is expressed in human B cell lines.\\n\\nYu LP(1), Sheehy MJ.\\n\\nAuthor information:\\n(1)Research Department, American Red Cross Blood Services, Madison, WI.\\n\\nAlthough there are two gene pairs in the HLA-DQ region, DQA1/B1 and DQA2/B2, the \\nlatter pair have never been found to be expressed. The DQA2 and DQB2 genes \\nappear to be normal at the DNA sequence level, there being no indication that \\nthey should be pseudogenes, and normal splicing occurs when DQA2 is included in \\na retroviral vector. The putative amino acid sequences of DQA2 and DQB2 have \\nbeen nearly invariant, raising the questions of how these genes have been \\nmaintained normal-appearing and nearly monomorphic if they are not expressed \\nand, if they are expressed, in what cell types are they expressed. We present \\nNorthern blot, nuclease protection assay, and mRNA sequence data that \\ndemonstrate the existence of spliced DQA2 mRNA in human B lymphoblastoid cell \\nlines. In contrast, we found no evidence of DQB2 mRNA in the same cell lines, \\nnor of DQA2 mRNA in the monocyte line U937 or in activated T cells.\\n\\nPMID: 1753107 [Indexed for MEDLINE]\\n\\n\\n86. Hum Immunol. 1991 Feb;30(2):136-9. doi: 10.1016/0198-8859(91)90082-k.\\n\\nLinkage disequilibrium between the HLA-DQA2 alleles and the HLA/DR/DQ complex.\\n\\nRatanachaiyavong S(1), Bidwell JL, Bidwell EA, McGregor AM.\\n\\nAuthor information:\\n(1)Department of Medicine, King\\'s College School of Medicine and Dentistry, \\nLondon, United Kingdom.\\n\\nA statistically significant association was observed between alleles of the \\nHLA-DQA2 and of the DR/DQ complex in a DNA-restriction fragment length \\npolymorphism study of 219 members from 21 multiplex families of patients with \\nhyperthyroid Graves\\' disease and 773 unrelated individuals selected for \\nhomozygosity of the HLA-DQA2 alleles. This data provides evidence for linkage \\ndisequilibrium rather than for a hot spot of recombination within the HLA-DQ \\nsubregion.\\n\\nDOI: 10.1016/0198-8859(91)90082-k\\nPMID: 1673673 [Indexed for MEDLINE]\\n\\n\\n87. Immunogenetics. 1991;34(2):69-79. doi: 10.1007/BF00211419.\\n\\nNucleotide sequence, polymorphism, and evolution of ovine MHC class II DQA \\ngenes.\\n\\nScott PC(1), Gogolin-Ewens KJ, Adams TE, Brandon MR.\\n\\nAuthor information:\\n(1)Centre for Animal Biotechnology, School of Veterinary Science, University of \\nMelbourne, Parkville, Victoria, Australia.\\n\\nThe nucleotide sequence of all exons and introns, excluding exon 1, of the ovine \\nmajor histocompatibility complex (MhcOvar) genes analogous to the HLA-DQA1 and \\n-DQA2 genes has been determined and the gene structure found to be similar to \\nthat reported for other species. The predicted amino acid sequences of the \\nOvar-DQA genes have been compared with the equivalent DQA genes in man, mouse, \\nrat, rabbit, and cattle and used to determine the evolutionary relationships of \\nthe sheep class II genes to these other species. Northern blot analysis of sheep \\nmRNA using exon specific probes for each of the two Ovar-DQA genes show that \\nboth genes are transcribed, whereas in humans there is no evidence that HLA-DQA2 \\nis transcriptionally active. Restriction fragment length polymorphisms (RFLPs) \\nhave been used to define a polymorphic series of alleles in both Ovar-DQA genes \\nand have indicated that the number of DQA genes is not constant in sheep as it \\nis in humans, but varies with the haplotype.\\n\\nDOI: 10.1007/BF00211419\\nPMID: 1678367 [Indexed for MEDLINE]\\n\\n\\n88. Hum Immunol. 1990 Dec;29(4):256-62. doi: 10.1016/0198-8859(90)90039-r.\\n\\nHLA-DQA2 (DX alpha) polymorphism and insulin dependent diabetes.\\n\\nRowe JR(1), Neme de Gimenez MH, Emler CA, Sheehy MJ.\\n\\nAuthor information:\\n(1)Research Department, American Red Cross Blood Services, Madison, WI 53705.\\n\\nA certain HLA-DQA2 locus TaqI fragment, DX alpha\"U\", has been reported to be \\nassociated with insulin-dependent diabetes mellitus (IDDM). Reports of various \\nstudies in this vein have ranged from stating that the association of DQA2\"U\" \\nwith IDDM exists even among subjects positive for HLA-DR3 and -DR4 to stating \\nthat the association of DQA2\"U\" with diabetes can be attributed to linkage \\ndisequilibrium between the DQA2\"U\" and some component(s) on the affected \\nhaplotypes. Using a synthetic 97-base probe corresponding to a portion of an \\nintron of DQA2, in a Southern blot analysis of IDDM and control subjects from \\nWisconsin, we were able to confirm the association of DQA2\"U\" with diabetes. \\nHowever, among DR3 subjects there was no significant association between DQA2\"U\" \\nand diabetes (p = 0.26). Although there was a (nonsignificant) association of \\nIDDM with DQA2\"U\" among DR4-positive subjects (p = 0.14), this can be completely \\nattributed to linkage disequilibrium between DQA2\"U\" and DQw8. We also sequenced \\nmost of the second exon (corresponding to the alpha 1 domain of the DQA2 \\nglycoprotein) from five individuals that were homozygous for either DQA2\"U\" or \\nDQA2\"L.\" The only polymorphisms observed were a \"silent\" mutation at position 36 \\nand one example of a difference that would result in a change of amino acid at \\nposition 41.\\n\\nDOI: 10.1016/0198-8859(90)90039-r\\nPMID: 1981060 [Indexed for MEDLINE]\\n\\n\\n89. Immunogenetics. 1990;31(4):229-32. doi: 10.1007/BF00204892.\\n\\nA SINE insertion provides information on the divergence of the HLA-DQA1 and \\nHLA-DQA2 genes.\\n\\nDel Pozzo G(1), Guardiola J.\\n\\nAuthor information:\\n(1)International Institute of Genetics and Biophysics, CNR, Napoli, Italy.\\n\\nThe class II region of the human major histocompatibility complex (MHC) contains \\na cluster of highly polymorphic genes organized into at least three subloci (DR, \\nDQ, and DP), each encoding a subset of surface antigens participating in the \\nmodulation of the immune response. Genetic diversity in this system is brought \\nabout by two major mechanisms, hypermutation and trans-species evolution. The DQ \\nsubregion contains a pair of closely related A genes, HLA-DQA1 and HLA-DQA2, \\nwhose phylogenetic relationship is uncertain, although their generation by \\nduplication of an ancestral A gene before or after speciation can be implied. We \\nreport here the presence of a member of the Alu repetitive family immediately 5\\' \\nto the HLA-DQA1 gene. The sequence of this element indicates that it may have \\nintegrated by transposition at the time of divergence of hominoids from Old \\nWorld monkeys. HLA-DQA2 carries an empty integration target site in place of the \\nAlu, thereby suggesting that the insertion of Alu near HLA-DQA1 was preceded by \\nthe separation of the two genes.\\n\\nDOI: 10.1007/BF00204892\\nPMID: 2158485 [Indexed for MEDLINE]\\n\\n\\n90. Immunogenetics. 1989;30(6):432-9. doi: 10.1007/BF02421175.\\n\\nRFLP analysis of the HLA-, ChLA-, and RhLA-DQ alpha chain gene regions: \\nconservation of restriction sites during evolution.\\n\\nBontrop RE(1), Otting N, Broos LA, Noort MC, Kenter M, Jonker M.\\n\\nAuthor information:\\n(1)TNO Primate Center, Rijswijk, The Netherlands.\\n\\nGenomic DNA samples, derived from a panel of 60 chimpanzees and 45 rhesus \\nmonkeys, were digested with the restriction enzymes Taq I and Bgl II and \\nhybridized with an HLA-DQ alpha chain cDNA probe. The results were compared with \\nthe data available on a human reference panel. Use of the restriction enzyme Taq \\nI and the DQ alpha chain probe allows the detection of five HLA-DQA1 and two \\nHLA-DQA2 gene-associated fragments within the human population. For the ChLA and \\nRhLA systems, 3 and 7 different DQA1-associated restriction patterns were \\ndetected, respectively, while for the chimpanzee a nonpolymorphic DQA2 (DX \\nalpha) gene-associated fragment was also observed. The equivalent of the HLA- \\nand ChLA-DQA2 genes appears to be absent in the rhesus monkey. The ChLA-DQA1 and \\n-DQA2 gene-associated RFLP patterns are identical in man and chimpanzee, whereas \\nsuch restriction site conservation is not seen in the rhesus monkey. The \\nconclusion drawn is that the genetic organization of the HLA-DQA and ChLA-DQA \\ngene regions, and possibly some of their allelic variabilities, already existed \\nbefore man and chimpanzee separated in evolution. Moreover, the particular \\nduplication which led to the generation of the HLA- and ChLA-DQA2 genes must \\nhave happened before speciation of members belonging to the superfamily \\nHominoidea (man, chimpanzee, etc), but probably after the separation of \\nsuperfamily Cercopitecoidea (rhesus monkeys, baboons, etc.) from Hominoidea.\\n\\nDOI: 10.1007/BF02421175\\nPMID: 2574158 [Indexed for MEDLINE]'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'HLA-DQA2',save='y', retmax=90) # Todos os resultados"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 20 resultados em pubmed. Irão ser processados 10 resultados.\n",
      "\n",
      "1. Genome Med. 2022 May 24;14(1):55. doi: 10.1186/s13073-022-01058-2.\n",
      "\n",
      "Fine-mapping studies distinguish genetic risks for childhood- and adult-onset \n",
      "asthma in the HLA region.\n",
      "\n",
      "Clay SM(1), Schoettler N(2), Goldstein AM(3), Carbonetto P(4), Dapas M(4), \n",
      "Altman MC(5)(6), Rosasco MG(6), Gern JE(7), Jackson DJ(7), Im HK(8), Stephens \n",
      "M(3), Nicolae DL(4)(3), Ober C(9).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "seleneclay@uchicago.edu.\n",
      "(2)Section of Pulmonary and Critical Care, Department of Medicine, University of \n",
      "Chicago, Chicago, IL, 60637, USA.\n",
      "(3)Department of Statistics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(4)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA.\n",
      "(5)Division of Allergy and Infectious Diseases, Department of Medicine, \n",
      "University of Washington, Seattle, WA, 98109, USA.\n",
      "(6)Systems Immunology Program, Benaroya Research Institute, Seattle, WA, 98101, \n",
      "USA.\n",
      "(7)Department of Pediatrics, University of Wisconsin, School of Medicine and \n",
      "Public Health, Madison, WI, 53706, USA.\n",
      "(8)Section of Genetic Medicine, Department of Medicine, University of Chicago, \n",
      "Chicago, IL, 60637, USA.\n",
      "(9)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \n",
      "c-ober@genetics.uchicago.edu.\n",
      "\n",
      "BACKGROUND: Genome-wide association studies of asthma have revealed robust \n",
      "associations with variation across the human leukocyte antigen (HLA) complex \n",
      "with independent associations in the HLA class I and class II regions for both \n",
      "childhood-onset asthma (COA) and adult-onset asthma (AOA). However, the specific \n",
      "variants and genes contributing to risk are unknown.\n",
      "METHODS: We used Bayesian approaches to perform genetic fine-mapping for COA and \n",
      "AOA (n=9432 and 21,556, respectively; n=318,167 shared controls) in White \n",
      "British individuals from the UK Biobank and to perform expression quantitative \n",
      "trait locus (eQTL) fine-mapping in immune (lymphoblastoid cell lines, n=398; \n",
      "peripheral blood mononuclear cells, n=132) and airway (nasal epithelial cells, \n",
      "n=188) cells from ethnically diverse individuals. We also examined putatively \n",
      "causal protein coding variation from protein crystal structures and conducted \n",
      "replication studies in independent multi-ethnic cohorts from the UK Biobank (COA \n",
      "n=1686; AOA n=3666; controls n=56,063).\n",
      "RESULTS: Genetic fine-mapping revealed both shared and distinct causal variation \n",
      "between COA and AOA in the class I region but only distinct causal variation in \n",
      "the class II region. Both gene expression levels and amino acid variation \n",
      "contributed to risk. Our results from eQTL fine-mapping and amino acid \n",
      "visualization suggested that the HLA-DQA1*03:01 allele and variation associated \n",
      "with expression of the nonclassical HLA-DQA2 and HLA-DQB2 genes accounted \n",
      "entirely for the most significant association with AOA in GWAS. Our studies also \n",
      "suggested a potentially prominent role for HLA-C protein coding variation in the \n",
      "class I region in COA. We replicated putatively causal variant associations in a \n",
      "multi-ethnic cohort.\n",
      "CONCLUSIONS: We highlight roles for both gene expression and protein coding \n",
      "variation in asthma risk and identified putatively causal variation and genes in \n",
      "the HLA region. A convergence of genomic, transcriptional, and protein coding \n",
      "evidence implicates the HLA-DQA2 and HLA-DQB2 genes and HLA-DQA1*03:01 allele in \n",
      "AOA.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s13073-022-01058-2\n",
      "PMCID: PMC9128203\n",
      "PMID: 35606880 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "2. Transl Psychiatry. 2022 Apr 18;12(1):163. doi: 10.1038/s41398-022-01913-1.\n",
      "\n",
      "Allele-specific analysis reveals exon- and cell-type-specific regulatory effects \n",
      "of Alzheimer's disease-associated genetic variants.\n",
      "\n",
      "He L(1), Loika Y(2), Kulminski AM(3).\n",
      "\n",
      "Author information:\n",
      "(1)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. lh235@duke.edu.\n",
      "(2)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA.\n",
      "(3)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \n",
      "University, Durham, NC, USA. alexander.kulminski@duke.edu.\n",
      "\n",
      "Elucidating regulatory effects of Alzheimer's disease (AD)-associated genetic \n",
      "variants is critical for unraveling their causal pathways and understanding the \n",
      "pathology. However, their cell-type-specific regulatory mechanisms in the brain \n",
      "remain largely unclear. Here, we conducted an analysis of allele-specific \n",
      "expression quantitative trait loci (aseQTLs) for 33 AD-associated variants in \n",
      "four brain regions and seven cell types using ~3000 bulk RNA-seq samples and \n",
      ">0.25 million single nuclei. We first develop a flexible hierarchical Poisson \n",
      "mixed model (HPMM) and demonstrate its superior statistical power to a \n",
      "beta-binomial model achieved by unifying samples in both allelic and \n",
      "genotype-level expression data. Using the HPMM, we identified 24 (~73%) aseQTLs \n",
      "in at least one brain region, including three new eQTLs associated with CA12, \n",
      "CHRNE, and CASS4. Notably, the APOE ε4 variant reduces APOE expression across \n",
      "all regions, even in AD-unaffected controls. Our results reveal region-dependent \n",
      "and exon-specific effects of multiple aseQTLs, such as rs2093760 with CR1, \n",
      "rs7982 with CLU, and rs3865444 with CD33. In an attempt to pinpoint the cell \n",
      "types responsible for the observed tissue-level aseQTLs using the snRNA-seq \n",
      "data, we detected many aseQTLs in microglia or monocytes associated with \n",
      "immune-related genes, including HLA-DQB1, HLA-DQA2, CD33, FCER1G, MS4A6A, SPI1, \n",
      "and BIN1, highlighting the regulatory role of AD-associated variants in the \n",
      "immune response. These findings provide further insights into potential causal \n",
      "pathways and cell types mediating the effects of the AD-associated variants.\n",
      "\n",
      "© 2022. The Author(s).\n",
      "\n",
      "DOI: 10.1038/s41398-022-01913-1\n",
      "PMCID: PMC9016079\n",
      "PMID: 35436980 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "3. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\n",
      "eCollection  2022.\n",
      "\n",
      "The Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren's \n",
      "Syndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\n",
      "\n",
      "Hong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \n",
      "Zhao T(2), Fu J(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "\n",
      "OBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \n",
      "of multiple sclerosis (MS) and Sjögren's syndrome (SS).\n",
      "METHODS: MS- and SS-related susceptibility genes and differentially expressed \n",
      "genes (DEGs) were identified by bioinformatics analysis based on genome-wide \n",
      "association studies (GWAS) and transcriptome data from GWAS catalog and Gene \n",
      "Expression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \n",
      "analysis, and protein-protein interaction analysis for susceptibility genes and \n",
      "DEGs were performed. The drugs targeting common pathways/genes were obtained \n",
      "through Comparative Toxicogenomics Database (CTD), DrugBank database, and \n",
      "Drug-Gene Interaction (DGI) Database. The target genes of \n",
      "approved/investigational drugs for MS and SS were obtained through DrugBank and \n",
      "compared with the common susceptibility genes.\n",
      "RESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \n",
      "(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \n",
      "HLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \n",
      "than two genes, and 28 common susceptibility pathways, with 15 drugs targeting \n",
      "three or more than three pathways. Based on transcriptome data, we found 3 hub \n",
      "common DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \n",
      "435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \n",
      "pathways and common risk pathways at the same time. There were 133 overlaps \n",
      "including JAK-STAT inhibitors between agents from GWAS and transcriptome data. \n",
      "Besides, we found that IL2RA and HLA-DRB1, identified as hub common \n",
      "susceptibility genes, were the targets of daclizumab and glatiramer that were \n",
      "used for MS, indicating that daclizumab and glatiramer may be therapeutic for \n",
      "SS.\n",
      "CONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \n",
      "signaling pathway played a vital role, which may be the genetic and molecular \n",
      "bases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \n",
      "therapies for MS and SS, especially for their comorbidity.\n",
      "\n",
      "Copyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.857014\n",
      "PMCID: PMC8959321\n",
      "PMID: 35356004 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "4. Front Immunol. 2022 Feb 16;13:814239. doi: 10.3389/fimmu.2022.814239. \n",
      "eCollection 2022.\n",
      "\n",
      "Global Characterization of Peripheral B Cells in Parkinson's Disease by \n",
      "Single-Cell RNA and BCR Sequencing.\n",
      "\n",
      "Wang P(1), Luo M(1), Zhou W(1), Jin X(1), Xu Z(1), Yan S(2), Li Y(1), Xu C(1), \n",
      "Cheng R(1), Huang Y(1), Lin X(1), Yao L(2), Nie H(1), Jiang Q(1)(3).\n",
      "\n",
      "Author information:\n",
      "(1)School of Life Science and Technology, Harbin Institute of Technology, \n",
      "Harbin, China.\n",
      "(2)Department of Neurology, First Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "(3)Key Laboratory of Biological Big Data (Harbin Institute of Technology), \n",
      "Ministry of Education, Harbin, China.\n",
      "\n",
      "Immune system plays important roles in the pathogenesis of Parkinson's disease \n",
      "(PD). However, the role of B cells in this complex disease are still not fully \n",
      "understood. B cells produce antibodies but can also regulate immune responses. \n",
      "In order to decode the relative contribution of peripheral B cell subtypes to \n",
      "the etiology of PD, we performed single cell RNA and BCR sequencing for 10,466 B \n",
      "cells from 8 PD patients and 6 age-matched healthy controls. We observed \n",
      "significant increased memory B cells and significant decreased naïve B cells in \n",
      "PD patients compared to healthy controls. Notably, we also discovered increased \n",
      "IgG and IgA isotypes and more frequent class switch recombination events in PD \n",
      "patients. Moreover, we identified preferential V and J gene segments of B cell \n",
      "receptors in PD patients as the evidence of convergent selection in PD. Finally, \n",
      "we found a marked clonal expanded memory B cell population in PD patients, \n",
      "up-regulating both MHC II genes (HLA-DRB5, HLA-DQA2 and HLA-DPB1) and \n",
      "transcription factor activator protein 1 (AP-1), suggesting that the antigen \n",
      "presentation capacity of B cells was enhanced and B cells were activated in PD \n",
      "patients. Overall, this study conducted a comprehensive analysis of peripheral B \n",
      "cell characteristics of PD patients, which provided novel insights into the \n",
      "humoral immune response in the pathogenesis of PD.\n",
      "\n",
      "Copyright © 2022 Wang, Luo, Zhou, Jin, Xu, Yan, Li, Xu, Cheng, Huang, Lin, Yao, \n",
      "Nie and Jiang.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.814239\n",
      "PMCID: PMC8888848\n",
      "PMID: 35250991 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "5. Comput Math Methods Med. 2021 Sep 21;2021:6015473. doi: 10.1155/2021/6015473. \n",
      "eCollection 2021.\n",
      "\n",
      "Comprehensive Analysis of RNA Expression Profile Identifies Hub miRNA-circRNA \n",
      "Interaction Networks in the Hypoxic Ischemic Encephalopathy.\n",
      "\n",
      "Wei L(#)(1), Li X(#)(1), Wang L(2), Song Y(3), Dong H(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neonatology, Jinan Maternal and Child Health Care Hospital, \n",
      "Shandong, China.\n",
      "(2)Department of Pediatrics, Jinan Maternal and Child Health Care Hospital, \n",
      "Shandong, China.\n",
      "(3)Department of Pediatrics, Chinese People's Liberation Army of 960 Hospital, \n",
      "Shandong, China.\n",
      "(#)Contributed equally\n",
      "\n",
      "Hypoxic ischemic encephalopathy (HIE) is classified as a sort of serious nervous \n",
      "system syndrome that occurs in the early life period. Noncoding RNAs had been \n",
      "confirmed to have crucial roles in human diseases. So far, there were few \n",
      "systematical and comprehensive studies towards the expression profile of RNAs in \n",
      "the brain after hypoxia ischemia. In this study, 31 differentially expressed \n",
      "microRNAs (miRNAs) with upregulation were identified. In addition, 5512 \n",
      "differentially expressed mRNAs, long noncoding RNAs (lncRNAs), and circular RNAs \n",
      "(circRNAs) were identified in HIE groups. Bioinformatics analysis showed these \n",
      "circRNAs and mRNAs were significantly enriched in regulation of leukocyte \n",
      "activation, response to virus, and neutrophil degranulation. Pathway and its \n",
      "related gene network analysis indicated that HLA - DPA1, HLA - DQA2, HLA - DQB1, \n",
      "and HLA - DRB4 have a more crucial role in HIE. Finally, miRNA-circRNA-mRNA \n",
      "interaction network analysis was also performed to identify hub miRNAs and \n",
      "circRNAs. We found that miR-592 potentially targeting 5 circRNAs, thus affecting \n",
      "15 mRNA expressions in HIR. hsa_circ_0068397 and hsa_circ_0045698 were \n",
      "identified as hub circRNAs in HIE. Collectively, using RNA-seq, bioinformatics \n",
      "analysis, and circRNA/miRNA interaction prediction, we systematically \n",
      "investigated the differentially expressed RNAs in HIE, which could give a new \n",
      "hint of understanding the pathogenesis of HIE.\n",
      "\n",
      "Copyright © 2021 Lin Wei et al.\n",
      "\n",
      "DOI: 10.1155/2021/6015473\n",
      "PMCID: PMC8481051\n",
      "PMID: 34603484 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: No conflict of interest.\n",
      "\n",
      "\n",
      "6. Restor Neurol Neurosci. 2021;39(4):247-266. doi: 10.3233/RNN-211161.\n",
      "\n",
      "Transcriptome analysis of human dorsal striatum implicates attenuated canonical \n",
      "WNT signaling in neuroinflammation and in age-related impairment of striatal \n",
      "neurogenesis and synaptic plasticity.\n",
      "\n",
      "Wu C(1), Bendriem RM(2), Freed WJ(3), Lee CT(4).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Molecular and Cellular Pharmacology, Miller School of Medicine, \n",
      "University of Miami, Miami, FL, USA.\n",
      "(2)Brain and Mind Research Institute, Weill Cornell Graduate School of Medical \n",
      "Sciences, New York, NY, USA.\n",
      "(3)Department of Biology, Lebanon Valley College, Annville, PA, USA.\n",
      "(4)Department of Neurology, Miller School of Medicine, University of Miami, \n",
      "Miami, FL, USA.\n",
      "\n",
      "BACKGROUND: Motor and cognitive decline as part of the normal aging process is \n",
      "linked to alterations in synaptic plasticity and reduction of adult neurogenesis \n",
      "in the dorsal striatum. Neuroinflammation, particularly in the form of \n",
      "microglial activation, is suggested to contribute to these age-associated \n",
      "changes.\n",
      "OBJECTIVE AND METHODS: To explore the molecular basis of alterations in striatal \n",
      "function during aging we analyzed RNA-Seq data for 117 postmortem human dorsal \n",
      "caudate samples and 97 putamen samples acquired through GTEx.\n",
      "RESULTS: Increased expression of neuroinflammatory transcripts including TREM2, \n",
      "MHC II molecules HLA-DMB, HLA-DQA2, HLA-DPA1, HLA-DPB1, HLA-DMA and HLA-DRA, \n",
      "complement genes C1QA, C1QB, CIQC and C3AR1, and MHCI molecules HLA-B and HLA-F \n",
      "was identified. We also identified down-regulation of transcripts involved in \n",
      "neurogenesis, synaptogenesis, and synaptic pruning, including DCX, CX3CL1, and \n",
      "CD200, and the canonical WNTs WNT7A, WNT7B, and WNT8A. The canonical WNT \n",
      "signaling pathway has previously been shown to mediate adult neurogenesis and \n",
      "synapse formation and growth. Recent findings also highlight the link between \n",
      "WNT/β-catenin signaling and inflammation pathways.\n",
      "CONCLUSIONS: These findings suggest that age-dependent attenuation of canonical \n",
      "WNT signaling plays a pivotal role in regulating striatal plasticity during \n",
      "aging. Dysregulation of WNT/β-catenin signaling via astrocyte-microglial \n",
      "interactions is suggested to be a novel mechanism that drives the decline of \n",
      "striatal neurogenesis and altered synaptic connectivity and plasticity, leading \n",
      "to a subsequent decrease in motor and cognitive performance with age. These \n",
      "findings may aid in the development of therapies targeting WNT/β-catenin \n",
      "signaling to combat cognitive and motor impairments associated with aging.\n",
      "\n",
      "DOI: 10.3233/RNN-211161\n",
      "PMID: 34275915 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. J Cell Mol Med. 2021 May;25(10):4684-4695. doi: 10.1111/jcmm.16407. Epub 2021 \n",
      "Mar 22.\n",
      "\n",
      "A single-cell survey of the human glomerulonephritis.\n",
      "\n",
      "Chen Z(1), Zhang T(2), Mao K(2), Shao X(1), Xu Y(1), Zhu M(1), Zhou H(1), Wang \n",
      "Q(1), Li Z(1), Xie Y(1), Yuan X(3), Ying L(3), Zhang M(3), Hu J(4), Mou S(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Nephrology, Molecular Cell Laboratory for Kidney Disease, Ren \n",
      "Ji Hospital, School of Medicine, Shanghai Jiao Tong University, Shanghai, China.\n",
      "(2)Shanghai Institute of Immunology, Department of Microbiology and Immunology, \n",
      "Shanghai Jiao Tong University School of Medicine (SJTU-SM), Shanghai, China.\n",
      "(3)Transplantation Center of Ren Ji Hospital, School of Medicine, Shanghai Jiao \n",
      "Tong University, Shanghai, China.\n",
      "(4)Department of Nuclear Medicine, Ruijin Hospital, Shanghai Jiao Tong \n",
      "University School of Medicine, Shanghai, China.\n",
      "\n",
      "Glomerulonephritis is the one of the major causes of the end-stage kidney \n",
      "disease, whereas the pathological process of glomerulonephritis is still not \n",
      "completely understood. Single-cell RNA sequencing (scRNA-seq) emerges to be a \n",
      "powerful tool to evaluate the full heterogeneity of kidney diseases. To reveal \n",
      "cellular gene expression profiles of glomerulonephritis, we performed scRNA-seq \n",
      "of 2 human kidney transplantation donor samples, 4 human glomerulonephritis \n",
      "samples, 1 human malignant hypertension (MH) sample and 1 human chronic \n",
      "interstitial nephritis (CIN) sample, all tissues were taken from the biopsy. \n",
      "After filtering the cells with < 200 genes and > 10% mitochondria (MT) genes, \n",
      "the resulting 14 932 cells can be divided into 20 cell clusters, consistently \n",
      "with the previous report, in disease samples dramatic immune cells infiltration \n",
      "was found, among which a proximal tubule (PT) subset characterized by wnt-β \n",
      "catenin activation and a natural killer T (NKT) subset high expressing LTB were \n",
      "found. Furthermore, in the cluster of the podocyte, three glomerulonephritis \n",
      "related genes named FXYD5, CD74 and B2M were found. Compared with the mesangial \n",
      "of donor, the gene CLIC1 and RPS26 were up-regulated in mesangial of IgA \n",
      "nephropathy(IgAN), whereas the gene JUNB was up-regulated in podocyte of IgAN in \n",
      "comparison with that of donor. Meanwhile, some membranous nephropathy (MN) high \n",
      "expressed genes such as HLA-DRB5, HLA-DQA2, IFNG, CCL2 and NR4A2, which involve \n",
      "in highest enrichment pathway, display the cellular-specific expression style, \n",
      "whereas monocyte marker of lupus nephritis (LN) named TNFSF13B was also found \n",
      "and interferon alpha/beta signalling pathway was enriched in B and NKT of LN \n",
      "comparing with donor. By scRNA-seq, we first defined the podocyte markers of \n",
      "glomerulonephritis and specific markers in IgA, MN and LN were found at cellular \n",
      "level. Furthermore, the critical role of interferon alpha/beta signalling \n",
      "pathway was enriched in B and NKT of LN was declared.\n",
      "\n",
      "© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \n",
      "Foundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/jcmm.16407\n",
      "PMCID: PMC8107090\n",
      "PMID: 33754492 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "8. Front Pharmacol. 2020 Jul 24;11:1138. doi: 10.3389/fphar.2020.01138.\n",
      "eCollection  2020.\n",
      "\n",
      "Discovery of Selenocysteine as a Potential Nanomedicine Promotes Cartilage \n",
      "Regeneration With Enhanced Immune Response by Text Mining and Biomedical \n",
      "Databases.\n",
      "\n",
      "Ye J(1), Xu B(1), Fan B(2), Zhang J(1), Yuan F(1), Chen Y(1), Sun Z(1), Yan \n",
      "X(1), Song Y(1), Song S(1), Yang M(2), Yu JK(1).\n",
      "\n",
      "Author information:\n",
      "(1)Knee Surgery Department of the Institution of Sports Medicine, Peking \n",
      "University Third Hospital, Beijing Key Laboratory of Sports Injuries, Beijing, \n",
      "China.\n",
      "(2)School of Clinical Medicine, Weifang Medical University, Weifang, China.\n",
      "\n",
      "BACKGROUND: Unlike bone tissue, little progress has been made regarding \n",
      "cartilage regeneration, and many challenges remain. Furthermore, the key roles \n",
      "of cartilage lesion caused by traumas, focal lesion, or articular overstress \n",
      "remain unclear. Traumatic injuries to the meniscus as well as its degeneration \n",
      "are important risk factors for long-term joint dysfunction, degenerative joint \n",
      "lesions, and knee osteoarthritis (OA) a chronic joint disease characterized by \n",
      "degeneration of articular cartilage and hyperosteogeny. Nearly 50% of the \n",
      "individuals with meniscus injuries develop OA over time. Due to the limited \n",
      "inherent self-repair capacity of cartilage lesion, the Biomaterial \n",
      "drug-nanomedicine is considered to be a promising alternative. Therefore, it is \n",
      "important to elucidate the gene potential regeneration mechanisms and discover \n",
      "novel precise medication, which are identified through this study to investigate \n",
      "their function and role in pathogenesis.\n",
      "METHODS: We downloaded the mRNA microarray statistics GSE117999, involving \n",
      "paired cartilage lesion tissue samples from 12 OA patients and 12 patients from \n",
      "a control group. First, we analyzed these statistics to recognize the \n",
      "differentially expressed genes (DEGs). We then exposed the gene ontology (GO) \n",
      "annotation and the Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathway \n",
      "enrichment analyses for these DEGs. Protein-protein interaction (PPI) networks \n",
      "were then constructed, from which we attained eight significant genes after a \n",
      "functional interaction analysis. Finally, we identified a potential nanomedicine \n",
      "attained from this assay set, using a wide range of inhibitor information \n",
      "archived in the Search Tool for the Retrieval of Interacting Genes (STRING) \n",
      "database.\n",
      "RESULTS: Sixty-six DEGs were identified with our standards for meaning (adjusted \n",
      "P-value < 0.01, |log2 - FC| ≥1.2). Furthermore, we identified eight hub genes \n",
      "and one potential nanomedicine - Selenocysteine based on these integrative data.\n",
      "CONCLUSION: We identified eight hub genes that could work as prospective \n",
      "biomarkers for the diagnostic and biomaterial drug treatment of cartilage \n",
      "lesion, involving the novel genes CAMP, DEFA3, TOLLIP, HLA-DQA2, SLC38A6, \n",
      "SLC3A1, FAM20A, and ANO8. Meanwhile, these genes were mainly associated with \n",
      "immune response, immune mediator induction, and cell chemotaxis. Significant \n",
      "support is provided for obtaining a series of novel gene targets, and we \n",
      "identify potential mechanisms for cartilage regeneration and final nanomedicine \n",
      "immunotherapy in regenerative medicine.\n",
      "\n",
      "Copyright © 2020 Ye, Xu, Fan, Zhang, Yuan, Chen, Sun, Yan, Song, Song, Yang and \n",
      "Yu.\n",
      "\n",
      "DOI: 10.3389/fphar.2020.01138\n",
      "PMCID: PMC7394085\n",
      "PMID: 32792959\n",
      "\n",
      "\n",
      "9. Sci Rep. 2020 Jul 2;10(1):10868. doi: 10.1038/s41598-020-67730-y.\n",
      "\n",
      "Synovial cell cross-talk with cartilage plays a major role in the pathogenesis \n",
      "of osteoarthritis.\n",
      "\n",
      "Chou CH(#)(1), Jain V(#)(1), Gibson J(1), Attarian DE(2), Haraden CA(1), Yohn \n",
      "CB(3), Laberge RM(3), Gregory S(1), Kraus VB(4)(5).\n",
      "\n",
      "Author information:\n",
      "(1)Duke Molecular Physiology Institute, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA.\n",
      "(2)Department of Orthopaedic Surgery, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA.\n",
      "(3)Unity Biotechnology, South San Francisco, CA, USA.\n",
      "(4)Duke Molecular Physiology Institute, Duke University School of Medicine, \n",
      "Durham, NC, 27701, USA. vbk@duke.edu.\n",
      "(5)Division of Rheumatology, Department of Medicine, Duke University School of \n",
      "Medicine, Durham, NC, 27701, USA. vbk@duke.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "We elucidated the molecular cross-talk between cartilage and synovium in \n",
      "osteoarthritis, the most widespread arthritis in the world, using the powerful \n",
      "tool of single-cell RNA-sequencing. Multiple cell types were identified based on \n",
      "profiling of 10,640 synoviocytes and 26,192 chondrocytes: 12 distinct synovial \n",
      "cell types and 7 distinct articular chondrocyte phenotypes from matched tissues. \n",
      "Intact cartilage was enriched for homeostatic and hypertrophic chondrocytes, \n",
      "while damaged cartilage was enriched for prefibro- and fibro-, regulatory, \n",
      "reparative and prehypertrophic chondrocytes. A total of 61 cytokines and growth \n",
      "factors were predicted to regulate the 7 chondrocyte cell phenotypes. Based on \n",
      "production by > 1% of cells, 55% of the cytokines were produced by synovial \n",
      "cells (39% exclusive to synoviocytes and not expressed by chondrocytes) and \n",
      "their presence in osteoarthritic synovial fluid confirmed. The synoviocytes \n",
      "producing IL-1beta (a classic pathogenic cytokine in osteoarthritis), mainly \n",
      "inflammatory macrophages and dendritic cells, were characterized by \n",
      "co-expression of surface proteins corresponding to HLA-DQA1, HLA-DQA2, OLR1 or \n",
      "TLR2. Strategies to deplete these pathogenic intra-articular cell subpopulations \n",
      "could be a therapeutic option for human osteoarthritis.\n",
      "\n",
      "DOI: 10.1038/s41598-020-67730-y\n",
      "PMCID: PMC7331607\n",
      "PMID: 32616761 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: C.H.C., V.J., J.G., D.E.A., C.A.H. and S.G. \n",
      "declare no competing interests. C.B.Y. and R.M.L. are employees of Unity \n",
      "Biotechnology and played a role in the conceptualization of the study and in \n",
      "preparation of the manuscript. This work was funded in part by Unity \n",
      "Biotechnology, however, Unity Biotechnology had no role in the preparation of \n",
      "the manuscript, data collection or decision to publish this study. V.B.K. has \n",
      "been a periodic consultant for Unity Biotechnology providing periodic advice on \n",
      "clinical study design, unrelated to this study and has no competing interests to \n",
      "declare.\n",
      "\n",
      "\n",
      "10. Medicina (Kaunas). 2020 Mar 31;56(4):154. doi: 10.3390/medicina56040154.\n",
      "\n",
      "An Intergenic rs9275596 Polymorphism on Chr. 6p21 Is Associated with Multiple \n",
      "Sclerosis in Latvians.\n",
      "\n",
      "Paramonova N(1), Trapina I(1), Dokane K(1), Kalnina J(1), Sjakste T(1), Sjakste \n",
      "N(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Genomics and Bioinformatics, Institute of Biology of the University of \n",
      "Latvia, LV-1004 Riga, Latvia.\n",
      "(2)Department of Medical Biochemistry of the University of Latvia, LV-1004 Riga, \n",
      "Latvia.\n",
      "\n",
      "Background and objectives: Multiple sclerosis (MS) is a chronic inflammatory \n",
      "disease of the central nervous system, leading to demyelination of neurons and \n",
      "potentially debilitating physical and mental symptoms. The disease is more \n",
      "prevalent in women than in men. The major histocompatibility complex (MHC) \n",
      "region has been identified as a major genetic determinant for autoimmune \n",
      "diseases, and its role in some neurological disorders including MS was \n",
      "evaluated. An intergenic single-nucleotide polymorphism (SNP), rs9275596, \n",
      "located between the HLA-DQB1 and HLA-DQA2 genes, is in significant association \n",
      "with various autoimmune diseases according to genome-wide association studies \n",
      "(GWASs). A cumulative effect of this SNP with other polymorphisms from this \n",
      "region was revealed. The aim of the study was to verify the data on rs9275596 \n",
      "association in multiple sclerosis in a case/control study of the Latvian \n",
      "population and to evaluate eventual functional significance of allele \n",
      "substitutions. Materials and Methods: rs9275596 (chr6:32713854; GRCh38.p12) was \n",
      "genotyped in 273 MS patients and 208 controls on main and sex-specific \n",
      "associations. Eventual functional significance of allele substitutions was \n",
      "evaluated in silico using publicly available tools. Results: The rs9275596 rare \n",
      "alleles were identified as a disease susceptibility factor in association with \n",
      "the MS main group and in affected females (p < 0.001 and p < 0.01, \n",
      "respectively). Risk factor genotypes with rare alleles included were associated \n",
      "with the MS common cohort (p < 0.002) and female cohort (odds ratio, OR = 2.24) \n",
      "and were identified as disease susceptible in males (OR = 2.41). It was shown \n",
      "that structural changes of rs9275596 affect the secondary structure of DNA. \n",
      "Functional significance of allele substitutions was evaluated on the eventual \n",
      "sequence affinity to transcription factors (TFs) and splicing signals \n",
      "similarity. A possible impact of the particular polymorphisms on the \n",
      "transcription and splicing efficiency is discussed. Conclusions: Our results \n",
      "suggest susceptibility of rs9275596 to multiple sclerosis in Latvians.\n",
      "\n",
      "DOI: 10.3390/medicina56040154\n",
      "PMCID: PMC7230508\n",
      "PMID: 32244438 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no conflict \n",
      "of interest.\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1. Genome Med. 2022 May 24;14(1):55. doi: 10.1186/s13073-022-01058-2.\\n\\nFine-mapping studies distinguish genetic risks for childhood- and adult-onset \\nasthma in the HLA region.\\n\\nClay SM(1), Schoettler N(2), Goldstein AM(3), Carbonetto P(4), Dapas M(4), \\nAltman MC(5)(6), Rosasco MG(6), Gern JE(7), Jackson DJ(7), Im HK(8), Stephens \\nM(3), Nicolae DL(4)(3), Ober C(9).\\n\\nAuthor information:\\n(1)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \\nseleneclay@uchicago.edu.\\n(2)Section of Pulmonary and Critical Care, Department of Medicine, University of \\nChicago, Chicago, IL, 60637, USA.\\n(3)Department of Statistics, University of Chicago, Chicago, IL, 60637, USA.\\n(4)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA.\\n(5)Division of Allergy and Infectious Diseases, Department of Medicine, \\nUniversity of Washington, Seattle, WA, 98109, USA.\\n(6)Systems Immunology Program, Benaroya Research Institute, Seattle, WA, 98101, \\nUSA.\\n(7)Department of Pediatrics, University of Wisconsin, School of Medicine and \\nPublic Health, Madison, WI, 53706, USA.\\n(8)Section of Genetic Medicine, Department of Medicine, University of Chicago, \\nChicago, IL, 60637, USA.\\n(9)Department of Human Genetics, University of Chicago, Chicago, IL, 60637, USA. \\nc-ober@genetics.uchicago.edu.\\n\\nBACKGROUND: Genome-wide association studies of asthma have revealed robust \\nassociations with variation across the human leukocyte antigen (HLA) complex \\nwith independent associations in the HLA class I and class II regions for both \\nchildhood-onset asthma (COA) and adult-onset asthma (AOA). However, the specific \\nvariants and genes contributing to risk are unknown.\\nMETHODS: We used Bayesian approaches to perform genetic fine-mapping for COA and \\nAOA (n=9432 and 21,556, respectively; n=318,167 shared controls) in White \\nBritish individuals from the UK Biobank and to perform expression quantitative \\ntrait locus (eQTL) fine-mapping in immune (lymphoblastoid cell lines, n=398; \\nperipheral blood mononuclear cells, n=132) and airway (nasal epithelial cells, \\nn=188) cells\\xa0from ethnically diverse individuals. We also examined putatively \\ncausal protein coding variation from protein crystal structures and conducted \\nreplication studies in independent multi-ethnic cohorts from the UK Biobank (COA \\nn=1686; AOA n=3666; controls n=56,063).\\nRESULTS: Genetic fine-mapping revealed both shared and distinct causal variation \\nbetween COA and AOA in the class I region but only distinct causal variation in \\nthe class II region. Both gene expression levels and amino acid variation \\ncontributed to risk. Our results from eQTL fine-mapping and amino acid \\nvisualization suggested that the HLA-DQA1*03:01 allele and variation associated \\nwith expression of the nonclassical HLA-DQA2 and HLA-DQB2 genes accounted \\nentirely for the most significant association with AOA in GWAS. Our studies also \\nsuggested a potentially prominent role for HLA-C protein coding variation in the \\nclass I region in COA. We replicated putatively causal variant associations in a \\nmulti-ethnic cohort.\\nCONCLUSIONS: We highlight roles for both gene expression and protein coding \\nvariation in asthma risk and identified putatively causal variation and genes in \\nthe HLA region. A convergence of genomic, transcriptional, and protein coding \\nevidence implicates the HLA-DQA2 and HLA-DQB2 genes and HLA-DQA1*03:01 allele in \\nAOA.\\n\\n© 2022. The Author(s).\\n\\nDOI: 10.1186/s13073-022-01058-2\\nPMCID: PMC9128203\\nPMID: 35606880 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n2. Transl Psychiatry. 2022 Apr 18;12(1):163. doi: 10.1038/s41398-022-01913-1.\\n\\nAllele-specific analysis reveals exon- and cell-type-specific regulatory effects \\nof Alzheimer\\'s disease-associated genetic variants.\\n\\nHe L(1), Loika Y(2), Kulminski AM(3).\\n\\nAuthor information:\\n(1)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA. lh235@duke.edu.\\n(2)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA.\\n(3)Biodemography of Aging Research Unit, Social Science Research Institute, Duke \\nUniversity, Durham, NC, USA. alexander.kulminski@duke.edu.\\n\\nElucidating regulatory effects of Alzheimer\\'s disease (AD)-associated genetic \\nvariants is critical for unraveling their causal pathways and understanding the \\npathology. However, their cell-type-specific regulatory mechanisms in the brain \\nremain largely unclear. Here, we conducted an analysis of allele-specific \\nexpression quantitative trait loci (aseQTLs) for 33 AD-associated variants in \\nfour brain regions and seven cell types using ~3000 bulk RNA-seq samples and \\n>0.25 million single nuclei. We first develop a flexible hierarchical Poisson \\nmixed model (HPMM) and demonstrate its superior statistical power to a \\nbeta-binomial model achieved by unifying samples in both allelic and \\ngenotype-level expression data. Using the HPMM, we identified 24 (~73%) aseQTLs \\nin at least one brain region, including three new eQTLs associated with CA12, \\nCHRNE, and CASS4. Notably, the APOE ε4 variant reduces APOE expression across \\nall regions, even in AD-unaffected controls. Our results reveal region-dependent \\nand exon-specific effects of multiple aseQTLs, such as rs2093760 with CR1, \\nrs7982 with CLU, and rs3865444 with CD33. In an attempt to pinpoint the cell \\ntypes responsible for the observed tissue-level aseQTLs using the snRNA-seq \\ndata, we detected many aseQTLs in microglia or monocytes associated with \\nimmune-related genes, including HLA-DQB1, HLA-DQA2, CD33, FCER1G, MS4A6A, SPI1, \\nand BIN1, highlighting the regulatory role of AD-associated variants in the \\nimmune response. These findings provide further insights into potential causal \\npathways and cell types mediating the effects of the AD-associated variants.\\n\\n© 2022. The Author(s).\\n\\nDOI: 10.1038/s41398-022-01913-1\\nPMCID: PMC9016079\\nPMID: 35436980 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n3. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\\neCollection  2022.\\n\\nThe Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren\\'s \\nSyndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\\n\\nHong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \\nZhao T(2), Fu J(1).\\n\\nAuthor information:\\n(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n\\nOBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \\nof multiple sclerosis (MS) and Sjögren\\'s syndrome (SS).\\nMETHODS: MS- and SS-related susceptibility genes and differentially expressed \\ngenes (DEGs) were identified by bioinformatics analysis based on genome-wide \\nassociation studies (GWAS) and transcriptome data from GWAS catalog and Gene \\nExpression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \\nanalysis, and protein-protein interaction analysis for susceptibility genes and \\nDEGs were performed. The drugs targeting common pathways/genes were obtained \\nthrough Comparative Toxicogenomics Database (CTD), DrugBank database, and \\nDrug-Gene Interaction (DGI) Database. The target genes of \\napproved/investigational drugs for MS and SS were obtained through DrugBank and \\ncompared with the common susceptibility genes.\\nRESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \\n(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \\nHLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \\nthan two genes, and 28 common susceptibility pathways, with 15 drugs targeting \\nthree or more than three pathways. Based on transcriptome data, we found 3 hub \\ncommon DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \\n435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \\npathways and common risk pathways at the same time. There were 133 overlaps \\nincluding JAK-STAT inhibitors between agents from GWAS and transcriptome data. \\nBesides, we found that IL2RA and HLA-DRB1, identified as hub common \\nsusceptibility genes, were the targets of daclizumab and glatiramer that were \\nused for MS, indicating that daclizumab and glatiramer may be therapeutic for \\nSS.\\nCONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \\nsignaling pathway played a vital role, which may be the genetic and molecular \\nbases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \\ntherapies for MS and SS, especially for their comorbidity.\\n\\nCopyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\\n\\nDOI: 10.3389/fimmu.2022.857014\\nPMCID: PMC8959321\\nPMID: 35356004 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n4. Front Immunol. 2022 Feb 16;13:814239. doi: 10.3389/fimmu.2022.814239. \\neCollection 2022.\\n\\nGlobal Characterization of Peripheral B Cells in Parkinson\\'s Disease by \\nSingle-Cell RNA and BCR Sequencing.\\n\\nWang P(1), Luo M(1), Zhou W(1), Jin X(1), Xu Z(1), Yan S(2), Li Y(1), Xu C(1), \\nCheng R(1), Huang Y(1), Lin X(1), Yao L(2), Nie H(1), Jiang Q(1)(3).\\n\\nAuthor information:\\n(1)School of Life Science and Technology, Harbin Institute of Technology, \\nHarbin, China.\\n(2)Department of Neurology, First Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n(3)Key Laboratory of Biological Big Data (Harbin Institute of Technology), \\nMinistry of Education, Harbin, China.\\n\\nImmune system plays important roles in the pathogenesis of Parkinson\\'s disease \\n(PD). However, the role of B cells in this complex disease are still not fully \\nunderstood. B cells produce antibodies but can also regulate immune responses. \\nIn order to decode the relative contribution of peripheral B cell subtypes to \\nthe etiology of PD, we performed single cell RNA and BCR sequencing for 10,466 B \\ncells from 8 PD patients and 6 age-matched healthy controls. We observed \\nsignificant increased memory B cells and significant decreased naïve B cells in \\nPD patients compared to healthy controls. Notably, we also discovered increased \\nIgG and IgA isotypes and more frequent class switch recombination events in PD \\npatients. Moreover, we identified preferential V and J gene segments of B cell \\nreceptors in PD patients as the evidence of convergent selection in PD. Finally, \\nwe found a marked clonal expanded memory B cell population in PD patients, \\nup-regulating both MHC II genes (HLA-DRB5, HLA-DQA2 and HLA-DPB1) and \\ntranscription factor activator protein 1 (AP-1), suggesting that the antigen \\npresentation capacity of B cells was enhanced and B cells were activated in PD \\npatients. Overall, this study conducted a comprehensive analysis of peripheral B \\ncell characteristics of PD patients, which provided novel insights into the \\nhumoral immune response in the pathogenesis of PD.\\n\\nCopyright © 2022 Wang, Luo, Zhou, Jin, Xu, Yan, Li, Xu, Cheng, Huang, Lin, Yao, \\nNie and Jiang.\\n\\nDOI: 10.3389/fimmu.2022.814239\\nPMCID: PMC8888848\\nPMID: 35250991 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n5. Comput Math Methods Med. 2021 Sep 21;2021:6015473. doi: 10.1155/2021/6015473. \\neCollection 2021.\\n\\nComprehensive Analysis of RNA Expression Profile Identifies Hub miRNA-circRNA \\nInteraction Networks in the Hypoxic Ischemic Encephalopathy.\\n\\nWei L(#)(1), Li X(#)(1), Wang L(2), Song Y(3), Dong H(1).\\n\\nAuthor information:\\n(1)Department of Neonatology, Jinan Maternal and Child Health Care Hospital, \\nShandong, China.\\n(2)Department of Pediatrics, Jinan Maternal and Child Health Care Hospital, \\nShandong, China.\\n(3)Department of Pediatrics, Chinese People\\'s Liberation Army of 960 Hospital, \\nShandong, China.\\n(#)Contributed equally\\n\\nHypoxic ischemic encephalopathy (HIE) is classified as a sort of serious nervous \\nsystem syndrome that occurs in the early life period. Noncoding RNAs had been \\nconfirmed to have crucial roles in human diseases. So far, there were few \\nsystematical and comprehensive studies towards the expression profile of RNAs in \\nthe brain after hypoxia ischemia. In this study, 31 differentially expressed \\nmicroRNAs (miRNAs) with upregulation were identified. In addition, 5512 \\ndifferentially expressed mRNAs, long noncoding RNAs (lncRNAs), and circular RNAs \\n(circRNAs) were identified in HIE groups. Bioinformatics analysis showed these \\ncircRNAs and mRNAs were significantly enriched in regulation of leukocyte \\nactivation, response to virus, and neutrophil degranulation. Pathway and its \\nrelated gene network analysis indicated that HLA\\u2009-\\u2009DPA1, HLA\\u2009-\\u2009DQA2, HLA\\u2009-\\u2009DQB1, \\nand HLA\\u2009-\\u2009DRB4 have a more crucial role in HIE. Finally, miRNA-circRNA-mRNA \\ninteraction network analysis was also performed to identify hub miRNAs and \\ncircRNAs. We found that miR-592 potentially targeting 5 circRNAs, thus affecting \\n15 mRNA expressions in HIR. hsa_circ_0068397 and hsa_circ_0045698 were \\nidentified as hub circRNAs in HIE. Collectively, using RNA-seq, bioinformatics \\nanalysis, and circRNA/miRNA interaction prediction, we systematically \\ninvestigated the differentially expressed RNAs in HIE, which could give a new \\nhint of understanding the pathogenesis of HIE.\\n\\nCopyright © 2021 Lin Wei et al.\\n\\nDOI: 10.1155/2021/6015473\\nPMCID: PMC8481051\\nPMID: 34603484 [Indexed for MEDLINE]\\n\\nConflict of interest statement: No conflict of interest.\\n\\n\\n6. Restor Neurol Neurosci. 2021;39(4):247-266. doi: 10.3233/RNN-211161.\\n\\nTranscriptome analysis of human dorsal striatum implicates attenuated canonical \\nWNT signaling in neuroinflammation and in age-related impairment of striatal \\nneurogenesis and synaptic plasticity.\\n\\nWu C(1), Bendriem RM(2), Freed WJ(3), Lee CT(4).\\n\\nAuthor information:\\n(1)Department of Molecular and Cellular Pharmacology, Miller School of Medicine, \\nUniversity of Miami, Miami, FL, USA.\\n(2)Brain and Mind Research Institute, Weill Cornell Graduate School of Medical \\nSciences, New York, NY, USA.\\n(3)Department of Biology, Lebanon Valley College, Annville, PA, USA.\\n(4)Department of Neurology, Miller School of Medicine, University of Miami, \\nMiami, FL, USA.\\n\\nBACKGROUND: Motor and cognitive decline as part of the normal aging process is \\nlinked to alterations in synaptic plasticity and reduction of adult neurogenesis \\nin the dorsal striatum. Neuroinflammation, particularly in the form of \\nmicroglial activation, is suggested to contribute to these age-associated \\nchanges.\\nOBJECTIVE AND METHODS: To explore the molecular basis of alterations in striatal \\nfunction during aging we analyzed RNA-Seq data for 117 postmortem human dorsal \\ncaudate samples and 97 putamen samples acquired through GTEx.\\nRESULTS: Increased expression of neuroinflammatory transcripts including TREM2, \\nMHC II molecules HLA-DMB, HLA-DQA2, HLA-DPA1, HLA-DPB1, HLA-DMA and HLA-DRA, \\ncomplement genes C1QA, C1QB, CIQC and C3AR1, and MHCI molecules HLA-B and HLA-F \\nwas identified. We also identified down-regulation of transcripts involved in \\nneurogenesis, synaptogenesis, and synaptic pruning, including DCX, CX3CL1, and \\nCD200, and the canonical WNTs WNT7A, WNT7B, and WNT8A. The canonical WNT \\nsignaling pathway has previously been shown to mediate adult neurogenesis and \\nsynapse formation and growth. Recent findings also highlight the link between \\nWNT/β-catenin signaling and inflammation pathways.\\nCONCLUSIONS: These findings suggest that age-dependent attenuation of canonical \\nWNT signaling plays a pivotal role in regulating striatal plasticity during \\naging. Dysregulation of WNT/β-catenin signaling via astrocyte-microglial \\ninteractions is suggested to be a novel mechanism that drives the decline of \\nstriatal neurogenesis and altered synaptic connectivity and plasticity, leading \\nto a subsequent decrease in motor and cognitive performance with age. These \\nfindings may aid in the development of therapies targeting WNT/β-catenin \\nsignaling to combat cognitive and motor impairments associated with aging.\\n\\nDOI: 10.3233/RNN-211161\\nPMID: 34275915 [Indexed for MEDLINE]\\n\\n\\n7. J Cell Mol Med. 2021 May;25(10):4684-4695. doi: 10.1111/jcmm.16407. Epub 2021 \\nMar 22.\\n\\nA single-cell survey of the human glomerulonephritis.\\n\\nChen Z(1), Zhang T(2), Mao K(2), Shao X(1), Xu Y(1), Zhu M(1), Zhou H(1), Wang \\nQ(1), Li Z(1), Xie Y(1), Yuan X(3), Ying L(3), Zhang M(3), Hu J(4), Mou S(1).\\n\\nAuthor information:\\n(1)Department of Nephrology, Molecular Cell Laboratory for Kidney Disease, Ren \\nJi Hospital, School of Medicine, Shanghai Jiao Tong University, Shanghai, China.\\n(2)Shanghai Institute of Immunology, Department of Microbiology and Immunology, \\nShanghai Jiao Tong University School of Medicine (SJTU-SM), Shanghai, China.\\n(3)Transplantation Center of Ren Ji Hospital, School of Medicine, Shanghai Jiao \\nTong University, Shanghai, China.\\n(4)Department of Nuclear Medicine, Ruijin Hospital, Shanghai Jiao Tong \\nUniversity School of Medicine, Shanghai, China.\\n\\nGlomerulonephritis is the one of the major causes of the end-stage kidney \\ndisease, whereas the pathological process of glomerulonephritis is still not \\ncompletely understood. Single-cell RNA sequencing (scRNA-seq) emerges to be a \\npowerful tool to evaluate the full heterogeneity of kidney diseases. To reveal \\ncellular gene expression profiles of glomerulonephritis, we performed scRNA-seq \\nof 2 human kidney transplantation donor samples, 4 human glomerulonephritis \\nsamples, 1 human malignant hypertension (MH) sample and 1 human chronic \\ninterstitial nephritis (CIN) sample, all tissues were taken from the biopsy. \\nAfter filtering the cells with\\xa0<\\xa0200 genes and\\xa0>\\xa010% mitochondria (MT) genes, \\nthe resulting 14\\xa0932 cells can be divided into 20 cell clusters, consistently \\nwith the previous report, in disease samples dramatic immune cells infiltration \\nwas found, among which a proximal tubule (PT) subset characterized by wnt-β \\ncatenin activation and a natural killer T (NKT) subset high expressing LTB were \\nfound. Furthermore, in the cluster of the podocyte, three glomerulonephritis \\nrelated genes named FXYD5, CD74 and B2M were found. Compared with the mesangial \\nof donor, the gene CLIC1 and RPS26 were up-regulated in mesangial of IgA \\nnephropathy(IgAN), whereas the gene JUNB was up-regulated in podocyte of IgAN in \\ncomparison with that of donor. Meanwhile, some membranous nephropathy (MN) high \\nexpressed genes such as HLA-DRB5, HLA-DQA2, IFNG, CCL2 and NR4A2, which involve \\nin highest enrichment pathway, display the cellular-specific expression style, \\nwhereas monocyte marker of lupus nephritis (LN) named TNFSF13B was also found \\nand interferon alpha/beta signalling pathway was enriched in B and NKT of LN \\ncomparing with donor. By scRNA-seq, we first defined the podocyte markers of \\nglomerulonephritis and specific markers in IgA, MN and LN were found at cellular \\nlevel. Furthermore, the critical role of interferon alpha/beta signalling \\npathway was enriched in B and NKT of LN was declared.\\n\\n© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \\nFoundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/jcmm.16407\\nPMCID: PMC8107090\\nPMID: 33754492 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n8. Front Pharmacol. 2020 Jul 24;11:1138. doi: 10.3389/fphar.2020.01138.\\neCollection  2020.\\n\\nDiscovery of Selenocysteine as a\\xa0Potential Nanomedicine Promotes Cartilage \\nRegeneration With Enhanced Immune Response by Text Mining and Biomedical \\nDatabases.\\n\\nYe J(1), Xu B(1), Fan B(2), Zhang J(1), Yuan F(1), Chen Y(1), Sun Z(1), Yan \\nX(1), Song Y(1), Song S(1), Yang M(2), Yu JK(1).\\n\\nAuthor information:\\n(1)Knee Surgery Department of the Institution of Sports Medicine, Peking \\nUniversity Third Hospital, Beijing Key Laboratory of Sports Injuries, Beijing, \\nChina.\\n(2)School of Clinical Medicine, Weifang Medical University, Weifang, China.\\n\\nBACKGROUND: Unlike bone tissue, little progress has been made regarding \\ncartilage regeneration, and many challenges remain. Furthermore, the key roles \\nof cartilage lesion caused by traumas, focal lesion, or articular overstress \\nremain unclear. Traumatic injuries to the meniscus as well as its degeneration \\nare important risk factors for long-term joint dysfunction, degenerative joint \\nlesions, and knee osteoarthritis (OA) a chronic joint disease characterized by \\ndegeneration of articular cartilage and hyperosteogeny. Nearly 50% of the \\nindividuals with meniscus injuries develop OA over time. Due to the limited \\ninherent self-repair capacity of cartilage lesion, the Biomaterial \\ndrug-nanomedicine is considered to be a promising alternative. Therefore, it is \\nimportant to elucidate the gene potential regeneration mechanisms and discover \\nnovel precise medication, which are identified through this study to investigate \\ntheir function and role in pathogenesis.\\nMETHODS: We downloaded the mRNA microarray statistics GSE117999, involving \\npaired cartilage lesion tissue samples from 12 OA patients and 12 patients from \\na control group. First, we analyzed these statistics to recognize the \\ndifferentially expressed genes (DEGs). We then exposed the gene ontology (GO) \\nannotation and the Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathway \\nenrichment analyses for these DEGs. Protein-protein interaction (PPI) networks \\nwere then constructed, from which we attained eight significant genes after a \\nfunctional interaction analysis. Finally, we identified a potential nanomedicine \\nattained from this assay set, using a wide range of inhibitor information \\narchived in the Search Tool for the Retrieval of Interacting Genes (STRING) \\ndatabase.\\nRESULTS: Sixty-six DEGs were identified with our standards for meaning (adjusted \\nP-value < 0.01, |log2 - FC| ≥1.2). Furthermore, we identified eight hub genes \\nand one potential nanomedicine - Selenocysteine based on these integrative data.\\nCONCLUSION: We identified eight hub genes that could work as prospective \\nbiomarkers for the diagnostic and biomaterial drug treatment of cartilage \\nlesion, involving the novel genes CAMP, DEFA3, TOLLIP, HLA-DQA2, SLC38A6, \\nSLC3A1, FAM20A, and ANO8. Meanwhile, these genes were mainly associated with \\nimmune response, immune mediator induction, and cell chemotaxis. Significant \\nsupport is provided for obtaining a series of novel gene targets, and we \\nidentify potential mechanisms for cartilage regeneration and final nanomedicine \\nimmunotherapy in regenerative medicine.\\n\\nCopyright © 2020 Ye, Xu, Fan, Zhang, Yuan, Chen, Sun, Yan, Song, Song, Yang and \\nYu.\\n\\nDOI: 10.3389/fphar.2020.01138\\nPMCID: PMC7394085\\nPMID: 32792959\\n\\n\\n9. Sci Rep. 2020 Jul 2;10(1):10868. doi: 10.1038/s41598-020-67730-y.\\n\\nSynovial cell cross-talk with cartilage plays a major role in the pathogenesis \\nof osteoarthritis.\\n\\nChou CH(#)(1), Jain V(#)(1), Gibson J(1), Attarian DE(2), Haraden CA(1), Yohn \\nCB(3), Laberge RM(3), Gregory S(1), Kraus VB(4)(5).\\n\\nAuthor information:\\n(1)Duke Molecular Physiology Institute, Duke University School of Medicine, \\nDurham, NC, 27701, USA.\\n(2)Department of Orthopaedic Surgery, Duke University School of Medicine, \\nDurham, NC, 27701, USA.\\n(3)Unity Biotechnology, South San Francisco, CA, USA.\\n(4)Duke Molecular Physiology Institute, Duke University School of Medicine, \\nDurham, NC, 27701, USA. vbk@duke.edu.\\n(5)Division of Rheumatology, Department of Medicine, Duke University School of \\nMedicine, Durham, NC, 27701, USA. vbk@duke.edu.\\n(#)Contributed equally\\n\\nWe elucidated the molecular cross-talk between cartilage and synovium in \\nosteoarthritis, the most widespread arthritis in the world, using the powerful \\ntool of single-cell RNA-sequencing. Multiple cell types were identified based on \\nprofiling of 10,640 synoviocytes and 26,192 chondrocytes: 12 distinct synovial \\ncell types and 7 distinct articular chondrocyte phenotypes from matched tissues. \\nIntact cartilage was enriched for homeostatic and hypertrophic chondrocytes, \\nwhile damaged cartilage was enriched for prefibro- and fibro-, regulatory, \\nreparative and prehypertrophic chondrocytes. A total of 61 cytokines and growth \\nfactors were predicted to regulate the 7 chondrocyte cell phenotypes. Based on \\nproduction by >\\u20091% of cells, 55% of the cytokines were produced by synovial \\ncells (39% exclusive to synoviocytes and not expressed by chondrocytes) and \\ntheir presence in osteoarthritic synovial fluid confirmed. The synoviocytes \\nproducing IL-1beta (a classic pathogenic cytokine in osteoarthritis), mainly \\ninflammatory macrophages and dendritic cells, were characterized by \\nco-expression of surface proteins corresponding to HLA-DQA1, HLA-DQA2, OLR1 or \\nTLR2. Strategies to deplete these pathogenic intra-articular cell subpopulations \\ncould be a therapeutic option for human osteoarthritis.\\n\\nDOI: 10.1038/s41598-020-67730-y\\nPMCID: PMC7331607\\nPMID: 32616761 [Indexed for MEDLINE]\\n\\nConflict of interest statement: C.H.C., V.J., J.G., D.E.A., C.A.H. and S.G. \\ndeclare no competing interests. C.B.Y. and R.M.L. are employees of Unity \\nBiotechnology and played a role in the conceptualization of the study and in \\npreparation of the manuscript. This work was funded in part by Unity \\nBiotechnology, however, Unity Biotechnology had no role in the preparation of \\nthe manuscript, data collection or decision to publish this study. V.B.K. has \\nbeen a periodic consultant for Unity Biotechnology providing periodic advice on \\nclinical study design, unrelated to this study and has no competing interests to \\ndeclare.\\n\\n\\n10. Medicina (Kaunas). 2020 Mar 31;56(4):154. doi: 10.3390/medicina56040154.\\n\\nAn Intergenic rs9275596 Polymorphism on Chr. 6p21 Is Associated with Multiple \\nSclerosis in Latvians.\\n\\nParamonova N(1), Trapina I(1), Dokane K(1), Kalnina J(1), Sjakste T(1), Sjakste \\nN(1)(2).\\n\\nAuthor information:\\n(1)Genomics and Bioinformatics, Institute of Biology of the University of \\nLatvia, LV-1004 Riga, Latvia.\\n(2)Department of Medical Biochemistry of the University of Latvia, LV-1004 Riga, \\nLatvia.\\n\\nBackground and objectives: Multiple sclerosis (MS) is a chronic inflammatory \\ndisease of the central nervous system, leading to demyelination of neurons and \\npotentially debilitating physical and mental symptoms. The disease is more \\nprevalent in women than in men. The major histocompatibility complex (MHC) \\nregion has been identified as a major genetic determinant for autoimmune \\ndiseases, and its role in some neurological disorders including MS was \\nevaluated. An intergenic single-nucleotide polymorphism (SNP), rs9275596, \\nlocated between the HLA-DQB1 and HLA-DQA2 genes, is in significant association \\nwith various autoimmune diseases according to genome-wide association studies \\n(GWASs). A cumulative effect of this SNP with other polymorphisms from this \\nregion was revealed. The aim of the study was to verify the data on rs9275596 \\nassociation in multiple sclerosis in a case/control study of the Latvian \\npopulation and to evaluate eventual functional significance of allele \\nsubstitutions. Materials and Methods: rs9275596 (chr6:32713854; GRCh38.p12) was \\ngenotyped in 273 MS patients and 208 controls on main and sex-specific \\nassociations. Eventual functional significance of allele substitutions was \\nevaluated in silico using publicly available tools. Results: The rs9275596 rare \\nalleles were identified as a disease susceptibility factor in association with \\nthe MS main group and in affected females (p < 0.001 and p < 0.01, \\nrespectively). Risk factor genotypes with rare alleles included were associated \\nwith the MS common cohort (p < 0.002) and female cohort (odds ratio, OR = 2.24) \\nand were identified as disease susceptible in males (OR = 2.41). It was shown \\nthat structural changes of rs9275596 affect the secondary structure of DNA. \\nFunctional significance of allele substitutions was evaluated on the eventual \\nsequence affinity to transcription factors (TFs) and splicing signals \\nsimilarity. A possible impact of the particular polymorphisms on the \\ntranscription and splicing efficiency is discussed. Conclusions: Our results \\nsuggest susceptibility of rs9275596 to multiple sclerosis in Latvians.\\n\\nDOI: 10.3390/medicina56040154\\nPMCID: PMC7230508\\nPMID: 32244438 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no conflict \\nof interest.'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'HLA-DQA2 role',save='y', retmax=10) # Apenas os primeiros 10 resultados/mais relevantes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em pubmed. Irão ser processados 10 resultados.\n",
      "\n",
      "1. Sci Rep. 2018 Jan 18;8(1):1069. doi: 10.1038/s41598-017-18241-w.\n",
      "\n",
      "Genome-wide association study of self-reported food reactions in Japanese \n",
      "identifies shrimp and peach specific loci in the HLA-DR/DQ gene region.\n",
      "\n",
      "Khor SS(1), Morino R(2), Nakazono K(3), Kamitsuji S(3), Akita M(2), Kawajiri \n",
      "M(2), Yamasaki T(4), Kami A(2), Hoshi Y(4), Tada A(2), Ishikawa K(2), Hine M(5), \n",
      "Kobayashi M(5), Kurume N(5), Kamatani N(3), Tokunaga K(1), Johnson TA(6).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Human Genetics, Graduate School of Medicine, The University of \n",
      "Tokyo, Tokyo, 113-0033, Japan.\n",
      "(2)EverGene Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\n",
      "(3)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan.\n",
      "(4)Life Science Group, Healthcare Division, Department of Healthcare Business, \n",
      "MTI Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\n",
      "(5)LunaLuna Division, Department of Healthcare Business, MTI Ltd., Shinjuku-ku, \n",
      "Tokyo, 163-1435, Japan.\n",
      "(6)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan. \n",
      "todd.johnson@stagen.co.jp.\n",
      "\n",
      "Food allergy is an increasingly important health problem in the world. Several \n",
      "genome-wide association studies (GWAS) focused on European ancestry samples have \n",
      "identified food allergy-specific loci in the HLA class II region. We conducted \n",
      "GWAS of self-reported reactivity with common foods using the data from 11011 \n",
      "Japanese women and identified shrimp and peach allergy-specific loci in the \n",
      "HLA-DR/DQ gene region tagged by rs74995702 (P = 6.30 × 10-17, OR = 1.91) and \n",
      "rs28359884 (P = 2.3 × 10-12, OR = 1.80), respectively. After HLA imputation \n",
      "using a Japanese population-specific reference, the most strongly associated \n",
      "haplotype was HLA-DRB1*04:05-HLA-DQB1*04:01 for shrimp allergy \n",
      "(P = 3.92 × 10-19, OR = 1.99) and HLA-DRB1*09:01-HLA-DQB1*03:03 for peach \n",
      "allergy (P = 1.15 × 10-7, OR = 1.68). Additionally, both allergies' associated \n",
      "variants were eQTLs for several HLA genes, with HLA-DQA2 the single eQTL gene \n",
      "shared between the two traits. Our study suggests that allergy to certain foods \n",
      "may be related to genetic differences that tag both HLA alleles having \n",
      "particular epitope binding specificities as well as variants modulating \n",
      "expression of particular HLA genes. Investigating this further could increase \n",
      "our understanding of food allergy aetiology and potentially lead to better \n",
      "therapeutic strategies for allergen immunotherapies.\n",
      "\n",
      "DOI: 10.1038/s41598-017-18241-w\n",
      "PMCID: PMC5773682\n",
      "PMID: 29348432 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: T.A.J., S.K., N.Ka., S.K. are employees of \n",
      "StaGen Co. Ltd. R.M., M.A., M.Ka., T.Y., A.K., M.T., Y.H., A.T., K.I., M.H., \n",
      "M.Ko., and N.Ku. are employees of EverGene/MTI. EverGene/MTI has applied for \n",
      "patents related to the food allergy associations described in this report. N.Ka. \n",
      "is also Chairman of StaGen Co. Ltd. and a Director of the Tsukuba International \n",
      "Clinical Pharmacology Clinic. S.S.K. and K.T. declare no competing financial \n",
      "interests.\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1. Sci Rep. 2018 Jan 18;8(1):1069. doi: 10.1038/s41598-017-18241-w.\\n\\nGenome-wide association study of self-reported food reactions in Japanese \\nidentifies shrimp and peach specific loci in the HLA-DR/DQ gene region.\\n\\nKhor SS(1), Morino R(2), Nakazono K(3), Kamitsuji S(3), Akita M(2), Kawajiri \\nM(2), Yamasaki T(4), Kami A(2), Hoshi Y(4), Tada A(2), Ishikawa K(2), Hine M(5), \\nKobayashi M(5), Kurume N(5), Kamatani N(3), Tokunaga K(1), Johnson TA(6).\\n\\nAuthor information:\\n(1)Department of Human Genetics, Graduate School of Medicine, The University of \\nTokyo, Tokyo, 113-0033, Japan.\\n(2)EverGene Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\\n(3)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan.\\n(4)Life Science Group, Healthcare Division, Department of Healthcare Business, \\nMTI Ltd., Shinjuku-ku, Tokyo, 163-1435, Japan.\\n(5)LunaLuna Division, Department of Healthcare Business, MTI Ltd., Shinjuku-ku, \\nTokyo, 163-1435, Japan.\\n(6)StaGen Co., Ltd., Taito-ku, Tokyo, 111-0051, Japan. \\ntodd.johnson@stagen.co.jp.\\n\\nFood allergy is an increasingly important health problem in the world. Several \\ngenome-wide association studies (GWAS) focused on European ancestry samples have \\nidentified food allergy-specific loci in the HLA class II region. We conducted \\nGWAS of self-reported reactivity with common foods using the data from 11011 \\nJapanese women and identified shrimp and peach allergy-specific loci in the \\nHLA-DR/DQ gene region tagged by rs74995702 (P\\u2009=\\u20096.30\\u2009×\\u200910-17, OR\\u2009=\\u20091.91) and \\nrs28359884 (P\\u2009=\\u20092.3\\u2009×\\u200910-12, OR\\u2009=\\u20091.80), respectively. After HLA imputation \\nusing a Japanese population-specific reference, the most strongly associated \\nhaplotype was HLA-DRB1*04:05-HLA-DQB1*04:01 for shrimp allergy \\n(P\\u2009=\\u20093.92\\u2009×\\u200910-19, OR\\u2009=\\u20091.99) and HLA-DRB1*09:01-HLA-DQB1*03:03 for peach \\nallergy (P\\u2009=\\u20091.15\\u2009×\\u200910-7, OR\\u2009=\\u20091.68). Additionally, both allergies' associated \\nvariants were eQTLs for several HLA genes, with HLA-DQA2 the single eQTL gene \\nshared between the two traits. Our study suggests that allergy to certain foods \\nmay be related to genetic differences that tag both HLA alleles having \\nparticular epitope binding specificities as well as variants modulating \\nexpression of particular HLA genes. Investigating this further could increase \\nour understanding of food allergy aetiology and potentially lead to better \\ntherapeutic strategies for allergen immunotherapies.\\n\\nDOI: 10.1038/s41598-017-18241-w\\nPMCID: PMC5773682\\nPMID: 29348432 [Indexed for MEDLINE]\\n\\nConflict of interest statement: T.A.J., S.K., N.Ka., S.K. are employees of \\nStaGen Co. Ltd. R.M., M.A., M.Ka., T.Y., A.K., M.T., Y.H., A.T., K.I., M.H., \\nM.Ko., and N.Ku. are employees of EverGene/MTI. EverGene/MTI has applied for \\npatents related to the food allergy associations described in this report. N.Ka. \\nis also Chairman of StaGen Co. Ltd. and a Director of the Tsukuba International \\nClinical Pharmacology Clinic. S.S.K. and K.T. declare no competing financial \\ninterests.\""
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'HLA-DQA2 food allergy', save='y', retmax=10) # Focado na alergia alimentar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 11 resultados em pubmed. Irão ser processados 11 resultados.\n",
      "\n",
      "1. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\n",
      " 2023 Oct 10.\n",
      "\n",
      "Genome-wide identification of m(6)A-associated single nucleotide polymorphisms \n",
      "in complex diseases of nervous system.\n",
      "\n",
      "Guo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \n",
      "Guo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\n",
      "\n",
      "Author information:\n",
      "(1)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China.\n",
      "(2)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China.\n",
      "(3)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "jiangyongshuai@hrbmu.edu.cn.\n",
      "(4)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "zhangmingming@hrbmu.edu.cn.\n",
      "\n",
      "N6-methyladenosine (m6A) is one of the most abundant chemical modifications on \n",
      "RNA and can affect the occurrence and development of diseases. Some studies have \n",
      "shown that the expressions of some m6A-related genes are significantly regulated \n",
      "by single nucleotide variants (SNV). However, the function of m6A-associated \n",
      "single nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \n",
      "(MS), Alzheimer's disease (AD) and Parkinson's disease (PD). Here, we identified \n",
      "the disease-associated m6A-SNPs by integrating genome-wide association study \n",
      "(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \n",
      "between these identified m6A-SNPs and their target genes in eQTL analysis and \n",
      "gene differential expression analysis. Finally, 26 genes corresponding to 20 \n",
      "m6A-SNPs with eQTL signals were identified and differentially expressed \n",
      "(P < 0.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P < 1e-04) were \n",
      "differentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \n",
      "expression of 31 genes were identified. There were 5 HLA genes with eQTL signals \n",
      "(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \n",
      "three diseases. In summary, our study provided new insights into understanding \n",
      "the potential roles of these m6A-SNPs in disease pathogenesis as well as \n",
      "therapeutic target.\n",
      "\n",
      "Copyright © 2023 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.neulet.2023.137513\n",
      "PMID: 37827449 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "2. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\n",
      "eCollection  2022.\n",
      "\n",
      "The Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren's \n",
      "Syndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\n",
      "\n",
      "Hong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \n",
      "Zhao T(2), Fu J(1).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \n",
      "University, Harbin, China.\n",
      "\n",
      "OBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \n",
      "of multiple sclerosis (MS) and Sjögren's syndrome (SS).\n",
      "METHODS: MS- and SS-related susceptibility genes and differentially expressed \n",
      "genes (DEGs) were identified by bioinformatics analysis based on genome-wide \n",
      "association studies (GWAS) and transcriptome data from GWAS catalog and Gene \n",
      "Expression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \n",
      "analysis, and protein-protein interaction analysis for susceptibility genes and \n",
      "DEGs were performed. The drugs targeting common pathways/genes were obtained \n",
      "through Comparative Toxicogenomics Database (CTD), DrugBank database, and \n",
      "Drug-Gene Interaction (DGI) Database. The target genes of \n",
      "approved/investigational drugs for MS and SS were obtained through DrugBank and \n",
      "compared with the common susceptibility genes.\n",
      "RESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \n",
      "(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \n",
      "HLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \n",
      "than two genes, and 28 common susceptibility pathways, with 15 drugs targeting \n",
      "three or more than three pathways. Based on transcriptome data, we found 3 hub \n",
      "common DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \n",
      "435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \n",
      "pathways and common risk pathways at the same time. There were 133 overlaps \n",
      "including JAK-STAT inhibitors between agents from GWAS and transcriptome data. \n",
      "Besides, we found that IL2RA and HLA-DRB1, identified as hub common \n",
      "susceptibility genes, were the targets of daclizumab and glatiramer that were \n",
      "used for MS, indicating that daclizumab and glatiramer may be therapeutic for \n",
      "SS.\n",
      "CONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \n",
      "signaling pathway played a vital role, which may be the genetic and molecular \n",
      "bases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \n",
      "therapies for MS and SS, especially for their comorbidity.\n",
      "\n",
      "Copyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\n",
      "\n",
      "DOI: 10.3389/fimmu.2022.857014\n",
      "PMCID: PMC8959321\n",
      "PMID: 35356004 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationships that could \n",
      "be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "3. J Cell Mol Med. 2021 Jul;25(14):7001-7012. doi: 10.1111/jcmm.16714. Epub 2021 \n",
      "Jun 16.\n",
      "\n",
      "Single-cell RNA sequencing identify SDCBP in ACE2-positive bronchial epithelial \n",
      "cells negatively correlates with COVID-19 severity.\n",
      "\n",
      "Ma D(1)(2), Liu S(1), Hu L(1)(3), He Q(1), Shi W(1)(2), Yan D(1)(3), Cao \n",
      "Y(1)(2)(3), Zhang G(1)(2)(3), Wang Z(1)(2)(3), Wu J(1), Jiang C(1)(2)(3).\n",
      "\n",
      "Author information:\n",
      "(1)Jiangsu Key Laboratory of Molecular Medicine, National Institute of \n",
      "Healthcare Data Science at Nanjing University, Medical School of Nanjing \n",
      "University, Nanjing, China.\n",
      "(2)Department of Hepatobiliary Surgery, The Affiliated Drum Tower Hospital of \n",
      "Nanjing University Medical School, Nanjing, China.\n",
      "(3)Department of Hepatobiliary Surgery, Drum Tower Clinical College of Nanjing \n",
      "Medical University, Nanjing, China.\n",
      "\n",
      "Erratum in\n",
      "    J Cell Mol Med. 2023 Oct;27(19):3012-3015.\n",
      "\n",
      "The coronavirus disease 2019 (COVID-19), caused by the novel severe acute \n",
      "respiratory syndrome coronavirus 2 (SARS-CoV-2), has resulted in many deaths \n",
      "throughout the world. It is vital to identify the novel prognostic biomarkers \n",
      "and therapeutic targets to assist with the subsequent diagnosis and treatment \n",
      "plan to mitigate the expansion of COVID-19. Since angiotensin-converting enzyme \n",
      "2 (ACE2)-positive cells are hosts for COVID-19, we focussed on this cell type to \n",
      "explore the underlying mechanisms of COVID-19. In this study, we identified that \n",
      "ACE2-positive cells from the bronchoalveolar lavage fluid (BALF) of patients \n",
      "with COVID-19 belong to bronchial epithelial cells. Comparing with patients of \n",
      "COVID-19 showing severe symptoms, the antigen processing and presentation \n",
      "pathway was increased and 12 typical genes, HLA-DRB5, HLA-DRB1, CD74, HLA-DRA, \n",
      "HLA-DPA1, HLA-DQA1, HSP90AA1, HSP90AB1, HLA-DPB1, HLA-DQB1, HLA-DQA2, and \n",
      "HLA-DMA, particularly HLA-DPB1, were obviously up-regulated in ACE2-positive \n",
      "bronchial epithelial cells of patients with mild disease. We further discovered \n",
      "SDCBP was positively correlated with above 12 genes particularly with HLA-DPB1 \n",
      "in ACE2-positive bronchial epithelial cells of COVID-19 patients. Moreover, \n",
      "SDCBP may increase the immune infiltration of B cells, CD8+ T cells, CD4+ T \n",
      "cells, macrophages, neutrophils and dendritic cells in different lung carcinoma. \n",
      "Moreover, we found the expression of SDCBP was positively correlated with the \n",
      "expression of antigen processing and presentation genes in post-mortem lung \n",
      "biopsies tissues, which is consistent with previous discoveries. These results \n",
      "suggest that SDCBP has good potential to be further developed as a novel \n",
      "diagnostic and therapeutic target in the treatment of COVID-19.\n",
      "\n",
      "© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \n",
      "Foundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/jcmm.16714\n",
      "PMCID: PMC8278084\n",
      "PMID: 34137173 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that there are no conflicts \n",
      "of interest.\n",
      "\n",
      "\n",
      "4. PLoS One. 2021 Apr 22;16(4):e0248429. doi: 10.1371/journal.pone.0248429. \n",
      "eCollection 2021.\n",
      "\n",
      "Hypomethylation mediates genetic association with the major histocompatibility \n",
      "complex genes in Sjögren's syndrome.\n",
      "\n",
      "Chi C(1)(2), Taylor KE(3), Quach H(2), Quach D(2), Criswell LA(3), Barcellos \n",
      "LF(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Computational Biology, College of Engineering, University of \n",
      "California, Berkeley, Berkeley, California, United States of America.\n",
      "(2)Genetic Epidemiology and Genomics Laboratory, School of Public Health, \n",
      "University of California, Berkeley, Berkeley, California, United States of \n",
      "America.\n",
      "(3)Department of Medicine, Russell/Engleman Rheumatology Research Center, \n",
      "University of California, San Francisco, San Francisco, California, United \n",
      "States of America.\n",
      "\n",
      "Erratum in\n",
      "    PLoS One. 2021 Jul 27;16(7):e0255549.\n",
      "\n",
      "Differential methylation of immune genes has been a consistent theme observed in \n",
      "Sjögren's syndrome (SS) in CD4+ T cells, CD19+ B cells, whole blood, and labial \n",
      "salivary glands (LSGs). Multiple studies have found associations supporting \n",
      "genetic control of DNA methylation in SS, which in the absence of reverse \n",
      "causation, has positive implications for the potential of epigenetic therapy. \n",
      "However, a formal study of the causal relationship between genetic variation, \n",
      "DNA methylation, and disease status is lacking. We performed a causal mediation \n",
      "analysis of DNA methylation as a mediator of nearby genetic association with SS \n",
      "using LSGs and genotype data collected from 131 female members of the Sjögren's \n",
      "International Collaborative Clinical Alliance registry, comprising of 64 SS \n",
      "cases and 67 non-cases. Bumphunter was used to first identify \n",
      "differentially-methylated regions (DMRs), then the causal inference test (CIT) \n",
      "was applied to identify DMRs mediating the association of nearby methylation \n",
      "quantitative trait loci (MeQTL) with SS. Bumphunter discovered 215 DMRs, with \n",
      "the majority located in the major histocompatibility complex (MHC) on chromosome \n",
      "6p21.3. Consistent with previous findings, regions hypomethylated in SS cases \n",
      "were enriched for gene sets associated with immune processes. Using the CIT, we \n",
      "observed a total of 19 DMR-MeQTL pairs that exhibited strong evidence for a \n",
      "causal mediation relationship. Close to half of these DMRs reside in the MHC and \n",
      "their corresponding meQTLs are in the region spanning the HLA-DQA1, HLA-DQB1, \n",
      "and HLA-DQA2 loci. The risk of SS conferred by these corresponding MeQTLs in the \n",
      "MHC was further substantiated by previous genome-wide association study results, \n",
      "with modest evidence for independent effects. By validating the presence of \n",
      "causal mediation, our findings suggest both genetic and epigenetic factors \n",
      "contribute to disease susceptibility, and inform the development of targeted \n",
      "epigenetic modification as a therapeutic approach for SS.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0248429\n",
      "PMCID: PMC8062105\n",
      "PMID: 33886574 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have declared that no competing \n",
      "interests exist.\n",
      "\n",
      "\n",
      "5. Medicine (Baltimore). 2020 Nov 25;99(48):e23477. doi: \n",
      "10.1097/MD.0000000000023477.\n",
      "\n",
      "CD74, a novel predictor for bronchopulmonary dysplasia in preterm infants.\n",
      "\n",
      "Gao J(1), Wu M(1), Wang F(1), Jiang L(1), Tian R(2), Zhu X(3), He S(3)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Pediatrics, Affiliated Hospital of Yangzhou University, \n",
      "Yangzhou, Jiangsu.\n",
      "(2)Department of Pediatrics, The First People's Hospital of Kunming City, \n",
      "Kunming, Yunnan.\n",
      "(3)Department of Neonatology, Children's Hospital of Soochow University, Suzhou, \n",
      "Jiangsu.\n",
      "(4)Department of Pediatrics, The First People's Hospital of Yunnan Province, \n",
      "Kunming, Yunnan, China.\n",
      "\n",
      "Bronchopulmonary dysplasia (BPD) remains a major complication and accounts for \n",
      "high morbidity and mortality of preterm infants. The present study aimed to \n",
      "identify the key genes in the development of BPD and to provide some new \n",
      "insights into the pathogenesis of BPD. The GSE108754 dataset was downloaded from \n",
      "Gene Expression Omnibus database containing 5 samples of BPD patients and 6 of \n",
      "non-BPD infants. The differentially expressed genes (DEGs) between BPD and \n",
      "non-BPD patients were identified by R software. The pathway and function \n",
      "enrichment analyses were performed through Database for Annotation Visualization \n",
      "and Integrated Discovery website. The protein-protein interaction network for \n",
      "DEGs was established by Cytoscape software and the most highly connected module \n",
      "was selected through MCODE plugin. Furthermore, the clinical sample verification \n",
      "among 25 BPD patients and 10 non-BPD infants was carried out in our center. \n",
      "Finally, based on the results above, the gene set enrichment analysis focusing \n",
      "on CD74 upregulated status was employed. Totally, 189 DEGs including 147 \n",
      "upregulated genes and 42 downregulated genes between BPD and non-BPD patients \n",
      "were screened out. The pathway and function enrichments revealed these DEGs were \n",
      "mainly enriched in asthma, intestinal immune network for IgA production, antigen \n",
      "processing and presentation and immune response. Thirteen DEGs (CD74, HLA-DMA, \n",
      "HLA-DRA, HLA-DMB, HLA-DOB, HLA-DQA1, HLA-DRB5, HLA-DPA1, HLA-DOA, HLA-DPB1, \n",
      "HLA-DQB2, HLA-DQA2, and HLA-DQB1) were determined as hub genes. The mRNA \n",
      "expression levels of the 13 hub genes were tested by quantitative real-time \n",
      "polymerase chain reaction among our clinical samples. Eventually, CD74 was \n",
      "confirmed to be the most significant highly expressed in BPD samples (P < .001) \n",
      "and its expression level was negatively correlated with gestational age \n",
      "(r = -0.653) and birth weight (r = -0.675). The gene set enrichment analysis \n",
      "results showed the gene sets associated with lupus erythematosus, viral \n",
      "myocarditis, immune network for IgA production, graft versus host disease, cell \n",
      "adhesion molecules and so no were differentially enriched with the phenotype of \n",
      "high-expression CD74. In conclusion, CD74 may serve to predict the BPD \n",
      "development and provide a new therapeutic target for BPD.\n",
      "\n",
      "DOI: 10.1097/MD.0000000000023477\n",
      "PMCID: PMC7710202\n",
      "PMID: 33235138 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors have no conflicts of interest to \n",
      "disclose.\n",
      "\n",
      "\n",
      "6. Kidney Int. 2021 Mar;99(3):671-685. doi: 10.1016/j.kint.2020.08.007. Epub 2020\n",
      " Sep 2.\n",
      "\n",
      "HLA-D and PLA2R1 risk alleles associate with recurrent primary membranous \n",
      "nephropathy in kidney transplant recipients.\n",
      "\n",
      "Berchtold L(1), Letouzé E(2), Alexander MP(3), Canaud G(4), Logt AV(5), Hamilton \n",
      "P(6), Mousson C(7), Vuiblet V(8), Moyer AM(9), Guibert S(10), Mrázová P(11), \n",
      "Levi C(12), Dubois V(13), Cruzado JM(14), Torres A(15), Gandhi MJ(16), Yousfi \n",
      "N(17), Tesar V(18), OndrejViklický(19), Hourmant M(20), Moulin B(21), Rieu \n",
      "P(22), Choukroun G(23), Legendre C(24), Wetzels J(5), Brenchley P(6), Ballarín \n",
      "Castan JA(25), Debiec H(17), Ronco P(26).\n",
      "\n",
      "Author information:\n",
      "(1)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France; Division of Nephrology, Department of \n",
      "Internal Medicine Specialties, University Hospital of Geneva, Geneva, \n",
      "Switzerland.\n",
      "(2)Centre de Recherche des Cordeliers, Sorbonne Université, INSERM, France; \n",
      "Functional Genomics of Solid Tumor, Labex Immuno-Oncology, Equipe Labellisée \n",
      "Ligue Contre le Cancer, Université Paris 13, Paris, France.\n",
      "(3)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, \n",
      "Minnesota, USA.\n",
      "(4)Inserm U1151, Necker-Enfants Malades Hospital, Paris, France; University \n",
      "Paris Descartes, Sorbonne Paris Cité, Paris, France; Department of Adult \n",
      "Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, France.\n",
      "(5)Department of Nephrology, Radboud University Medical Center, Radboud \n",
      "Institute for Health Sciences, Nijmegen, The Netherlands.\n",
      "(6)Division of Cardiovascular Sciences, School of Medical Sciences, Faculty of \n",
      "Biology, Medicine and Health, University of Manchester, Manchester, UK.\n",
      "(7)Department of Nephrology and Transplantation, University Hospital, Dijon, \n",
      "France.\n",
      "(8)BioSpec T Laboratory, EA 7506 URCA, Reims, France; Nephropathology Department \n",
      "of Biopathology Laboratory, Reims University Hospital, Reims, France; Division \n",
      "of Nephrology, Reims University Hospital, Reims, France.\n",
      "(9)Department of Laboratory Medicine and Pathology, Personalized Genomics \n",
      "Laboratory Mayo Clinic, Rochester, Minnesota, USA.\n",
      "(10)IntegraGen, Evry, France.\n",
      "(11)Transplant Laboratory, Institute for Clinical and Experimental Medicine, \n",
      "Prague, Czech Republic.\n",
      "(12)Department of Transplantation, Nephrology, and Clinical Immunology, Edouard \n",
      "Herriot Hospital, Hospices Civils de Lyon, Lyon, France.\n",
      "(13)Laboratoire HLA, Etablissement Français du Sang Auvergne Rhone-Alpes, Lyon, \n",
      "France.\n",
      "(14)Nephrology Department, Hospital Universitari Bellvitge, Bellvitge Research \n",
      "Institute (IDIBELL), Barcelona, Spain; RedInRen, RD16/0009/0031, University of \n",
      "Barcelona, L'Hospitalet de Llobregat, Barcelona, Spain.\n",
      "(15)Hospital Universitario de Canarias, Instituto de Tecnologías Biomédicas \n",
      "(ITB), Universidad de La Laguna, Tenerife, Spain.\n",
      "(16)Division of Transfusion Medicine, Mayo Clinic, Rochester, Minnesota, USA.\n",
      "(17)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France.\n",
      "(18)Department of Nephrology, First Faculty of Medicine, Charles University and \n",
      "General University Hospital, Prague, Czech Republic.\n",
      "(19)Department of Nephrology, Transplant Center, Institute for Clinical and \n",
      "Experimental Medicine, Prague, Czech Republic.\n",
      "(20)Department of Nephrology, University Hospital, Nantes, France.\n",
      "(21)Institut National de la Santé et de la Recherche Médicale, Unité Mixte de \n",
      "Recherche S 1109, Strasbourg University, Strasbourg, France; Fédération de \n",
      "Médecine Translationnelle de Strasbourg (FMTS), Strasbourg, France; Department \n",
      "of Nephrology, Strasbourg University Hospital, Strasbourg, France.\n",
      "(22)Division of Nephrology, Reims University Hospital, Reims, France; Université \n",
      "de Reims Champagne-Ardenne, CNRS UMR 7369 (Matrice Extracellulaire et Dynamique \n",
      "Cellulaire, MEDyC), Reims, France.\n",
      "(23)Department of Nephrology, University Hospital, Amiens, France.\n",
      "(24)University Paris Descartes, Sorbonne Paris Cité, Paris, France; Department \n",
      "of Adult Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, \n",
      "France.\n",
      "(25)Department of Nephrology, Fundación Puigvert, Barcelona, Spain.\n",
      "(26)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \n",
      "France; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \n",
      "de Recherche S1155, Paris, France; Department of Nephrology (Day Hospital), \n",
      "AP-HP, Tenon Hospital, Paris, France. Electronic address: pierre.ronco@upmc.fr.\n",
      "\n",
      "Erratum in\n",
      "    Kidney Int. 2021 Jul;100(1):243.\n",
      "\n",
      "Comment in\n",
      "    Kidney Int. 2021 Mar;99(3):545-548.\n",
      "\n",
      "Recurrence of primary membranous nephropathy after transplantation occurs in up \n",
      "to 44% of patients and is driven by PLA2R antibody. Here, we asked whether \n",
      "genetic determinants could improve risk prediction. First, we sequenced PLA2R1 \n",
      "and HLA-D loci in 248 patients with primary membranous nephropathy and \n",
      "identified two independent single nucleotide polymorphisms (SNPs) at risk for \n",
      "primary membranous nephropathy at each locus. These were rs9271188 (intergenic \n",
      "between HLA-DRB1 and HLA-DQA1,) and rs9275086 (intergenic between HLA-DQB1 and \n",
      "HLA-DQA2) at the HLA-D locus along with rs6726925 and rs13018963 at the PLA2R1 \n",
      "locus. Then we investigated whether primary membranous nephropathy at-risk \n",
      "variants were associated with recurrence in a retrospective cohort of 105 \n",
      "donor-recipient pairs and a replication cohort of 40 pairs. Seven SNPs located \n",
      "between HLA-DRB1 and HLA-DQA1 in linkage disequilibrium with rs9271188, and \n",
      "three SNPs in the PLA2R1 region predicted recurrence when presented by the \n",
      "donor, but not when presented by the recipient. The two SNPs in the HLA-D region \n",
      "most strongly associated with recurrence (rs9271705 and rs9271550) were \n",
      "confirmed in the replication cohort. A genetic risk score based on the two best \n",
      "predictors at each locus (rs9271705, rs9271550, rs17830558, and rs3828323) \n",
      "identified a group of patients with high risk of recurrence. Thus, our results \n",
      "suggest that the graft contributes to recurrence of primary membranous \n",
      "nephropathy through the disease susceptibility HLA-D and PLA2R1 SNPs in an \n",
      "autoimmune milieu. Further studies are needed before implementation of genetic \n",
      "testing for these in donor selection.\n",
      "\n",
      "Copyright © 2020. Published by Elsevier Inc.\n",
      "\n",
      "DOI: 10.1016/j.kint.2020.08.007\n",
      "PMID: 32889013 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. Lupus. 2013 Feb;22(2):198-204. doi: 10.1177/0961203312470183. Epub 2012 Dec\n",
      "20.\n",
      "\n",
      "HLA variants rs9271366 and rs9275328 are associated with systemic lupus \n",
      "erythematosus susceptibility in Malays and Chinese.\n",
      "\n",
      "Chai HC(1), Phipps ME, Othman I, Tan LP, Chua KH.\n",
      "\n",
      "Author information:\n",
      "(1)Jeffrey Cheah School of Medicine and Health Sciences, Monash University \n",
      "(Sunway Campus), Malaysia.\n",
      "\n",
      "BACKGROUND: Human leukocyte antigen (HLA) antigens and genes have long been \n",
      "reported associated with systemic lupus erythematosus (SLE) susceptibility in \n",
      "many populations. With the advance in technologies such as genome-wide \n",
      "association studies, many newly discovered SLE-associated single-nucleotide \n",
      "polymorphisms (SNPs) have been reported in recent years. These include \n",
      "HLA-DRB1/HLA-DQA1 rs9271366 and HLA-DQB1/HLA-DQA2 rs9275328. Our aim was to \n",
      "investigate these SNPs in a Malaysian SLE cohort.\n",
      "MATERIALS AND METHODS: SNPs rs9271366 and rs9275328 were screened across 790 \n",
      "Malaysian citizens from three ethnic groups (360 patients and 430 healthy \n",
      "volunteers) by Taqman SNP genotyping assays. Allele and genotyping frequencies, \n",
      "Hardy-Weinberg equilibrium, Fisher's exact test and odds ratio were calculated \n",
      "for each SNP and ethnic group. Linkage disequilibrium and interaction between \n",
      "the two SNPs were also evaluated.\n",
      "RESULTS: The minor allele G and its homozygous genotype GG of HLA-DRB1/HLA-DQA1 \n",
      "rs9271366 significantly increased the SLE susceptibility in Malaysian patients, \n",
      "including those of Malay and Chinese ethnicity (odds ratio (OR) > 1, p < 0.05). \n",
      "As for HLA-DQB1/HLA-DQA2 rs9275328, the minor allele T and the heterozygous \n",
      "genotype CT conferred protective effect to SLE in Malaysians, as well as in \n",
      "Malays and Chinese, by having OR < 1 and p value <0.05. Both SNPs did not show \n",
      "associations to SLE in Indians. D' and r (2) values for the two SNPs in LD \n",
      "analysis were 0.941 and 0.065, respectively, with haplotype GC and AT being \n",
      "significantly associated with SLE (p < 5.0 × 10(-4)) after 10,000 permutations \n",
      "were performed. The MDR test clustered the genotype combinations of GG and CC, \n",
      "and AG and CC of rs9271366 and rs9275328, accordingly, as high-risk group, and \n",
      "the two SNPs interacted redundantly by removing 1.96% of the entropy.\n",
      "CONCLUSIONS: Our findings suggest that in addition to some classical HLA \n",
      "variants, rs9271366 and rs9275328 are additional polymorphisms worth considering \n",
      "in the Malaysian and possibly in a larger Asian SLE scenario.\n",
      "\n",
      "DOI: 10.1177/0961203312470183\n",
      "PMID: 23257407 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "8. Clin Exp Allergy. 2012 Dec;42(12):1724-33. doi: 10.1111/cea.12000.\n",
      "\n",
      "HLA-DQ strikes again: genome-wide association study further confirms HLA-DQ in \n",
      "the diagnosis of asthma among adults.\n",
      "\n",
      "Lasky-Su J(1), Himes BE, Raby BA, Klanderman BJ, Sylvia JS, Lange C, Melen E, \n",
      "Martinez FD, Israel E, Gauderman J, Gilliland F, Sleiman P, Hakonarson H, \n",
      "Celedón JC, Soto-Quiros M, Avila L, Lima JJ, Irvin CG, Peters SP, Boushey H, \n",
      "Chinchilli VM, Mauger D, Tantisira K, Weiss ST; SHARP investigators.\n",
      "\n",
      "Author information:\n",
      "(1)Channing Laboratory, Department of Medicine, Brigham and Women's Hospital, \n",
      "and Harvard Medical School, Boston, MA, USA.\n",
      "\n",
      "BACKGROUND: Asthma is a common chronic respiratory disease in children and \n",
      "adults. An important genetic component to asthma susceptibility has long been \n",
      "recognized, most recently through the identification of several genes (e.g., \n",
      "ORMDL3, PDE4D, HLA-DQ, and TLE4) via genome-wide association studies.\n",
      "OBJECTIVE: To identify genetic variants associated with asthma affection status \n",
      "using genome-wide association data.\n",
      "METHODS: We describe results from a genome-wide association study on asthma \n",
      "performed in 3855 subjects using a panel of 455 089 single nucleotide \n",
      "polymorphisms (SNPs).\n",
      "RESULT: The genome-wide association study resulted in the prioritization of 33 \n",
      "variants for immediate follow-up in a multi-staged replication effort. Of these, \n",
      "a common polymorphism (rs9272346) localizing to within 1 Kb of HLA-DQA1 \n",
      "(chromosome 6p21.3) was associated with asthma in adults (P-value = 2.2E-08) \n",
      "with consistent evidence in the more heterogeneous group of adults and children \n",
      "(P-value = 1.0E-04). Moreover, some genes identified in prior asthma GWAS were \n",
      "nominally associated with asthma in our populations.\n",
      "CONCLUSION: Overall, our findings further replicate the HLA-DQ region in the \n",
      "pathogenesis of asthma. HLA-DQA1 is the fourth member of the HLA family found to \n",
      "be associated with asthma, in addition to the previously identified HLA-DRA, \n",
      "HLA-DQB1 and HLA-DQA2.\n",
      "\n",
      "© 2012 Blackwell Publishing Ltd.\n",
      "\n",
      "DOI: 10.1111/cea.12000\n",
      "PMCID: PMC6343489\n",
      "PMID: 23181788 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Conflict of Interests: Charles Irvin’s work was \n",
      "funded by a grant from the American Lung Association. Stephen Peters has been a \n",
      "member of the NHLBI’s ACRN since its creation in 1993. Homer Boushey received \n",
      "funding from the NIH for his participation in the research of the Inner City \n",
      "Asthma Consortium, from Genentech for research on the airway microbiome of \n",
      "severe asthma, and from GlaxoSmithKline for research on bronchial epithelial \n",
      "cell susceptibility to viral infection. He has served as an ad-hoc consultant or \n",
      "advisory board member for Genentech, GlaxoSmithKline, Kalobios, Pharmaxis, Merck \n",
      "and Johnson & Johnson. Vernon Chinchilli, Blanca Himes, Scott Weiss, Juan \n",
      "Celedon, Barbara Klanderman, Erik Melen, Hakon Hakonarson, Lydiana Avila, \n",
      "Benjamin Raby, Kelan Tantisira, David Mauger, Patrick Sleiman, Fernando \n",
      "Martinez, W. James Gauderman, Frank Gilliland, John Lima, Elliot Israel, \n",
      "Christoph Lange, Jody Senter Sylvia, Manuel Soto Quiros and Jessica Lasky-Su \n",
      "have no conflict of interests to declare\n",
      "\n",
      "\n",
      "9. J Immunol. 2012 Apr 15;188(8):3903-11. doi: 10.4049/jimmunol.1103048. Epub\n",
      "2012  Mar 9.\n",
      "\n",
      "HLA-DQA2 and HLA-DQB2 genes are specifically expressed in human Langerhans cells \n",
      "and encode a new HLA class II molecule.\n",
      "\n",
      "Lenormand C(1), Bausinger H, Gross F, Signorino-Gelo F, Koch S, Peressin M, \n",
      "Fricker D, Cazenave JP, Bieber T, Hanau D, de la Salle H, Tourne S.\n",
      "\n",
      "Author information:\n",
      "(1)INSERM, Unité Mixte de Recherche S725, Biologie des Cellules Dendritiques \n",
      "Humaines, Strasbourg, 67065 France.\n",
      "\n",
      "The precise role of human epidermal Langerhans cells (LCs) in immune response is \n",
      "highly controversial. While studying the gene expression profile of these cells, \n",
      "we were intrigued to identify the HLA-DQB2 gene as potentially expressed in LCs. \n",
      "Despite a strong evolutionary conservation of their sequences, the concomitant \n",
      "expression of the poorly polymorphic HLA-DQA2/HLA-DQB2 genes, paralogous to the \n",
      "HLA-DQA1/HLA-DQB1 genes, has never been detected in any cell type. We confirmed \n",
      "by RT-PCR that the HLA-DQA2 and -DQB2 genes are both expressed in LCs, but not \n",
      "in monocyte-derived dendritic cells, or in blood CD1c(+) or plasmacytoid \n",
      "dendritic cells. The presence of the HLA-DQβ2 chain in LCs could be demonstrated \n",
      "by Western blotting, whereas immunofluorescence revealed its localization in \n",
      "early endosomes. As in the case of other HLA class II molecules, the HLA-DQα2 \n",
      "and -DQβ2 chains formed heterodimers that had to associate with the invariant \n",
      "chain to reach endosomal compartments. HLA-DQα2/β2 heterodimers were expressed \n",
      "at the cell surface, where they could mediate staphylococcal superantigen \n",
      "stimulation of T cells. Interestingly, HLA-DQα2 and HLA-DQβ1 chains formed mixed \n",
      "heterodimers which efficiently left the endoplasmic reticulum. These \n",
      "observations strongly suggest that the poorly polymorphic HLA-DQA2 and -DQB2 \n",
      "genes should be considered to be of immunological importance. The HLA-DQα2/β2 \n",
      "molecules could influence the complexity of the repertoire of Ags presented by \n",
      "LCs.\n",
      "\n",
      "DOI: 10.4049/jimmunol.1103048\n",
      "PMID: 22407913 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "10. PLoS One. 2010 Apr 13;5(4):e10142. doi: 10.1371/journal.pone.0010142.\n",
      "\n",
      "The effect of single nucleotide polymorphisms from genome wide association \n",
      "studies in multiple sclerosis on gene expression.\n",
      "\n",
      "Handel AE(1), Handunnetthi L, Berlanga AJ, Watson CT, Morahan JM, Ramagopalan \n",
      "SV.\n",
      "\n",
      "Author information:\n",
      "(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, \n",
      "United Kingdom.\n",
      "\n",
      "BACKGROUND: Multiple sclerosis (MS) is a complex neurological disorder. Its \n",
      "aetiology involves both environmental and genetic factors. Recent genome-wide \n",
      "association studies have identified a number of single nucleotide polymorphisms \n",
      "(SNPs) associated with susceptibility to (MS). We investigated whether these \n",
      "genetic variations were associated with alteration in gene expression.\n",
      "METHODS/PRINCIPAL FINDINGS: We used a database of mRNA expression and genetic \n",
      "variation derived from immortalised peripheral lymphocytes to investigate \n",
      "polymorphisms associated with MS for correlation with gene expression. Several \n",
      "SNPs were found to be associated with changes in expression: in particular two \n",
      "with HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB1, HLA-DRB4 and HLA-DRB5, one with \n",
      "ZFP57, one with CD58, two with IL7 and FAM164A, and one with FAM119B, TSFM and \n",
      "KUB3. We found minimal cross-over with a recent whole genome expression study in \n",
      "MS patients.\n",
      "DISCUSSION: We have shown that many susceptibility loci in MS are associated \n",
      "with changes in gene expression using an unbiased expression database. Several \n",
      "of these findings suggest novel gene candidates underlying the effects of \n",
      "MS-associated genetic variation.\n",
      "\n",
      "DOI: 10.1371/journal.pone.0010142\n",
      "PMCID: PMC2854120\n",
      "PMID: 20405052 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Competing Interests: The authors have declared \n",
      "that no competing interests exist.\n",
      "\n",
      "\n",
      "11. Hum Immunol. 2001 May;62(5):504-8. doi: 10.1016/s0198-8859(01)00236-1.\n",
      "\n",
      "Absence of in vivo DNA-protein interactions in the DQA2 and DQB2 promoter \n",
      "regions.\n",
      "\n",
      "Indovina P(1), Megiorni F, Fontemaggi G, Coni P, Mora B, Mazzilli MC.\n",
      "\n",
      "Author information:\n",
      "(1)Dipartimento di Medicina Sperimentale e Patologia, Università \"La Sapienza,\", \n",
      "Rome, Italy.\n",
      "\n",
      "The DQ subregion of the human major histocompatibility complex (HLA) contains \n",
      "two pairs of loci: the DQA1/B1 genes (hereafter called DQ1), coding for the DQ \n",
      "molecules, and the DQA2/B2 pseudogenes (hereafter called DQ2). These pseudogenes \n",
      "are highly homologous to the functional DQ1 genes and they have no apparent \n",
      "abnormal features in their sequences that could prevent their activity. Only \n",
      "recently a low expression of the DQA2 gene has been observed whereas the DQB2 \n",
      "transcript was never found. The comparison between the DQ1 and DQ2 regulatory \n",
      "sequences revealed several differences in their W, X, and Y cis-acting elements. \n",
      "To examine the DNA/protein interactions in the DQ promoter regions, we performed \n",
      "in vivo footprinting experiments. Whereas the functional DQ1 loci showed a \n",
      "series of DNA-protein contact points in the X and Y boxes, the promoters of the \n",
      "DQ2 pseudogenes displayed an unoccupied phenotype. These findings suggest that \n",
      "the very low level of DQA2 expression and the apparent lack of DQB2 activity are \n",
      "caused by the reduced binding affinity of specific transcription factors.\n",
      "\n",
      "DOI: 10.1016/s0198-8859(01)00236-1\n",
      "PMID: 11334674 [Indexed for MEDLINE]\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\\n 2023 Oct 10.\\n\\nGenome-wide identification of m(6)A-associated single nucleotide polymorphisms \\nin complex diseases of nervous system.\\n\\nGuo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \\nGuo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\\n\\nAuthor information:\\n(1)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China.\\n(2)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China.\\n(3)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\njiangyongshuai@hrbmu.edu.cn.\\n(4)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\nzhangmingming@hrbmu.edu.cn.\\n\\nN6-methyladenosine (m6A) is one of the most abundant chemical modifications on \\nRNA and can affect the occurrence and development of diseases. Some studies have \\nshown that the expressions of some m6A-related genes are significantly regulated \\nby single nucleotide variants (SNV). However, the function of m6A-associated \\nsingle nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \\n(MS), Alzheimer\\'s disease (AD) and Parkinson\\'s disease (PD). Here, we identified \\nthe disease-associated m6A-SNPs by integrating genome-wide association study \\n(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \\nbetween these identified m6A-SNPs and their target genes in eQTL analysis and \\ngene differential expression analysis. Finally, 26 genes corresponding to 20 \\nm6A-SNPs with eQTL signals were identified and differentially expressed \\n(P\\xa0<\\xa00.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P\\xa0<\\xa01e-04) were \\ndifferentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \\nexpression of 31 genes were identified. There were 5 HLA genes with eQTL signals \\n(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \\nthree diseases. In summary, our study provided new insights into understanding \\nthe potential roles of these m6A-SNPs in disease pathogenesis as well as \\ntherapeutic target.\\n\\nCopyright © 2023 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.neulet.2023.137513\\nPMID: 37827449 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n2. Front Immunol. 2022 Mar 9;13:857014. doi: 10.3389/fimmu.2022.857014.\\neCollection  2022.\\n\\nThe Shared Mechanism and Candidate Drugs of Multiple Sclerosis and Sjögren\\'s \\nSyndrome Analyzed by Bioinformatics Based on GWAS and Transcriptome Data.\\n\\nHong X(1), Wang X(1), Rang X(1), Yin X(1), Zhang X(1), Wang R(1), Wang D(1), \\nZhao T(2), Fu J(1).\\n\\nAuthor information:\\n(1)Department of Neurology, The Second Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n(2)Department of Neurology, The First Affiliated Hospital of Harbin Medical \\nUniversity, Harbin, China.\\n\\nOBJECTIVE: This study aimed to explore the shared mechanism and candidate drugs \\nof multiple sclerosis (MS) and Sjögren\\'s syndrome (SS).\\nMETHODS: MS- and SS-related susceptibility genes and differentially expressed \\ngenes (DEGs) were identified by bioinformatics analysis based on genome-wide \\nassociation studies (GWAS) and transcriptome data from GWAS catalog and Gene \\nExpression Omnibus (GEO) database. Pathway enrichment, Gene Ontology (GO) \\nanalysis, and protein-protein interaction analysis for susceptibility genes and \\nDEGs were performed. The drugs targeting common pathways/genes were obtained \\nthrough Comparative Toxicogenomics Database (CTD), DrugBank database, and \\nDrug-Gene Interaction (DGI) Database. The target genes of \\napproved/investigational drugs for MS and SS were obtained through DrugBank and \\ncompared with the common susceptibility genes.\\nRESULTS: Based on GWAS data, we found 14 hub common susceptibility genes \\n(HLA-DRB1, HLA-DRA, STAT3, JAK1, HLA-B, HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB5, \\nHLA-DPA1, HLA-DPB1, TYK2, IL2RA, and MAPK1), with 8 drugs targeting two or more \\nthan two genes, and 28 common susceptibility pathways, with 15 drugs targeting \\nthree or more than three pathways. Based on transcriptome data, we found 3 hub \\ncommon DEGs (STAT1, GATA3, PIK3CA) with 3 drugs and 10 common risk pathways with \\n435 drugs. \"JAK-STAT signaling pathway\" was included in common susceptibility \\npathways and common risk pathways at the same time. There were 133 overlaps \\nincluding JAK-STAT inhibitors between agents from GWAS and transcriptome data. \\nBesides, we found that IL2RA and HLA-DRB1, identified as hub common \\nsusceptibility genes, were the targets of daclizumab and glatiramer that were \\nused for MS, indicating that daclizumab and glatiramer may be therapeutic for \\nSS.\\nCONCLUSION: We observed the shared mechanism of MS and SS, in which JAK-STAT \\nsignaling pathway played a vital role, which may be the genetic and molecular \\nbases of comorbidity of MS with SS. Moreover, JAK-STAT inhibitors were potential \\ntherapies for MS and SS, especially for their comorbidity.\\n\\nCopyright © 2022 Hong, Wang, Rang, Yin, Zhang, Wang, Wang, Zhao and Fu.\\n\\nDOI: 10.3389/fimmu.2022.857014\\nPMCID: PMC8959321\\nPMID: 35356004 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationships that could \\nbe construed as a potential conflict of interest.\\n\\n\\n3. J Cell Mol Med. 2021 Jul;25(14):7001-7012. doi: 10.1111/jcmm.16714. Epub 2021 \\nJun 16.\\n\\nSingle-cell RNA sequencing identify SDCBP in ACE2-positive bronchial epithelial \\ncells negatively correlates with COVID-19 severity.\\n\\nMa D(1)(2), Liu S(1), Hu L(1)(3), He Q(1), Shi W(1)(2), Yan D(1)(3), Cao \\nY(1)(2)(3), Zhang G(1)(2)(3), Wang Z(1)(2)(3), Wu J(1), Jiang C(1)(2)(3).\\n\\nAuthor information:\\n(1)Jiangsu Key Laboratory of Molecular Medicine, National Institute of \\nHealthcare Data Science at Nanjing University, Medical School of Nanjing \\nUniversity, Nanjing, China.\\n(2)Department of Hepatobiliary Surgery, The Affiliated Drum Tower Hospital of \\nNanjing University Medical School, Nanjing, China.\\n(3)Department of Hepatobiliary Surgery, Drum Tower Clinical College of Nanjing \\nMedical University, Nanjing, China.\\n\\nErratum in\\n    J Cell Mol Med. 2023 Oct;27(19):3012-3015.\\n\\nThe coronavirus disease 2019 (COVID-19), caused by the novel severe acute \\nrespiratory syndrome coronavirus 2 (SARS-CoV-2), has resulted in many deaths \\nthroughout the world. It is vital to identify the novel prognostic biomarkers \\nand therapeutic targets to assist with the subsequent diagnosis and treatment \\nplan to mitigate the expansion of COVID-19. Since angiotensin-converting enzyme \\n2 (ACE2)-positive cells are hosts for COVID-19, we focussed on this cell type to \\nexplore the underlying mechanisms of COVID-19. In this study, we identified that \\nACE2-positive cells from the bronchoalveolar lavage fluid (BALF) of patients \\nwith COVID-19 belong to bronchial epithelial cells. Comparing with patients of \\nCOVID-19 showing severe symptoms, the antigen processing and presentation \\npathway was increased and 12 typical genes, HLA-DRB5, HLA-DRB1, CD74, HLA-DRA, \\nHLA-DPA1, HLA-DQA1, HSP90AA1, HSP90AB1, HLA-DPB1, HLA-DQB1, HLA-DQA2, and \\nHLA-DMA, particularly HLA-DPB1, were obviously up-regulated in ACE2-positive \\nbronchial epithelial cells of patients with mild disease. We further discovered \\nSDCBP was positively correlated with above 12 genes particularly with HLA-DPB1 \\nin ACE2-positive bronchial epithelial cells of COVID-19 patients. Moreover, \\nSDCBP may increase the immune infiltration of B cells, CD8+ T cells, CD4+ T \\ncells, macrophages, neutrophils and dendritic cells in different lung carcinoma. \\nMoreover, we found the expression of SDCBP was positively correlated with the \\nexpression of antigen processing and presentation genes in post-mortem lung \\nbiopsies tissues, which is consistent with previous discoveries. These results \\nsuggest that SDCBP has good potential to be further developed as a novel \\ndiagnostic and therapeutic target in the treatment of COVID-19.\\n\\n© 2021 The Authors. Journal of Cellular and Molecular Medicine published by \\nFoundation for Cellular and Molecular Medicine and John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/jcmm.16714\\nPMCID: PMC8278084\\nPMID: 34137173 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that there are no conflicts \\nof interest.\\n\\n\\n4. PLoS One. 2021 Apr 22;16(4):e0248429. doi: 10.1371/journal.pone.0248429. \\neCollection 2021.\\n\\nHypomethylation mediates genetic association with the major histocompatibility \\ncomplex genes in Sjögren\\'s syndrome.\\n\\nChi C(1)(2), Taylor KE(3), Quach H(2), Quach D(2), Criswell LA(3), Barcellos \\nLF(1)(2).\\n\\nAuthor information:\\n(1)Center for Computational Biology, College of Engineering, University of \\nCalifornia, Berkeley, Berkeley, California, United States of America.\\n(2)Genetic Epidemiology and Genomics Laboratory, School of Public Health, \\nUniversity of California, Berkeley, Berkeley, California, United States of \\nAmerica.\\n(3)Department of Medicine, Russell/Engleman Rheumatology Research Center, \\nUniversity of California, San Francisco, San Francisco, California, United \\nStates of America.\\n\\nErratum in\\n    PLoS One. 2021 Jul 27;16(7):e0255549.\\n\\nDifferential methylation of immune genes has been a consistent theme observed in \\nSjögren\\'s syndrome (SS) in CD4+ T cells, CD19+ B cells, whole blood, and labial \\nsalivary glands (LSGs). Multiple studies have found associations supporting \\ngenetic control of DNA methylation in SS, which in the absence of reverse \\ncausation, has positive implications for the potential of epigenetic therapy. \\nHowever, a formal study of the causal relationship between genetic variation, \\nDNA methylation, and disease status is lacking. We performed a causal mediation \\nanalysis of DNA methylation as a mediator of nearby genetic association with SS \\nusing LSGs and genotype data collected from 131 female members of the Sjögren\\'s \\nInternational Collaborative Clinical Alliance registry, comprising of 64 SS \\ncases and 67 non-cases. Bumphunter was used to first identify \\ndifferentially-methylated regions (DMRs), then the causal inference test (CIT) \\nwas applied to identify DMRs mediating the association of nearby methylation \\nquantitative trait loci (MeQTL) with SS. Bumphunter discovered 215 DMRs, with \\nthe majority located in the major histocompatibility complex (MHC) on chromosome \\n6p21.3. Consistent with previous findings, regions hypomethylated in SS cases \\nwere enriched for gene sets associated with immune processes. Using the CIT, we \\nobserved a total of 19 DMR-MeQTL pairs that exhibited strong evidence for a \\ncausal mediation relationship. Close to half of these DMRs reside in the MHC and \\ntheir corresponding meQTLs are in the region spanning the HLA-DQA1, HLA-DQB1, \\nand HLA-DQA2 loci. The risk of SS conferred by these corresponding MeQTLs in the \\nMHC was further substantiated by previous genome-wide association study results, \\nwith modest evidence for independent effects. By validating the presence of \\ncausal mediation, our findings suggest both genetic and epigenetic factors \\ncontribute to disease susceptibility, and inform the development of targeted \\nepigenetic modification as a therapeutic approach for SS.\\n\\nDOI: 10.1371/journal.pone.0248429\\nPMCID: PMC8062105\\nPMID: 33886574 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have declared that no competing \\ninterests exist.\\n\\n\\n5. Medicine (Baltimore). 2020 Nov 25;99(48):e23477. doi: \\n10.1097/MD.0000000000023477.\\n\\nCD74, a novel predictor for bronchopulmonary dysplasia in preterm infants.\\n\\nGao J(1), Wu M(1), Wang F(1), Jiang L(1), Tian R(2), Zhu X(3), He S(3)(4).\\n\\nAuthor information:\\n(1)Department of Pediatrics, Affiliated Hospital of Yangzhou University, \\nYangzhou, Jiangsu.\\n(2)Department of Pediatrics, The First People\\'s Hospital of Kunming City, \\nKunming, Yunnan.\\n(3)Department of Neonatology, Children\\'s Hospital of Soochow University, Suzhou, \\nJiangsu.\\n(4)Department of Pediatrics, The First People\\'s Hospital of Yunnan Province, \\nKunming, Yunnan, China.\\n\\nBronchopulmonary dysplasia (BPD) remains a major complication and accounts for \\nhigh morbidity and mortality of preterm infants. The present study aimed to \\nidentify the key genes in the development of BPD and to provide some new \\ninsights into the pathogenesis of BPD. The GSE108754 dataset was downloaded from \\nGene Expression Omnibus database containing 5 samples of BPD patients and 6 of \\nnon-BPD infants. The differentially expressed genes (DEGs) between BPD and \\nnon-BPD patients were identified by R software. The pathway and function \\nenrichment analyses were performed through Database for Annotation Visualization \\nand Integrated Discovery website. The protein-protein interaction network for \\nDEGs was established by Cytoscape software and the most highly connected module \\nwas selected through MCODE plugin. Furthermore, the clinical sample verification \\namong 25 BPD patients and 10 non-BPD infants was carried out in our center. \\nFinally, based on the results above, the gene set enrichment analysis focusing \\non CD74 upregulated status was employed. Totally, 189 DEGs including 147 \\nupregulated genes and 42 downregulated genes between BPD and non-BPD patients \\nwere screened out. The pathway and function enrichments revealed these DEGs were \\nmainly enriched in asthma, intestinal immune network for IgA production, antigen \\nprocessing and presentation and immune response. Thirteen DEGs (CD74, HLA-DMA, \\nHLA-DRA, HLA-DMB, HLA-DOB, HLA-DQA1, HLA-DRB5, HLA-DPA1, HLA-DOA, HLA-DPB1, \\nHLA-DQB2, HLA-DQA2, and HLA-DQB1) were determined as hub genes. The mRNA \\nexpression levels of the 13 hub genes were tested by quantitative real-time \\npolymerase chain reaction among our clinical samples. Eventually, CD74 was \\nconfirmed to be the most significant highly expressed in BPD samples (P\\u200a<\\u200a.001) \\nand its expression level was negatively correlated with gestational age \\n(r\\u200a=\\u200a-0.653) and birth weight (r\\u200a=\\u200a-0.675). The gene set enrichment analysis \\nresults showed the gene sets associated with lupus erythematosus, viral \\nmyocarditis, immune network for IgA production, graft versus host disease, cell \\nadhesion molecules and so no were differentially enriched with the phenotype of \\nhigh-expression CD74. In conclusion, CD74 may serve to predict the BPD \\ndevelopment and provide a new therapeutic target for BPD.\\n\\nDOI: 10.1097/MD.0000000000023477\\nPMCID: PMC7710202\\nPMID: 33235138 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors have no conflicts of interest to \\ndisclose.\\n\\n\\n6. Kidney Int. 2021 Mar;99(3):671-685. doi: 10.1016/j.kint.2020.08.007. Epub 2020\\n Sep 2.\\n\\nHLA-D and PLA2R1 risk alleles associate with recurrent primary membranous \\nnephropathy in kidney transplant recipients.\\n\\nBerchtold L(1), Letouzé E(2), Alexander MP(3), Canaud G(4), Logt AV(5), Hamilton \\nP(6), Mousson C(7), Vuiblet V(8), Moyer AM(9), Guibert S(10), Mrázová P(11), \\nLevi C(12), Dubois V(13), Cruzado JM(14), Torres A(15), Gandhi MJ(16), Yousfi \\nN(17), Tesar V(18), OndrejViklický(19), Hourmant M(20), Moulin B(21), Rieu \\nP(22), Choukroun G(23), Legendre C(24), Wetzels J(5), Brenchley P(6), Ballarín \\nCastan JA(25), Debiec H(17), Ronco P(26).\\n\\nAuthor information:\\n(1)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France; Division of Nephrology, Department of \\nInternal Medicine Specialties, University Hospital of Geneva, Geneva, \\nSwitzerland.\\n(2)Centre de Recherche des Cordeliers, Sorbonne Université, INSERM, France; \\nFunctional Genomics of Solid Tumor, Labex Immuno-Oncology, Equipe Labellisée \\nLigue Contre le Cancer, Université Paris 13, Paris, France.\\n(3)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, \\nMinnesota, USA.\\n(4)Inserm U1151, Necker-Enfants Malades Hospital, Paris, France; University \\nParis Descartes, Sorbonne Paris Cité, Paris, France; Department of Adult \\nNephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, France.\\n(5)Department of Nephrology, Radboud University Medical Center, Radboud \\nInstitute for Health Sciences, Nijmegen, The Netherlands.\\n(6)Division of Cardiovascular Sciences, School of Medical Sciences, Faculty of \\nBiology, Medicine and Health, University of Manchester, Manchester, UK.\\n(7)Department of Nephrology and Transplantation, University Hospital, Dijon, \\nFrance.\\n(8)BioSpec T Laboratory, EA 7506 URCA, Reims, France; Nephropathology Department \\nof Biopathology Laboratory, Reims University Hospital, Reims, France; Division \\nof Nephrology, Reims University Hospital, Reims, France.\\n(9)Department of Laboratory Medicine and Pathology, Personalized Genomics \\nLaboratory Mayo Clinic, Rochester, Minnesota, USA.\\n(10)IntegraGen, Evry, France.\\n(11)Transplant Laboratory, Institute for Clinical and Experimental Medicine, \\nPrague, Czech Republic.\\n(12)Department of Transplantation, Nephrology, and Clinical Immunology, Edouard \\nHerriot Hospital, Hospices Civils de Lyon, Lyon, France.\\n(13)Laboratoire HLA, Etablissement Français du Sang Auvergne Rhone-Alpes, Lyon, \\nFrance.\\n(14)Nephrology Department, Hospital Universitari Bellvitge, Bellvitge Research \\nInstitute (IDIBELL), Barcelona, Spain; RedInRen, RD16/0009/0031, University of \\nBarcelona, L\\'Hospitalet de Llobregat, Barcelona, Spain.\\n(15)Hospital Universitario de Canarias, Instituto de Tecnologías Biomédicas \\n(ITB), Universidad de La Laguna, Tenerife, Spain.\\n(16)Division of Transfusion Medicine, Mayo Clinic, Rochester, Minnesota, USA.\\n(17)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France.\\n(18)Department of Nephrology, First Faculty of Medicine, Charles University and \\nGeneral University Hospital, Prague, Czech Republic.\\n(19)Department of Nephrology, Transplant Center, Institute for Clinical and \\nExperimental Medicine, Prague, Czech Republic.\\n(20)Department of Nephrology, University Hospital, Nantes, France.\\n(21)Institut National de la Santé et de la Recherche Médicale, Unité Mixte de \\nRecherche S 1109, Strasbourg University, Strasbourg, France; Fédération de \\nMédecine Translationnelle de Strasbourg (FMTS), Strasbourg, France; Department \\nof Nephrology, Strasbourg University Hospital, Strasbourg, France.\\n(22)Division of Nephrology, Reims University Hospital, Reims, France; Université \\nde Reims Champagne-Ardenne, CNRS UMR 7369 (Matrice Extracellulaire et Dynamique \\nCellulaire, MEDyC), Reims, France.\\n(23)Department of Nephrology, University Hospital, Amiens, France.\\n(24)University Paris Descartes, Sorbonne Paris Cité, Paris, France; Department \\nof Adult Nephrology and Transplantation, Necker-Enfants Malades Hospital, Paris, \\nFrance.\\n(25)Department of Nephrology, Fundación Puigvert, Barcelona, Spain.\\n(26)Sorbonne Université, Université Pierre et Marie Curie Paris 06, Paris, \\nFrance; Institut National de la Santé et de la Recherche Médicale, Unité Mixte \\nde Recherche S1155, Paris, France; Department of Nephrology (Day Hospital), \\nAP-HP, Tenon Hospital, Paris, France. Electronic address: pierre.ronco@upmc.fr.\\n\\nErratum in\\n    Kidney Int. 2021 Jul;100(1):243.\\n\\nComment in\\n    Kidney Int. 2021 Mar;99(3):545-548.\\n\\nRecurrence of primary membranous nephropathy after transplantation occurs in up \\nto 44% of patients and is driven by PLA2R antibody. Here, we asked whether \\ngenetic determinants could improve risk prediction. First, we sequenced PLA2R1 \\nand HLA-D loci in 248 patients with primary membranous nephropathy and \\nidentified two independent single nucleotide polymorphisms (SNPs) at risk for \\nprimary membranous nephropathy at each locus. These were rs9271188 (intergenic \\nbetween HLA-DRB1 and HLA-DQA1,) and rs9275086 (intergenic between HLA-DQB1 and \\nHLA-DQA2) at the HLA-D locus along with rs6726925 and rs13018963 at the PLA2R1 \\nlocus. Then we investigated whether primary membranous nephropathy at-risk \\nvariants were associated with recurrence in a retrospective cohort of 105 \\ndonor-recipient pairs and a replication cohort of 40 pairs. Seven SNPs located \\nbetween HLA-DRB1 and HLA-DQA1 in linkage disequilibrium with rs9271188, and \\nthree SNPs in the PLA2R1 region predicted recurrence when presented by the \\ndonor, but not when presented by the recipient. The two SNPs in the HLA-D region \\nmost strongly associated with recurrence (rs9271705 and rs9271550) were \\nconfirmed in the replication cohort. A genetic risk score based on the two best \\npredictors at each locus (rs9271705, rs9271550, rs17830558, and rs3828323) \\nidentified a group of patients with high risk of recurrence. Thus, our results \\nsuggest that the graft contributes to recurrence of primary membranous \\nnephropathy through the disease susceptibility HLA-D and PLA2R1 SNPs in an \\nautoimmune milieu. Further studies are needed before implementation of genetic \\ntesting for these in donor selection.\\n\\nCopyright © 2020. Published by Elsevier Inc.\\n\\nDOI: 10.1016/j.kint.2020.08.007\\nPMID: 32889013 [Indexed for MEDLINE]\\n\\n\\n7. Lupus. 2013 Feb;22(2):198-204. doi: 10.1177/0961203312470183. Epub 2012 Dec\\n20.\\n\\nHLA variants rs9271366 and rs9275328 are associated with systemic lupus \\nerythematosus susceptibility in Malays and Chinese.\\n\\nChai HC(1), Phipps ME, Othman I, Tan LP, Chua KH.\\n\\nAuthor information:\\n(1)Jeffrey Cheah School of Medicine and Health Sciences, Monash University \\n(Sunway Campus), Malaysia.\\n\\nBACKGROUND: Human leukocyte antigen (HLA) antigens and genes have long been \\nreported associated with systemic lupus erythematosus (SLE) susceptibility in \\nmany populations. With the advance in technologies such as genome-wide \\nassociation studies, many newly discovered SLE-associated single-nucleotide \\npolymorphisms (SNPs) have been reported in recent years. These include \\nHLA-DRB1/HLA-DQA1 rs9271366 and HLA-DQB1/HLA-DQA2 rs9275328. Our aim was to \\ninvestigate these SNPs in a Malaysian SLE cohort.\\nMATERIALS AND METHODS: SNPs rs9271366 and rs9275328 were screened across 790 \\nMalaysian citizens from three ethnic groups (360 patients and 430 healthy \\nvolunteers) by Taqman SNP genotyping assays. Allele and genotyping frequencies, \\nHardy-Weinberg equilibrium, Fisher\\'s exact test and odds ratio were calculated \\nfor each SNP and ethnic group. Linkage disequilibrium and interaction between \\nthe two SNPs were also evaluated.\\nRESULTS: The minor allele G and its homozygous genotype GG of HLA-DRB1/HLA-DQA1 \\nrs9271366 significantly increased the SLE susceptibility in Malaysian patients, \\nincluding those of Malay and Chinese ethnicity (odds ratio (OR)\\u2009>\\u20091, p\\u2009<\\u20090.05). \\nAs for HLA-DQB1/HLA-DQA2 rs9275328, the minor allele T and the heterozygous \\ngenotype CT conferred protective effect to SLE in Malaysians, as well as in \\nMalays and Chinese, by having OR\\u2009<\\u20091 and p value <0.05. Both SNPs did not show \\nassociations to SLE in Indians. D\\' and r (2) values for the two SNPs in LD \\nanalysis were 0.941 and 0.065, respectively, with haplotype GC and AT being \\nsignificantly associated with SLE (p\\u2009<\\u20095.0\\u2009×\\u200910(-4)) after 10,000 permutations \\nwere performed. The MDR test clustered the genotype combinations of GG and CC, \\nand AG and CC of rs9271366 and rs9275328, accordingly, as high-risk group, and \\nthe two SNPs interacted redundantly by removing 1.96% of the entropy.\\nCONCLUSIONS: Our findings suggest that in addition to some classical HLA \\nvariants, rs9271366 and rs9275328 are additional polymorphisms worth considering \\nin the Malaysian and possibly in a larger Asian SLE scenario.\\n\\nDOI: 10.1177/0961203312470183\\nPMID: 23257407 [Indexed for MEDLINE]\\n\\n\\n8. Clin Exp Allergy. 2012 Dec;42(12):1724-33. doi: 10.1111/cea.12000.\\n\\nHLA-DQ strikes again: genome-wide association study further confirms HLA-DQ in \\nthe diagnosis of asthma among adults.\\n\\nLasky-Su J(1), Himes BE, Raby BA, Klanderman BJ, Sylvia JS, Lange C, Melen E, \\nMartinez FD, Israel E, Gauderman J, Gilliland F, Sleiman P, Hakonarson H, \\nCeledón JC, Soto-Quiros M, Avila L, Lima JJ, Irvin CG, Peters SP, Boushey H, \\nChinchilli VM, Mauger D, Tantisira K, Weiss ST; SHARP investigators.\\n\\nAuthor information:\\n(1)Channing Laboratory, Department of Medicine, Brigham and Women\\'s Hospital, \\nand Harvard Medical School, Boston, MA, USA.\\n\\nBACKGROUND: Asthma is a common chronic respiratory disease in children and \\nadults. An important genetic component to asthma susceptibility has long been \\nrecognized, most recently through the identification of several genes (e.g., \\nORMDL3, PDE4D, HLA-DQ, and TLE4) via genome-wide association studies.\\nOBJECTIVE: To identify genetic variants associated with asthma affection status \\nusing genome-wide association data.\\nMETHODS: We describe results from a genome-wide association study on asthma \\nperformed in 3855 subjects using a panel of 455 089 single nucleotide \\npolymorphisms (SNPs).\\nRESULT: The genome-wide association study resulted in the prioritization of 33 \\nvariants for immediate follow-up in a multi-staged replication effort. Of these, \\na common polymorphism (rs9272346) localizing to within 1 Kb of HLA-DQA1 \\n(chromosome 6p21.3) was associated with asthma in adults (P-value = 2.2E-08) \\nwith consistent evidence in the more heterogeneous group of adults and children \\n(P-value = 1.0E-04). Moreover, some genes identified in prior asthma GWAS were \\nnominally associated with asthma in our populations.\\nCONCLUSION: Overall, our findings further replicate the HLA-DQ region in the \\npathogenesis of asthma. HLA-DQA1 is the fourth member of the HLA family found to \\nbe associated with asthma, in addition to the previously identified HLA-DRA, \\nHLA-DQB1 and HLA-DQA2.\\n\\n© 2012 Blackwell Publishing Ltd.\\n\\nDOI: 10.1111/cea.12000\\nPMCID: PMC6343489\\nPMID: 23181788 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Conflict of Interests: Charles Irvin’s work was \\nfunded by a grant from the American Lung Association. Stephen Peters has been a \\nmember of the NHLBI’s ACRN since its creation in 1993. Homer Boushey received \\nfunding from the NIH for his participation in the research of the Inner City \\nAsthma Consortium, from Genentech for research on the airway microbiome of \\nsevere asthma, and from GlaxoSmithKline for research on bronchial epithelial \\ncell susceptibility to viral infection. He has served as an ad-hoc consultant or \\nadvisory board member for Genentech, GlaxoSmithKline, Kalobios, Pharmaxis, Merck \\nand Johnson & Johnson. Vernon Chinchilli, Blanca Himes, Scott Weiss, Juan \\nCeledon, Barbara Klanderman, Erik Melen, Hakon Hakonarson, Lydiana Avila, \\nBenjamin Raby, Kelan Tantisira, David Mauger, Patrick Sleiman, Fernando \\nMartinez, W. James Gauderman, Frank Gilliland, John Lima, Elliot Israel, \\nChristoph Lange, Jody Senter Sylvia, Manuel Soto Quiros and Jessica Lasky-Su \\nhave no conflict of interests to declare\\n\\n\\n9. J Immunol. 2012 Apr 15;188(8):3903-11. doi: 10.4049/jimmunol.1103048. Epub\\n2012  Mar 9.\\n\\nHLA-DQA2 and HLA-DQB2 genes are specifically expressed in human Langerhans cells \\nand encode a new HLA class II molecule.\\n\\nLenormand C(1), Bausinger H, Gross F, Signorino-Gelo F, Koch S, Peressin M, \\nFricker D, Cazenave JP, Bieber T, Hanau D, de la Salle H, Tourne S.\\n\\nAuthor information:\\n(1)INSERM, Unité Mixte de Recherche S725, Biologie des Cellules Dendritiques \\nHumaines, Strasbourg, 67065 France.\\n\\nThe precise role of human epidermal Langerhans cells (LCs) in immune response is \\nhighly controversial. While studying the gene expression profile of these cells, \\nwe were intrigued to identify the HLA-DQB2 gene as potentially expressed in LCs. \\nDespite a strong evolutionary conservation of their sequences, the concomitant \\nexpression of the poorly polymorphic HLA-DQA2/HLA-DQB2 genes, paralogous to the \\nHLA-DQA1/HLA-DQB1 genes, has never been detected in any cell type. We confirmed \\nby RT-PCR that the HLA-DQA2 and -DQB2 genes are both expressed in LCs, but not \\nin monocyte-derived dendritic cells, or in blood CD1c(+) or plasmacytoid \\ndendritic cells. The presence of the HLA-DQβ2 chain in LCs could be demonstrated \\nby Western blotting, whereas immunofluorescence revealed its localization in \\nearly endosomes. As in the case of other HLA class II molecules, the HLA-DQα2 \\nand -DQβ2 chains formed heterodimers that had to associate with the invariant \\nchain to reach endosomal compartments. HLA-DQα2/β2 heterodimers were expressed \\nat the cell surface, where they could mediate staphylococcal superantigen \\nstimulation of T cells. Interestingly, HLA-DQα2 and HLA-DQβ1 chains formed mixed \\nheterodimers which efficiently left the endoplasmic reticulum. These \\nobservations strongly suggest that the poorly polymorphic HLA-DQA2 and -DQB2 \\ngenes should be considered to be of immunological importance. The HLA-DQα2/β2 \\nmolecules could influence the complexity of the repertoire of Ags presented by \\nLCs.\\n\\nDOI: 10.4049/jimmunol.1103048\\nPMID: 22407913 [Indexed for MEDLINE]\\n\\n\\n10. PLoS One. 2010 Apr 13;5(4):e10142. doi: 10.1371/journal.pone.0010142.\\n\\nThe effect of single nucleotide polymorphisms from genome wide association \\nstudies in multiple sclerosis on gene expression.\\n\\nHandel AE(1), Handunnetthi L, Berlanga AJ, Watson CT, Morahan JM, Ramagopalan \\nSV.\\n\\nAuthor information:\\n(1)Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, \\nUnited Kingdom.\\n\\nBACKGROUND: Multiple sclerosis (MS) is a complex neurological disorder. Its \\naetiology involves both environmental and genetic factors. Recent genome-wide \\nassociation studies have identified a number of single nucleotide polymorphisms \\n(SNPs) associated with susceptibility to (MS). We investigated whether these \\ngenetic variations were associated with alteration in gene expression.\\nMETHODS/PRINCIPAL FINDINGS: We used a database of mRNA expression and genetic \\nvariation derived from immortalised peripheral lymphocytes to investigate \\npolymorphisms associated with MS for correlation with gene expression. Several \\nSNPs were found to be associated with changes in expression: in particular two \\nwith HLA-DQA1, HLA-DQA2, HLA-DQB1, HLA-DRB1, HLA-DRB4 and HLA-DRB5, one with \\nZFP57, one with CD58, two with IL7 and FAM164A, and one with FAM119B, TSFM and \\nKUB3. We found minimal cross-over with a recent whole genome expression study in \\nMS patients.\\nDISCUSSION: We have shown that many susceptibility loci in MS are associated \\nwith changes in gene expression using an unbiased expression database. Several \\nof these findings suggest novel gene candidates underlying the effects of \\nMS-associated genetic variation.\\n\\nDOI: 10.1371/journal.pone.0010142\\nPMCID: PMC2854120\\nPMID: 20405052 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Competing Interests: The authors have declared \\nthat no competing interests exist.\\n\\n\\n11. Hum Immunol. 2001 May;62(5):504-8. doi: 10.1016/s0198-8859(01)00236-1.\\n\\nAbsence of in vivo DNA-protein interactions in the DQA2 and DQB2 promoter \\nregions.\\n\\nIndovina P(1), Megiorni F, Fontemaggi G, Coni P, Mora B, Mazzilli MC.\\n\\nAuthor information:\\n(1)Dipartimento di Medicina Sperimentale e Patologia, Università \"La Sapienza,\", \\nRome, Italy.\\n\\nThe DQ subregion of the human major histocompatibility complex (HLA) contains \\ntwo pairs of loci: the DQA1/B1 genes (hereafter called DQ1), coding for the DQ \\nmolecules, and the DQA2/B2 pseudogenes (hereafter called DQ2). These pseudogenes \\nare highly homologous to the functional DQ1 genes and they have no apparent \\nabnormal features in their sequences that could prevent their activity. Only \\nrecently a low expression of the DQA2 gene has been observed whereas the DQB2 \\ntranscript was never found. The comparison between the DQ1 and DQ2 regulatory \\nsequences revealed several differences in their W, X, and Y cis-acting elements. \\nTo examine the DNA/protein interactions in the DQ promoter regions, we performed \\nin vivo footprinting experiments. Whereas the functional DQ1 loci showed a \\nseries of DNA-protein contact points in the X and Y boxes, the promoters of the \\nDQ2 pseudogenes displayed an unoccupied phenotype. These findings suggest that \\nthe very low level of DQA2 expression and the apparent lack of DQB2 activity are \\ncaused by the reduced binding affinity of specific transcription factors.\\n\\nDOI: 10.1016/s0198-8859(01)00236-1\\nPMID: 11334674 [Indexed for MEDLINE]'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'HLA-DQA1 HLA-DQA2 HLA-DQB1',retmax=11,save='y')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 13 resultados em pubmed. Irão ser processados 10 resultados.\n",
      "\n",
      "1. J Clin Invest. 2022 Jan 4;132(1):e155609. doi: 10.1172/JCI155609.\n",
      "\n",
      "Early peanut introduction wins over the HLA-DQA1*01:02 allele in the interplay \n",
      "between environment and genetics.\n",
      "\n",
      "Manohar M, Nadeau KC, Kasowski M.\n",
      "\n",
      "Comment on\n",
      "    J Clin Invest. 2022 Jan 4;132(1):\n",
      "\n",
      "The rising incidence of food allergy in children underscores the importance of \n",
      "environmental exposures; however, genetic factors play a major role. How the \n",
      "environment and genetics interact to cause food allergy remains unclear. The \n",
      "landmark Learning Early About Peanut Allergy (LEAP) clinical trial established \n",
      "that early peanut introduction protects high-risk infants, consistent with the \n",
      "tolerizing effects of gut exposure. In this issue of the JCI, Kanchan et al. \n",
      "leveraged the LEAP trial data to examine molecular genetic mechanisms of early \n",
      "sensitization. A previously identified HLA risk allele for peanut allergy \n",
      "(DQA1*01:02) was associated with peanut-specific IgG4 levels in consumers. \n",
      "Notably, IgG4 antibodies likely provide protection by reducing the binding of \n",
      "allergen to IgE. The association of the same allele with peanut allergy in \n",
      "avoiders while potentially conferring protection in consumers reinforces the \n",
      "need to integrate genetic information toward a personalized therapeutic strategy \n",
      "for the best outcome in addressing food allergies.\n",
      "\n",
      "DOI: 10.1172/JCI155609\n",
      "PMCID: PMC8718134\n",
      "PMID: 34981779 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "2. J Clin Invest. 2022 Jan 4;132(1):e152070. doi: 10.1172/JCI152070.\n",
      "\n",
      "HLA alleles and sustained peanut consumption promote IgG4 responses in subjects \n",
      "protected from peanut allergy.\n",
      "\n",
      "Kanchan K(1), Grinek S(2), Bahnson HT(3)(4), Ruczinski I(5), Shankar G(1), \n",
      "Larson D(4), Du Toit G(6), Barnes KC(7), Sampson HA(8), Suarez-Farinas M(2)(9), \n",
      "Lack G(6), Nepom GT(3)(4), Cerosaletti K(3), Mathias RA(1).\n",
      "\n",
      "Author information:\n",
      "(1)Division of Allergy and Clinical Immunology, Department of Medicine, School \n",
      "of Medicine, Johns Hopkins University, Baltimore, Maryland, USA.\n",
      "(2)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \n",
      "Mount Sinai, New York, New York, USA.\n",
      "(3)Benaroya Research Institute at Virginia Mason, Seattle, Washington, USA.\n",
      "(4)The Immune Tolerance Network, Bethesda, Maryland, USA.\n",
      "(5)Department of Biostatistics, Bloomberg School of Public Health, Johns Hopkins \n",
      "University, Baltimore, Maryland, USA.\n",
      "(6)The Department of Pediatric Allergy, Division of Asthma, Allergy and Lung \n",
      "Biology, King's College London, and Guy's and St. Thomas' NHS Foundation Trust, \n",
      "London, United Kingdom.\n",
      "(7)The Department of Medicine, University of Colorado, Anschutz, Colorado, USA.\n",
      "(8)Department of Pediatrics, Allergy and Immunology and.\n",
      "(9)Department of Population Health Science and Policy, Icahn School of Medicine \n",
      "at Mount Sinai, New York, New York, USA.\n",
      "\n",
      "Comment in\n",
      "    J Clin Invest. 2022 Jan 4;132(1):\n",
      "\n",
      "We investigated the interplay between genetics and oral peanut protein exposure \n",
      "in the determination of the immunological response to peanut using the targeted \n",
      "intervention in the LEAP clinical trial. We identified an association between \n",
      "peanut-specific IgG4 and HLA-DQA1*01:02 that was only observed in the presence \n",
      "of sustained oral peanut protein exposure. The association between IgG4 and \n",
      "HLA-DQA1*01:02 was driven by IgG4 specific for the Ara h 2 component. Once \n",
      "peanut consumption ceased, the association between IgG4-specific Ara h 2 and \n",
      "HLA-DQA1*01:02 was attenuated. The association was validated by observing \n",
      "expanded IgG4-specific epitopes in people who carried HLA-DQA1*01:02. Notably, \n",
      "we confirmed the previously reported associations with HLA-DQA1*01:02 and peanut \n",
      "allergy risk in the absence of oral peanut protein exposure. Interaction between \n",
      "HLA and presence or absence of exposure to peanut in an allergen- and \n",
      "epitope-specific manner implicates a mechanism of antigen recognition that is \n",
      "fundamental to driving immune responses related to allergy risk or protection.\n",
      "\n",
      "DOI: 10.1172/JCI152070\n",
      "PMCID: PMC8718139\n",
      "PMID: 34981778 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Conflict of interest: GL reports holding stock \n",
      "and stock options in DBV Technologies. HTB has provided statistical consulting \n",
      "to DBV Technologies.\n",
      "\n",
      "\n",
      "3. Ann Allergy Asthma Immunol. 2015 Dec;115(6):471-6. doi: \n",
      "10.1016/j.anai.2015.10.008. Epub 2015 Oct 27.\n",
      "\n",
      "The evolving story of human leukocyte antigen and the immunogenetics of peanut \n",
      "allergy.\n",
      "\n",
      "Hemler JA(1), Phillips EJ(2), Mallal SA(3), Kendall PL(4).\n",
      "\n",
      "Author information:\n",
      "(1)Division of Allergy, Pulmonary and Critical Care, Department of Medicine, \n",
      "Vanderbilt University School of Medicine, Nashville, Tennessee.\n",
      "(2)Division of Infectious Diseases, Department of Medicine, Vanderbilt \n",
      "University School of Medicine, Nashville, Tennessee; Department of Pathology, \n",
      "Microbiology and Immunology, Vanderbilt University School of Medicine, \n",
      "Nashville, Tennessee; Department of Pharmacology, Oates Institute for \n",
      "Experimental Therapeutics, Vanderbilt University School of Medicine, Nashville, \n",
      "Tennessee; Institute for Immunology and Infectious Diseases, Murdoch University, \n",
      "Murdoch, Western Australia, Australia.\n",
      "(3)Division of Infectious Diseases, Department of Medicine, Vanderbilt \n",
      "University School of Medicine, Nashville, Tennessee; Center for Translational \n",
      "Immunology and Infectious Diseases, Vanderbilt University School of Medicine, \n",
      "Nashville, Tennessee; Department of Pathology, Microbiology and Immunology, \n",
      "Vanderbilt University School of Medicine, Nashville, Tennessee; Institute for \n",
      "Immunology and Infectious Diseases, Murdoch University, Murdoch, Western \n",
      "Australia, Australia.\n",
      "(4)Division of Allergy, Pulmonary and Critical Care, Department of Medicine, \n",
      "Vanderbilt University School of Medicine, Nashville, Tennessee; Department of \n",
      "Pathology, Microbiology and Immunology, Vanderbilt University School of \n",
      "Medicine, Nashville, Tennessee. Electronic address: \n",
      "peggy.kendall@vanderbilt.edu.\n",
      "\n",
      "OBJECTIVE: Peanut allergy (PA) clearly has a heritable component. Specific \n",
      "genetic contributions are unknown, but human leukocyte antigen (HLA) loci are \n",
      "obvious candidates. This review focuses on emerging studies of HLA associations \n",
      "with PA.\n",
      "DATA SOURCES: PubMed was searched with no time limitations using key terms human \n",
      "leukocyte antigen, HLA, MHC, peanut, peanut hypersensitivity, and peanut \n",
      "allergy.\n",
      "STUDY SELECTIONS: Qualifying studies were English-language reports of genetic \n",
      "analyses examining PA and HLA associations.\n",
      "RESULTS: Seven relevant citations were identified, which were published from \n",
      "1996 to 2015. Early studies using candidate gene approaches found associations \n",
      "between PA and HLA-DR and -DQ alleles (HLA-DRB1*08 and DQB1*06:03P) when \n",
      "comparing subjects with peanut allergy with nonallergic unrelated control \n",
      "groups. No significant associations were found between siblings with and without \n",
      "peanut allergy. However, a recent large genomewide association study of patients \n",
      "with peanut allergy and their family members found 2 PA-associated \n",
      "single-nucleotide polymorphisms (rs9275596 and rs7192) mapping to regions \n",
      "involving the HLA-DR and HLA-DQ genes. Associations with differential DNA \n",
      "methylation partly mediated the associations between PA and single-nucleotide \n",
      "polymorphisms.\n",
      "CONCLUSION: Early studies using candidate gene approaches identified HLA \n",
      "associations with PA compared with the general population, suggesting a link \n",
      "with atopy but failing to identify a PA-specific association. These studies had \n",
      "various limitations that included small samples. The most compelling evidence \n",
      "for a PA-specific HLA association comes from a genomewide association study, \n",
      "which examined the entire genome in large, well-defined, related cohorts. More \n",
      "research is needed to validate and replicate these findings, to perform fine \n",
      "genetic mapping of specific HLA loci, and to demonstrate underlying mechanisms \n",
      "of HLA contributions to PA.\n",
      "\n",
      "Copyright © 2015 American College of Allergy, Asthma & Immunology. Published by \n",
      "Elsevier Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.anai.2015.10.008\n",
      "PMCID: PMC5665015\n",
      "PMID: 26522257 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "4. Curr Opin Allergy Clin Immunol. 2015 Oct;15(5):409-16. doi: \n",
      "10.1097/ACI.0000000000000196.\n",
      "\n",
      "Epidemiology of food allergy and food-induced anaphylaxis: is there really a \n",
      "Western world epidemic?\n",
      "\n",
      "Koplin JJ(1), Mills EN, Allen KJ.\n",
      "\n",
      "Author information:\n",
      "(1)aMurdoch Childrens Research Institute, University of Melbourne Department of \n",
      "Paediatrics and Royal Children's Hospital bSchool of Population and Global \n",
      "Health, University of Melbourne, Parkville, VIC, Australia cUniversity of \n",
      "Manchester, Institute of Inflammation and Repair and Manchester Institute of \n",
      "Biotechnology, Manchester, UK.\n",
      "\n",
      "PURPOSE OF REVIEW: Food-induced anaphylaxis continues to be an important cause \n",
      "of hospital admissions, particularly in children. This review outlines recent \n",
      "advances in understanding the epidemiology of IgE-mediated food allergy and \n",
      "potential mechanisms for its rise in prevalence.\n",
      "RECENT FINDINGS: The rise in food allergy prevalence in Western countries has \n",
      "happened more quickly than changes to the genome can occur; thus, environmental \n",
      "changes are likely to be important. Recent studies, however, suggest that \n",
      "genetic risk determines responses to environmental risk factors. Environmental \n",
      "peanut exposure has been associated with increased peanut allergy risk in \n",
      "individuals with filaggrin null mutations, consistently with sensitization \n",
      "occurring through a damaged skin barrier. Reduced microbial and vitamin D \n",
      "exposure is also leading candidates for risk factors for food allergy in the \n",
      "context of genetic susceptibility. In addition, HLA-DR and HLA-DQ gene region \n",
      "variants appear to play a role in peanut allergy, although no studies have yet \n",
      "assessed their susceptibility to environmental cues. Finally, findings from \n",
      "observational cohorts and the first large-scale intervention trials for food \n",
      "allergy prevention support early oral allergen exposure to reduce the prevalence \n",
      "of specific food allergies, which is informing changes in public health \n",
      "guidelines at the population level. Further research will be required to assess \n",
      "the impact of these guideline changes on the population prevalence of food \n",
      "allergy.\n",
      "SUMMARY: New studies are providing important insights into the prevalence, \n",
      "causes, and mechanisms of food allergy. Recent findings are informing changes to \n",
      "population health guidelines in developed countries, which have the potential to \n",
      "halt or reverse the increase in food allergy prevalence. By contrast, food \n",
      "allergy in the developing world remains understudied.\n",
      "\n",
      "DOI: 10.1097/ACI.0000000000000196\n",
      "PMID: 26258921 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "5. Nat Commun. 2015 Feb 24;6:6304. doi: 10.1038/ncomms7304.\n",
      "\n",
      "Genome-wide association study identifies peanut allergy-specific loci and \n",
      "evidence of epigenetic mediation in US children.\n",
      "\n",
      "Hong X(1), Hao K(2), Ladd-Acosta C(3), Hansen KD(4), Tsai HJ(5), Liu X(6), Xu \n",
      "X(7), Thornton TA(8), Caruso D(1), Keet CA(9), Sun Y(10), Wang G(1), Luo W(11), \n",
      "Kumar R(12), Fuleihan R(12), Singh AM(13), Kim JS(14), Story RE(15), Gupta \n",
      "RS(16), Gao P(17), Chen Z(1), Walker SO(1), Bartell TR(16), Beaty TH(3), Fallin \n",
      "MD(18), Schleimer R(19), Holt PG(20), Nadeau KC(21), Wood RA(22), Pongracic \n",
      "JA(12), Weeks DE(23), Wang X(24).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Population, Family and Reproductive Health, Center on the Early \n",
      "Life Origins of Disease, Johns Hopkins University Bloomberg School of Public \n",
      "Health, 615 North Wolfe Street, E4132, Baltimore, Maryland 21205, USA.\n",
      "(2)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \n",
      "Mount Sinai, New York, New York 10029, USA.\n",
      "(3)Department of Epidemiology, Johns Hopkins University Bloomberg School of \n",
      "Public Health, Baltimore, Maryland 21205, USA.\n",
      "(4)1] Department of Biostatistics, Johns Hopkins University Bloomberg School of \n",
      "Public Health Baltimore, Baltimore, Maryland 21205, USA [2] McKusick-Nathans \n",
      "Insitute of Genetic Medicine, Johns Hopkins University School of Medicine, \n",
      "Baltimore, Maryland 21205, USA.\n",
      "(5)1] Mary Ann and J. Milburn Smith Child Health Research Program, Department of \n",
      "Pediatrics, Northwestern University Feinberg School of Medicine and Stanley \n",
      "Manne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \n",
      "of Chicago, Chicago, Illinois 60611, USA [2] Division of Biostatistics and \n",
      "Bioinformatics, Institute of Population Health Sciences, National Health \n",
      "Research Institutes, Zhunan 35053, Taiwan [3] Department of Public Health, China \n",
      "Medical University, Taichung 40402, Taiwan.\n",
      "(6)1] Mary Ann and J. Milburn Smith Child Health Research Program, Department of \n",
      "Pediatrics, Northwestern University Feinberg School of Medicine and Stanley \n",
      "Manne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \n",
      "of Chicago, Chicago, Illinois 60611, USA [2] Department of Preventive Medicine, \n",
      "Northwestern University Feinberg School of Medicine, Chicago, Illinois 60611, \n",
      "USA.\n",
      "(7)Guangdong Provincial Institute of Nephrology, Southern Medical University, \n",
      "Guangzhou 510515, China.\n",
      "(8)Department of Biostatistics, University of Washington, Seattle, Washington \n",
      "98195, USA.\n",
      "(9)1] Department of Epidemiology, Johns Hopkins University Bloomberg School of \n",
      "Public Health, Baltimore, Maryland 21205, USA [2] Division of Pediatric Allergy \n",
      "and Immunology, Johns Hopkins University School of Medicine, Baltimore, Maryland \n",
      "21287, USA.\n",
      "(10)Department of Biostatistics, Johns Hopkins University Bloomberg School of \n",
      "Public Health Baltimore, Baltimore, Maryland 21205, USA.\n",
      "(11)1] Department of Genetics and Genomic Sciences, Icahn School of Medicine at \n",
      "Mount Sinai, New York, New York 10029, USA [2] College of Computer Science and \n",
      "Technology, Huaqiao University, Xiamen 361021, China.\n",
      "(12)Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \n",
      "Hospital of Chicago, Chicago, Illinois 60611, USA.\n",
      "(13)Department of Pediatrics and Medicine, Ann and Robert H. Lurie Children's \n",
      "Hospital of Chicago, Northwestern Feinberg School of Medicine, Chicago, Illinois \n",
      "61611, USA.\n",
      "(14)1] Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \n",
      "Hospital of Chicago, Chicago, Illinois 60611, USA [2] NorthShore University \n",
      "HealthSystem, Evanston, Illinois 60201, USA.\n",
      "(15)1] Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \n",
      "Hospital of Chicago, Chicago, Illinois 60611, USA [2] NorthShore University \n",
      "Health Systems, Pritzker School of Medicine, University of Chicago, Chicago, \n",
      "Illinois 60637, USA.\n",
      "(16)Mary Ann and J. Milburn Smith Child Health Research Program, Department of \n",
      "Pediatrics, Northwestern University Feinberg School of Medicine and Stanley \n",
      "Manne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \n",
      "of Chicago, Chicago, Illinois 60611, USA.\n",
      "(17)Division of Allergy and Clinical Immunology, Johns Hopkins University School \n",
      "of Medicine, Baltimore, Maryland 21224, USA.\n",
      "(18)Department of Mental Health, Wendy Klag Center for Autism and Developmental \n",
      "Disabilities, Bloomberg School of Public Health, Johns Hopkins University, \n",
      "Baltimore, Maryland 21205, USA.\n",
      "(19)Division of Allergy-Immunology, Northwestern University Feinberg School of \n",
      "Medicine, Chicago, Illinois 60611, USA.\n",
      "(20)Telethon Kids Institute, University of Western Australia; Perth and \n",
      "Queensland Children's Medical Research Institute, University of Queensland, \n",
      "Brisbane, Queensland 4029, Australia.\n",
      "(21)Division of Allergy, Immunology and Rheumatology, Stanford University School \n",
      "of Medicine, Stanford, California 94305, USA.\n",
      "(22)Division of Pediatric Allergy and Immunology, Johns Hopkins University \n",
      "School of Medicine, Baltimore, Maryland 21287, USA.\n",
      "(23)Departments of Human Genetics and Biostatistics, Graduate School of Public \n",
      "Health, University of Pittsburgh, Pittsburgh, Pennsylvania 15261, USA.\n",
      "(24)1] Department of Population, Family and Reproductive Health, Center on the \n",
      "Early Life Origins of Disease, Johns Hopkins University Bloomberg School of \n",
      "Public Health, 615 North Wolfe Street, E4132, Baltimore, Maryland 21205, USA [2] \n",
      "Division of General Pediatrics and Adolescent Medicine, Department of \n",
      "Pediatrics, Johns Hopkins University School of Medicine, Baltimore, Maryland \n",
      "21287, USA.\n",
      "\n",
      "Food allergy (FA) affects 2%-10% of US children and is a growing clinical and \n",
      "public health problem. Here we conduct the first genome-wide association study \n",
      "of well-defined FA, including specific subtypes (peanut, milk and egg) in 2,759 \n",
      "US participants (1,315 children and 1,444 parents) from the Chicago Food Allergy \n",
      "Study, and identify peanut allergy (PA)-specific loci in the HLA-DR and -DQ gene \n",
      "region at 6p21.32, tagged by rs7192 (P=5.5 × 10(-8)) and rs9275596 (P=6.8 × \n",
      "10(-10)), in 2,197 participants of European ancestry. We replicate these \n",
      "associations in an independent sample of European ancestry. These associations \n",
      "are further supported by meta-analyses across the discovery and replication \n",
      "samples. Both single-nucleotide polymorphisms (SNPs) are associated with \n",
      "differential DNA methylation levels at multiple CpG sites (P<5 × 10(-8)), and \n",
      "differential DNA methylation of the HLA-DQB1 and HLA-DRB1 genes partially \n",
      "mediate the identified SNP-PA associations. This study suggests that the HLA-DR \n",
      "and -DQ gene region probably poses significant genetic risk for PA.\n",
      "\n",
      "DOI: 10.1038/ncomms7304\n",
      "PMCID: PMC4340086\n",
      "PMID: 25710614 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: CONFLICTS OF INTERESTS The authors have declared \n",
      "that no conflicting interests exist.\n",
      "\n",
      "\n",
      "6. Clin Exp Allergy. 2013 Jun;43(6):684-97. doi: 10.1111/cea.12113.\n",
      "\n",
      "Ara h 1 CD4+ T cell epitope-based peptides: candidates for a peanut allergy \n",
      "therapeutic.\n",
      "\n",
      "Prickett SR(1), Voskamp AL, Phan T, Dacumos-Hill A, Mannering SI, Rolland JM, \n",
      "O'Hehir RE.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Immunology, Monash University, Melbourne, Vic., Australia.\n",
      "\n",
      "BACKGROUND: Peanut allergy is a life-threatening condition; there is currently \n",
      "no cure. While whole allergen extracts are used for specific immunotherapy for \n",
      "many allergies, they can cause severe reactions and even fatalities in peanut \n",
      "allergy.\n",
      "OBJECTIVE: To identify short, HLA-degenerate CD4(+) T cell epitope-based \n",
      "peptides of the major peanut allergen Ara h 1 that target allergen-specific T \n",
      "cells without causing IgE-mediated inflammatory cell activation, as candidates \n",
      "for safe peanut-specific immunotherapy.\n",
      "METHODS: Ara h 1-specific CD4(+) T cell lines (TCL) were generated from \n",
      "peripheral blood mononuclear cells (PBMC) of peanut-allergic subjects using \n",
      "CFSE-based methodology. T cell epitopes were identified using CFSE and \n",
      "thymidine-based proliferation assays. Epitope HLA-restriction was investigated \n",
      "using blocking antibodies, HLA-genotyping and epitope prediction algorithms. \n",
      "Functional peanut-specific IgE reactivity to peptides was assessed by basophil \n",
      "activation assay.\n",
      "RESULTS: A total of 145 Ara h 1-specific TCL were generated from 18 HLA-diverse \n",
      "peanut-allergic subjects. The TCL recognized 20-mer peptides throughout Ara h 1. \n",
      "Nine 20-mers containing the most frequently recognized epitopes were selected \n",
      "and their recognition confirmed in 18 additional peanut-allergic subjects. Ten \n",
      "core epitopes were mapped within these 20-mers. These were HLA-DQ and/or HLA-DR \n",
      "restricted, with each presented on at least two different HLA-molecules. Seven \n",
      "short (≤ 20 aa) non-basophil-reactive peptides encompassing all core epitopes \n",
      "were designed and validated in peanut-allergic donor PBMC T cell assays.\n",
      "CONCLUSIONS AND CLINICAL RELEVANCE: Short CD4(+) T cell epitope-based Ara h 1 \n",
      "peptides were identified as novel candidates for a safe, T cell targeted \n",
      "peanut-specific immunotherapy for HLA-diverse populations.\n",
      "\n",
      "© 2013 John Wiley & Sons Ltd.\n",
      "\n",
      "DOI: 10.1111/cea.12113\n",
      "PMCID: PMC3709139\n",
      "PMID: 23711131 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. Eur J Hum Genet. 2013 Oct;21(10):1181-4. doi: 10.1038/ejhg.2013.13. Epub 2013 \n",
      "Feb 27.\n",
      "\n",
      "HLA-DQB1*02 and DQB1*06:03P are associated with peanut allergy.\n",
      "\n",
      "Madore AM(1), Vaillancourt VT, Asai Y, Alizadehfar R, Ben-Shoshan M, Michel DL, \n",
      "Kozyrskyj AL, Becker A, Chan-Yeung M, Clarke AE, Hull P, Daley D, Sandford AJ, \n",
      "Laprise C.\n",
      "\n",
      "Author information:\n",
      "(1)Département des Sciences Fondamentales, Université du Québec à Chicoutimi, \n",
      "Saguenay, Québec, Canada.\n",
      "\n",
      "Peanut allergy (PA) is a common and serious food allergy and its prevalence has \n",
      "increased in the past decade. Although there is strong evidence of inheritance, \n",
      "the genetic causes of this disease are not well understood. Previously, a \n",
      "large-scale genome-wide association study described an association between human \n",
      "leukocyte antigen (HLA)-DQB1 and asthma; the aim of this study was to evaluate \n",
      "the association between HLA-DQB1 and PA. Genotypic and allelic profiles were \n",
      "established for 311 Caucasian members of a well-described Canadian group of \n",
      "children with PA and 226 Caucasian controls. Firth's logistic regression \n",
      "analyses showed associations between HLA-DQB1 alleles and PA for DQB1*02 (P=1.1 \n",
      "× 10(-8), odds ratio (OR)=0.09 (CI=0.03-0.23)) and DQB1*06:03P alleles (P=2.1 × \n",
      "10(-2), OR=2.82 (CI=1.48-5.45)). This study of HLA in PA demonstrates specific \n",
      "association between two allelic groups of the HLA-DQB1 gene (DQB1*02 and \n",
      "DQB1*06:03P) and PA, highlighting its possible role in the development of this \n",
      "disease.\n",
      "\n",
      "DOI: 10.1038/ejhg.2013.13\n",
      "PMCID: PMC3778350\n",
      "PMID: 23443026 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "8. Clin Exp Allergy. 2013 Jan;43(1):116-27. doi: 10.1111/cea.12014.\n",
      "\n",
      "In silico prediction of Ara h 2 T cell epitopes in peanut-allergic children.\n",
      "\n",
      "Pascal M(1), Konstantinou GN, Masilamani M, Lieberman J, Sampson HA.\n",
      "\n",
      "Author information:\n",
      "(1)Division of Allergy & Immunology and The Jaffe Food Allergy Research \n",
      "Institute, Mount Sinai School of Medicine, New York, NY, USA.\n",
      "\n",
      "BACKGROUND: Despite the frequency and severity of peanut allergy, the only \n",
      "approved treatment is strict avoidance. Different types of immunotherapy with \n",
      "crude peanut extract are not universally effective and have been associated with \n",
      "relatively high adverse reaction rates.\n",
      "OBJECTIVE: We sought to determine whether in silico predictive algorithms were \n",
      "useful in identifying candidate peptides for an Ara h 2 peptide-based vaccine \n",
      "using peanut-allergic patients' peripheral blood mononuclear cells (PBMCs) in \n",
      "vitro. A human leucocyte antigen (HLA) distribution analysis was also performed.\n",
      "METHODS: Major histocompatibility complex (MHC)-class II-binding peptides were \n",
      "predicted using NetMHCIIpan-2.0 and NetMHCII-2.2 algorithms. PBMCs from 80 \n",
      "peanut-allergic patients were stimulated with overlapping 20-mer Ara h 2 \n",
      "peptides. Cell supernatant cytokine profiles were evaluated by multiplex assays. \n",
      "HLA-DRB1* and HLA-DQB1* typing were performed.\n",
      "RESULTS: Four regions of overlapping sequences induced PBMC proliferation and \n",
      "predominant Th2 cytokine production. HLA genotyping showed 30 different DRB1* \n",
      "allele specificities and eight DQ serological specificities. The in silico \n",
      "analysis revealed similar relevant regions and predicted identical or similar \n",
      "core 9-mer epitopes to those identified in vitro. If relevant peptides, as \n",
      "determined by either in vitro or in silico analysis (15 peptides and 9 core \n",
      "epitopes respectively), were used in a peptide-based vaccine, they would cover \n",
      "virtually all subjects in the cohort studied.\n",
      "CONCLUSIONS AND CLINICAL RELEVANCE: Four dominant regions in Ara h 2 have been \n",
      "identified, containing sequences that could serve as potential candidates for \n",
      "peptide-based immunotherapy. MHC-class II-based T cell epitope prediction \n",
      "algorithms for HLA-DR and -DQ loci accurately predicted Ara h 2 T cell epitopes \n",
      "in peanut-allergic subjects, suggesting their potential utility in a \n",
      "peptide-based vaccine design for food allergy.\n",
      "\n",
      "© 2012 Blackwell Publishing Ltd.\n",
      "\n",
      "DOI: 10.1111/cea.12014\n",
      "PMID: 23278886 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "9. J Allergy Clin Immunol. 2011 Mar;127(3):608-15.e1-5. doi: \n",
      "10.1016/j.jaci.2010.09.027. Epub 2010 Nov 18.\n",
      "\n",
      "Ara h 2 peptides containing dominant CD4+ T-cell epitopes: candidates for a \n",
      "peanut allergy therapeutic.\n",
      "\n",
      "Prickett SR(1), Voskamp AL, Dacumos-Hill A, Symons K, Rolland JM, O'Hehir RE.\n",
      "\n",
      "Author information:\n",
      "(1)Department of Immunology, Monash University, Melbourne, Australia.\n",
      "\n",
      "BACKGROUND: Peanut allergy is a life-threatening condition; there is currently \n",
      "no cure. Although whole allergen extracts are used for specific immunotherapy \n",
      "for many allergies, they can cause severe reactions, and even fatalities, in \n",
      "peanut allergy.\n",
      "OBJECTIVE: This study aimed to identify short, T-cell epitope-based peptides \n",
      "that target allergen-specific CD4(+) T cells but do not bind IgE as candidates \n",
      "for safe peanut-specific immunotherapy.\n",
      "METHODS: Multiple CD4(+) T-cell lines specific for the major peanut allergen Ara \n",
      "h 2 were generated from PBMCs of 16 HLA-diverse subjects with peanut allergy by \n",
      "using 5,6-carboxyfluorescein diacetate succinimidylester-based methodology. \n",
      "Proliferation and ELISPOT assays were used to identify dominant epitopes \n",
      "recognized by T-cell lines and to confirm recognition by peripheral blood T \n",
      "cells of epitope-based peptides modified for therapeutic production. HLA \n",
      "restriction of core epitope recognition was investigated by using anti-HLA \n",
      "blocking antibodies and HLA genotyping. Serum-IgE peptide-binding was assessed \n",
      "by dot-blot.\n",
      "RESULTS: Five dominant CD4(+) T-cell epitopes were identified in Ara h 2. In \n",
      "combination, these were presented by HLA-DR, HLA-DP, and HLA-DQ molecules and \n",
      "recognized by T cells from all 16 subjects. Three short peptide variants \n",
      "containing these T-cell epitopes were designed with cysteine-to-serine \n",
      "substitutions to facilitate stability and therapeutic production. Variant \n",
      "peptides showed HLA-binding degeneracy, did not bind peanut-specific serum IgE, \n",
      "and could directly target T(H)2-type T cells in peripheral blood of subjects \n",
      "with allergy.\n",
      "CONCLUSION: Short CD4(+) T-cell epitope-based Ara h 2 peptides were identified \n",
      "as novel candidates for a T-cell-targeted peanut-specific immunotherapy for an \n",
      "HLA-diverse population.\n",
      "\n",
      "Copyright © 2010 American Academy of Allergy, Asthma & Immunology. Published by \n",
      "Mosby, Inc. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.jaci.2010.09.027\n",
      "PMID: 21093025 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "10. Ann Allergy Asthma Immunol. 2006 Jun;96(6):865-9. doi: \n",
      "10.1016/S1081-1206(10)61351-8.\n",
      "\n",
      "Lack of association of HLA class II alleles with peanut allergy.\n",
      "\n",
      "Shreffler WG(1), Charlop-Powers Z, Sicherer SH.\n",
      "\n",
      "Author information:\n",
      "(1)The Elliot and Roslyn Jaffe Food Allergy Institute, Division of Allergy and \n",
      "Immunology, Department of Pediatrics, Mount Sinai School of Medicine, New York, \n",
      "New York, USA.\n",
      "\n",
      "Comment in\n",
      "    Ann Allergy Asthma Immunol. 2006 Jun;96(6):766-8.\n",
      "\n",
      "BACKGROUND: Peanut allergy is a common and severe phenotype of food allergy with \n",
      "a strong genetic component; HLA class II polymorphisms are attractive candidate \n",
      "genes for this disorder.\n",
      "OBJECTIVE: To determine possible genotypic associations of HLA class II with \n",
      "peanut allergy and attempt replication of previously reported associations.\n",
      "METHODS: Sibling pairs discordant for peanut allergy were genotyped (low \n",
      "resolution) by polymerase chain reaction-based methods to 7 DQ and 18 DR allele \n",
      "groups. A chi2 analysis was undertaken against sibling controls with statistical \n",
      "adjustment for multiple analyses.\n",
      "RESULTS: Seventy-three children with confirmed peanut allergy (mean age, 6.5 \n",
      "years; male, 72%; asthma, 58%; atopic dermatitis, 62%; allergic rhinitis, 67%; \n",
      "other food allergies, 41%) and 75 of their siblings who eat peanut (mean age, 8 \n",
      "years; male, 52%; asthma, 12%; atopic dermatitis, 22%; allergic rhinitis, 37%; \n",
      "other food allergy, 7%) were genotyped. Distribution of DQ7 (29% of children \n",
      "with peanut allergy vs 47% sibling controls) was statistically significantly \n",
      "different (P = .04) before statistical correction for multiple comparisons was \n",
      "made by multiplying them by the number of alleles tested (and not statistically \n",
      "significant after correction; P = .30). Distribution of DR11 was nearly \n",
      "statistically significant without statistical adjustment (26% with peanut \n",
      "allergy vs 41% of sibling controls; P = .07; corrected P = 1.3). Alleles that \n",
      "were previously reported to have a weak association with peanut allergy (DRB1 \n",
      "*03, *08; DQB1 *0302, *04) were not verified in this cohort (unadjusted P > \n",
      ".44).\n",
      "CONCLUSIONS: We could not establish an association between the HLA class II \n",
      "alleles evaluated in this cohort of sibling pairs discordant for peanut allergy.\n",
      "\n",
      "DOI: 10.1016/S1081-1206(10)61351-8\n",
      "PMID: 16802776 [Indexed for MEDLINE]\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1. J Clin Invest. 2022 Jan 4;132(1):e155609. doi: 10.1172/JCI155609.\\n\\nEarly peanut introduction wins over the HLA-DQA1*01:02 allele in the interplay \\nbetween environment and genetics.\\n\\nManohar M, Nadeau KC, Kasowski M.\\n\\nComment on\\n    J Clin Invest. 2022 Jan 4;132(1):\\n\\nThe rising incidence of food allergy in children underscores the importance of \\nenvironmental exposures; however, genetic factors play a major role. How the \\nenvironment and genetics interact to cause food allergy remains unclear. The \\nlandmark Learning Early About Peanut Allergy (LEAP) clinical trial established \\nthat early peanut introduction protects high-risk infants, consistent with the \\ntolerizing effects of gut exposure. In this issue of the JCI, Kanchan et al. \\nleveraged the LEAP trial data to examine molecular genetic mechanisms of early \\nsensitization. A previously identified HLA risk allele for peanut allergy \\n(DQA1*01:02) was associated with peanut-specific IgG4 levels in consumers. \\nNotably, IgG4 antibodies likely provide protection by reducing the binding of \\nallergen to IgE. The association of the same allele with peanut allergy in \\navoiders while potentially conferring protection in consumers reinforces the \\nneed to integrate genetic information toward a personalized therapeutic strategy \\nfor the best outcome in addressing food allergies.\\n\\nDOI: 10.1172/JCI155609\\nPMCID: PMC8718134\\nPMID: 34981779 [Indexed for MEDLINE]\\n\\n\\n2. J Clin Invest. 2022 Jan 4;132(1):e152070. doi: 10.1172/JCI152070.\\n\\nHLA alleles and sustained peanut consumption promote IgG4 responses in subjects \\nprotected from peanut allergy.\\n\\nKanchan K(1), Grinek S(2), Bahnson HT(3)(4), Ruczinski I(5), Shankar G(1), \\nLarson D(4), Du Toit G(6), Barnes KC(7), Sampson HA(8), Suarez-Farinas M(2)(9), \\nLack G(6), Nepom GT(3)(4), Cerosaletti K(3), Mathias RA(1).\\n\\nAuthor information:\\n(1)Division of Allergy and Clinical Immunology, Department of Medicine, School \\nof Medicine, Johns Hopkins University, Baltimore, Maryland, USA.\\n(2)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \\nMount Sinai, New York, New York, USA.\\n(3)Benaroya Research Institute at Virginia Mason, Seattle, Washington, USA.\\n(4)The Immune Tolerance Network, Bethesda, Maryland, USA.\\n(5)Department of Biostatistics, Bloomberg School of Public Health, Johns Hopkins \\nUniversity, Baltimore, Maryland, USA.\\n(6)The Department of Pediatric Allergy, Division of Asthma, Allergy and Lung \\nBiology, King's College London, and Guy's and St. Thomas' NHS Foundation Trust, \\nLondon, United Kingdom.\\n(7)The Department of Medicine, University of Colorado, Anschutz, Colorado, USA.\\n(8)Department of Pediatrics, Allergy and Immunology and.\\n(9)Department of Population Health Science and Policy, Icahn School of Medicine \\nat Mount Sinai, New York, New York, USA.\\n\\nComment in\\n    J Clin Invest. 2022 Jan 4;132(1):\\n\\nWe investigated the interplay between genetics and oral peanut protein exposure \\nin the determination of the immunological response to peanut using the targeted \\nintervention in the LEAP clinical trial. We identified an association between \\npeanut-specific IgG4 and HLA-DQA1*01:02 that was only observed in the presence \\nof sustained oral peanut protein exposure. The association between IgG4 and \\nHLA-DQA1*01:02 was driven by IgG4 specific for the Ara h 2 component. Once \\npeanut consumption ceased, the association between IgG4-specific Ara h 2 and \\nHLA-DQA1*01:02 was attenuated. The association was validated by observing \\nexpanded IgG4-specific epitopes in people who carried HLA-DQA1*01:02. Notably, \\nwe confirmed the previously reported associations with HLA-DQA1*01:02 and peanut \\nallergy risk in the absence of oral peanut protein exposure. Interaction between \\nHLA and presence or absence of exposure to peanut in an allergen- and \\nepitope-specific manner implicates a mechanism of antigen recognition that is \\nfundamental to driving immune responses related to allergy risk or protection.\\n\\nDOI: 10.1172/JCI152070\\nPMCID: PMC8718139\\nPMID: 34981778 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Conflict of interest: GL reports holding stock \\nand stock options in DBV Technologies. HTB has provided statistical consulting \\nto DBV Technologies.\\n\\n\\n3. Ann Allergy Asthma Immunol. 2015 Dec;115(6):471-6. doi: \\n10.1016/j.anai.2015.10.008. Epub 2015 Oct 27.\\n\\nThe evolving story of human leukocyte antigen and the immunogenetics of peanut \\nallergy.\\n\\nHemler JA(1), Phillips EJ(2), Mallal SA(3), Kendall PL(4).\\n\\nAuthor information:\\n(1)Division of Allergy, Pulmonary and Critical Care, Department of Medicine, \\nVanderbilt University School of Medicine, Nashville, Tennessee.\\n(2)Division of Infectious Diseases, Department of Medicine, Vanderbilt \\nUniversity School of Medicine, Nashville, Tennessee; Department of Pathology, \\nMicrobiology and Immunology, Vanderbilt University School of Medicine, \\nNashville, Tennessee; Department of Pharmacology, Oates Institute for \\nExperimental Therapeutics, Vanderbilt University School of Medicine, Nashville, \\nTennessee; Institute for Immunology and Infectious Diseases, Murdoch University, \\nMurdoch, Western Australia, Australia.\\n(3)Division of Infectious Diseases, Department of Medicine, Vanderbilt \\nUniversity School of Medicine, Nashville, Tennessee; Center for Translational \\nImmunology and Infectious Diseases, Vanderbilt University School of Medicine, \\nNashville, Tennessee; Department of Pathology, Microbiology and Immunology, \\nVanderbilt University School of Medicine, Nashville, Tennessee; Institute for \\nImmunology and Infectious Diseases, Murdoch University, Murdoch, Western \\nAustralia, Australia.\\n(4)Division of Allergy, Pulmonary and Critical Care, Department of Medicine, \\nVanderbilt University School of Medicine, Nashville, Tennessee; Department of \\nPathology, Microbiology and Immunology, Vanderbilt University School of \\nMedicine, Nashville, Tennessee. Electronic address: \\npeggy.kendall@vanderbilt.edu.\\n\\nOBJECTIVE: Peanut allergy (PA) clearly has a heritable component. Specific \\ngenetic contributions are unknown, but human leukocyte antigen (HLA) loci are \\nobvious candidates. This review focuses on emerging studies of HLA associations \\nwith PA.\\nDATA SOURCES: PubMed was searched with no time limitations using key terms human \\nleukocyte antigen, HLA, MHC, peanut, peanut hypersensitivity, and peanut \\nallergy.\\nSTUDY SELECTIONS: Qualifying studies were English-language reports of genetic \\nanalyses examining PA and HLA associations.\\nRESULTS: Seven relevant citations were identified, which were published from \\n1996 to 2015. Early studies using candidate gene approaches found associations \\nbetween PA and HLA-DR and -DQ alleles (HLA-DRB1*08 and DQB1*06:03P) when \\ncomparing subjects with peanut allergy with nonallergic unrelated control \\ngroups. No significant associations were found between siblings with and without \\npeanut allergy. However, a recent large genomewide association study of patients \\nwith peanut allergy and their family members found 2 PA-associated \\nsingle-nucleotide polymorphisms (rs9275596 and rs7192) mapping to regions \\ninvolving the HLA-DR and HLA-DQ genes. Associations with differential DNA \\nmethylation partly mediated the associations between PA and single-nucleotide \\npolymorphisms.\\nCONCLUSION: Early studies using candidate gene approaches identified HLA \\nassociations with PA compared with the general population, suggesting a link \\nwith atopy but failing to identify a PA-specific association. These studies had \\nvarious limitations that included small samples. The most compelling evidence \\nfor a PA-specific HLA association comes from a genomewide association study, \\nwhich examined the entire genome in large, well-defined, related cohorts. More \\nresearch is needed to validate and replicate these findings, to perform fine \\ngenetic mapping of specific HLA loci, and to demonstrate underlying mechanisms \\nof HLA contributions to PA.\\n\\nCopyright © 2015 American College of Allergy, Asthma & Immunology. Published by \\nElsevier Inc. All rights reserved.\\n\\nDOI: 10.1016/j.anai.2015.10.008\\nPMCID: PMC5665015\\nPMID: 26522257 [Indexed for MEDLINE]\\n\\n\\n4. Curr Opin Allergy Clin Immunol. 2015 Oct;15(5):409-16. doi: \\n10.1097/ACI.0000000000000196.\\n\\nEpidemiology of food allergy and food-induced anaphylaxis: is there really a \\nWestern world epidemic?\\n\\nKoplin JJ(1), Mills EN, Allen KJ.\\n\\nAuthor information:\\n(1)aMurdoch Childrens Research Institute, University of Melbourne Department of \\nPaediatrics and Royal Children's Hospital bSchool of Population and Global \\nHealth, University of Melbourne, Parkville, VIC, Australia cUniversity of \\nManchester, Institute of Inflammation and Repair and Manchester Institute of \\nBiotechnology, Manchester, UK.\\n\\nPURPOSE OF REVIEW: Food-induced anaphylaxis continues to be an important cause \\nof hospital admissions, particularly in children. This review outlines recent \\nadvances in understanding the epidemiology of IgE-mediated food allergy and \\npotential mechanisms for its rise in prevalence.\\nRECENT FINDINGS: The rise in food allergy prevalence in Western countries has \\nhappened more quickly than changes to the genome can occur; thus, environmental \\nchanges are likely to be important. Recent studies, however, suggest that \\ngenetic risk determines responses to environmental risk factors. Environmental \\npeanut exposure has been associated with increased peanut allergy risk in \\nindividuals with filaggrin null mutations, consistently with sensitization \\noccurring through a damaged skin barrier. Reduced microbial and vitamin D \\nexposure is also leading candidates for risk factors for food allergy in the \\ncontext of genetic susceptibility. In addition, HLA-DR and HLA-DQ gene region \\nvariants appear to play a role in peanut allergy, although no studies have yet \\nassessed their susceptibility to environmental cues. Finally, findings from \\nobservational cohorts and the first large-scale intervention trials for food \\nallergy prevention support early oral allergen exposure to reduce the prevalence \\nof specific food allergies, which is informing changes in public health \\nguidelines at the population level. Further research will be required to assess \\nthe impact of these guideline changes on the population prevalence of food \\nallergy.\\nSUMMARY: New studies are providing important insights into the prevalence, \\ncauses, and mechanisms of food allergy. Recent findings are informing changes to \\npopulation health guidelines in developed countries, which have the potential to \\nhalt or reverse the increase in food allergy prevalence. By contrast, food \\nallergy in the developing world remains understudied.\\n\\nDOI: 10.1097/ACI.0000000000000196\\nPMID: 26258921 [Indexed for MEDLINE]\\n\\n\\n5. Nat Commun. 2015 Feb 24;6:6304. doi: 10.1038/ncomms7304.\\n\\nGenome-wide association study identifies peanut allergy-specific loci and \\nevidence of epigenetic mediation in US children.\\n\\nHong X(1), Hao K(2), Ladd-Acosta C(3), Hansen KD(4), Tsai HJ(5), Liu X(6), Xu \\nX(7), Thornton TA(8), Caruso D(1), Keet CA(9), Sun Y(10), Wang G(1), Luo W(11), \\nKumar R(12), Fuleihan R(12), Singh AM(13), Kim JS(14), Story RE(15), Gupta \\nRS(16), Gao P(17), Chen Z(1), Walker SO(1), Bartell TR(16), Beaty TH(3), Fallin \\nMD(18), Schleimer R(19), Holt PG(20), Nadeau KC(21), Wood RA(22), Pongracic \\nJA(12), Weeks DE(23), Wang X(24).\\n\\nAuthor information:\\n(1)Department of Population, Family and Reproductive Health, Center on the Early \\nLife Origins of Disease, Johns Hopkins University Bloomberg School of Public \\nHealth, 615 North Wolfe Street, E4132, Baltimore, Maryland 21205, USA.\\n(2)Department of Genetics and Genomic Sciences, Icahn School of Medicine at \\nMount Sinai, New York, New York 10029, USA.\\n(3)Department of Epidemiology, Johns Hopkins University Bloomberg School of \\nPublic Health, Baltimore, Maryland 21205, USA.\\n(4)1] Department of Biostatistics, Johns Hopkins University Bloomberg School of \\nPublic Health Baltimore, Baltimore, Maryland 21205, USA [2] McKusick-Nathans \\nInsitute of Genetic Medicine, Johns Hopkins University School of Medicine, \\nBaltimore, Maryland 21205, USA.\\n(5)1] Mary Ann and J. Milburn Smith Child Health Research Program, Department of \\nPediatrics, Northwestern University Feinberg School of Medicine and Stanley \\nManne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \\nof Chicago, Chicago, Illinois 60611, USA [2] Division of Biostatistics and \\nBioinformatics, Institute of Population Health Sciences, National Health \\nResearch Institutes, Zhunan 35053, Taiwan [3] Department of Public Health, China \\nMedical University, Taichung 40402, Taiwan.\\n(6)1] Mary Ann and J. Milburn Smith Child Health Research Program, Department of \\nPediatrics, Northwestern University Feinberg School of Medicine and Stanley \\nManne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \\nof Chicago, Chicago, Illinois 60611, USA [2] Department of Preventive Medicine, \\nNorthwestern University Feinberg School of Medicine, Chicago, Illinois 60611, \\nUSA.\\n(7)Guangdong Provincial Institute of Nephrology, Southern Medical University, \\nGuangzhou 510515, China.\\n(8)Department of Biostatistics, University of Washington, Seattle, Washington \\n98195, USA.\\n(9)1] Department of Epidemiology, Johns Hopkins University Bloomberg School of \\nPublic Health, Baltimore, Maryland 21205, USA [2] Division of Pediatric Allergy \\nand Immunology, Johns Hopkins University School of Medicine, Baltimore, Maryland \\n21287, USA.\\n(10)Department of Biostatistics, Johns Hopkins University Bloomberg School of \\nPublic Health Baltimore, Baltimore, Maryland 21205, USA.\\n(11)1] Department of Genetics and Genomic Sciences, Icahn School of Medicine at \\nMount Sinai, New York, New York 10029, USA [2] College of Computer Science and \\nTechnology, Huaqiao University, Xiamen 361021, China.\\n(12)Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \\nHospital of Chicago, Chicago, Illinois 60611, USA.\\n(13)Department of Pediatrics and Medicine, Ann and Robert H. Lurie Children's \\nHospital of Chicago, Northwestern Feinberg School of Medicine, Chicago, Illinois \\n61611, USA.\\n(14)1] Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \\nHospital of Chicago, Chicago, Illinois 60611, USA [2] NorthShore University \\nHealthSystem, Evanston, Illinois 60201, USA.\\n(15)1] Division of Allergy and Immunology, Ann and Robert H. Lurie Children's \\nHospital of Chicago, Chicago, Illinois 60611, USA [2] NorthShore University \\nHealth Systems, Pritzker School of Medicine, University of Chicago, Chicago, \\nIllinois 60637, USA.\\n(16)Mary Ann and J. Milburn Smith Child Health Research Program, Department of \\nPediatrics, Northwestern University Feinberg School of Medicine and Stanley \\nManne Children's Research Institute, Ann and Robert H. Lurie Children's Hospital \\nof Chicago, Chicago, Illinois 60611, USA.\\n(17)Division of Allergy and Clinical Immunology, Johns Hopkins University School \\nof Medicine, Baltimore, Maryland 21224, USA.\\n(18)Department of Mental Health, Wendy Klag Center for Autism and Developmental \\nDisabilities, Bloomberg School of Public Health, Johns Hopkins University, \\nBaltimore, Maryland 21205, USA.\\n(19)Division of Allergy-Immunology, Northwestern University Feinberg School of \\nMedicine, Chicago, Illinois 60611, USA.\\n(20)Telethon Kids Institute, University of Western Australia; Perth and \\nQueensland Children's Medical Research Institute, University of Queensland, \\nBrisbane, Queensland 4029, Australia.\\n(21)Division of Allergy, Immunology and Rheumatology, Stanford University School \\nof Medicine, Stanford, California 94305, USA.\\n(22)Division of Pediatric Allergy and Immunology, Johns Hopkins University \\nSchool of Medicine, Baltimore, Maryland 21287, USA.\\n(23)Departments of Human Genetics and Biostatistics, Graduate School of Public \\nHealth, University of Pittsburgh, Pittsburgh, Pennsylvania 15261, USA.\\n(24)1] Department of Population, Family and Reproductive Health, Center on the \\nEarly Life Origins of Disease, Johns Hopkins University Bloomberg School of \\nPublic Health, 615 North Wolfe Street, E4132, Baltimore, Maryland 21205, USA [2] \\nDivision of General Pediatrics and Adolescent Medicine, Department of \\nPediatrics, Johns Hopkins University School of Medicine, Baltimore, Maryland \\n21287, USA.\\n\\nFood allergy (FA) affects 2%-10% of US children and is a growing clinical and \\npublic health problem. Here we conduct the first genome-wide association study \\nof well-defined FA, including specific subtypes (peanut, milk and egg) in 2,759 \\nUS participants (1,315 children and 1,444 parents) from the Chicago Food Allergy \\nStudy, and identify peanut allergy (PA)-specific loci in the HLA-DR and -DQ gene \\nregion at 6p21.32, tagged by rs7192 (P=5.5 × 10(-8)) and rs9275596 (P=6.8 × \\n10(-10)), in 2,197 participants of European ancestry. We replicate these \\nassociations in an independent sample of European ancestry. These associations \\nare further supported by meta-analyses across the discovery and replication \\nsamples. Both single-nucleotide polymorphisms (SNPs) are associated with \\ndifferential DNA methylation levels at multiple CpG sites (P<5 × 10(-8)), and \\ndifferential DNA methylation of the HLA-DQB1 and HLA-DRB1 genes partially \\nmediate the identified SNP-PA associations. This study suggests that the HLA-DR \\nand -DQ gene region probably poses significant genetic risk for PA.\\n\\nDOI: 10.1038/ncomms7304\\nPMCID: PMC4340086\\nPMID: 25710614 [Indexed for MEDLINE]\\n\\nConflict of interest statement: CONFLICTS OF INTERESTS The authors have declared \\nthat no conflicting interests exist.\\n\\n\\n6. Clin Exp Allergy. 2013 Jun;43(6):684-97. doi: 10.1111/cea.12113.\\n\\nAra h 1 CD4+ T cell epitope-based peptides: candidates for a peanut allergy \\ntherapeutic.\\n\\nPrickett SR(1), Voskamp AL, Phan T, Dacumos-Hill A, Mannering SI, Rolland JM, \\nO'Hehir RE.\\n\\nAuthor information:\\n(1)Department of Immunology, Monash University, Melbourne, Vic., Australia.\\n\\nBACKGROUND: Peanut allergy is a life-threatening condition; there is currently \\nno cure. While whole allergen extracts are used for specific immunotherapy for \\nmany allergies, they can cause severe reactions and even fatalities in peanut \\nallergy.\\nOBJECTIVE: To identify short, HLA-degenerate CD4(+) T cell epitope-based \\npeptides of the major peanut allergen Ara h 1 that target allergen-specific T \\ncells without causing IgE-mediated inflammatory cell activation, as candidates \\nfor safe peanut-specific immunotherapy.\\nMETHODS: Ara h 1-specific CD4(+) T cell lines (TCL) were generated from \\nperipheral blood mononuclear cells (PBMC) of peanut-allergic subjects using \\nCFSE-based methodology. T cell epitopes were identified using CFSE and \\nthymidine-based proliferation assays. Epitope HLA-restriction was investigated \\nusing blocking antibodies, HLA-genotyping and epitope prediction algorithms. \\nFunctional peanut-specific IgE reactivity to peptides was assessed by basophil \\nactivation assay.\\nRESULTS: A total of 145 Ara h 1-specific TCL were generated from 18 HLA-diverse \\npeanut-allergic subjects. The TCL recognized 20-mer peptides throughout Ara h 1. \\nNine 20-mers containing the most frequently recognized epitopes were selected \\nand their recognition confirmed in 18 additional peanut-allergic subjects. Ten \\ncore epitopes were mapped within these 20-mers. These were HLA-DQ and/or HLA-DR \\nrestricted, with each presented on at least two different HLA-molecules. Seven \\nshort (≤\\xa020 aa) non-basophil-reactive peptides encompassing all core epitopes \\nwere designed and validated in peanut-allergic donor PBMC T cell assays.\\nCONCLUSIONS AND CLINICAL RELEVANCE: Short CD4(+) T cell epitope-based Ara h 1 \\npeptides were identified as novel candidates for a safe, T cell targeted \\npeanut-specific immunotherapy for HLA-diverse populations.\\n\\n© 2013 John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/cea.12113\\nPMCID: PMC3709139\\nPMID: 23711131 [Indexed for MEDLINE]\\n\\n\\n7. Eur J Hum Genet. 2013 Oct;21(10):1181-4. doi: 10.1038/ejhg.2013.13. Epub 2013 \\nFeb 27.\\n\\nHLA-DQB1*02 and DQB1*06:03P are associated with peanut allergy.\\n\\nMadore AM(1), Vaillancourt VT, Asai Y, Alizadehfar R, Ben-Shoshan M, Michel DL, \\nKozyrskyj AL, Becker A, Chan-Yeung M, Clarke AE, Hull P, Daley D, Sandford AJ, \\nLaprise C.\\n\\nAuthor information:\\n(1)Département des Sciences Fondamentales, Université du Québec à Chicoutimi, \\nSaguenay, Québec, Canada.\\n\\nPeanut allergy (PA) is a common and serious food allergy and its prevalence has \\nincreased in the past decade. Although there is strong evidence of inheritance, \\nthe genetic causes of this disease are not well understood. Previously, a \\nlarge-scale genome-wide association study described an association between human \\nleukocyte antigen (HLA)-DQB1 and asthma; the aim of this study was to evaluate \\nthe association between HLA-DQB1 and PA. Genotypic and allelic profiles were \\nestablished for 311 Caucasian members of a well-described Canadian group of \\nchildren with PA and 226 Caucasian controls. Firth's logistic regression \\nanalyses showed associations between HLA-DQB1 alleles and PA for DQB1*02 (P=1.1 \\n× 10(-8), odds ratio (OR)=0.09 (CI=0.03-0.23)) and DQB1*06:03P alleles (P=2.1 × \\n10(-2), OR=2.82 (CI=1.48-5.45)). This study of HLA in PA demonstrates specific \\nassociation between two allelic groups of the HLA-DQB1 gene (DQB1*02 and \\nDQB1*06:03P) and PA, highlighting its possible role in the development of this \\ndisease.\\n\\nDOI: 10.1038/ejhg.2013.13\\nPMCID: PMC3778350\\nPMID: 23443026 [Indexed for MEDLINE]\\n\\n\\n8. Clin Exp Allergy. 2013 Jan;43(1):116-27. doi: 10.1111/cea.12014.\\n\\nIn silico prediction of Ara h 2 T cell epitopes in peanut-allergic children.\\n\\nPascal M(1), Konstantinou GN, Masilamani M, Lieberman J, Sampson HA.\\n\\nAuthor information:\\n(1)Division of Allergy & Immunology and The Jaffe Food Allergy Research \\nInstitute, Mount Sinai School of Medicine, New York, NY, USA.\\n\\nBACKGROUND: Despite the frequency and severity of peanut allergy, the only \\napproved treatment is strict avoidance. Different types of immunotherapy with \\ncrude peanut extract are not universally effective and have been associated with \\nrelatively high adverse reaction rates.\\nOBJECTIVE: We sought to determine whether in silico predictive algorithms were \\nuseful in identifying candidate peptides for an Ara h 2 peptide-based vaccine \\nusing peanut-allergic patients' peripheral blood mononuclear cells (PBMCs) in \\nvitro. A human leucocyte antigen (HLA) distribution analysis was also performed.\\nMETHODS: Major histocompatibility complex (MHC)-class II-binding peptides were \\npredicted using NetMHCIIpan-2.0 and NetMHCII-2.2 algorithms. PBMCs from 80 \\npeanut-allergic patients were stimulated with overlapping 20-mer Ara h 2 \\npeptides. Cell supernatant cytokine profiles were evaluated by multiplex assays. \\nHLA-DRB1* and HLA-DQB1* typing were performed.\\nRESULTS: Four regions of overlapping sequences induced PBMC proliferation and \\npredominant Th2 cytokine production. HLA genotyping showed 30 different DRB1* \\nallele specificities and eight DQ serological specificities. The in silico \\nanalysis revealed similar relevant regions and predicted identical or similar \\ncore 9-mer epitopes to those identified in vitro. If relevant peptides, as \\ndetermined by either in vitro or in silico analysis (15 peptides and 9 core \\nepitopes respectively), were used in a peptide-based vaccine, they would cover \\nvirtually all subjects in the cohort studied.\\nCONCLUSIONS AND CLINICAL RELEVANCE: Four dominant regions in Ara h 2 have been \\nidentified, containing sequences that could serve as potential candidates for \\npeptide-based immunotherapy. MHC-class II-based T cell epitope prediction \\nalgorithms for HLA-DR and -DQ loci accurately predicted Ara h 2 T cell epitopes \\nin peanut-allergic subjects, suggesting their potential utility in a \\npeptide-based vaccine design for food allergy.\\n\\n© 2012 Blackwell Publishing Ltd.\\n\\nDOI: 10.1111/cea.12014\\nPMID: 23278886 [Indexed for MEDLINE]\\n\\n\\n9. J Allergy Clin Immunol. 2011 Mar;127(3):608-15.e1-5. doi: \\n10.1016/j.jaci.2010.09.027. Epub 2010 Nov 18.\\n\\nAra h 2 peptides containing dominant CD4+ T-cell epitopes: candidates for a \\npeanut allergy therapeutic.\\n\\nPrickett SR(1), Voskamp AL, Dacumos-Hill A, Symons K, Rolland JM, O'Hehir RE.\\n\\nAuthor information:\\n(1)Department of Immunology, Monash University, Melbourne, Australia.\\n\\nBACKGROUND: Peanut allergy is a life-threatening condition; there is currently \\nno cure. Although whole allergen extracts are used for specific immunotherapy \\nfor many allergies, they can cause severe reactions, and even fatalities, in \\npeanut allergy.\\nOBJECTIVE: This study aimed to identify short, T-cell epitope-based peptides \\nthat target allergen-specific CD4(+) T cells but do not bind IgE as candidates \\nfor safe peanut-specific immunotherapy.\\nMETHODS: Multiple CD4(+) T-cell lines specific for the major peanut allergen Ara \\nh 2 were generated from PBMCs of 16 HLA-diverse subjects with peanut allergy by \\nusing 5,6-carboxyfluorescein diacetate succinimidylester-based methodology. \\nProliferation and ELISPOT assays were used to identify dominant epitopes \\nrecognized by T-cell lines and to confirm recognition by peripheral blood T \\ncells of epitope-based peptides modified for therapeutic production. HLA \\nrestriction of core epitope recognition was investigated by using anti-HLA \\nblocking antibodies and HLA genotyping. Serum-IgE peptide-binding was assessed \\nby dot-blot.\\nRESULTS: Five dominant CD4(+) T-cell epitopes were identified in Ara h 2. In \\ncombination, these were presented by HLA-DR, HLA-DP, and HLA-DQ molecules and \\nrecognized by T cells from all 16 subjects. Three short peptide variants \\ncontaining these T-cell epitopes were designed with cysteine-to-serine \\nsubstitutions to facilitate stability and therapeutic production. Variant \\npeptides showed HLA-binding degeneracy, did not bind peanut-specific serum IgE, \\nand could directly target T(H)2-type T cells in peripheral blood of subjects \\nwith allergy.\\nCONCLUSION: Short CD4(+) T-cell epitope-based Ara h 2 peptides were identified \\nas novel candidates for a T-cell-targeted peanut-specific immunotherapy for an \\nHLA-diverse population.\\n\\nCopyright © 2010 American Academy of Allergy, Asthma & Immunology. Published by \\nMosby, Inc. All rights reserved.\\n\\nDOI: 10.1016/j.jaci.2010.09.027\\nPMID: 21093025 [Indexed for MEDLINE]\\n\\n\\n10. Ann Allergy Asthma Immunol. 2006 Jun;96(6):865-9. doi: \\n10.1016/S1081-1206(10)61351-8.\\n\\nLack of association of HLA class II alleles with peanut allergy.\\n\\nShreffler WG(1), Charlop-Powers Z, Sicherer SH.\\n\\nAuthor information:\\n(1)The Elliot and Roslyn Jaffe Food Allergy Institute, Division of Allergy and \\nImmunology, Department of Pediatrics, Mount Sinai School of Medicine, New York, \\nNew York, USA.\\n\\nComment in\\n    Ann Allergy Asthma Immunol. 2006 Jun;96(6):766-8.\\n\\nBACKGROUND: Peanut allergy is a common and severe phenotype of food allergy with \\na strong genetic component; HLA class II polymorphisms are attractive candidate \\ngenes for this disorder.\\nOBJECTIVE: To determine possible genotypic associations of HLA class II with \\npeanut allergy and attempt replication of previously reported associations.\\nMETHODS: Sibling pairs discordant for peanut allergy were genotyped (low \\nresolution) by polymerase chain reaction-based methods to 7 DQ and 18 DR allele \\ngroups. A chi2 analysis was undertaken against sibling controls with statistical \\nadjustment for multiple analyses.\\nRESULTS: Seventy-three children with confirmed peanut allergy (mean age, 6.5 \\nyears; male, 72%; asthma, 58%; atopic dermatitis, 62%; allergic rhinitis, 67%; \\nother food allergies, 41%) and 75 of their siblings who eat peanut (mean age, 8 \\nyears; male, 52%; asthma, 12%; atopic dermatitis, 22%; allergic rhinitis, 37%; \\nother food allergy, 7%) were genotyped. Distribution of DQ7 (29% of children \\nwith peanut allergy vs 47% sibling controls) was statistically significantly \\ndifferent (P = .04) before statistical correction for multiple comparisons was \\nmade by multiplying them by the number of alleles tested (and not statistically \\nsignificant after correction; P = .30). Distribution of DR11 was nearly \\nstatistically significant without statistical adjustment (26% with peanut \\nallergy vs 41% of sibling controls; P = .07; corrected P = 1.3). Alleles that \\nwere previously reported to have a weak association with peanut allergy (DRB1 \\n*03, *08; DQB1 *0302, *04) were not verified in this cohort (unadjusted P > \\n.44).\\nCONCLUSIONS: We could not establish an association between the HLA class II \\nalleles evaluated in this cohort of sibling pairs discordant for peanut allergy.\\n\\nDOI: 10.1016/S1081-1206(10)61351-8\\nPMID: 16802776 [Indexed for MEDLINE]\""
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email, 'HLA-DQ peanut allergy')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em gene. Irá ser processado 1 resultado.\n",
      "\n",
      "HLA-DQA2 major histocompatibility complex, class II, DQ alpha 2[Homo sapiens]\n",
      "Gene ID: 3118, updated on 27-Dec-2023\n",
      "\n",
      "\n",
      "Reference GRCh38.p14 Primary Assembly NC_000006.12  from: 32741391 to: 32747198\n",
      "mRNA  NM_020056.5, 5 exons,  total annotated spliced exon length: 1458\n",
      "protein precursor NP_064440.1 (CCDS4753.1), 4 coding  exons,  annotated AA length: 255\n",
      "\n",
      "Exon table for  mRNA  NM_020056.5 and protein NP_064440.1\n",
      "Genomic Interval Exon\t\tGenomic Interval Coding\t\tGene Interval Exon\t\tGene Interval Coding\t\tExon Length\tCoding Length\tIntron Length\n",
      "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
      "32741391-32741525\t\t32741444-32741525\t\t1-135\t\t54-135\t\t135\t\t82\t\t3633\n",
      "32745159-32745407\t\t32745159-32745407\t\t3769-4017\t\t3769-4017\t\t249\t\t249\t\t383\n",
      "32745791-32746072\t\t32745791-32746072\t\t4401-4682\t\t4401-4682\t\t282\t\t282\t\t167\n",
      "32746240-32746414\t\t32746240-32746394\t\t4850-5024\t\t4850-5004\t\t175\t\t155\t\t167\n",
      "32746582-32747198\t\t5192-5808\t\t617\n",
      "\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'HLA-DQA2 major histocompatibility complex, class II, DQ alpha 2[Homo sapiens]\\nGene ID: 3118, updated on 27-Dec-2023\\n\\n\\nReference GRCh38.p14 Primary Assembly NC_000006.12  from: 32741391 to: 32747198\\nmRNA  NM_020056.5, 5 exons,  total annotated spliced exon length: 1458\\nprotein precursor NP_064440.1 (CCDS4753.1), 4 coding  exons,  annotated AA length: 255\\n\\nExon table for  mRNA  NM_020056.5 and protein NP_064440.1\\nGenomic Interval Exon\\t\\tGenomic Interval Coding\\t\\tGene Interval Exon\\t\\tGene Interval Coding\\t\\tExon Length\\tCoding Length\\tIntron Length\\n------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\\n32741391-32741525\\t\\t32741444-32741525\\t\\t1-135\\t\\t54-135\\t\\t135\\t\\t82\\t\\t3633\\n32745159-32745407\\t\\t32745159-32745407\\t\\t3769-4017\\t\\t3769-4017\\t\\t249\\t\\t249\\t\\t383\\n32745791-32746072\\t\\t32745791-32746072\\t\\t4401-4682\\t\\t4401-4682\\t\\t282\\t\\t282\\t\\t167\\n32746240-32746414\\t\\t32746240-32746394\\t\\t4850-5024\\t\\t4850-5004\\t\\t175\\t\\t155\\t\\t167\\n32746582-32747198\\t\\t5192-5808\\t\\t617\\n'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'3118',db='gene',retmax=1,rettype='gene_table')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em nuccore. Irão ser processados 10 resultados.\n",
      "\n",
      "LOCUS       NM_020056               1458 bp    mRNA    linear   PRI 25-DEC-2022\n",
      "DEFINITION  Homo sapiens major histocompatibility complex, class II, DQ alpha 2\n",
      "            (HLA-DQA2), mRNA.\n",
      "ACCESSION   NM_020056 XM_005249050 XM_005272806 XM_005274854 XM_005275110\n",
      "            XM_005275385 XM_005275545\n",
      "VERSION     NM_020056.5\n",
      "KEYWORDS    RefSeq; MANE Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1458)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   2  (bases 1 to 1458)\n",
      "  AUTHORS   Liang SQ, Deng JM, Wei X, Chen ZR, Yang ML, Qin HJ, Zhang JQ and He\n",
      "            ZY.\n",
      "  TITLE     Association of GWAS-supported noncoding area loci rs404860,\n",
      "            rs3117098, and rs7775228 with asthma in Chinese Zhuang population\n",
      "  JOURNAL   J Clin Lab Anal 34 (2), e23066 (2020)\n",
      "   PUBMED   31605414\n",
      "  REMARK    GeneRIF: Association of GWAS-supported noncoding area loci\n",
      "            rs404860, rs3117098, and rs7775228 with asthma in Chinese Zhuang\n",
      "            population.\n",
      "REFERENCE   3  (bases 1 to 1458)\n",
      "  AUTHORS   Armstrong DL, Zidovetzki R, Alarcon-Riquelme ME, Tsao BP, Criswell\n",
      "            LA, Kimberly RP, Harley JB, Sivils KL, Vyse TJ, Gaffney PM,\n",
      "            Langefeld CD and Jacob CO.\n",
      "  TITLE     GWAS identifies novel SLE susceptibility genes and explains the\n",
      "            association of the HLA region\n",
      "  JOURNAL   Genes Immun 15 (6), 347-354 (2014)\n",
      "   PUBMED   24871463\n",
      "REFERENCE   4  (bases 1 to 1458)\n",
      "  AUTHORS   Chang SW, Fann CS, Su WH, Wang YC, Weng CC, Yu CJ, Hsu CL, Hsieh\n",
      "            AR, Chien RN, Chu CM and Tai DI.\n",
      "  TITLE     A genome-wide association study on chronic HBV infection and its\n",
      "            clinical progression in male Han-Taiwanese\n",
      "  JOURNAL   PLoS One 9 (6), e99724 (2014)\n",
      "   PUBMED   24940741\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   5  (bases 1 to 1458)\n",
      "  AUTHORS   Jiang L, Yin J, Ye L, Yang J, Hemani G, Liu AJ, Zou H, He D, Sun L,\n",
      "            Zeng X, Li Z, Zheng Y, Lin Y, Liu Y, Fang Y, Xu J, Li Y, Dai S,\n",
      "            Guan J, Jiang L, Wei Q, Wang Y, Li Y, Huang C, Zuo X, Liu Y, Wu X,\n",
      "            Zhang L, Zhou L, Zhang Q, Li T, Chen L, Xu Z, Yang X, Qian F, Xie\n",
      "            W, Liu W, Guo Q, Huang S, Zhao J, Li M, Jin Y, Gao J, Lv Y, Wang Y,\n",
      "            Lin L, Guo A, Danoy P, Willner D, Cremin C, Hadler J, Zhang F, Zhao\n",
      "            Y, Li M, Yue T, Fan X, Guo J, Mu R, Li J, Wu C, Zeng M, Wang J, Li\n",
      "            S, Jin L, Wang B, Wang J, Ma X, Sun L, Zhang X, Brown MA, Visscher\n",
      "            PM, Su DF and Xu H.\n",
      "  TITLE     Novel risk loci for rheumatoid arthritis in Han Chinese and\n",
      "            congruence with risk variants in Europeans\n",
      "  JOURNAL   Arthritis Rheumatol 66 (5), 1121-1132 (2014)\n",
      "   PUBMED   24782177\n",
      "  REMARK    Erratum:[Arthritis Rheumatol. 2014 Jul;66(7):1881]\n",
      "REFERENCE   6  (bases 1 to 1458)\n",
      "  AUTHORS   Marsh SG and Bodmer JG.\n",
      "  TITLE     HLA class II nucleotide sequences, 1992\n",
      "  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\n",
      "   PUBMED   1362295\n",
      "  REMARK    Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\n",
      "            1529427]\n",
      "REFERENCE   7  (bases 1 to 1458)\n",
      "  AUTHORS   Wang Y, Lu P, Zhou K, Clay T, Wood N, Bradley B and Bidwell J.\n",
      "  TITLE     Nucleotide sequencing of HLA-DQ gene second exons in Chinese\n",
      "            homozygous cells\n",
      "  JOURNAL   Eur J Immunogenet 19 (5), 287-293 (1992)\n",
      "   PUBMED   1420115\n",
      "REFERENCE   8  (bases 1 to 1458)\n",
      "  AUTHORS   Kenter M, Otting N, Anholts J, Leunissen J, Jonker M and Bontrop\n",
      "            RE.\n",
      "  TITLE     Evolutionary relationships among the primate Mhc-DQA1 and DQA2\n",
      "            alleles\n",
      "  JOURNAL   Immunogenetics 36 (2), 71-78 (1992)\n",
      "   PUBMED   1612647\n",
      "REFERENCE   9  (bases 1 to 1458)\n",
      "  AUTHORS   Gyllensten UB and Erlich HA.\n",
      "  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\n",
      "            primates\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\n",
      "   PUBMED   2513578\n",
      "REFERENCE   10 (bases 1 to 1458)\n",
      "  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\n",
      "            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\n",
      "  TITLE     Class II genes of the human major histocompatibility complex.\n",
      "            Comparisons of the DQ and DX alpha and beta genes\n",
      "  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\n",
      "   PUBMED   3036828\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL672104.4.\n",
      "            \n",
      "            On May 17, 2019 this sequence version replaced NM_020056.4.\n",
      "            \n",
      "            Summary: This gene belongs to the HLA class II alpha chain family.\n",
      "            The encoded protein forms a heterodimer with a class II beta chain.\n",
      "            It is located in intracellular vesicles and plays a central role in\n",
      "            the peptide loading of MHC class II molecules by helping to release\n",
      "            the CLIP molecule from the peptide binding site. Class II molecules\n",
      "            are expressed in antigen presenting cells (B lymphocytes, dendritic\n",
      "            cells, macrophages) and are used to present antigenic peptides on\n",
      "            the cell surface to be recognized by CD4 T-cells. [provided by\n",
      "            RefSeq, Jun 2010].\n",
      "            \n",
      "            Sequence Note: The RefSeq transcript and protein were derived from\n",
      "            genomic sequence to make the sequence consistent with the reference\n",
      "            genome assembly. The genomic coordinates used for the transcript\n",
      "            record were based on alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            RNAseq introns :: single sample supports all introns SAMEA2144335,\n",
      "                              SAMEA2145893 [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            MANE Ensembl match     :: ENST00000374940.4/ ENSP00000364076.3\n",
      "            RefSeq Select criteria :: based on single protein-coding transcript\n",
      "            ##RefSeq-Attributes-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-135               AL672104.4         5734-5868\n",
      "            136-384             AL672104.4         9502-9750\n",
      "            385-666             AL672104.4         10134-10415\n",
      "            667-841             AL672104.4         10583-10757\n",
      "            842-1458            AL672104.4         10925-11541\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1458\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"6\"\n",
      "                     /map=\"6p21.32\"\n",
      "     gene            1..1458\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"major histocompatibility complex, class II, DQ\n",
      "                     alpha 2\"\n",
      "                     /db_xref=\"GeneID:3118\"\n",
      "                     /db_xref=\"HGNC:HGNC:4943\"\n",
      "                     /db_xref=\"MIM:613503\"\n",
      "     exon            1..135\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    33..35\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             54..821\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"HLA class II histocompatibility antigen, DQ alpha 2\n",
      "                     chain; MHC class II DQA2; DX alpha chain; HLA class II\n",
      "                     histocompatibility antigen, DQ(6) alpha chain; MHC class\n",
      "                     II antigen; MHC class II DQA1; HLA class II\n",
      "                     histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha\n",
      "                     chain; MHC class I antigen\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain precursor\"\n",
      "                     /protein_id=\"NP_064440.1\"\n",
      "                     /db_xref=\"CCDS:CCDS4753.1\"\n",
      "                     /db_xref=\"GeneID:3118\"\n",
      "                     /db_xref=\"HGNC:HGNC:4943\"\n",
      "                     /db_xref=\"MIM:613503\"\n",
      "                     /translation=\"MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGP\n",
      "                     SGQYTHEFDGDEEFYVDLETKETVWQLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQ\n",
      "                     SNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSE\n",
      "                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTET\n",
      "                     LVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\"\n",
      "     sig_peptide     54..122\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\n",
      "     mat_peptide     123..818\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain\"\n",
      "     misc_feature    123..383\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Alpha-1\"\n",
      "     misc_feature    363..365\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\n",
      "     misc_feature    384..665\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Alpha-2\"\n",
      "     misc_feature    483..485\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\n",
      "     misc_feature    666..704\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Connecting peptide\"\n",
      "     misc_feature    705..773\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     transmembrane region\"\n",
      "     exon            136..384\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            385..666\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            667..841\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            842..1458\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      1134..1139\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1158\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 acaattgctc tacagctcag agcagcaact gctgaggctg ccttgggaag aagatgatcc\n",
      "       61 taaacaaagc tctgctgctg ggggccctcg ccctgactgc cgtgatgagc ccctgtggag\n",
      "      121 gtgaagacat tgtggctgac catgttgcct cctatggtgt gaacttctac cagtctcacg\n",
      "      181 gtccctctgg ccagtacacc catgaatttg atggagacga ggagttctat gtggacctgg\n",
      "      241 agacgaaaga gactgtctgg cagttgccta tgtttagcaa atttataagt tttgacccgc\n",
      "      301 agagtgcact gagaaatatg gctgtgggaa aacacacctt ggaattcatg atgagacagt\n",
      "      361 ccaactctac cgctgccacc aatgaggttc ctgaggtcac agtgttttcc aagtttcctg\n",
      "      421 tgacgctggg tcagcccaac accctcatct gtcttgtgga caacatcttt cctcctgtgg\n",
      "      481 tcaacatcac ctggctgagc aatgggcact cagtcacaga aggtgtttct gagaccagct\n",
      "      541 tcctctccaa gagtgatcat tccttcttca agatcagtta cctcaccttc ctcccttctg\n",
      "      601 ctgatgagat ttatgactgc aaggtggagc actggggcct ggacgagcct cttctgaaac\n",
      "      661 actgggagcc tgagattcca gcccctatgt cagagctcac agagactttg gtctgcgccc\n",
      "      721 tggggttgtc tgtgggcctc atgggcattg tggtgggcac tgtcttcatc atccaaggcc\n",
      "      781 tgcgttcagt tggtgcttcc agacaccaag ggctcttatg aatcccatcc tgaaaaggaa\n",
      "      841 ggtgcatcac catctacagg agaagaagaa tggacttgct aaatgaccta gcactattct\n",
      "      901 ctggcctgat ttatcatatc ccttttctcc tccaaatgtt tcttctctca cctcttctct\n",
      "      961 gggacttaag gtgctatatt ccctcagagc tcacaaatgc ctttcaattc tttccctgac\n",
      "     1021 ctcctttcct gaattttttt attttctcaa atgttaccta ctaagggatg cctgggtaag\n",
      "     1081 ccactcagct acctaattcc tcaatgacct ttatctaaaa tctccatgga agcaataaat\n",
      "     1141 tcccttttga tgcctctatt gaatttttcc catctttcat ctcagggctg actgagagca\n",
      "     1201 taacttagaa tgggcgactc ttatgtttta ggccaatttc atatcattcc ccagatcata\n",
      "     1261 tttcaagtcc agtaacacag gagcaaccaa gtacagtgta tcctgataat ttgttgattt\n",
      "     1321 cttaactggt gttaatattt ctttcttcct tttgttccta cccttggcca ctgccagcca\n",
      "     1381 cccctcaatt caggtaccaa cgaaccctct gcccttggct cagaatggtt atagcagaaa\n",
      "     1441 tacaaaaaaa aaaaaaaa\n",
      "//\n",
      "\n",
      "\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'LOCUS       NM_020056               1458 bp    mRNA    linear   PRI 25-DEC-2022\\nDEFINITION  Homo sapiens major histocompatibility complex, class II, DQ alpha 2\\n            (HLA-DQA2), mRNA.\\nACCESSION   NM_020056 XM_005249050 XM_005272806 XM_005274854 XM_005275110\\n            XM_005275385 XM_005275545\\nVERSION     NM_020056.5\\nKEYWORDS    RefSeq; MANE Select.\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1  (bases 1 to 1458)\\n  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\\n            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\\n            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\\n            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\\n            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\\n            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\\n            Colabella C, Coppin G, D\\'Amata C, De Ridder D, De Rouck S,\\n            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\\n            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\\n            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\\n            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\\n            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\\n            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\\n            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\\n            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\\n            MA.\\n  TITLE     A reference map of the human binary protein interactome\\n  JOURNAL   Nature 580 (7803), 402-408 (2020)\\n   PUBMED   32296183\\nREFERENCE   2  (bases 1 to 1458)\\n  AUTHORS   Liang SQ, Deng JM, Wei X, Chen ZR, Yang ML, Qin HJ, Zhang JQ and He\\n            ZY.\\n  TITLE     Association of GWAS-supported noncoding area loci rs404860,\\n            rs3117098, and rs7775228 with asthma in Chinese Zhuang population\\n  JOURNAL   J Clin Lab Anal 34 (2), e23066 (2020)\\n   PUBMED   31605414\\n  REMARK    GeneRIF: Association of GWAS-supported noncoding area loci\\n            rs404860, rs3117098, and rs7775228 with asthma in Chinese Zhuang\\n            population.\\nREFERENCE   3  (bases 1 to 1458)\\n  AUTHORS   Armstrong DL, Zidovetzki R, Alarcon-Riquelme ME, Tsao BP, Criswell\\n            LA, Kimberly RP, Harley JB, Sivils KL, Vyse TJ, Gaffney PM,\\n            Langefeld CD and Jacob CO.\\n  TITLE     GWAS identifies novel SLE susceptibility genes and explains the\\n            association of the HLA region\\n  JOURNAL   Genes Immun 15 (6), 347-354 (2014)\\n   PUBMED   24871463\\nREFERENCE   4  (bases 1 to 1458)\\n  AUTHORS   Chang SW, Fann CS, Su WH, Wang YC, Weng CC, Yu CJ, Hsu CL, Hsieh\\n            AR, Chien RN, Chu CM and Tai DI.\\n  TITLE     A genome-wide association study on chronic HBV infection and its\\n            clinical progression in male Han-Taiwanese\\n  JOURNAL   PLoS One 9 (6), e99724 (2014)\\n   PUBMED   24940741\\n  REMARK    Publication Status: Online-Only\\nREFERENCE   5  (bases 1 to 1458)\\n  AUTHORS   Jiang L, Yin J, Ye L, Yang J, Hemani G, Liu AJ, Zou H, He D, Sun L,\\n            Zeng X, Li Z, Zheng Y, Lin Y, Liu Y, Fang Y, Xu J, Li Y, Dai S,\\n            Guan J, Jiang L, Wei Q, Wang Y, Li Y, Huang C, Zuo X, Liu Y, Wu X,\\n            Zhang L, Zhou L, Zhang Q, Li T, Chen L, Xu Z, Yang X, Qian F, Xie\\n            W, Liu W, Guo Q, Huang S, Zhao J, Li M, Jin Y, Gao J, Lv Y, Wang Y,\\n            Lin L, Guo A, Danoy P, Willner D, Cremin C, Hadler J, Zhang F, Zhao\\n            Y, Li M, Yue T, Fan X, Guo J, Mu R, Li J, Wu C, Zeng M, Wang J, Li\\n            S, Jin L, Wang B, Wang J, Ma X, Sun L, Zhang X, Brown MA, Visscher\\n            PM, Su DF and Xu H.\\n  TITLE     Novel risk loci for rheumatoid arthritis in Han Chinese and\\n            congruence with risk variants in Europeans\\n  JOURNAL   Arthritis Rheumatol 66 (5), 1121-1132 (2014)\\n   PUBMED   24782177\\n  REMARK    Erratum:[Arthritis Rheumatol. 2014 Jul;66(7):1881]\\nREFERENCE   6  (bases 1 to 1458)\\n  AUTHORS   Marsh SG and Bodmer JG.\\n  TITLE     HLA class II nucleotide sequences, 1992\\n  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\\n   PUBMED   1362295\\n  REMARK    Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\\n            1529427]\\nREFERENCE   7  (bases 1 to 1458)\\n  AUTHORS   Wang Y, Lu P, Zhou K, Clay T, Wood N, Bradley B and Bidwell J.\\n  TITLE     Nucleotide sequencing of HLA-DQ gene second exons in Chinese\\n            homozygous cells\\n  JOURNAL   Eur J Immunogenet 19 (5), 287-293 (1992)\\n   PUBMED   1420115\\nREFERENCE   8  (bases 1 to 1458)\\n  AUTHORS   Kenter M, Otting N, Anholts J, Leunissen J, Jonker M and Bontrop\\n            RE.\\n  TITLE     Evolutionary relationships among the primate Mhc-DQA1 and DQA2\\n            alleles\\n  JOURNAL   Immunogenetics 36 (2), 71-78 (1992)\\n   PUBMED   1612647\\nREFERENCE   9  (bases 1 to 1458)\\n  AUTHORS   Gyllensten UB and Erlich HA.\\n  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\\n            primates\\n  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\\n   PUBMED   2513578\\nREFERENCE   10 (bases 1 to 1458)\\n  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\\n            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\\n  TITLE     Class II genes of the human major histocompatibility complex.\\n            Comparisons of the DQ and DX alpha and beta genes\\n  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\\n   PUBMED   3036828\\nCOMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\\n            reference sequence was derived from AL672104.4.\\n            \\n            On May 17, 2019 this sequence version replaced NM_020056.4.\\n            \\n            Summary: This gene belongs to the HLA class II alpha chain family.\\n            The encoded protein forms a heterodimer with a class II beta chain.\\n            It is located in intracellular vesicles and plays a central role in\\n            the peptide loading of MHC class II molecules by helping to release\\n            the CLIP molecule from the peptide binding site. Class II molecules\\n            are expressed in antigen presenting cells (B lymphocytes, dendritic\\n            cells, macrophages) and are used to present antigenic peptides on\\n            the cell surface to be recognized by CD4 T-cells. [provided by\\n            RefSeq, Jun 2010].\\n            \\n            Sequence Note: The RefSeq transcript and protein were derived from\\n            genomic sequence to make the sequence consistent with the reference\\n            genome assembly. The genomic coordinates used for the transcript\\n            record were based on alignments.\\n            \\n            Publication Note:  This RefSeq record includes a subset of the\\n            publications that are available for this gene. Please see the Gene\\n            record to access additional publications.\\n            \\n            ##Evidence-Data-START##\\n            RNAseq introns :: single sample supports all introns SAMEA2144335,\\n                              SAMEA2145893 [ECO:0000348]\\n            ##Evidence-Data-END##\\n            \\n            ##RefSeq-Attributes-START##\\n            MANE Ensembl match     :: ENST00000374940.4/ ENSP00000364076.3\\n            RefSeq Select criteria :: based on single protein-coding transcript\\n            ##RefSeq-Attributes-END##\\n            COMPLETENESS: complete on the 3\\' end.\\nPRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\\n            1-135               AL672104.4         5734-5868\\n            136-384             AL672104.4         9502-9750\\n            385-666             AL672104.4         10134-10415\\n            667-841             AL672104.4         10583-10757\\n            842-1458            AL672104.4         10925-11541\\nFEATURES             Location/Qualifiers\\n     source          1..1458\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"mRNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /chromosome=\"6\"\\n                     /map=\"6p21.32\"\\n     gene            1..1458\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"major histocompatibility complex, class II, DQ\\n                     alpha 2\"\\n                     /db_xref=\"GeneID:3118\"\\n                     /db_xref=\"HGNC:HGNC:4943\"\\n                     /db_xref=\"MIM:613503\"\\n     exon            1..135\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     misc_feature    33..35\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"upstream in-frame stop codon\"\\n     CDS             54..821\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"HLA class II histocompatibility antigen, DQ alpha 2\\n                     chain; MHC class II DQA2; DX alpha chain; HLA class II\\n                     histocompatibility antigen, DQ(6) alpha chain; MHC class\\n                     II antigen; MHC class II DQA1; HLA class II\\n                     histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha\\n                     chain; MHC class I antigen\"\\n                     /codon_start=1\\n                     /product=\"HLA class II histocompatibility antigen, DQ\\n                     alpha 2 chain precursor\"\\n                     /protein_id=\"NP_064440.1\"\\n                     /db_xref=\"CCDS:CCDS4753.1\"\\n                     /db_xref=\"GeneID:3118\"\\n                     /db_xref=\"HGNC:HGNC:4943\"\\n                     /db_xref=\"MIM:613503\"\\n                     /translation=\"MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGP\\n                     SGQYTHEFDGDEEFYVDLETKETVWQLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQ\\n                     SNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSE\\n                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTET\\n                     LVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\"\\n     sig_peptide     54..122\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\\n     mat_peptide     123..818\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /product=\"HLA class II histocompatibility antigen, DQ\\n                     alpha 2 chain\"\\n     misc_feature    123..383\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Alpha-1\"\\n     misc_feature    363..365\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255; propagated from\\n                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\\n     misc_feature    384..665\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Alpha-2\"\\n     misc_feature    483..485\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255; propagated from\\n                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\\n     misc_feature    666..704\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Connecting peptide\"\\n     misc_feature    705..773\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     transmembrane region\"\\n     exon            136..384\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            385..666\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            667..841\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            842..1458\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     regulatory      1134..1139\\n                     /regulatory_class=\"polyA_signal_sequence\"\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"hexamer: AATAAA\"\\n     polyA_site      1158\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"major polyA site\"\\nORIGIN      \\n        1 acaattgctc tacagctcag agcagcaact gctgaggctg ccttgggaag aagatgatcc\\n       61 taaacaaagc tctgctgctg ggggccctcg ccctgactgc cgtgatgagc ccctgtggag\\n      121 gtgaagacat tgtggctgac catgttgcct cctatggtgt gaacttctac cagtctcacg\\n      181 gtccctctgg ccagtacacc catgaatttg atggagacga ggagttctat gtggacctgg\\n      241 agacgaaaga gactgtctgg cagttgccta tgtttagcaa atttataagt tttgacccgc\\n      301 agagtgcact gagaaatatg gctgtgggaa aacacacctt ggaattcatg atgagacagt\\n      361 ccaactctac cgctgccacc aatgaggttc ctgaggtcac agtgttttcc aagtttcctg\\n      421 tgacgctggg tcagcccaac accctcatct gtcttgtgga caacatcttt cctcctgtgg\\n      481 tcaacatcac ctggctgagc aatgggcact cagtcacaga aggtgtttct gagaccagct\\n      541 tcctctccaa gagtgatcat tccttcttca agatcagtta cctcaccttc ctcccttctg\\n      601 ctgatgagat ttatgactgc aaggtggagc actggggcct ggacgagcct cttctgaaac\\n      661 actgggagcc tgagattcca gcccctatgt cagagctcac agagactttg gtctgcgccc\\n      721 tggggttgtc tgtgggcctc atgggcattg tggtgggcac tgtcttcatc atccaaggcc\\n      781 tgcgttcagt tggtgcttcc agacaccaag ggctcttatg aatcccatcc tgaaaaggaa\\n      841 ggtgcatcac catctacagg agaagaagaa tggacttgct aaatgaccta gcactattct\\n      901 ctggcctgat ttatcatatc ccttttctcc tccaaatgtt tcttctctca cctcttctct\\n      961 gggacttaag gtgctatatt ccctcagagc tcacaaatgc ctttcaattc tttccctgac\\n     1021 ctcctttcct gaattttttt attttctcaa atgttaccta ctaagggatg cctgggtaag\\n     1081 ccactcagct acctaattcc tcaatgacct ttatctaaaa tctccatgga agcaataaat\\n     1141 tcccttttga tgcctctatt gaatttttcc catctttcat ctcagggctg actgagagca\\n     1201 taacttagaa tgggcgactc ttatgtttta ggccaatttc atatcattcc ccagatcata\\n     1261 tttcaagtcc agtaacacag gagcaaccaa gtacagtgta tcctgataat ttgttgattt\\n     1321 cttaactggt gttaatattt ctttcttcct tttgttccta cccttggcca ctgccagcca\\n     1381 cccctcaatt caggtaccaa cgaaccctct gcccttggct cagaatggtt atagcagaaa\\n     1441 tacaaaaaaa aaaaaaaa\\n//\\n\\n'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'NM_020056','nuccore')\n",
    "#pesquisa_ncbi(email,'NM_020056','nuccore',rettype='gb',save='y',display='n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em protein. Irão ser processados 10 resultados.\n",
      "\n",
      "LOCUS       NP_064440                255 aa            linear   PRI 25-DEC-2022\n",
      "DEFINITION  HLA class II histocompatibility antigen, DQ alpha 2 chain precursor\n",
      "            [Homo sapiens].\n",
      "ACCESSION   NP_064440 XP_005249107 XP_005272863 XP_005274911 XP_005275167\n",
      "            XP_005275442 XP_005275602\n",
      "VERSION     NP_064440.1\n",
      "DBSOURCE    REFSEQ: accession NM_020056.5\n",
      "KEYWORDS    RefSeq; MANE Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (residues 1 to 255)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   2  (residues 1 to 255)\n",
      "  AUTHORS   Liang SQ, Deng JM, Wei X, Chen ZR, Yang ML, Qin HJ, Zhang JQ and He\n",
      "            ZY.\n",
      "  TITLE     Association of GWAS-supported noncoding area loci rs404860,\n",
      "            rs3117098, and rs7775228 with asthma in Chinese Zhuang population\n",
      "  JOURNAL   J Clin Lab Anal 34 (2), e23066 (2020)\n",
      "   PUBMED   31605414\n",
      "  REMARK    GeneRIF: Association of GWAS-supported noncoding area loci\n",
      "            rs404860, rs3117098, and rs7775228 with asthma in Chinese Zhuang\n",
      "            population.\n",
      "REFERENCE   3  (residues 1 to 255)\n",
      "  AUTHORS   Armstrong DL, Zidovetzki R, Alarcon-Riquelme ME, Tsao BP, Criswell\n",
      "            LA, Kimberly RP, Harley JB, Sivils KL, Vyse TJ, Gaffney PM,\n",
      "            Langefeld CD and Jacob CO.\n",
      "  TITLE     GWAS identifies novel SLE susceptibility genes and explains the\n",
      "            association of the HLA region\n",
      "  JOURNAL   Genes Immun 15 (6), 347-354 (2014)\n",
      "   PUBMED   24871463\n",
      "REFERENCE   4  (residues 1 to 255)\n",
      "  AUTHORS   Chang SW, Fann CS, Su WH, Wang YC, Weng CC, Yu CJ, Hsu CL, Hsieh\n",
      "            AR, Chien RN, Chu CM and Tai DI.\n",
      "  TITLE     A genome-wide association study on chronic HBV infection and its\n",
      "            clinical progression in male Han-Taiwanese\n",
      "  JOURNAL   PLoS One 9 (6), e99724 (2014)\n",
      "   PUBMED   24940741\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   5  (residues 1 to 255)\n",
      "  AUTHORS   Jiang L, Yin J, Ye L, Yang J, Hemani G, Liu AJ, Zou H, He D, Sun L,\n",
      "            Zeng X, Li Z, Zheng Y, Lin Y, Liu Y, Fang Y, Xu J, Li Y, Dai S,\n",
      "            Guan J, Jiang L, Wei Q, Wang Y, Li Y, Huang C, Zuo X, Liu Y, Wu X,\n",
      "            Zhang L, Zhou L, Zhang Q, Li T, Chen L, Xu Z, Yang X, Qian F, Xie\n",
      "            W, Liu W, Guo Q, Huang S, Zhao J, Li M, Jin Y, Gao J, Lv Y, Wang Y,\n",
      "            Lin L, Guo A, Danoy P, Willner D, Cremin C, Hadler J, Zhang F, Zhao\n",
      "            Y, Li M, Yue T, Fan X, Guo J, Mu R, Li J, Wu C, Zeng M, Wang J, Li\n",
      "            S, Jin L, Wang B, Wang J, Ma X, Sun L, Zhang X, Brown MA, Visscher\n",
      "            PM, Su DF and Xu H.\n",
      "  TITLE     Novel risk loci for rheumatoid arthritis in Han Chinese and\n",
      "            congruence with risk variants in Europeans\n",
      "  JOURNAL   Arthritis Rheumatol 66 (5), 1121-1132 (2014)\n",
      "   PUBMED   24782177\n",
      "  REMARK    Erratum:[Arthritis Rheumatol. 2014 Jul;66(7):1881]\n",
      "REFERENCE   6  (residues 1 to 255)\n",
      "  AUTHORS   Marsh SG and Bodmer JG.\n",
      "  TITLE     HLA class II nucleotide sequences, 1992\n",
      "  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\n",
      "   PUBMED   1362295\n",
      "  REMARK    Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\n",
      "            1529427]\n",
      "REFERENCE   7  (residues 1 to 255)\n",
      "  AUTHORS   Wang Y, Lu P, Zhou K, Clay T, Wood N, Bradley B and Bidwell J.\n",
      "  TITLE     Nucleotide sequencing of HLA-DQ gene second exons in Chinese\n",
      "            homozygous cells\n",
      "  JOURNAL   Eur J Immunogenet 19 (5), 287-293 (1992)\n",
      "   PUBMED   1420115\n",
      "REFERENCE   8  (residues 1 to 255)\n",
      "  AUTHORS   Kenter M, Otting N, Anholts J, Leunissen J, Jonker M and Bontrop\n",
      "            RE.\n",
      "  TITLE     Evolutionary relationships among the primate Mhc-DQA1 and DQA2\n",
      "            alleles\n",
      "  JOURNAL   Immunogenetics 36 (2), 71-78 (1992)\n",
      "   PUBMED   1612647\n",
      "REFERENCE   9  (residues 1 to 255)\n",
      "  AUTHORS   Gyllensten UB and Erlich HA.\n",
      "  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\n",
      "            primates\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\n",
      "   PUBMED   2513578\n",
      "REFERENCE   10 (residues 1 to 255)\n",
      "  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\n",
      "            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\n",
      "  TITLE     Class II genes of the human major histocompatibility complex.\n",
      "            Comparisons of the DQ and DX alpha and beta genes\n",
      "  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\n",
      "   PUBMED   3036828\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL672104.4.\n",
      "            \n",
      "            On or before Aug 31, 2013 this sequence version replaced\n",
      "            XP_005275167.1, XP_005275442.1, XP_005275602.1, XP_005249107.1,\n",
      "            XP_005272863.1, XP_005274911.1.\n",
      "            \n",
      "            Summary: This gene belongs to the HLA class II alpha chain family.\n",
      "            The encoded protein forms a heterodimer with a class II beta chain.\n",
      "            It is located in intracellular vesicles and plays a central role in\n",
      "            the peptide loading of MHC class II molecules by helping to release\n",
      "            the CLIP molecule from the peptide binding site. Class II molecules\n",
      "            are expressed in antigen presenting cells (B lymphocytes, dendritic\n",
      "            cells, macrophages) and are used to present antigenic peptides on\n",
      "            the cell surface to be recognized by CD4 T-cells. [provided by\n",
      "            RefSeq, Jun 2010].\n",
      "            \n",
      "            Sequence Note: The RefSeq transcript and protein were derived from\n",
      "            genomic sequence to make the sequence consistent with the reference\n",
      "            genome assembly. The genomic coordinates used for the transcript\n",
      "            record were based on alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            RNAseq introns :: single sample supports all introns SAMEA2144335,\n",
      "                              SAMEA2145893 [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            MANE Ensembl match     :: ENST00000374940.4/ ENSP00000364076.3\n",
      "            RefSeq Select criteria :: based on single protein-coding transcript\n",
      "            ##RefSeq-Attributes-END##\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..255\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"6\"\n",
      "                     /map=\"6p21.32\"\n",
      "     Protein         1..255\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain precursor\"\n",
      "                     /note=\"HLA class II histocompatibility antigen, DQ alpha 2\n",
      "                     chain; MHC class II DQA2; DX alpha chain; HLA class II\n",
      "                     histocompatibility antigen, DQ(6) alpha chain; MHC class\n",
      "                     II antigen; MHC class II DQA1; HLA class II\n",
      "                     histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha\n",
      "                     chain; MHC class I antigen\"\n",
      "                     /calculated_mol_wt=25793\n",
      "     sig_peptide     1..23\n",
      "                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\n",
      "                     /calculated_mol_wt=2258\n",
      "     mat_peptide     24..255\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain\"\n",
      "                     /calculated_mol_wt=25793\n",
      "     Region          24..110\n",
      "                     /region_name=\"Alpha-1\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     Region          29..109\n",
      "                     /region_name=\"MHC_II_alpha\"\n",
      "                     /note=\"Class II histocompatibility antigen, alpha domain;\n",
      "                     pfam00993\"\n",
      "                     /db_xref=\"CDD:425978\"\n",
      "     Site            104\n",
      "                     /site_type=\"glycosylation\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     Region          111..205\n",
      "                     /region_name=\"IgC1_MHC_II_alpha_HLA-DQ\"\n",
      "                     /note=\"Class II major histocompatibility complex (MHC)\n",
      "                     alpha chain immunoglobulin domain of histocompatibility\n",
      "                     antigen (HLA) DQ and related proteins; member of the\n",
      "                     C1-set of Ig superfamily (IgSF) domains; cd21008\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          111..204\n",
      "                     /region_name=\"Alpha-2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     Region          112..120\n",
      "                     /region_name=\"Ig strand A\"\n",
      "                     /note=\"Ig strand A [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Site            order(116,118..120,122,174,176)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"heterodimer interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          128..137\n",
      "                     /region_name=\"Ig strand B\"\n",
      "                     /note=\"Ig strand B [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Site            order(140..142,161..164,170..171,176)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"MHC binding domain interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          143..149\n",
      "                     /region_name=\"Ig strand C\"\n",
      "                     /note=\"Ig strand C [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Site            144\n",
      "                     /site_type=\"glycosylation\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     Region          151..154\n",
      "                     /region_name=\"Ig strand C'\"\n",
      "                     /note=\"Ig strand C' [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          158..166\n",
      "                     /region_name=\"Ig strand D\"\n",
      "                     /note=\"Ig strand D [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          169..179\n",
      "                     /region_name=\"Ig strand E\"\n",
      "                     /note=\"Ig strand E [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          186..193\n",
      "                     /region_name=\"Ig strand F\"\n",
      "                     /note=\"Ig strand F [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          199..204\n",
      "                     /region_name=\"Ig strand G\"\n",
      "                     /note=\"Ig strand G [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          205..217\n",
      "                     /region_name=\"Connecting peptide\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     Site            218..240\n",
      "                     /site_type=\"transmembrane region\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2)\"\n",
      "     CDS             1..255\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /coded_by=\"NM_020056.5:54..821\"\n",
      "                     /db_xref=\"CCDS:CCDS4753.1\"\n",
      "                     /db_xref=\"GeneID:3118\"\n",
      "                     /db_xref=\"HGNC:HGNC:4943\"\n",
      "                     /db_xref=\"MIM:613503\"\n",
      "ORIGIN      \n",
      "        1 milnkalllg alaltavmsp cggedivadh vasygvnfyq shgpsgqyth efdgdeefyv\n",
      "       61 dletketvwq lpmfskfisf dpqsalrnma vgkhtlefmm rqsnstaatn evpevtvfsk\n",
      "      121 fpvtlgqpnt liclvdnifp pvvnitwlsn ghsvtegvse tsflsksdhs ffkisyltfl\n",
      "      181 psadeiydck vehwgldepl lkhwepeipa pmseltetlv calglsvglm givvgtvfii\n",
      "      241 qglrsvgasr hqgll\n",
      "//\n",
      "\n",
      "\n",
      "Concluído.\n",
      "Encontrados 1 resultados em protein. Irão ser processados 10 resultados.\n",
      "\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'>NP_064440.1 HLA class II histocompatibility antigen, DQ alpha 2 chain precursor [Homo sapiens]\\nMILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGPSGQYTHEFDGDEEFYVDLETKETVWQ\\nLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQSNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFP\\nPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPA\\nPMSELTETLVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\\n\\n'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'NP_064440','protein')\n",
    "pesquisa_ncbi(email,'NP_064440','protein',rettype='fasta',save='y',display='n')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
