{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ATENÇÃO À UTILIZAÇÃO DESTA FUNÇÃO\n",
    "# DEMASIADAS QUERIES SEGUIDAS PODEM LEVAR A BLOQUEIO POR PARTE DO SERVIDOR\n",
    "# EM PRINCÍPIO O MÓDULO ENTREZ ESTÁ PREPARADO PARA LIDAR COM ISSO MAS CONVÉM TER SEMPRE CUIDADO\n",
    "\n",
    "def lit_search(email, term,db = 'pubmed',save = 'N',ext='.txt',amplitude = 10,print='Y'):\n",
    "    import os\n",
    "\n",
    "    # TODO Adicionar indicações para fazer o print e elencar a lista de resultados\n",
    "    # TODO Adicionar documentação\n",
    "\n",
    "\n",
    "    # Importamos o módulo \n",
    "    from Bio import Entrez\n",
    "\n",
    "    # Definimos o email para os fins de pesquisa\n",
    "    Entrez.email = email\n",
    "    \n",
    "    \n",
    "\n",
    "    # Secção egquery que pode ser skipable\n",
    "\n",
    "    handle = Entrez.egquery(term=term)\n",
    "    egq_res = Entrez.read(handle)\n",
    "\n",
    "    for _ in egq_res['eGQueryResult']:\n",
    "        if db in _.values():\n",
    "            print('Resultados em \"pubmed\":',_['Count'])\n",
    "\n",
    "    # Sacar as IDs\n",
    "    handle      = Entrez.esearch(db=db,term=term, retmax=amplitude)\n",
    "    esearch_res = Entrez.read(handle)\n",
    "\n",
    "    lista_ids = esearch_res['IdList']\n",
    "    print('Top 10 artigos ->',lista_ids) # Converter para uma lista de títulos mais à frente\n",
    "    \n",
    "    \n",
    "\n",
    "    # Sacar os abstracts\n",
    "    handle = Entrez.efetch(db=db,id=','.join(lista_ids),rettype='abstract',retmode='text')\n",
    "    fetch_res = handle.read()\n",
    "    \n",
    "    # Usar com print para já, para não estar a bagunçar demasiado.\n",
    "    # Perceber como se pode implementar a escrita de ficheiros para ser mais fácil de mastigar os resultados.\n",
    "\n",
    "    # NÃO FUNCIONA NO GITHUB / LIVECODING\n",
    "    # ADICIONAR PARAMETRO save = 'Y' SE SE PRETENDER GUARDAR FICHEIRO \n",
    "    \n",
    "    if save.upper() == 'N':\n",
    "        print(fetch_res,sep='\\n')\n",
    "\n",
    "    # Verifica a condição de gravação de output como ficheiro de texto\n",
    "    if save.upper() == 'Y':\n",
    "\n",
    "        # Verifica se existe pasta para output e cria-a caso não exista\n",
    "        cwd = os.getcwd()\n",
    "        outputdir = os.path.join(cwd,'lit-search-output')\n",
    "        if not os.path.exists(outputdir):\n",
    "            os.mkdir(outputdir)\n",
    "\n",
    "\n",
    "        # Muda o working directory para a pasta de output para gerar o ficheiro\n",
    "        os.chdir(outputdir)\n",
    "\n",
    "        # Gera o ficheiro           \n",
    "        with open(term+ext , 'w') as _:\n",
    "            _.write(fetch_res)\n",
    "\n",
    "        # Retorna ao working directory anterior\n",
    "        os.chdir(cwd)\n",
    "\n",
    "    handle.close()\n",
    "\n",
    "    return fetch_res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = 'pg21019@alunos.uminho.pt'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Resultados em \"pubmed\": 2007\n",
      "Top 10 artigos -> ['38063198', '38059274', '38036787', '38029420', '38012279', '38001428', '37979046', '37971080', '37958966', '37906636']\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1. JCI Insight. 2023 Dec 8;8(23):e171417. doi: 10.1172/jci.insight.171417.\\n\\nSingle-cell transcriptomics reveals variations in monocytes and Tregs between \\ngout flare and remission.\\n\\nYu H(1), Xue W(1), Yu H(1), Song Y(1), Liu X(1), Qin L(1), Wang S(1), Bao H(1), \\nGu H(1), Chen G(1), Zhao D(1), Tu Y(1), Cheng J(1), Wang L(1), Ai Z(2), Hu D(1), \\nWang L(1), Peng A(1).\\n\\nAuthor information:\\n(1)Center for Nephrology and Clinical Metabolomics and Division of Nephrology, \\nShanghai Tenth People\\'s Hospital, and.\\n(2)Department of Medical Statistics, Tongji University School of Medicine, \\nShanghai, China.\\n\\nGout commonly manifests as a painful, self-limiting inflammatory arthritis. \\nNevertheless, the understanding of the inflammatory and immune responses \\nunderlying gout flares and remission remains ambiguous. Here, based on \\nsingle-cell RNA-Seq and an independent validation cohort, we identified the \\npotential mechanism of gout flare, which likely involves the upregulation of \\nHLA-DQA1+ nonclassical monocytes and is related to antigen processing and \\npresentation. Furthermore, Tregs also play an essential role in the suppressive \\ncapacity during gout remission. Cell communication analysis suggested the \\nexistence of altered crosstalk between monocytes and other T cell types, such as \\nTregs. Moreover, we observed the systemic upregulation of inflammatory and \\ncytokine genes, primarily in classical monocytes, during gout flares. All \\nmonocyte subtypes showed increased arachidonic acid metabolic activity along \\nwith upregulation of prostaglandin-endoperoxide synthase 2 (PTGS2). We also \\ndetected a decrease in blood arachidonic acid and an increase in leukotriene B4 \\nlevels during gout flares. In summary, our study illustrates the distinctive \\nimmune cell responses and systemic inflammation patterns that characterize the \\ntransition from gout flares to remission, and it suggests that blood monocyte \\nsubtypes and Tregs are potential intervention targets for preventing recurrent \\ngout attacks and progression.\\n\\nDOI: 10.1172/jci.insight.171417\\nPMID: 38063198 [Indexed for MEDLINE]\\n\\n\\n2. Arthritis Rheumatol. 2023 Dec 7. doi: 10.1002/art.42768. Online ahead of\\nprint.\\n\\nClinical features and immunogenetic risk factors associated with additional \\nautoantibodies in anti-transcriptional intermediary factor 1γ juvenile-onset \\ndermatomyositis.\\n\\nSherman MA(1), Yang Q(2), Gutierrez-Alamillo L(2), Pak K(1), Flegel WA(3), \\nMammen AL(1)(4)(5), Rider LG(#)(6), Casciola-Rosen LA(#)(2); Childhood Myositis \\nHeterogeneity Collaborative Study Group.\\n\\nCollaborators: Arabshahi B, Ballinger S, Bingham CA, Bohnsack JF, Carrasco R, \\nEberhardt BA, Edelheit BS, Farhadi PN, Finkel TH, Fuhlbrigge RC, Goldmuntz EA, \\nGottlieb BS, Griffin TA, Guirola R, Hannan W, Henrickson M, Hong S, Jones OY, \\nKingsbury DJ, Kishi T, Lindsley CB, Mamyrova G, McCarthy PL, Miller FW, Nativ S, \\nOral EA, Pachman LM, de la Pena W, Perez MD, Person DA, Pinal-Fernandez I, \\nSabbagh SE, Sarkar K, Schiffenbauer A, Schlefman A, Shaham B, Soep J, Sule S, \\nTargoff IN, Vogelgesang SA, Volochayev R, Wahezi D.\\n\\nAuthor information:\\n(1)Muscle Disease Unit, National Institute of Arthritis and Musculoskeletal and \\nSkin Diseases, National Institutes of Health, Bethesda, Maryland, USA.\\n(2)Division of Rheumatology, Johns Hopkins University School of Medicine, \\nBaltimore, Maryland, USA.\\n(3)Department of Transfusion Medicine, NIH Clinical Center, National Institutes \\nof Health, Bethesda, Maryland, USA.\\n(4)Department of Neurology, Johns Hopkins University School of Medicine, \\nBaltimore, Maryland, USA.\\n(5)Department of Medicine, Johns Hopkins University School of Medicine, \\nBaltimore, Maryland, USA.\\n(6)Environmental Autoimmunity Group, Clinical Research Branch, National \\nInstitute of Environmental Health Sciences, National Institutes of Health, \\nBethesda, Maryland, USA.\\n(#)Contributed equally\\n\\nOBJECTIVE: Novel autoantibody specificities including anti-CCAR1 were recently \\ndiscovered in adult patients with anti-TIF1γ-positive dermatomyositis (DM) and \\nwere associated with attenuated cancer emergence. The aims of the present study \\nwere to examine whether these autoantibodies occur in patients with \\njuvenile-onset dermatomyositis (JDM) and to determine their associated features.\\nMETHODS: Sera from 150 patients with anti-TIF1γ autoantibody-positive JDM in a \\ncross-sectional cohort and 90 juvenile healthy controls were assayed for \\nanti-CCAR1, anti-C1Z1, anti-IMMT, anti-TBL1XR1, and anti-Sp4 autoantibodies. \\nDemographics, myositis autoantibodies, clinical features, medications, outcomes, \\nand HLA-DRB1 and HLA-DQA1 alleles were compared between those with and without \\nthese autoantibodies.\\nRESULTS: Any one of the anti-TIF1γ-associated autoantibodies was present in 44 \\n(29%) patients overall, including 25 (17%) with anti-Sp4, 22 (15%) with \\nanti-TBL1XR1, 14 (9%) with anti-CCAR1, 2 (1%) with anti-C1Z1, and 2 (1%) with \\nanti-IMMT autoantibodies. These anti-TIF1γ-associated autoantibodies frequently \\nco-occurred. Patients with any of the anti-TIF1γ-associated autoantibodies had \\nless frequent falling (34% [15] vs 53% [56], p=0.032) and lower peak muscle \\nenzymes. None of the patients had cancer. Among White patients, HLA-DRB1*03 was \\nprotective against an anti-TIF1γ-associated autoantibody (OR 0.20, 95% CI \\n0.07-0.52).\\nCONCLUSION: Autoantibodies associated with anti-TIF1γ were found in isolation \\nand in combination among a subset of patients with JDM. Patients with these \\nautoantibodies had less severe muscle disease and were not enriched for \\nHLA-DRB1*03. Additional autoantibodies among anti-TIF1γ-positive patients with \\nJDM likely contribute to the heterogeneity of the anti-TIF1γ serologic subgroup. \\nThis article is protected by copyright. All rights reserved.\\n\\nThis article is protected by copyright. All rights reserved.\\n\\nDOI: 10.1002/art.42768\\nPMID: 38059274\\n\\n\\n3. Nat Genet. 2023 Dec;55(12):2255-2268. doi: 10.1038/s41588-023-01586-6. Epub\\n2023  Nov 30.\\n\\nMapping the dynamic genetic regulatory architecture of HLA genes at single-cell \\nresolution.\\n\\nKang JB(1)(2)(3)(4)(5), Shen AZ(1)(2)(3)(4)(5), Gurajala S(1)(2)(3)(4)(5), \\nNathan A(1)(2)(3)(4)(5), Rumker L(1)(2)(3)(4)(5), Aguiar VRC(4)(6), Valencia \\nC(1)(2)(3)(4)(5), Lagattuta KA(1)(2)(3)(4)(5), Zhang F(1)(2)(3)(4)(5)(7), \\nJonsson AH(5), Yazar S(8), Alquicira-Hernandez J(8), Khalili H(9), \\nAnanthakrishnan AN(9), Jagadeesh K(10), Dey K(10)(11)(12); Accelerating \\nMedicines Partnership Program: Rheumatoid Arthritis and Systemic Lupus \\nErythematosus (AMP RA/SLE) Network; Daly MJ(13)(14)(15)(16), Xavier \\nRJ(17)(18)(19), Donlin LT(20)(21), Anolik JH(22), Powell JE(8), Rao DA(5), \\nBrenner MB(5), Gutierrez-Arcelus M(4)(6), Luo Y(1)(2)(3)(4)(5)(23), Sakaue \\nS(1)(2)(3)(4)(5), Raychaudhuri S(24)(25)(26)(27)(28).\\n\\nCollaborators: Albrecht J, Apruzzese W, Banda N, Barnas JL, Bathon JM, Ben-Artzi \\nA, Boyce BF, Boyle DL, Bridges SL Jr, Bykerk VP, Campbell D, Carr HL, Ceponis A, \\nChicoine A, Cordle A, Curtis M, Deane KD, DiCarlo E, Dunn P, Filer A, Firestein \\nGS, Forbess L, Geraldino-Pardilla L, Goodman SM, Gravallese EM, Gregersen PK, \\nGuthridge JM, Holers VM, Horowitz D, Hughes LB, Ishigaki K, Ivashkiv LB, James \\nJA, Keras G, Korsunsky I, Lakhanpal A, Lederer JA, Lewis M, Li ZJ, Li Y, Liao \\nKP, Mandelin AM 2nd, Mantel I, Marks KE, Maybury M, McDavid A, McGeachy MJ, \\nMears J, Meednu N, Millard N, Moreland LW, Nayar S, Nerviani A, Orange DE, \\nPerlman H, Pitzalis C, Rangel-Moreno J, Raza K, Reshef Y, Ritchlin C, Rivellese \\nF, Robinson WH, Sahbudin I, Singaraju A, Seifert JA, Slowikowski K, Smith MH, \\nTabechian D, Scheel-Toellner D, Utz PJ, Watts GFM, Wei K, Weinand K, Weisenfeld \\nD, Weisman MH, Wyse A, Xiao Q, Zhu Z.\\n\\nAuthor information:\\n(1)Center for Data Sciences, Brigham and Women\\'s Hospital, Boston, MA, USA.\\n(2)Division of Genetics, Department of Medicine, Brigham and Women\\'s Hospital \\nand Harvard Medical School, Boston, MA, USA.\\n(3)Department of Biomedical Informatics, Harvard Medical School, Boston, MA, \\nUSA.\\n(4)Program in Medical and Population Genetics, Broad Institute of MIT and \\nHarvard, Cambridge, MA, USA.\\n(5)Division of Rheumatology, Inflammation, and Immunity, Department of Medicine, \\nBrigham and Women\\'s Hospital and Harvard Medical School, Boston, MA, USA.\\n(6)Division of Immunology, Boston Children\\'s Hospital, Harvard Medical School, \\nBoston, MA, USA.\\n(7)Division of Rheumatology and the Center for Health Artificial Intelligence, \\nUniversity of Colorado School of Medicine, Aurora, CO, USA.\\n(8)Garvan Institute of Medical Research, Sydney, New South Wales, Australia.\\n(9)Division of Gastroenterology, Massachusetts General Hospital and Harvard \\nMedical School, Boston, MA, USA.\\n(10)Harvard T. H. Chan School of Public Health, Boston, MA, USA.\\n(11)Computational and Systems Biology Program, Sloan Kettering Institute, \\nMemorial Sloan Kettering Cancer Center, New York, NY, USA.\\n(12)Physiology, Biophysics and Systems Biology Program, Weill Cornell Medicine, \\nNew York, NY, USA.\\n(13)Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General \\nHospital, Boston, MA, USA.\\n(14)The Stanley Center for Psychiatric Research, The Broad Institute of MIT and \\nHarvard, Cambridge, MA, USA.\\n(15)Institute for Molecular Medicine Finland (FIMM), University of Helsinki, \\nHelsinki, Finland.\\n(16)Analytic and Translational Genetics Unit, Department of Medicine, \\nMassachusetts General Hospital, Boston, MA, USA.\\n(17)Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA, \\nUSA.\\n(18)Center for Computational and Integrative Biology, Massachusetts General \\nHospital and Harvard Medical School, Boston, MA, USA.\\n(19)Department of Molecular Biology, Massachusetts General Hospital and Harvard \\nMedical School, Boston, MA, USA.\\n(20)Hospital for Special Surgery, New York, NY, USA.\\n(21)Weill Cornell Medicine, New York, NY, USA.\\n(22)Department of Medicine, University of Rochester Medical Center, Rochester, \\nNY, USA.\\n(23)Kennedy Institute of Rheumatology, University of Oxford, Oxford, UK.\\n(24)Center for Data Sciences, Brigham and Women\\'s Hospital, Boston, MA, USA. \\nsoumya@broadinstitute.org.\\n(25)Division of Genetics, Department of Medicine, Brigham and Women\\'s Hospital \\nand Harvard Medical School, Boston, MA, USA. soumya@broadinstitute.org.\\n(26)Department of Biomedical Informatics, Harvard Medical School, Boston, MA, \\nUSA. soumya@broadinstitute.org.\\n(27)Program in Medical and Population Genetics, Broad Institute of MIT and \\nHarvard, Cambridge, MA, USA. soumya@broadinstitute.org.\\n(28)Division of Rheumatology, Inflammation, and Immunity, Department of \\nMedicine, Brigham and Women\\'s Hospital and Harvard Medical School, Boston, MA, \\nUSA. soumya@broadinstitute.org.\\n\\nUpdate of\\n    medRxiv. 2023 Mar 20;:\\n\\nThe human leukocyte antigen (HLA) locus plays a critical role in complex traits \\nspanning autoimmune and infectious diseases, transplantation and cancer. While \\ncoding variation in HLA genes has been extensively documented, regulatory \\ngenetic variation modulating HLA expression levels has not been comprehensively \\ninvestigated. Here we mapped expression quantitative trait loci (eQTLs) for \\nclassical HLA genes across 1,073 individuals and 1,131,414 single cells from \\nthree tissues. To mitigate technical confounding, we developed scHLApers, a \\npipeline to accurately quantify single-cell HLA expression using personalized \\nreference genomes. We identified cell-type-specific cis-eQTLs for every \\nclassical HLA gene. Modeling eQTLs at single-cell resolution revealed that many \\neQTL effects are dynamic across cell states even within a cell type. HLA-DQ \\ngenes exhibit particularly cell-state-dependent effects within myeloid, B and T \\ncells. For example, a T cell HLA-DQA1 eQTL ( rs3104371 ) is strongest in \\ncytotoxic cells. Dynamic HLA regulation may underlie important interindividual \\nvariability in immune responses.\\n\\n© 2023. The Author(s), under exclusive licence to Springer Nature America, Inc.\\n\\nDOI: 10.1038/s41588-023-01586-6\\nPMID: 38036787 [Indexed for MEDLINE]\\n\\n\\n4. Inflamm Bowel Dis. 2023 Nov 29:izad273. doi: 10.1093/ibd/izad273. Online ahead\\n of print.\\n\\nHLA-DQA1*05 Not Associated With Ustekinumab Loss of Response and Antidrug \\nAntibodies in Ulcerative Colitis and Crohn\\'s Disease Patients.\\n\\nColombel JF(1), Martín-Arranz MD(2), Brinkman B(3), Guan M(4), Hart A(4), Gasink \\nC(5).\\n\\nAuthor information:\\n(1)Icahn School of Medicine at Mt Sinai, New York, NY, USA.\\n(2)Gastroenterology Department, Hospital Universitario de La Paz, Institute for \\nHealth Research La Paz, Universidad Autónoma de Madrid, Madrid, Spain.\\n(3)Janssen Pharmaceutica, Beerse, Belgium.\\n(4)Janssen Research & Development, LLC, Spring House, PA, USA.\\n(5)Janssen Scientific Affairs, LLC, Horsham, PA, USA.\\n\\nPlain Language Summary: It is unknown if inflammatory bowel disease patients \\ncarrying the HLA-DQA1*05 allele have a greater risk of immunogenicity to \\nustekinumab. We observed that ustekinumab-treated patients carrying the allele \\ndid not have increased immunogenicity or reduced efficacy/serum ustekinumab \\nconcentrations vs noncarriers.\\n\\nDOI: 10.1093/ibd/izad273\\nPMID: 38029420\\n\\n\\n5. Sci Rep. 2023 Nov 27;13(1):20820. doi: 10.1038/s41598-023-47632-5.\\n\\nA web-based genome-wide association study reveals the susceptibility loci of \\ncommon adverse events following COVID-19 vaccination in the Japanese population.\\n\\nNogawa S(1), Kanamori H(2)(3), Tokuda K(2)(3), Kawafune K(1), Chijiiwa M(1), \\nSaito K(1), Takahashi S(4).\\n\\nAuthor information:\\n(1)Genequest Inc., Siba 5-29-11, Minato-ku, Tokyo, 108-0014, Japan.\\n(2)Department of Infectious Diseases, Internal Medicine, Tohoku University \\nGraduate School of Medicine, 2-1 Seiryo-machi, Aoba-ku, Sendai, 980-8574, Japan.\\n(3)Division of Infection Control, Tohoku University Hospital, 1-1 Seiryo-machi, \\nAoba-ku, Sendai, 980-8574, Japan.\\n(4)Genequest Inc., Siba 5-29-11, Minato-ku, Tokyo, 108-0014, Japan. \\ntakahashi@genequest.jp.\\n\\nThe coronavirus disease 2019 (COVID-19) pandemic has spread rapidly worldwide. \\nTo prevent its spread, mRNA-based vaccines made by Pfizer/BioNTech (BNT162b1) \\nand Moderna (mRNA-1273) have been widely used, including in Japan. Various \\nadverse events have been reported following the COVID-19 mRNA vaccination, with \\ndifferences observed among individuals. However, analyses of the genetic \\nbackground associated with the susceptibility to side effects have been limited. \\nIn the present study, we performed genome-wide association studies (GWAS) for \\nself-reported adverse events of the COVID-19 mRNA vaccination in 4545 Japanese \\nindividuals and identified 14 associated loci. Among these, 6p21 was associated \\nwith 37.5\\xa0°C or higher fever, 38\\xa0°C or higher fever, and muscle pain. HLA allele \\nassociation analysis revealed that various HLA alleles were associated with the \\nadverse effects; HLA-DQA1*03:01 and HLA-A*11:01 were more reliably associated \\nwith the adverse effects. Our results may enable the preparation and management \\nof adverse effects by identifying the susceptibility to these adverse events. \\nFurthermore, we obtained valuable data that may lead to a better understanding \\nof the mechanisms of action of the COVID-19 mRNA vaccines.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1038/s41598-023-47632-5\\nPMCID: PMC10682012\\nPMID: 38012279 [Indexed for MEDLINE]\\n\\nConflict of interest statement: S.N., K.K., and M.C. are employees of Genequest \\nInc.; K.S. and S.T. are board members of Genequest Inc.; H.K. and K.T. declare \\nno competing interests.\\n\\n\\n6. BMC Cancer. 2023 Nov 24;23(1):1141. doi: 10.1186/s12885-023-11579-7.\\n\\nMolecular subtyping based on immune cell marker genes predicts prognosis and \\ntherapeutic response in patients with lung adenocarcinoma.\\n\\nLiu ZT(#)(1), Shen JT(#)(1), Lei YJ(#)(1), Huang YC(1), Zhao GQ(1), Zheng CH(2), \\nWang X(1), Wang YT(1), Chen L(3), Li ZX(1), Li SZ(1), Liao J(1), Yu TD(4).\\n\\nAuthor information:\\n(1)Department of Thoracic Surgery, The Third Affiliated Hospital of Kunming \\nMedical University, Kunming, China.\\n(2)Department of Ultrasound, The Third Affiliated Hospital of Kunming Medical \\nUniversity, Yunnan Cancer Hospital, Kunming, China.\\n(3)Department of PET/CT Center, Cancer Center of Yunnan Province, The Third \\nAffiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, \\nKunming, China.\\n(4)Department of Thoracic Surgery, The Third Affiliated Hospital of Kunming \\nMedical University, Kunming, China. yutingdong@outlook.com.\\n(#)Contributed equally\\n\\nOBJECTIVE: Lung adenocarcinoma (LA) is one of the most common malignancies and \\nis responsible for the greatest number of tumor-related deaths. Our research \\naimed to explore the molecular subtype signatures of LA to clarify the \\ncorrelation among the immune microenvironment, clinical outcomes, and \\ntherapeutic response.\\nMETHODS: The LA immune cell marker genes (LICMGs) identified by single-cell RNA \\nsequencing (scRNA-seq) analysis were used to discriminate the molecular subtypes \\nand homologous immune and metabolic traits of GSE72094 LA cases. In addition, \\nthe model-building genes were identified from 1441 LICMGs by Cox-regression \\nanalysis, and a LA immune difference score (LIDscore) was developed to quantify \\nindividual differences in each patient, thereby predicting prognosis and \\nsusceptibility to immunotherapy and chemotherapy of LA patients.\\nRESULTS: Patients of the GSE72094 cohort were divided into two distinct \\nmolecular subtypes based on LICMGs: immune activating subtype (Cluster-C1) and \\nmetabolically activating subtype (cluster-C2). The two molecular subtypes have \\ndistinct characteristics regarding prognosis, clinicopathology, genomics, immune \\nmicroenvironment, and response to immunotherapy. Among the LICMGs, LGR4, GOLM1, \\nCYP24A1, SFTPB, COL1A1, HLA-DQA1, MS4A7, PPARG, and IL7R were enrolled to \\nconstruct a LIDscore model. Low-LIDscore patients had a higher survival rate due \\nto abundant immune cell infiltration, activated immunity, and lower genetic \\nvariation, but probably the higher levels of Treg cells in the immune \\nmicroenvironment lead to immune cell dysfunction and promote tumor immune \\nescape, thus decreasing the responsiveness to immunotherapy compared with that \\nof the high-LIDscore patients. Overall, high-LIDscore patients had a higher \\nresponsiveness to immunotherapy and a higher sensitivity to chemotherapy than \\nthe low-LIDscore group.\\nCONCLUSIONS: Molecular subtypes based on LICMGs provided a promising strategy \\nfor predicting patient prognosis, biological characteristics, and immune \\nmicroenvironment features. In addition, they helped identify the patients most \\nlikely to benefit from immunotherapy and chemotherapy.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s12885-023-11579-7\\nPMCID: PMC10668343\\nPMID: 38001428 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n7. Immunogenetics. 2023 Nov 18. doi: 10.1007/s00251-023-01325-5. Online ahead of \\nprint.\\n\\nInvestigating associations between HLA DQA1\\u2009~\\u2009DQB1 haplotypes, H. pylori \\ninfection, metaplasia, and anti-CagA IgA seropositivity in a Turkish gastritis \\ncohort.\\n\\nColakogullari M(1), Karatas L(2), Tatar Z(2).\\n\\nAuthor information:\\n(1)Izmir Democracy University, Izmir, Turkey. \\nmukaddes.colakogullari@yahoo.co.uk.\\n(2)Izmir Democracy University, Izmir, Turkey.\\n\\nHelicobacter pylori was reported as an important cause of gastritis, and gastric \\nulcers and CagA oncoprotein-producing H. pylori subgroups were blamed to \\nincrease the severity of gastritis. Disparities were reported in that the \\npresence of serum anti-CagA IgA was not parallel with CagA-positive H. pylori \\ncohabitation. We hypothesized that the HLA-DQA1\\u2009~\\u2009DQB1 haplotypes in human \\npopulations include protective haplotypes that more effectively present \\nimmunogenic CagA peptides and susceptible haplotypes with an impaired capacity \\nto present CagA peptides. We recruited patients (n\\u2009=\\u2009201) admitted for \\ngastroendoscopy procedures and performed high-resolution HLA-DQA1 and DQB1 \\ntyping. Serum anti-CagA IgA levels were analyzed by ELISA (23.0% positive), and \\nH. pylori was classified as positive or negative in gastric mucosal tissue \\nslides (72.6% positive). The HLA DQA1*05:05 allele (29.1%) and HLA DQB1*03:01 \\nallele (32.8%) were found at the highest frequency among gastritis patients of \\nTurkish descent. In HLA DQA1*05:05\\u2009~\\u2009DQB1*03:01 double homozygous (7.3%) and \\nheterozygous (40.7%) haplotype carriers, the presence of anti-CagA IgA decreased \\ndramatically, the presence of H. pylori increased, and the presence of \\nmetaplasia followed a decreasing trend. The DQ protein encoded by HLA \\nDQA1*05:05-DQ*03:01 showed a low binding affinity to the CagA peptide when \\nbinding capacity was analyzed by the NetMHCIIPan 4.0 prediction method. In \\nconclusion, HLA DQA1\\u2009~\\u2009DQB1 polymorphisms are crucial as host defense mechanisms \\nagainst CagA H. pylori since antigen binding capacity plays a crucial role in \\nanti-CagA IgA production.\\n\\n© 2023. The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, \\npart of Springer Nature.\\n\\nDOI: 10.1007/s00251-023-01325-5\\nPMID: 37979046\\n\\n\\n8. HLA. 2023 Nov 16. doi: 10.1111/tan.15291. Online ahead of print.\\n\\nIdentification of the novel HLA-DQA1*01:138 allele by next-generation \\nsequencing.\\n\\nLee JK(1), Bae GE(1), Lee S(1), Park JW(1), Kang ES(1).\\n\\nAuthor information:\\n(1)Department of Laboratory Medicine and Genetics, Samsung Medical Center, \\nSungkyunkwan University School of Medicine, Seoul, South Korea.\\n\\nHLA-DQA1*01:138 is identical to HLA-DQA1*01:03 except for a single nucleotide \\nsubstitution in exon 3.\\n\\n© 2023 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd.\\n\\nDOI: 10.1111/tan.15291\\nPMID: 37971080\\n\\n\\n9. Int J Mol Sci. 2023 Nov 5;24(21):15984. doi: 10.3390/ijms242115984.\\n\\nCandidate Genes for IgA Nephropathy in Pediatric Patients: Exome-Wide \\nAssociation Study.\\n\\nBuianova AA(1), Proskura MV(2), Cheranev VV(1), Belova VA(1), Shmitko AO(1), \\nPavlova AS(1), Vasiliadis IA(1), Suchalko ON(1), Rebrikov DV(1), Petrosyan \\nEK(2), Korostin DO(1).\\n\\nAuthor information:\\n(1)Center for Precision Genome Editing and Genetic Technologies for Biomedicine, \\nPirogov Russian National Research Medical University, Ostrovityanova Str., 1, p. \\n1, 117513 Moscow, Russia.\\n(2)Nephrology Department, Russian Children\\'s Clinical Hospital, Leninsky \\nProspect 117, 119571 Moscow, Russia.\\n\\nIgA nephropathy (IgAN) is an autoimmune disorder which is believed to be \\nnon-monogenic. We performed an exome-wide association study of 70 children with \\nIgAN and 637 healthy donors. The HLA allele frequencies were compared between \\nthe patients and healthy donors from the bone marrow registry of the Pirogov \\nUniversity. We tested 78,020 gene markers for association and performed \\nfunctional enrichment analysis and transcription factor binding preference \\ndetection. We identified 333 genetic variants, employing three inheritance \\nmodels. The most significant association with the disorder was observed for \\nrs143409664 (PRAG1) in the case of the additive and dominant models (PBONF = \\n1.808 × 10-15 and PBONF = 1.654 × 10-15, respectively), and for rs13028230 \\n(UBR3) in the case of the recessive model (PBONF = 1.545 × 10-9). Enrichment \\nanalysis indicated the strongly overrepresented \"immune system\" and \"kidney \\ndevelopment\" terms. The HLA-DQA1*01:01:01G allele (p = 0.0076; OR, 2.021 [95% \\nCI, 1.322-3.048]) was significantly the most frequent among IgAN patients. Here, \\nwe characterized, for the first time, the genetic background of Russian IgAN \\npatients, identifying the risk alleles typical of the population. The most \\nimportant signals were detected in previously undescribed loci.\\n\\nDOI: 10.3390/ijms242115984\\nPMCID: PMC10647220\\nPMID: 37958966 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no conflict of interest.\\n\\n\\n10. Clin Cancer Res. 2023 Oct 31. doi: 10.1158/1078-0432.CCR-23-2213. Online\\nahead  of print.\\n\\nBiomarkers of Pathological Complete Response to Neoadjuvant Immunotherapy in \\nMismatch Repair Deficiency Colorectal Cancer.\\n\\nLi J(1), Hu H(2), Qin G(1), Bai F(1), Wu X(1), Ke H(1), Zhang J(1), Xie Y(1), Wu \\nZ(2), Fu Y(1), Zheng H(3), Gong L(4), Xie Z(5), Deng Y(6).\\n\\nAuthor information:\\n(1)Sixth Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, \\nChina.\\n(2)Sixth Affiliated Hospital of Sun Yat-sen University, Guangzhou, China.\\n(3)Genecast Biotechnology Co., Ltd, Guangzhou, China.\\n(4)Beijing Genecast Biotechnology Co., Beijing, China.\\n(5)State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun \\nYat-sen University, Guangzhou, China.\\n(6)The Sixth Affiliated Hospital, Sun Yat-sen University, Guangzhou, Guangdong, \\nChina.\\n\\nPURPOSE: Immune checkpoint inhibitors (ICIs) have become the standard of care \\nfor patients with mismatch repair-deficient/microsatellite instability-high \\n(dMMR/MSI-H) colorectal cancer (CRC). However, biomarkers of response to ICIs \\nare still lacking.\\nEXPERIMENTAL DESIGN: Forty-two patients with dMMR CRC treated with neoadjuvant \\nPD-1 blockade were prospectively enrolled. To identify biomarkers of \\npathological complete response (pCR) to neoadjuvant therapy, we analyzed genomic \\nand transcriptomic profiles based on next-generation sequencing, and immune cell \\ndensity based on multiplex immunofluorescence (mIF) staining. An integrated \\nanalysis of single-cell RNA sequencing from our previous study and GSE178341, as \\nwell as mIF was performed to further explore the significance of the tumor \\nmicroenvironment (TME) on pCR response.\\nRESULTS: The tumor mutation burden of both tumor tissue and plasma blood samples \\nwas comparable between the pCR and non-pCR groups, while HLA-DQA1 and HLA-DQB1 \\nwere significantly overexpressed in the pCR group. Gene signature enrichment \\nanalysis showed that pathways including T cell receptor pathway, antigen \\npresentation pathway were significantly enriched in the pCR group. In addition, \\nhigher pre-existing CD8+ T cell density was associated with pCR response (767.47 \\nper.mm2 vs. 326.64 per.mm2, P=0.013 Wilcoxon test). Further integrated analysis \\nshowed that CD8+ T cells with low PD-1 expression (PD-1lo CD8+ T cells) \\nexpressing high levels of TRGC2, CD160, KLRB1 and low levels of proliferated and \\nexhausted genes were significantly associated with pCR response.\\nCONCLUSIONS: Immune-associated transcriptomic features, particularly CD8+ T \\ncells were associated with pCR response to ICI in dMMR CRC. Heterogeneity of TME \\nwithin dMMR CRC may help to discriminate patients with complete response to \\nneoadjuvant ICI.\\n\\nDOI: 10.1158/1078-0432.CCR-23-2213\\nPMID: 37906636'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lit_search('pg21019@alunos.uminho.pt','hla-dqa2')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Resultados em \"pubmed\": 19\n",
      "Top 10 artigos -> ['155971', '3106', '920', '3119', '3630', '155871', '155030', '3107', '3117', '156110']\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\n1. env\\nEnvelope surface glycoprotein gp160, precursor [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp8\\nOther Designations: Envelope surface glycoprotein gp160, precursor; Envelope surface glycoprotein gp120; Envelope transmembrane domain; hypothetical protein\\nAnnotation:  NC_001802.1 (5771..8341)\\nID: 155971\\n\\n2. HLA-B\\nOfficial Symbol: HLA-B and Name: major histocompatibility complex, class I, B [Homo sapiens (human)]\\nOther Aliases: AS, B-4901, HLAB\\nOther Designations: major histocompatibility complex, class I, B; HLA class I antigen HLA-B; HLA class I histocompatibility antigen, B alpha chain; MHC HLA-B cell surface glycoprotein; MHC HLA-B transmembrane glycoprotein; MHC class 1 antigen; MHC class I antigen HLA-B alpha chain; MHC class I antigen HLA-B heavy chain; MHC class I antigen SHCHA; MHC class I molecule; leukocyte antigen class I-B\\nChromosome: 6; Location: 6p21.33\\nAnnotation: Chromosome 6 NC_000006.12 (31353875..31357179, complement)\\nMIM: 142830\\nID: 3106\\n\\n3. CD4\\nOfficial Symbol: CD4 and Name: CD4 molecule [Homo sapiens (human)]\\nOther Aliases: CD4mut, IMD79, Leu-3, OKT4D, T4\\nOther Designations: T-cell surface glycoprotein CD4; CD4 antigen (p55); CD4 receptor; T-cell surface antigen T4/Leu-3\\nChromosome: 12; Location: 12p13.31\\nAnnotation: Chromosome 12 NC_000012.12 (6789528..6820799)\\nMIM: 186940\\nID: 920\\n\\n4. HLA-DQB1\\nOfficial Symbol: HLA-DQB1 and Name: major histocompatibility complex, class II, DQ beta 1 [Homo sapiens (human)]\\nOther Aliases: CELIAC1, HLA-DQB, IDDM1\\nOther Designations: HLA class II histocompatibility antigen, DQ beta 1 chain; MHC class II DQ beta chain; MHC class II HLA-DQ beta glycoprotein; MHC class II antigen DQB1; MHC class II antigen HLA-DQ-beta-1\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32659467..32666657, complement)\\nMIM: 604305\\nID: 3119\\n\\n5. INS\\nOfficial Symbol: INS and Name: insulin [Homo sapiens (human)]\\nOther Aliases: IDDM, IDDM1, IDDM2, ILPR, IRDN, MODY10, PNDM4\\nOther Designations: insulin; preproinsulin; proinsulin\\nChromosome: 11; Location: 11p15.5\\nAnnotation: Chromosome 11 NC_000011.10 (2159779..2161209, complement)\\nMIM: 176730\\nID: 3630\\n\\n6. tat\\nTat [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp5\\nOther Designations: Tat\\nAnnotation:  NC_001802.1 (5377..7970)\\nID: 155871\\n\\n7. gag\\nPr55(Gag) [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp2\\nOther Designations: Pr55(Gag); capsid; matrix; nucleocapsid; p1; p2; p6\\nAnnotation:  NC_001802.1 (336..1838)\\nID: 155030\\n\\n8. HLA-C\\nOfficial Symbol: HLA-C and Name: major histocompatibility complex, class I, C [Homo sapiens (human)]\\nOther Aliases: D6S204, HLA-JY3, HLAC, HLC-C, MHC, PSORS1\\nOther Designations: HLA class I histocompatibility antigen, C alpha chain; HLA-C antigen; MHC class I antigen heavy chain HLA-C; human leukocyte antigen-C alpha chain; major histocompatibility antigen HLA-C\\nChromosome: 6; Location: 6p21.33\\nAnnotation: Chromosome 6 NC_000006.12 (31268749..31272092, complement)\\nMIM: 142840\\nID: 3107\\n\\n9. HLA-DQA1\\nOfficial Symbol: HLA-DQA1 and Name: major histocompatibility complex, class II, DQ alpha 1 [Homo sapiens (human)]\\nOther Aliases: CELIAC1, DQ-A1, DQA1, HLA-DQA*, HLA-DQB1, HLA-DQA1\\nOther Designations: HLA class II histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha chain; DC-alpha; HLA class II histocompatibility antigen DQ alpha chain; HLA-DCA; MHC HLA-DQ alpha; MHC class II DQ alpha chain; MHC class II DQA1; MHC class II HLA-DQ-alpha-1; MHC class II antigen DQA1; MHC class II protein\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32637406..32655272)\\nMIM: 146880\\nID: 3117\\n\\n10. nef\\nNef [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp9\\nOther Designations: Nef\\nAnnotation:  NC_001802.1 (8343..8963)\\nID: 156110\\n\\n'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lit_search(email,'hla-dqa2','gene')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Top 10 artigos -> ['2594567951']\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'LOCUS       OR557191                5232 bp    DNA     linear   PRI 25-OCT-2023\\nDEFINITION  Homo sapiens isolate PSCDA0402 MHC class II antigen (HLA-DQA2)\\n            gene, HLA-DQA2*01:05:01:01extended allele, complete cds.\\nACCESSION   OR557191\\nVERSION     OR557191.1\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1  (bases 1 to 5232)\\n  AUTHORS   Khor,S.-S., Huijse,L., Endlich,S. and Tokunaga,K.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (15-SEP-2023) Genome Medical Science Project, National\\n            Center for Global Health and Medicine, Toyama 1-21-1, Shinjuku\\n            Ward, Tokyo 162-8655, Japan\\nCOMMENT     ##Assembly-Data-START##\\n            Assembly Method       :: NGSengine v. 2.30.1.29498\\n            Sequencing Technology :: PacBio\\n            ##Assembly-Data-END##\\nFEATURES             Location/Qualifiers\\n     source          1..5232\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /isolate=\"PSCDA0402\"\\n                     /db_xref=\"taxon:9606\"\\n     gene            1..5232\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n     mRNA            join(1..162,3797..4045,4429..4710,4878..5232)\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /product=\"MHC class II antigen\"\\n     exon            1..162\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=1\\n     5\\'UTR           1..80\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n     CDS             join(81..162,3797..4045,4429..4710,4878..5032)\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /codon_start=1\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"WOF76037.1\"\\n                     /translation=\"MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGP\\n                     SGQYTHEFDGDEEFYVDLEMKETVWQLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQ\\n                     SNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSE\\n                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTET\\n                     LVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\"\\n     intron          163..3796\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=1\\n     exon            3797..4045\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=2\\n     intron          4046..4428\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=2\\n     exon            4429..4710\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=3\\n     intron          4711..4877\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=3\\n     exon            4878..5232\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\n                     /number=4\\n     3\\'UTR           5033..5232\\n                     /gene=\"HLA-DQA2\"\\n                     /allele=\"HLA-DQA2*01:05:01:01extended\"\\nORIGIN      \\n        1 ttcagatttg cttgtctcga ggtcctcaca attgctctac agctcagagc agcaactgct\\n       61 gaggctgcct tgggaagaag atgatcctaa acaaagctct gctgctgggg gccctcgccc\\n      121 tgactgccgt gatgagcccc tgtggaggtg aagacattgt gggtgagtgc atgagtgagg\\n      181 aatgttctct ggagctgaaa aacagcaaat taaaggaaaa gaaagagtgc aatttgctaa\\n      241 gaaatagtag aaatttccca agggtctttt caatattaag aaattttaaa attatggcag\\n      301 ttcctccttt aggaaaccag agctccaacc gactctcttt gctacctgtg ctattggagt\\n      361 ttaccaagga cgttgttctg tttatattat atccagagac tatagcctgg aggtctgtgt\\n      421 ggcattccat catgattgcc tcaaagacta gggatgtttc catgaatgga gtattttttt\\n      481 gttattaaaa atttctgaac tgttactccc aaatttctct gaacaacttt tgaagctttt\\n      541 catatgcctc ctatagcata tgttggggta gatagttcca tgaagtatgt acactctata\\n      601 gatataaaga aagaggttct tttctttctc tcagacttac atttccacat gggaattggc\\n      661 acaggtgggg agtaggtgaa agagcccagc aggctgaatg ccttcaacaa tcattttacc\\n      721 acgtggtaaa tgtggtactt actctctgct acctcatata tgtcacctcg cttatgatca\\n      781 aataaaatgg gcatgtagat atgctttatg aatagtaaaa acatgaatgt caactttttt\\n      841 taacttattc ctattacagg tataacttcg tattttttct ttagcaaagt aaggaatata\\n      901 ttttaaaact gagaacttta tgataaaatg cttggtaaat taaattattt tattctcaaa\\n      961 ttgtcaaccc aaattacttg ttcttcacct tatctaatga agtcttataa agagaaaaat\\n     1021 gggcaggcac agattatttg gtcccttagt cccctctgcc tttgtcgtcc atctctttcc\\n     1081 acctctcttc atgcatccct ttctccctct tccctttcag gatccatctc tgactccctg\\n     1141 ctcctttaca gacatgggca gtgggtttgt aaaacaaaag ttggaaagtc aaatagttaa\\n     1201 aaggggaagt gaactggaag ctactcgaaa cttccacaac cttattaacc atagctgctc\\n     1261 ccattctgat tttgtttggc agtggaagtt tcacctgctt ctccagagca cttggctttt\\n     1321 ttttttcaaa tctcctttct tcaacctcac accagagtgc cccggtcagg ctcgacttat\\n     1381 ccattaggaa cagtgtgggc agtgaaggag actctccaaa ctgtaaagct acaagagaac\\n     1441 gttttaactc gttttaaaat tagaagaaaa atgaagtttt acagtctatg aaaatgtttt\\n     1501 aacttttttt tttttttgac ggagtctcgc tctgtcgccc aggctggagt gcagtggcac\\n     1561 gatctcagct cactgcaagc tccgcctccc gggttcacgc cattctcctg cctcagcctc\\n     1621 ccgagtagct gggactacag gcgcccgcca ccgcgcacgg ctaatttttt gtatttttag\\n     1681 tagggacggg gtttcaccgt gtgagccagg atggtctcga tctcctgacc tcatgatccg\\n     1741 cccgtctcaa cctcccaaag tgctgggatt acaggcgtga tccaccgtgt ccggccttaa\\n     1801 cttttaatgt agcctggatt gtatttgtct ttataccaat acaatcagaa gctgtaattt\\n     1861 tccgtatttt tatggaggaa ggcgcccaca aaagcaacag tgctcggggc tcacaagtca\\n     1921 gaattcagcc ctgggcatcc ctgatcctgg gctttgcgtg gttctgctac ctgggtgcct\\n     1981 gtcagtcttc cccaaaatct atgtaattgt caaaaattgc aattgtcatt caatacacat\\n     2041 gtttgagcac acaatgagct aacttttggg aattcaaaga taaaaaatca tgctgtctgc\\n     2101 cttgcagagg gtgcacaaac cagtgatgga aacagtatgg ggcacaggaa agcagaaggc\\n     2161 cctgctgagc aggacagtgg cccagcagag gctgaaacta taaaaatgac ttggttccag\\n     2221 ctgggccagt agagtgatgt cctccagcaa cactcagcac ccaggacaag taccagatga\\n     2281 aaagaagaat tgcatgtatt ccacatatat tcatgtttga acaaggagtc aaagtttatt\\n     2341 gtaaggataa ggagtctttg ttggtggcct gttaagtaac caaccagggc agtcatgctg\\n     2401 ggtagggaag aaggtgagct ggaggaggaa aagacaaact tggagagcca gacattgaga\\n     2461 ttccattgag gcgttggagg tcacaacgcg gtcaaaaaca tgttgagagg acttagctgc\\n     2521 aaagttgtta actaagtaga aacctaaagg atgaatttta ggatttctcc aggaaatcct\\n     2581 aaaagataac ttctttcagg gagaaaaaac agacccttgc aaagacatga aaggaaatgt\\n     2641 aatttggttt gattggcaga tagttgtgaa gaatgttgga ctgtaaggct gtcgatatcc\\n     2701 tcctcacaga actccccaaa gtacattgta tttgctccct taccgacctg attctcccac\\n     2761 tattcagttc attccttgat gctgttttaa gcaacccctg ctctgtctga cacttttgga\\n     2821 tgctcagtaa atgaggaaag aaggaaggaa agataaaatg gtaaagggct cacacatgtc\\n     2881 ttaacaaaaa tgtccagttc ggctcatttg gctatacttc atggctgctg ctctgccctt\\n     2941 gcatcctcgg ataagctcac tgcccattag aggaaaaagg gtttaattta cctgagtcct\\n     3001 cgagtgaatg taattgttga atcagaacac tatagatatt tagtaacctc cttcagagga\\n     3061 aaaaaaaaaa agtgggggca atgacagaaa ttaaaaaacc agtcgagctt ccacttttca\\n     3121 tttcagaaga aatcaggtgc tctcctctaa ggaccactac tattaacaaa acagagacct\\n     3181 tagaagaatt gtttatttgt tataaatgta taatgttgct attcttgtaa tagtctttct\\n     3241 cgtaccctat aattgttaga agaaattatt ttaagttaat acgttcctac atgcttttct\\n     3301 ttggtttaaa aaaaaaaaaa aaaaaaggaa actctgtgta gaaagtgtcc tgttctgatc\\n     3361 tagtcctgac aggaaacgaa gtataatcaa cttgttatta actgagagag aaaacttagg\\n     3421 aagcagaggg aaataaactg aatctctgag taagaaaact aaatcctatg ataactcatt\\n     3481 cattccttcc tttgtttatt gcaatattca tcataagctt atgatgtgcc aggcactaag\\n     3541 taggcactca ggaaataaca gacgtgtgac gttctgcctt tgtggagcat atgttatagt\\n     3601 gagaaagaca gaatcagttc taacctgatg actaccaacg ttaggcaagg aggaagcagg\\n     3661 tgttaggaag attgttcagg gactgtgcca aagatgaagc ccataatatt tgaaagtgag\\n     3721 tttcttcaat cactttctgt attaaggttc tttctccctg tgttccaccc tcctgcttgt\\n     3781 caccttcact cgtcagctga ccatgttgcc tcctatggtg tgaacttcta ccagtctcac\\n     3841 ggtccctctg gccagtacac ccatgaattt gatggagacg aggagttcta tgtggacctg\\n     3901 gagatgaaag agactgtctg gcagttgcct atgtttagca aatttataag ttttgacccg\\n     3961 cagagtgcac tgagaaatat ggctgtggga aaacacacct tggaattcat gatgagacag\\n     4021 tccaactcta ccgctgccac caatggtatg tgtccaccat tccgcctctc tttactgaaa\\n     4081 ctaatctttc ataccaagtt ttactccctt cttctcaaga gatttccaga tcttctcatg\\n     4141 gtaattgctg aaattttatc atctcccatc tctaaaatca catattccca tgtaatacaa\\n     4201 gggtctttcc attatgtatt aattcctact ttattaaaca tgcccacaga gagaagggca\\n     4261 caggaataaa gcagaggcaa tgtgtcgttg ctcccaagca gaaggtaaat aagacctctt\\n     4321 tgactatcag gtggtgaaat gctggtagga gggctcttcc aggatgtaat gcagaagctc\\n     4381 atggcagagc tattcacact tcacatcagt gctgtttcct caccacagag gttcctgagg\\n     4441 tcacagtgtt ttccaagttt cctgtgacgc tgggtcagcc caacaccctc atctgtcttg\\n     4501 tggacaacat ctttcctcct gtggtcaaca tcacctggct gagcaatggg cactcagtca\\n     4561 cagaaggtgt ttctgagacc agcttcctct ccaagagtga tcattccttc ttcaagatca\\n     4621 gttacctcac cttcctccct tctgctgatg agatttatga ctgcaaggtg gagcactggg\\n     4681 gcctggacga gcctcttctg aaacactggg gtaaggatga gttccaccac ttcatgggtt\\n     4741 tctaataata gacttcactc ttctccctaa gcctggggcc ttgagtcttg cagagccagc\\n     4801 cctccacccc atcccatccc acacacatgc acatgagcac actgcacatt ctgacctcaa\\n     4861 cagctccact ttcacagagc ctgagattcc agcccctatg tcagagctca cagagacttt\\n     4921 ggtctgcgcc ctggggttgt ctgtgggcct catgggcatt gtggtgggca ctgtcttcat\\n     4981 catccaaggc ctgcgttcag ttggtgcttc cagacaccaa gggctcttat gaatcccatc\\n     5041 ctgaaaagga aggtaagatt gagatttgtt ggagctgaaa cctcagtatg agagggagga\\n     5101 aagtgggagg gggttgtgga catgaatgtg gttgaaagtt gtaggcgaat tgggaagtgg\\n     5161 catgatgatc acacaggagg cccctcagac ccatcgatct catgtctgtc ctgttgcagg\\n     5221 tgcatcacca tc\\n//\\n\\n'"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lit_search(email,'hla-dqa2','nucleotide',amplitude=1,save='y')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
