{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 90 resultados em pubmed. Irão ser processados 10 resultados.\n",
      "\n",
      "1. Hepatol Int. 2023 Nov 11. doi: 10.1007/s12072-023-10608-8. Online ahead of \n",
      "print.\n",
      "\n",
      "Proteome-wide Mendelian randomization highlights AIF1 and HLA-DQA2 as targets \n",
      "for primary sclerosing cholangitis.\n",
      "\n",
      "Chen L(1), Zhao Y(1), Li M(1), Lv G(2).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \n",
      "First Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \n",
      "Jilin, China.\n",
      "(2)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \n",
      "First Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \n",
      "Jilin, China. lvgy@jlu.edu.cn.\n",
      "\n",
      "BACKGROUND: Primary sclerosing cholangitis (PSC) is a kind of cholestatic liver \n",
      "disease without effective therapies and its pathogenesis is largely unknown.\n",
      "METHODS: We performed the proteome-wide Mendelian randomization (MR) design to \n",
      "estimate the causal associations of protein levels with PSC risk. Therein, \n",
      "genetic associations with 4,907 plasma protein levels were extracted from a \n",
      "proteome-wide genome-wide association study (GWAS) with 35,559 individuals and \n",
      "those with PSC were obtained from the International PSC Study Group (2,871 cases \n",
      "and 12,019 controls) and the FinnGen study (1,491 cases and 301,383 controls). \n",
      "The colocalization analysis was performed to detect causal variants shared by \n",
      "proteins and PSC. The identified proteins were further enriched in pathways and \n",
      "diseases. A phenome-wide association screening was performed and potential drugs \n",
      "were assessed as well.\n",
      "RESULTS: The results indicated that genetically predicted plasma levels of 14 \n",
      "proteins were positively associated with an increased risk of PSC and 8 proteins \n",
      "were inversely associated with PSC risk in both PSC GWAS data sets, and they all \n",
      "survived in sensitivity analyses. The colocalization indicated that AIF1 \n",
      "(allograft inflammatory factor 1) and HLA-DQA2 (major histocompatibility \n",
      "complex, class II, DQ alpha 2) were shared proteins with PSC, and they should be \n",
      "direct targets for PSC. The phenome-wide screening suggested that variants \n",
      "located at AIF1 or HLA-DQA2 region were closely associated with several \n",
      "autoimmune diseases, such as rheumatoid arthritis, implicating the shared \n",
      "pathogenesis among them.\n",
      "CONCLUSIONS: Our study highly pinpointed two candidate targets (AIF1 and \n",
      "HLA-DQA2) for PSC.\n",
      "\n",
      "© 2023. Asian Pacific Association for the Study of the Liver.\n",
      "\n",
      "DOI: 10.1007/s12072-023-10608-8\n",
      "PMID: 37950809\n",
      "\n",
      "\n",
      "2. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\n",
      " 2023 Oct 10.\n",
      "\n",
      "Genome-wide identification of m(6)A-associated single nucleotide polymorphisms \n",
      "in complex diseases of nervous system.\n",
      "\n",
      "Guo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \n",
      "Guo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\n",
      "\n",
      "Author information:\n",
      "(1)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China.\n",
      "(2)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China.\n",
      "(3)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "jiangyongshuai@hrbmu.edu.cn.\n",
      "(4)College of Bioinformatics Science and Technology, Harbin Medical University, \n",
      "Harbin, China; The EWAS Project, China. Electronic address: \n",
      "zhangmingming@hrbmu.edu.cn.\n",
      "\n",
      "N6-methyladenosine (m6A) is one of the most abundant chemical modifications on \n",
      "RNA and can affect the occurrence and development of diseases. Some studies have \n",
      "shown that the expressions of some m6A-related genes are significantly regulated \n",
      "by single nucleotide variants (SNV). However, the function of m6A-associated \n",
      "single nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \n",
      "(MS), Alzheimer's disease (AD) and Parkinson's disease (PD). Here, we identified \n",
      "the disease-associated m6A-SNPs by integrating genome-wide association study \n",
      "(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \n",
      "between these identified m6A-SNPs and their target genes in eQTL analysis and \n",
      "gene differential expression analysis. Finally, 26 genes corresponding to 20 \n",
      "m6A-SNPs with eQTL signals were identified and differentially expressed \n",
      "(P < 0.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P < 1e-04) were \n",
      "differentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \n",
      "expression of 31 genes were identified. There were 5 HLA genes with eQTL signals \n",
      "(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \n",
      "three diseases. In summary, our study provided new insights into understanding \n",
      "the potential roles of these m6A-SNPs in disease pathogenesis as well as \n",
      "therapeutic target.\n",
      "\n",
      "Copyright © 2023 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.neulet.2023.137513\n",
      "PMID: 37827449 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "3. Br J Cancer. 2023 Oct;129(9):1510-1515. doi: 10.1038/s41416-023-02419-3. Epub \n",
      "2023 Sep 7.\n",
      "\n",
      "Integrating genomics and proteomics data to identify candidate plasma biomarkers \n",
      "for lung cancer risk among European descendants.\n",
      "\n",
      "Yang Y(#)(1), Xu S(#)(2), Jia G(2), Yuan F(2), Ping J(2), Guo X(2), Tao R(3)(4), \n",
      "Shu XO(2), Zheng W(2), Long J(2), Cai Q(5).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Public Health Genomics, Department of Public Health Sciences, UVA \n",
      "Comprehensive Cancer Center, School of Medicine, University of Virginia, \n",
      "Charlottesville, VA, USA. vta8we@virginia.edu.\n",
      "(2)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(3)Department of Biostatistics, Vanderbilt University Medical Center, Nashville, \n",
      "TN, USA.\n",
      "(4)Vanderbilt Genetics Institute, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA.\n",
      "(5)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \n",
      "Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \n",
      "Nashville, TN, USA. qiuyin.cai@vanderbilt.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: Plasma proteins are potential biomarkers for complex diseases. We \n",
      "aimed to identify plasma protein biomarkers for lung cancer.\n",
      "METHODS: We investigated genetically predicted plasma levels of 1130 proteins in \n",
      "association with lung cancer risk among 29,266 cases and 56,450 controls of \n",
      "European descent. For proteins significantly associated with lung cancer risk, \n",
      "we evaluated associations of genetically predicted expression of their coding \n",
      "genes with the risk of lung cancer.\n",
      "RESULTS: Nine proteins were identified with genetically predicted plasma levels \n",
      "significantly associated with overall lung cancer risk at a false discovery rate \n",
      "(FDR) of <0.05. Proteins C2, MICA, AIF1, and CTSH were associated with increased \n",
      "lung cancer risk, while proteins SFTPB, HLA-DQA2, MICB, NRP1, and GMFG were \n",
      "associated with decreased lung cancer risk. Stratified analyses by histological \n",
      "types revealed the cross-subtype consistency of these nine associations and \n",
      "identified an additional protein, ICAM5, significantly associated with lung \n",
      "adenocarcinoma risk (FDR < 0.05). Coding genes of NRP1 and ICAM5 proteins are \n",
      "located at two loci that have never been reported by previous GWAS. Genetically \n",
      "predicted blood levels of genes C2, AIF1, and CTSH were associated with lung \n",
      "cancer risk, in directions consistent with those shown in protein-level \n",
      "analyses.\n",
      "CONCLUSION: Identification of novel plasma protein biomarkers provided new \n",
      "insights into the biology of lung cancer.\n",
      "\n",
      "© 2023. The Author(s), under exclusive licence to Springer Nature Limited.\n",
      "\n",
      "DOI: 10.1038/s41416-023-02419-3\n",
      "PMCID: PMC10628278\n",
      "PMID: 37679517 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "4. Front Oncol. 2023 Jun 23;13:1150098. doi: 10.3389/fonc.2023.1150098.\n",
      "eCollection  2023.\n",
      "\n",
      "Immune microenvironment analysis and novel biomarkers of early-stage lung \n",
      "adenocarcinoma evolution.\n",
      "\n",
      "Liu J(1), Ji Y(2), Weng X(3), Shao W(2), Zhao J(2), Chen H(4), Shen L(4), Wang \n",
      "F(4), Meng Q(4), Wu X(4), Wang X(4), Ou Q(4), Ke H(3).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Chemotherapy, Affiliated Hospital of Nantong University, \n",
      "Nantong, China.\n",
      "(2)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \n",
      "Medical School of Nantong University, Nantong, China.\n",
      "(3)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \n",
      "Nantong, China.\n",
      "(4)Geneseeq Research Institute, Nanjing Geneseeq Technology Inc., \n",
      "Nanjing, China.\n",
      "\n",
      "BACKGROUND: Lung cancer is the deadliest and most diagnosed type of cancer \n",
      "worldwide. The 5-year survival rate of lung adenocarcinoma (LUAD) dropped \n",
      "significantly when tumor stages advanced. Patients who received surgically \n",
      "resecting at the pre-invasive stage had a 5-year survival rate of nearly 100%. \n",
      "However, the study on the differences in gene expression profiles and immune \n",
      "microenvironment among pre-invasive LUAD patients is still lacking.\n",
      "METHODS: In this study, the gene expression profiles of three pre-invasive LUAD \n",
      "stages were compared using the RNA-sequencing data of 10 adenocarcinoma in situ \n",
      "(AIS) samples, 12 minimally invasive adenocarcinoma (MIA) samples, and 10 \n",
      "invasive adenocarcinoma (IAC) samples.\n",
      "RESULTS: The high expression levels of PTGFRN (Hazard Ratio [HR] = 1.45; 95% \n",
      "Confidence Interval [CI]: 1.08-1.94; log-rank P = 0.013) and SPP1 (HR = 1.44; \n",
      "95% CI: 1.07-1.93; log-rank P = 0.015) were identified to be associated with \n",
      "LUAD prognosis. Moreover, the early LUAD invasion was accompanied by the \n",
      "enhancement of antigen presentation ability, reflected by the increase of \n",
      "myeloid dendritic cells infiltration rate (Cuzick test P < 0.01) and the \n",
      "upregulation of seven important genes participating in the antigen presentation, \n",
      "including HLA-A (Cuzick test P = 0.03), MICA (Cuzick test P = 0.01), MICB \n",
      "(Cuzick test P = 0.01), HLA-DPA1 (Cuzick test P = 0.04), HLA-DQA2 (Cuzick test P \n",
      "< 0.01), HLA-DQB1 (Cuzick test P = 0.03), and HLA-DQB2 (Cuzick test P < 0.01). \n",
      "However, the tumor-killing ability of the immune system was inhibited during \n",
      "this process, as there were no rising cytotoxic T cell activity (Cuzick test P = \n",
      "0.20) and no increasing expression in genes encoding cytotoxic proteins.\n",
      "CONCLUSION: In all, our research elucidated the changes in the immune \n",
      "microenvironment during early-stage LUAD evolution and may provide a theoretical \n",
      "basis for developing novel early-stage lung cancer therapeutic targets.\n",
      "\n",
      "Copyright © 2023 Liu, Ji, Weng, Shao, Zhao, Chen, Shen, Wang, Meng, Wu, Wang, Ou \n",
      "and Ke.\n",
      "\n",
      "DOI: 10.3389/fonc.2023.1150098\n",
      "PMCID: PMC10328385\n",
      "PMID: 37427097\n",
      "\n",
      "Conflict of interest statement: HC, LS, FW, QM, XWU, XWA, and QO are employees \n",
      "of Nanjing Geneseeq Technology Inc., China. The remaining authors declare that \n",
      "the research was conducted in the absence of any commercial or financial \n",
      "relationships that could be construed as a potential conflict of interest.\n",
      "\n",
      "\n",
      "5. Hereditas. 2023 May 31;160(1):27. doi: 10.1186/s41065-023-00289-6.\n",
      "\n",
      "NPM3 as a novel oncogenic factor and poor prognostic marker contributes to cell \n",
      "proliferation and migration in lung adenocarcinoma.\n",
      "\n",
      "Wei S(1)(2), Xing J(2)(3), Lu K(4), Wang K(1), Yu W(5).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Respiratory and Critical Care Medicine, The Fourth Affiliated \n",
      "Hospital, School of Medicine, Zhejiang University, Yiwu, 322000, Zhejiang, \n",
      "People's Republic of China.\n",
      "(2)Department of Respiratory and Critical Care Medicine, The Affiliated People's \n",
      "Hospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \n",
      "Baizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China.\n",
      "(3)Ningbo University School of Medicine, Zhejiang Province, Ningbo, People's \n",
      "Republic of China.\n",
      "(4)Department of Urology, The Affiliated First Hospital of Ningbo University \n",
      "(Ningbo First Hospital), No.59, Liuting Street, Ningbo, 315010, Zhejiang, \n",
      "People's Republic of China.\n",
      "(5)Department of Respiratory and Critical Care Medicine, The Affiliated People's \n",
      "Hospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \n",
      "Baizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China. \n",
      "nbyuwanjun@163.com.\n",
      "\n",
      "BACKGROUND: Lung cancer is the leading cause of cancer-related deaths worldwide, \n",
      "and despite recent advances in targeted therapies and immunotherapies, the \n",
      "clinical benefit remains limited. Therefore, there is an urgent need to further \n",
      "investigate the molecular mechanisms underlying lung cancer. The aim of this \n",
      "study was to investigate the expression and function of NPM3 in the tumor \n",
      "microenvironment of lung adenocarcinoma (LUAD).\n",
      "METHODS: We utilized bioinformatics tools and databases, including UALCAN, \n",
      "GEPIA2, HPA, and Sangerbox, to analyze NPM3 expression in LUAD samples and its \n",
      "association with prognosis and mutational landscape. NPM3 expression in various \n",
      "cell types was assessed at the single cell level using the TISCH database. We \n",
      "also used algorithms such as TIMER and EPIC to explore the crosstalk between \n",
      "NPM3 expression and immune features. KEGG enrichment analysis was performed to \n",
      "identify potential signaling pathways of NPM3. Finally, we employed siRNA \n",
      "knockdown strategy to investigate the effect of NPM3 on LUAD cell proliferation \n",
      "and migration in vitro.\n",
      "RESULTS: NPM3 was significantly upregulated in LUAD tissues and was strongly \n",
      "associated with poor prognosis and TP53 gene mutations. Single-cell sequencing \n",
      "analysis revealed that NPM3 was expressed in immune cells (dendritic cells and \n",
      "monocytes/macrophages) in the tumor microenvironment. Moreover, NPM3 expression \n",
      "was negatively associated with immune B cell and CD4 T cell infiltration, as \n",
      "well as with several immune-related genes (including CCL22, CXCR2, CX3CR1, CCR6, \n",
      "HLA-DOA, HLA-DQA2). KEGG enrichment analysis indicated that NPM3 expression was \n",
      "associated with cell cycle, CAMs, and NSCLC pathway genes. Finally, in vitro \n",
      "experiments showed that NPM3 knockdown inhibited LUAD cell proliferation and \n",
      "migration in NCI-H1299 and SPC-A1 cells, and suppressed the expression of CCNA2 \n",
      "and MAD2L1.\n",
      "CONCLUSION: Elevated NPM3 expression predicts poor clinical outcome and an \n",
      "immunosuppressive microenvironment in LUAD tissues. NPM3 promotes LUAD \n",
      "progression by promoting cell proliferation and migration, and targeting NPM3 \n",
      "may represent a novel therapeutic strategy for LUAD.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s41065-023-00289-6\n",
      "PMCID: PMC10230701\n",
      "PMID: 37254219 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors claim that there is no possible \n",
      "conflict of interest.\n",
      "\n",
      "\n",
      "6. Rheumatology (Oxford). 2023 Sep 1;62(9):3213-3218. doi: \n",
      "10.1093/rheumatology/kead152.\n",
      "\n",
      "Stratified genetic analysis reveals sex differences in MPO-ANCA-associated \n",
      "vasculitis.\n",
      "\n",
      "Ekman D(1), Sennblad B(2), Knight A(3), Karlsson Å(4), Rantapää-Dahlqvist S(5), \n",
      "Berglin E(5), Stegmayr B(5), Baslund B(6), Palm Ø(7), Haukeland H(8), Gunnarsson \n",
      "I(9)(10), Bruchfeld A(11)(12), Segelmark M(13), Ohlsson S(13), Mohammad \n",
      "AJ(14)(15), Svärd A(16), Pullerits R(17)(18), Herlitz H(19), Söderbergh A(20), \n",
      "Omdal R(21)(22), Jonsson R(23), Rönnblom L(3), Eriksson P(24), Lindblad-Toh \n",
      "K(4)(25), Dahlqvist J(3)(4)(25).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Biochemistry and Biophysics, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Stockholm University, \n",
      "Sweden.\n",
      "(2)Department of Cell and Molecular Biology, National Bioinformatics \n",
      "Infrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala, \n",
      "Sweden.\n",
      "(3)Department of Medical Sciences, Uppsala University, Uppsala, Sweden.\n",
      "(4)Science for Life Laboratory, Department of Medical Biochemistry and \n",
      "Microbiology, Uppsala University, Uppsala, Sweden.\n",
      "(5)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \n",
      "Sweden.\n",
      "(6)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \n",
      "Diseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\n",
      "(7)Department of Rheumatology, Oslo University Hospital, Oslo, Norway.\n",
      "(8)Department of Rheumatology, Martina Hansens Hospital, Gjettum, Norway.\n",
      "(9)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \n",
      "Stockholm, Sweden.\n",
      "(10)Unit of Rheumatology, Karolinska University Hospital, Stockholm, Sweden.\n",
      "(11)Department of Health, Medicine and Caring Sciences, Linköping University, \n",
      "Linköping, Sweden.\n",
      "(12)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \n",
      "Karolinska Institutet, Stockholm, Sweden.\n",
      "(13)Department of Clinical Sciences, Division of Nephrology, Lund University and \n",
      "Skåne University Hospital, Lund, Sweden.\n",
      "(14)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \n",
      "University Hospital, Lund University, Lund, Sweden.\n",
      "(15)Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "(16)Center for Clinical Research Dalarna, Uppsala University, Uppsala, Sweden.\n",
      "(17)Department of Rheumatology and Inflammation Research, Institution of \n",
      "Medicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden.\n",
      "(18)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \n",
      "University Hospital, Gothenburg, Sweden.\n",
      "(19)Department of Molecular and Clinical Medicine/Nephrology, Institute of \n",
      "Medicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden.\n",
      "(20)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\n",
      "(21)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \n",
      "University Hospital, Stavanger, Norway.\n",
      "(22)Department of Clinical Science, University of Bergen, Bergen, Norway.\n",
      "(23)Broegelmann Research Laboratory, Department of Clinical Science, University \n",
      "of Bergen, Bergen, Norway.\n",
      "(24)Department of Biomedical and Clinical Sciences, Division of Inflammation and \n",
      "Infection, Linköping University, Linköping, Sweden.\n",
      "(25)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\n",
      "\n",
      "OBJECTIVE: To identify and genetically characterize subgroups of patients with \n",
      "ANCA-associated vasculitides (AAV) based on sex and ANCA subtype.\n",
      "METHODS: A previously established SNP dataset derived from DNA sequencing of \n",
      "1853 genes and genotyping of 1088 Scandinavian cases with AAV and 1589 controls \n",
      "was stratified for sex and ANCA subtype and analysed for association with five \n",
      "top AAV SNPs. rs9274619, a lead variant at the HLA-DQB1/HLA-DQA2 locus \n",
      "previously associated with AAV positive for myeloperoxidase (MPO)-ANCA, was \n",
      "analysed for association with the cumulative disease involvement of ten \n",
      "different organ systems.\n",
      "RESULTS: rs9274619 showed a significantly stronger association to \n",
      "MPO-ANCA-positive females than males [P = 2.0 × 10-4, OR = 2.3 (95% CI 1.5, \n",
      "3.5)], whereas proteinase 3 (PR3)-ANCA-associated variants rs1042335, rs9277341 \n",
      "(HLA-DPB1/A1) and rs28929474 (SERPINA1) were equally associated with females and \n",
      "males with PR3-ANCA. In MPO-ANCA-positive cases, carriers of the rs9274619 risk \n",
      "allele were more prone to disease engagement of eyes [P = 0.021, OR = 11 (95% CI \n",
      "2.2, 205)] but less prone to pulmonary involvement [P = 0.026, OR = 0.52 (95% CI \n",
      "0.30, 0.92)]. Moreover, AAV with both MPO-ANCA and PR3-ANCA was associated with \n",
      "the PR3-ANCA lead SNP rs1042335 [P = 0.0015, OR = 0.091 (95% CI 0.0022, 0.55)] \n",
      "but not with rs9274619.\n",
      "CONCLUSIONS: Females and males with MPO-ANCA-positive AAV differ in genetic \n",
      "predisposition to disease, suggesting at least partially distinct disease \n",
      "mechanisms between the sexes. Double ANCA-positive AAV cases are genetically \n",
      "similar to PR3-ANCA-positive cases, providing clues to the clinical follow-up \n",
      "and treatment of these patients.\n",
      "\n",
      "© The Author(s) 2023. Published by Oxford University Press on behalf of the \n",
      "British Society for Rheumatology.\n",
      "\n",
      "DOI: 10.1093/rheumatology/kead152\n",
      "PMCID: PMC10473270\n",
      "PMID: 37004177 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "7. BMC Genomics. 2023 Mar 27;24(1):153. doi: 10.1186/s12864-023-09227-2.\n",
      "\n",
      "Genome-wide identification of RNA modification-related single nucleotide \n",
      "polymorphisms associated with rheumatoid arthritis.\n",
      "\n",
      "Wang M(#)(1)(2), Wu J(#)(1)(2), Lei S(1)(2), Mo X(3)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China.\n",
      "(2)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China.\n",
      "(3)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \n",
      "School of Public Health, Medical College of Soochow University, 199 Renai Road, \n",
      "Suzhou, Jiangsu, 215123, People's Republic of China. xbmo@suda.edu.cn.\n",
      "(4)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \n",
      "Diseases, School of Public Health, Medical College of Soochow University, \n",
      "Suzhou, China. xbmo@suda.edu.cn.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: RNA modification plays important roles in many biological processes, \n",
      "such as gene expression control. The aim of this study was to identify single \n",
      "nucleotide polymorphisms related to RNA modification (RNAm-SNPs) for rheumatoid \n",
      "arthritis (RA) as putative functional variants.\n",
      "METHODS: We examined the association of RNAm-SNPs with RA in summary data from a \n",
      "genome-wide association study of 19,234 RA cases and 61,565 controls. We \n",
      "performed eQTL and pQTL analyses for the RNAm-SNPs to find associated gene \n",
      "expression and protein levels. Furthermore, we examined the associations of gene \n",
      "expression and circulating protein levels with RA using two-sample Mendelian \n",
      "randomization analysis methods.\n",
      "RESULTS: A total of 160 RNAm-SNPs related to m6A, m1A, A-to-I, m7G, m5C, m5U and \n",
      "m6Am modifications were identified to be significantly associated with RA. These \n",
      "RNAm-SNPs were located in 62 protein-coding genes, which were significantly \n",
      "enriched in immune-related pathways. RNAm-SNPs in important RA susceptibility \n",
      "genes, such as PADI2, SPRED2, PLCL2, HLA-A, HLA-B, HLA-DRB1, HLA-DPB1, TRAF1 and \n",
      "TXNDC11, were identified. Most of these RNAm-SNPs showed eQTL effects, and the \n",
      "expression levels of 26 of the modifiable genes (e.g., PADI2, TRAF1, HLA-A, \n",
      "HLA-DRB1, HLA-DPB1 and HLA-B) in blood cells were associated with RA. \n",
      "Circulating protein levels, such as CFB, GZMA, HLA-DQA2, IL21, LRPAP1 and TFF3, \n",
      "were affected by RNAm-SNPs and were associated with RA.\n",
      "CONCLUSION: The present study identified RNAm-SNPs in the reported RA \n",
      "susceptibility genes and suggested that RNAm-SNPs may affect RA risk by \n",
      "affecting the expression levels of corresponding genes and proteins.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12864-023-09227-2\n",
      "PMCID: PMC10045113\n",
      "PMID: 36973646 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare that they have no competing \n",
      "interests.\n",
      "\n",
      "\n",
      "8. Yi Chuan. 2023 Jan 20;45(1):52-66. doi: 10.16288/j.yczz.22-293.\n",
      "\n",
      "Application of transcriptome in time analysis and donor characterization in \n",
      "blood samples.\n",
      "\n",
      "Zhang J(1)(2), Liu KH(2), Zhang Y(2), Hao JP(2), Zhang GF(2), Xu XY(2), Chang \n",
      "J(2), Liu XP(3), Yang XY(2), Ye J(1)(2).\n",
      "\n",
      "Author information:\n",
      "(1)1. People's Public Security University of China, Beijing 100038, China.\n",
      "(2)2. Institute of Forensic Science Ministry of Public Security, Beijing 100038, \n",
      "China.\n",
      "(3)3. Ruijin Public Security Bureau, Ruijin 342500, China.\n",
      "\n",
      "As an effective supplement to the current forensic DNA typing and one of the \n",
      "research hotpots in forensic science, the in-depth mining and characterization \n",
      "of biological evidence can provide rich and reliable clues for case \n",
      "investigation. In this study, the time-dependent variations of transcriptome \n",
      "were confirmed in in vitro blood samples within 0-168 days and a random forest \n",
      "model was established to realize the classification of blood samples with \n",
      "different TSD (time since deposition). Meanwhile, significant differences were \n",
      "observed in the transcripts of blood samples with different smoking habits and \n",
      "genders within a certain time period. HLA-DRB1, HLA-DQB1 and HLA-DQA2 were \n",
      "identified as markers for smoking habit identification, while the transcripts \n",
      "for RPS4Y1 and EIF1AY from the non-recombining region of the Y chromosome (NRY) \n",
      "were identified as markers for male sex identification. Thus, this study \n",
      "provides a theoretical foundation and experimental strategy for establishing a \n",
      "transcriptome-based method for characterizing blood sample retention time and \n",
      "donor characteristics in the field of forensic investigation.\n",
      "\n",
      "Publisher: \n",
      "案件现场生物物证信息深度挖掘与刻画，可以为案件侦查、涉案人员查找提供丰富可靠线索，是当前法医DNA检验的有效补充，也是国内外法庭科学的研究热点之一。本研究以血液样本为研究对象，证实了0~168天内离体血液样本转录组变化的时间相关性，并建立随机森林模型实现不同离体时间血液样本分类。同时，进一步证实相同离体时间段内，不同吸烟习惯和不同性别供体来源的血液样本转录本具有显著差异，HLA-DRB1、HLA-DQB1和HLA-DQA2可以作为供体吸烟习惯判别标志，Y染色体非重组区(non-recombining \n",
      "Y, \n",
      "NRY)的转录本RPS4Y1和EIF1AY可以作为供体性别特征判别标志。本研究为法庭科学领域建立基于转录组分析的血液样本遗留时间和供体特征刻画方法提供了理论基础和实验依据。.\n",
      "\n",
      "DOI: 10.16288/j.yczz.22-293\n",
      "PMID: 36927638 [Indexed for MEDLINE]\n",
      "\n",
      "\n",
      "9. Pediatr Rheumatol Online J. 2023 Feb 15;21(1):17. doi: \n",
      "10.1186/s12969-023-00787-x.\n",
      "\n",
      "Profiling the peripheral immune response to ex vivo TNF stimulation in untreated \n",
      "juvenile idiopathic arthritis using single cell RNA sequencing.\n",
      "\n",
      "Imbach KJ(#)(1), Treadway NJ(#)(2), Prahalad V(2), Kosters A(3), Arafat D(1), \n",
      "Duan M(1), Gergely T(2), Ponder LA(4), Chandrakasan S(4)(5), Ghosn EEB(2)(3)(4), \n",
      "Prahalad S(#)(6)(7)(8), Gibson G(#)(1)(4).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Integrative Genomics, School of Biological Sciences, Georgia \n",
      "Institute of Technology, Atlanta, GA, 30332, USA.\n",
      "(2)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \n",
      "30223, USA.\n",
      "(3)Lowance Center for Human Immunology, Division of Immunology, Department of \n",
      "Medicine, Emory University School of Medicine, Atlanta, GA, 30223, USA.\n",
      "(4)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \n",
      "Atlanta, GA, 30223, USA.\n",
      "(5)Aflac Cancer and Blood Disorders Center, Department of Pediatrics, Children's \n",
      "Healthcare of Atlanta, Emory University School of Medicine, Atlanta, GA, 30223, \n",
      "USA.\n",
      "(6)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \n",
      "30223, USA. sprahal@emory.edu.\n",
      "(7)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \n",
      "Atlanta, GA, 30223, USA. sprahal@emory.edu.\n",
      "(8)Department of Human Genetics, Emory University School of Medicine, Atlanta, \n",
      "GA, 30223, USA. sprahal@emory.edu.\n",
      "(#)Contributed equally\n",
      "\n",
      "BACKGROUND: Juvenile Idiopathic Arthritis (JIA) is an autoimmune disease with a \n",
      "heterogenous clinical presentation and unpredictable response to available \n",
      "therapies. This personalized transcriptomics study sought proof-of-concept for \n",
      "single-cell RNA sequencing to characterize patient-specific immune profiles.\n",
      "METHODS: Whole blood samples from six untreated children, newly diagnosed with \n",
      "JIA, and two healthy controls were cultured for 24 h with or without ex vivo TNF \n",
      "stimulation and subjected to scRNAseq to examine cellular populations and \n",
      "transcript expression in PBMCs. A novel analytical pipeline, scPool, was \n",
      "developed wherein cells are first pooled into pseudocells prior to expression \n",
      "analysis, facilitating variance partitioning of the effects of TNF stimulus, JIA \n",
      "disease status, and individual donor.\n",
      "RESULTS: Seventeen robust immune cell-types were identified, the abundance of \n",
      "which was significantly affected by TNF stimulus, which resulted in notable \n",
      "elevation of memory CD8 + T-cells and NK56 cells, but down-regulation of naïve \n",
      "B-cell proportions. Memory CD8 + and CD4 + T-cells were also both reduced in the \n",
      "JIA cases relative to two controls. Significant differential expression \n",
      "responses to TNF stimulus were also characterized, with monocytes showing more \n",
      "transcriptional shifts than T-lymphocyte subsets, while the B-cell response was \n",
      "more limited. We also show that donor variability exceeds the small degree of \n",
      "possible intrinsic differentiation between JIA and control profiles. An \n",
      "incidental finding of interest was association of HLA-DQA2 and HLA-DRB5 \n",
      "expression with JIA status.\n",
      "CONCLUSIONS: These results support the development of personalized \n",
      "immune-profiling combined with ex-vivo immune stimulation for evaluation of \n",
      "patient-specific modes of immune cell activity in autoimmune rheumatic disease.\n",
      "\n",
      "© 2023. The Author(s).\n",
      "\n",
      "DOI: 10.1186/s12969-023-00787-x\n",
      "PMCID: PMC9929251\n",
      "PMID: 36793127 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: Dr. Prahalad has served on a macrophage \n",
      "activation syndrome adjudication committee for Novartis, but this had no bearing \n",
      "on the work described in this manuscript. None of the other authors have any \n",
      "competing interests in the manuscript.\n",
      "\n",
      "\n",
      "10. Epilepsia. 2023 Jan;64(1):218-230. doi: 10.1111/epi.17457. Epub 2022 Nov 16.\n",
      "\n",
      "Brain molecular mechanisms in Rasmussen encephalitis.\n",
      "\n",
      "Leitner DF(1)(2), Lin Z(3), Sawaged Z(3), Kanshin E(4), Friedman D(1), Devore \n",
      "S(1), Ueberheide B(2)(4)(5), Chang JW(6), Mathern GW(6), Anink JJ(7), Aronica \n",
      "E(7)(8), Wisniewski T(2)(9)(10), Devinsky O(1).\n",
      "\n",
      "Author information:\n",
      "(1)Comprehensive Epilepsy Center, NYU Grossman School of Medicine, New York, New \n",
      "York, USA.\n",
      "(2)Center for Cognitive Neurology, Department of Neurology, NYU Grossman School \n",
      "of Medicine, New York, New York, USA.\n",
      "(3)Applied Bioinformatics Laboratories, NYU Grossman School of Medicine, New \n",
      "York, New York, USA.\n",
      "(4)Proteomics Laboratory, Division of Advanced Research Technologies, NYU \n",
      "Grossman School of Medicine, New York, New York, USA.\n",
      "(5)Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of \n",
      "Medicine, New York, New York, USA.\n",
      "(6)Departments of Neurosurgery and Psychiatry, David Geffen School of Medicine, \n",
      "University of California, Los Angeles, Los Angeles, California, USA.\n",
      "(7)Department of (Neuro) Pathology, Amsterdam University Medical Center, \n",
      "University of Amsterdam, Amsterdam Neuroscience, Amsterdam, the Netherlands.\n",
      "(8)Epilepsy Institutes of the Netherlands Foundation, Heemstede, the \n",
      "Netherlands.\n",
      "(9)Department of Pathology, NYU Grossman School of Medicine, New York, New York, \n",
      "USA.\n",
      "(10)Department of Psychiatry, NYU Grossman School of Medicine, New York, New \n",
      "York, USA.\n",
      "\n",
      "OBJECTIVE: This study was undertaken to identify molecular mechanisms in brain \n",
      "tissue of Rasmussen encephalitis (RE) when compared to people with non-RE \n",
      "epilepsy (PWE) and control cases using whole exome sequencing (WES), RNAseq, and \n",
      "proteomics.\n",
      "METHODS: Frozen brain tissue (ages = 2-19 years) was obtained from control \n",
      "autopsy (n = 14), surgical PWE (n = 10), and surgical RE cases (n = 27). We \n",
      "evaluated WES variants in RE associated with epilepsy, seizures, RE, and human \n",
      "leukocyte antigens (HLAs). Differential expression was evaluated by RNAseq \n",
      "(adjusted p < .05) and label-free quantitative mass spectrometry (false \n",
      "discovery rate < 5%) in the three groups.\n",
      "RESULTS: WES revealed no common pathogenic variants in RE, but several rare and \n",
      "likely deleterious variants of unknown significance (VUS; ANGPTL7/MTOR, SCN1A, \n",
      "FCGR3B, MTOR) and more common HLA VUS in >25% of RE cases (HLA-DRB1, HLA-DQA2), \n",
      "all with allele frequency < 5% in the general population. RNAseq in RE versus \n",
      "PWE (1516 altered transcripts) revealed significant activation of crosstalk \n",
      "between dendritic and natural killer cells (p = 7.94 × 10-6 , z = 2.65), in RE \n",
      "versus control (7466 transcripts) neuroinflammation signaling activation \n",
      "(p = 6.31 × 10-13 , z = 5.07), and in PWE versus control (945 transcripts) \n",
      "phagosome formation activation (p = 2.00 × 10-13 , z = 5.61). Proteomics \n",
      "detected fewer altered targets.\n",
      "SIGNIFICANCE: In RE, we identified activated immune signaling pathways and \n",
      "immune cell type annotation enrichment that suggest roles of the innate and \n",
      "adaptive immune responses, as well as HLA variants that may increase \n",
      "vulnerability to RE. Follow-up studies could evaluate cell type density and \n",
      "subregional localization associated with top targets, clinical history \n",
      "(neuropathology, disease duration), and whether modulating crosstalk between \n",
      "dendritic and natural killer cells may limit disease progression.\n",
      "\n",
      "© 2022 International League Against Epilepsy.\n",
      "\n",
      "DOI: 10.1111/epi.17457\n",
      "PMCID: PMC9852002\n",
      "PMID: 36336987 [Indexed for MEDLINE]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1. Hepatol Int. 2023 Nov 11. doi: 10.1007/s12072-023-10608-8. Online ahead of \\nprint.\\n\\nProteome-wide Mendelian randomization highlights AIF1 and HLA-DQA2 as targets \\nfor primary sclerosing cholangitis.\\n\\nChen L(1), Zhao Y(1), Li M(1), Lv G(2).\\n\\nAuthor information:\\n(1)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \\nFirst Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \\nJilin, China.\\n(2)Department of Hepatobiliary and Pancreatic Surgery, General Surgery Center, \\nFirst Hospital of Jilin University, No. 1 Xinmin Street, Changchun, 130021, \\nJilin, China. lvgy@jlu.edu.cn.\\n\\nBACKGROUND: Primary sclerosing cholangitis (PSC) is a kind of cholestatic liver \\ndisease without effective therapies and its pathogenesis is largely unknown.\\nMETHODS: We performed the proteome-wide Mendelian randomization (MR) design to \\nestimate the causal associations of protein levels with PSC risk. Therein, \\ngenetic associations with 4,907 plasma protein levels were extracted from a \\nproteome-wide genome-wide association study (GWAS) with 35,559 individuals and \\nthose with PSC were obtained from the International PSC Study Group (2,871 cases \\nand 12,019 controls) and the FinnGen study (1,491 cases and 301,383 controls). \\nThe colocalization analysis was performed to detect causal variants shared by \\nproteins and PSC. The identified proteins were further enriched in pathways and \\ndiseases. A phenome-wide association screening was performed and potential drugs \\nwere assessed as well.\\nRESULTS: The results indicated that genetically predicted plasma levels of 14 \\nproteins were positively associated with an increased risk of PSC and 8 proteins \\nwere inversely associated with PSC risk in both PSC GWAS data sets, and they all \\nsurvived in sensitivity analyses. The colocalization indicated that AIF1 \\n(allograft inflammatory factor 1) and HLA-DQA2 (major histocompatibility \\ncomplex, class II, DQ alpha 2) were shared proteins with PSC, and they should be \\ndirect targets for PSC. The phenome-wide screening suggested that variants \\nlocated at AIF1 or HLA-DQA2 region were closely associated with several \\nautoimmune diseases, such as rheumatoid arthritis, implicating the shared \\npathogenesis among them.\\nCONCLUSIONS: Our study highly pinpointed two candidate targets (AIF1 and \\nHLA-DQA2) for PSC.\\n\\n© 2023. Asian Pacific Association for the Study of the Liver.\\n\\nDOI: 10.1007/s12072-023-10608-8\\nPMID: 37950809\\n\\n\\n2. Neurosci Lett. 2023 Nov 20;817:137513. doi: 10.1016/j.neulet.2023.137513. Epub\\n 2023 Oct 10.\\n\\nGenome-wide identification of m(6)A-associated single nucleotide polymorphisms \\nin complex diseases of nervous system.\\n\\nGuo F(1), Kang J(1), Xu J(1), Wei S(2), Tao J(1), Dong Y(1), Ma Y(1), Tian H(2), \\nGuo X(2), Bi S(2), Zhang C(2), Lv H(2), Shang Z(2), Jiang Y(3), Zhang M(4).\\n\\nAuthor information:\\n(1)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China.\\n(2)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China.\\n(3)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\njiangyongshuai@hrbmu.edu.cn.\\n(4)College of Bioinformatics Science and Technology, Harbin Medical University, \\nHarbin, China; The EWAS Project, China. Electronic address: \\nzhangmingming@hrbmu.edu.cn.\\n\\nN6-methyladenosine (m6A) is one of the most abundant chemical modifications on \\nRNA and can affect the occurrence and development of diseases. Some studies have \\nshown that the expressions of some m6A-related genes are significantly regulated \\nby single nucleotide variants (SNV). However, the function of m6A-associated \\nsingle nucleotide polymorphisms (m6A-SNP) remains unclear in multiple sclerosis \\n(MS), Alzheimer's disease (AD) and Parkinson's disease (PD). Here, we identified \\nthe disease-associated m6A-SNPs by integrating genome-wide association study \\n(GWAS) and m6A-SNPs from the RMVar database, and confirmed the relationship \\nbetween these identified m6A-SNPs and their target genes in eQTL analysis and \\ngene differential expression analysis. Finally, 26 genes corresponding to 20 \\nm6A-SNPs with eQTL signals were identified and differentially expressed \\n(P\\xa0<\\xa00.05) in MS, 15 genes corresponding to 12 m6A-SNPs (P\\xa0<\\xa01e-04) were \\ndifferentially expressed in AD, and 27 PD-associated m6A-SNPs that regulated the \\nexpression of 31 genes were identified. There were 5 HLA genes with eQTL signals \\n(HLA-DQB1, HLA-DRB1, HLA-DQA1, HLA-DQA2 and HLA-DQB1-AS1) to be detected in the \\nthree diseases. In summary, our study provided new insights into understanding \\nthe potential roles of these m6A-SNPs in disease pathogenesis as well as \\ntherapeutic target.\\n\\nCopyright © 2023 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.neulet.2023.137513\\nPMID: 37827449 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n3. Br J Cancer. 2023 Oct;129(9):1510-1515. doi: 10.1038/s41416-023-02419-3. Epub \\n2023 Sep 7.\\n\\nIntegrating genomics and proteomics data to identify candidate plasma biomarkers \\nfor lung cancer risk among European descendants.\\n\\nYang Y(#)(1), Xu S(#)(2), Jia G(2), Yuan F(2), Ping J(2), Guo X(2), Tao R(3)(4), \\nShu XO(2), Zheng W(2), Long J(2), Cai Q(5).\\n\\nAuthor information:\\n(1)Center for Public Health Genomics, Department of Public Health Sciences, UVA \\nComprehensive Cancer Center, School of Medicine, University of Virginia, \\nCharlottesville, VA, USA. vta8we@virginia.edu.\\n(2)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \\nCenter, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \\nNashville, TN, USA.\\n(3)Department of Biostatistics, Vanderbilt University Medical Center, Nashville, \\nTN, USA.\\n(4)Vanderbilt Genetics Institute, Vanderbilt University Medical Center, \\nNashville, TN, USA.\\n(5)Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology \\nCenter, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, \\nNashville, TN, USA. qiuyin.cai@vanderbilt.edu.\\n(#)Contributed equally\\n\\nBACKGROUND: Plasma proteins are potential biomarkers for complex diseases. We \\naimed to identify plasma protein biomarkers for lung cancer.\\nMETHODS: We investigated genetically predicted plasma levels of 1130 proteins in \\nassociation with lung cancer risk among 29,266 cases and 56,450 controls of \\nEuropean descent. For proteins significantly associated with lung cancer risk, \\nwe evaluated associations of genetically predicted expression of their coding \\ngenes with the risk of lung cancer.\\nRESULTS: Nine proteins were identified with genetically predicted plasma levels \\nsignificantly associated with overall lung cancer risk at a\\xa0false discovery rate \\n(FDR) of <0.05. Proteins C2, MICA, AIF1, and CTSH were associated with increased \\nlung cancer risk, while proteins SFTPB, HLA-DQA2, MICB, NRP1, and GMFG were \\nassociated with decreased lung cancer risk. Stratified analyses by histological \\ntypes revealed the cross-subtype consistency of these nine associations and \\nidentified an additional protein, ICAM5, significantly associated with lung \\nadenocarcinoma risk (FDR\\u2009<\\u20090.05). Coding genes of NRP1 and ICAM5 proteins\\xa0are \\nlocated at two loci that have never been reported by previous GWAS. Genetically \\npredicted blood levels of genes C2, AIF1, and CTSH were associated with lung \\ncancer risk, in directions consistent with those shown in protein-level \\nanalyses.\\nCONCLUSION: Identification of novel plasma protein biomarkers provided new \\ninsights into the biology of lung cancer.\\n\\n© 2023. The Author(s), under exclusive licence to Springer Nature Limited.\\n\\nDOI: 10.1038/s41416-023-02419-3\\nPMCID: PMC10628278\\nPMID: 37679517 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n4. Front Oncol. 2023 Jun 23;13:1150098. doi: 10.3389/fonc.2023.1150098.\\neCollection  2023.\\n\\nImmune microenvironment analysis and novel biomarkers of early-stage lung \\nadenocarcinoma evolution.\\n\\nLiu J(1), Ji Y(2), Weng X(3), Shao W(2), Zhao J(2), Chen H(4), Shen L(4), Wang \\nF(4), Meng Q(4), Wu X(4), Wang X(4), Ou Q(4), Ke H(3).\\n\\nAuthor information:\\n(1)Department of Chemotherapy, Affiliated Hospital of Nantong University, \\nNantong,\\xa0China.\\n(2)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \\nMedical School of Nantong University, Nantong,\\xa0China.\\n(3)Department of Thoracic Surgery, Affiliated Hospital of Nantong University, \\nNantong,\\xa0China.\\n(4)Geneseeq Research Institute, Nanjing Geneseeq Technology Inc., \\nNanjing,\\xa0China.\\n\\nBACKGROUND: Lung cancer is the deadliest and most diagnosed type of cancer \\nworldwide. The 5-year survival rate of lung adenocarcinoma (LUAD) dropped \\nsignificantly when tumor stages advanced. Patients who received surgically \\nresecting at the pre-invasive stage had a 5-year survival rate of nearly 100%. \\nHowever, the study on the differences in gene expression profiles and immune \\nmicroenvironment among pre-invasive LUAD patients is still lacking.\\nMETHODS: In this study, the gene expression profiles of three pre-invasive LUAD \\nstages were compared using the RNA-sequencing data of 10 adenocarcinoma in situ \\n(AIS) samples, 12 minimally invasive adenocarcinoma (MIA) samples, and 10 \\ninvasive adenocarcinoma (IAC) samples.\\nRESULTS: The high expression levels of PTGFRN (Hazard Ratio [HR] = 1.45; 95% \\nConfidence Interval [CI]: 1.08-1.94; log-rank P = 0.013) and SPP1 (HR = 1.44; \\n95% CI: 1.07-1.93; log-rank P = 0.015) were identified to be associated with \\nLUAD prognosis. Moreover, the early LUAD invasion was accompanied by the \\nenhancement of antigen presentation ability, reflected by the increase of \\nmyeloid dendritic cells infiltration rate (Cuzick test P < 0.01) and the \\nupregulation of seven important genes participating in the antigen presentation, \\nincluding HLA-A (Cuzick test P = 0.03), MICA (Cuzick test P = 0.01), MICB \\n(Cuzick test P = 0.01), HLA-DPA1 (Cuzick test P = 0.04), HLA-DQA2 (Cuzick test P \\n< 0.01), HLA-DQB1 (Cuzick test P = 0.03), and HLA-DQB2 (Cuzick test P < 0.01). \\nHowever, the tumor-killing ability of the immune system was inhibited during \\nthis process, as there were no rising cytotoxic T cell activity (Cuzick test P = \\n0.20) and no increasing expression in genes encoding cytotoxic proteins.\\nCONCLUSION: In all, our research elucidated the changes in the immune \\nmicroenvironment during early-stage LUAD evolution and may provide a theoretical \\nbasis for developing novel early-stage lung cancer therapeutic targets.\\n\\nCopyright © 2023 Liu, Ji, Weng, Shao, Zhao, Chen, Shen, Wang, Meng, Wu, Wang, Ou \\nand Ke.\\n\\nDOI: 10.3389/fonc.2023.1150098\\nPMCID: PMC10328385\\nPMID: 37427097\\n\\nConflict of interest statement: HC, LS, FW, QM, XWU, XWA, and QO are employees \\nof Nanjing Geneseeq Technology Inc., China. The remaining authors declare that \\nthe research was conducted in the absence of any commercial or financial \\nrelationships that could be construed as a potential conflict of interest.\\n\\n\\n5. Hereditas. 2023 May 31;160(1):27. doi: 10.1186/s41065-023-00289-6.\\n\\nNPM3 as a novel oncogenic factor and poor prognostic marker contributes to cell \\nproliferation and migration in lung adenocarcinoma.\\n\\nWei S(1)(2), Xing J(2)(3), Lu K(4), Wang K(1), Yu W(5).\\n\\nAuthor information:\\n(1)Department of Respiratory and Critical Care Medicine, The Fourth Affiliated \\nHospital, School of Medicine, Zhejiang University, Yiwu, 322000, Zhejiang, \\nPeople's Republic of China.\\n(2)Department of Respiratory and Critical Care Medicine, The Affiliated People's \\nHospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \\nBaizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China.\\n(3)Ningbo University School of Medicine, Zhejiang Province, Ningbo, People's \\nRepublic of China.\\n(4)Department of Urology, The Affiliated First Hospital of Ningbo University \\n(Ningbo First Hospital), No.59, Liuting Street, Ningbo, 315010, Zhejiang, \\nPeople's Republic of China.\\n(5)Department of Respiratory and Critical Care Medicine, The Affiliated People's \\nHospital of Ningbo University (Ningbo Yinzhou People's Hospital), No.251, \\nBaizhang Road, Ningbo, 315040, Zhejiang, People's Republic of China. \\nnbyuwanjun@163.com.\\n\\nBACKGROUND: Lung cancer is the leading cause of cancer-related deaths worldwide, \\nand despite recent advances in targeted therapies and immunotherapies, the \\nclinical benefit remains limited. Therefore, there is an urgent need to further \\ninvestigate the molecular mechanisms underlying lung cancer. The aim of this \\nstudy was to investigate the expression and function of NPM3 in the tumor \\nmicroenvironment of lung adenocarcinoma (LUAD).\\nMETHODS: We utilized bioinformatics tools and databases, including UALCAN, \\nGEPIA2, HPA, and Sangerbox, to analyze NPM3 expression in LUAD samples and its \\nassociation with prognosis and mutational landscape. NPM3 expression in various \\ncell types was assessed at the single cell level using the TISCH database. We \\nalso used algorithms such as TIMER and EPIC to explore the crosstalk between \\nNPM3 expression and immune features. KEGG enrichment analysis was performed to \\nidentify potential signaling pathways of NPM3. Finally, we employed siRNA \\nknockdown strategy to investigate the effect of NPM3 on LUAD cell proliferation \\nand migration in vitro.\\nRESULTS: NPM3 was significantly upregulated in LUAD tissues and was strongly \\nassociated with poor prognosis and TP53 gene mutations. Single-cell sequencing \\nanalysis revealed that NPM3 was expressed in immune cells (dendritic cells and \\nmonocytes/macrophages) in the tumor microenvironment. Moreover, NPM3 expression \\nwas negatively associated with immune B cell and CD4 T cell infiltration, as \\nwell as with several immune-related genes (including CCL22, CXCR2, CX3CR1, CCR6, \\nHLA-DOA, HLA-DQA2). KEGG enrichment analysis indicated that NPM3 expression was \\nassociated with cell cycle, CAMs, and NSCLC pathway genes. Finally, in vitro \\nexperiments showed that NPM3 knockdown inhibited LUAD cell proliferation and \\nmigration in NCI-H1299 and SPC-A1 cells, and suppressed the expression of CCNA2 \\nand MAD2L1.\\nCONCLUSION: Elevated NPM3 expression predicts poor clinical outcome and an \\nimmunosuppressive microenvironment in LUAD tissues. NPM3 promotes LUAD \\nprogression by promoting cell proliferation and migration, and targeting NPM3 \\nmay represent a novel therapeutic strategy for LUAD.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s41065-023-00289-6\\nPMCID: PMC10230701\\nPMID: 37254219 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors claim that there is no possible \\nconflict of interest.\\n\\n\\n6. Rheumatology (Oxford). 2023 Sep 1;62(9):3213-3218. doi: \\n10.1093/rheumatology/kead152.\\n\\nStratified genetic analysis reveals sex differences in MPO-ANCA-associated \\nvasculitis.\\n\\nEkman D(1), Sennblad B(2), Knight A(3), Karlsson Å(4), Rantapää-Dahlqvist S(5), \\nBerglin E(5), Stegmayr B(5), Baslund B(6), Palm Ø(7), Haukeland H(8), Gunnarsson \\nI(9)(10), Bruchfeld A(11)(12), Segelmark M(13), Ohlsson S(13), Mohammad \\nAJ(14)(15), Svärd A(16), Pullerits R(17)(18), Herlitz H(19), Söderbergh A(20), \\nOmdal R(21)(22), Jonsson R(23), Rönnblom L(3), Eriksson P(24), Lindblad-Toh \\nK(4)(25), Dahlqvist J(3)(4)(25).\\n\\nAuthor information:\\n(1)Department of Biochemistry and Biophysics, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Stockholm University, \\nSweden.\\n(2)Department of Cell and Molecular Biology, National Bioinformatics \\nInfrastructure Sweden, Science for Life Laboratory, Uppsala University, Uppsala, \\nSweden.\\n(3)Department of Medical Sciences, Uppsala University, Uppsala, Sweden.\\n(4)Science for Life Laboratory, Department of Medical Biochemistry and \\nMicrobiology, Uppsala University, Uppsala, Sweden.\\n(5)Department of Public Health and Clinical Medicine, Umeå University, Umeå, \\nSweden.\\n(6)Copenhagen Lupus and Vasculitis Clinic, Center for Rheumatology and Spine \\nDiseases, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark.\\n(7)Department of Rheumatology, Oslo University Hospital, Oslo, Norway.\\n(8)Department of Rheumatology, Martina Hansens Hospital, Gjettum, Norway.\\n(9)Department of Medicine, Division of Rheumatology, Karolinska Institutet, \\nStockholm, Sweden.\\n(10)Unit of Rheumatology, Karolinska University Hospital, Stockholm, Sweden.\\n(11)Department of Health, Medicine and Caring Sciences, Linköping University, \\nLinköping, Sweden.\\n(12)Department of Renal Medicine, Karolinska University Hospital and CLINTEC \\nKarolinska Institutet, Stockholm, Sweden.\\n(13)Department of Clinical Sciences, Division of Nephrology, Lund University and \\nSkåne University Hospital, Lund, Sweden.\\n(14)Department of Clinical Sciences Lund, Section of Rheumatology, Skåne \\nUniversity Hospital, Lund University, Lund, Sweden.\\n(15)Department of Medicine, University of Cambridge, Cambridge, UK.\\n(16)Center for Clinical Research Dalarna, Uppsala University, Uppsala, Sweden.\\n(17)Department of Rheumatology and Inflammation Research, Institution of \\nMedicine, Sahlgrenska Academy at University of Gothenburg, Gothenburg, Sweden.\\n(18)Department of Clinical Immunology and Transfusion Medicine, Sahlgrenska \\nUniversity Hospital, Gothenburg, Sweden.\\n(19)Department of Molecular and Clinical Medicine/Nephrology, Institute of \\nMedicine, the Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden.\\n(20)Department of Rheumatology, Örebro University Hospital, Örebro, Sweden.\\n(21)Clinical Immunology Unit, Department of Internal Medicine, Stavanger \\nUniversity Hospital, Stavanger, Norway.\\n(22)Department of Clinical Science, University of Bergen, Bergen, Norway.\\n(23)Broegelmann Research Laboratory, Department of Clinical Science, University \\nof Bergen, Bergen, Norway.\\n(24)Department of Biomedical and Clinical Sciences, Division of Inflammation and \\nInfection, Linköping University, Linköping, Sweden.\\n(25)Broad Institute of MIT and Harvard University, Cambridge, MA, USA.\\n\\nOBJECTIVE: To identify and genetically characterize subgroups of patients with \\nANCA-associated vasculitides (AAV) based on sex and ANCA subtype.\\nMETHODS: A previously established SNP dataset derived from DNA sequencing of \\n1853 genes and genotyping of 1088 Scandinavian cases with AAV and 1589 controls \\nwas stratified for sex and ANCA subtype and analysed for association with five \\ntop AAV SNPs. rs9274619, a lead variant at the HLA-DQB1/HLA-DQA2 locus \\npreviously associated with AAV positive for myeloperoxidase (MPO)-ANCA, was \\nanalysed for association with the cumulative disease involvement of ten \\ndifferent organ systems.\\nRESULTS: rs9274619 showed a significantly stronger association to \\nMPO-ANCA-positive females than males [P\\u2009=\\u20092.0\\u2009×\\u200910-4, OR\\u2009=\\u20092.3 (95% CI 1.5, \\n3.5)], whereas proteinase 3 (PR3)-ANCA-associated variants rs1042335, rs9277341 \\n(HLA-DPB1/A1) and rs28929474 (SERPINA1) were equally associated with females and \\nmales with PR3-ANCA. In MPO-ANCA-positive cases, carriers of the rs9274619 risk \\nallele were more prone to disease engagement of eyes [P\\u2009=\\u20090.021, OR\\u2009=\\u200911 (95% CI \\n2.2, 205)] but less prone to pulmonary involvement [P\\u2009=\\u20090.026, OR\\u2009=\\u20090.52 (95% CI \\n0.30, 0.92)]. Moreover, AAV with both MPO-ANCA and PR3-ANCA was associated with \\nthe PR3-ANCA lead SNP rs1042335 [P\\u2009=\\u20090.0015, OR\\u2009=\\u20090.091 (95% CI 0.0022, 0.55)] \\nbut not with rs9274619.\\nCONCLUSIONS: Females and males with MPO-ANCA-positive AAV differ in genetic \\npredisposition to disease, suggesting at least partially distinct disease \\nmechanisms between the sexes. Double ANCA-positive AAV cases are genetically \\nsimilar to PR3-ANCA-positive cases, providing clues to the clinical follow-up \\nand treatment of these patients.\\n\\n© The Author(s) 2023. Published by Oxford University Press on behalf of the \\nBritish Society for Rheumatology.\\n\\nDOI: 10.1093/rheumatology/kead152\\nPMCID: PMC10473270\\nPMID: 37004177 [Indexed for MEDLINE]\\n\\n\\n7. BMC Genomics. 2023 Mar 27;24(1):153. doi: 10.1186/s12864-023-09227-2.\\n\\nGenome-wide identification of RNA modification-related single nucleotide \\npolymorphisms associated with rheumatoid arthritis.\\n\\nWang M(#)(1)(2), Wu J(#)(1)(2), Lei S(1)(2), Mo X(3)(4).\\n\\nAuthor information:\\n(1)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \\nSchool of Public Health, Medical College of Soochow University, 199 Renai Road, \\nSuzhou, Jiangsu, 215123, People's Republic of China.\\n(2)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \\nDiseases, School of Public Health, Medical College of Soochow University, \\nSuzhou, China.\\n(3)Center for Genetic Epidemiology and Genomics, Department of Epidemiology, \\nSchool of Public Health, Medical College of Soochow University, 199 Renai Road, \\nSuzhou, Jiangsu, 215123, People's Republic of China. xbmo@suda.edu.cn.\\n(4)Jiangsu Key Laboratory of Preventive and Translational Medicine for Geriatric \\nDiseases, School of Public Health, Medical College of Soochow University, \\nSuzhou, China. xbmo@suda.edu.cn.\\n(#)Contributed equally\\n\\nBACKGROUND: RNA modification plays important roles in many biological processes, \\nsuch as gene expression control. The aim of this study was to identify single \\nnucleotide polymorphisms related to RNA modification (RNAm-SNPs) for rheumatoid \\narthritis (RA) as putative functional variants.\\nMETHODS: We examined the association of RNAm-SNPs with RA in summary data from a \\ngenome-wide association study of 19,234 RA cases and 61,565 controls. We \\nperformed eQTL and pQTL analyses for the RNAm-SNPs to find associated gene \\nexpression and protein levels. Furthermore, we examined the associations of gene \\nexpression and circulating protein levels with RA using two-sample Mendelian \\nrandomization analysis methods.\\nRESULTS: A total of 160 RNAm-SNPs related to m6A, m1A, A-to-I, m7G, m5C, m5U and \\nm6Am modifications were identified to be significantly associated with RA. These \\nRNAm-SNPs were located in 62 protein-coding genes, which were significantly \\nenriched in immune-related pathways. RNAm-SNPs in important RA susceptibility \\ngenes, such as PADI2, SPRED2, PLCL2, HLA-A, HLA-B, HLA-DRB1, HLA-DPB1, TRAF1 and \\nTXNDC11, were identified. Most of these RNAm-SNPs showed eQTL effects, and the \\nexpression levels of 26 of the modifiable genes (e.g., PADI2, TRAF1, HLA-A, \\nHLA-DRB1, HLA-DPB1 and HLA-B) in blood cells were associated with RA. \\nCirculating protein levels, such as CFB, GZMA, HLA-DQA2, IL21, LRPAP1 and TFF3, \\nwere affected by RNAm-SNPs and were associated with RA.\\nCONCLUSION: The present study identified RNAm-SNPs in the reported RA \\nsusceptibility genes and suggested that RNAm-SNPs may affect RA risk by \\naffecting the expression levels of corresponding genes and proteins.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s12864-023-09227-2\\nPMCID: PMC10045113\\nPMID: 36973646 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare that they have no competing \\ninterests.\\n\\n\\n8. Yi Chuan. 2023 Jan 20;45(1):52-66. doi: 10.16288/j.yczz.22-293.\\n\\nApplication of transcriptome in time analysis and donor characterization in \\nblood samples.\\n\\nZhang J(1)(2), Liu KH(2), Zhang Y(2), Hao JP(2), Zhang GF(2), Xu XY(2), Chang \\nJ(2), Liu XP(3), Yang XY(2), Ye J(1)(2).\\n\\nAuthor information:\\n(1)1. People's Public Security University of China, Beijing 100038, China.\\n(2)2. Institute of Forensic Science Ministry of Public Security, Beijing 100038, \\nChina.\\n(3)3. Ruijin Public Security Bureau, Ruijin 342500, China.\\n\\nAs an effective supplement to the current forensic DNA typing and one of the \\nresearch hotpots in forensic science, the in-depth mining and characterization \\nof biological evidence can provide rich and reliable clues for case \\ninvestigation. In this study, the time-dependent variations of transcriptome \\nwere confirmed in in vitro blood samples within 0-168 days and a random forest \\nmodel was established to realize the classification of blood samples with \\ndifferent TSD (time since deposition). Meanwhile, significant differences were \\nobserved in the transcripts of blood samples with different smoking habits and \\ngenders within a certain time period. HLA-DRB1, HLA-DQB1 and HLA-DQA2 were \\nidentified as markers for smoking habit identification, while the transcripts \\nfor RPS4Y1 and EIF1AY from the non-recombining region of the Y chromosome (NRY) \\nwere identified as markers for male sex identification. Thus, this study \\nprovides a theoretical foundation and experimental strategy for establishing a \\ntranscriptome-based method for characterizing blood sample retention time and \\ndonor characteristics in the field of forensic investigation.\\n\\nPublisher: \\n案件现场生物物证信息深度挖掘与刻画，可以为案件侦查、涉案人员查找提供丰富可靠线索，是当前法医DNA检验的有效补充，也是国内外法庭科学的研究热点之一。本研究以血液样本为研究对象，证实了0~168天内离体血液样本转录组变化的时间相关性，并建立随机森林模型实现不同离体时间血液样本分类。同时，进一步证实相同离体时间段内，不同吸烟习惯和不同性别供体来源的血液样本转录本具有显著差异，HLA-DRB1、HLA-DQB1和HLA-DQA2可以作为供体吸烟习惯判别标志，Y染色体非重组区(non-recombining \\nY, \\nNRY)的转录本RPS4Y1和EIF1AY可以作为供体性别特征判别标志。本研究为法庭科学领域建立基于转录组分析的血液样本遗留时间和供体特征刻画方法提供了理论基础和实验依据。.\\n\\nDOI: 10.16288/j.yczz.22-293\\nPMID: 36927638 [Indexed for MEDLINE]\\n\\n\\n9. Pediatr Rheumatol Online J. 2023 Feb 15;21(1):17. doi: \\n10.1186/s12969-023-00787-x.\\n\\nProfiling the peripheral immune response to ex vivo TNF stimulation in untreated \\njuvenile idiopathic arthritis using single cell RNA sequencing.\\n\\nImbach KJ(#)(1), Treadway NJ(#)(2), Prahalad V(2), Kosters A(3), Arafat D(1), \\nDuan M(1), Gergely T(2), Ponder LA(4), Chandrakasan S(4)(5), Ghosn EEB(2)(3)(4), \\nPrahalad S(#)(6)(7)(8), Gibson G(#)(1)(4).\\n\\nAuthor information:\\n(1)Center for Integrative Genomics, School of Biological Sciences, Georgia \\nInstitute of Technology, Atlanta, GA, 30332, USA.\\n(2)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \\n30223, USA.\\n(3)Lowance Center for Human Immunology, Division of Immunology, Department of \\nMedicine, Emory University School of Medicine, Atlanta, GA, 30223, USA.\\n(4)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \\nAtlanta, GA, 30223, USA.\\n(5)Aflac Cancer and Blood Disorders Center, Department of Pediatrics, Children's \\nHealthcare of Atlanta, Emory University School of Medicine, Atlanta, GA, 30223, \\nUSA.\\n(6)Department of Pediatrics, Emory University School of Medicine, Atlanta, GA, \\n30223, USA. sprahal@emory.edu.\\n(7)Center for Immunity and Applied Genomics, Children's Healthcare of Atlanta, \\nAtlanta, GA, 30223, USA. sprahal@emory.edu.\\n(8)Department of Human Genetics, Emory University School of Medicine, Atlanta, \\nGA, 30223, USA. sprahal@emory.edu.\\n(#)Contributed equally\\n\\nBACKGROUND: Juvenile Idiopathic Arthritis (JIA) is an autoimmune disease with a \\nheterogenous clinical presentation and unpredictable response to available \\ntherapies. This personalized transcriptomics study sought proof-of-concept for \\nsingle-cell RNA sequencing to characterize patient-specific immune profiles.\\nMETHODS: Whole blood samples from six untreated children, newly diagnosed with \\nJIA, and two healthy controls were cultured for 24\\xa0h with or without ex vivo TNF \\nstimulation and subjected to scRNAseq to examine cellular populations and \\ntranscript expression in PBMCs. A novel analytical pipeline, scPool, was \\ndeveloped wherein cells are first pooled into pseudocells prior to expression \\nanalysis, facilitating variance partitioning of the effects of TNF stimulus, JIA \\ndisease status, and individual donor.\\nRESULTS: Seventeen robust immune cell-types were identified, the abundance of \\nwhich was significantly affected by TNF stimulus, which resulted in notable \\nelevation of memory CD8\\u2009+\\u2009T-cells and NK56 cells, but down-regulation of naïve \\nB-cell proportions. Memory CD8\\u2009+\\u2009and CD4\\u2009+\\u2009T-cells were also both reduced in the \\nJIA cases relative to two controls. Significant differential expression \\nresponses to TNF stimulus were also characterized, with monocytes showing more \\ntranscriptional shifts than T-lymphocyte subsets, while the B-cell response was \\nmore limited. We also show that donor variability exceeds the small degree of \\npossible intrinsic differentiation between JIA and control profiles. An \\nincidental finding of interest was association of HLA-DQA2 and HLA-DRB5 \\nexpression with JIA status.\\nCONCLUSIONS: These results support the development of personalized \\nimmune-profiling combined with ex-vivo immune stimulation for evaluation of \\npatient-specific modes of immune cell activity in autoimmune rheumatic disease.\\n\\n© 2023. The Author(s).\\n\\nDOI: 10.1186/s12969-023-00787-x\\nPMCID: PMC9929251\\nPMID: 36793127 [Indexed for MEDLINE]\\n\\nConflict of interest statement: Dr. Prahalad has served on a macrophage \\nactivation syndrome adjudication committee for Novartis, but this had no bearing \\non the work described in this manuscript. None of the other authors have any \\ncompeting interests in the manuscript.\\n\\n\\n10. Epilepsia. 2023 Jan;64(1):218-230. doi: 10.1111/epi.17457. Epub 2022 Nov 16.\\n\\nBrain molecular mechanisms in Rasmussen encephalitis.\\n\\nLeitner DF(1)(2), Lin Z(3), Sawaged Z(3), Kanshin E(4), Friedman D(1), Devore \\nS(1), Ueberheide B(2)(4)(5), Chang JW(6), Mathern GW(6), Anink JJ(7), Aronica \\nE(7)(8), Wisniewski T(2)(9)(10), Devinsky O(1).\\n\\nAuthor information:\\n(1)Comprehensive Epilepsy Center, NYU Grossman School of Medicine, New York, New \\nYork, USA.\\n(2)Center for Cognitive Neurology, Department of Neurology, NYU Grossman School \\nof Medicine, New York, New York, USA.\\n(3)Applied Bioinformatics Laboratories, NYU Grossman School of Medicine, New \\nYork, New York, USA.\\n(4)Proteomics Laboratory, Division of Advanced Research Technologies, NYU \\nGrossman School of Medicine, New York, New York, USA.\\n(5)Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of \\nMedicine, New York, New York, USA.\\n(6)Departments of Neurosurgery and Psychiatry, David Geffen School of Medicine, \\nUniversity of California, Los Angeles, Los Angeles, California, USA.\\n(7)Department of (Neuro) Pathology, Amsterdam University Medical Center, \\nUniversity of Amsterdam, Amsterdam Neuroscience, Amsterdam, the Netherlands.\\n(8)Epilepsy Institutes of the Netherlands Foundation, Heemstede, the \\nNetherlands.\\n(9)Department of Pathology, NYU Grossman School of Medicine, New York, New York, \\nUSA.\\n(10)Department of Psychiatry, NYU Grossman School of Medicine, New York, New \\nYork, USA.\\n\\nOBJECTIVE: This study was undertaken to identify molecular mechanisms in brain \\ntissue of Rasmussen encephalitis (RE) when compared to people with non-RE \\nepilepsy (PWE) and control cases using whole exome sequencing (WES), RNAseq, and \\nproteomics.\\nMETHODS: Frozen brain tissue (ages\\xa0=\\xa02-19\\u2009years) was obtained from control \\nautopsy (n\\xa0=\\xa014), surgical PWE (n\\xa0=\\xa010), and surgical RE cases (n\\xa0=\\xa027). We \\nevaluated WES variants in RE associated with epilepsy, seizures, RE, and human \\nleukocyte antigens (HLAs). Differential expression was evaluated by RNAseq \\n(adjusted p\\u2009<\\u2009.05) and label-free quantitative mass spectrometry (false \\ndiscovery rate\\u2009<\\u20095%) in the three groups.\\nRESULTS: WES revealed no common pathogenic variants in RE, but several rare and \\nlikely deleterious variants of unknown significance (VUS; ANGPTL7/MTOR, SCN1A, \\nFCGR3B, MTOR) and more common HLA VUS in >25% of RE cases (HLA-DRB1, HLA-DQA2), \\nall with allele frequency\\u2009<\\u20095% in the general population. RNAseq in RE versus \\nPWE (1516 altered transcripts) revealed significant activation of crosstalk \\nbetween dendritic and natural killer cells (p\\xa0=\\xa07.94\\u2009×\\u200910-6 , z\\xa0=\\xa02.65), in RE \\nversus control (7466 transcripts) neuroinflammation signaling activation \\n(p\\xa0=\\xa06.31\\u2009×\\u200910-13 , z\\xa0=\\xa05.07), and in PWE versus control (945 transcripts) \\nphagosome formation activation (p\\xa0=\\xa02.00\\u2009×\\u200910-13 , z\\xa0=\\xa05.61). Proteomics \\ndetected fewer altered targets.\\nSIGNIFICANCE: In RE, we identified activated immune signaling pathways and \\nimmune cell type annotation enrichment that suggest roles of the innate and \\nadaptive immune responses, as well as HLA variants that may increase \\nvulnerability to RE. Follow-up studies could evaluate cell type density and \\nsubregional localization associated with top targets, clinical history \\n(neuropathology, disease duration), and whether modulating crosstalk between \\ndendritic and natural killer cells may limit disease progression.\\n\\n© 2022 International League Against Epilepsy.\\n\\nDOI: 10.1111/epi.17457\\nPMCID: PMC9852002\\nPMID: 36336987 [Indexed for MEDLINE]\""
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Exemplos da utilização da função pesquisa_ncbi\n",
    "\n",
    "# Para processar artigos\n",
    "email = 'pg21019@alunos.uminho.pt'\n",
    "termo = 'hla-dqa2'\n",
    "\n",
    "from scripts import pesquisa_ncbi\n",
    "pesquisa_ncbi(email,termo)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em gene. Irá ser processado 1 resultado.\n",
      "\n",
      "HLA-DQA2 major histocompatibility complex, class II, DQ alpha 2[Homo sapiens]\n",
      "Gene ID: 3118, updated on 27-Dec-2023\n",
      "\n",
      "\n",
      "Reference GRCh38.p14 Primary Assembly NC_000006.12  from: 32741391 to: 32747198\n",
      "mRNA  NM_020056.5, 5 exons,  total annotated spliced exon length: 1458\n",
      "protein precursor NP_064440.1 (CCDS4753.1), 4 coding  exons,  annotated AA length: 255\n",
      "\n",
      "Exon table for  mRNA  NM_020056.5 and protein NP_064440.1\n",
      "Genomic Interval Exon\t\tGenomic Interval Coding\t\tGene Interval Exon\t\tGene Interval Coding\t\tExon Length\tCoding Length\tIntron Length\n",
      "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
      "32741391-32741525\t\t32741444-32741525\t\t1-135\t\t54-135\t\t135\t\t82\t\t3633\n",
      "32745159-32745407\t\t32745159-32745407\t\t3769-4017\t\t3769-4017\t\t249\t\t249\t\t383\n",
      "32745791-32746072\t\t32745791-32746072\t\t4401-4682\t\t4401-4682\t\t282\t\t282\t\t167\n",
      "32746240-32746414\t\t32746240-32746394\t\t4850-5024\t\t4850-5004\t\t175\t\t155\t\t167\n",
      "32746582-32747198\t\t5192-5808\t\t617\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'HLA-DQA2 major histocompatibility complex, class II, DQ alpha 2[Homo sapiens]\\nGene ID: 3118, updated on 27-Dec-2023\\n\\n\\nReference GRCh38.p14 Primary Assembly NC_000006.12  from: 32741391 to: 32747198\\nmRNA  NM_020056.5, 5 exons,  total annotated spliced exon length: 1458\\nprotein precursor NP_064440.1 (CCDS4753.1), 4 coding  exons,  annotated AA length: 255\\n\\nExon table for  mRNA  NM_020056.5 and protein NP_064440.1\\nGenomic Interval Exon\\t\\tGenomic Interval Coding\\t\\tGene Interval Exon\\t\\tGene Interval Coding\\t\\tExon Length\\tCoding Length\\tIntron Length\\n------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\\n32741391-32741525\\t\\t32741444-32741525\\t\\t1-135\\t\\t54-135\\t\\t135\\t\\t82\\t\\t3633\\n32745159-32745407\\t\\t32745159-32745407\\t\\t3769-4017\\t\\t3769-4017\\t\\t249\\t\\t249\\t\\t383\\n32745791-32746072\\t\\t32745791-32746072\\t\\t4401-4682\\t\\t4401-4682\\t\\t282\\t\\t282\\t\\t167\\n32746240-32746414\\t\\t32746240-32746394\\t\\t4850-5024\\t\\t4850-5004\\t\\t175\\t\\t155\\t\\t167\\n32746582-32747198\\t\\t5192-5808\\t\\t617\\n'"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Para encontrar accessions e ficheiros GenBank\n",
    "pesquisa_ncbi(email,'3118','gene',1,'gene_table')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em nuccore. Irá ser processado 1 resultado.\n",
      "\n",
      "LOCUS       NM_020056               1458 bp    mRNA    linear   PRI 25-DEC-2022\n",
      "DEFINITION  Homo sapiens major histocompatibility complex, class II, DQ alpha 2\n",
      "            (HLA-DQA2), mRNA.\n",
      "ACCESSION   NM_020056 XM_005249050 XM_005272806 XM_005274854 XM_005275110\n",
      "            XM_005275385 XM_005275545\n",
      "VERSION     NM_020056.5\n",
      "KEYWORDS    RefSeq; MANE Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1458)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   2  (bases 1 to 1458)\n",
      "  AUTHORS   Liang SQ, Deng JM, Wei X, Chen ZR, Yang ML, Qin HJ, Zhang JQ and He\n",
      "            ZY.\n",
      "  TITLE     Association of GWAS-supported noncoding area loci rs404860,\n",
      "            rs3117098, and rs7775228 with asthma in Chinese Zhuang population\n",
      "  JOURNAL   J Clin Lab Anal 34 (2), e23066 (2020)\n",
      "   PUBMED   31605414\n",
      "  REMARK    GeneRIF: Association of GWAS-supported noncoding area loci\n",
      "            rs404860, rs3117098, and rs7775228 with asthma in Chinese Zhuang\n",
      "            population.\n",
      "REFERENCE   3  (bases 1 to 1458)\n",
      "  AUTHORS   Armstrong DL, Zidovetzki R, Alarcon-Riquelme ME, Tsao BP, Criswell\n",
      "            LA, Kimberly RP, Harley JB, Sivils KL, Vyse TJ, Gaffney PM,\n",
      "            Langefeld CD and Jacob CO.\n",
      "  TITLE     GWAS identifies novel SLE susceptibility genes and explains the\n",
      "            association of the HLA region\n",
      "  JOURNAL   Genes Immun 15 (6), 347-354 (2014)\n",
      "   PUBMED   24871463\n",
      "REFERENCE   4  (bases 1 to 1458)\n",
      "  AUTHORS   Chang SW, Fann CS, Su WH, Wang YC, Weng CC, Yu CJ, Hsu CL, Hsieh\n",
      "            AR, Chien RN, Chu CM and Tai DI.\n",
      "  TITLE     A genome-wide association study on chronic HBV infection and its\n",
      "            clinical progression in male Han-Taiwanese\n",
      "  JOURNAL   PLoS One 9 (6), e99724 (2014)\n",
      "   PUBMED   24940741\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   5  (bases 1 to 1458)\n",
      "  AUTHORS   Jiang L, Yin J, Ye L, Yang J, Hemani G, Liu AJ, Zou H, He D, Sun L,\n",
      "            Zeng X, Li Z, Zheng Y, Lin Y, Liu Y, Fang Y, Xu J, Li Y, Dai S,\n",
      "            Guan J, Jiang L, Wei Q, Wang Y, Li Y, Huang C, Zuo X, Liu Y, Wu X,\n",
      "            Zhang L, Zhou L, Zhang Q, Li T, Chen L, Xu Z, Yang X, Qian F, Xie\n",
      "            W, Liu W, Guo Q, Huang S, Zhao J, Li M, Jin Y, Gao J, Lv Y, Wang Y,\n",
      "            Lin L, Guo A, Danoy P, Willner D, Cremin C, Hadler J, Zhang F, Zhao\n",
      "            Y, Li M, Yue T, Fan X, Guo J, Mu R, Li J, Wu C, Zeng M, Wang J, Li\n",
      "            S, Jin L, Wang B, Wang J, Ma X, Sun L, Zhang X, Brown MA, Visscher\n",
      "            PM, Su DF and Xu H.\n",
      "  TITLE     Novel risk loci for rheumatoid arthritis in Han Chinese and\n",
      "            congruence with risk variants in Europeans\n",
      "  JOURNAL   Arthritis Rheumatol 66 (5), 1121-1132 (2014)\n",
      "   PUBMED   24782177\n",
      "  REMARK    Erratum:[Arthritis Rheumatol. 2014 Jul;66(7):1881]\n",
      "REFERENCE   6  (bases 1 to 1458)\n",
      "  AUTHORS   Marsh SG and Bodmer JG.\n",
      "  TITLE     HLA class II nucleotide sequences, 1992\n",
      "  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\n",
      "   PUBMED   1362295\n",
      "  REMARK    Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\n",
      "            1529427]\n",
      "REFERENCE   7  (bases 1 to 1458)\n",
      "  AUTHORS   Wang Y, Lu P, Zhou K, Clay T, Wood N, Bradley B and Bidwell J.\n",
      "  TITLE     Nucleotide sequencing of HLA-DQ gene second exons in Chinese\n",
      "            homozygous cells\n",
      "  JOURNAL   Eur J Immunogenet 19 (5), 287-293 (1992)\n",
      "   PUBMED   1420115\n",
      "REFERENCE   8  (bases 1 to 1458)\n",
      "  AUTHORS   Kenter M, Otting N, Anholts J, Leunissen J, Jonker M and Bontrop\n",
      "            RE.\n",
      "  TITLE     Evolutionary relationships among the primate Mhc-DQA1 and DQA2\n",
      "            alleles\n",
      "  JOURNAL   Immunogenetics 36 (2), 71-78 (1992)\n",
      "   PUBMED   1612647\n",
      "REFERENCE   9  (bases 1 to 1458)\n",
      "  AUTHORS   Gyllensten UB and Erlich HA.\n",
      "  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\n",
      "            primates\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\n",
      "   PUBMED   2513578\n",
      "REFERENCE   10 (bases 1 to 1458)\n",
      "  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\n",
      "            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\n",
      "  TITLE     Class II genes of the human major histocompatibility complex.\n",
      "            Comparisons of the DQ and DX alpha and beta genes\n",
      "  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\n",
      "   PUBMED   3036828\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL672104.4.\n",
      "            \n",
      "            On May 17, 2019 this sequence version replaced NM_020056.4.\n",
      "            \n",
      "            Summary: This gene belongs to the HLA class II alpha chain family.\n",
      "            The encoded protein forms a heterodimer with a class II beta chain.\n",
      "            It is located in intracellular vesicles and plays a central role in\n",
      "            the peptide loading of MHC class II molecules by helping to release\n",
      "            the CLIP molecule from the peptide binding site. Class II molecules\n",
      "            are expressed in antigen presenting cells (B lymphocytes, dendritic\n",
      "            cells, macrophages) and are used to present antigenic peptides on\n",
      "            the cell surface to be recognized by CD4 T-cells. [provided by\n",
      "            RefSeq, Jun 2010].\n",
      "            \n",
      "            Sequence Note: The RefSeq transcript and protein were derived from\n",
      "            genomic sequence to make the sequence consistent with the reference\n",
      "            genome assembly. The genomic coordinates used for the transcript\n",
      "            record were based on alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            RNAseq introns :: single sample supports all introns SAMEA2144335,\n",
      "                              SAMEA2145893 [ECO:0000348]\n",
      "            ##Evidence-Data-END##\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            MANE Ensembl match     :: ENST00000374940.4/ ENSP00000364076.3\n",
      "            RefSeq Select criteria :: based on single protein-coding transcript\n",
      "            ##RefSeq-Attributes-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-135               AL672104.4         5734-5868\n",
      "            136-384             AL672104.4         9502-9750\n",
      "            385-666             AL672104.4         10134-10415\n",
      "            667-841             AL672104.4         10583-10757\n",
      "            842-1458            AL672104.4         10925-11541\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1458\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"6\"\n",
      "                     /map=\"6p21.32\"\n",
      "     gene            1..1458\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"major histocompatibility complex, class II, DQ\n",
      "                     alpha 2\"\n",
      "                     /db_xref=\"GeneID:3118\"\n",
      "                     /db_xref=\"HGNC:HGNC:4943\"\n",
      "                     /db_xref=\"MIM:613503\"\n",
      "     exon            1..135\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    33..35\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             54..821\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"HLA class II histocompatibility antigen, DQ alpha 2\n",
      "                     chain; MHC class II DQA2; DX alpha chain; HLA class II\n",
      "                     histocompatibility antigen, DQ(6) alpha chain; MHC class\n",
      "                     II antigen; MHC class II DQA1; HLA class II\n",
      "                     histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha\n",
      "                     chain; MHC class I antigen\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain precursor\"\n",
      "                     /protein_id=\"NP_064440.1\"\n",
      "                     /db_xref=\"CCDS:CCDS4753.1\"\n",
      "                     /db_xref=\"GeneID:3118\"\n",
      "                     /db_xref=\"HGNC:HGNC:4943\"\n",
      "                     /db_xref=\"MIM:613503\"\n",
      "                     /translation=\"MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGP\n",
      "                     SGQYTHEFDGDEEFYVDLETKETVWQLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQ\n",
      "                     SNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSE\n",
      "                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTET\n",
      "                     LVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\"\n",
      "     sig_peptide     54..122\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\n",
      "     mat_peptide     123..818\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 2 chain\"\n",
      "     misc_feature    123..383\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Alpha-1\"\n",
      "     misc_feature    363..365\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\n",
      "     misc_feature    384..665\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Alpha-2\"\n",
      "     misc_feature    483..485\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\n",
      "     misc_feature    666..704\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     Region: Connecting peptide\"\n",
      "     misc_feature    705..773\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\n",
      "                     transmembrane region\"\n",
      "     exon            136..384\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            385..666\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            667..841\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            842..1458\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      1134..1139\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1158\n",
      "                     /gene=\"HLA-DQA2\"\n",
      "                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\n",
      "                     HLADQA2\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 acaattgctc tacagctcag agcagcaact gctgaggctg ccttgggaag aagatgatcc\n",
      "       61 taaacaaagc tctgctgctg ggggccctcg ccctgactgc cgtgatgagc ccctgtggag\n",
      "      121 gtgaagacat tgtggctgac catgttgcct cctatggtgt gaacttctac cagtctcacg\n",
      "      181 gtccctctgg ccagtacacc catgaatttg atggagacga ggagttctat gtggacctgg\n",
      "      241 agacgaaaga gactgtctgg cagttgccta tgtttagcaa atttataagt tttgacccgc\n",
      "      301 agagtgcact gagaaatatg gctgtgggaa aacacacctt ggaattcatg atgagacagt\n",
      "      361 ccaactctac cgctgccacc aatgaggttc ctgaggtcac agtgttttcc aagtttcctg\n",
      "      421 tgacgctggg tcagcccaac accctcatct gtcttgtgga caacatcttt cctcctgtgg\n",
      "      481 tcaacatcac ctggctgagc aatgggcact cagtcacaga aggtgtttct gagaccagct\n",
      "      541 tcctctccaa gagtgatcat tccttcttca agatcagtta cctcaccttc ctcccttctg\n",
      "      601 ctgatgagat ttatgactgc aaggtggagc actggggcct ggacgagcct cttctgaaac\n",
      "      661 actgggagcc tgagattcca gcccctatgt cagagctcac agagactttg gtctgcgccc\n",
      "      721 tggggttgtc tgtgggcctc atgggcattg tggtgggcac tgtcttcatc atccaaggcc\n",
      "      781 tgcgttcagt tggtgcttcc agacaccaag ggctcttatg aatcccatcc tgaaaaggaa\n",
      "      841 ggtgcatcac catctacagg agaagaagaa tggacttgct aaatgaccta gcactattct\n",
      "      901 ctggcctgat ttatcatatc ccttttctcc tccaaatgtt tcttctctca cctcttctct\n",
      "      961 gggacttaag gtgctatatt ccctcagagc tcacaaatgc ctttcaattc tttccctgac\n",
      "     1021 ctcctttcct gaattttttt attttctcaa atgttaccta ctaagggatg cctgggtaag\n",
      "     1081 ccactcagct acctaattcc tcaatgacct ttatctaaaa tctccatgga agcaataaat\n",
      "     1141 tcccttttga tgcctctatt gaatttttcc catctttcat ctcagggctg actgagagca\n",
      "     1201 taacttagaa tgggcgactc ttatgtttta ggccaatttc atatcattcc ccagatcata\n",
      "     1261 tttcaagtcc agtaacacag gagcaaccaa gtacagtgta tcctgataat ttgttgattt\n",
      "     1321 cttaactggt gttaatattt ctttcttcct tttgttccta cccttggcca ctgccagcca\n",
      "     1381 cccctcaatt caggtaccaa cgaaccctct gcccttggct cagaatggtt atagcagaaa\n",
      "     1441 tacaaaaaaa aaaaaaaa\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'LOCUS       NM_020056               1458 bp    mRNA    linear   PRI 25-DEC-2022\\nDEFINITION  Homo sapiens major histocompatibility complex, class II, DQ alpha 2\\n            (HLA-DQA2), mRNA.\\nACCESSION   NM_020056 XM_005249050 XM_005272806 XM_005274854 XM_005275110\\n            XM_005275385 XM_005275545\\nVERSION     NM_020056.5\\nKEYWORDS    RefSeq; MANE Select.\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1  (bases 1 to 1458)\\n  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\\n            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\\n            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\\n            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\\n            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\\n            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\\n            Colabella C, Coppin G, D\\'Amata C, De Ridder D, De Rouck S,\\n            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\\n            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\\n            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\\n            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\\n            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\\n            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\\n            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\\n            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\\n            MA.\\n  TITLE     A reference map of the human binary protein interactome\\n  JOURNAL   Nature 580 (7803), 402-408 (2020)\\n   PUBMED   32296183\\nREFERENCE   2  (bases 1 to 1458)\\n  AUTHORS   Liang SQ, Deng JM, Wei X, Chen ZR, Yang ML, Qin HJ, Zhang JQ and He\\n            ZY.\\n  TITLE     Association of GWAS-supported noncoding area loci rs404860,\\n            rs3117098, and rs7775228 with asthma in Chinese Zhuang population\\n  JOURNAL   J Clin Lab Anal 34 (2), e23066 (2020)\\n   PUBMED   31605414\\n  REMARK    GeneRIF: Association of GWAS-supported noncoding area loci\\n            rs404860, rs3117098, and rs7775228 with asthma in Chinese Zhuang\\n            population.\\nREFERENCE   3  (bases 1 to 1458)\\n  AUTHORS   Armstrong DL, Zidovetzki R, Alarcon-Riquelme ME, Tsao BP, Criswell\\n            LA, Kimberly RP, Harley JB, Sivils KL, Vyse TJ, Gaffney PM,\\n            Langefeld CD and Jacob CO.\\n  TITLE     GWAS identifies novel SLE susceptibility genes and explains the\\n            association of the HLA region\\n  JOURNAL   Genes Immun 15 (6), 347-354 (2014)\\n   PUBMED   24871463\\nREFERENCE   4  (bases 1 to 1458)\\n  AUTHORS   Chang SW, Fann CS, Su WH, Wang YC, Weng CC, Yu CJ, Hsu CL, Hsieh\\n            AR, Chien RN, Chu CM and Tai DI.\\n  TITLE     A genome-wide association study on chronic HBV infection and its\\n            clinical progression in male Han-Taiwanese\\n  JOURNAL   PLoS One 9 (6), e99724 (2014)\\n   PUBMED   24940741\\n  REMARK    Publication Status: Online-Only\\nREFERENCE   5  (bases 1 to 1458)\\n  AUTHORS   Jiang L, Yin J, Ye L, Yang J, Hemani G, Liu AJ, Zou H, He D, Sun L,\\n            Zeng X, Li Z, Zheng Y, Lin Y, Liu Y, Fang Y, Xu J, Li Y, Dai S,\\n            Guan J, Jiang L, Wei Q, Wang Y, Li Y, Huang C, Zuo X, Liu Y, Wu X,\\n            Zhang L, Zhou L, Zhang Q, Li T, Chen L, Xu Z, Yang X, Qian F, Xie\\n            W, Liu W, Guo Q, Huang S, Zhao J, Li M, Jin Y, Gao J, Lv Y, Wang Y,\\n            Lin L, Guo A, Danoy P, Willner D, Cremin C, Hadler J, Zhang F, Zhao\\n            Y, Li M, Yue T, Fan X, Guo J, Mu R, Li J, Wu C, Zeng M, Wang J, Li\\n            S, Jin L, Wang B, Wang J, Ma X, Sun L, Zhang X, Brown MA, Visscher\\n            PM, Su DF and Xu H.\\n  TITLE     Novel risk loci for rheumatoid arthritis in Han Chinese and\\n            congruence with risk variants in Europeans\\n  JOURNAL   Arthritis Rheumatol 66 (5), 1121-1132 (2014)\\n   PUBMED   24782177\\n  REMARK    Erratum:[Arthritis Rheumatol. 2014 Jul;66(7):1881]\\nREFERENCE   6  (bases 1 to 1458)\\n  AUTHORS   Marsh SG and Bodmer JG.\\n  TITLE     HLA class II nucleotide sequences, 1992\\n  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\\n   PUBMED   1362295\\n  REMARK    Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\\n            1529427]\\nREFERENCE   7  (bases 1 to 1458)\\n  AUTHORS   Wang Y, Lu P, Zhou K, Clay T, Wood N, Bradley B and Bidwell J.\\n  TITLE     Nucleotide sequencing of HLA-DQ gene second exons in Chinese\\n            homozygous cells\\n  JOURNAL   Eur J Immunogenet 19 (5), 287-293 (1992)\\n   PUBMED   1420115\\nREFERENCE   8  (bases 1 to 1458)\\n  AUTHORS   Kenter M, Otting N, Anholts J, Leunissen J, Jonker M and Bontrop\\n            RE.\\n  TITLE     Evolutionary relationships among the primate Mhc-DQA1 and DQA2\\n            alleles\\n  JOURNAL   Immunogenetics 36 (2), 71-78 (1992)\\n   PUBMED   1612647\\nREFERENCE   9  (bases 1 to 1458)\\n  AUTHORS   Gyllensten UB and Erlich HA.\\n  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\\n            primates\\n  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\\n   PUBMED   2513578\\nREFERENCE   10 (bases 1 to 1458)\\n  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\\n            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\\n  TITLE     Class II genes of the human major histocompatibility complex.\\n            Comparisons of the DQ and DX alpha and beta genes\\n  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\\n   PUBMED   3036828\\nCOMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\\n            reference sequence was derived from AL672104.4.\\n            \\n            On May 17, 2019 this sequence version replaced NM_020056.4.\\n            \\n            Summary: This gene belongs to the HLA class II alpha chain family.\\n            The encoded protein forms a heterodimer with a class II beta chain.\\n            It is located in intracellular vesicles and plays a central role in\\n            the peptide loading of MHC class II molecules by helping to release\\n            the CLIP molecule from the peptide binding site. Class II molecules\\n            are expressed in antigen presenting cells (B lymphocytes, dendritic\\n            cells, macrophages) and are used to present antigenic peptides on\\n            the cell surface to be recognized by CD4 T-cells. [provided by\\n            RefSeq, Jun 2010].\\n            \\n            Sequence Note: The RefSeq transcript and protein were derived from\\n            genomic sequence to make the sequence consistent with the reference\\n            genome assembly. The genomic coordinates used for the transcript\\n            record were based on alignments.\\n            \\n            Publication Note:  This RefSeq record includes a subset of the\\n            publications that are available for this gene. Please see the Gene\\n            record to access additional publications.\\n            \\n            ##Evidence-Data-START##\\n            RNAseq introns :: single sample supports all introns SAMEA2144335,\\n                              SAMEA2145893 [ECO:0000348]\\n            ##Evidence-Data-END##\\n            \\n            ##RefSeq-Attributes-START##\\n            MANE Ensembl match     :: ENST00000374940.4/ ENSP00000364076.3\\n            RefSeq Select criteria :: based on single protein-coding transcript\\n            ##RefSeq-Attributes-END##\\n            COMPLETENESS: complete on the 3\\' end.\\nPRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\\n            1-135               AL672104.4         5734-5868\\n            136-384             AL672104.4         9502-9750\\n            385-666             AL672104.4         10134-10415\\n            667-841             AL672104.4         10583-10757\\n            842-1458            AL672104.4         10925-11541\\nFEATURES             Location/Qualifiers\\n     source          1..1458\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"mRNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /chromosome=\"6\"\\n                     /map=\"6p21.32\"\\n     gene            1..1458\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"major histocompatibility complex, class II, DQ\\n                     alpha 2\"\\n                     /db_xref=\"GeneID:3118\"\\n                     /db_xref=\"HGNC:HGNC:4943\"\\n                     /db_xref=\"MIM:613503\"\\n     exon            1..135\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     misc_feature    33..35\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"upstream in-frame stop codon\"\\n     CDS             54..821\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"HLA class II histocompatibility antigen, DQ alpha 2\\n                     chain; MHC class II DQA2; DX alpha chain; HLA class II\\n                     histocompatibility antigen, DQ(6) alpha chain; MHC class\\n                     II antigen; MHC class II DQA1; HLA class II\\n                     histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha\\n                     chain; MHC class I antigen\"\\n                     /codon_start=1\\n                     /product=\"HLA class II histocompatibility antigen, DQ\\n                     alpha 2 chain precursor\"\\n                     /protein_id=\"NP_064440.1\"\\n                     /db_xref=\"CCDS:CCDS4753.1\"\\n                     /db_xref=\"GeneID:3118\"\\n                     /db_xref=\"HGNC:HGNC:4943\"\\n                     /db_xref=\"MIM:613503\"\\n                     /translation=\"MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGP\\n                     SGQYTHEFDGDEEFYVDLETKETVWQLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQ\\n                     SNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSE\\n                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTET\\n                     LVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\"\\n     sig_peptide     54..122\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\\n     mat_peptide     123..818\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /product=\"HLA class II histocompatibility antigen, DQ\\n                     alpha 2 chain\"\\n     misc_feature    123..383\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Alpha-1\"\\n     misc_feature    363..365\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255; propagated from\\n                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\\n     misc_feature    384..665\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Alpha-2\"\\n     misc_feature    483..485\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255; propagated from\\n                     UniProtKB/Swiss-Prot (P01906.2); glycosylation site\"\\n     misc_feature    666..704\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     Region: Connecting peptide\"\\n     misc_feature    705..773\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"propagated from UniProtKB/Swiss-Prot (P01906.2);\\n                     transmembrane region\"\\n     exon            136..384\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            385..666\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            667..841\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     exon            842..1458\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /inference=\"alignment:Splign:2.1.0\"\\n     regulatory      1134..1139\\n                     /regulatory_class=\"polyA_signal_sequence\"\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"hexamer: AATAAA\"\\n     polyA_site      1158\\n                     /gene=\"HLA-DQA2\"\\n                     /gene_synonym=\"DC-alpha; DQA1; DX-ALPHA; HLA-DCA; HLA-DXA;\\n                     HLADQA2\"\\n                     /note=\"major polyA site\"\\nORIGIN      \\n        1 acaattgctc tacagctcag agcagcaact gctgaggctg ccttgggaag aagatgatcc\\n       61 taaacaaagc tctgctgctg ggggccctcg ccctgactgc cgtgatgagc ccctgtggag\\n      121 gtgaagacat tgtggctgac catgttgcct cctatggtgt gaacttctac cagtctcacg\\n      181 gtccctctgg ccagtacacc catgaatttg atggagacga ggagttctat gtggacctgg\\n      241 agacgaaaga gactgtctgg cagttgccta tgtttagcaa atttataagt tttgacccgc\\n      301 agagtgcact gagaaatatg gctgtgggaa aacacacctt ggaattcatg atgagacagt\\n      361 ccaactctac cgctgccacc aatgaggttc ctgaggtcac agtgttttcc aagtttcctg\\n      421 tgacgctggg tcagcccaac accctcatct gtcttgtgga caacatcttt cctcctgtgg\\n      481 tcaacatcac ctggctgagc aatgggcact cagtcacaga aggtgtttct gagaccagct\\n      541 tcctctccaa gagtgatcat tccttcttca agatcagtta cctcaccttc ctcccttctg\\n      601 ctgatgagat ttatgactgc aaggtggagc actggggcct ggacgagcct cttctgaaac\\n      661 actgggagcc tgagattcca gcccctatgt cagagctcac agagactttg gtctgcgccc\\n      721 tggggttgtc tgtgggcctc atgggcattg tggtgggcac tgtcttcatc atccaaggcc\\n      781 tgcgttcagt tggtgcttcc agacaccaag ggctcttatg aatcccatcc tgaaaaggaa\\n      841 ggtgcatcac catctacagg agaagaagaa tggacttgct aaatgaccta gcactattct\\n      901 ctggcctgat ttatcatatc ccttttctcc tccaaatgtt tcttctctca cctcttctct\\n      961 gggacttaag gtgctatatt ccctcagagc tcacaaatgc ctttcaattc tttccctgac\\n     1021 ctcctttcct gaattttttt attttctcaa atgttaccta ctaagggatg cctgggtaag\\n     1081 ccactcagct acctaattcc tcaatgacct ttatctaaaa tctccatgga agcaataaat\\n     1141 tcccttttga tgcctctatt gaatttttcc catctttcat ctcagggctg actgagagca\\n     1201 taacttagaa tgggcgactc ttatgtttta ggccaatttc atatcattcc ccagatcata\\n     1261 tttcaagtcc agtaacacag gagcaaccaa gtacagtgta tcctgataat ttgttgattt\\n     1321 cttaactggt gttaatattt ctttcttcct tttgttccta cccttggcca ctgccagcca\\n     1381 cccctcaatt caggtaccaa cgaaccctct gcccttggct cagaatggtt atagcagaaa\\n     1441 tacaaaaaaa aaaaaaaa\\n//\\n\\n'"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'NM_020056','nuccore',1,'gb')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 0 resultados em nuccore. Irá ser processado 1 resultado.\n",
      "\n",
      "Supplied+id+parameter+is+empty.\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'Supplied+id+parameter+is+empty.\\n'"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Para transferir ficheiros FASTA\n",
    "pesquisa_ncbi(email,'NM_020056','nuccore',1,'fasta')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em protein. Irá ser processado 1 resultado.\n",
      "\n",
      ">NP_064440.1 HLA class II histocompatibility antigen, DQ alpha 2 chain precursor [Homo sapiens]\n",
      "MILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGPSGQYTHEFDGDEEFYVDLETKETVWQ\n",
      "LPMFSKFISFDPQSALRNMAVGKHTLEFMMRQSNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFP\n",
      "PVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPA\n",
      "PMSELTETLVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\n",
      "\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'>NP_064440.1 HLA class II histocompatibility antigen, DQ alpha 2 chain precursor [Homo sapiens]\\nMILNKALLLGALALTAVMSPCGGEDIVADHVASYGVNFYQSHGPSGQYTHEFDGDEEFYVDLETKETVWQ\\nLPMFSKFISFDPQSALRNMAVGKHTLEFMMRQSNSTAATNEVPEVTVFSKFPVTLGQPNTLICLVDNIFP\\nPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPA\\nPMSELTETLVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGLL\\n\\n'"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'NP_064440','protein',1,'fasta')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
