{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Para realizar a **análise de literatura**, foi desenvolvida uma função com o objetivo de realizar a pesquisa de bibliografia na base de dados NCBI através do BioPython. Esta função oferece uma abordagem flexível uma vez que permite a adaptação de parâmetros conforme os objetivos desejados.\n",
    "\n",
    "A função escrita recebe um email e um termo de pesquisa como entrada, depois, oferece a capacidade de selecionar a base de dados desejada conforme o leque de bases de dados disponíveis. Por exemplo, podemos selecionar:\n",
    "\n",
    "- pubmed\n",
    "- gene\n",
    "- nuccore\n",
    "- protein\n",
    "- ...\n",
    "\n",
    "Além disso, o número de resultados que integram o output pode ser maleável, tal como o tipo de informação a ser extraído (como abstracts ou sequências genéticas) e também o formato do output. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Função **pesquisa_ncbi**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "def pesquisa_ncbi(email, term, db = 'pubmed', retmax = 10, rettype = 'abstract', retmode = 'text', display = 'Y' , save = 'N', ext='txt'):\n",
    "    '''\n",
    "\n",
    "    Desenvolvido por Rui Sousa\n",
    "\n",
    "    NOTA - PARA USAR RETMODE XML TÊM DE SER DECLARADA UMA NOVA VARIÁVEL POIS A FUNÇÃO AINDA NÃO CRIA FICHEIROS XML\n",
    "    \n",
    "    Função para a pesquisa de bibliografia na base de dados NCBI utilizando BioPython\n",
    "    Esta função contempla um conjunto de parâmetros para seleção de base de dados e outputs desejado adaptando-se\n",
    "    ao objetivo da pesquisa. \n",
    "\n",
    "    Por defeito a função pesquisa a base de dados pubmed e capta os 10 primeiros resultados para uma pesquisa\n",
    "    generalizada. \n",
    "\n",
    "    Tem a opção de gerar um ficheiro de texto com os resultados da pesquisa de artigos e respetivos abstracts.\n",
    "\n",
    "    No entanto a função pode ser especializada para uma pesquisa em base de dados de genes para captar ficheiros\n",
    "    GenBank e FASTA para posterior análise com as funções específicas que estamos a desenvolver.\n",
    "\n",
    "    Parâmetros\n",
    "    ----------\n",
    "\n",
    "    email : str\n",
    "        email a ser utilizado pelo módulo Entrez para  pesquisa (e.g. exempl@mail.com)\n",
    "\n",
    "    term : str\n",
    "        termo a ser utilizado na pesquisa (e.g. 'gene-123 functions')\n",
    "\n",
    "    db  : str\n",
    "        base de dados onde irá ser realizada a pesquisa. Por defeito é utilizada a 'pubmed'\n",
    "        Para efeitos deste trabalho iremos também utilizar: 'gene', 'nuccore' e 'protein'\n",
    "        Outras opções disponíveis podem ser encontradas aqui: https://www.ncbi.nlm.nih.gov/books/NBK25497/table/chapter2.T._entrez_unique_identifiers_ui/?report=objectonly\n",
    "\n",
    "    retmax : int\n",
    "        define o número de resultados que queremos retirar para o output final (10 por defeito)\n",
    "\n",
    "    rettype : str\n",
    "        define o tipo de resultado que pretendemos retirar da base de dados ('abstract' por defeito)\n",
    "        pode ser mudado para 'gb' para base de dados 'gene' ou 'fasta' para base de dados de nucleotidos ou proteina (sequencias)\n",
    "        \n",
    "    retmode : str\n",
    "        define o retmode para a captura de resultados da pesquisa na base de dados. Por defeito \n",
    "        o retmode é text mas pode ser adaptado para 'xml' quando é pretendido obter um ficheiro xml (e.g base de dados nucleotide, ou genbank)\n",
    "\n",
    "    display : str\n",
    "        define se é pretendido o output da função legível (útil para quando se quer refinar a pesquisa antes/sem gravar o ficheiro)\n",
    "        por defeito apresenta o resultado em formato legível\n",
    "    \n",
    "    save : str\n",
    "        define se é pretendido gravar ou não o ficheiro de resultados na pasta 'lit-search-output'\n",
    "        por defeito este parâmetro está inactivo\n",
    "\n",
    "            \n",
    "    \n",
    "    '''\n",
    "    # TODO criar ficheiros XML\n",
    "\n",
    "    # Importação de módulos\n",
    "    from Bio import Entrez\n",
    "    import os\n",
    "\n",
    "    # Definimos o email para os fins de pesquisa\n",
    "    Entrez.email = email\n",
    "        \n",
    "    # Secção egquery que pode ser skipable\n",
    "\n",
    "    handle = Entrez.egquery(term=term)\n",
    "    egq_res = Entrez.read(handle)\n",
    "\n",
    "    if rettype == 'fasta' or rettype == 'gb': retmax = 1\n",
    "\n",
    "    for _ in egq_res['eGQueryResult']:\n",
    "        if db in _.values():\n",
    "            if retmax > 1:\n",
    "                print('Encontrados {} resultados em {}. Irão ser processados {} resultados.\\n'.format(_['Count'],db,retmax))\n",
    "            else:\n",
    "                print('Encontrados {} resultados em {}. Irá ser processado 1 resultado.\\n'.format(_['Count'],db))\n",
    "\n",
    "    # Sacar as IDs\n",
    "    handle      = Entrez.esearch(db=db,term=term, retmode=retmode, retmax=retmax)\n",
    "    esearch_res = Entrez.read(handle)\n",
    "\n",
    "    lista_ids = esearch_res['IdList']\n",
    "    \n",
    "    # Transfere a informação\n",
    "    handle = Entrez.efetch(db=db,id=','.join(lista_ids),rettype=rettype,retmode=retmode)\n",
    "    fetch_res = handle.read()\n",
    "    \n",
    "    if display.upper() == 'Y':\n",
    "        print(fetch_res,sep='\\n')\n",
    "\n",
    "\n",
    "    # Verifica a condição de gravação de output como ficheiro de texto\n",
    "    \n",
    "    if retmode == 'xml': save = 'n'\n",
    "    \n",
    "    if save.upper() == 'Y':  \n",
    "\n",
    "        # Definimos a extensão em função da base de dados e retmode\n",
    "        \n",
    "\n",
    "        filename = term + type_handler(rettype,retmode)\n",
    "\n",
    "        if os.path.isfile(filename): \n",
    "            print('Ficheiro já existe. Não será criado novo ficheiro.')\n",
    "            return fetch_res\n",
    "\n",
    "        # Gera o ficheiro           \n",
    "        guardar_ficheiro(filename, fetch_res)\n",
    "\n",
    "\n",
    "    handle.close()\n",
    "\n",
    "    return fetch_res\n",
    "\n",
    "\n",
    "# SUBFUNÇÕES\n",
    "\n",
    "def guardar_ficheiro(ficheiro, content):\n",
    "\n",
    "    '''\n",
    "    Função que recebe um nome de ficheiro (ficheiro) e carrega o conteúdo \n",
    "    da variável content para esse ficheiro.\n",
    "    '''    \n",
    "    with open(ficheiro, 'w', encoding = 'utf-8') as _:\n",
    "        _.write(content)\n",
    "\n",
    "    print(f'{ficheiro} guardado com sucesso.')\n",
    "\n",
    "def type_handler(rettype,retmode):\n",
    "\n",
    "    '''\n",
    "    Função auxiliar da função pesquisa NCBI que permite\n",
    "    controlar o tipo de ficheiro de output da função\n",
    "    '''\n",
    "\n",
    "    if rettype == 'gb':\n",
    "        ext = '.gbk'\n",
    "        \n",
    "    elif rettype == 'fasta':\n",
    "        ext = '.fasta'\n",
    "        \n",
    "    elif retmode == 'xml':\n",
    "        ext = '.xml'\n",
    "\n",
    "    else: ext = '.txt'\n",
    "\n",
    "    return ext"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Função **pesquisa_ncbi** aplicada\n",
    "\n",
    "Passando agora à utilização da função com os parâmetros pretendidos. Foi definido o email e o termo de pesquisa pretendido de forma a diminuir o espetro de resultados"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "email = 'pg52648@alunos.uminho.pt'\n",
    "termo = 'hla-dqa1'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 2011 resultados em pubmed. Irão ser processados 5 resultados.\n",
      "\n",
      "1. Clin Immunol. 2024 Jan 8:109897. doi: 10.1016/j.clim.2024.109897. Online ahead\n",
      " of print.\n",
      "\n",
      "The genome-wide association study of serum IgE levels demonstrated the shared \n",
      "genetic background in allergic diseases.\n",
      "\n",
      "Lu HF(1), Chou CH(2), Lin YJ(3), Uchiyama S(4), Terao C(5), Wang YW(6), Yang \n",
      "JS(6), Liu TY(6), Wong HS(7), Chen SC(7), Tsai FJ(8).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan; Laboratory for Statistical and Translational Genetics, RIKEN \n",
      "Center for Integrative Medical Sciences, Yokohama, Japan; Genetic Center, \n",
      "Department of Medical Research, China Medical University Hospital, Taichung, \n",
      "Taiwan.\n",
      "(2)PhD Program for Health Science and Industry, College of Health Care, China \n",
      "Medical University, Taichung, Taiwan; Department of Health Services \n",
      "Administration, China Medical University, Taichung, Taiwan; Department of \n",
      "Medical Research, China Medical University Hospital, Taichung 404327, Taiwan.\n",
      "(3)Genetic Center, Department of Medical Research, China Medical University \n",
      "Hospital, Taichung, Taiwan; School of Chinese Medicine, China Medical \n",
      "University, Taichung, Taiwan.\n",
      "(4)Laboratory for Statistical and Translational Genetics, RIKEN Center for \n",
      "Integrative Medical Sciences, Yokohama, Japan.\n",
      "(5)Laboratory for Statistical and Translational Genetics, RIKEN Center for \n",
      "Integrative Medical Sciences, Yokohama, Japan; Clinical Research Center, \n",
      "Shizuoka General Hospital, Shizuoka, Japan; The Department of Applied Genetics, \n",
      "The School of Pharmaceutical Sciences, University of Shizuoka, Shizuoka, Japan.\n",
      "(6)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan.\n",
      "(7)Graduate Institute of Biomedical Informatics, College of Medical Science and \n",
      "Technology, Taipei Medical University, Taipei, Taiwan.\n",
      "(8)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan; School of Chinese Medicine, China Medical University, Taichung, \n",
      "Taiwan; School of Chinese Medicine, China Medical University, Taichung, Taiwan; \n",
      "Department of Biotechnology and Bioinformatics, Asia University, Taichung, \n",
      "Taiwan. Electronic address: d0704@mail.cmuh.org.tw.\n",
      "\n",
      "Immunoglobulin E (IgE) is highly related to a variety of atopic diseases, and \n",
      "several genome-wide association studies (GWASs) have demonstrated the \n",
      "association between genes and IgE. In this study, we conducted the largest \n",
      "genome-wide association study of IgE in a Taiwanese Han population. A total of 8 \n",
      "independent variants showed genome-wide significance, and rs147642819 at 6p21.32 \n",
      "was the most significant signal (p = 1.8 × 10-19). Among them, an intronic SNP \n",
      "of CD28, rs1181388, and an intergenic SNP, rs1002957030, on 11q23.2 were \n",
      "identified as novel signals for IgE. Seven of the loci were replicated \n",
      "successfully after a meta-analysis of the Japanese population. Among all the \n",
      "human leukocyte antigen (HLA) regions, HLA-DQA1*03:02 - HLA-DQB1*03:03 was the \n",
      "most significant haplotype (OR = 1.25, SE = 0.02, FDR = 1.6 × 10-14), \n",
      "corresponding to HLA-DQA1 Asp160 and HLA-DQB1 Leu87 amino residues. The genetic \n",
      "correlation showed significance between IgE and allergic diseases including \n",
      "asthma, atopic dermatitis, and pollinosis. The IgE PRS significantly correlated \n",
      "with the total IgE level. Furthermore, the top 10 quantile IgE polygenic risk \n",
      "score (PRS) group had a potentially higher risk for asthma, which was also \n",
      "replicated in the Taiwan Biobank and Japanese populations. IgE PRS may be used \n",
      "to aid in predicting the occurrence of allergic reactions before symptoms occur \n",
      "and biomarkers are detectable. Our study provided a more comprehensive \n",
      "understanding of the impact of genomic variants on serum IgE.\n",
      "\n",
      "Copyright © 2024. Published by Elsevier Inc.\n",
      "\n",
      "DOI: 10.1016/j.clim.2024.109897\n",
      "PMID: 38199299\n",
      "\n",
      "Conflict of interest statement: Declaration of competing interest All authors \n",
      "declare that they have no other relevant conflicts of interest.\n",
      "\n",
      "\n",
      "2. Commun Biol. 2024 Jan 5;7(1):41. doi: 10.1038/s42003-023-05701-5.\n",
      "\n",
      "Human leukocyte antigen-DQA1*04:01 and rs2040406 variants are associated with \n",
      "elevated risk of childhood Burkitt lymphoma.\n",
      "\n",
      "Liu Z(#)(1), Luo Y(#)(2)(3)(4)(5)(6)(7), Kirimunda S(8), Verboom M(9), Onabajo \n",
      "OO(1), Gouveia MH(10), Ogwang MD(11)(12), Kerchan P(12)(13), Reynolds SJ(14), \n",
      "Tenge CN(12)(15), Were PA(12)(16), Kuremu RT(12)(15), Wekesa WN(12)(15), Masalu \n",
      "N(17), Kawira E(12)(18), Kinyera T(11)(12), Otim I(11)(12), Legason ID(12)(13), \n",
      "Nabalende H(11)(12), Dhudha H(12)(18), Ayers LW(19), Bhatia K(1), Goedert JJ(1), \n",
      "Cole N(20), Luo W(20), Liu J(20), Manning M(20), Hicks B(20), Prokunina-Olsson \n",
      "L(1), Chagaluka G(21), Johnston WT(22), Mutalima N(22)(23), Borgstein E(21), \n",
      "Liomba GN(21), Kamiza S(21), Mkandawire N(21), Mitambo C(24), Molyneux EM(21), \n",
      "Newton R(22), Hsing AW(25), Mensah JE(26), Adjei AA(26), Hutchinson A(20), \n",
      "Carrington M(27)(28), Yeager M(20), Blasczyk R(9), Chanock SJ(1), Raychaudhuri \n",
      "S(3)(4)(5)(6)(7), Mbulaiteye SM(29).\n",
      "\n",
      "Author information:\n",
      "(1)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \n",
      "Rockville, MD, USA.\n",
      "(2)Kennedy Institute of Rheumatology, Nuffield Department of Orthopaedics, \n",
      "Rheumatology and Musculoskeletal Sciences, University of Oxford, Oxford, UK.\n",
      "(3)Center for Data Sciences, Brigham and Women's Hospital, Harvard Medical \n",
      "School, Boston, MA, USA.\n",
      "(4)Division of Rheumatology, Immunology, and Immunity, Brigham and Women's \n",
      "Hospital, Harvard Medical School, Boston, MA, USA.\n",
      "(5)Division of Genetics, Brigham and Women's Hospital, Harvard Medical School, \n",
      "Boston, MA, USA.\n",
      "(6)Department of Biomedical Informatics, Harvard Medical School, Boston, MA, \n",
      "USA.\n",
      "(7)Broad Institute of MIT and Harvard, Cambridge, MA, USA.\n",
      "(8)College of Health Sciences, Makerere University, Kampala, Uganda.\n",
      "(9)Institute of Transfusion Medicine and Transplant Engineering, Hanover, \n",
      "Germany.\n",
      "(10)Center for Research on Genomics & Global Health, NHGRI, National Institutes \n",
      "of Health, Bethesda, MD, USA.\n",
      "(11)St. Mary's Hospital, Lacor, Gulu, Uganda.\n",
      "(12)EMBLEM Study, African Field Epidemiology Network, Kampala, Uganda.\n",
      "(13)Kuluva Hospital, Arua, Uganda.\n",
      "(14)Division of Intramural Research, National Institute of Allergy and \n",
      "Infectious Diseases, National Institutes of Health, Bethesda, MD, USA.\n",
      "(15)Moi University College of Health Sciences, Eldoret, Kenya.\n",
      "(16)Academic Model Providing Access To Healthcare (AMPATH), Eldoret, Kenya.\n",
      "(17)Bugando Medical Center, Mwanza, Tanzania.\n",
      "(18)Shirati Health, Education, and Development Foundation, Shirati, Tanzania.\n",
      "(19)Department of Pathology, The Ohio State University, Columbus, OH, USA.\n",
      "(20)Cancer Genomics Research Laboratory, Frederick National Laboratory for \n",
      "Cancer Research, Frederick, MD, USA.\n",
      "(21)Departments of Pediatrics and Surgery, Kamuzu University of Health Sciences \n",
      "(formerly College of Medicine), University of Malawi, Blantyre, Malawi.\n",
      "(22)Epidemiology and Cancer Statistics Group, Department of Health Sciences, \n",
      "University of York, York, UK.\n",
      "(23)Cancer Epidemiology Unit, University of Oxford, Oxford, UK.\n",
      "(24)National Health Sciences Research Committee, Research Department, Ministry \n",
      "of Health, Lilongwe, Malawi.\n",
      "(25)Stanford Cancer Institute, Stanford University, Stanford, CA, USA.\n",
      "(26)University of Ghana Medical School, Accra, Ghana.\n",
      "(27)Basic Science Program, Frederick National Laboratory for Cancer Research, \n",
      "National Cancer Institute, Frederick, MD, USA and Laboratory of Integrative \n",
      "Cancer Immunology, Center for Cancer Research, National Cancer Institute, \n",
      "Bethesda, MD, USA.\n",
      "(28)Ragon Institute of MGH, MIT and Harvard, Cambridge, MA, USA.\n",
      "(29)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \n",
      "Rockville, MD, USA. mbulaits@mail.nih.gov.\n",
      "(#)Contributed equally\n",
      "\n",
      "Burkitt lymphoma (BL) is responsible for many childhood cancers in sub-Saharan \n",
      "Africa, where it is linked to recurrent or chronic infection by Epstein-Barr \n",
      "virus or Plasmodium falciparum. However, whether human leukocyte antigen (HLA) \n",
      "polymorphisms, which regulate immune response, are associated with BL has not \n",
      "been well investigated, which limits our understanding of BL etiology. Here we \n",
      "investigate this association among 4,645 children aged 0-15 years, 800 with BL, \n",
      "enrolled in Uganda, Tanzania, Kenya, and Malawi. HLA alleles are imputed with \n",
      "accuracy >90% for HLA class I and 85-89% for class II alleles. BL risk is \n",
      "elevated with HLA-DQA1*04:01 (adjusted odds ratio [OR] = 1.61, 95% confidence \n",
      "interval [CI] = 1.32-1.97, P = 3.71 × 10-6), with rs2040406(G) in HLA-DQA1 \n",
      "region (OR = 1.43, 95% CI = 1.26-1.63, P = 4.62 × 10-8), and with amino acid Gln \n",
      "at position 53 versus other variants in HLA-DQA1 (OR = 1.36, P = 2.06 × 10-6). \n",
      "The associations with HLA-DQA1*04:01 (OR = 1.29, P = 0.03) and rs2040406(G) \n",
      "(OR = 1.68, P = 0.019) persist in mutually adjusted models. The higher risk \n",
      "rs2040406(G) variant for BL is associated with decreased HLA-DQB1 expression in \n",
      "eQTLs in EBV transformed lymphocytes. Our results support the role of HLA \n",
      "variation in the etiology of BL and suggest that a promising area of research \n",
      "might be understanding the link between HLA variation and EBV control.\n",
      "\n",
      "© 2024. This is a U.S. Government work and not under copyright protection in the \n",
      "US; foreign copyright protection may apply.\n",
      "\n",
      "DOI: 10.1038/s42003-023-05701-5\n",
      "PMCID: PMC10770398\n",
      "PMID: 38182727 [Indexed for MEDLINE]\n",
      "\n",
      "Conflict of interest statement: The authors declare no competing interests.\n",
      "\n",
      "\n",
      "3. Mult Scler Relat Disord. 2023 Dec 14;82:105373. doi: \n",
      "10.1016/j.msard.2023.105373. Online ahead of print.\n",
      "\n",
      "Gene expression in multiple sclerosis during pregnancy based on integrated \n",
      "bioinformatics analysis.\n",
      "\n",
      "Hernández-Preciado MR(1), Torres-Mendoza BM(2), Mireles-Ramírez MA(3), \n",
      "Kobayashi-Gutiérrez A(4), Sánchez-Rosales NA(3), Vázquez-Valls E(5), \n",
      "Marquez-Pedroza J(6).\n",
      "\n",
      "Author information:\n",
      "(1)High Specialty Medical Unit, Pediatric Hospital, Western National Medical \n",
      "Center of the Mexican Institute of Social Security, Guadalajara 44340, Mexico; \n",
      "Department of Philosophical and Methodological Disciplines, University Health \n",
      "Sciences Center, University of Guadalajara, Guadalajara 44340, Mexico.\n",
      "(2)Department of Philosophical and Methodological Disciplines, University Health \n",
      "Sciences Center, University of Guadalajara, Guadalajara 44340, Mexico; \n",
      "Neurosciences Division, Western Biomedical Research Center (IMSS), Guadalajara \n",
      "44340, Mexico.\n",
      "(3)High Specialty Medical Unit, Western National Medical Center of the Mexican \n",
      "Institute of Social Security, Guadalajara 44340, Mexico.\n",
      "(4)High Specialty Medical Unit, Western National Medical Center of the Mexican \n",
      "Institute of Social Security, Guadalajara 44340, Mexico; PhD in Psychology of \n",
      "Health, University Health Sciences Center, University of Guadalajara, \n",
      "Guadalajara 44340, Mexico.\n",
      "(5)Directorate of Generation of Professional Resources, Research, and \n",
      "Development, Secretary of Health, Guadalajara 44100, Mexico.\n",
      "(6)Neurosciences Division, Western Biomedical Research Center (IMSS), \n",
      "Guadalajara 44340, Mexico. Electronic address: jaz180688@gmail.com.\n",
      "\n",
      "BACKGROUND: The modulation of the activity disease in patients with Multiple \n",
      "Sclerosis (MS) that occurs during pregnancy is a helpful model which could \n",
      "provide insight into central disease mechanisms and facilitate treatment. \n",
      "Therefore, the aim of the study was to identify differentially expressed genes \n",
      "in-silico to perform biological function pathway enrichment analysis and \n",
      "protein-protein interaction from pregnant women with MS.\n",
      "METHODS: Transcriptome data were obtained from the Gene Expression Omnibus (GEO) \n",
      "database. We selected the microarray dataset GSE17449. The gene expression \n",
      "dataset contains the data of mononuclear cells from four different groups \n",
      "sought, including seven healthy women (H), four healthy pregnant women (HP), \n",
      "eight women with multiple sclerosis (WMS), and nine women nine months pregnant \n",
      "with multiple sclerosis (PMS). The GSEA software was employed for enrichment \n",
      "analysis, and the REACTOME database was used for biological pathways. The \n",
      "protein-protein interaction (PPI) network was plotted with STRING. The databases \n",
      "used to identify the connection of DEGs with different signaling pathways were \n",
      "KEGG and WIKIPATHWAYS.\n",
      "RESULTS: We identified 42 differentially expressed genes in pregnant women with \n",
      "MS. The significant pathways included IL-10 signaling pathway, ErbB2 activates, \n",
      "the hemoglobin complex (HBD, HBB, HBA1, AHSP, and HBA2), IL-17 signaling pathway \n",
      "(LCN2 and MMP9), antigen processing and presentation, and Th17 cell \n",
      "differentiation (HLA-DQA1), Rap1 signaling pathway (ID1), NOD-Like receptor \n",
      "signaling pathway (CAMP and DEFA4), PD-L1 Signaling, Interferon gamma signaling \n",
      "(MMP9 and ARG1), Neutrophil degranulation (CAMP, DEFA4, ELANE, CEACAM8, S100P, \n",
      "CHI3L1, AZU1, OLFM4, CRISP3, LTF, ARG1, PGLYRP1, and TCN1). In the WIKIPATHWAYS \n",
      "set, significance was found Vitamin B12 metabolism (TCN1, HBB, and HBA2), and \n",
      "IL-18 signaling pathway (S100P).\n",
      "CONCLUSION: This study can be used to understand several essential target genes \n",
      "and pathways identified in the present study, which may serve as feasible \n",
      "targets for MS therapies.\n",
      "\n",
      "Copyright © 2023 Elsevier B.V. All rights reserved.\n",
      "\n",
      "DOI: 10.1016/j.msard.2023.105373\n",
      "PMID: 38154347\n",
      "\n",
      "Conflict of interest statement: Declaration of Competing Interest The authors \n",
      "declare that they have no known competing financial interests or personal \n",
      "relationships that could have appeared to influence the work reported in this \n",
      "paper.\n",
      "\n",
      "\n",
      "4. Biomedicines. 2023 Dec 11;11(12):3273. doi: 10.3390/biomedicines11123273.\n",
      "\n",
      "Genetic Susceptibility of HLA Alleles to Non-Steroidal Anti-Inflammatory Drug \n",
      "Hypersensitivity in the Taiwanese Population.\n",
      "\n",
      "Chang SL(1)(2)(3)(4), Lai CH(2)(4)(5), Lin GC(6), Chen YM(2)(4)(6)(7)(8), Lee \n",
      "MH(9), Hsu HS(3)(10), Chen IC(6).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Anesthesiology, Taichung Veterans General Hospital, Taichung \n",
      "407219, Taiwan.\n",
      "(2)Department of Postbaccalaureate Medicine, College of Medicine, National Chung \n",
      "Hsing University, Taichung 402202, Taiwan.\n",
      "(3)Institute of Emergency and Critical Care Medicine, School of Medicine, \n",
      "National Yang Ming Chiao Tung University, Taipei 112304, Taiwan.\n",
      "(4)School of Medicine, National Yang Ming Chiao Tung University, Taipei 112304, \n",
      "Taiwan.\n",
      "(5)Department of Medicine and Cardiovascular Center, Taichung Veterans General \n",
      "Hospital, Taichung 407219, Taiwan.\n",
      "(6)Department of Medical Research, Taichung Veterans General Hospital, Taichung \n",
      "407219, Taiwan.\n",
      "(7)Division of Allergy, Immunology and Rheumatology, Department of Medical \n",
      "Research, Taichung Veterans General Hospital, Taichung 407219, Taiwan.\n",
      "(8)Institute of Biomedical Science and Rong Hsing Research Center for \n",
      "Translational Medicine, National Chung Hsing University, Taichung 402202, \n",
      "Taiwan.\n",
      "(9)Institute of Clinical Medicine, National Yang Ming Chiao Tung University, \n",
      "Taipei 112304, Taiwan.\n",
      "(10)Division of Thoracic Surgery, Department of Surgery, Taipei Veterans General \n",
      "Hospital, Taipei 112201, Taiwan.\n",
      "\n",
      "BACKGROUND: Human leukocyte antigen (HLA) genes are important in many immune \n",
      "processes and contribute to many adverse drug reactions. Whether genetic \n",
      "variations in the HLA region are associated with non-steroid anti-inflammatory \n",
      "drug (NSAID) hypersensitivity remains uncertain. Therefore, the aim of our study \n",
      "was to identify HLA genetic variations in patients with NSAID hypersensitivity \n",
      "in the Taiwanese population.\n",
      "METHODS: This hospital-based, retrospective case-control study enrolled 37,156 \n",
      "participants with NSAID exposure from the Taiwan Precision Medicine Initiative \n",
      "(TPMI), who were all genotyped and imputed to fine map HLA typing. Our study \n",
      "assigned 1217 cases to the NSAID allergy group and 12,170 controls to a matched \n",
      "group. Logistic regression analyses were utilized to explore associations \n",
      "between HLA alleles and NSAID hypersensitivity.\n",
      "RESULTS: Overall, 13,387 patients were genotyped for eight major HLA alleles. \n",
      "Allele frequencies were different between the two groups. In the NSAID allergy \n",
      "group, the genotype frequencies of HLA-A*02:01, HLA-A*34:01, and HLA-DQA1*06:01 \n",
      "were found to be markedly elevated compared to the control group, a significance \n",
      "that persisted even after applying the Bonferroni correction. Furthermore, the \n",
      "risk of NSAID allergy demonstrated a significant association with HLA-A*02:01 \n",
      "(OR = 1.29, p < 0.001) and HLA-A*34:01 (OR = 9.90, p = 0.001), in comparison to \n",
      "their respective counterparts. Notably, the genotype frequency of HLA-B*46:01 \n",
      "exhibited a significant increase in the severe allergy group when compared with \n",
      "the mild allergy group.\n",
      "CONCLUSIONS: We identified HLA genotypes linked to the onset and severity of \n",
      "NSAID hypersensitivity. Our findings establish a basis for precision \n",
      "prescription in future clinical applications.\n",
      "\n",
      "DOI: 10.3390/biomedicines11123273\n",
      "PMCID: PMC10741656\n",
      "PMID: 38137494\n",
      "\n",
      "Conflict of interest statement: The authors declare that the research was \n",
      "conducted in the absence of any commercial or financial relationship that could \n",
      "be construed as a potential conflict of interest. The authors have no relevant \n",
      "financial or non-financial interests to disclose. The funders had no role in the \n",
      "design of the study; in the collection, analyses, or interpretation of data; in \n",
      "the writing of the manuscript; or in the decision to publish the results.\n",
      "\n",
      "\n",
      "5. JCI Insight. 2023 Dec 8;8(23):e171417. doi: 10.1172/jci.insight.171417.\n",
      "\n",
      "Single-cell transcriptomics reveals variations in monocytes and Tregs between \n",
      "gout flare and remission.\n",
      "\n",
      "Yu H(1), Xue W(1), Yu H(1), Song Y(1), Liu X(1), Qin L(1), Wang S(1), Bao H(1), \n",
      "Gu H(1), Chen G(1), Zhao D(1), Tu Y(1), Cheng J(1), Wang L(1), Ai Z(2), Hu D(1), \n",
      "Wang L(1), Peng A(1).\n",
      "\n",
      "Author information:\n",
      "(1)Center for Nephrology and Clinical Metabolomics and Division of Nephrology, \n",
      "Shanghai Tenth People's Hospital, and.\n",
      "(2)Department of Medical Statistics, Tongji University School of Medicine, \n",
      "Shanghai, China.\n",
      "\n",
      "Gout commonly manifests as a painful, self-limiting inflammatory arthritis. \n",
      "Nevertheless, the understanding of the inflammatory and immune responses \n",
      "underlying gout flares and remission remains ambiguous. Here, based on \n",
      "single-cell RNA-Seq and an independent validation cohort, we identified the \n",
      "potential mechanism of gout flare, which likely involves the upregulation of \n",
      "HLA-DQA1+ nonclassical monocytes and is related to antigen processing and \n",
      "presentation. Furthermore, Tregs also play an essential role in the suppressive \n",
      "capacity during gout remission. Cell communication analysis suggested the \n",
      "existence of altered crosstalk between monocytes and other T cell types, such as \n",
      "Tregs. Moreover, we observed the systemic upregulation of inflammatory and \n",
      "cytokine genes, primarily in classical monocytes, during gout flares. All \n",
      "monocyte subtypes showed increased arachidonic acid metabolic activity along \n",
      "with upregulation of prostaglandin-endoperoxide synthase 2 (PTGS2). We also \n",
      "detected a decrease in blood arachidonic acid and an increase in leukotriene B4 \n",
      "levels during gout flares. In summary, our study illustrates the distinctive \n",
      "immune cell responses and systemic inflammation patterns that characterize the \n",
      "transition from gout flares to remission, and it suggests that blood monocyte \n",
      "subtypes and Tregs are potential intervention targets for preventing recurrent \n",
      "gout attacks and progression.\n",
      "\n",
      "DOI: 10.1172/jci.insight.171417\n",
      "PMID: 38063198 [Indexed for MEDLINE]\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1. Clin Immunol. 2024 Jan 8:109897. doi: 10.1016/j.clim.2024.109897. Online ahead\\n of print.\\n\\nThe genome-wide association study of serum IgE levels demonstrated the shared \\ngenetic background in allergic diseases.\\n\\nLu HF(1), Chou CH(2), Lin YJ(3), Uchiyama S(4), Terao C(5), Wang YW(6), Yang \\nJS(6), Liu TY(6), Wong HS(7), Chen SC(7), Tsai FJ(8).\\n\\nAuthor information:\\n(1)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; Laboratory for Statistical and Translational Genetics, RIKEN \\nCenter for Integrative Medical Sciences, Yokohama, Japan; Genetic Center, \\nDepartment of Medical Research, China Medical University Hospital, Taichung, \\nTaiwan.\\n(2)PhD Program for Health Science and Industry, College of Health Care, China \\nMedical University, Taichung, Taiwan; Department of Health Services \\nAdministration, China Medical University, Taichung, Taiwan; Department of \\nMedical Research, China Medical University Hospital, Taichung 404327, Taiwan.\\n(3)Genetic Center, Department of Medical Research, China Medical University \\nHospital, Taichung, Taiwan; School of Chinese Medicine, China Medical \\nUniversity, Taichung, Taiwan.\\n(4)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan.\\n(5)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan; Clinical Research Center, \\nShizuoka General Hospital, Shizuoka, Japan; The Department of Applied Genetics, \\nThe School of Pharmaceutical Sciences, University of Shizuoka, Shizuoka, Japan.\\n(6)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan.\\n(7)Graduate Institute of Biomedical Informatics, College of Medical Science and \\nTechnology, Taipei Medical University, Taipei, Taiwan.\\n(8)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; School of Chinese Medicine, China Medical University, Taichung, \\nTaiwan; School of Chinese Medicine, China Medical University, Taichung, Taiwan; \\nDepartment of Biotechnology and Bioinformatics, Asia University, Taichung, \\nTaiwan. Electronic address: d0704@mail.cmuh.org.tw.\\n\\nImmunoglobulin E (IgE) is highly related to a variety of atopic diseases, and \\nseveral genome-wide association studies (GWASs) have demonstrated the \\nassociation between genes and IgE. In this study, we conducted the largest \\ngenome-wide association study of IgE in a Taiwanese Han population. A total of 8 \\nindependent variants showed genome-wide significance, and rs147642819 at 6p21.32 \\nwas the most significant signal (p\\u202f=\\u202f1.8\\u202f×\\u202f10-19). Among them, an intronic SNP \\nof CD28, rs1181388, and an intergenic SNP, rs1002957030, on 11q23.2 were \\nidentified as novel signals for IgE. Seven of the loci were replicated \\nsuccessfully after a meta-analysis of the Japanese population. Among all the \\nhuman leukocyte antigen (HLA) regions, HLA-DQA1*03:02 - HLA-DQB1*03:03 was the \\nmost significant haplotype (OR\\u202f=\\u202f1.25, SE\\u202f=\\u202f0.02, FDR\\u202f=\\u202f1.6\\u202f×\\u202f10-14), \\ncorresponding to HLA-DQA1 Asp160 and HLA-DQB1 Leu87 amino residues. The genetic \\ncorrelation showed significance between IgE and allergic diseases including \\nasthma, atopic dermatitis, and pollinosis. The IgE PRS significantly correlated \\nwith the total IgE level. Furthermore, the top 10 quantile IgE polygenic risk \\nscore (PRS) group had a potentially higher risk for asthma, which was also \\nreplicated in the Taiwan Biobank and Japanese populations. IgE PRS may be used \\nto aid in predicting the occurrence of allergic reactions before symptoms occur \\nand biomarkers are detectable. Our study provided a more comprehensive \\nunderstanding of the impact of genomic variants on serum IgE.\\n\\nCopyright © 2024. Published by Elsevier Inc.\\n\\nDOI: 10.1016/j.clim.2024.109897\\nPMID: 38199299\\n\\nConflict of interest statement: Declaration of competing interest All authors \\ndeclare that they have no other relevant conflicts of interest.\\n\\n\\n2. Commun Biol. 2024 Jan 5;7(1):41. doi: 10.1038/s42003-023-05701-5.\\n\\nHuman leukocyte antigen-DQA1*04:01 and rs2040406 variants are associated with \\nelevated risk of childhood Burkitt lymphoma.\\n\\nLiu Z(#)(1), Luo Y(#)(2)(3)(4)(5)(6)(7), Kirimunda S(8), Verboom M(9), Onabajo \\nOO(1), Gouveia MH(10), Ogwang MD(11)(12), Kerchan P(12)(13), Reynolds SJ(14), \\nTenge CN(12)(15), Were PA(12)(16), Kuremu RT(12)(15), Wekesa WN(12)(15), Masalu \\nN(17), Kawira E(12)(18), Kinyera T(11)(12), Otim I(11)(12), Legason ID(12)(13), \\nNabalende H(11)(12), Dhudha H(12)(18), Ayers LW(19), Bhatia K(1), Goedert JJ(1), \\nCole N(20), Luo W(20), Liu J(20), Manning M(20), Hicks B(20), Prokunina-Olsson \\nL(1), Chagaluka G(21), Johnston WT(22), Mutalima N(22)(23), Borgstein E(21), \\nLiomba GN(21), Kamiza S(21), Mkandawire N(21), Mitambo C(24), Molyneux EM(21), \\nNewton R(22), Hsing AW(25), Mensah JE(26), Adjei AA(26), Hutchinson A(20), \\nCarrington M(27)(28), Yeager M(20), Blasczyk R(9), Chanock SJ(1), Raychaudhuri \\nS(3)(4)(5)(6)(7), Mbulaiteye SM(29).\\n\\nAuthor information:\\n(1)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \\nRockville, MD, USA.\\n(2)Kennedy Institute of Rheumatology, Nuffield Department of Orthopaedics, \\nRheumatology and Musculoskeletal Sciences, University of Oxford, Oxford, UK.\\n(3)Center for Data Sciences, Brigham and Women's Hospital, Harvard Medical \\nSchool, Boston, MA, USA.\\n(4)Division of Rheumatology, Immunology, and Immunity, Brigham and Women's \\nHospital, Harvard Medical School, Boston, MA, USA.\\n(5)Division of Genetics, Brigham and Women's Hospital, Harvard Medical School, \\nBoston, MA, USA.\\n(6)Department of Biomedical Informatics, Harvard Medical School, Boston, MA, \\nUSA.\\n(7)Broad Institute of MIT and Harvard, Cambridge, MA, USA.\\n(8)College of Health Sciences, Makerere University, Kampala, Uganda.\\n(9)Institute of Transfusion Medicine and Transplant Engineering, Hanover, \\nGermany.\\n(10)Center for Research on Genomics & Global Health, NHGRI, National Institutes \\nof Health, Bethesda, MD, USA.\\n(11)St. Mary's Hospital, Lacor, Gulu, Uganda.\\n(12)EMBLEM Study, African Field Epidemiology Network, Kampala, Uganda.\\n(13)Kuluva Hospital, Arua, Uganda.\\n(14)Division of Intramural Research, National Institute of Allergy and \\nInfectious Diseases, National Institutes of Health, Bethesda, MD, USA.\\n(15)Moi University College of Health Sciences, Eldoret, Kenya.\\n(16)Academic Model Providing Access To Healthcare (AMPATH), Eldoret, Kenya.\\n(17)Bugando Medical Center, Mwanza, Tanzania.\\n(18)Shirati Health, Education, and Development Foundation, Shirati, Tanzania.\\n(19)Department of Pathology, The Ohio State University, Columbus, OH, USA.\\n(20)Cancer Genomics Research Laboratory, Frederick National Laboratory for \\nCancer Research, Frederick, MD, USA.\\n(21)Departments of Pediatrics and Surgery, Kamuzu University of Health Sciences \\n(formerly College of Medicine), University of Malawi, Blantyre, Malawi.\\n(22)Epidemiology and Cancer Statistics Group, Department of Health Sciences, \\nUniversity of York, York, UK.\\n(23)Cancer Epidemiology Unit, University of Oxford, Oxford, UK.\\n(24)National Health Sciences Research Committee, Research Department, Ministry \\nof Health, Lilongwe, Malawi.\\n(25)Stanford Cancer Institute, Stanford University, Stanford, CA, USA.\\n(26)University of Ghana Medical School, Accra, Ghana.\\n(27)Basic Science Program, Frederick National Laboratory for Cancer Research, \\nNational Cancer Institute, Frederick, MD, USA and Laboratory of Integrative \\nCancer Immunology, Center for Cancer Research, National Cancer Institute, \\nBethesda, MD, USA.\\n(28)Ragon Institute of MGH, MIT and Harvard, Cambridge, MA, USA.\\n(29)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \\nRockville, MD, USA. mbulaits@mail.nih.gov.\\n(#)Contributed equally\\n\\nBurkitt lymphoma (BL) is responsible for many childhood cancers in sub-Saharan \\nAfrica, where it is linked to recurrent or chronic infection by Epstein-Barr \\nvirus or Plasmodium falciparum. However, whether human leukocyte antigen (HLA) \\npolymorphisms, which regulate immune response, are associated with BL has not \\nbeen well investigated, which limits our understanding of BL etiology. Here we \\ninvestigate this association among 4,645 children aged 0-15 years, 800 with BL, \\nenrolled in Uganda, Tanzania, Kenya, and Malawi. HLA alleles are imputed with \\naccuracy >90% for HLA class I and 85-89% for class II alleles. BL risk is \\nelevated with HLA-DQA1*04:01 (adjusted odds ratio [OR]\\u2009=\\u20091.61, 95% confidence \\ninterval [CI]\\u2009=\\u20091.32-1.97, P\\u2009=\\u20093.71\\u2009×\\u200910-6), with rs2040406(G) in HLA-DQA1 \\nregion (OR\\u2009=\\u20091.43, 95% CI\\u2009=\\u20091.26-1.63, P\\u2009=\\u20094.62\\u2009×\\u200910-8), and with amino acid Gln \\nat position 53 versus other variants in HLA-DQA1 (OR\\u2009=\\u20091.36, P\\u2009=\\u20092.06\\u2009×\\u200910-6). \\nThe associations with HLA-DQA1*04:01 (OR\\u2009=\\u20091.29, P\\u2009=\\u20090.03) and rs2040406(G) \\n(OR\\u2009=\\u20091.68, P\\u2009=\\u20090.019) persist in mutually adjusted models. The higher risk \\nrs2040406(G) variant for BL is associated with decreased HLA-DQB1 expression in \\neQTLs in EBV transformed lymphocytes. Our results support the role of HLA \\nvariation in the etiology of BL and suggest that a promising area of research \\nmight be understanding the link between HLA variation and EBV control.\\n\\n© 2024. This is a U.S. Government work and not under copyright protection in the \\nUS; foreign copyright protection may apply.\\n\\nDOI: 10.1038/s42003-023-05701-5\\nPMCID: PMC10770398\\nPMID: 38182727 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n3. Mult Scler Relat Disord. 2023 Dec 14;82:105373. doi: \\n10.1016/j.msard.2023.105373. Online ahead of print.\\n\\nGene expression in multiple sclerosis during pregnancy based on integrated \\nbioinformatics analysis.\\n\\nHernández-Preciado MR(1), Torres-Mendoza BM(2), Mireles-Ramírez MA(3), \\nKobayashi-Gutiérrez A(4), Sánchez-Rosales NA(3), Vázquez-Valls E(5), \\nMarquez-Pedroza J(6).\\n\\nAuthor information:\\n(1)High Specialty Medical Unit, Pediatric Hospital, Western National Medical \\nCenter of the Mexican Institute of Social Security, Guadalajara 44340, Mexico; \\nDepartment of Philosophical and Methodological Disciplines, University Health \\nSciences Center, University of Guadalajara, Guadalajara 44340, Mexico.\\n(2)Department of Philosophical and Methodological Disciplines, University Health \\nSciences Center, University of Guadalajara, Guadalajara 44340, Mexico; \\nNeurosciences Division, Western Biomedical Research Center (IMSS), Guadalajara \\n44340, Mexico.\\n(3)High Specialty Medical Unit, Western National Medical Center of the Mexican \\nInstitute of Social Security, Guadalajara 44340, Mexico.\\n(4)High Specialty Medical Unit, Western National Medical Center of the Mexican \\nInstitute of Social Security, Guadalajara 44340, Mexico; PhD in Psychology of \\nHealth, University Health Sciences Center, University of Guadalajara, \\nGuadalajara 44340, Mexico.\\n(5)Directorate of Generation of Professional Resources, Research, and \\nDevelopment, Secretary of Health, Guadalajara 44100, Mexico.\\n(6)Neurosciences Division, Western Biomedical Research Center (IMSS), \\nGuadalajara 44340, Mexico. Electronic address: jaz180688@gmail.com.\\n\\nBACKGROUND: The modulation of the activity disease in patients with Multiple \\nSclerosis (MS) that occurs during pregnancy is a helpful model which could \\nprovide insight into central disease mechanisms and facilitate treatment. \\nTherefore, the aim of the study was to identify differentially expressed genes \\nin-silico to perform biological function pathway enrichment analysis and \\nprotein-protein interaction from pregnant women with MS.\\nMETHODS: Transcriptome data were obtained from the Gene Expression Omnibus (GEO) \\ndatabase. We selected the microarray dataset GSE17449. The gene expression \\ndataset contains the data of mononuclear cells from four different groups \\nsought, including seven healthy women (H), four healthy pregnant women (HP), \\neight women with multiple sclerosis (WMS), and nine women nine months pregnant \\nwith multiple sclerosis (PMS). The GSEA software was employed for enrichment \\nanalysis, and the REACTOME database was used for biological pathways. The \\nprotein-protein interaction (PPI) network was plotted with STRING. The databases \\nused to identify the connection of DEGs with different signaling pathways were \\nKEGG and WIKIPATHWAYS.\\nRESULTS: We identified 42 differentially expressed genes in pregnant women with \\nMS. The significant pathways included IL-10 signaling pathway, ErbB2 activates, \\nthe hemoglobin complex (HBD, HBB, HBA1, AHSP, and HBA2), IL-17 signaling pathway \\n(LCN2 and MMP9), antigen processing and presentation, and Th17 cell \\ndifferentiation (HLA-DQA1), Rap1 signaling pathway (ID1), NOD-Like receptor \\nsignaling pathway (CAMP and DEFA4), PD-L1 Signaling, Interferon gamma signaling \\n(MMP9 and ARG1), Neutrophil degranulation (CAMP, DEFA4, ELANE, CEACAM8, S100P, \\nCHI3L1, AZU1, OLFM4, CRISP3, LTF, ARG1, PGLYRP1, and TCN1). In the WIKIPATHWAYS \\nset, significance was found Vitamin B12 metabolism (TCN1, HBB, and HBA2), and \\nIL-18 signaling pathway (S100P).\\nCONCLUSION: This study can be used to understand several essential target genes \\nand pathways identified in the present study, which may serve as feasible \\ntargets for MS therapies.\\n\\nCopyright © 2023 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.msard.2023.105373\\nPMID: 38154347\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n4. Biomedicines. 2023 Dec 11;11(12):3273. doi: 10.3390/biomedicines11123273.\\n\\nGenetic Susceptibility of HLA Alleles to Non-Steroidal Anti-Inflammatory Drug \\nHypersensitivity in the Taiwanese Population.\\n\\nChang SL(1)(2)(3)(4), Lai CH(2)(4)(5), Lin GC(6), Chen YM(2)(4)(6)(7)(8), Lee \\nMH(9), Hsu HS(3)(10), Chen IC(6).\\n\\nAuthor information:\\n(1)Department of Anesthesiology, Taichung Veterans General Hospital, Taichung \\n407219, Taiwan.\\n(2)Department of Postbaccalaureate Medicine, College of Medicine, National Chung \\nHsing University, Taichung 402202, Taiwan.\\n(3)Institute of Emergency and Critical Care Medicine, School of Medicine, \\nNational Yang Ming Chiao Tung University, Taipei 112304, Taiwan.\\n(4)School of Medicine, National Yang Ming Chiao Tung University, Taipei 112304, \\nTaiwan.\\n(5)Department of Medicine and Cardiovascular Center, Taichung Veterans General \\nHospital, Taichung 407219, Taiwan.\\n(6)Department of Medical Research, Taichung Veterans General Hospital, Taichung \\n407219, Taiwan.\\n(7)Division of Allergy, Immunology and Rheumatology, Department of Medical \\nResearch, Taichung Veterans General Hospital, Taichung 407219, Taiwan.\\n(8)Institute of Biomedical Science and Rong Hsing Research Center for \\nTranslational Medicine, National Chung Hsing University, Taichung 402202, \\nTaiwan.\\n(9)Institute of Clinical Medicine, National Yang Ming Chiao Tung University, \\nTaipei 112304, Taiwan.\\n(10)Division of Thoracic Surgery, Department of Surgery, Taipei Veterans General \\nHospital, Taipei 112201, Taiwan.\\n\\nBACKGROUND: Human leukocyte antigen (HLA) genes are important in many immune \\nprocesses and contribute to many adverse drug reactions. Whether genetic \\nvariations in the HLA region are associated with non-steroid anti-inflammatory \\ndrug (NSAID) hypersensitivity remains uncertain. Therefore, the aim of our study \\nwas to identify HLA genetic variations in patients with NSAID hypersensitivity \\nin the Taiwanese population.\\nMETHODS: This hospital-based, retrospective case-control study enrolled 37,156 \\nparticipants with NSAID exposure from the Taiwan Precision Medicine Initiative \\n(TPMI), who were all genotyped and imputed to fine map HLA typing. Our study \\nassigned 1217 cases to the NSAID allergy group and 12,170 controls to a matched \\ngroup. Logistic regression analyses were utilized to explore associations \\nbetween HLA alleles and NSAID hypersensitivity.\\nRESULTS: Overall, 13,387 patients were genotyped for eight major HLA alleles. \\nAllele frequencies were different between the two groups. In the NSAID allergy \\ngroup, the genotype frequencies of HLA-A*02:01, HLA-A*34:01, and HLA-DQA1*06:01 \\nwere found to be markedly elevated compared to the control group, a significance \\nthat persisted even after applying the Bonferroni correction. Furthermore, the \\nrisk of NSAID allergy demonstrated a significant association with HLA-A*02:01 \\n(OR = 1.29, p < 0.001) and HLA-A*34:01 (OR = 9.90, p = 0.001), in comparison to \\ntheir respective counterparts. Notably, the genotype frequency of HLA-B*46:01 \\nexhibited a significant increase in the severe allergy group when compared with \\nthe mild allergy group.\\nCONCLUSIONS: We identified HLA genotypes linked to the onset and severity of \\nNSAID hypersensitivity. Our findings establish a basis for precision \\nprescription in future clinical applications.\\n\\nDOI: 10.3390/biomedicines11123273\\nPMCID: PMC10741656\\nPMID: 38137494\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationship that could \\nbe construed as a potential conflict of interest. The authors have no relevant \\nfinancial or non-financial interests to disclose. The funders had no role in the \\ndesign of the study; in the collection, analyses, or interpretation of data; in \\nthe writing of the manuscript; or in the decision to publish the results.\\n\\n\\n5. JCI Insight. 2023 Dec 8;8(23):e171417. doi: 10.1172/jci.insight.171417.\\n\\nSingle-cell transcriptomics reveals variations in monocytes and Tregs between \\ngout flare and remission.\\n\\nYu H(1), Xue W(1), Yu H(1), Song Y(1), Liu X(1), Qin L(1), Wang S(1), Bao H(1), \\nGu H(1), Chen G(1), Zhao D(1), Tu Y(1), Cheng J(1), Wang L(1), Ai Z(2), Hu D(1), \\nWang L(1), Peng A(1).\\n\\nAuthor information:\\n(1)Center for Nephrology and Clinical Metabolomics and Division of Nephrology, \\nShanghai Tenth People's Hospital, and.\\n(2)Department of Medical Statistics, Tongji University School of Medicine, \\nShanghai, China.\\n\\nGout commonly manifests as a painful, self-limiting inflammatory arthritis. \\nNevertheless, the understanding of the inflammatory and immune responses \\nunderlying gout flares and remission remains ambiguous. Here, based on \\nsingle-cell RNA-Seq and an independent validation cohort, we identified the \\npotential mechanism of gout flare, which likely involves the upregulation of \\nHLA-DQA1+ nonclassical monocytes and is related to antigen processing and \\npresentation. Furthermore, Tregs also play an essential role in the suppressive \\ncapacity during gout remission. Cell communication analysis suggested the \\nexistence of altered crosstalk between monocytes and other T cell types, such as \\nTregs. Moreover, we observed the systemic upregulation of inflammatory and \\ncytokine genes, primarily in classical monocytes, during gout flares. All \\nmonocyte subtypes showed increased arachidonic acid metabolic activity along \\nwith upregulation of prostaglandin-endoperoxide synthase 2 (PTGS2). We also \\ndetected a decrease in blood arachidonic acid and an increase in leukotriene B4 \\nlevels during gout flares. In summary, our study illustrates the distinctive \\nimmune cell responses and systemic inflammation patterns that characterize the \\ntransition from gout flares to remission, and it suggests that blood monocyte \\nsubtypes and Tregs are potential intervention targets for preventing recurrent \\ngout attacks and progression.\\n\\nDOI: 10.1172/jci.insight.171417\\nPMID: 38063198 [Indexed for MEDLINE]\""
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,termo)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Como podemos observar, foram processados processados somente 5 resultados dos 2011 resultados encontrados. \n",
    "\n",
    "O primeiro resultado apresentado é um artigo intitulado de \"The genome-wide association study of serum IgE levels demonstrated the shared \n",
    "genetic background in allergic diseases\", publicado em janeiro de 2024 na revista Clinical Immunology. O estudo foi conduzido por 8 autores que se encontram listados acima em conjunto com informação relevante sobre os mesmos.\n",
    "\n",
    "No final do output podemos observar que tanto o DOI e o PMID são fornecidos. \n",
    "\n",
    "Podemos concluir que o output da função **pesquisa_ncbi** permite a extração de informações relevantes na análise dos genes a nível da análise de artigos científicos disponíveis em diversas bases de dados. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Flexibilidade da função pesquisa_ncbi\n",
    "\n",
    "Como foi descrito anteriormente, é possível adaptar os parâmetros da função de forma a atingir os objetivos esperados. Desta forma, é possível:\n",
    "\n",
    "- **Manipular quantidade de resultados processados**\n",
    "\n",
    "Podemos manipular a quantidade de resultados apresentados modificando o número do parâmetro \"retmax\". Na função seguinte foi alterado o parâmetro \"retmax\" para \"1\" de modo a ser processo um único resultado."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 2011 resultados em pubmed. Irá ser processado 1 resultado.\n",
      "\n",
      "1. Clin Immunol. 2024 Jan 8:109897. doi: 10.1016/j.clim.2024.109897. Online ahead\n",
      " of print.\n",
      "\n",
      "The genome-wide association study of serum IgE levels demonstrated the shared \n",
      "genetic background in allergic diseases.\n",
      "\n",
      "Lu HF(1), Chou CH(2), Lin YJ(3), Uchiyama S(4), Terao C(5), Wang YW(6), Yang \n",
      "JS(6), Liu TY(6), Wong HS(7), Chen SC(7), Tsai FJ(8).\n",
      "\n",
      "Author information:\n",
      "(1)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan; Laboratory for Statistical and Translational Genetics, RIKEN \n",
      "Center for Integrative Medical Sciences, Yokohama, Japan; Genetic Center, \n",
      "Department of Medical Research, China Medical University Hospital, Taichung, \n",
      "Taiwan.\n",
      "(2)PhD Program for Health Science and Industry, College of Health Care, China \n",
      "Medical University, Taichung, Taiwan; Department of Health Services \n",
      "Administration, China Medical University, Taichung, Taiwan; Department of \n",
      "Medical Research, China Medical University Hospital, Taichung 404327, Taiwan.\n",
      "(3)Genetic Center, Department of Medical Research, China Medical University \n",
      "Hospital, Taichung, Taiwan; School of Chinese Medicine, China Medical \n",
      "University, Taichung, Taiwan.\n",
      "(4)Laboratory for Statistical and Translational Genetics, RIKEN Center for \n",
      "Integrative Medical Sciences, Yokohama, Japan.\n",
      "(5)Laboratory for Statistical and Translational Genetics, RIKEN Center for \n",
      "Integrative Medical Sciences, Yokohama, Japan; Clinical Research Center, \n",
      "Shizuoka General Hospital, Shizuoka, Japan; The Department of Applied Genetics, \n",
      "The School of Pharmaceutical Sciences, University of Shizuoka, Shizuoka, Japan.\n",
      "(6)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan.\n",
      "(7)Graduate Institute of Biomedical Informatics, College of Medical Science and \n",
      "Technology, Taipei Medical University, Taipei, Taiwan.\n",
      "(8)Department of Medical Research, China Medical University Hospital, Taichung \n",
      "404327, Taiwan; School of Chinese Medicine, China Medical University, Taichung, \n",
      "Taiwan; School of Chinese Medicine, China Medical University, Taichung, Taiwan; \n",
      "Department of Biotechnology and Bioinformatics, Asia University, Taichung, \n",
      "Taiwan. Electronic address: d0704@mail.cmuh.org.tw.\n",
      "\n",
      "Immunoglobulin E (IgE) is highly related to a variety of atopic diseases, and \n",
      "several genome-wide association studies (GWASs) have demonstrated the \n",
      "association between genes and IgE. In this study, we conducted the largest \n",
      "genome-wide association study of IgE in a Taiwanese Han population. A total of 8 \n",
      "independent variants showed genome-wide significance, and rs147642819 at 6p21.32 \n",
      "was the most significant signal (p = 1.8 × 10-19). Among them, an intronic SNP \n",
      "of CD28, rs1181388, and an intergenic SNP, rs1002957030, on 11q23.2 were \n",
      "identified as novel signals for IgE. Seven of the loci were replicated \n",
      "successfully after a meta-analysis of the Japanese population. Among all the \n",
      "human leukocyte antigen (HLA) regions, HLA-DQA1*03:02 - HLA-DQB1*03:03 was the \n",
      "most significant haplotype (OR = 1.25, SE = 0.02, FDR = 1.6 × 10-14), \n",
      "corresponding to HLA-DQA1 Asp160 and HLA-DQB1 Leu87 amino residues. The genetic \n",
      "correlation showed significance between IgE and allergic diseases including \n",
      "asthma, atopic dermatitis, and pollinosis. The IgE PRS significantly correlated \n",
      "with the total IgE level. Furthermore, the top 10 quantile IgE polygenic risk \n",
      "score (PRS) group had a potentially higher risk for asthma, which was also \n",
      "replicated in the Taiwan Biobank and Japanese populations. IgE PRS may be used \n",
      "to aid in predicting the occurrence of allergic reactions before symptoms occur \n",
      "and biomarkers are detectable. Our study provided a more comprehensive \n",
      "understanding of the impact of genomic variants on serum IgE.\n",
      "\n",
      "Copyright © 2024. Published by Elsevier Inc.\n",
      "\n",
      "DOI: 10.1016/j.clim.2024.109897\n",
      "PMID: 38199299\n",
      "\n",
      "Conflict of interest statement: Declaration of competing interest All authors \n",
      "declare that they have no other relevant conflicts of interest.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'1. Clin Immunol. 2024 Jan 8:109897. doi: 10.1016/j.clim.2024.109897. Online ahead\\n of print.\\n\\nThe genome-wide association study of serum IgE levels demonstrated the shared \\ngenetic background in allergic diseases.\\n\\nLu HF(1), Chou CH(2), Lin YJ(3), Uchiyama S(4), Terao C(5), Wang YW(6), Yang \\nJS(6), Liu TY(6), Wong HS(7), Chen SC(7), Tsai FJ(8).\\n\\nAuthor information:\\n(1)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; Laboratory for Statistical and Translational Genetics, RIKEN \\nCenter for Integrative Medical Sciences, Yokohama, Japan; Genetic Center, \\nDepartment of Medical Research, China Medical University Hospital, Taichung, \\nTaiwan.\\n(2)PhD Program for Health Science and Industry, College of Health Care, China \\nMedical University, Taichung, Taiwan; Department of Health Services \\nAdministration, China Medical University, Taichung, Taiwan; Department of \\nMedical Research, China Medical University Hospital, Taichung 404327, Taiwan.\\n(3)Genetic Center, Department of Medical Research, China Medical University \\nHospital, Taichung, Taiwan; School of Chinese Medicine, China Medical \\nUniversity, Taichung, Taiwan.\\n(4)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan.\\n(5)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan; Clinical Research Center, \\nShizuoka General Hospital, Shizuoka, Japan; The Department of Applied Genetics, \\nThe School of Pharmaceutical Sciences, University of Shizuoka, Shizuoka, Japan.\\n(6)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan.\\n(7)Graduate Institute of Biomedical Informatics, College of Medical Science and \\nTechnology, Taipei Medical University, Taipei, Taiwan.\\n(8)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; School of Chinese Medicine, China Medical University, Taichung, \\nTaiwan; School of Chinese Medicine, China Medical University, Taichung, Taiwan; \\nDepartment of Biotechnology and Bioinformatics, Asia University, Taichung, \\nTaiwan. Electronic address: d0704@mail.cmuh.org.tw.\\n\\nImmunoglobulin E (IgE) is highly related to a variety of atopic diseases, and \\nseveral genome-wide association studies (GWASs) have demonstrated the \\nassociation between genes and IgE. In this study, we conducted the largest \\ngenome-wide association study of IgE in a Taiwanese Han population. A total of 8 \\nindependent variants showed genome-wide significance, and rs147642819 at 6p21.32 \\nwas the most significant signal (p\\u202f=\\u202f1.8\\u202f×\\u202f10-19). Among them, an intronic SNP \\nof CD28, rs1181388, and an intergenic SNP, rs1002957030, on 11q23.2 were \\nidentified as novel signals for IgE. Seven of the loci were replicated \\nsuccessfully after a meta-analysis of the Japanese population. Among all the \\nhuman leukocyte antigen (HLA) regions, HLA-DQA1*03:02 - HLA-DQB1*03:03 was the \\nmost significant haplotype (OR\\u202f=\\u202f1.25, SE\\u202f=\\u202f0.02, FDR\\u202f=\\u202f1.6\\u202f×\\u202f10-14), \\ncorresponding to HLA-DQA1 Asp160 and HLA-DQB1 Leu87 amino residues. The genetic \\ncorrelation showed significance between IgE and allergic diseases including \\nasthma, atopic dermatitis, and pollinosis. The IgE PRS significantly correlated \\nwith the total IgE level. Furthermore, the top 10 quantile IgE polygenic risk \\nscore (PRS) group had a potentially higher risk for asthma, which was also \\nreplicated in the Taiwan Biobank and Japanese populations. IgE PRS may be used \\nto aid in predicting the occurrence of allergic reactions before symptoms occur \\nand biomarkers are detectable. Our study provided a more comprehensive \\nunderstanding of the impact of genomic variants on serum IgE.\\n\\nCopyright © 2024. Published by Elsevier Inc.\\n\\nDOI: 10.1016/j.clim.2024.109897\\nPMID: 38199299\\n\\nConflict of interest statement: Declaration of competing interest All authors \\ndeclare that they have no other relevant conflicts of interest.'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,termo,retmax=1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Podemos observar que, dos 2011 resultados obtidos, apenas 1 foi processado.\n",
    "\n",
    "Para além da alteração da quantidade de resultados processados é igualmente possível realizar as seguintes manipulações da função:\n",
    "\n",
    "-  **Manipular o display do output**\n",
    "\n",
    "-  **Manipulação da gravação do ficheiro**\n",
    "\n",
    "A manipulação do display do output é alcançada através da alteração do parâmetro \"display\" e o modo de gravação dos resultados processados é feita através da alteração do parâmetro \"save\"."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 2011 resultados em pubmed. Irão ser processados 5 resultados.\n",
      "\n",
      "Ficheiro já existe. Não será criado novo ficheiro.\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"1. Clin Immunol. 2024 Jan 8:109897. doi: 10.1016/j.clim.2024.109897. Online ahead\\n of print.\\n\\nThe genome-wide association study of serum IgE levels demonstrated the shared \\ngenetic background in allergic diseases.\\n\\nLu HF(1), Chou CH(2), Lin YJ(3), Uchiyama S(4), Terao C(5), Wang YW(6), Yang \\nJS(6), Liu TY(6), Wong HS(7), Chen SC(7), Tsai FJ(8).\\n\\nAuthor information:\\n(1)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; Laboratory for Statistical and Translational Genetics, RIKEN \\nCenter for Integrative Medical Sciences, Yokohama, Japan; Genetic Center, \\nDepartment of Medical Research, China Medical University Hospital, Taichung, \\nTaiwan.\\n(2)PhD Program for Health Science and Industry, College of Health Care, China \\nMedical University, Taichung, Taiwan; Department of Health Services \\nAdministration, China Medical University, Taichung, Taiwan; Department of \\nMedical Research, China Medical University Hospital, Taichung 404327, Taiwan.\\n(3)Genetic Center, Department of Medical Research, China Medical University \\nHospital, Taichung, Taiwan; School of Chinese Medicine, China Medical \\nUniversity, Taichung, Taiwan.\\n(4)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan.\\n(5)Laboratory for Statistical and Translational Genetics, RIKEN Center for \\nIntegrative Medical Sciences, Yokohama, Japan; Clinical Research Center, \\nShizuoka General Hospital, Shizuoka, Japan; The Department of Applied Genetics, \\nThe School of Pharmaceutical Sciences, University of Shizuoka, Shizuoka, Japan.\\n(6)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan.\\n(7)Graduate Institute of Biomedical Informatics, College of Medical Science and \\nTechnology, Taipei Medical University, Taipei, Taiwan.\\n(8)Department of Medical Research, China Medical University Hospital, Taichung \\n404327, Taiwan; School of Chinese Medicine, China Medical University, Taichung, \\nTaiwan; School of Chinese Medicine, China Medical University, Taichung, Taiwan; \\nDepartment of Biotechnology and Bioinformatics, Asia University, Taichung, \\nTaiwan. Electronic address: d0704@mail.cmuh.org.tw.\\n\\nImmunoglobulin E (IgE) is highly related to a variety of atopic diseases, and \\nseveral genome-wide association studies (GWASs) have demonstrated the \\nassociation between genes and IgE. In this study, we conducted the largest \\ngenome-wide association study of IgE in a Taiwanese Han population. A total of 8 \\nindependent variants showed genome-wide significance, and rs147642819 at 6p21.32 \\nwas the most significant signal (p\\u202f=\\u202f1.8\\u202f×\\u202f10-19). Among them, an intronic SNP \\nof CD28, rs1181388, and an intergenic SNP, rs1002957030, on 11q23.2 were \\nidentified as novel signals for IgE. Seven of the loci were replicated \\nsuccessfully after a meta-analysis of the Japanese population. Among all the \\nhuman leukocyte antigen (HLA) regions, HLA-DQA1*03:02 - HLA-DQB1*03:03 was the \\nmost significant haplotype (OR\\u202f=\\u202f1.25, SE\\u202f=\\u202f0.02, FDR\\u202f=\\u202f1.6\\u202f×\\u202f10-14), \\ncorresponding to HLA-DQA1 Asp160 and HLA-DQB1 Leu87 amino residues. The genetic \\ncorrelation showed significance between IgE and allergic diseases including \\nasthma, atopic dermatitis, and pollinosis. The IgE PRS significantly correlated \\nwith the total IgE level. Furthermore, the top 10 quantile IgE polygenic risk \\nscore (PRS) group had a potentially higher risk for asthma, which was also \\nreplicated in the Taiwan Biobank and Japanese populations. IgE PRS may be used \\nto aid in predicting the occurrence of allergic reactions before symptoms occur \\nand biomarkers are detectable. Our study provided a more comprehensive \\nunderstanding of the impact of genomic variants on serum IgE.\\n\\nCopyright © 2024. Published by Elsevier Inc.\\n\\nDOI: 10.1016/j.clim.2024.109897\\nPMID: 38199299\\n\\nConflict of interest statement: Declaration of competing interest All authors \\ndeclare that they have no other relevant conflicts of interest.\\n\\n\\n2. Commun Biol. 2024 Jan 5;7(1):41. doi: 10.1038/s42003-023-05701-5.\\n\\nHuman leukocyte antigen-DQA1*04:01 and rs2040406 variants are associated with \\nelevated risk of childhood Burkitt lymphoma.\\n\\nLiu Z(#)(1), Luo Y(#)(2)(3)(4)(5)(6)(7), Kirimunda S(8), Verboom M(9), Onabajo \\nOO(1), Gouveia MH(10), Ogwang MD(11)(12), Kerchan P(12)(13), Reynolds SJ(14), \\nTenge CN(12)(15), Were PA(12)(16), Kuremu RT(12)(15), Wekesa WN(12)(15), Masalu \\nN(17), Kawira E(12)(18), Kinyera T(11)(12), Otim I(11)(12), Legason ID(12)(13), \\nNabalende H(11)(12), Dhudha H(12)(18), Ayers LW(19), Bhatia K(1), Goedert JJ(1), \\nCole N(20), Luo W(20), Liu J(20), Manning M(20), Hicks B(20), Prokunina-Olsson \\nL(1), Chagaluka G(21), Johnston WT(22), Mutalima N(22)(23), Borgstein E(21), \\nLiomba GN(21), Kamiza S(21), Mkandawire N(21), Mitambo C(24), Molyneux EM(21), \\nNewton R(22), Hsing AW(25), Mensah JE(26), Adjei AA(26), Hutchinson A(20), \\nCarrington M(27)(28), Yeager M(20), Blasczyk R(9), Chanock SJ(1), Raychaudhuri \\nS(3)(4)(5)(6)(7), Mbulaiteye SM(29).\\n\\nAuthor information:\\n(1)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \\nRockville, MD, USA.\\n(2)Kennedy Institute of Rheumatology, Nuffield Department of Orthopaedics, \\nRheumatology and Musculoskeletal Sciences, University of Oxford, Oxford, UK.\\n(3)Center for Data Sciences, Brigham and Women's Hospital, Harvard Medical \\nSchool, Boston, MA, USA.\\n(4)Division of Rheumatology, Immunology, and Immunity, Brigham and Women's \\nHospital, Harvard Medical School, Boston, MA, USA.\\n(5)Division of Genetics, Brigham and Women's Hospital, Harvard Medical School, \\nBoston, MA, USA.\\n(6)Department of Biomedical Informatics, Harvard Medical School, Boston, MA, \\nUSA.\\n(7)Broad Institute of MIT and Harvard, Cambridge, MA, USA.\\n(8)College of Health Sciences, Makerere University, Kampala, Uganda.\\n(9)Institute of Transfusion Medicine and Transplant Engineering, Hanover, \\nGermany.\\n(10)Center for Research on Genomics & Global Health, NHGRI, National Institutes \\nof Health, Bethesda, MD, USA.\\n(11)St. Mary's Hospital, Lacor, Gulu, Uganda.\\n(12)EMBLEM Study, African Field Epidemiology Network, Kampala, Uganda.\\n(13)Kuluva Hospital, Arua, Uganda.\\n(14)Division of Intramural Research, National Institute of Allergy and \\nInfectious Diseases, National Institutes of Health, Bethesda, MD, USA.\\n(15)Moi University College of Health Sciences, Eldoret, Kenya.\\n(16)Academic Model Providing Access To Healthcare (AMPATH), Eldoret, Kenya.\\n(17)Bugando Medical Center, Mwanza, Tanzania.\\n(18)Shirati Health, Education, and Development Foundation, Shirati, Tanzania.\\n(19)Department of Pathology, The Ohio State University, Columbus, OH, USA.\\n(20)Cancer Genomics Research Laboratory, Frederick National Laboratory for \\nCancer Research, Frederick, MD, USA.\\n(21)Departments of Pediatrics and Surgery, Kamuzu University of Health Sciences \\n(formerly College of Medicine), University of Malawi, Blantyre, Malawi.\\n(22)Epidemiology and Cancer Statistics Group, Department of Health Sciences, \\nUniversity of York, York, UK.\\n(23)Cancer Epidemiology Unit, University of Oxford, Oxford, UK.\\n(24)National Health Sciences Research Committee, Research Department, Ministry \\nof Health, Lilongwe, Malawi.\\n(25)Stanford Cancer Institute, Stanford University, Stanford, CA, USA.\\n(26)University of Ghana Medical School, Accra, Ghana.\\n(27)Basic Science Program, Frederick National Laboratory for Cancer Research, \\nNational Cancer Institute, Frederick, MD, USA and Laboratory of Integrative \\nCancer Immunology, Center for Cancer Research, National Cancer Institute, \\nBethesda, MD, USA.\\n(28)Ragon Institute of MGH, MIT and Harvard, Cambridge, MA, USA.\\n(29)Division of Cancer Epidemiology and Genetics, National Cancer Institute, \\nRockville, MD, USA. mbulaits@mail.nih.gov.\\n(#)Contributed equally\\n\\nBurkitt lymphoma (BL) is responsible for many childhood cancers in sub-Saharan \\nAfrica, where it is linked to recurrent or chronic infection by Epstein-Barr \\nvirus or Plasmodium falciparum. However, whether human leukocyte antigen (HLA) \\npolymorphisms, which regulate immune response, are associated with BL has not \\nbeen well investigated, which limits our understanding of BL etiology. Here we \\ninvestigate this association among 4,645 children aged 0-15 years, 800 with BL, \\nenrolled in Uganda, Tanzania, Kenya, and Malawi. HLA alleles are imputed with \\naccuracy >90% for HLA class I and 85-89% for class II alleles. BL risk is \\nelevated with HLA-DQA1*04:01 (adjusted odds ratio [OR]\\u2009=\\u20091.61, 95% confidence \\ninterval [CI]\\u2009=\\u20091.32-1.97, P\\u2009=\\u20093.71\\u2009×\\u200910-6), with rs2040406(G) in HLA-DQA1 \\nregion (OR\\u2009=\\u20091.43, 95% CI\\u2009=\\u20091.26-1.63, P\\u2009=\\u20094.62\\u2009×\\u200910-8), and with amino acid Gln \\nat position 53 versus other variants in HLA-DQA1 (OR\\u2009=\\u20091.36, P\\u2009=\\u20092.06\\u2009×\\u200910-6). \\nThe associations with HLA-DQA1*04:01 (OR\\u2009=\\u20091.29, P\\u2009=\\u20090.03) and rs2040406(G) \\n(OR\\u2009=\\u20091.68, P\\u2009=\\u20090.019) persist in mutually adjusted models. The higher risk \\nrs2040406(G) variant for BL is associated with decreased HLA-DQB1 expression in \\neQTLs in EBV transformed lymphocytes. Our results support the role of HLA \\nvariation in the etiology of BL and suggest that a promising area of research \\nmight be understanding the link between HLA variation and EBV control.\\n\\n© 2024. This is a U.S. Government work and not under copyright protection in the \\nUS; foreign copyright protection may apply.\\n\\nDOI: 10.1038/s42003-023-05701-5\\nPMCID: PMC10770398\\nPMID: 38182727 [Indexed for MEDLINE]\\n\\nConflict of interest statement: The authors declare no competing interests.\\n\\n\\n3. Mult Scler Relat Disord. 2023 Dec 14;82:105373. doi: \\n10.1016/j.msard.2023.105373. Online ahead of print.\\n\\nGene expression in multiple sclerosis during pregnancy based on integrated \\nbioinformatics analysis.\\n\\nHernández-Preciado MR(1), Torres-Mendoza BM(2), Mireles-Ramírez MA(3), \\nKobayashi-Gutiérrez A(4), Sánchez-Rosales NA(3), Vázquez-Valls E(5), \\nMarquez-Pedroza J(6).\\n\\nAuthor information:\\n(1)High Specialty Medical Unit, Pediatric Hospital, Western National Medical \\nCenter of the Mexican Institute of Social Security, Guadalajara 44340, Mexico; \\nDepartment of Philosophical and Methodological Disciplines, University Health \\nSciences Center, University of Guadalajara, Guadalajara 44340, Mexico.\\n(2)Department of Philosophical and Methodological Disciplines, University Health \\nSciences Center, University of Guadalajara, Guadalajara 44340, Mexico; \\nNeurosciences Division, Western Biomedical Research Center (IMSS), Guadalajara \\n44340, Mexico.\\n(3)High Specialty Medical Unit, Western National Medical Center of the Mexican \\nInstitute of Social Security, Guadalajara 44340, Mexico.\\n(4)High Specialty Medical Unit, Western National Medical Center of the Mexican \\nInstitute of Social Security, Guadalajara 44340, Mexico; PhD in Psychology of \\nHealth, University Health Sciences Center, University of Guadalajara, \\nGuadalajara 44340, Mexico.\\n(5)Directorate of Generation of Professional Resources, Research, and \\nDevelopment, Secretary of Health, Guadalajara 44100, Mexico.\\n(6)Neurosciences Division, Western Biomedical Research Center (IMSS), \\nGuadalajara 44340, Mexico. Electronic address: jaz180688@gmail.com.\\n\\nBACKGROUND: The modulation of the activity disease in patients with Multiple \\nSclerosis (MS) that occurs during pregnancy is a helpful model which could \\nprovide insight into central disease mechanisms and facilitate treatment. \\nTherefore, the aim of the study was to identify differentially expressed genes \\nin-silico to perform biological function pathway enrichment analysis and \\nprotein-protein interaction from pregnant women with MS.\\nMETHODS: Transcriptome data were obtained from the Gene Expression Omnibus (GEO) \\ndatabase. We selected the microarray dataset GSE17449. The gene expression \\ndataset contains the data of mononuclear cells from four different groups \\nsought, including seven healthy women (H), four healthy pregnant women (HP), \\neight women with multiple sclerosis (WMS), and nine women nine months pregnant \\nwith multiple sclerosis (PMS). The GSEA software was employed for enrichment \\nanalysis, and the REACTOME database was used for biological pathways. The \\nprotein-protein interaction (PPI) network was plotted with STRING. The databases \\nused to identify the connection of DEGs with different signaling pathways were \\nKEGG and WIKIPATHWAYS.\\nRESULTS: We identified 42 differentially expressed genes in pregnant women with \\nMS. The significant pathways included IL-10 signaling pathway, ErbB2 activates, \\nthe hemoglobin complex (HBD, HBB, HBA1, AHSP, and HBA2), IL-17 signaling pathway \\n(LCN2 and MMP9), antigen processing and presentation, and Th17 cell \\ndifferentiation (HLA-DQA1), Rap1 signaling pathway (ID1), NOD-Like receptor \\nsignaling pathway (CAMP and DEFA4), PD-L1 Signaling, Interferon gamma signaling \\n(MMP9 and ARG1), Neutrophil degranulation (CAMP, DEFA4, ELANE, CEACAM8, S100P, \\nCHI3L1, AZU1, OLFM4, CRISP3, LTF, ARG1, PGLYRP1, and TCN1). In the WIKIPATHWAYS \\nset, significance was found Vitamin B12 metabolism (TCN1, HBB, and HBA2), and \\nIL-18 signaling pathway (S100P).\\nCONCLUSION: This study can be used to understand several essential target genes \\nand pathways identified in the present study, which may serve as feasible \\ntargets for MS therapies.\\n\\nCopyright © 2023 Elsevier B.V. All rights reserved.\\n\\nDOI: 10.1016/j.msard.2023.105373\\nPMID: 38154347\\n\\nConflict of interest statement: Declaration of Competing Interest The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper.\\n\\n\\n4. Biomedicines. 2023 Dec 11;11(12):3273. doi: 10.3390/biomedicines11123273.\\n\\nGenetic Susceptibility of HLA Alleles to Non-Steroidal Anti-Inflammatory Drug \\nHypersensitivity in the Taiwanese Population.\\n\\nChang SL(1)(2)(3)(4), Lai CH(2)(4)(5), Lin GC(6), Chen YM(2)(4)(6)(7)(8), Lee \\nMH(9), Hsu HS(3)(10), Chen IC(6).\\n\\nAuthor information:\\n(1)Department of Anesthesiology, Taichung Veterans General Hospital, Taichung \\n407219, Taiwan.\\n(2)Department of Postbaccalaureate Medicine, College of Medicine, National Chung \\nHsing University, Taichung 402202, Taiwan.\\n(3)Institute of Emergency and Critical Care Medicine, School of Medicine, \\nNational Yang Ming Chiao Tung University, Taipei 112304, Taiwan.\\n(4)School of Medicine, National Yang Ming Chiao Tung University, Taipei 112304, \\nTaiwan.\\n(5)Department of Medicine and Cardiovascular Center, Taichung Veterans General \\nHospital, Taichung 407219, Taiwan.\\n(6)Department of Medical Research, Taichung Veterans General Hospital, Taichung \\n407219, Taiwan.\\n(7)Division of Allergy, Immunology and Rheumatology, Department of Medical \\nResearch, Taichung Veterans General Hospital, Taichung 407219, Taiwan.\\n(8)Institute of Biomedical Science and Rong Hsing Research Center for \\nTranslational Medicine, National Chung Hsing University, Taichung 402202, \\nTaiwan.\\n(9)Institute of Clinical Medicine, National Yang Ming Chiao Tung University, \\nTaipei 112304, Taiwan.\\n(10)Division of Thoracic Surgery, Department of Surgery, Taipei Veterans General \\nHospital, Taipei 112201, Taiwan.\\n\\nBACKGROUND: Human leukocyte antigen (HLA) genes are important in many immune \\nprocesses and contribute to many adverse drug reactions. Whether genetic \\nvariations in the HLA region are associated with non-steroid anti-inflammatory \\ndrug (NSAID) hypersensitivity remains uncertain. Therefore, the aim of our study \\nwas to identify HLA genetic variations in patients with NSAID hypersensitivity \\nin the Taiwanese population.\\nMETHODS: This hospital-based, retrospective case-control study enrolled 37,156 \\nparticipants with NSAID exposure from the Taiwan Precision Medicine Initiative \\n(TPMI), who were all genotyped and imputed to fine map HLA typing. Our study \\nassigned 1217 cases to the NSAID allergy group and 12,170 controls to a matched \\ngroup. Logistic regression analyses were utilized to explore associations \\nbetween HLA alleles and NSAID hypersensitivity.\\nRESULTS: Overall, 13,387 patients were genotyped for eight major HLA alleles. \\nAllele frequencies were different between the two groups. In the NSAID allergy \\ngroup, the genotype frequencies of HLA-A*02:01, HLA-A*34:01, and HLA-DQA1*06:01 \\nwere found to be markedly elevated compared to the control group, a significance \\nthat persisted even after applying the Bonferroni correction. Furthermore, the \\nrisk of NSAID allergy demonstrated a significant association with HLA-A*02:01 \\n(OR = 1.29, p < 0.001) and HLA-A*34:01 (OR = 9.90, p = 0.001), in comparison to \\ntheir respective counterparts. Notably, the genotype frequency of HLA-B*46:01 \\nexhibited a significant increase in the severe allergy group when compared with \\nthe mild allergy group.\\nCONCLUSIONS: We identified HLA genotypes linked to the onset and severity of \\nNSAID hypersensitivity. Our findings establish a basis for precision \\nprescription in future clinical applications.\\n\\nDOI: 10.3390/biomedicines11123273\\nPMCID: PMC10741656\\nPMID: 38137494\\n\\nConflict of interest statement: The authors declare that the research was \\nconducted in the absence of any commercial or financial relationship that could \\nbe construed as a potential conflict of interest. The authors have no relevant \\nfinancial or non-financial interests to disclose. The funders had no role in the \\ndesign of the study; in the collection, analyses, or interpretation of data; in \\nthe writing of the manuscript; or in the decision to publish the results.\\n\\n\\n5. JCI Insight. 2023 Dec 8;8(23):e171417. doi: 10.1172/jci.insight.171417.\\n\\nSingle-cell transcriptomics reveals variations in monocytes and Tregs between \\ngout flare and remission.\\n\\nYu H(1), Xue W(1), Yu H(1), Song Y(1), Liu X(1), Qin L(1), Wang S(1), Bao H(1), \\nGu H(1), Chen G(1), Zhao D(1), Tu Y(1), Cheng J(1), Wang L(1), Ai Z(2), Hu D(1), \\nWang L(1), Peng A(1).\\n\\nAuthor information:\\n(1)Center for Nephrology and Clinical Metabolomics and Division of Nephrology, \\nShanghai Tenth People's Hospital, and.\\n(2)Department of Medical Statistics, Tongji University School of Medicine, \\nShanghai, China.\\n\\nGout commonly manifests as a painful, self-limiting inflammatory arthritis. \\nNevertheless, the understanding of the inflammatory and immune responses \\nunderlying gout flares and remission remains ambiguous. Here, based on \\nsingle-cell RNA-Seq and an independent validation cohort, we identified the \\npotential mechanism of gout flare, which likely involves the upregulation of \\nHLA-DQA1+ nonclassical monocytes and is related to antigen processing and \\npresentation. Furthermore, Tregs also play an essential role in the suppressive \\ncapacity during gout remission. Cell communication analysis suggested the \\nexistence of altered crosstalk between monocytes and other T cell types, such as \\nTregs. Moreover, we observed the systemic upregulation of inflammatory and \\ncytokine genes, primarily in classical monocytes, during gout flares. All \\nmonocyte subtypes showed increased arachidonic acid metabolic activity along \\nwith upregulation of prostaglandin-endoperoxide synthase 2 (PTGS2). We also \\ndetected a decrease in blood arachidonic acid and an increase in leukotriene B4 \\nlevels during gout flares. In summary, our study illustrates the distinctive \\nimmune cell responses and systemic inflammation patterns that characterize the \\ntransition from gout flares to remission, and it suggests that blood monocyte \\nsubtypes and Tregs are potential intervention targets for preventing recurrent \\ngout attacks and progression.\\n\\nDOI: 10.1172/jci.insight.171417\\nPMID: 38063198 [Indexed for MEDLINE]\""
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Pesquisa com output, sem display legível e permitindo a gravação dos resultados processados\n",
    "pesquisa_ncbi(email,termo,save='y',display='n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Como podemos observar pelo resultado apresentado, não ocorreu o display legível dos resultados e o ficheiro foi gravado.\n",
    "\n",
    "Também é possível realizar a seguinte manipulação da função:\n",
    "\n",
    "- **Manipular bases de dados onde a pesquisa é efetuada**\n",
    "\n",
    "O parâmetro \"db\" pode ser alterado de forma a alterar a base de dados onde a pesquisa é realizada. No seguinte exemplo podemos observar que a base de dados selecionada para efeitos de pesquisa dos resultados foi a \"nuccore\"."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1769 resultados em nuccore. Irão ser processados 5 resultados.\n",
      "\n",
      "LOCUS       OY986031                7342 bp    DNA     linear   PRI 27-DEC-2023\n",
      "DEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\n",
      "            HLA-DQA1*02:01:01:NEW.\n",
      "ACCESSION   OY986031\n",
      "VERSION     OY986031.1\n",
      "DBLINK      BioProject: PRJEB71396\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1\n",
      "  AUTHORS   French,A.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\n",
      "            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\n",
      "            Kingdom\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..7342\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /clone=\"AN302966\"\n",
      "     gene            732..5954\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     CDS             join(732..813,4508..4753,5166..5447,5800..5954)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /function=\"antigen presenting molecule\"\n",
      "                     /codon_start=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "                     /product=\"MHC class II antigen\"\n",
      "                     /protein_id=\"CAK6681862.1\"\n",
      "                     /translation=\"MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGP\n",
      "                     SGQFTHEFDGDEEFYVDLERKETVWKLPLFHRLRFDPQFALTNIAVLKHNLNILIKRS\n",
      "                     NSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSET\n",
      "                     SFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTETV\n",
      "                     VCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL\"\n",
      "     exon            732..813\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          814..4507\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            4508..4753\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          4754..5165\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            5166..5447\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          5448..5799\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            5800..5954\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=4\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "ORIGIN      \n",
      "        1 ttgtaaccca gaaaatcctg aaggatgcca tataattgat gacctcatct atccatgagg\n",
      "       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tccaagcact\n",
      "      121 ttgggaggct gaggcaggca gatcacgagg tcaggagttc aagaccagcg tggccaacat\n",
      "      181 agtgaaactc tgtctctact aaaaatacag aaattagccg ggcatggtgg caggcacctg\n",
      "      241 cagtcccagc tacttgggag ggtgaggcag gagaatcgct tgaacccggg aggcaaaggt\n",
      "      301 tgcagtgagc cgagaccatg ccattgtacc tcagcctggg tgacagagtg agactacgtc\n",
      "      361 tcgaaaaata aagaaaaaga aaagaaaaag aaaaaaaaag aaaaattccc atcccttttg\n",
      "      421 cgaatggcag acatgcacac accagagaag attccaattt agtgtcttcc ctctcttcat\n",
      "      481 agaacaattc ctcaagtcca ctctgagtag aggctgcatc acaacaaggg gattgccctg\n",
      "      541 tctccttcca gggctcttaa tagaaactct tcaactagta actgagatgt caccatgggg\n",
      "      601 gatttttcta attggccaaa acctgacttg gcagggtttg gtttgggtgt cttcagattt\n",
      "      661 ccttgtcttg aggtcctcac aattactcta cagctcagaa cagcaactgc tgaggctgcc\n",
      "      721 ttgggaagaa gatgatccta aacaaagctc tgatgctggg ggccctcgcc ctgaccaccg\n",
      "      781 tgatgagccc ttgtggaggt gaagacattg tgggtgagtg cgtgagtgag gaatgttctc\n",
      "      841 tggagctgaa aaacagtaaa ttgaaggaaa agagagaaag cgatttgcag agaaattgta\n",
      "      901 gagatttcct aagacccctt tcagtattaa gagaattaaa aattatagct gttcctcctt\n",
      "      961 caggaaacca gagccccaac ctactctttt tgttatgtat gcttttgtgt tcactaagga\n",
      "     1021 tgctattctg tttatattat attcagtgac tacagcctgg aggtctctat gtcattccat\n",
      "     1081 catgattgcc tcaaaaatta gtgaggtttc catcagtgga taatttttta ttattaaaaa\n",
      "     1141 tgtatgaagt gtcattctca aatttccctg aacaactttt gaagattttc ggatgtctcc\n",
      "     1201 tgtagtagat cttggggtcg ttccatcaat tatatactct atagatatta aaaaagttgc\n",
      "     1261 ccgtttcttt ctctcagact tactcacatt tccacatggg aactggcaca ggtggggagt\n",
      "     1321 aggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\n",
      "     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\n",
      "     1441 ctaagctttg taaatagtga aaacatggat gtcaattgtt tttacatatt tctattacag\n",
      "     1501 gtatagcttc acatttcttt tctttagcaa aataagggat ccttttagtt taaaattgag\n",
      "     1561 aagtagaaaa aattggtaaa ttaaatcatt ttattctcaa attatcaacc caaattacct\n",
      "     1621 gttcttcacc tcatctaata aagtcctata aaaagaaaag tgggccagac atggtggctc\n",
      "     1681 atgcctgtaa tcccagcact ttgggaggcc gaagcaggag gatcatttga gcctgggagt\n",
      "     1741 ttgagaccag cctgggcaac acagcaagac ctcatctcta ccaaaaaata aaataaaaat\n",
      "     1801 tagccaggca tggtagtgca tgcctgtggt gccagctact cagaaggctg cagtgggagg\n",
      "     1861 agcacttgag tccaggaggt ggaagctgca gtgagccatg atggcaccac tacactccag\n",
      "     1921 ccagggcaac agagagagac cctgtctcaa aaagaaagcg gaaagaaaga gagaaaggaa\n",
      "     1981 ggaaagaagg aaagaaggaa gcaaggaagg agaaagggaa gggaagaaag aagaaagaaa\n",
      "     2041 gaaagaaaac agaaggaagg aagcacagat taattatttg gtctcttagt ctcctctgcc\n",
      "     2101 tttgtcgtcc atctcttccc acctctcttc atgcattcct ttctccctct tccctttcag\n",
      "     2161 gatccatctc tgactccctg ctcctttata gagatggaca tgagtttgta aaacaaaagt\n",
      "     2221 tgaaaagtca gatagttaaa aggggaagta aactggaagg tactctaaac tttcacaacc\n",
      "     2281 ttattaaccg tggcagctcc cattctgatt ttgttcagca gtggaagttt caccctctcc\n",
      "     2341 tccagagcgc ttggcttctt tgttccaaat ttcctttctt cagcctcaca ccagagtgcc\n",
      "     2401 ctggtcaggc tcagctcatc cattaggcac aatgtgggca gtgcagggga acctccatac\n",
      "     2461 tgtaaagcca catgagaatg ttttaactcc ttttaaaatt ataaaaaaat gaaattgtag\n",
      "     2521 agcctaagaa aatgttttaa cttttaattc agcctatatt atattgtctt tataccaatt\n",
      "     2581 cagtcataaa atataatttt ccatattttt atggaggaag gcgtccacac aagcaagagt\n",
      "     2641 gcttggggct cacatgtcag aacgcatccc tgatcatggc tgatcctgac cttcgtgtgg\n",
      "     2701 ttctgctaac tatgtgcctg tcagtcttcc ccaaaatcta tgtggtcctc aaatataaca\n",
      "     2761 actgtcattc aatacacatg tttgagcacc cagtgagcta agttttaagg attcaaagat\n",
      "     2821 gaaaagtcat gctgtctccc ctgcagaggg tgctcagact agtgatggaa acagtatggg\n",
      "     2881 atgaaagaaa gcagaaggcc attgctgagc aggcagtgga ctcagcagag gctgaaacta\n",
      "     2941 tacaagtgac ttggttccag ctgggccagc aggataacca gacgaaaaga aggattgcat\n",
      "     3001 atattccata tatatttatg tttgaacaaa gagtcaaggt ttattgcaag gataaggagg\n",
      "     3061 ctttgttggt ggcctgttaa gaccatccag cgtggtcata ctggataggg aagaaggtga\n",
      "     3121 gctggaagag ggatagacaa acttggatgg ccagatgttg agatggagga gctggaggtc\n",
      "     3181 ataacgtggt caaaaacatg ttgatgagag gacttagcta caaagttgtt aacttaagca\n",
      "     3241 gaaacctcaa ggattgattt tatgatttct ccaggaagtc ctaaaagata atttcatttc\n",
      "     3301 agggaggaaa acaacagacc actgcaaaga ccaggaacat gaaaggataa tgtagtttgg\n",
      "     3361 tttgcttggc agatacttgt gaaagatgtt ggactgtaag gctgtcaata tcctcctcgc\n",
      "     3421 agaacttact acagtacatt gtatctgctc cctcacctac ctgactctcc cactattcag\n",
      "     3481 tttgttcctt aatggtagac catgcctgat tggtgtttta cacttcccct gctatgtctg\n",
      "     3541 atacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta aaaggcttac\n",
      "     3601 acatgtcttg accagaatgt tcagtttggc tcatttggct ggagtcatac tgcatggctg\n",
      "     3661 ccattctgct ctggcatcct cagagaagca cactgcccat tagaggaaaa agggtgaata\n",
      "     3721 taaatgttga gtcagaacac tgcagacatt tagtaacctc cttcagagga aaaaaagggt\n",
      "     3781 ggggggaatg acagaaatcc aaaaactagt agagcttcca ctttttcatt tcagaagaaa\n",
      "     3841 tcagttactc tcctctaagg accattacta ttaacaaaac agagacctta gaaggaagca\n",
      "     3901 ttatttattt atcatatatt ttgtaatgtt attaccgttc ttgttatact ctttcttata\n",
      "     3961 ccctaccatt gttagcagaa attattttaa attaataaga tcctgcatgc ttttcctttt\n",
      "     4021 tctaaaaaaa gaaagatctc tgtgtagaat gtcctgttct gagccagtcc tgagaggaaa\n",
      "     4081 ggaagtataa tcaatttgtt attaactgat gaaagaatta agtgaaagat aaaccttagg\n",
      "     4141 aagcagaggg aagttaatct atgactaaga aagttaagta ctctgataac tcattcattc\n",
      "     4201 cttcttctgt tcatttacat tatttaatca caagtccatg atgtgccagg cactcaggaa\n",
      "     4261 atagtgaaaa tcggacacgc gatattctgc ccttgtgtag cacacactgt agtgggaaag\n",
      "     4321 aaagtgcact tttaactgga caactatcaa cacgaagagg ggaggaagca ggggctggaa\n",
      "     4381 atgtccacag actttgccaa agacaaagcc cataatattt gaaagtcagt ttcttccatc\n",
      "     4441 attttgtgta ttaaggtttt ttattctcct gttctctgcc ttcctgcttg tcatcttcac\n",
      "     4501 tcatcagctg accacgttgc ctcttacggt gtaaacttgt accagtctta cggtccctct\n",
      "     4561 ggccagttca cccatgaatt tgatggagac gaggagttct atgtggacct ggagaggaag\n",
      "     4621 gagactgtct ggaagttgcc tctgttccac agacttagat ttgacccgca atttgcactg\n",
      "     4681 acaaacatcg ctgtgctaaa acataacttg aacatcctga ttaaacgctc caactctacc\n",
      "     4741 gctgctacca atggtatgtg tccaccattc tgcctttctt tactgatcta tccctttata\n",
      "     4801 ccaagtttca ttattttctt tccaagaggt ccccagatct tctcatggca attgctgaaa\n",
      "     4861 ttttatcatt tctcatctct aaaatcacat atccccatgt aatacaaggg tctttccatt\n",
      "     4921 atgcattcat taaatcattc taggagaggt ctcatcaacc tcctacttta ttaaacatgc\n",
      "     4981 ccacagagag aagggcacag gagtaaagca gaggcaatgt gtcattgctc ccaagtagaa\n",
      "     5041 ggtaaataag gcctctttga ccagcaggag aggaaatgct ggtaggaaga ctcttccagg\n",
      "     5101 atgtaatgca gaagctcagg gcagagctat tcacacttca caccagtgct gtttcctcac\n",
      "     5161 catagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\n",
      "     5221 caccctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca cctggctgag\n",
      "     5281 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\n",
      "     5341 ttccttcttc aagatcagtt acctcacctt cctcccttct gctgatgaga tttatgactg\n",
      "     5401 caaggtggag cactggggcc tggatgagcc tcttctgaaa cactggggta aggatgagtt\n",
      "     5461 tcaccatttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactctat\n",
      "     5521 gtcccaaaac ttgttttcca cacttcatga gtttctttta tctttttttt tttttgaaag\n",
      "     5581 aattaagcaa caaaagcaca gatttattaa aaaagaaagt acactccaca gggtgggagc\n",
      "     5641 aggcctgcca cttcatgggt ttctaataac agacttcact ctcctccctg agccaggggc\n",
      "     5701 cttgagtctt tgcagagcca accctccacc ccatcccatc cctcacacat gcacatgagc\n",
      "     5761 acactctgca ttctgacctc aacaacttca cttccacaga gcctgagatt ccagcaccta\n",
      "     5821 tgtcagagct cacagagact gtggtctgtg ccctggggtt gtctgtgggc ctcgtgggca\n",
      "     5881 ttgtggtggg gaccgtcttg atcatccgag gcctgcgttc agttggtgct tccagacacc\n",
      "     5941 aagggccctt gtgaatccca tcctgaaaaa gaaggtaagt ttgagatttg ttagagctga\n",
      "     6001 agctgcagga aggaaagtgg gaggaggctg tggacatgaa tgtggttgaa agttgtaggg\n",
      "     6061 gaattgggaa gtggcatgat gatgacacag gagccccctt ggacccatcg atctcatgtc\n",
      "     6121 tgtcctgttg caggtgcatc accatctaca gcagcggaag agtggacttg ctacatgacc\n",
      "     6181 tagcactatt ctctggcccg atttatcata ccctttttct cctgcaaatg tttctcctct\n",
      "     6241 taccttttct ctgctttttt ttccttaagc ttctgtatcc cctcagagct cacaaatgcc\n",
      "     6301 tttgaattct ttccctgacc tcctgatttt tttttctttt ctcaggtgtt acctactaag\n",
      "     6361 agatgcctgg ggtaagccgc ccagctacct aattcctcag taacatcgat ctaaaatctc\n",
      "     6421 catggaagca ataaattccc tttaagagat ctatgtcaaa tttttccatc tttcatccgg\n",
      "     6481 ggctgactga acctatggct aagaattggg acactctcat gtttcaagcc aatttcatct\n",
      "     6541 catttcccag atcatatttc atatccagta acacagaagc aaccaagtac aatatagcct\n",
      "     6601 gataatatgt tgatttctta gctgacatta atatttcttt cttctttgtg ttctcaccct\n",
      "     6661 tggcactgcc gcccatccct caattcaggc aacaatgaag ttaatggata ctctctgccc\n",
      "     6721 tttgctcaga attgttatag caaaaatttt aaaaccaaaa aataagtttg tactaatttc\n",
      "     6781 aatatggctt ttaaaagtat gatggagaaa taaattagga taaaggaact ttgaatcaca\n",
      "     6841 aaaatatcaa aagtaaaaat ttattctcaa aactttgaat ttgtaaagaa tgatgacagt\n",
      "     6901 agaagccttc ctctcccctc ctcgccttta gggaataaaa attctttagg taggaaaaga\n",
      "     6961 aatggaagtc agaaaaacat tagaaaaaga cagtaatgtg ggtatctgaa aaggaacaaa\n",
      "     7021 tacttattcc tcacataggg ttagtgacaa tgggaaaagg gataggagta gaagccacag\n",
      "     7081 acatatctag gagccctgaa tagaggcgca gtctgcctca cctcctgaat gaagctttgc\n",
      "     7141 tagataacca cgtagctttc cctgtgccac ccttgcatga agcagacagt atagtggata\n",
      "     7201 tggcaggatg tttctaggaa acatgccgat acaaaacaat gccagtatct tcagaaatcc\n",
      "     7261 ccagcccttt cccctcaccc ctcctggcta aggaaagcac tagcttatga gagaaaccct\n",
      "     7321 aggaggaaca acacagttga ga\n",
      "//\n",
      "\n",
      "LOCUS       OY986028                7419 bp    DNA     linear   PRI 27-DEC-2023\n",
      "DEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\n",
      "            HLA-DQA1*01:02:01:NEW.\n",
      "ACCESSION   OY986028\n",
      "VERSION     OY986028.1\n",
      "DBLINK      BioProject: PRJEB71396\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1\n",
      "  AUTHORS   French,A.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\n",
      "            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\n",
      "            Kingdom\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..7419\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /clone=\"AN302945\"\n",
      "     gene            728..6045\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     CDS             join(728..809,4591..4839,5253..5534,5891..6045)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /function=\"antigen presenting molecule\"\n",
      "                     /codon_start=1\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "                     /product=\"MHC class II antigen\"\n",
      "                     /protein_id=\"CAK6681859.1\"\n",
      "                     /translation=\"MILNKALLLGALALTTVMSPCGGEDIVADHVASCGVNLYQFYGP\n",
      "                     SGQYTHEFDGDEQFYVDLERKETAWRWPEFSKFGGFDPQGALRNMAVAKHNLNIMIKR\n",
      "                     YNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGQSVTEGVSE\n",
      "                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDQPLLKHWEPEIPAPMSELTET\n",
      "                     VVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGPL\"\n",
      "     exon            728..809\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     intron          810..4590\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     exon            4591..4839\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     intron          4840..5252\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     exon            5253..5534\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     intron          5535..5890\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "     exon            5891..6045\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=4\n",
      "                     /allele=\"HLA-DQA1*01:02:01:NEW\"\n",
      "ORIGIN      \n",
      "        1 ttgtaagcca gaacatcctg aaggatgccg tataactgat gacctcatct atccacgagg\n",
      "       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tctgagcact\n",
      "      121 ttgggaggct gagacgggca gatcatgagg tcaggagttc aagaccagcg tggccaacat\n",
      "      181 agtgaaaccc cgtctctact aaaaatacaa aaattagctg ggcgtggtgg caggtgcctg\n",
      "      241 tagtcccagc tacttgggag ggtgagggag gagaatcgct tgaatccggg aggcaaaggt\n",
      "      301 tgcagtgagc cgagaccttg ccattgtacc acagcctggg tgacagagtg agactacgtc\n",
      "      361 tcaaaaaaaa agaaaaagaa aagaaaaaga aaaaaaaaga aaaacaccca tccctcttgc\n",
      "      421 gactggcaga catgcacaca ccagagaaga ttccaatttc gtgtcctccc tttattcata\n",
      "      481 gaacaattcc tcaagtccac tctgagtaga ggctgcatca caaggggatt gccccgtctc\n",
      "      541 cttccagggc tcttaataca aactcttcaa ctagtaactg agatgtcacc ataggggatt\n",
      "      601 tttctaattg gccaaaacct gacttggcag ggtttggttt gggtgtcttc agatttcctt\n",
      "      661 gtcttgaggt cctcacaatt actctacagc tcagaacacc aactgctgag gctgccttgg\n",
      "      721 gaagaggatg atcctaaaca aagctctgct gctgggggcc ctcgctctga ccaccgtgat\n",
      "      781 gagcccctgt ggaggtgaag acattgtggg tgagtgcatg agtgagggat gttctctgga\n",
      "      841 gctgaaaaac agtaaattga aggaaaagag ataaagcgat ttgcagagaa actgtagaga\n",
      "      901 tttcctaagg gccctttcag tattaagaca attaaaaatt atagctgttc ctccttcagg\n",
      "      961 aaaccagagc cccaacctac tctttttgtt atgtatgctt ttgtgttcac taaggatgct\n",
      "     1021 attctgttta tattatattc agtgactaca gcctggaggt ctctatgtcg ttccgtcatg\n",
      "     1081 attgcctcaa aaattagtga ggtttccatc agtggataat tttttattat taaaaattta\n",
      "     1141 tgaagtgtca ttctcaaatt tccctgaaca acttttgaag cttttcggat gtctccggtg\n",
      "     1201 gtagatcttg gggtcattcc atcaattatg tactctatag atatttaaaa tgttgcccgt\n",
      "     1261 ttctttctct cagacttact cacatttcca catgggaact ggcacaggtg gggagtgggt\n",
      "     1321 aaaggagtcc agcaggctga atgccttcaa caatcatttt accacatggt cctcacttac\n",
      "     1381 tctcagctgc ctcatatgtg tcacctcaca agtaatcaaa taaaatgggc atgtggctaa\n",
      "     1441 gctttgtaaa tagtgaaaac atgaatgtca atttttttta catatttcta ttacagatat\n",
      "     1501 agcttcacat ttcttttctt tagcaaaata agggatcatt ttattttaaa attgagaagt\n",
      "     1561 agaaaaaatt ggtaaaataa atcattttat tctcaaatta tcaacccaaa ttacctgttc\n",
      "     1621 ttcacctcat ctaataaagt cctataaaaa gaaaagtggg ccagacaagg tggctcatgc\n",
      "     1681 ctgtaatccc agcactttgg gaggccgagg caggaggatc atttgagcct gggagcttga\n",
      "     1741 gaccagcctg ggcaacatag caagacctca tctctacaaa aaaagaaaaa gaaaaattag\n",
      "     1801 ccaggcatgg tggtgtgtgc ctgtggtgcc agctactcag aaggctgccg tgggaggacc\n",
      "     1861 acttgagccc atgaggcgga agctgcagtg agccatggtg gcaccactac actccagcca\n",
      "     1921 gggcaacaga gagagacgct gtctcaaaaa gaaagaggaa agaaagcgag gaaggaagga\n",
      "     1981 aggaaggaag gaaggaagga gaaggaagga aggaaggaaa agaaaagaaa agggaaggga\n",
      "     2041 ggaaggaagg aaaaaagaaa gacagaaaga aagaggaagg aaggaagaaa ggaagcacag\n",
      "     2101 attaattatt tggtctctta ctctcctctg cctttgtcgt ccatctcttt ccacctctct\n",
      "     2161 tgatgcatcc ctttctccct cttccgtttc aggagccatc tctggctccc tgctccttta\n",
      "     2221 tagagatgga cagtgggttt gtaaaacaaa agttgaaaag tcagatagtt aaaaggggaa\n",
      "     2281 gtgaactgga aggtacttta aactttcaca actttattaa ccgtggctgc tcccattatg\n",
      "     2341 attttgttcg gcagtggaag tttcacctgc ttctccagag cacttggcgt ctttgttcca\n",
      "     2401 aatttccttt cttcaacctc acaccagagt gccctggtca ggcttggctc attcattagg\n",
      "     2461 cacaatgtgg gcagtgcagg ggaccctcca tactgtaaag ccacatgaga aggttttaac\n",
      "     2521 tccttttaaa attagaaaaa taatggaata ttagagccta agaaaatgtt ttaactttta\n",
      "     2581 attcagccta gattatattg tctttatacc aattcagtca taaaatataa ttttccatat\n",
      "     2641 ttttgtggag gaagaggccc atacaagcaa gaatgctcgg ggctcacatg tcagaatgca\n",
      "     2701 gccttggtca tgactgatcc tggccttcgt atggttctgc tacctgtgtg cctgcccgtc\n",
      "     2761 ttccccaaaa tctatgtggt cctcgaatat agcaactgtc attcaataca catgtttgag\n",
      "     2821 cacacagtga gctaagtttt aaggattcaa agatgaaaag tcatgctgtc ttccctgcag\n",
      "     2881 agggtgctca gattagtgat ggaaacagta tggggtgcag gaaagcagaa ggccattgct\n",
      "     2941 gagcagggca gtggacccag cagaggctga aacaatacaa gtgacttggt tccaggtggg\n",
      "     3001 ccagcagggt gacgtcctct agcaaaattt ggcacccaag acaagtacca gaagaaaaga\n",
      "     3061 aagactgcat gtattccaca tatattcatg tttgaacaag gagtcaagat ttatggcaag\n",
      "     3121 gataaggagg ctttgttggt gacctgttaa gaccaaccag ggcagtcatg ctggataggg\n",
      "     3181 aagaaggtga gctggaagag gaacagacaa cttggacagc cagatgttga gacggagggg\n",
      "     3241 ttggaggtca taatgtggtc aaaaacatgt tgatgagagg actcagctac aaagttgtta\n",
      "     3301 acttaagcag aaacctcaag gatggatttt agaatttctc caggaagtcc taaaagataa\n",
      "     3361 ttccatttca gggagaaaaa caacagacca ctgcaaagac cagggaacat gaaaggataa\n",
      "     3421 tgtagtttgt cttgcttggc agatactttt gaaggatgtt ggactgtaag gctgtcgata\n",
      "     3481 tcctcctcag agaacttact acagtacatt gtatctgttc ccttacctac ctgactcttc\n",
      "     3541 cactattcag tttgttcctt aatggtagac catgcctgat tggtgtttta cacatctccg\n",
      "     3601 gctatgtctg acacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta\n",
      "     3661 aaaggcttac acatgtcttg acaaaaaagt ccagtttggc tcatttggct ggagtcgtac\n",
      "     3721 tgcatggctg ccattctgct ctggcatcct cagacaagca cactgcccat tagaggaaag\n",
      "     3781 agtgtgatat aagtgttgag tcagaatgct gtagacattt agtaacctcc ttcacaggaa\n",
      "     3841 aaaaaaaaaa aggtgggggg aatgacagaa atccaaaaac tagtagagct tccacttttc\n",
      "     3901 atttcagaag aaatcagttg ctctcctcta aggaccacta ctattaacaa aacagagacc\n",
      "     3961 ttagaaggag gcattgttta tttattatat attttgtaat gttattacca atcttgttat\n",
      "     4021 actctttctt ataccctaca attgttagca gaaattattt taaattaata agatcctgca\n",
      "     4081 tgcttttcct ttaaaaaaaa aaaaaaagaa agatctctgt gtagagtgtc ctattctgag\n",
      "     4141 ccagtcctga gaggaaagga agtataatca atttgttatt aaccaatgaa agaattaagt\n",
      "     4201 gaaagataaa tctcaggaag cagagggaag taaacctaat ttctgactaa gaaagctaaa\n",
      "     4261 tactatgata actcattcat tccttctttt gttcaattac attatttaat cataagtcca\n",
      "     4321 tgacgtgcca ggcactcagg aaatagtaaa aattggacat gcgatattct gcccttgtgt\n",
      "     4381 agcgcacact agagtgggaa agaaagtgca cttttaactg gacaactacc aacatgaaga\n",
      "     4441 ggggaggaag caggggctgg aaatgtccac agactgtgcc aaaaaatgaa gcccataata\n",
      "     4501 tttgaaagtc aggttcttcc atcattttgt gtattaaggt tctttcttcc tctgttctcc\n",
      "     4561 gccttcctgc ttgtcatctt cactcatcag ctgaccacgt tgcctcttgt ggtgtaaact\n",
      "     4621 tgtaccagtt ttacggtccc tctggccagt acacccatga atttgatgga gatgagcagt\n",
      "     4681 tctacgtgga cctggagagg aaggagactg cctggcggtg gcctgagttc agcaaatttg\n",
      "     4741 gaggttttga cccgcagggt gcactgagaa acatggctgt ggcaaaacac aacttgaaca\n",
      "     4801 tcatgattaa acgctacaac tctaccgctg ctaccaatgg tatgcgtcca ccattctgcc\n",
      "     4861 tctctttact taagctatcc ctccatacca gggttcatta ttttcttccc aagaggtccc\n",
      "     4921 cagatcttgt tatggcaatt gctgaaattt tatcatctcc catctctaaa atcacatatt\n",
      "     4981 cccatgtaat acaagggtct ttccattatg cattcagcaa atccttctag gagaggtctc\n",
      "     5041 atcaacctcc tactttatta aacatgccca cagagagaag ggcacaggaa taaagcagag\n",
      "     5101 gcaatgtgtc gttgctccca agcagaaggt aaataagacc tctttgacta tcaggtggtg\n",
      "     5161 aaatgctggt aggagggctc ttccaggatg taatgcagaa cttcagggca gagctattca\n",
      "     5221 cacttcacac cagtgctgtt tcctcaccac agaggttcct gaggtcacag tgttttccaa\n",
      "     5281 gtctcccgtg acactgggtc agcccaacac cctcatttgt cttgtggaca acatctttcc\n",
      "     5341 tcctgtggtc aacatcacat ggctgagcaa tgggcagtca gtcacagaag gtgtttctga\n",
      "     5401 gaccagcttc ctctccaaga gtgatcattc cttcttcaag atcagttacc tcaccttcct\n",
      "     5461 cccttctgct gatgagattt atgactgcaa ggtggagcac tggggcctgg accagcctct\n",
      "     5521 tctgaaacac tggggtaagg atgagtttca tcattttttg attctttctt gtctgtcaag\n",
      "     5581 ttcagaactt cctgcctttt actcctatat ccaaaaactt gttttccaca cttcatgggt\n",
      "     5641 ttcttttctg tctctctttt ttttttgaaa gaataaagca aaaaaagcag agatttattg\n",
      "     5701 aaaatgaaag tacactctac aggatgggag tgggcctgcc acttcatggt tttctaatga\n",
      "     5761 tagacttcac tcttctccct aagcctgggg ccttgagtct ttgcagagcc aaccctctac\n",
      "     5821 cccatcccat cccacacaca tgcacatgag cacactctgc attctgacct caacaacttc\n",
      "     5881 acttccacag agcctgagat tccagcccct atgtcagagc tcacagagac tgtggtctgt\n",
      "     5941 gccctggggt tgtctgtggg cctcatgggc attgtggtgg gcactgtctt catcatccaa\n",
      "     6001 ggcctgcgtt cagttggtgc ttccagacac caagggccat tgtgaatccc atcctggaag\n",
      "     6061 ggaaggtaag attgagactg gttacagttg aagcggcagt atgaaaggaa ggaaagtggg\n",
      "     6121 agggcgttgt ggacatgaat gtggttgaaa gttgtagggg aattgggaag tggcatgatg\n",
      "     6181 atgacacagg agccccctcg gacccattga tctcatgtct gccctgttgc aggtgcatcg\n",
      "     6241 ccatctacag gagcagaaga atggacttgc taaatgacct agcactattc tctggcccga\n",
      "     6301 tttatcatat cccttttctc ctccaaatat ttctcctctc accttttctc tgggacttaa\n",
      "     6361 gctgctatat cccctcagag ctcacaaatg cctttacatt ctttccctga cctcctgatt\n",
      "     6421 ttttttttct tttctcaaat gttacctaca cagacatgcc tggagtaagc cacccggcta\n",
      "     6481 cctaattcct cagtaacctc catctaaaat ctccaaggaa gcaataaatt ccttttatga\n",
      "     6541 gatctatgtc aaatttttcc atctttcatc cagggctgac tgaaactatg gctaataatt\n",
      "     6601 ggggtactct tatgtttcaa tccaatttaa cctcatttcc cagatcattt ttcatgtcca\n",
      "     6661 gtaacacaga agccaccaag tacagtatag cctgataata tgttgatttc ttagctgaca\n",
      "     6721 ttaatatttc ttgcttcctt gtgttcccac ccttggcact gccacccacc cctcaattca\n",
      "     6781 ggcaacaatg aaattaatgg ataccgtctg cccttggccc agaattgtta tagcaaaaat\n",
      "     6841 tttagaacca aaaaataagt ctgtactaat ttcaatgtgg cttttaaaag tatgacagag\n",
      "     6901 aaataagtta ggataaagga aatttgaatc tcaaaaatat caaaagtaaa aatgtattct\n",
      "     6961 caaaacttta aatttatgaa gaatgatgac agtagaagcc ttcctcttcc cttgtcacct\n",
      "     7021 tgagataaaa attctttagg caggaaaaga aatggaagtc agaaaaacat tagaataaga\n",
      "     7081 caataatgtg ggtgtctgaa aaggaacaaa tacttattcc tcacataggg ttagtgacaa\n",
      "     7141 tgggaaaagg gacgggagta gaagccacag acatatctag gagccctgaa tagaggcgca\n",
      "     7201 gtctgcctca cctcctgaat gaagctgtac tagataacca tgtagctttc cctgtgccac\n",
      "     7261 tctggcatga aggagacagt atagtgaata tggctacagg atgtttcttg gaaacatgcc\n",
      "     7321 aatatcttca gaaatcccca gccctttccc ctaacccctc ctggctaagg aaagcatgag\n",
      "     7381 cttatgagag aaaccctagg aagaacaaca cagttgaga\n",
      "//\n",
      "\n",
      "LOCUS       OY986027                7342 bp    DNA     linear   PRI 27-DEC-2023\n",
      "DEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\n",
      "            HLA-DQA1*02:01:01:NEW.\n",
      "ACCESSION   OY986027\n",
      "VERSION     OY986027.1\n",
      "DBLINK      BioProject: PRJEB71396\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1\n",
      "  AUTHORS   French,A.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\n",
      "            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\n",
      "            Kingdom\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..7342\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /clone=\"AN302944\"\n",
      "     gene            732..5954\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     CDS             join(732..813,4508..4753,5166..5447,5800..5954)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /function=\"antigen presenting molecule\"\n",
      "                     /codon_start=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "                     /product=\"MHC class II antigen\"\n",
      "                     /protein_id=\"CAK6681858.1\"\n",
      "                     /translation=\"MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGP\n",
      "                     SGQFTHEFDGDEEFYVDLERKETVWKLPLFHRLRFDPQFALTNIAVLKHNLNILIKRS\n",
      "                     NSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSET\n",
      "                     SFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTETV\n",
      "                     VCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL\"\n",
      "     exon            732..813\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          814..4507\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            4508..4753\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          4754..5165\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            5166..5447\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     intron          5448..5799\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "     exon            5800..5954\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=4\n",
      "                     /allele=\"HLA-DQA1*02:01:01:NEW\"\n",
      "ORIGIN      \n",
      "        1 ttgtaaccca gaaaatcctg aaggatgcca tataattgat gacctcatct atccatgagg\n",
      "       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tccaagcact\n",
      "      121 ttgggaggct gaggcaggca gatcacgagg tcaggagttc aagaccagcg tggccaacat\n",
      "      181 agtgaaactc tgtctctact aaaaatacag aaattagccg ggcatggtgg caggcacctg\n",
      "      241 cagtcccagc tacttgggag ggtgaggcag gagaatcgct tgaacccggg aggcaaaggt\n",
      "      301 tgcagtgagc cgagaccatg ccattgtacc tcagcctggg tgacagagtg agactacgtc\n",
      "      361 tcgaaaaata aagaaaaaga aaagaaaaag aaaaaaaaag aaaaattccc atcccttttg\n",
      "      421 cgaatggcag acatgcacac accagagaag attccaattt agtgtcttcc ctctcttcat\n",
      "      481 agaacaattc ctcaagtcca ctctgagtag aggctgcatc acaacaaggg gattgccctg\n",
      "      541 tctccttcca gggctcttaa tagaaactct tcaactagta actgagatgt caccatgggg\n",
      "      601 gatttttcta attggccaaa acctgacttg gcagggtttg gtttgggtgt cttcagattt\n",
      "      661 ccttgtcttg aggtcctcac aattactcta cagctcagaa cagcaactgc tgaggctgcc\n",
      "      721 ttgggaagaa gatgatccta aacaaagctc tgatgctggg ggccctcgcc ctgaccaccg\n",
      "      781 tgatgagccc ttgtggaggt gaagacattg tgggtgagtg cgtgagtgag gaatgttctc\n",
      "      841 tggagctgaa aaacagtaaa ttgaaggaaa agagagaaag cgatttgcag agaaattgta\n",
      "      901 gagatttcct aagacccctt tcagtattaa gagaattaaa aattatagct gttcctcctt\n",
      "      961 caggaaacca gagccccaac ctactctttt tgttatgtat gcttttgtgt tcactaagga\n",
      "     1021 tgctattctg tttatattat attcagtgac tacagcctgg aggtctctat gtcattccat\n",
      "     1081 catgattgcc tcaaaaatta gtgaggtttc catcagtgga taatttttta ttattaaaaa\n",
      "     1141 tgtatgaagt gtcattctca aatttccctg aacaactttt gaagattttc ggatgtctcc\n",
      "     1201 tgtagtagat cttggggtcg ttccatcaat tatatactct atagatatta aaaaagttgc\n",
      "     1261 ccgtttcttt ctctcagact tactcacatt tccacatggg aactggcaca ggtggggagt\n",
      "     1321 aggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\n",
      "     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\n",
      "     1441 ctaagctttg taaatagtga aaacatggat gtcaattgtt tttacatatt tctattacag\n",
      "     1501 gtatagcttc acatttcttt tctttagcaa aataagggat ccttttagtt taaaattgag\n",
      "     1561 aagtagaaaa aattggtaaa ttaaatcatt ttattctcaa attatcaacc caaattacct\n",
      "     1621 gttcttcacc tcatctaata aagtcctata aaaagaaaag tgggccagac atggtggctc\n",
      "     1681 atgcctgtaa tcccagcact ttgggaggcc gaagcaggag gatcatttga gcctgggagt\n",
      "     1741 ttgagaccag cctgggcaac acagcaagac ctcatctcta ccaaaaaata aaataaaaat\n",
      "     1801 tagccaggca tggtagtgca tgcctgtggt gccagctact cagaaggctg cagtgggagg\n",
      "     1861 agcacttgag tccaggaggt ggaagctgca gtgagccatg atggcaccac tacactccag\n",
      "     1921 ccagggcaac agagagagac cctatctcaa aaagaaagcg gaaagaaaga gagaaaggaa\n",
      "     1981 ggaaagaagg aaagaaggaa gcaaggaagg agaaagggaa gggaagaaag aagaaagaaa\n",
      "     2041 gaaagaaaac agaaggaagg aagcacagat taattatttg gtctcttagt ctcctctgcc\n",
      "     2101 tttgtcgtcc atctcttccc acctctcttc atgcattcct ttctccctct tccctttcag\n",
      "     2161 gatccatctc tgactccctg ctcctttata gagatggaca tgagtttgta aaacaaaagt\n",
      "     2221 tgaaaagtca gatagttaaa aggggaagta aactggaagg tactctaaac tttcacaacc\n",
      "     2281 ttattaaccg tggcagctcc cattctgatt ttgttcagca gtggaagttt caccctctcc\n",
      "     2341 tccagagcgc ttggcttctt tgttccaaat ttcctttctt cagcctcaca ccagagtgcc\n",
      "     2401 ctggtcaggc tcagctcatc cattaggcac aatgtgggca gtgcagggga acctccatac\n",
      "     2461 tgtaaagcca catgagaatg ttttaactcc ttttaaaatt ataaaaaaat gaaattgtag\n",
      "     2521 agcctaagaa aatgttttaa cttttaattc agcctatatt atattgtctt tataccaatt\n",
      "     2581 cagtcataaa atataatttt ccatattttt atggaggaag gcgtccacac aagcaagagt\n",
      "     2641 gcttggggct cacatgtcag aacgcatccc tgatcatggc tgatcctgac cttcgtgtgg\n",
      "     2701 ttctgctaac tatgtgcctg tcagtcttcc ccaaaatcta tgtggtcctc aaatataaca\n",
      "     2761 actgtcattc aatacacatg tttgagcacc cagtgagcta agttttaagg attcaaagat\n",
      "     2821 gaaaagtcat gctgtctccc ctgcagaggg tgctcagact agtgatggaa acagtatggg\n",
      "     2881 atgaaagaaa gcagaaggcc attgctgagc aggcagtgga ctcagcagag gctgaaacta\n",
      "     2941 tacaagtgac ttggttccag ctgggccagc aggataacca gacgaaaaga aggattgcat\n",
      "     3001 atattccata tatatttatg tttgaacaaa gagtcaaggt ttattgcaag gataaggagg\n",
      "     3061 ctttgttggt ggcctgttaa gaccatccag cgtggtcata ctggataggg aagaaggtga\n",
      "     3121 gctggaagag ggatagacaa acttggatgg ccagatgttg agatggagga gctggaggtc\n",
      "     3181 ataacgtggt caaaaacatg ttgatgagag gacttagcta caaagttgtt aacttaagca\n",
      "     3241 gaaacctcaa ggattgattt tatgatttct ccaggaagtc ctaaaagata atttcatttc\n",
      "     3301 agggaggaaa acaacagacc actgcaaaga ccaggaacat gaaaggataa tgtagtttgg\n",
      "     3361 tttgcttggc agatacttgt gaaagatgtt ggactgtaag gctgtcaata tcctcctcgc\n",
      "     3421 agaacttact acagtacatt gtatctgctc ccttacctac ctgactctcc cactattcag\n",
      "     3481 tttgttcctt aatggtagac catgcctgat tggtgtttta cacttcccct gctatgtctg\n",
      "     3541 atacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta aaaggcttac\n",
      "     3601 acatgtcttg accagaatgt tcagtttggc tcatttggct ggagtcatac tgcatggctg\n",
      "     3661 ccattctgct ctggcatcct cagagaagca cactgcccat tagaggaaaa agggtgaata\n",
      "     3721 taaatgttga gtcagaacac tgcagacatt tagtaacctc cttcagagga aaaaaagggt\n",
      "     3781 ggggggaatg acagaaatcc aaaaactagt agagcttcca ctttttcatt tcagaagaaa\n",
      "     3841 tcagttactc tcctctaagg accattacta ttaacaaaac agagacctta gaaggaagca\n",
      "     3901 ttatttattt atcatatatt ttgtaatgtt attaccgttc ttgttatact ctttcttata\n",
      "     3961 ccctaccatt gttagcagaa attattttaa attaataaga tcctgcatgc ttttcctttt\n",
      "     4021 tctaaaaaaa gaaagatctc tgtgtagaat gtcctgttct gagccagtcc tgagaggaaa\n",
      "     4081 ggaagtataa tcaatttgtt attaactgat gaaagaatta agtgaaagat aaaccttagg\n",
      "     4141 aagcagaggg aagttaatct atgactaaga aagttaagta ctctgataac tcattcattc\n",
      "     4201 cttcttctgt tcatttacat tatttaatca caagtccatg atgtgccagg cactcaggaa\n",
      "     4261 atagtgaaaa tcggacacgc gatattctgc ccttgtgtag cacacactgt agtgggaaag\n",
      "     4321 aaagtgcact tttaactgga caactatcaa cacgaagagg ggaggaagca ggggctggaa\n",
      "     4381 atgtccacag actttgccaa agacaaagcc cataatattt gaaagtcagt ttcttccatc\n",
      "     4441 attttgtgta ttaaggtttt ttattctcct gttctctgcc ttcctgcttg tcatcttcac\n",
      "     4501 tcatcagctg accacgttgc ctcttacggt gtaaacttgt accagtctta cggtccctct\n",
      "     4561 ggccagttca cccatgaatt tgatggagac gaggagttct atgtggacct ggagaggaag\n",
      "     4621 gagactgtct ggaagttgcc tctgttccac agacttagat ttgacccgca atttgcactg\n",
      "     4681 acaaacatcg ctgtgctaaa acataacttg aacatcctga ttaaacgctc caactctacc\n",
      "     4741 gctgctacca atggtatgtg tccaccattc tgcctttctt tactgatcta tccctttata\n",
      "     4801 ccaagtttca ttattttctt tccaagaggt ccccagatct tctcatggca attgctgaaa\n",
      "     4861 ttttatcatt tctcatctct aaaatcacat atccccatgt aatacaaggg tctttccatt\n",
      "     4921 atgcattcat taaatcattc taggagaggt ctcatcaacc tcctacttta ttaaacatgc\n",
      "     4981 ccacagagag aagggcacag gagtaaagca gaggcaatgt gtcattgctc ccaagtagaa\n",
      "     5041 ggtaaataag gcctctttga ccagcaggag aggaaatgct ggtaggaaga ctcttccagg\n",
      "     5101 atgtaatgca gaagctcagg gcagagctat tcacacttca caccagtgct gtttcctcac\n",
      "     5161 catagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\n",
      "     5221 caccctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca cctggctgag\n",
      "     5281 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\n",
      "     5341 ttccttcttc aagatcagtt acctcacctt cctcccttct gctgatgaga tttatgactg\n",
      "     5401 caaggtggag cactggggcc tggatgagcc tcttctgaaa cactggggta aggatgagtt\n",
      "     5461 tcaccatttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactctat\n",
      "     5521 gtcccaaaac ttgttttcca cacttcatga gtttctttta tctttttttt tttttgaaag\n",
      "     5581 aattaagcaa caaaagcaca gatttattaa aaaagaaagt acactccaca gggtgggagc\n",
      "     5641 aggcctgcca cttcatgggt ttctaataac agacttcact ctcctccctg agccaggggc\n",
      "     5701 cttgagtctt tgcagagcca accctccacc ccatcccatc cctcacacat gcacatgagc\n",
      "     5761 acactctgca ttctgacctc aacaacttca cttccacaga gcctgagatt ccagcaccta\n",
      "     5821 tgtcagagct cacagagact gtggtctgtg ccctggggtt gtctgtgggc ctcgtgggca\n",
      "     5881 ttgtggtggg gaccgtcttg atcatccgag gcctgcgttc agttggtgct tccagacacc\n",
      "     5941 aagggccctt gtgaatccca tcctgaaaaa gaaggtaagt ttgagatttg ttagagctga\n",
      "     6001 agctgcagga aggaaagtgg gaggaggctg tggacatgaa tgtggttgaa agttgtaggg\n",
      "     6061 gaattgggaa gtggcatgat gatgacacag gagccccctt ggacccatcg atctcatgtc\n",
      "     6121 tgtcctgttg caggtgcatc accatctaca gcagcggaag agtggacttg ctacatgacc\n",
      "     6181 tagcactatt ctctggcccg atttatcata ccctttttct cctgcaaatg tttctcctct\n",
      "     6241 taccttttct ctgctttttt ttccttaagc ttctgtatcc cctcagagct cacaaatgcc\n",
      "     6301 tttgaattct ttccctgacc tcctgatttt tttttctttt ctcaggtgtt acctactaag\n",
      "     6361 agatgcctgg ggtaagccgc ccagctacct aattcctcag taacatcgat ctaaaatctc\n",
      "     6421 catggaagca ataaattccc tttaagagat ctatgtcaaa tttttccatc tttcatccgg\n",
      "     6481 ggctgactga acctatggct aagaattggg acactctcat gtttcaagcc aatttcatct\n",
      "     6541 catttcccag atcatatttc atatccagta acacagaagc aaccaagtac aatatagcct\n",
      "     6601 gataatatgt tgatttctta gctgacatta atatttcttt cttctttgtg ttctcaccct\n",
      "     6661 tggcactgcc gcccatccct caattcaggc aacaatgaag ttaatggata ctctctgccc\n",
      "     6721 tttgctcaga attgttatag caaaaatttt aaaaccaaaa aataagtttg tactaatttc\n",
      "     6781 aatatggctt ttaaaagtat gatggagaaa taaattagga taaaggaact ttgaatcaca\n",
      "     6841 aaaatatcaa aagtaaaaat ttattctcaa aactttgaat ttgtaaagaa tgatgacagt\n",
      "     6901 agaagccttc ctctcccctc ctcgccttta gggaataaaa attctttagg taggaaaaga\n",
      "     6961 aatggaagtc agaaaaacat tagaaaaaga cagtaatgtg ggtatctgaa aaggaacaaa\n",
      "     7021 tacttattcc tcacataggg ttagtgacaa tgggaaaagg gataggagta gaagccacag\n",
      "     7081 acatatctag gagccctgaa tagaggcgca gtctgcctca cctcctgaat gaagctttgc\n",
      "     7141 tagataacca cgtagctttc cctgtgccac ccttgcatga agcagacagt atagtggata\n",
      "     7201 tggcaggatg tttctaggaa acatgccgat acaaaacaat gccagtatct tcagaaatcc\n",
      "     7261 ccagcccttt cccctcaccc ctcctggcta aggaaagcac tagcttatga gagaaaccct\n",
      "     7321 aggaggaaca acacagttga ga\n",
      "//\n",
      "\n",
      "LOCUS       OY986025                7529 bp    DNA     linear   PRI 27-DEC-2023\n",
      "DEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\n",
      "            HLA-DQA1*05:09:01:NEW.\n",
      "ACCESSION   OY986025\n",
      "VERSION     OY986025.1\n",
      "DBLINK      BioProject: PRJEB71396\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1\n",
      "  AUTHORS   French,A.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\n",
      "            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\n",
      "            Kingdom\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..7529\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /clone=\"AN302920\"\n",
      "     gene            727..6143\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     CDS             join(727..808,4628..4873,5286..5567,5989..6143)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /function=\"antigen presenting molecule\"\n",
      "                     /codon_start=1\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "                     /product=\"MHC class II antigen\"\n",
      "                     /protein_id=\"CAK6681856.1\"\n",
      "                     /translation=\"MILNKALMLGTLALTTVMSPCGGKDIVADHVASYGVNLYQSYGP\n",
      "                     SGQYTHEFDGDEQFYVDLGRKETVWCLPVLRQFRFDPQFALTNIAVLKHNLNSLIKRS\n",
      "                     NSTAATNEVPEVTVFSKSPVTLGQPNILICLVDNIFPPVVNITWLSNGHSVTEGVSET\n",
      "                     SFLSKSDHSFFKISYLTLLPSAEESYDCKVEHWGLDKPLLKHWEPEIPAPMSELTETV\n",
      "                     VCALGLSVGLVGIVVGTVFIIRGLRSVGASRHQGPL\"\n",
      "     exon            727..808\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     intron          809..4627\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=1\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     exon            4628..4873\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     intron          4874..5285\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     exon            5286..5567\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     intron          5568..5988\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "     exon            5989..6143\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=4\n",
      "                     /allele=\"HLA-DQA1*05:09:01:NEW\"\n",
      "ORIGIN      \n",
      "        1 ttgtaactca gaaaatcctg aaggatgccg tagaattgat gacatcatct atccacgagg\n",
      "       61 ctgctcagaa atgcccaccc ctggccaggg cggtggctca tgcctgtaat ctgagcactt\n",
      "      121 tgggaggctg agatgggcag atcacgaatt caggagttcg agaccagcct ggccaacata\n",
      "      181 gtgaaactct gtctctacta aaaatacaaa aattagccgg gcgtggtggc aggcgcctgt\n",
      "      241 aatcccagct acttgggagg gtgagacagg agaatcactt gaacccggga gtcaaagttt\n",
      "      301 gcagtgagcc gagaccatgc cattgcacct cagcctgggt gacagagtga gactacatct\n",
      "      361 caaaaaaaaa aaaaaaaaga aagaaaaaaa agaaaagaaa tgcccacccc tcttgccact\n",
      "      421 ggcagacatg cacacaccag agaagattcc gatttcgtgt cctccctcta ttcacagaac\n",
      "      481 atttcctcaa gtccactctg agtggaggct gcatcacaac aaggggattg ccctgtctcc\n",
      "      541 ttccagggct cttaatacaa actcttcaac tagtaactga ggtgtcatca taggggattt\n",
      "      601 ttctaattag ccaaaacctg acttggcagg gtttggtttg ggtgtcttca gattgccttg\n",
      "      661 tctcgaggtc ctcacaattg ctctacaact cagaacagca actgctaagg ctgccttggg\n",
      "      721 aagaggatga tcctaaacaa agctctgatg ctggggaccc ttgccctgac caccgtgatg\n",
      "      781 agcccctgtg gaggtaaaga cattgtgggt gagtgtatga gtgagggatg ttctctggag\n",
      "      841 ctggaagaga ggaaattgaa gcaaaagaga gaaagcgatt tgcagagaaa ttgtagagat\n",
      "      901 ttcctaaggg tcccttcagt attaagagat ttaaaaatta tggctgttcc tccttcagga\n",
      "      961 aaccagagtc ccaacctact ctttttgtta tctatgctgt tggcgttcac taaggatgct\n",
      "     1021 attctgttta tattgtattc agtgactata gcctggaggt ctctatgtca ttccatcatg\n",
      "     1081 attgcctcaa aaattagtga ggtttccatc aatggataat tttttattat taaaaatttg\n",
      "     1141 tgaagtatca ttctcaaatt tccctgaaca acttttgaag cttttcggat gtctcctgta\n",
      "     1201 gtagcgcttg gggtaaatga ttccatcaat tatatactct atagatatta agaaagatgc\n",
      "     1261 ccttttcttt ctctcagact tactaacatt tccacgtggg aactggcaca ggtggggagt\n",
      "     1321 gggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\n",
      "     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\n",
      "     1441 ctaagctttg taaatagtga aaacatgaat gtcaatttta tttttacata tttctattat\n",
      "     1501 aggtatagct tcacatttct tttcttttgc aaaataagga atccttttat tttaaaattg\n",
      "     1561 agaatttata gtagaaaaac ttggtaaatt aaatcatttt attctcaaat tatcaaccca\n",
      "     1621 aattacctgt tcttcaactc atctaatgaa gtcctataaa aagaaaagtg ggccagacat\n",
      "     1681 ggtggctcat gcctgtaatc ccagcacttt gggaggccga ggcaggagga tcctttgagc\n",
      "     1741 ccaggagttg agaccagcct gggcaacata gcaagacctc atctctacaa aaaataaaaa\n",
      "     1801 ttagccaggt gtggtggtgc atgcctgtgg tgccagctac tcagaaggct gcagtgggag\n",
      "     1861 gactgcttga gtccaggagg cggaagctgc agtgagccat gatggcacca ctacactcca\n",
      "     1921 gccagggcaa cagagagaaa ctctgtctca gaaagaaaaa ggaaagaaag agagaaagga\n",
      "     1981 aggaaagaaa gaaagaagga aggaaggaag gaagaagggg aagggaaggg aaggaaggga\n",
      "     2041 aggaaaggga agggaggaaa gaaggaagga aaaaagaaaa agaaagaaaa aagaaagaaa\n",
      "     2101 aagaaagaaa gaaagaaaag agggaaggaa agaaagaaaa aggaaggaag gaagcacaga\n",
      "     2161 ttaattatta atcatttggt ctctcttagt cttctctgtc tttgtcatcc atctatttcc\n",
      "     2221 acctctcttc atgcattcct ttctccctct tccctttcag gatccatctc tgactccctg\n",
      "     2281 cttctttata tggacagtgg ggtttgtaaa acaaaagttg aaaaatcaga tagttaaaag\n",
      "     2341 gtgaagtgaa ctggaaggtc taaacttcca caaccttatt aaccatggct gctcccattc\n",
      "     2401 tgattttgtt cgacggtgga agtttcacct ccttctccag agcacttggc ttctttgttc\n",
      "     2461 caaatttcct ttcttcaacc tcacaccaga gtgccctggt caggctcggc tcatccatta\n",
      "     2521 ggcacaacat gggcaatgca gagaaccctc catactgtaa agccacatga gaatgtttta\n",
      "     2581 actcctttta agattagaaa aaaatgaaat ttcagagcct aagaaaatgt tttaattcag\n",
      "     2641 cctagattgt attgtcttta taccaattca gtcataaaat acaattttcc atattttcat\n",
      "     2701 ggaggaaggg gcccacaaaa gcaagagtgc tcagggctca catgtcagaa tgcagccctg\n",
      "     2761 gtcatggctg atcctggcct tcttatgatt ctgctagtgt gcctgtccgt cttccccaaa\n",
      "     2821 atctatgtcg tcctcaaata tagcaactgt aattcaaaac acgtttgagc acacagtaag\n",
      "     2881 ctaagtttta aggattcaaa gatgaaaagt catgctgtct tccctgcaga gggtgctcag\n",
      "     2941 actagtgatg gaaacggtat gggatgcagg aaagcagaag gccattgctg accagggcag\n",
      "     3001 tggactcagc agaggctgaa actatacaaa tgacttggtt ccagctgggc cagcagggtg\n",
      "     3061 acatcctcca gcaaaactcg gcacccaaga caagtcccag atgaaaagaa ggattgcttt\n",
      "     3121 gtgtttgata aggagtcaag gtttattgca aggataaggc tttgttggtg gcctgttaag\n",
      "     3181 acaatccagg gcagtcatac tggataagga agaaggtgag ctggaagagg aacagacaaa\n",
      "     3241 cagccagata ttgaggtgga ggagctggag gtcataacgt ggtcaaaaac atgtcgatga\n",
      "     3301 gaggacttag ctacaaagtt gttaacttaa gcagaaacct caaggatgaa ttttaggatt\n",
      "     3361 tctccaggaa gtcctaaaag ataatttcat ttcagggaga aaaacaacaa accactggaa\n",
      "     3421 agaccaggga acatgaaagg ataatgtagt ttggtttgct tggcagatac ttatgaagga\n",
      "     3481 tgttggactg taaggctgtt gatagcctcc tcacagaaca tgctacagta cattgtatct\n",
      "     3541 gctcccttac ctacctgact cttccactct tcagtttgtt ccttaatggt agaccatgcc\n",
      "     3601 tgattggtgt tttacacatc ccctgctatg tctgacactt gtggatgctc agaaagtggg\n",
      "     3661 gaaggaagga aagatacaac ggtaaaaggc ttacacgtct tgacaggaat gtccagttcg\n",
      "     3721 gctcatttgg ctggagccac attgcacggc tgccattctg ctctggcatc ctcagacaag\n",
      "     3781 cacactgccc attagaggaa aaagggttta atttacctga gtcctcaagt gaatataagt\n",
      "     3841 gttaagtcag aacactgtag acatttagta acctccttca gaggaaaaaa aaaggtgggg\n",
      "     3901 ggaaatgaca gaaatccaaa aactagtaga gcttccactc ttcatttcag aagaaatcag\n",
      "     3961 ttgctctcct ctaaggacca ttactattaa caaaacagag accttgaagg aggcattgtt\n",
      "     4021 tatttattat atattttgta atgttattac cattcttgtt atactattcc ttatacccta\n",
      "     4081 caattgttag cagacattat tttaaattaa taagatcctg catgcttttc tttttttaaa\n",
      "     4141 aaaaagaaag acctctgtgt agagtgtcct gttctgagcc agtcctgaga ggaaagaaaa\n",
      "     4201 tacaatcagt ttgttattaa ctgaggaaag aattaagtga aagatgaatc ttaggaagca\n",
      "     4261 gaaggaagta aacctaatct ctgactaaga aagctaaata ccataataac tcattcattc\n",
      "     4321 cttcttttgt ttaattacat tatttaatca taagtccgtg atgtgccagg cactcaggaa\n",
      "     4381 atagtaaaaa ctggacatgt gatattctgc ccttgtgtag cgcacattat agtgggaaag\n",
      "     4441 aaagcgcaat tttaaccgga caactaccaa caataagagc ggaggaagca ggggttggaa\n",
      "     4501 atgtccacag gctgtgccaa agatgaagcc cgtaatattt gaaagtcagt ttctttcatc\n",
      "     4561 attttgtgta ttaaggttct ttcttcccct gttctccacc ttcctgcttg tcatcttcac\n",
      "     4621 tcatcagctg accacgtcgc ctcttatggt gtaaacttgt accagtctta cggtccctct\n",
      "     4681 ggccagtaca cccatgaatt tgatggagat gagcagttct acgtggacct ggggaggaag\n",
      "     4741 gagactgtct ggtgtttgcc tgttctcaga caatttagat ttgacccgca atttgcactg\n",
      "     4801 acaaacatcg ctgtcctaaa acataacttg aacagtctga ttaaacgctc caactctacc\n",
      "     4861 gctgctacca atggtatgtg tcaacaattc tgcccctctt tactgattta tcccttcata\n",
      "     4921 ccaagtttca ttattttatt tccaagaggt ccccagatct tctcatggca attgctgaaa\n",
      "     4981 ttttatcatc tcccatctct aaaatcacat attcccatgt aatacaaggg tctttccatt\n",
      "     5041 atccattcat taaatccttc tcggagaggt ctcatcaacc tcctacttta ttaaacatgc\n",
      "     5101 ccacagagag aagggcacag gaataaagcg gaggcaatgt gtcgttgctc ccaagcagaa\n",
      "     5161 ggtaaataag acctctttga ctatcaggtg gtgaaatgct ggtaggaggg ctcttccagg\n",
      "     5221 atgtaatgca gaagctcatg gcagagctat tcacacttca catcagtgct gtttcctcac\n",
      "     5281 cacagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\n",
      "     5341 catcctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca catggctgag\n",
      "     5401 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\n",
      "     5461 ttccttcttc aagatcagtt acctcaccct cctcccttct gctgaggaga gttatgactg\n",
      "     5521 caaggtggag cactggggac tggacaagcc tcttctgaaa cactggggta aggatgagtt\n",
      "     5581 ccaccacttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactccta\n",
      "     5641 tatcccaaaa cttgttttcc acacttcatg ggtttctttc tttctttctt tctttctttc\n",
      "     5701 tttctttctt tctttctttc tttctttctt tctttctttc ttttttgaaa gaataaagca\n",
      "     5761 acaaaagcaa agatttattg aaaatgaaag tacacttcac atggtgggag cgggcctgag\n",
      "     5821 cataggggct caagagccac ttcatgggtt tctaatgata gacttcactc tcctccctaa\n",
      "     5881 gctggggacc ttgagtcttt gcagagccaa ccctccaccc catcccatcc cacacacatg\n",
      "     5941 cacatgagca cactctgctt tctgacctca acgacttcat atccacagag cctgagattc\n",
      "     6001 cagcccctat gtcagagctc acagagactg tggtctgcgc cctggggttg tctgtgggcc\n",
      "     6061 tcgtgggcat tgtggtgggc actgtcttca tcatccgagg cctgcgttca gttggtgctt\n",
      "     6121 ccagacacca agggcccttg tgaatcccat cctggaatgg aaggtaagat tgagatttgt\n",
      "     6181 tagagctgaa tccgcagtat gagaggaagg aaagtggagg aggctgtgga catgaatggt\n",
      "     6241 tgaaagttgt aggggaattg ggaagtggca tgatgatgac ataggagcgg cctaggaccc\n",
      "     6301 atccatctca tgtctgtcct gttgcaggtg catcgccatc tacaggagca gaagagtgga\n",
      "     6361 cttgctacat gacctagcat tattttctgg ccccatttat catatccctt ttctcctcca\n",
      "     6421 aatgtttctc ctctcacctc ttctgtggga cttaaattgc tatatctgct cagagctcac\n",
      "     6481 aaatgccttt gaattatttc cctgacttcc tgattttttt cttttctcaa gtgttaccta\n",
      "     6541 ctaagggatg cctggagtaa gccacccagc tacctaattc ctcagtaacc tccatctata\n",
      "     6601 atctccatgg aagcaacaaa ttccctttat gagatatatg tcaaattttt ccatctttca\n",
      "     6661 tccagggctg actgaaaccg tggctaagaa ttgggagact ctcttgtttc aagccaattt\n",
      "     6721 aacatcattt accagatcat ttgtcatgtc cagtaacaca gaagcaacca actacagtat\n",
      "     6781 agcctgataa catgttgatt tcttagctga cgttaatatt tctttcttcc ttgtgttccc\n",
      "     6841 acccttggca ttgccaccca cccctcaatt aaggcaacaa tgaagttaat ggataccctc\n",
      "     6901 tgcctttggc tcagaaatgt tatagcaaaa attttaaaat aaaaaagtaa gtctgtacta\n",
      "     6961 atttcaatat gacttttaaa agtatgacag agaaatgggt tgggataaac gaaatttgaa\n",
      "     7021 tctcaaaaat atcaatagtg aaaatttatt ctcaaaacta aatttgtgaa gaatgatgac\n",
      "     7081 agtagaagcc ttcctctccc ctcctcaccc tgaaggaata aaatttcctt aggcaggaaa\n",
      "     7141 agaaatggaa gtcagaaaaa cattagaata agacaataat gtgggtatct gaaaaggaac\n",
      "     7201 aaatgctcat tcctcacata gggttagtga caatggggaa agggatggga gtagaagctg\n",
      "     7261 cagacatatc taggagccct agaatagagg cacagtctgc cccacctcct caatgaagct\n",
      "     7321 ttgctagata accatgtagc tttccctgtg ccagcctggc atgaaggaga cagtatagtg\n",
      "     7381 gatatggctg caagatgttt ctaggaaaca tgccgataaa aaaacaatac caatatcttc\n",
      "     7441 agaaatcccc agccctttcc cctaaccctc ctggctaagg aaagcactag cttatgagaa\n",
      "     7501 aaaccctagg aagaacaaca cagttaaga\n",
      "//\n",
      "\n",
      "LOCUS       OQ102522                 886 bp    DNA     linear   PRI 26-DEC-2023\n",
      "DEFINITION  Homo sapiens clone DQA1_113588_00517 MHC class II antigen\n",
      "            (HLA-DQA1) gene, exons 2 through 4 and partial cds.\n",
      "ACCESSION   OQ102522\n",
      "VERSION     OQ102522.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 886)\n",
      "  AUTHORS   Cereb,N. and Yang,S.Y.\n",
      "  TITLE     HLA new alleles found by Histogenetics\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 886)\n",
      "  AUTHORS   Cereb,N. and Yang,S.Y.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (21-DEC-2022) Histogenetics, 300 Executive Blvd,\n",
      "            Ossining, NY 10562, USA\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..886\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"genomic DNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /clone=\"DQA1_113588_00517\"\n",
      "     gene            <1..>886\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     mRNA            join(<1..249,350..631,732..>886)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /product=\"MHC class II antigen\"\n",
      "     CDS             join(<1..249,350..631,732..886)\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /codon_start=3\n",
      "                     /product=\"MHC class II antigen\"\n",
      "                     /protein_id=\"WBG53738.1\"\n",
      "                     /translation=\"DHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLP\n",
      "                     LFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNT\n",
      "                     LICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYD\n",
      "                     CKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVFIIRGLRSV\n",
      "                     GASRHQGPL\"\n",
      "     exon            1..249\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=2\n",
      "     gap             250..349\n",
      "                     /estimated_length=unknown\n",
      "     exon            350..631\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=3\n",
      "     gap             632..731\n",
      "                     /estimated_length=unknown\n",
      "     exon            732..886\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "                     /number=4\n",
      "ORIGIN      \n",
      "        1 ctgaccatgt tgcctcttac ggtgtaaact tgtaccagtc ttatggtccc tctgggcagt\n",
      "       61 acagccatga atttgatgga gacgaggagt tctatgtgga cctggagagg aaggagactg\n",
      "      121 tctggcagtt gcctctgttc cgcagattta gaagatttga cccgcaattt gcactgacaa\n",
      "      181 acatcgctgt gctaaaacat aacttgaaca tcgtgattaa acgctccaac tctaccgctg\n",
      "      241 ctaccaatgn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\n",
      "      301 nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnna ggttcctgag\n",
      "      361 gtcacagtgt tttccaagtc tcccgtgaca ctgggtcagc ccaacaccct catctgtctt\n",
      "      421 gtggacaaca tctttcctcc tgtggtcaac atcacctggc tgagcaatgg gcactcagtc\n",
      "      481 acagaaggtg tttctgagac cagcttcctc tccaagagtg atcattcctt cttcaagatc\n",
      "      541 agttacctca ccttcctccc ttctgctgat gagatttatg actgcaaggt ggagcactgg\n",
      "      601 ggcctggatg agcctcttct gaaacactgg gnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\n",
      "      661 nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\n",
      "      721 nnnnnnnnnn nagcctgaga ttccaacacc tatgtcagag ctcacagaga ctgtggtctg\n",
      "      781 cgccctgggg ttgtctgtgg gcctcgtggg cattgtggtg gggaccgtct tcatcatccg\n",
      "      841 aggcctgcgt tcagttggtg cttccagaca ccaagggccc ttgtga\n",
      "//\n",
      "\n",
      "\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'LOCUS       OY986031                7342 bp    DNA     linear   PRI 27-DEC-2023\\nDEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\\n            HLA-DQA1*02:01:01:NEW.\\nACCESSION   OY986031\\nVERSION     OY986031.1\\nDBLINK      BioProject: PRJEB71396\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1\\n  AUTHORS   French,A.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\\n            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\\n            Kingdom\\nFEATURES             Location/Qualifiers\\n     source          1..7342\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /clone=\"AN302966\"\\n     gene            732..5954\\n                     /gene=\"HLA-DQA1\"\\n     CDS             join(732..813,4508..4753,5166..5447,5800..5954)\\n                     /gene=\"HLA-DQA1\"\\n                     /function=\"antigen presenting molecule\"\\n                     /codon_start=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"CAK6681862.1\"\\n                     /translation=\"MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGP\\n                     SGQFTHEFDGDEEFYVDLERKETVWKLPLFHRLRFDPQFALTNIAVLKHNLNILIKRS\\n                     NSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSET\\n                     SFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTETV\\n                     VCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL\"\\n     exon            732..813\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          814..4507\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            4508..4753\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          4754..5165\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            5166..5447\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          5448..5799\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            5800..5954\\n                     /gene=\"HLA-DQA1\"\\n                     /number=4\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\nORIGIN      \\n        1 ttgtaaccca gaaaatcctg aaggatgcca tataattgat gacctcatct atccatgagg\\n       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tccaagcact\\n      121 ttgggaggct gaggcaggca gatcacgagg tcaggagttc aagaccagcg tggccaacat\\n      181 agtgaaactc tgtctctact aaaaatacag aaattagccg ggcatggtgg caggcacctg\\n      241 cagtcccagc tacttgggag ggtgaggcag gagaatcgct tgaacccggg aggcaaaggt\\n      301 tgcagtgagc cgagaccatg ccattgtacc tcagcctggg tgacagagtg agactacgtc\\n      361 tcgaaaaata aagaaaaaga aaagaaaaag aaaaaaaaag aaaaattccc atcccttttg\\n      421 cgaatggcag acatgcacac accagagaag attccaattt agtgtcttcc ctctcttcat\\n      481 agaacaattc ctcaagtcca ctctgagtag aggctgcatc acaacaaggg gattgccctg\\n      541 tctccttcca gggctcttaa tagaaactct tcaactagta actgagatgt caccatgggg\\n      601 gatttttcta attggccaaa acctgacttg gcagggtttg gtttgggtgt cttcagattt\\n      661 ccttgtcttg aggtcctcac aattactcta cagctcagaa cagcaactgc tgaggctgcc\\n      721 ttgggaagaa gatgatccta aacaaagctc tgatgctggg ggccctcgcc ctgaccaccg\\n      781 tgatgagccc ttgtggaggt gaagacattg tgggtgagtg cgtgagtgag gaatgttctc\\n      841 tggagctgaa aaacagtaaa ttgaaggaaa agagagaaag cgatttgcag agaaattgta\\n      901 gagatttcct aagacccctt tcagtattaa gagaattaaa aattatagct gttcctcctt\\n      961 caggaaacca gagccccaac ctactctttt tgttatgtat gcttttgtgt tcactaagga\\n     1021 tgctattctg tttatattat attcagtgac tacagcctgg aggtctctat gtcattccat\\n     1081 catgattgcc tcaaaaatta gtgaggtttc catcagtgga taatttttta ttattaaaaa\\n     1141 tgtatgaagt gtcattctca aatttccctg aacaactttt gaagattttc ggatgtctcc\\n     1201 tgtagtagat cttggggtcg ttccatcaat tatatactct atagatatta aaaaagttgc\\n     1261 ccgtttcttt ctctcagact tactcacatt tccacatggg aactggcaca ggtggggagt\\n     1321 aggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\\n     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\\n     1441 ctaagctttg taaatagtga aaacatggat gtcaattgtt tttacatatt tctattacag\\n     1501 gtatagcttc acatttcttt tctttagcaa aataagggat ccttttagtt taaaattgag\\n     1561 aagtagaaaa aattggtaaa ttaaatcatt ttattctcaa attatcaacc caaattacct\\n     1621 gttcttcacc tcatctaata aagtcctata aaaagaaaag tgggccagac atggtggctc\\n     1681 atgcctgtaa tcccagcact ttgggaggcc gaagcaggag gatcatttga gcctgggagt\\n     1741 ttgagaccag cctgggcaac acagcaagac ctcatctcta ccaaaaaata aaataaaaat\\n     1801 tagccaggca tggtagtgca tgcctgtggt gccagctact cagaaggctg cagtgggagg\\n     1861 agcacttgag tccaggaggt ggaagctgca gtgagccatg atggcaccac tacactccag\\n     1921 ccagggcaac agagagagac cctgtctcaa aaagaaagcg gaaagaaaga gagaaaggaa\\n     1981 ggaaagaagg aaagaaggaa gcaaggaagg agaaagggaa gggaagaaag aagaaagaaa\\n     2041 gaaagaaaac agaaggaagg aagcacagat taattatttg gtctcttagt ctcctctgcc\\n     2101 tttgtcgtcc atctcttccc acctctcttc atgcattcct ttctccctct tccctttcag\\n     2161 gatccatctc tgactccctg ctcctttata gagatggaca tgagtttgta aaacaaaagt\\n     2221 tgaaaagtca gatagttaaa aggggaagta aactggaagg tactctaaac tttcacaacc\\n     2281 ttattaaccg tggcagctcc cattctgatt ttgttcagca gtggaagttt caccctctcc\\n     2341 tccagagcgc ttggcttctt tgttccaaat ttcctttctt cagcctcaca ccagagtgcc\\n     2401 ctggtcaggc tcagctcatc cattaggcac aatgtgggca gtgcagggga acctccatac\\n     2461 tgtaaagcca catgagaatg ttttaactcc ttttaaaatt ataaaaaaat gaaattgtag\\n     2521 agcctaagaa aatgttttaa cttttaattc agcctatatt atattgtctt tataccaatt\\n     2581 cagtcataaa atataatttt ccatattttt atggaggaag gcgtccacac aagcaagagt\\n     2641 gcttggggct cacatgtcag aacgcatccc tgatcatggc tgatcctgac cttcgtgtgg\\n     2701 ttctgctaac tatgtgcctg tcagtcttcc ccaaaatcta tgtggtcctc aaatataaca\\n     2761 actgtcattc aatacacatg tttgagcacc cagtgagcta agttttaagg attcaaagat\\n     2821 gaaaagtcat gctgtctccc ctgcagaggg tgctcagact agtgatggaa acagtatggg\\n     2881 atgaaagaaa gcagaaggcc attgctgagc aggcagtgga ctcagcagag gctgaaacta\\n     2941 tacaagtgac ttggttccag ctgggccagc aggataacca gacgaaaaga aggattgcat\\n     3001 atattccata tatatttatg tttgaacaaa gagtcaaggt ttattgcaag gataaggagg\\n     3061 ctttgttggt ggcctgttaa gaccatccag cgtggtcata ctggataggg aagaaggtga\\n     3121 gctggaagag ggatagacaa acttggatgg ccagatgttg agatggagga gctggaggtc\\n     3181 ataacgtggt caaaaacatg ttgatgagag gacttagcta caaagttgtt aacttaagca\\n     3241 gaaacctcaa ggattgattt tatgatttct ccaggaagtc ctaaaagata atttcatttc\\n     3301 agggaggaaa acaacagacc actgcaaaga ccaggaacat gaaaggataa tgtagtttgg\\n     3361 tttgcttggc agatacttgt gaaagatgtt ggactgtaag gctgtcaata tcctcctcgc\\n     3421 agaacttact acagtacatt gtatctgctc cctcacctac ctgactctcc cactattcag\\n     3481 tttgttcctt aatggtagac catgcctgat tggtgtttta cacttcccct gctatgtctg\\n     3541 atacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta aaaggcttac\\n     3601 acatgtcttg accagaatgt tcagtttggc tcatttggct ggagtcatac tgcatggctg\\n     3661 ccattctgct ctggcatcct cagagaagca cactgcccat tagaggaaaa agggtgaata\\n     3721 taaatgttga gtcagaacac tgcagacatt tagtaacctc cttcagagga aaaaaagggt\\n     3781 ggggggaatg acagaaatcc aaaaactagt agagcttcca ctttttcatt tcagaagaaa\\n     3841 tcagttactc tcctctaagg accattacta ttaacaaaac agagacctta gaaggaagca\\n     3901 ttatttattt atcatatatt ttgtaatgtt attaccgttc ttgttatact ctttcttata\\n     3961 ccctaccatt gttagcagaa attattttaa attaataaga tcctgcatgc ttttcctttt\\n     4021 tctaaaaaaa gaaagatctc tgtgtagaat gtcctgttct gagccagtcc tgagaggaaa\\n     4081 ggaagtataa tcaatttgtt attaactgat gaaagaatta agtgaaagat aaaccttagg\\n     4141 aagcagaggg aagttaatct atgactaaga aagttaagta ctctgataac tcattcattc\\n     4201 cttcttctgt tcatttacat tatttaatca caagtccatg atgtgccagg cactcaggaa\\n     4261 atagtgaaaa tcggacacgc gatattctgc ccttgtgtag cacacactgt agtgggaaag\\n     4321 aaagtgcact tttaactgga caactatcaa cacgaagagg ggaggaagca ggggctggaa\\n     4381 atgtccacag actttgccaa agacaaagcc cataatattt gaaagtcagt ttcttccatc\\n     4441 attttgtgta ttaaggtttt ttattctcct gttctctgcc ttcctgcttg tcatcttcac\\n     4501 tcatcagctg accacgttgc ctcttacggt gtaaacttgt accagtctta cggtccctct\\n     4561 ggccagttca cccatgaatt tgatggagac gaggagttct atgtggacct ggagaggaag\\n     4621 gagactgtct ggaagttgcc tctgttccac agacttagat ttgacccgca atttgcactg\\n     4681 acaaacatcg ctgtgctaaa acataacttg aacatcctga ttaaacgctc caactctacc\\n     4741 gctgctacca atggtatgtg tccaccattc tgcctttctt tactgatcta tccctttata\\n     4801 ccaagtttca ttattttctt tccaagaggt ccccagatct tctcatggca attgctgaaa\\n     4861 ttttatcatt tctcatctct aaaatcacat atccccatgt aatacaaggg tctttccatt\\n     4921 atgcattcat taaatcattc taggagaggt ctcatcaacc tcctacttta ttaaacatgc\\n     4981 ccacagagag aagggcacag gagtaaagca gaggcaatgt gtcattgctc ccaagtagaa\\n     5041 ggtaaataag gcctctttga ccagcaggag aggaaatgct ggtaggaaga ctcttccagg\\n     5101 atgtaatgca gaagctcagg gcagagctat tcacacttca caccagtgct gtttcctcac\\n     5161 catagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\\n     5221 caccctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca cctggctgag\\n     5281 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\\n     5341 ttccttcttc aagatcagtt acctcacctt cctcccttct gctgatgaga tttatgactg\\n     5401 caaggtggag cactggggcc tggatgagcc tcttctgaaa cactggggta aggatgagtt\\n     5461 tcaccatttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactctat\\n     5521 gtcccaaaac ttgttttcca cacttcatga gtttctttta tctttttttt tttttgaaag\\n     5581 aattaagcaa caaaagcaca gatttattaa aaaagaaagt acactccaca gggtgggagc\\n     5641 aggcctgcca cttcatgggt ttctaataac agacttcact ctcctccctg agccaggggc\\n     5701 cttgagtctt tgcagagcca accctccacc ccatcccatc cctcacacat gcacatgagc\\n     5761 acactctgca ttctgacctc aacaacttca cttccacaga gcctgagatt ccagcaccta\\n     5821 tgtcagagct cacagagact gtggtctgtg ccctggggtt gtctgtgggc ctcgtgggca\\n     5881 ttgtggtggg gaccgtcttg atcatccgag gcctgcgttc agttggtgct tccagacacc\\n     5941 aagggccctt gtgaatccca tcctgaaaaa gaaggtaagt ttgagatttg ttagagctga\\n     6001 agctgcagga aggaaagtgg gaggaggctg tggacatgaa tgtggttgaa agttgtaggg\\n     6061 gaattgggaa gtggcatgat gatgacacag gagccccctt ggacccatcg atctcatgtc\\n     6121 tgtcctgttg caggtgcatc accatctaca gcagcggaag agtggacttg ctacatgacc\\n     6181 tagcactatt ctctggcccg atttatcata ccctttttct cctgcaaatg tttctcctct\\n     6241 taccttttct ctgctttttt ttccttaagc ttctgtatcc cctcagagct cacaaatgcc\\n     6301 tttgaattct ttccctgacc tcctgatttt tttttctttt ctcaggtgtt acctactaag\\n     6361 agatgcctgg ggtaagccgc ccagctacct aattcctcag taacatcgat ctaaaatctc\\n     6421 catggaagca ataaattccc tttaagagat ctatgtcaaa tttttccatc tttcatccgg\\n     6481 ggctgactga acctatggct aagaattggg acactctcat gtttcaagcc aatttcatct\\n     6541 catttcccag atcatatttc atatccagta acacagaagc aaccaagtac aatatagcct\\n     6601 gataatatgt tgatttctta gctgacatta atatttcttt cttctttgtg ttctcaccct\\n     6661 tggcactgcc gcccatccct caattcaggc aacaatgaag ttaatggata ctctctgccc\\n     6721 tttgctcaga attgttatag caaaaatttt aaaaccaaaa aataagtttg tactaatttc\\n     6781 aatatggctt ttaaaagtat gatggagaaa taaattagga taaaggaact ttgaatcaca\\n     6841 aaaatatcaa aagtaaaaat ttattctcaa aactttgaat ttgtaaagaa tgatgacagt\\n     6901 agaagccttc ctctcccctc ctcgccttta gggaataaaa attctttagg taggaaaaga\\n     6961 aatggaagtc agaaaaacat tagaaaaaga cagtaatgtg ggtatctgaa aaggaacaaa\\n     7021 tacttattcc tcacataggg ttagtgacaa tgggaaaagg gataggagta gaagccacag\\n     7081 acatatctag gagccctgaa tagaggcgca gtctgcctca cctcctgaat gaagctttgc\\n     7141 tagataacca cgtagctttc cctgtgccac ccttgcatga agcagacagt atagtggata\\n     7201 tggcaggatg tttctaggaa acatgccgat acaaaacaat gccagtatct tcagaaatcc\\n     7261 ccagcccttt cccctcaccc ctcctggcta aggaaagcac tagcttatga gagaaaccct\\n     7321 aggaggaaca acacagttga ga\\n//\\n\\nLOCUS       OY986028                7419 bp    DNA     linear   PRI 27-DEC-2023\\nDEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\\n            HLA-DQA1*01:02:01:NEW.\\nACCESSION   OY986028\\nVERSION     OY986028.1\\nDBLINK      BioProject: PRJEB71396\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1\\n  AUTHORS   French,A.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\\n            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\\n            Kingdom\\nFEATURES             Location/Qualifiers\\n     source          1..7419\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /clone=\"AN302945\"\\n     gene            728..6045\\n                     /gene=\"HLA-DQA1\"\\n     CDS             join(728..809,4591..4839,5253..5534,5891..6045)\\n                     /gene=\"HLA-DQA1\"\\n                     /function=\"antigen presenting molecule\"\\n                     /codon_start=1\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"CAK6681859.1\"\\n                     /translation=\"MILNKALLLGALALTTVMSPCGGEDIVADHVASCGVNLYQFYGP\\n                     SGQYTHEFDGDEQFYVDLERKETAWRWPEFSKFGGFDPQGALRNMAVAKHNLNIMIKR\\n                     YNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGQSVTEGVSE\\n                     TSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDQPLLKHWEPEIPAPMSELTET\\n                     VVCALGLSVGLMGIVVGTVFIIQGLRSVGASRHQGPL\"\\n     exon            728..809\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     intron          810..4590\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     exon            4591..4839\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     intron          4840..5252\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     exon            5253..5534\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     intron          5535..5890\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\n     exon            5891..6045\\n                     /gene=\"HLA-DQA1\"\\n                     /number=4\\n                     /allele=\"HLA-DQA1*01:02:01:NEW\"\\nORIGIN      \\n        1 ttgtaagcca gaacatcctg aaggatgccg tataactgat gacctcatct atccacgagg\\n       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tctgagcact\\n      121 ttgggaggct gagacgggca gatcatgagg tcaggagttc aagaccagcg tggccaacat\\n      181 agtgaaaccc cgtctctact aaaaatacaa aaattagctg ggcgtggtgg caggtgcctg\\n      241 tagtcccagc tacttgggag ggtgagggag gagaatcgct tgaatccggg aggcaaaggt\\n      301 tgcagtgagc cgagaccttg ccattgtacc acagcctggg tgacagagtg agactacgtc\\n      361 tcaaaaaaaa agaaaaagaa aagaaaaaga aaaaaaaaga aaaacaccca tccctcttgc\\n      421 gactggcaga catgcacaca ccagagaaga ttccaatttc gtgtcctccc tttattcata\\n      481 gaacaattcc tcaagtccac tctgagtaga ggctgcatca caaggggatt gccccgtctc\\n      541 cttccagggc tcttaataca aactcttcaa ctagtaactg agatgtcacc ataggggatt\\n      601 tttctaattg gccaaaacct gacttggcag ggtttggttt gggtgtcttc agatttcctt\\n      661 gtcttgaggt cctcacaatt actctacagc tcagaacacc aactgctgag gctgccttgg\\n      721 gaagaggatg atcctaaaca aagctctgct gctgggggcc ctcgctctga ccaccgtgat\\n      781 gagcccctgt ggaggtgaag acattgtggg tgagtgcatg agtgagggat gttctctgga\\n      841 gctgaaaaac agtaaattga aggaaaagag ataaagcgat ttgcagagaa actgtagaga\\n      901 tttcctaagg gccctttcag tattaagaca attaaaaatt atagctgttc ctccttcagg\\n      961 aaaccagagc cccaacctac tctttttgtt atgtatgctt ttgtgttcac taaggatgct\\n     1021 attctgttta tattatattc agtgactaca gcctggaggt ctctatgtcg ttccgtcatg\\n     1081 attgcctcaa aaattagtga ggtttccatc agtggataat tttttattat taaaaattta\\n     1141 tgaagtgtca ttctcaaatt tccctgaaca acttttgaag cttttcggat gtctccggtg\\n     1201 gtagatcttg gggtcattcc atcaattatg tactctatag atatttaaaa tgttgcccgt\\n     1261 ttctttctct cagacttact cacatttcca catgggaact ggcacaggtg gggagtgggt\\n     1321 aaaggagtcc agcaggctga atgccttcaa caatcatttt accacatggt cctcacttac\\n     1381 tctcagctgc ctcatatgtg tcacctcaca agtaatcaaa taaaatgggc atgtggctaa\\n     1441 gctttgtaaa tagtgaaaac atgaatgtca atttttttta catatttcta ttacagatat\\n     1501 agcttcacat ttcttttctt tagcaaaata agggatcatt ttattttaaa attgagaagt\\n     1561 agaaaaaatt ggtaaaataa atcattttat tctcaaatta tcaacccaaa ttacctgttc\\n     1621 ttcacctcat ctaataaagt cctataaaaa gaaaagtggg ccagacaagg tggctcatgc\\n     1681 ctgtaatccc agcactttgg gaggccgagg caggaggatc atttgagcct gggagcttga\\n     1741 gaccagcctg ggcaacatag caagacctca tctctacaaa aaaagaaaaa gaaaaattag\\n     1801 ccaggcatgg tggtgtgtgc ctgtggtgcc agctactcag aaggctgccg tgggaggacc\\n     1861 acttgagccc atgaggcgga agctgcagtg agccatggtg gcaccactac actccagcca\\n     1921 gggcaacaga gagagacgct gtctcaaaaa gaaagaggaa agaaagcgag gaaggaagga\\n     1981 aggaaggaag gaaggaagga gaaggaagga aggaaggaaa agaaaagaaa agggaaggga\\n     2041 ggaaggaagg aaaaaagaaa gacagaaaga aagaggaagg aaggaagaaa ggaagcacag\\n     2101 attaattatt tggtctctta ctctcctctg cctttgtcgt ccatctcttt ccacctctct\\n     2161 tgatgcatcc ctttctccct cttccgtttc aggagccatc tctggctccc tgctccttta\\n     2221 tagagatgga cagtgggttt gtaaaacaaa agttgaaaag tcagatagtt aaaaggggaa\\n     2281 gtgaactgga aggtacttta aactttcaca actttattaa ccgtggctgc tcccattatg\\n     2341 attttgttcg gcagtggaag tttcacctgc ttctccagag cacttggcgt ctttgttcca\\n     2401 aatttccttt cttcaacctc acaccagagt gccctggtca ggcttggctc attcattagg\\n     2461 cacaatgtgg gcagtgcagg ggaccctcca tactgtaaag ccacatgaga aggttttaac\\n     2521 tccttttaaa attagaaaaa taatggaata ttagagccta agaaaatgtt ttaactttta\\n     2581 attcagccta gattatattg tctttatacc aattcagtca taaaatataa ttttccatat\\n     2641 ttttgtggag gaagaggccc atacaagcaa gaatgctcgg ggctcacatg tcagaatgca\\n     2701 gccttggtca tgactgatcc tggccttcgt atggttctgc tacctgtgtg cctgcccgtc\\n     2761 ttccccaaaa tctatgtggt cctcgaatat agcaactgtc attcaataca catgtttgag\\n     2821 cacacagtga gctaagtttt aaggattcaa agatgaaaag tcatgctgtc ttccctgcag\\n     2881 agggtgctca gattagtgat ggaaacagta tggggtgcag gaaagcagaa ggccattgct\\n     2941 gagcagggca gtggacccag cagaggctga aacaatacaa gtgacttggt tccaggtggg\\n     3001 ccagcagggt gacgtcctct agcaaaattt ggcacccaag acaagtacca gaagaaaaga\\n     3061 aagactgcat gtattccaca tatattcatg tttgaacaag gagtcaagat ttatggcaag\\n     3121 gataaggagg ctttgttggt gacctgttaa gaccaaccag ggcagtcatg ctggataggg\\n     3181 aagaaggtga gctggaagag gaacagacaa cttggacagc cagatgttga gacggagggg\\n     3241 ttggaggtca taatgtggtc aaaaacatgt tgatgagagg actcagctac aaagttgtta\\n     3301 acttaagcag aaacctcaag gatggatttt agaatttctc caggaagtcc taaaagataa\\n     3361 ttccatttca gggagaaaaa caacagacca ctgcaaagac cagggaacat gaaaggataa\\n     3421 tgtagtttgt cttgcttggc agatactttt gaaggatgtt ggactgtaag gctgtcgata\\n     3481 tcctcctcag agaacttact acagtacatt gtatctgttc ccttacctac ctgactcttc\\n     3541 cactattcag tttgttcctt aatggtagac catgcctgat tggtgtttta cacatctccg\\n     3601 gctatgtctg acacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta\\n     3661 aaaggcttac acatgtcttg acaaaaaagt ccagtttggc tcatttggct ggagtcgtac\\n     3721 tgcatggctg ccattctgct ctggcatcct cagacaagca cactgcccat tagaggaaag\\n     3781 agtgtgatat aagtgttgag tcagaatgct gtagacattt agtaacctcc ttcacaggaa\\n     3841 aaaaaaaaaa aggtgggggg aatgacagaa atccaaaaac tagtagagct tccacttttc\\n     3901 atttcagaag aaatcagttg ctctcctcta aggaccacta ctattaacaa aacagagacc\\n     3961 ttagaaggag gcattgttta tttattatat attttgtaat gttattacca atcttgttat\\n     4021 actctttctt ataccctaca attgttagca gaaattattt taaattaata agatcctgca\\n     4081 tgcttttcct ttaaaaaaaa aaaaaaagaa agatctctgt gtagagtgtc ctattctgag\\n     4141 ccagtcctga gaggaaagga agtataatca atttgttatt aaccaatgaa agaattaagt\\n     4201 gaaagataaa tctcaggaag cagagggaag taaacctaat ttctgactaa gaaagctaaa\\n     4261 tactatgata actcattcat tccttctttt gttcaattac attatttaat cataagtcca\\n     4321 tgacgtgcca ggcactcagg aaatagtaaa aattggacat gcgatattct gcccttgtgt\\n     4381 agcgcacact agagtgggaa agaaagtgca cttttaactg gacaactacc aacatgaaga\\n     4441 ggggaggaag caggggctgg aaatgtccac agactgtgcc aaaaaatgaa gcccataata\\n     4501 tttgaaagtc aggttcttcc atcattttgt gtattaaggt tctttcttcc tctgttctcc\\n     4561 gccttcctgc ttgtcatctt cactcatcag ctgaccacgt tgcctcttgt ggtgtaaact\\n     4621 tgtaccagtt ttacggtccc tctggccagt acacccatga atttgatgga gatgagcagt\\n     4681 tctacgtgga cctggagagg aaggagactg cctggcggtg gcctgagttc agcaaatttg\\n     4741 gaggttttga cccgcagggt gcactgagaa acatggctgt ggcaaaacac aacttgaaca\\n     4801 tcatgattaa acgctacaac tctaccgctg ctaccaatgg tatgcgtcca ccattctgcc\\n     4861 tctctttact taagctatcc ctccatacca gggttcatta ttttcttccc aagaggtccc\\n     4921 cagatcttgt tatggcaatt gctgaaattt tatcatctcc catctctaaa atcacatatt\\n     4981 cccatgtaat acaagggtct ttccattatg cattcagcaa atccttctag gagaggtctc\\n     5041 atcaacctcc tactttatta aacatgccca cagagagaag ggcacaggaa taaagcagag\\n     5101 gcaatgtgtc gttgctccca agcagaaggt aaataagacc tctttgacta tcaggtggtg\\n     5161 aaatgctggt aggagggctc ttccaggatg taatgcagaa cttcagggca gagctattca\\n     5221 cacttcacac cagtgctgtt tcctcaccac agaggttcct gaggtcacag tgttttccaa\\n     5281 gtctcccgtg acactgggtc agcccaacac cctcatttgt cttgtggaca acatctttcc\\n     5341 tcctgtggtc aacatcacat ggctgagcaa tgggcagtca gtcacagaag gtgtttctga\\n     5401 gaccagcttc ctctccaaga gtgatcattc cttcttcaag atcagttacc tcaccttcct\\n     5461 cccttctgct gatgagattt atgactgcaa ggtggagcac tggggcctgg accagcctct\\n     5521 tctgaaacac tggggtaagg atgagtttca tcattttttg attctttctt gtctgtcaag\\n     5581 ttcagaactt cctgcctttt actcctatat ccaaaaactt gttttccaca cttcatgggt\\n     5641 ttcttttctg tctctctttt ttttttgaaa gaataaagca aaaaaagcag agatttattg\\n     5701 aaaatgaaag tacactctac aggatgggag tgggcctgcc acttcatggt tttctaatga\\n     5761 tagacttcac tcttctccct aagcctgggg ccttgagtct ttgcagagcc aaccctctac\\n     5821 cccatcccat cccacacaca tgcacatgag cacactctgc attctgacct caacaacttc\\n     5881 acttccacag agcctgagat tccagcccct atgtcagagc tcacagagac tgtggtctgt\\n     5941 gccctggggt tgtctgtggg cctcatgggc attgtggtgg gcactgtctt catcatccaa\\n     6001 ggcctgcgtt cagttggtgc ttccagacac caagggccat tgtgaatccc atcctggaag\\n     6061 ggaaggtaag attgagactg gttacagttg aagcggcagt atgaaaggaa ggaaagtggg\\n     6121 agggcgttgt ggacatgaat gtggttgaaa gttgtagggg aattgggaag tggcatgatg\\n     6181 atgacacagg agccccctcg gacccattga tctcatgtct gccctgttgc aggtgcatcg\\n     6241 ccatctacag gagcagaaga atggacttgc taaatgacct agcactattc tctggcccga\\n     6301 tttatcatat cccttttctc ctccaaatat ttctcctctc accttttctc tgggacttaa\\n     6361 gctgctatat cccctcagag ctcacaaatg cctttacatt ctttccctga cctcctgatt\\n     6421 ttttttttct tttctcaaat gttacctaca cagacatgcc tggagtaagc cacccggcta\\n     6481 cctaattcct cagtaacctc catctaaaat ctccaaggaa gcaataaatt ccttttatga\\n     6541 gatctatgtc aaatttttcc atctttcatc cagggctgac tgaaactatg gctaataatt\\n     6601 ggggtactct tatgtttcaa tccaatttaa cctcatttcc cagatcattt ttcatgtcca\\n     6661 gtaacacaga agccaccaag tacagtatag cctgataata tgttgatttc ttagctgaca\\n     6721 ttaatatttc ttgcttcctt gtgttcccac ccttggcact gccacccacc cctcaattca\\n     6781 ggcaacaatg aaattaatgg ataccgtctg cccttggccc agaattgtta tagcaaaaat\\n     6841 tttagaacca aaaaataagt ctgtactaat ttcaatgtgg cttttaaaag tatgacagag\\n     6901 aaataagtta ggataaagga aatttgaatc tcaaaaatat caaaagtaaa aatgtattct\\n     6961 caaaacttta aatttatgaa gaatgatgac agtagaagcc ttcctcttcc cttgtcacct\\n     7021 tgagataaaa attctttagg caggaaaaga aatggaagtc agaaaaacat tagaataaga\\n     7081 caataatgtg ggtgtctgaa aaggaacaaa tacttattcc tcacataggg ttagtgacaa\\n     7141 tgggaaaagg gacgggagta gaagccacag acatatctag gagccctgaa tagaggcgca\\n     7201 gtctgcctca cctcctgaat gaagctgtac tagataacca tgtagctttc cctgtgccac\\n     7261 tctggcatga aggagacagt atagtgaata tggctacagg atgtttcttg gaaacatgcc\\n     7321 aatatcttca gaaatcccca gccctttccc ctaacccctc ctggctaagg aaagcatgag\\n     7381 cttatgagag aaaccctagg aagaacaaca cagttgaga\\n//\\n\\nLOCUS       OY986027                7342 bp    DNA     linear   PRI 27-DEC-2023\\nDEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\\n            HLA-DQA1*02:01:01:NEW.\\nACCESSION   OY986027\\nVERSION     OY986027.1\\nDBLINK      BioProject: PRJEB71396\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1\\n  AUTHORS   French,A.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\\n            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\\n            Kingdom\\nFEATURES             Location/Qualifiers\\n     source          1..7342\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /clone=\"AN302944\"\\n     gene            732..5954\\n                     /gene=\"HLA-DQA1\"\\n     CDS             join(732..813,4508..4753,5166..5447,5800..5954)\\n                     /gene=\"HLA-DQA1\"\\n                     /function=\"antigen presenting molecule\"\\n                     /codon_start=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"CAK6681858.1\"\\n                     /translation=\"MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGP\\n                     SGQFTHEFDGDEEFYVDLERKETVWKLPLFHRLRFDPQFALTNIAVLKHNLNILIKRS\\n                     NSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSET\\n                     SFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTETV\\n                     VCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL\"\\n     exon            732..813\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          814..4507\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            4508..4753\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          4754..5165\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            5166..5447\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     intron          5448..5799\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\n     exon            5800..5954\\n                     /gene=\"HLA-DQA1\"\\n                     /number=4\\n                     /allele=\"HLA-DQA1*02:01:01:NEW\"\\nORIGIN      \\n        1 ttgtaaccca gaaaatcctg aaggatgcca tataattgat gacctcatct atccatgagg\\n       61 ctgctcagaa atgcccaccc ctggccaggc gcggtggctc atgcctgtaa tccaagcact\\n      121 ttgggaggct gaggcaggca gatcacgagg tcaggagttc aagaccagcg tggccaacat\\n      181 agtgaaactc tgtctctact aaaaatacag aaattagccg ggcatggtgg caggcacctg\\n      241 cagtcccagc tacttgggag ggtgaggcag gagaatcgct tgaacccggg aggcaaaggt\\n      301 tgcagtgagc cgagaccatg ccattgtacc tcagcctggg tgacagagtg agactacgtc\\n      361 tcgaaaaata aagaaaaaga aaagaaaaag aaaaaaaaag aaaaattccc atcccttttg\\n      421 cgaatggcag acatgcacac accagagaag attccaattt agtgtcttcc ctctcttcat\\n      481 agaacaattc ctcaagtcca ctctgagtag aggctgcatc acaacaaggg gattgccctg\\n      541 tctccttcca gggctcttaa tagaaactct tcaactagta actgagatgt caccatgggg\\n      601 gatttttcta attggccaaa acctgacttg gcagggtttg gtttgggtgt cttcagattt\\n      661 ccttgtcttg aggtcctcac aattactcta cagctcagaa cagcaactgc tgaggctgcc\\n      721 ttgggaagaa gatgatccta aacaaagctc tgatgctggg ggccctcgcc ctgaccaccg\\n      781 tgatgagccc ttgtggaggt gaagacattg tgggtgagtg cgtgagtgag gaatgttctc\\n      841 tggagctgaa aaacagtaaa ttgaaggaaa agagagaaag cgatttgcag agaaattgta\\n      901 gagatttcct aagacccctt tcagtattaa gagaattaaa aattatagct gttcctcctt\\n      961 caggaaacca gagccccaac ctactctttt tgttatgtat gcttttgtgt tcactaagga\\n     1021 tgctattctg tttatattat attcagtgac tacagcctgg aggtctctat gtcattccat\\n     1081 catgattgcc tcaaaaatta gtgaggtttc catcagtgga taatttttta ttattaaaaa\\n     1141 tgtatgaagt gtcattctca aatttccctg aacaactttt gaagattttc ggatgtctcc\\n     1201 tgtagtagat cttggggtcg ttccatcaat tatatactct atagatatta aaaaagttgc\\n     1261 ccgtttcttt ctctcagact tactcacatt tccacatggg aactggcaca ggtggggagt\\n     1321 aggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\\n     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\\n     1441 ctaagctttg taaatagtga aaacatggat gtcaattgtt tttacatatt tctattacag\\n     1501 gtatagcttc acatttcttt tctttagcaa aataagggat ccttttagtt taaaattgag\\n     1561 aagtagaaaa aattggtaaa ttaaatcatt ttattctcaa attatcaacc caaattacct\\n     1621 gttcttcacc tcatctaata aagtcctata aaaagaaaag tgggccagac atggtggctc\\n     1681 atgcctgtaa tcccagcact ttgggaggcc gaagcaggag gatcatttga gcctgggagt\\n     1741 ttgagaccag cctgggcaac acagcaagac ctcatctcta ccaaaaaata aaataaaaat\\n     1801 tagccaggca tggtagtgca tgcctgtggt gccagctact cagaaggctg cagtgggagg\\n     1861 agcacttgag tccaggaggt ggaagctgca gtgagccatg atggcaccac tacactccag\\n     1921 ccagggcaac agagagagac cctatctcaa aaagaaagcg gaaagaaaga gagaaaggaa\\n     1981 ggaaagaagg aaagaaggaa gcaaggaagg agaaagggaa gggaagaaag aagaaagaaa\\n     2041 gaaagaaaac agaaggaagg aagcacagat taattatttg gtctcttagt ctcctctgcc\\n     2101 tttgtcgtcc atctcttccc acctctcttc atgcattcct ttctccctct tccctttcag\\n     2161 gatccatctc tgactccctg ctcctttata gagatggaca tgagtttgta aaacaaaagt\\n     2221 tgaaaagtca gatagttaaa aggggaagta aactggaagg tactctaaac tttcacaacc\\n     2281 ttattaaccg tggcagctcc cattctgatt ttgttcagca gtggaagttt caccctctcc\\n     2341 tccagagcgc ttggcttctt tgttccaaat ttcctttctt cagcctcaca ccagagtgcc\\n     2401 ctggtcaggc tcagctcatc cattaggcac aatgtgggca gtgcagggga acctccatac\\n     2461 tgtaaagcca catgagaatg ttttaactcc ttttaaaatt ataaaaaaat gaaattgtag\\n     2521 agcctaagaa aatgttttaa cttttaattc agcctatatt atattgtctt tataccaatt\\n     2581 cagtcataaa atataatttt ccatattttt atggaggaag gcgtccacac aagcaagagt\\n     2641 gcttggggct cacatgtcag aacgcatccc tgatcatggc tgatcctgac cttcgtgtgg\\n     2701 ttctgctaac tatgtgcctg tcagtcttcc ccaaaatcta tgtggtcctc aaatataaca\\n     2761 actgtcattc aatacacatg tttgagcacc cagtgagcta agttttaagg attcaaagat\\n     2821 gaaaagtcat gctgtctccc ctgcagaggg tgctcagact agtgatggaa acagtatggg\\n     2881 atgaaagaaa gcagaaggcc attgctgagc aggcagtgga ctcagcagag gctgaaacta\\n     2941 tacaagtgac ttggttccag ctgggccagc aggataacca gacgaaaaga aggattgcat\\n     3001 atattccata tatatttatg tttgaacaaa gagtcaaggt ttattgcaag gataaggagg\\n     3061 ctttgttggt ggcctgttaa gaccatccag cgtggtcata ctggataggg aagaaggtga\\n     3121 gctggaagag ggatagacaa acttggatgg ccagatgttg agatggagga gctggaggtc\\n     3181 ataacgtggt caaaaacatg ttgatgagag gacttagcta caaagttgtt aacttaagca\\n     3241 gaaacctcaa ggattgattt tatgatttct ccaggaagtc ctaaaagata atttcatttc\\n     3301 agggaggaaa acaacagacc actgcaaaga ccaggaacat gaaaggataa tgtagtttgg\\n     3361 tttgcttggc agatacttgt gaaagatgtt ggactgtaag gctgtcaata tcctcctcgc\\n     3421 agaacttact acagtacatt gtatctgctc ccttacctac ctgactctcc cactattcag\\n     3481 tttgttcctt aatggtagac catgcctgat tggtgtttta cacttcccct gctatgtctg\\n     3541 atacttgtgg atgctcagaa agtggggaag gaaggaaaga tacgatggta aaaggcttac\\n     3601 acatgtcttg accagaatgt tcagtttggc tcatttggct ggagtcatac tgcatggctg\\n     3661 ccattctgct ctggcatcct cagagaagca cactgcccat tagaggaaaa agggtgaata\\n     3721 taaatgttga gtcagaacac tgcagacatt tagtaacctc cttcagagga aaaaaagggt\\n     3781 ggggggaatg acagaaatcc aaaaactagt agagcttcca ctttttcatt tcagaagaaa\\n     3841 tcagttactc tcctctaagg accattacta ttaacaaaac agagacctta gaaggaagca\\n     3901 ttatttattt atcatatatt ttgtaatgtt attaccgttc ttgttatact ctttcttata\\n     3961 ccctaccatt gttagcagaa attattttaa attaataaga tcctgcatgc ttttcctttt\\n     4021 tctaaaaaaa gaaagatctc tgtgtagaat gtcctgttct gagccagtcc tgagaggaaa\\n     4081 ggaagtataa tcaatttgtt attaactgat gaaagaatta agtgaaagat aaaccttagg\\n     4141 aagcagaggg aagttaatct atgactaaga aagttaagta ctctgataac tcattcattc\\n     4201 cttcttctgt tcatttacat tatttaatca caagtccatg atgtgccagg cactcaggaa\\n     4261 atagtgaaaa tcggacacgc gatattctgc ccttgtgtag cacacactgt agtgggaaag\\n     4321 aaagtgcact tttaactgga caactatcaa cacgaagagg ggaggaagca ggggctggaa\\n     4381 atgtccacag actttgccaa agacaaagcc cataatattt gaaagtcagt ttcttccatc\\n     4441 attttgtgta ttaaggtttt ttattctcct gttctctgcc ttcctgcttg tcatcttcac\\n     4501 tcatcagctg accacgttgc ctcttacggt gtaaacttgt accagtctta cggtccctct\\n     4561 ggccagttca cccatgaatt tgatggagac gaggagttct atgtggacct ggagaggaag\\n     4621 gagactgtct ggaagttgcc tctgttccac agacttagat ttgacccgca atttgcactg\\n     4681 acaaacatcg ctgtgctaaa acataacttg aacatcctga ttaaacgctc caactctacc\\n     4741 gctgctacca atggtatgtg tccaccattc tgcctttctt tactgatcta tccctttata\\n     4801 ccaagtttca ttattttctt tccaagaggt ccccagatct tctcatggca attgctgaaa\\n     4861 ttttatcatt tctcatctct aaaatcacat atccccatgt aatacaaggg tctttccatt\\n     4921 atgcattcat taaatcattc taggagaggt ctcatcaacc tcctacttta ttaaacatgc\\n     4981 ccacagagag aagggcacag gagtaaagca gaggcaatgt gtcattgctc ccaagtagaa\\n     5041 ggtaaataag gcctctttga ccagcaggag aggaaatgct ggtaggaaga ctcttccagg\\n     5101 atgtaatgca gaagctcagg gcagagctat tcacacttca caccagtgct gtttcctcac\\n     5161 catagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\\n     5221 caccctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca cctggctgag\\n     5281 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\\n     5341 ttccttcttc aagatcagtt acctcacctt cctcccttct gctgatgaga tttatgactg\\n     5401 caaggtggag cactggggcc tggatgagcc tcttctgaaa cactggggta aggatgagtt\\n     5461 tcaccatttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactctat\\n     5521 gtcccaaaac ttgttttcca cacttcatga gtttctttta tctttttttt tttttgaaag\\n     5581 aattaagcaa caaaagcaca gatttattaa aaaagaaagt acactccaca gggtgggagc\\n     5641 aggcctgcca cttcatgggt ttctaataac agacttcact ctcctccctg agccaggggc\\n     5701 cttgagtctt tgcagagcca accctccacc ccatcccatc cctcacacat gcacatgagc\\n     5761 acactctgca ttctgacctc aacaacttca cttccacaga gcctgagatt ccagcaccta\\n     5821 tgtcagagct cacagagact gtggtctgtg ccctggggtt gtctgtgggc ctcgtgggca\\n     5881 ttgtggtggg gaccgtcttg atcatccgag gcctgcgttc agttggtgct tccagacacc\\n     5941 aagggccctt gtgaatccca tcctgaaaaa gaaggtaagt ttgagatttg ttagagctga\\n     6001 agctgcagga aggaaagtgg gaggaggctg tggacatgaa tgtggttgaa agttgtaggg\\n     6061 gaattgggaa gtggcatgat gatgacacag gagccccctt ggacccatcg atctcatgtc\\n     6121 tgtcctgttg caggtgcatc accatctaca gcagcggaag agtggacttg ctacatgacc\\n     6181 tagcactatt ctctggcccg atttatcata ccctttttct cctgcaaatg tttctcctct\\n     6241 taccttttct ctgctttttt ttccttaagc ttctgtatcc cctcagagct cacaaatgcc\\n     6301 tttgaattct ttccctgacc tcctgatttt tttttctttt ctcaggtgtt acctactaag\\n     6361 agatgcctgg ggtaagccgc ccagctacct aattcctcag taacatcgat ctaaaatctc\\n     6421 catggaagca ataaattccc tttaagagat ctatgtcaaa tttttccatc tttcatccgg\\n     6481 ggctgactga acctatggct aagaattggg acactctcat gtttcaagcc aatttcatct\\n     6541 catttcccag atcatatttc atatccagta acacagaagc aaccaagtac aatatagcct\\n     6601 gataatatgt tgatttctta gctgacatta atatttcttt cttctttgtg ttctcaccct\\n     6661 tggcactgcc gcccatccct caattcaggc aacaatgaag ttaatggata ctctctgccc\\n     6721 tttgctcaga attgttatag caaaaatttt aaaaccaaaa aataagtttg tactaatttc\\n     6781 aatatggctt ttaaaagtat gatggagaaa taaattagga taaaggaact ttgaatcaca\\n     6841 aaaatatcaa aagtaaaaat ttattctcaa aactttgaat ttgtaaagaa tgatgacagt\\n     6901 agaagccttc ctctcccctc ctcgccttta gggaataaaa attctttagg taggaaaaga\\n     6961 aatggaagtc agaaaaacat tagaaaaaga cagtaatgtg ggtatctgaa aaggaacaaa\\n     7021 tacttattcc tcacataggg ttagtgacaa tgggaaaagg gataggagta gaagccacag\\n     7081 acatatctag gagccctgaa tagaggcgca gtctgcctca cctcctgaat gaagctttgc\\n     7141 tagataacca cgtagctttc cctgtgccac ccttgcatga agcagacagt atagtggata\\n     7201 tggcaggatg tttctaggaa acatgccgat acaaaacaat gccagtatct tcagaaatcc\\n     7261 ccagcccttt cccctcaccc ctcctggcta aggaaagcac tagcttatga gagaaaccct\\n     7321 aggaggaaca acacagttga ga\\n//\\n\\nLOCUS       OY986025                7529 bp    DNA     linear   PRI 27-DEC-2023\\nDEFINITION  Homo sapiens HLA-DQA1 gene for MHC class II antigen, allele\\n            HLA-DQA1*05:09:01:NEW.\\nACCESSION   OY986025\\nVERSION     OY986025.1\\nDBLINK      BioProject: PRJEB71396\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1\\n  AUTHORS   French,A.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (19-DEC-2023) Anthony Nolan Research Institute, Anthony\\n            Nolan Research Institute, 77C Fleet Road, Hampstead Heath, United\\n            Kingdom\\nFEATURES             Location/Qualifiers\\n     source          1..7529\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /clone=\"AN302920\"\\n     gene            727..6143\\n                     /gene=\"HLA-DQA1\"\\n     CDS             join(727..808,4628..4873,5286..5567,5989..6143)\\n                     /gene=\"HLA-DQA1\"\\n                     /function=\"antigen presenting molecule\"\\n                     /codon_start=1\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"CAK6681856.1\"\\n                     /translation=\"MILNKALMLGTLALTTVMSPCGGKDIVADHVASYGVNLYQSYGP\\n                     SGQYTHEFDGDEQFYVDLGRKETVWCLPVLRQFRFDPQFALTNIAVLKHNLNSLIKRS\\n                     NSTAATNEVPEVTVFSKSPVTLGQPNILICLVDNIFPPVVNITWLSNGHSVTEGVSET\\n                     SFLSKSDHSFFKISYLTLLPSAEESYDCKVEHWGLDKPLLKHWEPEIPAPMSELTETV\\n                     VCALGLSVGLVGIVVGTVFIIRGLRSVGASRHQGPL\"\\n     exon            727..808\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     intron          809..4627\\n                     /gene=\"HLA-DQA1\"\\n                     /number=1\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     exon            4628..4873\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     intron          4874..5285\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     exon            5286..5567\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     intron          5568..5988\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\n     exon            5989..6143\\n                     /gene=\"HLA-DQA1\"\\n                     /number=4\\n                     /allele=\"HLA-DQA1*05:09:01:NEW\"\\nORIGIN      \\n        1 ttgtaactca gaaaatcctg aaggatgccg tagaattgat gacatcatct atccacgagg\\n       61 ctgctcagaa atgcccaccc ctggccaggg cggtggctca tgcctgtaat ctgagcactt\\n      121 tgggaggctg agatgggcag atcacgaatt caggagttcg agaccagcct ggccaacata\\n      181 gtgaaactct gtctctacta aaaatacaaa aattagccgg gcgtggtggc aggcgcctgt\\n      241 aatcccagct acttgggagg gtgagacagg agaatcactt gaacccggga gtcaaagttt\\n      301 gcagtgagcc gagaccatgc cattgcacct cagcctgggt gacagagtga gactacatct\\n      361 caaaaaaaaa aaaaaaaaga aagaaaaaaa agaaaagaaa tgcccacccc tcttgccact\\n      421 ggcagacatg cacacaccag agaagattcc gatttcgtgt cctccctcta ttcacagaac\\n      481 atttcctcaa gtccactctg agtggaggct gcatcacaac aaggggattg ccctgtctcc\\n      541 ttccagggct cttaatacaa actcttcaac tagtaactga ggtgtcatca taggggattt\\n      601 ttctaattag ccaaaacctg acttggcagg gtttggtttg ggtgtcttca gattgccttg\\n      661 tctcgaggtc ctcacaattg ctctacaact cagaacagca actgctaagg ctgccttggg\\n      721 aagaggatga tcctaaacaa agctctgatg ctggggaccc ttgccctgac caccgtgatg\\n      781 agcccctgtg gaggtaaaga cattgtgggt gagtgtatga gtgagggatg ttctctggag\\n      841 ctggaagaga ggaaattgaa gcaaaagaga gaaagcgatt tgcagagaaa ttgtagagat\\n      901 ttcctaaggg tcccttcagt attaagagat ttaaaaatta tggctgttcc tccttcagga\\n      961 aaccagagtc ccaacctact ctttttgtta tctatgctgt tggcgttcac taaggatgct\\n     1021 attctgttta tattgtattc agtgactata gcctggaggt ctctatgtca ttccatcatg\\n     1081 attgcctcaa aaattagtga ggtttccatc aatggataat tttttattat taaaaatttg\\n     1141 tgaagtatca ttctcaaatt tccctgaaca acttttgaag cttttcggat gtctcctgta\\n     1201 gtagcgcttg gggtaaatga ttccatcaat tatatactct atagatatta agaaagatgc\\n     1261 ccttttcttt ctctcagact tactaacatt tccacgtggg aactggcaca ggtggggagt\\n     1321 gggtaaagga gtccagcagg ctgaatgcct tcaacaatca ttttaccaca tggtcctcac\\n     1381 ttactctcag ctgcctcata tgtgtcacct cacaaataat caaataaaat gggcatgtag\\n     1441 ctaagctttg taaatagtga aaacatgaat gtcaatttta tttttacata tttctattat\\n     1501 aggtatagct tcacatttct tttcttttgc aaaataagga atccttttat tttaaaattg\\n     1561 agaatttata gtagaaaaac ttggtaaatt aaatcatttt attctcaaat tatcaaccca\\n     1621 aattacctgt tcttcaactc atctaatgaa gtcctataaa aagaaaagtg ggccagacat\\n     1681 ggtggctcat gcctgtaatc ccagcacttt gggaggccga ggcaggagga tcctttgagc\\n     1741 ccaggagttg agaccagcct gggcaacata gcaagacctc atctctacaa aaaataaaaa\\n     1801 ttagccaggt gtggtggtgc atgcctgtgg tgccagctac tcagaaggct gcagtgggag\\n     1861 gactgcttga gtccaggagg cggaagctgc agtgagccat gatggcacca ctacactcca\\n     1921 gccagggcaa cagagagaaa ctctgtctca gaaagaaaaa ggaaagaaag agagaaagga\\n     1981 aggaaagaaa gaaagaagga aggaaggaag gaagaagggg aagggaaggg aaggaaggga\\n     2041 aggaaaggga agggaggaaa gaaggaagga aaaaagaaaa agaaagaaaa aagaaagaaa\\n     2101 aagaaagaaa gaaagaaaag agggaaggaa agaaagaaaa aggaaggaag gaagcacaga\\n     2161 ttaattatta atcatttggt ctctcttagt cttctctgtc tttgtcatcc atctatttcc\\n     2221 acctctcttc atgcattcct ttctccctct tccctttcag gatccatctc tgactccctg\\n     2281 cttctttata tggacagtgg ggtttgtaaa acaaaagttg aaaaatcaga tagttaaaag\\n     2341 gtgaagtgaa ctggaaggtc taaacttcca caaccttatt aaccatggct gctcccattc\\n     2401 tgattttgtt cgacggtgga agtttcacct ccttctccag agcacttggc ttctttgttc\\n     2461 caaatttcct ttcttcaacc tcacaccaga gtgccctggt caggctcggc tcatccatta\\n     2521 ggcacaacat gggcaatgca gagaaccctc catactgtaa agccacatga gaatgtttta\\n     2581 actcctttta agattagaaa aaaatgaaat ttcagagcct aagaaaatgt tttaattcag\\n     2641 cctagattgt attgtcttta taccaattca gtcataaaat acaattttcc atattttcat\\n     2701 ggaggaaggg gcccacaaaa gcaagagtgc tcagggctca catgtcagaa tgcagccctg\\n     2761 gtcatggctg atcctggcct tcttatgatt ctgctagtgt gcctgtccgt cttccccaaa\\n     2821 atctatgtcg tcctcaaata tagcaactgt aattcaaaac acgtttgagc acacagtaag\\n     2881 ctaagtttta aggattcaaa gatgaaaagt catgctgtct tccctgcaga gggtgctcag\\n     2941 actagtgatg gaaacggtat gggatgcagg aaagcagaag gccattgctg accagggcag\\n     3001 tggactcagc agaggctgaa actatacaaa tgacttggtt ccagctgggc cagcagggtg\\n     3061 acatcctcca gcaaaactcg gcacccaaga caagtcccag atgaaaagaa ggattgcttt\\n     3121 gtgtttgata aggagtcaag gtttattgca aggataaggc tttgttggtg gcctgttaag\\n     3181 acaatccagg gcagtcatac tggataagga agaaggtgag ctggaagagg aacagacaaa\\n     3241 cagccagata ttgaggtgga ggagctggag gtcataacgt ggtcaaaaac atgtcgatga\\n     3301 gaggacttag ctacaaagtt gttaacttaa gcagaaacct caaggatgaa ttttaggatt\\n     3361 tctccaggaa gtcctaaaag ataatttcat ttcagggaga aaaacaacaa accactggaa\\n     3421 agaccaggga acatgaaagg ataatgtagt ttggtttgct tggcagatac ttatgaagga\\n     3481 tgttggactg taaggctgtt gatagcctcc tcacagaaca tgctacagta cattgtatct\\n     3541 gctcccttac ctacctgact cttccactct tcagtttgtt ccttaatggt agaccatgcc\\n     3601 tgattggtgt tttacacatc ccctgctatg tctgacactt gtggatgctc agaaagtggg\\n     3661 gaaggaagga aagatacaac ggtaaaaggc ttacacgtct tgacaggaat gtccagttcg\\n     3721 gctcatttgg ctggagccac attgcacggc tgccattctg ctctggcatc ctcagacaag\\n     3781 cacactgccc attagaggaa aaagggttta atttacctga gtcctcaagt gaatataagt\\n     3841 gttaagtcag aacactgtag acatttagta acctccttca gaggaaaaaa aaaggtgggg\\n     3901 ggaaatgaca gaaatccaaa aactagtaga gcttccactc ttcatttcag aagaaatcag\\n     3961 ttgctctcct ctaaggacca ttactattaa caaaacagag accttgaagg aggcattgtt\\n     4021 tatttattat atattttgta atgttattac cattcttgtt atactattcc ttatacccta\\n     4081 caattgttag cagacattat tttaaattaa taagatcctg catgcttttc tttttttaaa\\n     4141 aaaaagaaag acctctgtgt agagtgtcct gttctgagcc agtcctgaga ggaaagaaaa\\n     4201 tacaatcagt ttgttattaa ctgaggaaag aattaagtga aagatgaatc ttaggaagca\\n     4261 gaaggaagta aacctaatct ctgactaaga aagctaaata ccataataac tcattcattc\\n     4321 cttcttttgt ttaattacat tatttaatca taagtccgtg atgtgccagg cactcaggaa\\n     4381 atagtaaaaa ctggacatgt gatattctgc ccttgtgtag cgcacattat agtgggaaag\\n     4441 aaagcgcaat tttaaccgga caactaccaa caataagagc ggaggaagca ggggttggaa\\n     4501 atgtccacag gctgtgccaa agatgaagcc cgtaatattt gaaagtcagt ttctttcatc\\n     4561 attttgtgta ttaaggttct ttcttcccct gttctccacc ttcctgcttg tcatcttcac\\n     4621 tcatcagctg accacgtcgc ctcttatggt gtaaacttgt accagtctta cggtccctct\\n     4681 ggccagtaca cccatgaatt tgatggagat gagcagttct acgtggacct ggggaggaag\\n     4741 gagactgtct ggtgtttgcc tgttctcaga caatttagat ttgacccgca atttgcactg\\n     4801 acaaacatcg ctgtcctaaa acataacttg aacagtctga ttaaacgctc caactctacc\\n     4861 gctgctacca atggtatgtg tcaacaattc tgcccctctt tactgattta tcccttcata\\n     4921 ccaagtttca ttattttatt tccaagaggt ccccagatct tctcatggca attgctgaaa\\n     4981 ttttatcatc tcccatctct aaaatcacat attcccatgt aatacaaggg tctttccatt\\n     5041 atccattcat taaatccttc tcggagaggt ctcatcaacc tcctacttta ttaaacatgc\\n     5101 ccacagagag aagggcacag gaataaagcg gaggcaatgt gtcgttgctc ccaagcagaa\\n     5161 ggtaaataag acctctttga ctatcaggtg gtgaaatgct ggtaggaggg ctcttccagg\\n     5221 atgtaatgca gaagctcatg gcagagctat tcacacttca catcagtgct gtttcctcac\\n     5281 cacagaggtt cctgaggtca cagtgttttc caagtctccc gtgacactgg gtcagcccaa\\n     5341 catcctcatc tgtcttgtgg acaacatctt tcctcctgtg gtcaacatca catggctgag\\n     5401 caatgggcac tcagtcacag aaggtgtttc tgagaccagc ttcctctcca agagtgatca\\n     5461 ttccttcttc aagatcagtt acctcaccct cctcccttct gctgaggaga gttatgactg\\n     5521 caaggtggag cactggggac tggacaagcc tcttctgaaa cactggggta aggatgagtt\\n     5581 ccaccacttt ttgatgcttt cttgtctgtc aagttcagaa cttcctgcct tttactccta\\n     5641 tatcccaaaa cttgttttcc acacttcatg ggtttctttc tttctttctt tctttctttc\\n     5701 tttctttctt tctttctttc tttctttctt tctttctttc ttttttgaaa gaataaagca\\n     5761 acaaaagcaa agatttattg aaaatgaaag tacacttcac atggtgggag cgggcctgag\\n     5821 cataggggct caagagccac ttcatgggtt tctaatgata gacttcactc tcctccctaa\\n     5881 gctggggacc ttgagtcttt gcagagccaa ccctccaccc catcccatcc cacacacatg\\n     5941 cacatgagca cactctgctt tctgacctca acgacttcat atccacagag cctgagattc\\n     6001 cagcccctat gtcagagctc acagagactg tggtctgcgc cctggggttg tctgtgggcc\\n     6061 tcgtgggcat tgtggtgggc actgtcttca tcatccgagg cctgcgttca gttggtgctt\\n     6121 ccagacacca agggcccttg tgaatcccat cctggaatgg aaggtaagat tgagatttgt\\n     6181 tagagctgaa tccgcagtat gagaggaagg aaagtggagg aggctgtgga catgaatggt\\n     6241 tgaaagttgt aggggaattg ggaagtggca tgatgatgac ataggagcgg cctaggaccc\\n     6301 atccatctca tgtctgtcct gttgcaggtg catcgccatc tacaggagca gaagagtgga\\n     6361 cttgctacat gacctagcat tattttctgg ccccatttat catatccctt ttctcctcca\\n     6421 aatgtttctc ctctcacctc ttctgtggga cttaaattgc tatatctgct cagagctcac\\n     6481 aaatgccttt gaattatttc cctgacttcc tgattttttt cttttctcaa gtgttaccta\\n     6541 ctaagggatg cctggagtaa gccacccagc tacctaattc ctcagtaacc tccatctata\\n     6601 atctccatgg aagcaacaaa ttccctttat gagatatatg tcaaattttt ccatctttca\\n     6661 tccagggctg actgaaaccg tggctaagaa ttgggagact ctcttgtttc aagccaattt\\n     6721 aacatcattt accagatcat ttgtcatgtc cagtaacaca gaagcaacca actacagtat\\n     6781 agcctgataa catgttgatt tcttagctga cgttaatatt tctttcttcc ttgtgttccc\\n     6841 acccttggca ttgccaccca cccctcaatt aaggcaacaa tgaagttaat ggataccctc\\n     6901 tgcctttggc tcagaaatgt tatagcaaaa attttaaaat aaaaaagtaa gtctgtacta\\n     6961 atttcaatat gacttttaaa agtatgacag agaaatgggt tgggataaac gaaatttgaa\\n     7021 tctcaaaaat atcaatagtg aaaatttatt ctcaaaacta aatttgtgaa gaatgatgac\\n     7081 agtagaagcc ttcctctccc ctcctcaccc tgaaggaata aaatttcctt aggcaggaaa\\n     7141 agaaatggaa gtcagaaaaa cattagaata agacaataat gtgggtatct gaaaaggaac\\n     7201 aaatgctcat tcctcacata gggttagtga caatggggaa agggatggga gtagaagctg\\n     7261 cagacatatc taggagccct agaatagagg cacagtctgc cccacctcct caatgaagct\\n     7321 ttgctagata accatgtagc tttccctgtg ccagcctggc atgaaggaga cagtatagtg\\n     7381 gatatggctg caagatgttt ctaggaaaca tgccgataaa aaaacaatac caatatcttc\\n     7441 agaaatcccc agccctttcc cctaaccctc ctggctaagg aaagcactag cttatgagaa\\n     7501 aaaccctagg aagaacaaca cagttaaga\\n//\\n\\nLOCUS       OQ102522                 886 bp    DNA     linear   PRI 26-DEC-2023\\nDEFINITION  Homo sapiens clone DQA1_113588_00517 MHC class II antigen\\n            (HLA-DQA1) gene, exons 2 through 4 and partial cds.\\nACCESSION   OQ102522\\nVERSION     OQ102522.1\\nKEYWORDS    .\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1  (bases 1 to 886)\\n  AUTHORS   Cereb,N. and Yang,S.Y.\\n  TITLE     HLA new alleles found by Histogenetics\\n  JOURNAL   Unpublished\\nREFERENCE   2  (bases 1 to 886)\\n  AUTHORS   Cereb,N. and Yang,S.Y.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (21-DEC-2022) Histogenetics, 300 Executive Blvd,\\n            Ossining, NY 10562, USA\\nFEATURES             Location/Qualifiers\\n     source          1..886\\n                     /organism=\"Homo sapiens\"\\n                     /mol_type=\"genomic DNA\"\\n                     /db_xref=\"taxon:9606\"\\n                     /clone=\"DQA1_113588_00517\"\\n     gene            <1..>886\\n                     /gene=\"HLA-DQA1\"\\n     mRNA            join(<1..249,350..631,732..>886)\\n                     /gene=\"HLA-DQA1\"\\n                     /product=\"MHC class II antigen\"\\n     CDS             join(<1..249,350..631,732..886)\\n                     /gene=\"HLA-DQA1\"\\n                     /codon_start=3\\n                     /product=\"MHC class II antigen\"\\n                     /protein_id=\"WBG53738.1\"\\n                     /translation=\"DHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLP\\n                     LFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNT\\n                     LICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYD\\n                     CKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVFIIRGLRSV\\n                     GASRHQGPL\"\\n     exon            1..249\\n                     /gene=\"HLA-DQA1\"\\n                     /number=2\\n     gap             250..349\\n                     /estimated_length=unknown\\n     exon            350..631\\n                     /gene=\"HLA-DQA1\"\\n                     /number=3\\n     gap             632..731\\n                     /estimated_length=unknown\\n     exon            732..886\\n                     /gene=\"HLA-DQA1\"\\n                     /number=4\\nORIGIN      \\n        1 ctgaccatgt tgcctcttac ggtgtaaact tgtaccagtc ttatggtccc tctgggcagt\\n       61 acagccatga atttgatgga gacgaggagt tctatgtgga cctggagagg aaggagactg\\n      121 tctggcagtt gcctctgttc cgcagattta gaagatttga cccgcaattt gcactgacaa\\n      181 acatcgctgt gctaaaacat aacttgaaca tcgtgattaa acgctccaac tctaccgctg\\n      241 ctaccaatgn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\\n      301 nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnna ggttcctgag\\n      361 gtcacagtgt tttccaagtc tcccgtgaca ctgggtcagc ccaacaccct catctgtctt\\n      421 gtggacaaca tctttcctcc tgtggtcaac atcacctggc tgagcaatgg gcactcagtc\\n      481 acagaaggtg tttctgagac cagcttcctc tccaagagtg atcattcctt cttcaagatc\\n      541 agttacctca ccttcctccc ttctgctgat gagatttatg actgcaaggt ggagcactgg\\n      601 ggcctggatg agcctcttct gaaacactgg gnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\\n      661 nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn nnnnnnnnnn\\n      721 nnnnnnnnnn nagcctgaga ttccaacacc tatgtcagag ctcacagaga ctgtggtctg\\n      781 cgccctgggg ttgtctgtgg gcctcgtggg cattgtggtg gggaccgtct tcatcatccg\\n      841 aggcctgcgt tcagttggtg cttccagaca ccaagggccc ttgtga\\n//\\n\\n'"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Pesquisa em base de dados de nucleotide/protein/gene\n",
    "pesquisa_ncbi(email,termo,db='nuccore')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A execução da função pesquisa_ncbi para a base de dados 'nuccore' resultou na identificação de 1769 entradas relacionadas. Destas, foram processados os primeiros 5 resultados.\n",
    "\n",
    "O primeiro resultado apresentado é o registo com o Accession Number OY986031, correspondente a um alelo do gene do antígeno de classe II do MHC em Homo sapiens (HLA-DQA1).\n",
    "\n",
    "O output dos resultados da pesquisa na base de dados \"nuccore\" foi em formato \"genbank\". Nos resultados são incorporados detalhes característicos desse tipo de formato incluindo a origem da sequência, características e a própria sequência, sendo essas informações enriquecedoras no que toca ao aprofundamento da análise do gene em questão. \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Transferência do ficheiro correspondente ao gene de interesse"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "De forma a realizar a transferência do ficheiro correspondente ao gene de interesse, em primeiro lugar, foi efetuada uma pesquisa na base de dados \"gene\" de forma a identificar o ID do gene de interesse. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 189 resultados em gene. Irão ser processados 15 resultados.\n",
      "\n",
      "\n",
      "1. HLA-DRB1\n",
      "Official Symbol: HLA-DRB1 and Name: major histocompatibility complex, class II, DR beta 1 [Homo sapiens (human)]\n",
      "Other Aliases: DRB1, HLA-DR1B, HLA-DRB, SS1\n",
      "Other Designations: major histocompatibility complex, class II, DR beta 1 precursor; HLA class II histocompatibility antigen, DR-1 beta chain; MHC class II HLA-DR beta 1 chain; human leucocyte antigen DRB1; lymphocyte antigen DRB1\n",
      "Chromosome: 6; Location: 6p21.32\n",
      "Annotation: Chromosome 6 NC_000006.12 (32578775..32589848, complement)\n",
      "MIM: 142857\n",
      "ID: 3123\n",
      "\n",
      "2. env\n",
      "Envelope surface glycoprotein gp160, precursor [Human immunodeficiency virus 1]\n",
      "Other Aliases: HIV1gp8\n",
      "Other Designations: Envelope surface glycoprotein gp160, precursor; Envelope surface glycoprotein gp120; Envelope transmembrane domain; hypothetical protein\n",
      "Annotation:  NC_001802.1 (5771..8341)\n",
      "ID: 155971\n",
      "\n",
      "3. TERT\n",
      "Official Symbol: TERT and Name: telomerase reverse transcriptase [Homo sapiens (human)]\n",
      "Other Aliases: CMM9, DKCA2, DKCB4, EST2, PFBMFT1, TCS1, TP2, TRT, hEST2, hTRT\n",
      "Other Designations: telomerase reverse transcriptase; telomerase catalytic subunit; telomerase-associated protein 2\n",
      "Chromosome: 5; Location: 5p15.33\n",
      "Annotation: Chromosome 5 NC_000005.10 (1253167..1295068, complement)\n",
      "MIM: 187270\n",
      "ID: 7015\n",
      "\n",
      "4. HLA-B\n",
      "Official Symbol: HLA-B and Name: major histocompatibility complex, class I, B [Homo sapiens (human)]\n",
      "Other Aliases: AS, B-4901, HLAB\n",
      "Other Designations: major histocompatibility complex, class I, B; HLA class I antigen HLA-B; HLA class I histocompatibility antigen, B alpha chain; MHC HLA-B cell surface glycoprotein; MHC HLA-B transmembrane glycoprotein; MHC class 1 antigen; MHC class I antigen HLA-B alpha chain; MHC class I antigen HLA-B heavy chain; MHC class I antigen SHCHA; MHC class I molecule; leukocyte antigen class I-B\n",
      "Chromosome: 6; Location: 6p21.33\n",
      "Annotation: Chromosome 6 NC_000006.12 (31353875..31357179, complement)\n",
      "MIM: 142830\n",
      "ID: 3106\n",
      "\n",
      "5. HLA-A\n",
      "Official Symbol: HLA-A and Name: major histocompatibility complex, class I, A [Homo sapiens (human)]\n",
      "Other Aliases: HLAA\n",
      "Other Designations: HLA class I histocompatibility antigen, A alpha chain; HLA class I histocompatibility antigen, A-1 alpha chain; MHC class I antigen HLA-A heavy chain; leukocyte antigen class I-A\n",
      "Chromosome: 6; Location: 6p22.1\n",
      "Annotation: Chromosome 6 NC_000006.12 (29942532..29945870)\n",
      "MIM: 142800\n",
      "ID: 3105\n",
      "\n",
      "6. HLA-DQB1\n",
      "Official Symbol: HLA-DQB1 and Name: major histocompatibility complex, class II, DQ beta 1 [Homo sapiens (human)]\n",
      "Other Aliases: CELIAC1, HLA-DQB, IDDM1\n",
      "Other Designations: HLA class II histocompatibility antigen, DQ beta 1 chain; MHC class II DQ beta chain; MHC class II HLA-DQ beta glycoprotein; MHC class II antigen DQB1; MHC class II antigen HLA-DQ-beta-1\n",
      "Chromosome: 6; Location: 6p21.32\n",
      "Annotation: Chromosome 6 NC_000006.12 (32659467..32666657, complement)\n",
      "MIM: 604305\n",
      "ID: 3119\n",
      "\n",
      "7. tat\n",
      "Tat [Human immunodeficiency virus 1]\n",
      "Other Aliases: HIV1gp5\n",
      "Other Designations: Tat\n",
      "Annotation:  NC_001802.1 (5377..7970)\n",
      "ID: 155871\n",
      "\n",
      "8. gag\n",
      "Pr55(Gag) [Human immunodeficiency virus 1]\n",
      "Other Aliases: HIV1gp2\n",
      "Other Designations: Pr55(Gag); capsid; matrix; nucleocapsid; p1; p2; p6\n",
      "Annotation:  NC_001802.1 (336..1838)\n",
      "ID: 155030\n",
      "\n",
      "9. HLA-C\n",
      "Official Symbol: HLA-C and Name: major histocompatibility complex, class I, C [Homo sapiens (human)]\n",
      "Other Aliases: D6S204, HLA-JY3, HLAC, HLC-C, MHC, PSORS1\n",
      "Other Designations: HLA class I histocompatibility antigen, C alpha chain; HLA-C antigen; MHC class I antigen heavy chain HLA-C; human leukocyte antigen-C alpha chain; major histocompatibility antigen HLA-C\n",
      "Chromosome: 6; Location: 6p21.33\n",
      "Annotation: Chromosome 6 NC_000006.12 (31268749..31272092, complement)\n",
      "MIM: 142840\n",
      "ID: 3107\n",
      "\n",
      "10. CD14\n",
      "Official Symbol: CD14 and Name: CD14 molecule [Homo sapiens (human)]\n",
      "Other Designations: monocyte differentiation antigen CD14; myeloid cell-specific leucine-rich glycoprotein\n",
      "Chromosome: 5; Location: 5q31.3\n",
      "Annotation: Chromosome 5 NC_000005.10 (140631732..140633701, complement)\n",
      "MIM: 158120\n",
      "ID: 929\n",
      "\n",
      "11. POU5F1\n",
      "Official Symbol: POU5F1 and Name: POU class 5 homeobox 1 [Homo sapiens (human)]\n",
      "Other Aliases: OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4, Oct3/4\n",
      "Other Designations: POU domain, class 5, transcription factor 1; POU domain transcription factor OCT4; POU-type homeodomain-containing DNA-binding protein; octamer-binding protein 3; octamer-binding protein 4; octamer-binding transcription factor 3\n",
      "Chromosome: 6; Location: 6p21.33\n",
      "Annotation: Chromosome 6 NC_000006.12 (31164337..31170682, complement)\n",
      "MIM: 164177\n",
      "ID: 5460\n",
      "\n",
      "12. LPA\n",
      "Official Symbol: LPA and Name: lipoprotein(a) [Homo sapiens (human)]\n",
      "Other Aliases: AK38, APOA, LP\n",
      "Other Designations: apolipoprotein(a); antiangiogenic AK38 protein; apo(a); lipoprotein, Lp(a); lp(a)\n",
      "Chromosome: 6; Location: 6q25.3-q26\n",
      "Annotation: Chromosome 6 NC_000006.12 (160531482..160664275, complement)\n",
      "MIM: 152200\n",
      "ID: 4018\n",
      "\n",
      "13. HLA-DQA1\n",
      "Official Symbol: HLA-DQA1 and Name: major histocompatibility complex, class II, DQ alpha 1 [Homo sapiens (human)]\n",
      "Other Aliases: CELIAC1, DQ-A1, DQA1, HLA-DQA*, HLA-DQB1, HLA-DQA1\n",
      "Other Designations: HLA class II histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha chain; DC-alpha; HLA class II histocompatibility antigen DQ alpha chain; HLA-DCA; MHC HLA-DQ alpha; MHC class II DQ alpha chain; MHC class II DQA1; MHC class II HLA-DQ-alpha-1; MHC class II antigen DQA1; MHC class II protein\n",
      "Chromosome: 6; Location: 6p21.32\n",
      "Annotation: Chromosome 6 NC_000006.12 (32637406..32655272)\n",
      "MIM: 146880\n",
      "ID: 3117\n",
      "\n",
      "14. nef\n",
      "Nef [Human immunodeficiency virus 1]\n",
      "Other Aliases: HIV1gp9\n",
      "Other Designations: Nef\n",
      "Annotation:  NC_001802.1 (8343..8963)\n",
      "ID: 156110\n",
      "\n",
      "15. STIM1\n",
      "Official Symbol: STIM1 and Name: stromal interaction molecule 1 [Homo sapiens (human)]\n",
      "Other Aliases: D11S4896E, GOK, IMD10, STRMK, TAM, TAM1\n",
      "Other Designations: stromal interaction molecule 1\n",
      "Chromosome: 11; Location: 11p15.4\n",
      "Annotation: Chromosome 11 NC_000011.10 (3854604..4093210)\n",
      "MIM: 605921\n",
      "ID: 6786\n",
      "\n",
      "\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\n1. HLA-DRB1\\nOfficial Symbol: HLA-DRB1 and Name: major histocompatibility complex, class II, DR beta 1 [Homo sapiens (human)]\\nOther Aliases: DRB1, HLA-DR1B, HLA-DRB, SS1\\nOther Designations: major histocompatibility complex, class II, DR beta 1 precursor; HLA class II histocompatibility antigen, DR-1 beta chain; MHC class II HLA-DR beta 1 chain; human leucocyte antigen DRB1; lymphocyte antigen DRB1\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32578775..32589848, complement)\\nMIM: 142857\\nID: 3123\\n\\n2. env\\nEnvelope surface glycoprotein gp160, precursor [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp8\\nOther Designations: Envelope surface glycoprotein gp160, precursor; Envelope surface glycoprotein gp120; Envelope transmembrane domain; hypothetical protein\\nAnnotation:  NC_001802.1 (5771..8341)\\nID: 155971\\n\\n3. TERT\\nOfficial Symbol: TERT and Name: telomerase reverse transcriptase [Homo sapiens (human)]\\nOther Aliases: CMM9, DKCA2, DKCB4, EST2, PFBMFT1, TCS1, TP2, TRT, hEST2, hTRT\\nOther Designations: telomerase reverse transcriptase; telomerase catalytic subunit; telomerase-associated protein 2\\nChromosome: 5; Location: 5p15.33\\nAnnotation: Chromosome 5 NC_000005.10 (1253167..1295068, complement)\\nMIM: 187270\\nID: 7015\\n\\n4. HLA-B\\nOfficial Symbol: HLA-B and Name: major histocompatibility complex, class I, B [Homo sapiens (human)]\\nOther Aliases: AS, B-4901, HLAB\\nOther Designations: major histocompatibility complex, class I, B; HLA class I antigen HLA-B; HLA class I histocompatibility antigen, B alpha chain; MHC HLA-B cell surface glycoprotein; MHC HLA-B transmembrane glycoprotein; MHC class 1 antigen; MHC class I antigen HLA-B alpha chain; MHC class I antigen HLA-B heavy chain; MHC class I antigen SHCHA; MHC class I molecule; leukocyte antigen class I-B\\nChromosome: 6; Location: 6p21.33\\nAnnotation: Chromosome 6 NC_000006.12 (31353875..31357179, complement)\\nMIM: 142830\\nID: 3106\\n\\n5. HLA-A\\nOfficial Symbol: HLA-A and Name: major histocompatibility complex, class I, A [Homo sapiens (human)]\\nOther Aliases: HLAA\\nOther Designations: HLA class I histocompatibility antigen, A alpha chain; HLA class I histocompatibility antigen, A-1 alpha chain; MHC class I antigen HLA-A heavy chain; leukocyte antigen class I-A\\nChromosome: 6; Location: 6p22.1\\nAnnotation: Chromosome 6 NC_000006.12 (29942532..29945870)\\nMIM: 142800\\nID: 3105\\n\\n6. HLA-DQB1\\nOfficial Symbol: HLA-DQB1 and Name: major histocompatibility complex, class II, DQ beta 1 [Homo sapiens (human)]\\nOther Aliases: CELIAC1, HLA-DQB, IDDM1\\nOther Designations: HLA class II histocompatibility antigen, DQ beta 1 chain; MHC class II DQ beta chain; MHC class II HLA-DQ beta glycoprotein; MHC class II antigen DQB1; MHC class II antigen HLA-DQ-beta-1\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32659467..32666657, complement)\\nMIM: 604305\\nID: 3119\\n\\n7. tat\\nTat [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp5\\nOther Designations: Tat\\nAnnotation:  NC_001802.1 (5377..7970)\\nID: 155871\\n\\n8. gag\\nPr55(Gag) [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp2\\nOther Designations: Pr55(Gag); capsid; matrix; nucleocapsid; p1; p2; p6\\nAnnotation:  NC_001802.1 (336..1838)\\nID: 155030\\n\\n9. HLA-C\\nOfficial Symbol: HLA-C and Name: major histocompatibility complex, class I, C [Homo sapiens (human)]\\nOther Aliases: D6S204, HLA-JY3, HLAC, HLC-C, MHC, PSORS1\\nOther Designations: HLA class I histocompatibility antigen, C alpha chain; HLA-C antigen; MHC class I antigen heavy chain HLA-C; human leukocyte antigen-C alpha chain; major histocompatibility antigen HLA-C\\nChromosome: 6; Location: 6p21.33\\nAnnotation: Chromosome 6 NC_000006.12 (31268749..31272092, complement)\\nMIM: 142840\\nID: 3107\\n\\n10. CD14\\nOfficial Symbol: CD14 and Name: CD14 molecule [Homo sapiens (human)]\\nOther Designations: monocyte differentiation antigen CD14; myeloid cell-specific leucine-rich glycoprotein\\nChromosome: 5; Location: 5q31.3\\nAnnotation: Chromosome 5 NC_000005.10 (140631732..140633701, complement)\\nMIM: 158120\\nID: 929\\n\\n11. POU5F1\\nOfficial Symbol: POU5F1 and Name: POU class 5 homeobox 1 [Homo sapiens (human)]\\nOther Aliases: OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4, Oct3/4\\nOther Designations: POU domain, class 5, transcription factor 1; POU domain transcription factor OCT4; POU-type homeodomain-containing DNA-binding protein; octamer-binding protein 3; octamer-binding protein 4; octamer-binding transcription factor 3\\nChromosome: 6; Location: 6p21.33\\nAnnotation: Chromosome 6 NC_000006.12 (31164337..31170682, complement)\\nMIM: 164177\\nID: 5460\\n\\n12. LPA\\nOfficial Symbol: LPA and Name: lipoprotein(a) [Homo sapiens (human)]\\nOther Aliases: AK38, APOA, LP\\nOther Designations: apolipoprotein(a); antiangiogenic AK38 protein; apo(a); lipoprotein, Lp(a); lp(a)\\nChromosome: 6; Location: 6q25.3-q26\\nAnnotation: Chromosome 6 NC_000006.12 (160531482..160664275, complement)\\nMIM: 152200\\nID: 4018\\n\\n13. HLA-DQA1\\nOfficial Symbol: HLA-DQA1 and Name: major histocompatibility complex, class II, DQ alpha 1 [Homo sapiens (human)]\\nOther Aliases: CELIAC1, DQ-A1, DQA1, HLA-DQA*, HLA-DQB1, HLA-DQA1\\nOther Designations: HLA class II histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha chain; DC-alpha; HLA class II histocompatibility antigen DQ alpha chain; HLA-DCA; MHC HLA-DQ alpha; MHC class II DQ alpha chain; MHC class II DQA1; MHC class II HLA-DQ-alpha-1; MHC class II antigen DQA1; MHC class II protein\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32637406..32655272)\\nMIM: 146880\\nID: 3117\\n\\n14. nef\\nNef [Human immunodeficiency virus 1]\\nOther Aliases: HIV1gp9\\nOther Designations: Nef\\nAnnotation:  NC_001802.1 (8343..8963)\\nID: 156110\\n\\n15. STIM1\\nOfficial Symbol: STIM1 and Name: stromal interaction molecule 1 [Homo sapiens (human)]\\nOther Aliases: D11S4896E, GOK, IMD10, STRMK, TAM, TAM1\\nOther Designations: stromal interaction molecule 1\\nChromosome: 11; Location: 11p15.4\\nAnnotation: Chromosome 11 NC_000011.10 (3854604..4093210)\\nMIM: 605921\\nID: 6786\\n\\n'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,\"HLA-DQA1\",db='gene',retmax=15)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Através da análise do output da função anterior, podemos observar que o ID correspondente ao gene de interesse é o \"3117\". Assim, torna-se possível proceder à gravação do ficheiro de interesse. \n",
    "\n",
    "Utilizando a função **pesquisa_ncbi** e colocando os seguintes parâmetros na função:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em gene. Irá ser processado 1 resultado.\n",
      "\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\n1. HLA-DQA1\\nOfficial Symbol: HLA-DQA1 and Name: major histocompatibility complex, class II, DQ alpha 1 [Homo sapiens (human)]\\nOther Aliases: CELIAC1, DQ-A1, DQA1, HLA-DQA*, HLA-DQB1, HLA-DQA1\\nOther Designations: HLA class II histocompatibility antigen, DQ alpha 1 chain; DC-1 alpha chain; DC-alpha; HLA class II histocompatibility antigen DQ alpha chain; HLA-DCA; MHC HLA-DQ alpha; MHC class II DQ alpha chain; MHC class II DQA1; MHC class II HLA-DQ-alpha-1; MHC class II antigen DQA1; MHC class II protein\\nChromosome: 6; Location: 6p21.32\\nAnnotation: Chromosome 6 NC_000006.12 (32637406..32655272)\\nMIM: 146880\\nID: 3117\\n\\n'"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'3117',db='gene',rettype='gb',retmax=1,save='y',display='n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Foi gravado com sucesso o ficheiro pretendido.\n",
    "\n",
    "### Transferência do ficheiro correspondente à proteína de interesse\n",
    "\n",
    "Passando agora para a identificação e preparação de dados relacionados à proteína codificada pelo gene HLA-DQA1 para análises subsequentes. Utilizando a função pesquisa_ncbi, direcionamos a pesquisa para a base de dados 'protein' da NCBI, procurando recuperar informações específicas sobre a proteína de interesse."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 1 resultados em protein. Irão ser processados 5 resultados.\n",
      "\n",
      "LOCUS       DQA1_HUMAN               254 aa            linear   PRI 08-NOV-2023\n",
      "DEFINITION  RecName: Full=HLA class II histocompatibility antigen, DQ alpha 1\n",
      "            chain; AltName: Full=DC-1 alpha chain; AltName: Full=DC-alpha;\n",
      "            AltName: Full=HLA-DCA; AltName: Full=MHC class II DQA1; Flags:\n",
      "            Precursor.\n",
      "ACCESSION   P01909\n",
      "VERSION     P01909.1\n",
      "DBSOURCE    UniProtKB: locus DQA1_HUMAN, accession P01909;\n",
      "            class: standard.\n",
      "            extra\n",
      "            accessions:O19630,O19706,P01907,P01908,P04225,P04226,P05536,P79553,\n",
      "            Q06751,Q29876,Q29994,Q2Q6Y6,Q2Q6Y7,Q2Q6Y8,Q2WCM3,Q30064,Q30067,\n",
      "            Q30068,Q30070,Q30071,Q30072,Q30073,Q30086,Q30101,Q5Y7D5,Q5Y7F5,\n",
      "            Q6ICU6,Q6PR46,Q6QDB1,Q860W2,Q860W4,Q9BD37,Q9TPM3,Q9UM31\n",
      "            created: Jul 21, 1986.\n",
      "            sequence updated: Jul 21, 1986.\n",
      "            annotation updated: Nov 8, 2023.\n",
      "            xrefs: X00033.1, CAA24917.1, X00370.1, X00452.1, CAA25141.1,\n",
      "            M16995.1, AAA59760.1, U92032.1, AAB91990.1, AY663395.1, AAU87978.1,\n",
      "            AY663398.1, AAU87987.1, AY663400.1, AAU87992.1, AY663406.1,\n",
      "            AAU88007.1, AY663411.1, AAU88022.1, AY663413.1, AAU88028.1,\n",
      "            DQ178403.1, ABA86855.1, DQ178400.1, DQ178401.1, DQ178402.1,\n",
      "            DQ178407.1, ABA86856.1, DQ178404.1, DQ178405.1, DQ178406.1,\n",
      "            DQ178411.1, ABA86857.1, DQ178408.1, DQ178409.1, DQ178410.1,\n",
      "            DQ178415.1, ABA86858.1, DQ178412.1, DQ178413.1, DQ178414.1,\n",
      "            M29616.1, AAA59759.1, M29613.1, AF322870.1, AAK11577.1, AF322867.1,\n",
      "            AF322868.1, AF322869.1, AF395700.1, AAM69677.1, AF395697.1,\n",
      "            AF395698.1, AF395699.1, CR450297.1, CAG29293.1, AM042559.1,\n",
      "            CAJ14960.1, AM042560.1, CAJ14961.1, AK313975.1, BAG36689.1,\n",
      "            BX248406.4, AL662789.11, Z84489.1, CAB06491.1, BC008585.1,\n",
      "            AAH08585.1, BC157865.1, AAI57866.1, L46875.1, L46876.1, L46877.1,\n",
      "            L46878.1, L46880.1, L46881.1, M11124.1, AAA59754.1, M20431.1,\n",
      "            AAA59758.1, L34082.1, AAC41950.1, L34085.1, AAC41953.1, L34086.1,\n",
      "            AAC41954.1, L34089.1, AAC41957.1, L34090.1, AAC41958.1, L34092.1,\n",
      "            AAC41960.1, L34093.1, AAC41961.1, L34094.1, AAC41962.1, L42625.1,\n",
      "            AAA85334.1, AY197775.1, AAO45622.1, AY547314.1, AAS49496.1,\n",
      "            AY206406.1, AAO47362.1, AY585236.1, AAT09985.1, M34997.1,\n",
      "            AAA35772.1, M34999.1, AAA74633.1, U85035.1, AAB41891.1, AF109734.1,\n",
      "            AAD56720.1, U03675.1, AAB60341.1, M20506.1, AAA59774.1, J00199.1,\n",
      "            M17846.1, AAA59707.1, HLHUD1, HLHUDQ, HLHUD7, HLHU3C, HLHUDC,\n",
      "            B27628, D34512, NP_002113.2, 1JK8_A, 1S9V_A, 1S9V_D, 1UVQ_A,\n",
      "            2NNA_A, 4GG6_A, 4GG6_C, 4OZF_A, 4OZG_A, 4OZG_C, 4OZH_A, 4OZH_C,\n",
      "            4OZI_A, 4OZI_C, 5KSA_A, 5KSB_A, 5KSB_C, 5KSU_A, 5KSU_D, 5KSV_A,\n",
      "            6MFF_A, 6MFG_A, 6MFG_C, 6U3M_A, 6U3M_C, 6U3N_A, 6XP6_A, 6XP6_D,\n",
      "            7SG0_A, 7SG1_A, 7SG1_F, 7SG2_A, 7SG2_F\n",
      "            xrefs (non-sequence databases): PDBsum:1JK8, PDBsum:1S9V,\n",
      "            PDBsum:1UVQ, PDBsum:2NNA, PDBsum:4GG6, PDBsum:4OZF, PDBsum:4OZG,\n",
      "            PDBsum:4OZH, PDBsum:4OZI, PDBsum:5KSA, PDBsum:5KSB, PDBsum:5KSU,\n",
      "            PDBsum:5KSV, PDBsum:6MFF, PDBsum:6MFG, PDBsum:6U3M, PDBsum:6U3N,\n",
      "            PDBsum:6XP6, PDBsum:7SG0, PDBsum:7SG1, PDBsum:7SG2,\n",
      "            AlphaFoldDB:P01909, SMR:P01909, BioGRID:109362, BioGRID:1529240,\n",
      "            IntAct:P01909, ChEMBL:CHEMBL4105884, TCDB:9.A.75.1.1,\n",
      "            GlyConnect:1986, GlyCosmos:P01909, GlyGen:P01909, iPTMnet:P01909,\n",
      "            BioMuta:HLA-DQA1, DMDM:122188, jPOST:P01909, MassIVE:P01909,\n",
      "            PeptideAtlas:P01909, ProteomicsDB:51510, ABCD:P01909,\n",
      "            Antibodypedia:2728, CPTC:P01909, DNASU:3117,\n",
      "            Ensembl:ENST00000343139.11, Ensembl:ENSP00000339398.5,\n",
      "            Ensembl:ENSG00000196735.13, Ensembl:ENST00000374949.2,\n",
      "            Ensembl:ENSP00000364087.2, Ensembl:ENST00000383251.6,\n",
      "            Ensembl:ENSP00000372738.2, Ensembl:ENSG00000206305.12,\n",
      "            Ensembl:ENST00000395363.5, Ensembl:ENSP00000378767.1,\n",
      "            Ensembl:ENST00000399675.5, Ensembl:ENSP00000382583.1,\n",
      "            Ensembl:ENST00000399678.5, Ensembl:ENSP00000382586.1,\n",
      "            Ensembl:ENST00000418023.5, Ensembl:ENSP00000387892.1,\n",
      "            Ensembl:ENSG00000232062.9, Ensembl:ENST00000444296.6,\n",
      "            Ensembl:ENSP00000413237.2, GeneID:3117, KEGG:hsa:3117,\n",
      "            MANE-Select:ENST00000343139.11, UCSC:uc003obr.4, AGR:HGNC:4942,\n",
      "            CTD:3117, DisGeNET:3117, GeneCards:HLA-DQA1, GeneReviews:HLA-DQA1,\n",
      "            HGNC:4942, HPA:ENSG00000196735, MalaCards:HLA-DQA1, MIM:146880,\n",
      "            neXtProt:NX_P01909, NIAGADS:ENSG00000196735, Orphanet:391490,\n",
      "            Orphanet:930, Orphanet:555, Orphanet:243377, PharmGKB:PA35066,\n",
      "            VEuPathDB:HostDB:ENSG00000196735, HOGENOM:CLU_069380_0_0_1,\n",
      "            InParanoid:P01909, OrthoDB:4118190at2759, PhylomeDB:P01909,\n",
      "            TreeFam:TF333797, PathwayCommons:P01909, Reactome:R-HSA-202424,\n",
      "            Reactome:R-HSA-202427, Reactome:R-HSA-202430,\n",
      "            Reactome:R-HSA-202433, Reactome:R-HSA-2132295,\n",
      "            Reactome:R-HSA-389948, Reactome:R-HSA-877300, SignaLink:P01909,\n",
      "            SIGNOR:P01909, BioGRID-ORCS:3117, ChiTaRS:HLA-DQA1,\n",
      "            EvolutionaryTrace:P01909, GenomeRNAi:3117, Pharos:P01909,\n",
      "            PRO:PR:P01909, Proteomes:UP000005640, RNAct:P01909,\n",
      "            Bgee:ENSG00000196735, ExpressionAtlas:P01909, Genevisible:P01909,\n",
      "            GO:0030669, GO:0030666, GO:0012507, GO:0000139, GO:0031902,\n",
      "            GO:0098553, GO:0005765, GO:0016020, GO:0042613, GO:0005886,\n",
      "            GO:0032588, GO:0030658, GO:0023026, GO:0032395, GO:0042605,\n",
      "            GO:0002250, GO:0019886, GO:0006955, GO:0002503, GO:0050778,\n",
      "            GO:0050870, CDD:cd21008, Gene3D:2.60.40.10, InterPro:IPR007110,\n",
      "            InterPro:IPR036179, InterPro:IPR013783, InterPro:IPR003006,\n",
      "            InterPro:IPR003597, InterPro:IPR011162, InterPro:IPR014745,\n",
      "            InterPro:IPR001003, PANTHER:PTHR19944:SF59, PANTHER:PTHR19944,\n",
      "            Pfam:PF07654, Pfam:PF00993, SMART:SM00407, SMART:SM00920,\n",
      "            SUPFAM:SSF48726, SUPFAM:SSF54452, PROSITE:PS50835, PROSITE:PS00290\n",
      "KEYWORDS    3D-structure; Adaptive immunity; Cell membrane; Direct protein\n",
      "            sequencing; Disulfide bond; Endoplasmic reticulum; Endosome;\n",
      "            Glycoprotein; Golgi apparatus; Immunity; Lysosome; Membrane; MHC\n",
      "            II; Reference proteome; Signal; Transmembrane; Transmembrane helix.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (residues 1 to 254)\n",
      "  AUTHORS   Chang,H.C., Moriuchi,T. and Silver,J.\n",
      "  TITLE     The heavy chain of human B-cell alloantigen HLA-DS has a variable\n",
      "            N-terminal region and a constant immunoglobulin-like region\n",
      "  JOURNAL   Nature 305 (5937), 813-815 (1983)\n",
      "   PUBMED   6415485\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*02:01).\n",
      "REFERENCE   2  (residues 1 to 254)\n",
      "  AUTHORS   Schenning,L., Larhammar,D., Bill,P., Wiman,K., Jonsson,A.K.,\n",
      "            Rask,L. and Peterson,P.A.\n",
      "  TITLE     Both alpha and beta chains of HLA-DC class II histocompatibility\n",
      "            antigens display extensive polymorphism in their amino-terminal\n",
      "            domains\n",
      "  JOURNAL   EMBO J 3 (2), 447-452 (1984)\n",
      "   PUBMED   6585297\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*05:01).\n",
      "REFERENCE   3  (residues 1 to 254)\n",
      "  AUTHORS   Auffray,C., Lillie,J.W., Arnot,D., Grossberger,D., Kappes,D. and\n",
      "            Strominger,J.L.\n",
      "  TITLE     Isotypic and allotypic variation of human class II\n",
      "            histocompatibility antigen alpha-chain genes\n",
      "  JOURNAL   Nature 308 (5957), 327-333 (1984)\n",
      "   PUBMED   6584734\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELES DQA1*01:02 AND ALLELE\n",
      "            DQA1*03:01).\n",
      "REFERENCE   4  (residues 1 to 254)\n",
      "  AUTHORS   Schiffenbauer,J., Didier,D.K., Klearman,M., Rice,K., Shuman,S.,\n",
      "            Tieber,V.L., Kittlesen,D.J. and Schwartz,B.D.\n",
      "  TITLE     Complete sequence of the HLA DQ alpha and DQ beta cDNA from a\n",
      "            DR5/DQw3 cell line\n",
      "  JOURNAL   J Immunol 139 (1), 228-233 (1987)\n",
      "   PUBMED   3584986\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*05:01).\n",
      "            Erratum:[J Immunol 1991 Mar 1;146(5):1715]\n",
      "REFERENCE   5  (residues 1 to 254)\n",
      "  AUTHORS   Ellis,M.C., Hetisimer,A.H., Ruddy,D.A., Hansen,S.L., Kronmal,G.S.,\n",
      "            McClelland,E., Quintana,L., Drayna,D.T., Aldrich,M.S. and Mignot,E.\n",
      "  TITLE     HLA class II haplotype and sequence analysis support a role for DQ\n",
      "            in narcolepsy\n",
      "  JOURNAL   Immunogenetics 46 (5), 410-417 (1997)\n",
      "   PUBMED   9271631\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*05:01).\n",
      "REFERENCE   6  (residues 1 to 254)\n",
      "  AUTHORS   Raymond,C.K., Kas,A., Paddock,M., Qiu,R., Zhou,Y., Subramanian,S.,\n",
      "            Chang,J., Palmieri,A., Haugen,E., Kaul,R. and Olson,M.V.\n",
      "  TITLE     Ancient haplotypes of the HLA Class II region\n",
      "  JOURNAL   Genome Res 15 (9), 1250-1257 (2005)\n",
      "   PUBMED   16140993\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:01; DQA1*01:02\n",
      "            AND DQA1*03:01).\n",
      "REFERENCE   7  (residues 1 to 254)\n",
      "  AUTHORS   Lee,K.W., Jung,Y.A. and Oh,D.H.\n",
      "  TITLE     Four novel human leukocyte antigen-DQA1 alleles identified in the\n",
      "            Korean population\n",
      "  JOURNAL   Tissue Antigens 68 (2), 167-172 (2006)\n",
      "   PUBMED   16866887\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:02; DQA1*05:06;\n",
      "            DQA1*05:07 AND DQA1*05:08).\n",
      "REFERENCE   8  (residues 1 to 254)\n",
      "  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\n",
      "            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\n",
      "  TITLE     Class II genes of the human major histocompatibility complex.\n",
      "            Comparisons of the DQ and DX alpha and beta genes\n",
      "  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\n",
      "   PUBMED   3036828\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*03:01).\n",
      "REFERENCE   9  (residues 1 to 254)\n",
      "  AUTHORS   Ashdown,M.L., Leas,L., Gavrilidis,A., Wood,J.M. and Simons,M.J.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JUN-2001) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*01:01).\n",
      "REFERENCE   10 (residues 1 to 254)\n",
      "  AUTHORS   Ebert,L., Schick,M., Neubert,P., Schatten,R., Henze,S. and Korn,B.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-MAY-2004) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELE DQA1*02:01).\n",
      "REFERENCE   11 (residues 1 to 254)\n",
      "  AUTHORS   Voorter,C.E. and van den Berg-Loonen,E.M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JUL-2005) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:02 AND\n",
      "            DQA1*05:09).\n",
      "REFERENCE   12 (residues 1 to 254)\n",
      "  AUTHORS   Ota,T., Suzuki,Y., Nishikawa,T., Otsuki,T., Sugiyama,T., Irie,R.,\n",
      "            Wakamatsu,A., Hayashi,K., Sato,H., Nagai,K., Kimura,K., Makita,H.,\n",
      "            Sekine,M., Obayashi,M., Nishi,T., Shibahara,T., Tanaka,T.,\n",
      "            Ishii,S., Yamamoto,J., Saito,K., Kawai,Y., Isono,Y., Nakamura,Y.,\n",
      "            Nagahari,K., Murakami,K., Yasuda,T., Iwayanagi,T., Wagatsuma,M.,\n",
      "            Shiratori,A., Sudo,H., Hosoiri,T., Kaku,Y., Kodaira,H., Kondo,H.,\n",
      "            Sugawara,M., Takahashi,M., Kanda,K., Yokoi,T., Furuya,T.,\n",
      "            Kikkawa,E., Omura,Y., Abe,K., Kamihara,K., Katsuta,N., Sato,K.,\n",
      "            Tanikawa,M., Yamazaki,M., Ninomiya,K., Ishibashi,T., Yamashita,H.,\n",
      "            Murakawa,K., Fujimori,K., Tanai,H., Kimata,M., Watanabe,M.,\n",
      "            Hiraoka,S., Chiba,Y., Ishida,S., Ono,Y., Takiguchi,S., Watanabe,S.,\n",
      "            Yosida,M., Hotuta,T., Kusano,J., Kanehori,K., Takahashi-Fujii,A.,\n",
      "            Hara,H., Tanase,T.O., Nomura,Y., Togiya,S., Komai,F., Hara,R.,\n",
      "            Takeuchi,K., Arita,M., Imose,N., Musashino,K., Yuuki,H., Oshima,A.,\n",
      "            Sasaki,N., Aotsuka,S., Yoshikawa,Y., Matsunawa,H., Ichihara,T.,\n",
      "            Shiohata,N., Sano,S., Moriya,S., Momiyama,H., Satoh,N., Takami,S.,\n",
      "            Terashima,Y., Suzuki,O., Nakagawa,S., Senoh,A., Mizoguchi,H.,\n",
      "            Goto,Y., Shimizu,F., Wakebe,H., Hishigaki,H., Watanabe,T.,\n",
      "            Sugiyama,A., Takemoto,M., Kawakami,B., Yamazaki,M., Watanabe,K.,\n",
      "            Kumagai,A., Itakura,S., Fukuzumi,Y., Fujimori,Y., Komiyama,M.,\n",
      "            Tashiro,H., Tanigami,A., Fujiwara,T., Ono,T., Yamada,K., Fujii,Y.,\n",
      "            Ozaki,K., Hirao,M., Ohmori,Y., Kawabata,A., Hikiji,T., Kobatake,N.,\n",
      "            Inagaki,H., Ikema,Y., Okamoto,S., Okitani,R., Kawakami,T.,\n",
      "            Noguchi,S., Itoh,T., Shigeta,K., Senba,T., Matsumura,K.,\n",
      "            Nakajima,Y., Mizuno,T., Morinaga,M., Sasaki,M., Togashi,T.,\n",
      "            Oyama,M., Hata,H., Watanabe,M., Komatsu,T., Mizushima-Sugano,J.,\n",
      "            Satoh,T., Shirai,Y., Takahashi,Y., Nakagawa,K., Okumura,K.,\n",
      "            Nagase,T., Nomura,N., Kikuchi,H., Masuho,Y., Yamashita,R.,\n",
      "            Nakai,K., Yada,T., Nakamura,Y., Ohara,O., Isogai,T. and Sugano,S.\n",
      "  TITLE     Complete sequencing and characterization of 21,243 full-length\n",
      "            human cDNAs\n",
      "  JOURNAL   Nat Genet 36 (1), 40-45 (2004)\n",
      "   PUBMED   14702039\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELE DQA1*01:01).;\n",
      "            TISSUE=Trachea\n",
      "REFERENCE   13 (residues 1 to 254)\n",
      "  AUTHORS   Mungall,A.J., Palmer,S.A., Sims,S.K., Edwards,C.A., Ashurst,J.L.,\n",
      "            Wilming,L., Jones,M.C., Horton,R., Hunt,S.E., Scott,C.E.,\n",
      "            Gilbert,J.G., Clamp,M.E., Bethel,G., Milne,S., Ainscough,R.,\n",
      "            Almeida,J.P., Ambrose,K.D., Andrews,T.D., Ashwell,R.I.,\n",
      "            Babbage,A.K., Bagguley,C.L., Bailey,J., Banerjee,R., Barker,D.J.,\n",
      "            Barlow,K.F., Bates,K., Beare,D.M., Beasley,H., Beasley,O.,\n",
      "            Bird,C.P., Blakey,S., Bray-Allen,S., Brook,J., Brown,A.J.,\n",
      "            Brown,J.Y., Burford,D.C., Burrill,W., Burton,J., Carder,C.,\n",
      "            Carter,N.P., Chapman,J.C., Clark,S.Y., Clark,G., Clee,C.M.,\n",
      "            Clegg,S., Cobley,V., Collier,R.E., Collins,J.E., Colman,L.K.,\n",
      "            Corby,N.R., Coville,G.J., Culley,K.M., Dhami,P., Davies,J.,\n",
      "            Dunn,M., Earthrowl,M.E., Ellington,A.E., Evans,K.A., Faulkner,L.,\n",
      "            Francis,M.D., Frankish,A., Frankland,J., French,L., Garner,P.,\n",
      "            Garnett,J., Ghori,M.J., Gilby,L.M., Gillson,C.J., Glithero,R.J.,\n",
      "            Grafham,D.V., Grant,M., Gribble,S., Griffiths,C., Griffiths,M.,\n",
      "            Hall,R., Halls,K.S., Hammond,S., Harley,J.L., Hart,E.A.,\n",
      "            Heath,P.D., Heathcott,R., Holmes,S.J., Howden,P.J., Howe,K.L.,\n",
      "            Howell,G.R., Huckle,E., Humphray,S.J., Humphries,M.D., Hunt,A.R.,\n",
      "            Johnson,C.M., Joy,A.A., Kay,M., Keenan,S.J., Kimberley,A.M.,\n",
      "            King,A., Laird,G.K., Langford,C., Lawlor,S., Leongamornlert,D.A.,\n",
      "            Leversha,M., Lloyd,C.R., Lloyd,D.M., Loveland,J.E., Lovell,J.,\n",
      "            Martin,S., Mashreghi-Mohammadi,M., Maslen,G.L., Matthews,L.,\n",
      "            McCann,O.T., McLaren,S.J., McLay,K., McMurray,A., Moore,M.J.,\n",
      "            Mullikin,J.C., Niblett,D., Nickerson,T., Novik,K.L., Oliver,K.,\n",
      "            Overton-Larty,E.K., Parker,A., Patel,R., Pearce,A.V., Peck,A.I.,\n",
      "            Phillimore,B., Phillips,S., Plumb,R.W., Porter,K.M., Ramsey,Y.,\n",
      "            Ranby,S.A., Rice,C.M., Ross,M.T., Searle,S.M., Sehra,H.K.,\n",
      "            Sheridan,E., Skuce,C.D., Smith,S., Smith,M., Spraggon,L.,\n",
      "            Squares,S.L., Steward,C.A., Sycamore,N., Tamlyn-Hall,G., Tester,J.,\n",
      "            Theaker,A.J., Thomas,D.W., Thorpe,A., Tracey,A., Tromans,A.,\n",
      "            Tubby,B., Wall,M., Wallis,J.M., West,A.P., White,S.S.,\n",
      "            Whitehead,S.L., Whittaker,H., Wild,A., Willey,D.J., Wilmer,T.E.,\n",
      "            Wood,J.M., Wray,P.W., Wyatt,J.C., Young,L., Younger,R.M.,\n",
      "            Bentley,D.R., Coulson,A., Durbin,R., Hubbard,T., Sulston,J.E.,\n",
      "            Dunham,I., Rogers,J. and Beck,S.\n",
      "  TITLE     The DNA sequence and analysis of human chromosome 6\n",
      "  JOURNAL   Nature 425 (6960), 805-811 (2003)\n",
      "   PUBMED   14574404\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA] (ALLELES DQA1*01:02;\n",
      "            DQA1*03:01 AND DQA1*05:01).\n",
      "REFERENCE   14 (residues 1 to 254)\n",
      "  AUTHORS   Gerhard,D.S., Wagner,L., Feingold,E.A., Shenmen,C.M., Grouse,L.H.,\n",
      "            Schuler,G., Klein,S.L., Old,S., Rasooly,R., Good,P., Guyer,M.,\n",
      "            Peck,A.M., Derge,J.G., Lipman,D., Collins,F.S., Jang,W., Sherry,S.,\n",
      "            Feolo,M., Misquitta,L., Lee,E., Rotmistrovsky,K., Greenhut,S.F.,\n",
      "            Schaefer,C.F., Buetow,K., Bonner,T.I., Haussler,D., Kent,J.,\n",
      "            Kiekhaus,M., Furey,T., Brent,M., Prange,C., Schreiber,K.,\n",
      "            Shapiro,N., Bhat,N.K., Hopkins,R.F., Hsie,F., Driscoll,T.,\n",
      "            Soares,M.B., Casavant,T.L., Scheetz,T.E., Brown-stein,M.J.,\n",
      "            Usdin,T.B., Toshiyuki,S., Carninci,P., Piao,Y., Dudekula,D.B.,\n",
      "            Ko,M.S., Kawakami,K., Suzuki,Y., Sugano,S., Gruber,C.E.,\n",
      "            Smith,M.R., Simmons,B., Moore,T., Waterman,R., Johnson,S.L.,\n",
      "            Ruan,Y., Wei,C.L., Mathavan,S., Gunaratne,P.H., Wu,J., Garcia,A.M.,\n",
      "            Hulyk,S.W., Fuh,E., Yuan,Y., Sneed,A., Kowis,C., Hodgson,A.,\n",
      "            Muzny,D.M., McPherson,J., Gibbs,R.A., Fahey,J., Helton,E.,\n",
      "            Ketteman,M., Madan,A., Rodrigues,S., Sanchez,A., Whiting,M.,\n",
      "            Madari,A., Young,A.C., Wetherby,K.D., Granite,S.J., Kwong,P.N.,\n",
      "            Brinkley,C.P., Pearson,R.L., Bouffard,G.G., Blakesly,R.W.,\n",
      "            Green,E.D., Dickson,M.C., Rodriguez,A.C., Grimwood,J., Schmutz,J.,\n",
      "            Myers,R.M., Butterfield,Y.S., Griffith,M., Griffith,O.L.,\n",
      "            Krzywinski,M.I., Liao,N., Morin,R., Palmquist,D., Petrescu,A.S.,\n",
      "            Skalska,U., Smailus,D.E., Stott,J.M., Schnerch,A., Schein,J.E.,\n",
      "            Jones,S.J., Holt,R.A., Baross,A., Marra,M.A., Clifton,S.,\n",
      "            Makowski,K.A., Bosak,S. and Malek,J.\n",
      "  CONSRTM   MGC Project Team\n",
      "  TITLE     The status, quality, and expansion of the NIH full-length cDNA\n",
      "            project: the Mammalian Gene Collection (MGC)\n",
      "  JOURNAL   Genome Res 14 (10B), 2121-2127 (2004)\n",
      "   PUBMED   15489334\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELES DQA1*03:01 AND\n",
      "            DQA1*05:01).;\n",
      "            TISSUE=Lymph\n",
      "            Erratum:[Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to\n",
      "            Morin, Ryan]]\n",
      "REFERENCE   15 (residues 1 to 254)\n",
      "  AUTHORS   Yasunaga,S., Kimura,A. and Sasazuki,T.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JUL-1996) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 1-27 (ALLELES\n",
      "            DQA1*01:01/DQA1*01:02/DQA1*01:03; DQA1*01:04; DQA1*01:05;\n",
      "            DQA1*02:01/DQA1*03:01/DQA1*03:03; DQA1*04:01/DQA1*06:01 AND\n",
      "            DQA1*05:01/DQA1*05:03/DQA1*05:05).;\n",
      "            TISSUE=B-cell\n",
      "REFERENCE   16 (residues 1 to 254)\n",
      "  AUTHORS   Moriuchi,J., Moriuchi,T. and Silver,J.\n",
      "  TITLE     Nucleotide sequence of an HLA-DQ alpha chain derived from a DRw9\n",
      "            cell line: genetic and evolutionary implications\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 82 (10), 3420-3424 (1985)\n",
      "   PUBMED   3879967\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 19-254 (ALLELE DQA1*03:02).\n",
      "REFERENCE   17 (residues 1 to 254)\n",
      "  AUTHORS   Lock,C.B., So,A.K., Welsh,K.I., Parkes,J.D. and Trowsdale,J.\n",
      "  TITLE     MHC class II sequences of an HLA-DR2 narcoleptic\n",
      "  JOURNAL   Immunogenetics 27 (6), 449-455 (1988)\n",
      "   PUBMED   3259543\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 23-254 (ALLELE DQA1*01:02).\n",
      "REFERENCE   18 (residues 1 to 254)\n",
      "  AUTHORS   Yasunaga,S., Kimura,A., Hamaguchi,K., Ronningen,K.S. and\n",
      "            Sasazuki,T.\n",
      "  TITLE     Different contribution of HLA-DR and -DQ genes in susceptibility\n",
      "            and resistance to insulin-dependent diabetes mellitus (IDDM)\n",
      "  JOURNAL   Tissue Antigens 47 (1), 37-48 (1996)\n",
      "   PUBMED   8929711\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-250 (ALLELE DQA1*03:02/DQA1*03:03)\n",
      "            AND NUCLEOTIDE SEQUENCE [MRNA] OF 24-249 (ALLELES DQA1*01:01;\n",
      "            DQA1*01:03; DQA1*01:04; DQA1*04:01; DQA1*05:03; DQA1*05:05 AND\n",
      "            DQA1*06:01).\n",
      "REFERENCE   19 (residues 1 to 254)\n",
      "  AUTHORS   Yasunaga,S.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JAN-1996) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-249 (ALLELE DQA1*01:05).\n",
      "REFERENCE   20 (residues 1 to 254)\n",
      "  AUTHORS   Goetz,H., Kratzin,H., Thinnes,F.P., Yang,C.-Y., Kruse,T., Pauly,E.,\n",
      "            Koelbel,S., Egert,G., Wernet,P. and Hilschmann,N.\n",
      "  TITLE     Primary structure of human class II histocompatibility antigens 3rd\n",
      "            communication. Amino acid sequence comparison between DR and DC\n",
      "            subclass antigens derived from a lymphoblastoid B cell line\n",
      "            homozygous at the HLA loci (HLA-A3,3; B7,7; Dw2,2; DR2,2: MT1,1;\n",
      "            Dc1,1: MB1,1)\n",
      "  JOURNAL   Hoppe-Seyler's Z. Physiol. Chem. 364, 749-755 (1983)\n",
      "   PUBMED   6576979\n",
      "  REMARK    PROTEIN SEQUENCE OF 24-89; 102-120 AND 164-212 (ALLELE DQA1*01:02).\n",
      "REFERENCE   21 (residues 1 to 254)\n",
      "  AUTHORS   Kao,H.T., Gregersen,P.K., Tang,J.C., Takahashi,T., Wang,C.Y. and\n",
      "            Silver,J.\n",
      "  TITLE     Molecular analysis of the HLA class II genes in two DRw6-related\n",
      "            haplotypes, DRw13 DQw1 and DRw14 DQw3\n",
      "  JOURNAL   J Immunol 142 (5), 1743-1747 (1989)\n",
      "   PUBMED   2493052\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-209 (ALLELE DQA1*05:01).\n",
      "REFERENCE   22 (residues 1 to 254)\n",
      "  AUTHORS   Cordovado,S.K., Hancock,L.N., Simone,A.E., Hendrix,M. and\n",
      "            Mueller,P.W.\n",
      "  TITLE     High-resolution genotyping of HLA-DQA1 in the GoKinD study and\n",
      "            identification of novel alleles HLA-DQA1*040102, HLA-DQA1*0402 and\n",
      "            HLA-DQA1*0404\n",
      "  JOURNAL   Tissue Antigens 65 (5), 448-458 (2005)\n",
      "   PUBMED   15853899\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELES DQA1*04:02 AND\n",
      "            DQA1*04:04).\n",
      "REFERENCE   23 (residues 1 to 254)\n",
      "  AUTHORS   Simone,A.E., Cordovado,S.K., Hendrix,M.M. and Mueller,P.W.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-DEC-2002) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELE DQA1*06:02).\n",
      "REFERENCE   24 (residues 1 to 254)\n",
      "  AUTHORS   Hancock,L.N., Cordovado,S.K. and Mueller,P.W.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-MAR-2004) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELE DQA1*01:07).\n",
      "REFERENCE   25 (residues 1 to 254)\n",
      "  AUTHORS   Horn,G.T., Bugawan,T.L., Long,C.M., Manos,M.M. and Erlich,H.A.\n",
      "  TITLE     Sequence analysis of HLA class II genes from insulin-dependent\n",
      "            diabetic individuals\n",
      "  JOURNAL   Hum Immunol 21 (4), 249-263 (1988)\n",
      "   PUBMED   3372263\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-110 (ALLELE DQA1*01:02),\n",
      "            AND NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-109 (ALLELE\n",
      "            DQA1*05:01).\n",
      "REFERENCE   26 (residues 1 to 254)\n",
      "  AUTHORS   Trejaut,J.A., Greville,W.D. and Dunckley,H.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JAN-1997) to the EMBL/GenBank/DDBJ databases\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-109 (ALLELE DQA1*05:04).\n",
      "REFERENCE   27 (residues 1 to 254)\n",
      "  AUTHORS   Luo,M., Blanchard,J., Maclean,I. and Brunham,R.\n",
      "  TITLE     Identification of a novel HLA-DQA1 allele (DQA1*0106) by\n",
      "            sequence-based DQA1 typing\n",
      "  JOURNAL   Tissue Antigens 53 (6), 595-596 (1999)\n",
      "   PUBMED   10395113\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 31-109 (ALLELE DQA1*01:06).\n",
      "REFERENCE   28 (residues 1 to 254)\n",
      "  AUTHORS   Marsh,S.G. and Bodmer,J.G.\n",
      "  TITLE     HLA class II nucleotide sequences, 1992\n",
      "  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\n",
      "   PUBMED   1362295\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 35-74 (ALLELE DQA1*05:02).\n",
      "            Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\n",
      "            1529427]\n",
      "REFERENCE   29 (residues 1 to 254)\n",
      "  AUTHORS   Liu,C.P., Bach,F.H. and Wu,S.K.\n",
      "  TITLE     Molecular studies of a rare DR2/LD-5a/DQw3 HLA class II haplotype.\n",
      "            Multiple genetic mechanisms in the generation of polymorphic HLA\n",
      "            class II genes\n",
      "  JOURNAL   J Immunol 140 (10), 3631-3639 (1988)\n",
      "   PUBMED   3129499\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 36-254 (ALLELE DQA1*05:01).\n",
      "            Erratum:[J Immunol 1990 Feb 15;144(4):1544]\n",
      "REFERENCE   30 (residues 1 to 254)\n",
      "  AUTHORS   Liu,C.P., Bach,F.H. and Wu,S.K.\n",
      "  JOURNAL   J. Immunol. 144, 15441-15441 (1990)\n",
      "  REMARK    ERRATUM OF PUBMED:3129499.\n",
      "REFERENCE   31 (residues 1 to 254)\n",
      "  AUTHORS   Auffray,C., Korman,A.J., Roux-Dosseto,M., Bono,R. and\n",
      "            Strominger,J.L.\n",
      "  TITLE     cDNA clone for the heavy chain of the human B cell alloantigen DC1:\n",
      "            strong sequence homology to the HLA-DR heavy chain\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 79 (20), 6337-6341 (1982)\n",
      "   PUBMED   6815651\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 40-254 (ALLELE DQA1*03:01).\n",
      "REFERENCE   32 (residues 1 to 254)\n",
      "  AUTHORS   Gyllensten,U.B. and Erlich,H.A.\n",
      "  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\n",
      "            primates\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\n",
      "   PUBMED   2513578\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 41-102 (ALLELE DQA1*05:01),\n",
      "            AND NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 41-103 (ALLELE\n",
      "            DQA1*01:02).\n",
      "REFERENCE   33 (residues 1 to 254)\n",
      "  AUTHORS   Turco,E., Care,A., Compagnone-Post,P., Robinson,C., Cascino,I. and\n",
      "            Trucco,M.\n",
      "  TITLE     Allelic forms of the alpha- and beta-chain genes encoding\n",
      "            DQw1-positive heterodimers\n",
      "  JOURNAL   Immunogenetics 26 (4-5), 282-290 (1987)\n",
      "   PUBMED   2888727\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 119-254 (ALLELE DQA1*01:02).\n",
      "REFERENCE   34 (residues 1 to 254)\n",
      "  AUTHORS   Cresswell,P.\n",
      "  TITLE     Invariant chain structure and MHC class II function\n",
      "  JOURNAL   Cell 84 (4), 505-507 (1996)\n",
      "   PUBMED   8598037\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   35 (residues 1 to 254)\n",
      "  AUTHORS   Villadangos,J.A.\n",
      "  TITLE     Presentation of antigens by MHC class II molecules: getting the\n",
      "            most out of them\n",
      "  JOURNAL   Mol Immunol 38 (5), 329-346 (2001)\n",
      "   PUBMED   11684289\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   36 (residues 1 to 254)\n",
      "  AUTHORS   Menendez-Benito,V. and Neefjes,J.\n",
      "  TITLE     Autophagy in MHC class II presentation: sampling from within\n",
      "  JOURNAL   Immunity 26 (1), 1-3 (2007)\n",
      "   PUBMED   17241953\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   37 (residues 1 to 254)\n",
      "  AUTHORS   Rocha,N. and Neefjes,J.\n",
      "  TITLE     MHC class II molecules on the move for successful antigen\n",
      "            presentation\n",
      "  JOURNAL   EMBO J 27 (1), 1-5 (2008)\n",
      "   PUBMED   18046453\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   38 (residues 1 to 254)\n",
      "  AUTHORS   Berger,A.C. and Roche,P.A.\n",
      "  TITLE     MHC class II transport at a glance\n",
      "  JOURNAL   J Cell Sci 122 (Pt 1), 1-4 (2009)\n",
      "   PUBMED   19092054\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   39 (residues 1 to 254)\n",
      "  AUTHORS   Beswick,E.J. and Reyes,V.E.\n",
      "  TITLE     CD74 in antigen presentation, inflammation, and cancers of the\n",
      "            gastrointestinal tract\n",
      "  JOURNAL   World J Gastroenterol 15 (23), 2855-2861 (2009)\n",
      "   PUBMED   19533806\n",
      "  REMARK    REVIEW.\n",
      "REFERENCE   40 (residues 1 to 254)\n",
      "  AUTHORS   Lee,K.H., Wucherpfennig,K.W. and Wiley,D.C.\n",
      "  TITLE     Structure of a human insulin peptide-HLA-DQ8 complex and\n",
      "            susceptibility to type 1 diabetes\n",
      "  JOURNAL   Nat Immunol 2 (6), 501-507 (2001)\n",
      "   PUBMED   11376336\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (2.4 ANGSTROMS) OF 27-207 OF\n",
      "            HLA-DQA1/HLA-DQB1 HETERODIMER IN COMPLEX WITH INS PEPTIDE, SUBUNIT,\n",
      "            GLYCOSYLATION AT ASN-103, AND DISULFIDE BOND.\n",
      "            Erratum:[Nat Immunol 2001 Sep;2(9):889]\n",
      "REFERENCE   41 (residues 1 to 254)\n",
      "  AUTHORS   Siebold,C., Hansen,B.E., Wyer,J.R., Harlos,K., Esnouf,R.E.,\n",
      "            Svejgaard,A., Bell,J.I., Strominger,J.L., Jones,E.Y. and Fugger,L.\n",
      "  TITLE     Crystal structure of HLA-DQ0602 that protects against type 1\n",
      "            diabetes and confers strong susceptibility to narcolepsy\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 101 (7), 1999-2004 (2004)\n",
      "   PUBMED   14769912\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (1.8 ANGSTROMS) OF 24-219 OF\n",
      "            HLA-DQA1/HLA-DQB1 HETERODIMER (HLA-DQ0602) IN COMPLEX WITH HCRT\n",
      "            PEPTIDE, POLYMORPHISM, SUBUNIT, AND DISULFIDE BOND.\n",
      "REFERENCE   42 (residues 1 to 254)\n",
      "  AUTHORS   Kim,C.Y., Quarsten,H., Bergseng,E., Khosla,C. and Sollid,L.M.\n",
      "  TITLE     Structural basis for HLA-DQ2-mediated presentation of gluten\n",
      "            epitopes in celiac disease\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 101 (12), 4175-4179 (2004)\n",
      "   PUBMED   15020763\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (2.22 ANGSTROMS) OF 24-216 OF\n",
      "            HLA-DQA1/HLA-DQB1 HETERODIMER (DQ2) IN COMPLEX WITH TRITICUM\n",
      "            AESTIVUM ALPHA/BETA-GLIADIN PEPTIDE, SUBUNIT, AND POLYMORPHISM.\n",
      "REFERENCE   43 (residues 1 to 254)\n",
      "  AUTHORS   Henderson,K.N., Tye-Din,J.A., Reid,H.H., Chen,Z., Borg,N.A.,\n",
      "            Beissbarth,T., Tatham,A., Mannering,S.I., Purcell,A.W., Dudek,N.L.,\n",
      "            van Heel,D.A., McCluskey,J., Rossjohn,J. and Anderson,R.P.\n",
      "  TITLE     A structural and immunological basis for the role of human\n",
      "            leukocyte antigen DQ8 in celiac disease\n",
      "  JOURNAL   Immunity 27 (1), 23-34 (2007)\n",
      "   PUBMED   17629515\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (2.1 ANGSTROMS) OF 24-206 OF\n",
      "            HLA-DQA1/HLA-DQB1 HETERODIMER IN COMPLEX WITH TRITICUM AESTIVUM\n",
      "            ALPHA/BETA-GLIADIN, SUBUNIT, AND DISULFIDE BOND.\n",
      "COMMENT     On or before Mar 30, 2010 this sequence version replaced P01908.2,\n",
      "            P05536.1, gi:74721757, gi:74705624, P01907.2, P04225.1, P04226.1,\n",
      "            gi:74706178, gi:74706836, gi:121945665, gi:121948343, gi:121945666,\n",
      "            gi:121941983, gi:74706863, gi:74706866, gi:74706867, gi:74706869,\n",
      "            gi:74706870, gi:74706871, gi:74706872, gi:74709986, gi:74727328,\n",
      "            gi:74713787, gi:74727342, gi:74717483, gi:74720013, gi:70108.\n",
      "            [FUNCTION] Binds peptides derived from antigens that access the\n",
      "            endocytic route of antigen presenting cells (APC) and presents them\n",
      "            on the cell surface for recognition by the CD4 T-cells. The peptide\n",
      "            binding cleft accommodates peptides of 10-30 residues. The peptides\n",
      "            presented by MHC class II molecules are generated mostly by\n",
      "            degradation of proteins that access the endocytic route, where they\n",
      "            are processed by lysosomal proteases and other hydrolases.\n",
      "            Exogenous antigens that have been endocytosed by the APC are thus\n",
      "            readily available for presentation via MHC II molecules, and for\n",
      "            this reason this antigen presentation pathway is usually referred\n",
      "            to as exogenous. As membrane proteins on their way to degradation\n",
      "            in lysosomes as part of their normal turn-over are also contained\n",
      "            in the endosomal/lysosomal compartments, exogenous antigens must\n",
      "            compete with those derived from endogenous components. Autophagy is\n",
      "            also a source of endogenous peptides, autophagosomes constitutively\n",
      "            fuse with MHC class II loading compartments. In addition to APCs,\n",
      "            other cells of the gastrointestinal tract, such as epithelial\n",
      "            cells, express MHC class II molecules and CD74 and act as APCs,\n",
      "            which is an unusual trait of the GI tract. To produce a MHC class\n",
      "            II molecule that presents an antigen, three MHC class II molecules\n",
      "            (heterodimers of an alpha and a beta chain) associate with a CD74\n",
      "            trimer in the ER to form a heterononamer. Soon after the entry of\n",
      "            this complex into the endosomal/lysosomal system where antigen\n",
      "            processing occurs, CD74 undergoes a sequential degradation by\n",
      "            various proteases, including CTSS and CTSL, leaving a small\n",
      "            fragment termed CLIP (class-II-associated invariant chain peptide).\n",
      "            The removal of CLIP is facilitated by HLA-DM via direct binding to\n",
      "            the alpha-beta-CLIP complex so that CLIP is released. HLA-DM\n",
      "            stabilizes MHC class II molecules until primary high affinity\n",
      "            antigenic peptides are bound. The MHC II molecule bound to a\n",
      "            peptide is then transported to the cell membrane surface. In\n",
      "            B-cells, the interaction between HLA-DM and MHC class II molecules\n",
      "            is regulated by HLA-DO. Primary dendritic cells (DCs) also to\n",
      "            express HLA-DO. Lysosomal microenvironment has been implicated in\n",
      "            the regulation of antigen loading into MHC II molecules, increased\n",
      "            acidification produces increased proteolysis and efficient peptide\n",
      "            loading.\n",
      "            [SUBUNIT] Heterodimer of an alpha and a beta subunit; also referred\n",
      "            as MHC class II molecule. In the endoplasmic reticulum (ER) it\n",
      "            forms a heterononamer; 3 MHC class II molecules bind to a CD74\n",
      "            homotrimer (also known as invariant chain or HLA class II\n",
      "            histocompatibility antigen gamma chain). In the endosomal/lysosomal\n",
      "            system; CD74 undergoes sequential degradation by various proteases;\n",
      "            leaving a small fragment termed CLIP on each MHC class II molecule.\n",
      "            MHC class II molecule interacts with HLA_DM, and HLA_DO in B-cells,\n",
      "            in order to release CLIP and facilitate the binding of antigenic\n",
      "            peptides. {ECO:0000269|PubMed:11376336,\n",
      "            ECO:0000269|PubMed:14769912, ECO:0000269|PubMed:15020763,\n",
      "            ECO:0000269|PubMed:17629515}.\n",
      "            [INTERACTION] P01909; P01920: HLA-DQB1; NbExp=8; IntAct=EBI-713389,\n",
      "            EBI-1038012.\n",
      "            [SUBCELLULAR LOCATION] Cell membrane; Single-pass type I membrane\n",
      "            protein. Endoplasmic reticulum membrane; Single-pass type I\n",
      "            membrane protein. Golgi apparatus, trans-Golgi network membrane;\n",
      "            Single-pass type I membrane protein. Endosome membrane; Single-pass\n",
      "            type I membrane protein. Lysosome membrane; Single-pass type I\n",
      "            membrane protein. Note=The MHC class II complex transits through a\n",
      "            number of intracellular compartments in the endocytic pathway until\n",
      "            it reaches the cell membrane for antigen presentation.\n",
      "            [POLYMORPHISM] The following alleles of DQA1 are known: DQA1*01:01,\n",
      "            DQA1*01:02, DQA1*01:03, DQA1*01:04, DQA1*01:05, DQA1*01:06,\n",
      "            DQA1*01:07, DQA1*02:01, DQA1*03:01, DQA1*03:02, DQA1*03:03,\n",
      "            DQA1*04:01, DQA1*04:02, DQA1*04:03, DQA1*04:04, DQA1*05:01,\n",
      "            DQA1*05:02, DQA1*05:03, DQA1*05:04, DQA1*05:05, DQA1*05:06,\n",
      "            DQA1*05:07, DQA1*05:08, DQA1*05:09, DQA1*06:01, DQA1*06:02. The\n",
      "            sequence shown is that of DQA1*05:01.\n",
      "            [POLYMORPHISM] DQ2 (heterodimer of DQA1*05:01/DQB1*02:01) is\n",
      "            associated with more than 90% of celiac disease patients. A\n",
      "            minority displays DQ8 (heterodimer of DQA1*03/DQB1*03:02). DQ0602\n",
      "            (heterodimer of DQA1*01:02/DQB1*06:02) confers dominant protection\n",
      "            against type 1 diabetes (T1D) and strong susceptibility to\n",
      "            narcolepsy.\n",
      "            [SIMILARITY] Belongs to the MHC class II family. {ECO:0000305}.\n",
      "            [SEQUENCE CAUTION] Sequence=AAD56720.1; Type=Erroneous gene model\n",
      "            prediction; Evidence={ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..254\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "     gene            1..254\n",
      "                     /gene=\"HLA-DQA1\"\n",
      "     Protein         1..254\n",
      "                     /product=\"HLA class II histocompatibility antigen, DQ\n",
      "                     alpha 1 chain\"\n",
      "                     /note=\"DC-1 alpha chain; DC-alpha; HLA-DCA; MHC class II\n",
      "                     DQA1\"\n",
      "                     /UniProtKB_evidence=\"Evidence at protein level\"\n",
      "     Region          1..23\n",
      "                     /region_name=\"Signal\"\n",
      "                     /note=\"/evidence=ECO:0000269|PubMed:6576979.\"\n",
      "     Region          8\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"M -> L (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*04:01 and allele DQA1*06:01; dbSNP:rs1047989).\n",
      "                     /id=VAR_033399.\"\n",
      "     Region          11\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"A -> S (in Ref. 4; AAA59760).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          11\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"A -> T (in allele DQA1*05:05, allele DQA1*05:08 and\n",
      "                     allele DQA1*05:09; dbSNP:rs1047992). /id=VAR_033400.\"\n",
      "     Region          17\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> M (in allele DQA1*01:04 and allele DQA1*01:05;\n",
      "                     dbSNP:rs12722039). /id=VAR_050380.\"\n",
      "     Region          18\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"M -> T (in allele DQA1*03:03; dbSNP:rs11545686).\n",
      "                     /id=VAR_050381.\"\n",
      "     Region          23..27\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"Missing (in Ref. 16; AAA59754).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          24..254\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /note=\"HLA class II histocompatibility antigen, DQ alpha 1\n",
      "                     chain. /id=PRO_0000018970.\"\n",
      "     Region          24..216\n",
      "                     /region_name=\"Topological domain\"\n",
      "                     /note=\"Extracellular. /evidence=ECO:0000255.\"\n",
      "     Region          24..119\n",
      "                     /region_name=\"Region of interest in the sequence\"\n",
      "                     /note=\"Alpha-1.\"\n",
      "     Region          24\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"E -> K (in allele DQA1*05:09; dbSNP:rs41545012).\n",
      "                     /id=VAR_060493.\"\n",
      "     Region          25\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"D -> G (in allele DQA1*01:04 and allele DQA1*01:05;\n",
      "                     dbSNP:rs12722042). /id=VAR_050382.\"\n",
      "     Region          29..108\n",
      "                     /region_name=\"MHC_II_alpha\"\n",
      "                     /note=\"Class II histocompatibility antigen, alpha domain;\n",
      "                     pfam00993\"\n",
      "                     /db_xref=\"CDD:425978\"\n",
      "     Region          29..40\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          34\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Y -> C (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1129740).\n",
      "                     /id=VAR_060494.\"\n",
      "     Region          41..44\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          41\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> F (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1071630).\n",
      "                     /id=VAR_033401.\"\n",
      "     Region          44\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"P -> L (in allele DQA1*05:04; dbSNP:rs41549715).\n",
      "                     /id=VAR_060495.\"\n",
      "     Region          45..52\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          48\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Y -> F (in allele DQA1*02:01, allele DQA1*01:03,\n",
      "                     allele DQA1*06:01 and allele DQA1*06:02;\n",
      "                     dbSNP:rs12722051). /id=VAR_033402.\"\n",
      "     Region          49\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"T -> S (in allele DQA1*03:01, allele DQA1*03:02 and\n",
      "                     allele DQA1*03:03; dbSNP:rs1048023). /id=VAR_033403.\"\n",
      "     Region          55..61\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          57\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Q -> E (in allele DQA1*01:01, allele DQA1*01:04,\n",
      "                     allele DQA1*01:05, allele DQA1*01:07, allele DQA1*02:01,\n",
      "                     allele DQA1*03:01, allele DQA1*03:02 and allele\n",
      "                     DQA1*03:03; dbSNP:rs10093). /id=VAR_014604.\"\n",
      "     Region          62..65\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          63\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"G -> E (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\n",
      "                     allele DQA1*03:01, allele DQA1*03:02 and allele\n",
      "                     DQA1*03:03; dbSNP:rs1142323). /id=VAR_060496.\"\n",
      "     Region          64\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> K (in allele DQA1*01:03; dbSNP:rs36219699).\n",
      "                     /id=VAR_050383.\"\n",
      "     Region          66..71\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          67\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"T -> A (in allele DQA1*01:06; dbSNP:rs41543221).\n",
      "                     /id=VAR_060497.\"\n",
      "     Region          68\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> A (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142324).\n",
      "                     /id=VAR_060498.\"\n",
      "     Region          70\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"C -> K (in allele DQA1*02:01; requires 2 nucleotide\n",
      "                     substitutions). /id=VAR_060499.\"\n",
      "     Region          70\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"C -> Q (in allele DQA1*03:01, allele DQA1*03:02 and\n",
      "                     allele DQA1*03:03; requires 2 nucleotide substitutions).\n",
      "                     /id=VAR_060500.\"\n",
      "     Region          70\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"C -> R (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142326).\n",
      "                     /id=VAR_060501.\"\n",
      "     Region          70\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"C -> Y (in dbSNP:rs3207983). /id=VAR_033404.\"\n",
      "     Region          71\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> W (in allele DQA1*01:01, allele\n",
      "                     DQA1*01:02,allele DQA1*01:03, allele DQA1*01:04, allele\n",
      "                     DQA1*01:05, allele DQA1*01:06 and allele DQA1*01:07;\n",
      "                     dbSNP:rs1142328). /id=VAR_060502.\"\n",
      "     Region          72..76\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          73\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> D (in dbSNP:rs760671632). /id=VAR_033406.\"\n",
      "     Region          73\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> E (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs3208105).\n",
      "                     /id=VAR_060503.\"\n",
      "     Region          73\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> L (in allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02 and allele DQA1*03:03;\n",
      "                     dbSNP:rs12722061). /id=VAR_033405.\"\n",
      "     Region          74\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> F (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\n",
      "                     allele DQA1*03:01, allele DQA1*03:02 and allele\n",
      "                     DQA1*03:03; dbSNP:rs9272698). /id=VAR_060504.\"\n",
      "     Region          75\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> H (in allele DQA1*02:01; dbSNP:rs28383449).\n",
      "                     /id=VAR_060505.\"\n",
      "     Region          75\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> S (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs9272699).\n",
      "                     /id=VAR_060506.\"\n",
      "     Region          76\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Q -> K (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1048052).\n",
      "                     /id=VAR_060507.\"\n",
      "     Region          76\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Q -> R (in allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02 and allele DQA1*03:03;\n",
      "                     dbSNP:rs12722069). /id=VAR_060508.\"\n",
      "     Region          77\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"F -> L (in allele DQA1*02:01; dbSNP:rs3188043).\n",
      "                     /id=VAR_060509.\"\n",
      "     Region          78\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> GG (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele\n",
      "                     DQA1*01:05,allele DQA1*01:06 and allele DQA1*01:07;\n",
      "                     dbSNP:rs4193). /id=VAR_060510.\"\n",
      "     Region          78\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> RR (in allele DQA1*03:01, allele DQA1*03:02\n",
      "                     and allele DQA1*03:03). /id=VAR_060511.\"\n",
      "     Region          81..101\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          81\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"P -> R (in allele DQA1*05:02; dbSNP:rs41541412).\n",
      "                     /id=VAR_060512.\"\n",
      "     Region          83\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"F -> G (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; requires 2\n",
      "                     nucleotide substitutions). /id=VAR_060513.\"\n",
      "     Region          86\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"T -> R (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142333).\n",
      "                     /id=VAR_033408.\"\n",
      "     Region          88\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"I -> M (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142334).\n",
      "                     /id=VAR_033409.\"\n",
      "     Region          91\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> A (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; requires 2\n",
      "                     nucleotide substitutions). /id=VAR_060514.\"\n",
      "     Region          91\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"L -> H (in Ref. 4; AAA59760).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          91\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> T (in allele DQA1*04:01, allele DQA1*04:02,\n",
      "                     allele DQA1*04:04, allele DQA1*06:01 and allele\n",
      "                     DQA1*06:02; requires 2 nucleotide substitutions).\n",
      "                     /id=VAR_060515.\"\n",
      "     Region          97\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> I (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\n",
      "                     allele DQA1*03:01, allele DQA1*03:02, allele DQA1*03:03,\n",
      "                     allele DQA1*04:01, allele DQA1*04:02, allele DQA1*04:04,\n",
      "                     allele DQA1*06:01 and allele DQA1*06:02; dbSNP:rs9279910).\n",
      "                     /id=VAR_060516.\"\n",
      "     Region          98\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> M (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1064944).\n",
      "                     /id=VAR_060517.\"\n",
      "     Region          98\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> V (in allele DQA1*03:01, allele DQA1*03:02 and\n",
      "                     allele DQA1*03:03; dbSNP:rs1064944). /id=VAR_060518.\"\n",
      "     Region          101\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> C (in allele DQA1*01:07; dbSNP:rs41542116).\n",
      "                     /id=VAR_060519.\"\n",
      "     Region          102..104\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:1JK8.\"\n",
      "     Region          102\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> Y (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1129808).\n",
      "                     /id=VAR_050384.\"\n",
      "     Site            103\n",
      "                     /site_type=\"glycosylation\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255.\"\n",
      "     Region          107\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"A -> P (in Ref. 16; AAA59754).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          110..204\n",
      "                     /region_name=\"IgC1_MHC_II_alpha_HLA-DQ\"\n",
      "                     /note=\"Class II major histocompatibility complex (MHC)\n",
      "                     alpha chain immunoglobulin domain of histocompatibility\n",
      "                     antigen (HLA) DQ and related proteins; member of the\n",
      "                     C1-set of Ig superfamily (IgSF) domains; cd21008\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          111..119\n",
      "                     /region_name=\"Ig strand A\"\n",
      "                     /note=\"Ig strand A [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          112..204\n",
      "                     /region_name=\"Domain\"\n",
      "                     /note=\"Ig-like C1-type.\"\n",
      "     Region          113..120\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Site            order(115,117..119,121,173,175)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"heterodimer interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          120..203\n",
      "                     /region_name=\"Region of interest in the sequence\"\n",
      "                     /note=\"Alpha-2.\"\n",
      "     Region          124..126\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:4OZF.\"\n",
      "     Region          124\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> V (in allele DQA1*05:06; dbSNP:rs41555012).\n",
      "                     /id=VAR_060520.\"\n",
      "     Region          127..136\n",
      "                     /region_name=\"Ig strand B\"\n",
      "                     /note=\"Ig strand B [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          128..137\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          129\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"I -> T (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\n",
      "                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\n",
      "                     and allele DQA1*06:02; dbSNP:rs707952). /id=VAR_050385.\"\n",
      "     Bond            bond(132,188)\n",
      "                     /bond_type=\"disulfide\"\n",
      "                     /note=\"/evidence=ECO:0000255|PROSITE-ProRule:PRU00114,\n",
      "                     ECO:0000269|PubMed:11376336, ECO:0000269|PubMed:14769912,\n",
      "                     ECO:0000269|PubMed:17629515.\"\n",
      "     Site            order(139..141,160..163,169..170,175)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"MHC binding domain interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          142..148\n",
      "                     /region_name=\"Ig strand C\"\n",
      "                     /note=\"Ig strand C [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          143..148\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Site            143\n",
      "                     /site_type=\"glycosylation\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255.\"\n",
      "     Region          150..153\n",
      "                     /region_name=\"Ig strand C'\"\n",
      "                     /note=\"Ig strand C' [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          151..153\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          151\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"H -> Q (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:04, allele DQA1*01:05 and allele\n",
      "                     DQA1*01:07; dbSNP:rs707950). /id=VAR_050386.\"\n",
      "     Region          152\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> A (in allele DQA1*01:03; dbSNP:rs41547417).\n",
      "                     /id=VAR_060521.\"\n",
      "     Region          156\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"G -> D (in Ref. 3; CAA25141).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          157..165\n",
      "                     /region_name=\"Ig strand D\"\n",
      "                     /note=\"Ig strand D [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          157..159\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          160\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"T -> I (in allele DQA1*04:02; dbSNP:rs41545514).\n",
      "                     /id=VAR_060522.\"\n",
      "     Region          161\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> I (in allele DQA1*05:08; dbSNP:rs41544114).\n",
      "                     /id=VAR_060524.\"\n",
      "     Region          161\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> R (in allele DQA1*06:02; dbSNP:rs41552014).\n",
      "                     /id=VAR_060523.\"\n",
      "     Region          168..178\n",
      "                     /region_name=\"Ig strand E\"\n",
      "                     /note=\"Ig strand E [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          170..178\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          175\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"Y -> H (in allele DQA1*04:04; dbSNP:rs41550317).\n",
      "                     /id=VAR_060525.\"\n",
      "     Region          178\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"L -> F (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\n",
      "                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\n",
      "                     and allele DQA1*06:02; dbSNP:rs707949). /id=VAR_060526.\"\n",
      "     Region          182\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"A -> D (in allele DQA1*03:02 and allele DQA1*03:03;\n",
      "                     dbSNP:rs7990). /id=VAR_060527.\"\n",
      "     Region          182\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"A -> S (in allele DQA1*05:03, allele DQA1*05:06 and\n",
      "                     allele DQA1*05:07; dbSNP:rs41561312). /id=VAR_060528.\"\n",
      "     Region          183\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"E -> D (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\n",
      "                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\n",
      "                     and allele DQA1*06:02; dbSNP:rs707963). /id=VAR_060529.\"\n",
      "     Region          185..192\n",
      "                     /region_name=\"Ig strand F\"\n",
      "                     /note=\"Ig strand F [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          185\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"S -> I (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\n",
      "                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\n",
      "                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\n",
      "                     and allele DQA1*06:02; dbSNP:rs707962). /id=VAR_060530.\"\n",
      "     Region          186..191\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          195..197\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          197\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"K -> E (in allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\n",
      "                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\n",
      "                     and allele DQA1*06:02; dbSNP:rs2308891). /id=VAR_060531.\"\n",
      "     Region          197\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"K -> Q (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05\n",
      "                     and allele DQA1*01:07; dbSNP:rs2308891). /id=VAR_060532.\"\n",
      "     Region          198..203\n",
      "                     /region_name=\"Ig strand G\"\n",
      "                     /note=\"Ig strand G [structural motif]\"\n",
      "                     /db_xref=\"CDD:409599\"\n",
      "     Region          199..203\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\n",
      "     Region          204..216\n",
      "                     /region_name=\"Region of interest in the sequence\"\n",
      "                     /note=\"Connecting peptide.\"\n",
      "     Region          209\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"A -> T (in allele DQA1*03:01, allele DQA1*03:02 and\n",
      "                     allele DQA1*03:03; dbSNP:rs9272785). /id=VAR_050387.\"\n",
      "     Region          213\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /note=\"E -> D (in Ref. 16; AAA59754).\n",
      "                     /evidence=ECO:0000305.\"\n",
      "     Region          217..239\n",
      "                     /region_name=\"Transmembrane region\"\n",
      "                     /note=\"Helical. /evidence=ECO:0000255.\"\n",
      "     Region          221\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"A -> T (in allele DQA1*01:04; dbSNP:rs35087390).\n",
      "                     /id=VAR_050388.\"\n",
      "     Region          229\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"V -> M (in allele DQA1*01:02; dbSNP:rs9260).\n",
      "                     /id=VAR_033411.\"\n",
      "     Region          230\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"G -> C (in allele DQA1*05:07; dbSNP:rs41545416).\n",
      "                     /id=VAR_060533.\"\n",
      "     Region          237\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"F -> L (in allele DQA1*02:01, allele DQA1*03:01,\n",
      "                     allele DQA1*03:02 and allele DQA1*03:03; dbSNP:rs1048430).\n",
      "                     /id=VAR_033412.\"\n",
      "     Region          240..254\n",
      "                     /region_name=\"Topological domain\"\n",
      "                     /note=\"Cytoplasmic.\"\n",
      "     Region          240\n",
      "                     /region_name=\"Variant\"\n",
      "                     /note=\"R -> Q (in allele DQA1*01:01, allele DQA1*01:02,\n",
      "                     allele DQA1*01:03, allele DQA1*01:04 and allele\n",
      "                     DQA1*01:05; dbSNP:rs1048439). /id=VAR_033413.\"\n",
      "ORIGIN      \n",
      "        1 milnkalmlg alalttvmsp cggedivadh vasygvnlyq sygpsgqyth efdgdeqfyv\n",
      "       61 dlgrketvwc lpvlrqfrfd pqfaltniav lkhnlnslik rsnstaatne vpevtvfsks\n",
      "      121 pvtlgqpnil iclvdnifpp vvnitwlsng hsvtegvset sflsksdhsf fkisyltllp\n",
      "      181 saeesydckv ehwgldkpll khwepeipap mseltetvvc alglsvglvg ivvgtvfiir\n",
      "      241 glrsvgasrh qgpl\n",
      "//\n",
      "\n",
      "\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'LOCUS       DQA1_HUMAN               254 aa            linear   PRI 08-NOV-2023\\nDEFINITION  RecName: Full=HLA class II histocompatibility antigen, DQ alpha 1\\n            chain; AltName: Full=DC-1 alpha chain; AltName: Full=DC-alpha;\\n            AltName: Full=HLA-DCA; AltName: Full=MHC class II DQA1; Flags:\\n            Precursor.\\nACCESSION   P01909\\nVERSION     P01909.1\\nDBSOURCE    UniProtKB: locus DQA1_HUMAN, accession P01909;\\n            class: standard.\\n            extra\\n            accessions:O19630,O19706,P01907,P01908,P04225,P04226,P05536,P79553,\\n            Q06751,Q29876,Q29994,Q2Q6Y6,Q2Q6Y7,Q2Q6Y8,Q2WCM3,Q30064,Q30067,\\n            Q30068,Q30070,Q30071,Q30072,Q30073,Q30086,Q30101,Q5Y7D5,Q5Y7F5,\\n            Q6ICU6,Q6PR46,Q6QDB1,Q860W2,Q860W4,Q9BD37,Q9TPM3,Q9UM31\\n            created: Jul 21, 1986.\\n            sequence updated: Jul 21, 1986.\\n            annotation updated: Nov 8, 2023.\\n            xrefs: X00033.1, CAA24917.1, X00370.1, X00452.1, CAA25141.1,\\n            M16995.1, AAA59760.1, U92032.1, AAB91990.1, AY663395.1, AAU87978.1,\\n            AY663398.1, AAU87987.1, AY663400.1, AAU87992.1, AY663406.1,\\n            AAU88007.1, AY663411.1, AAU88022.1, AY663413.1, AAU88028.1,\\n            DQ178403.1, ABA86855.1, DQ178400.1, DQ178401.1, DQ178402.1,\\n            DQ178407.1, ABA86856.1, DQ178404.1, DQ178405.1, DQ178406.1,\\n            DQ178411.1, ABA86857.1, DQ178408.1, DQ178409.1, DQ178410.1,\\n            DQ178415.1, ABA86858.1, DQ178412.1, DQ178413.1, DQ178414.1,\\n            M29616.1, AAA59759.1, M29613.1, AF322870.1, AAK11577.1, AF322867.1,\\n            AF322868.1, AF322869.1, AF395700.1, AAM69677.1, AF395697.1,\\n            AF395698.1, AF395699.1, CR450297.1, CAG29293.1, AM042559.1,\\n            CAJ14960.1, AM042560.1, CAJ14961.1, AK313975.1, BAG36689.1,\\n            BX248406.4, AL662789.11, Z84489.1, CAB06491.1, BC008585.1,\\n            AAH08585.1, BC157865.1, AAI57866.1, L46875.1, L46876.1, L46877.1,\\n            L46878.1, L46880.1, L46881.1, M11124.1, AAA59754.1, M20431.1,\\n            AAA59758.1, L34082.1, AAC41950.1, L34085.1, AAC41953.1, L34086.1,\\n            AAC41954.1, L34089.1, AAC41957.1, L34090.1, AAC41958.1, L34092.1,\\n            AAC41960.1, L34093.1, AAC41961.1, L34094.1, AAC41962.1, L42625.1,\\n            AAA85334.1, AY197775.1, AAO45622.1, AY547314.1, AAS49496.1,\\n            AY206406.1, AAO47362.1, AY585236.1, AAT09985.1, M34997.1,\\n            AAA35772.1, M34999.1, AAA74633.1, U85035.1, AAB41891.1, AF109734.1,\\n            AAD56720.1, U03675.1, AAB60341.1, M20506.1, AAA59774.1, J00199.1,\\n            M17846.1, AAA59707.1, HLHUD1, HLHUDQ, HLHUD7, HLHU3C, HLHUDC,\\n            B27628, D34512, NP_002113.2, 1JK8_A, 1S9V_A, 1S9V_D, 1UVQ_A,\\n            2NNA_A, 4GG6_A, 4GG6_C, 4OZF_A, 4OZG_A, 4OZG_C, 4OZH_A, 4OZH_C,\\n            4OZI_A, 4OZI_C, 5KSA_A, 5KSB_A, 5KSB_C, 5KSU_A, 5KSU_D, 5KSV_A,\\n            6MFF_A, 6MFG_A, 6MFG_C, 6U3M_A, 6U3M_C, 6U3N_A, 6XP6_A, 6XP6_D,\\n            7SG0_A, 7SG1_A, 7SG1_F, 7SG2_A, 7SG2_F\\n            xrefs (non-sequence databases): PDBsum:1JK8, PDBsum:1S9V,\\n            PDBsum:1UVQ, PDBsum:2NNA, PDBsum:4GG6, PDBsum:4OZF, PDBsum:4OZG,\\n            PDBsum:4OZH, PDBsum:4OZI, PDBsum:5KSA, PDBsum:5KSB, PDBsum:5KSU,\\n            PDBsum:5KSV, PDBsum:6MFF, PDBsum:6MFG, PDBsum:6U3M, PDBsum:6U3N,\\n            PDBsum:6XP6, PDBsum:7SG0, PDBsum:7SG1, PDBsum:7SG2,\\n            AlphaFoldDB:P01909, SMR:P01909, BioGRID:109362, BioGRID:1529240,\\n            IntAct:P01909, ChEMBL:CHEMBL4105884, TCDB:9.A.75.1.1,\\n            GlyConnect:1986, GlyCosmos:P01909, GlyGen:P01909, iPTMnet:P01909,\\n            BioMuta:HLA-DQA1, DMDM:122188, jPOST:P01909, MassIVE:P01909,\\n            PeptideAtlas:P01909, ProteomicsDB:51510, ABCD:P01909,\\n            Antibodypedia:2728, CPTC:P01909, DNASU:3117,\\n            Ensembl:ENST00000343139.11, Ensembl:ENSP00000339398.5,\\n            Ensembl:ENSG00000196735.13, Ensembl:ENST00000374949.2,\\n            Ensembl:ENSP00000364087.2, Ensembl:ENST00000383251.6,\\n            Ensembl:ENSP00000372738.2, Ensembl:ENSG00000206305.12,\\n            Ensembl:ENST00000395363.5, Ensembl:ENSP00000378767.1,\\n            Ensembl:ENST00000399675.5, Ensembl:ENSP00000382583.1,\\n            Ensembl:ENST00000399678.5, Ensembl:ENSP00000382586.1,\\n            Ensembl:ENST00000418023.5, Ensembl:ENSP00000387892.1,\\n            Ensembl:ENSG00000232062.9, Ensembl:ENST00000444296.6,\\n            Ensembl:ENSP00000413237.2, GeneID:3117, KEGG:hsa:3117,\\n            MANE-Select:ENST00000343139.11, UCSC:uc003obr.4, AGR:HGNC:4942,\\n            CTD:3117, DisGeNET:3117, GeneCards:HLA-DQA1, GeneReviews:HLA-DQA1,\\n            HGNC:4942, HPA:ENSG00000196735, MalaCards:HLA-DQA1, MIM:146880,\\n            neXtProt:NX_P01909, NIAGADS:ENSG00000196735, Orphanet:391490,\\n            Orphanet:930, Orphanet:555, Orphanet:243377, PharmGKB:PA35066,\\n            VEuPathDB:HostDB:ENSG00000196735, HOGENOM:CLU_069380_0_0_1,\\n            InParanoid:P01909, OrthoDB:4118190at2759, PhylomeDB:P01909,\\n            TreeFam:TF333797, PathwayCommons:P01909, Reactome:R-HSA-202424,\\n            Reactome:R-HSA-202427, Reactome:R-HSA-202430,\\n            Reactome:R-HSA-202433, Reactome:R-HSA-2132295,\\n            Reactome:R-HSA-389948, Reactome:R-HSA-877300, SignaLink:P01909,\\n            SIGNOR:P01909, BioGRID-ORCS:3117, ChiTaRS:HLA-DQA1,\\n            EvolutionaryTrace:P01909, GenomeRNAi:3117, Pharos:P01909,\\n            PRO:PR:P01909, Proteomes:UP000005640, RNAct:P01909,\\n            Bgee:ENSG00000196735, ExpressionAtlas:P01909, Genevisible:P01909,\\n            GO:0030669, GO:0030666, GO:0012507, GO:0000139, GO:0031902,\\n            GO:0098553, GO:0005765, GO:0016020, GO:0042613, GO:0005886,\\n            GO:0032588, GO:0030658, GO:0023026, GO:0032395, GO:0042605,\\n            GO:0002250, GO:0019886, GO:0006955, GO:0002503, GO:0050778,\\n            GO:0050870, CDD:cd21008, Gene3D:2.60.40.10, InterPro:IPR007110,\\n            InterPro:IPR036179, InterPro:IPR013783, InterPro:IPR003006,\\n            InterPro:IPR003597, InterPro:IPR011162, InterPro:IPR014745,\\n            InterPro:IPR001003, PANTHER:PTHR19944:SF59, PANTHER:PTHR19944,\\n            Pfam:PF07654, Pfam:PF00993, SMART:SM00407, SMART:SM00920,\\n            SUPFAM:SSF48726, SUPFAM:SSF54452, PROSITE:PS50835, PROSITE:PS00290\\nKEYWORDS    3D-structure; Adaptive immunity; Cell membrane; Direct protein\\n            sequencing; Disulfide bond; Endoplasmic reticulum; Endosome;\\n            Glycoprotein; Golgi apparatus; Immunity; Lysosome; Membrane; MHC\\n            II; Reference proteome; Signal; Transmembrane; Transmembrane helix.\\nSOURCE      Homo sapiens (human)\\n  ORGANISM  Homo sapiens\\n            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\\n            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\\n            Catarrhini; Hominidae; Homo.\\nREFERENCE   1  (residues 1 to 254)\\n  AUTHORS   Chang,H.C., Moriuchi,T. and Silver,J.\\n  TITLE     The heavy chain of human B-cell alloantigen HLA-DS has a variable\\n            N-terminal region and a constant immunoglobulin-like region\\n  JOURNAL   Nature 305 (5937), 813-815 (1983)\\n   PUBMED   6415485\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*02:01).\\nREFERENCE   2  (residues 1 to 254)\\n  AUTHORS   Schenning,L., Larhammar,D., Bill,P., Wiman,K., Jonsson,A.K.,\\n            Rask,L. and Peterson,P.A.\\n  TITLE     Both alpha and beta chains of HLA-DC class II histocompatibility\\n            antigens display extensive polymorphism in their amino-terminal\\n            domains\\n  JOURNAL   EMBO J 3 (2), 447-452 (1984)\\n   PUBMED   6585297\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*05:01).\\nREFERENCE   3  (residues 1 to 254)\\n  AUTHORS   Auffray,C., Lillie,J.W., Arnot,D., Grossberger,D., Kappes,D. and\\n            Strominger,J.L.\\n  TITLE     Isotypic and allotypic variation of human class II\\n            histocompatibility antigen alpha-chain genes\\n  JOURNAL   Nature 308 (5957), 327-333 (1984)\\n   PUBMED   6584734\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELES DQA1*01:02 AND ALLELE\\n            DQA1*03:01).\\nREFERENCE   4  (residues 1 to 254)\\n  AUTHORS   Schiffenbauer,J., Didier,D.K., Klearman,M., Rice,K., Shuman,S.,\\n            Tieber,V.L., Kittlesen,D.J. and Schwartz,B.D.\\n  TITLE     Complete sequence of the HLA DQ alpha and DQ beta cDNA from a\\n            DR5/DQw3 cell line\\n  JOURNAL   J Immunol 139 (1), 228-233 (1987)\\n   PUBMED   3584986\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] (ALLELE DQA1*05:01).\\n            Erratum:[J Immunol 1991 Mar 1;146(5):1715]\\nREFERENCE   5  (residues 1 to 254)\\n  AUTHORS   Ellis,M.C., Hetisimer,A.H., Ruddy,D.A., Hansen,S.L., Kronmal,G.S.,\\n            McClelland,E., Quintana,L., Drayna,D.T., Aldrich,M.S. and Mignot,E.\\n  TITLE     HLA class II haplotype and sequence analysis support a role for DQ\\n            in narcolepsy\\n  JOURNAL   Immunogenetics 46 (5), 410-417 (1997)\\n   PUBMED   9271631\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*05:01).\\nREFERENCE   6  (residues 1 to 254)\\n  AUTHORS   Raymond,C.K., Kas,A., Paddock,M., Qiu,R., Zhou,Y., Subramanian,S.,\\n            Chang,J., Palmieri,A., Haugen,E., Kaul,R. and Olson,M.V.\\n  TITLE     Ancient haplotypes of the HLA Class II region\\n  JOURNAL   Genome Res 15 (9), 1250-1257 (2005)\\n   PUBMED   16140993\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:01; DQA1*01:02\\n            AND DQA1*03:01).\\nREFERENCE   7  (residues 1 to 254)\\n  AUTHORS   Lee,K.W., Jung,Y.A. and Oh,D.H.\\n  TITLE     Four novel human leukocyte antigen-DQA1 alleles identified in the\\n            Korean population\\n  JOURNAL   Tissue Antigens 68 (2), 167-172 (2006)\\n   PUBMED   16866887\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:02; DQA1*05:06;\\n            DQA1*05:07 AND DQA1*05:08).\\nREFERENCE   8  (residues 1 to 254)\\n  AUTHORS   Jonsson,A.K., Hyldig-Nielsen,J.J., Servenius,B., Larhammar,D.,\\n            Andersson,G., Jorgensen,F., Peterson,P.A. and Rask,L.\\n  TITLE     Class II genes of the human major histocompatibility complex.\\n            Comparisons of the DQ and DX alpha and beta genes\\n  JOURNAL   J Biol Chem 262 (18), 8767-8777 (1987)\\n   PUBMED   3036828\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*03:01).\\nREFERENCE   9  (residues 1 to 254)\\n  AUTHORS   Ashdown,M.L., Leas,L., Gavrilidis,A., Wood,J.M. and Simons,M.J.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-JUN-2001) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELE DQA1*01:01).\\nREFERENCE   10 (residues 1 to 254)\\n  AUTHORS   Ebert,L., Schick,M., Neubert,P., Schatten,R., Henze,S. and Korn,B.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-MAY-2004) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELE DQA1*02:01).\\nREFERENCE   11 (residues 1 to 254)\\n  AUTHORS   Voorter,C.E. and van den Berg-Loonen,E.M.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-JUL-2005) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] (ALLELES DQA1*01:02 AND\\n            DQA1*05:09).\\nREFERENCE   12 (residues 1 to 254)\\n  AUTHORS   Ota,T., Suzuki,Y., Nishikawa,T., Otsuki,T., Sugiyama,T., Irie,R.,\\n            Wakamatsu,A., Hayashi,K., Sato,H., Nagai,K., Kimura,K., Makita,H.,\\n            Sekine,M., Obayashi,M., Nishi,T., Shibahara,T., Tanaka,T.,\\n            Ishii,S., Yamamoto,J., Saito,K., Kawai,Y., Isono,Y., Nakamura,Y.,\\n            Nagahari,K., Murakami,K., Yasuda,T., Iwayanagi,T., Wagatsuma,M.,\\n            Shiratori,A., Sudo,H., Hosoiri,T., Kaku,Y., Kodaira,H., Kondo,H.,\\n            Sugawara,M., Takahashi,M., Kanda,K., Yokoi,T., Furuya,T.,\\n            Kikkawa,E., Omura,Y., Abe,K., Kamihara,K., Katsuta,N., Sato,K.,\\n            Tanikawa,M., Yamazaki,M., Ninomiya,K., Ishibashi,T., Yamashita,H.,\\n            Murakawa,K., Fujimori,K., Tanai,H., Kimata,M., Watanabe,M.,\\n            Hiraoka,S., Chiba,Y., Ishida,S., Ono,Y., Takiguchi,S., Watanabe,S.,\\n            Yosida,M., Hotuta,T., Kusano,J., Kanehori,K., Takahashi-Fujii,A.,\\n            Hara,H., Tanase,T.O., Nomura,Y., Togiya,S., Komai,F., Hara,R.,\\n            Takeuchi,K., Arita,M., Imose,N., Musashino,K., Yuuki,H., Oshima,A.,\\n            Sasaki,N., Aotsuka,S., Yoshikawa,Y., Matsunawa,H., Ichihara,T.,\\n            Shiohata,N., Sano,S., Moriya,S., Momiyama,H., Satoh,N., Takami,S.,\\n            Terashima,Y., Suzuki,O., Nakagawa,S., Senoh,A., Mizoguchi,H.,\\n            Goto,Y., Shimizu,F., Wakebe,H., Hishigaki,H., Watanabe,T.,\\n            Sugiyama,A., Takemoto,M., Kawakami,B., Yamazaki,M., Watanabe,K.,\\n            Kumagai,A., Itakura,S., Fukuzumi,Y., Fujimori,Y., Komiyama,M.,\\n            Tashiro,H., Tanigami,A., Fujiwara,T., Ono,T., Yamada,K., Fujii,Y.,\\n            Ozaki,K., Hirao,M., Ohmori,Y., Kawabata,A., Hikiji,T., Kobatake,N.,\\n            Inagaki,H., Ikema,Y., Okamoto,S., Okitani,R., Kawakami,T.,\\n            Noguchi,S., Itoh,T., Shigeta,K., Senba,T., Matsumura,K.,\\n            Nakajima,Y., Mizuno,T., Morinaga,M., Sasaki,M., Togashi,T.,\\n            Oyama,M., Hata,H., Watanabe,M., Komatsu,T., Mizushima-Sugano,J.,\\n            Satoh,T., Shirai,Y., Takahashi,Y., Nakagawa,K., Okumura,K.,\\n            Nagase,T., Nomura,N., Kikuchi,H., Masuho,Y., Yamashita,R.,\\n            Nakai,K., Yada,T., Nakamura,Y., Ohara,O., Isogai,T. and Sugano,S.\\n  TITLE     Complete sequencing and characterization of 21,243 full-length\\n            human cDNAs\\n  JOURNAL   Nat Genet 36 (1), 40-45 (2004)\\n   PUBMED   14702039\\n  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELE DQA1*01:01).;\\n            TISSUE=Trachea\\nREFERENCE   13 (residues 1 to 254)\\n  AUTHORS   Mungall,A.J., Palmer,S.A., Sims,S.K., Edwards,C.A., Ashurst,J.L.,\\n            Wilming,L., Jones,M.C., Horton,R., Hunt,S.E., Scott,C.E.,\\n            Gilbert,J.G., Clamp,M.E., Bethel,G., Milne,S., Ainscough,R.,\\n            Almeida,J.P., Ambrose,K.D., Andrews,T.D., Ashwell,R.I.,\\n            Babbage,A.K., Bagguley,C.L., Bailey,J., Banerjee,R., Barker,D.J.,\\n            Barlow,K.F., Bates,K., Beare,D.M., Beasley,H., Beasley,O.,\\n            Bird,C.P., Blakey,S., Bray-Allen,S., Brook,J., Brown,A.J.,\\n            Brown,J.Y., Burford,D.C., Burrill,W., Burton,J., Carder,C.,\\n            Carter,N.P., Chapman,J.C., Clark,S.Y., Clark,G., Clee,C.M.,\\n            Clegg,S., Cobley,V., Collier,R.E., Collins,J.E., Colman,L.K.,\\n            Corby,N.R., Coville,G.J., Culley,K.M., Dhami,P., Davies,J.,\\n            Dunn,M., Earthrowl,M.E., Ellington,A.E., Evans,K.A., Faulkner,L.,\\n            Francis,M.D., Frankish,A., Frankland,J., French,L., Garner,P.,\\n            Garnett,J., Ghori,M.J., Gilby,L.M., Gillson,C.J., Glithero,R.J.,\\n            Grafham,D.V., Grant,M., Gribble,S., Griffiths,C., Griffiths,M.,\\n            Hall,R., Halls,K.S., Hammond,S., Harley,J.L., Hart,E.A.,\\n            Heath,P.D., Heathcott,R., Holmes,S.J., Howden,P.J., Howe,K.L.,\\n            Howell,G.R., Huckle,E., Humphray,S.J., Humphries,M.D., Hunt,A.R.,\\n            Johnson,C.M., Joy,A.A., Kay,M., Keenan,S.J., Kimberley,A.M.,\\n            King,A., Laird,G.K., Langford,C., Lawlor,S., Leongamornlert,D.A.,\\n            Leversha,M., Lloyd,C.R., Lloyd,D.M., Loveland,J.E., Lovell,J.,\\n            Martin,S., Mashreghi-Mohammadi,M., Maslen,G.L., Matthews,L.,\\n            McCann,O.T., McLaren,S.J., McLay,K., McMurray,A., Moore,M.J.,\\n            Mullikin,J.C., Niblett,D., Nickerson,T., Novik,K.L., Oliver,K.,\\n            Overton-Larty,E.K., Parker,A., Patel,R., Pearce,A.V., Peck,A.I.,\\n            Phillimore,B., Phillips,S., Plumb,R.W., Porter,K.M., Ramsey,Y.,\\n            Ranby,S.A., Rice,C.M., Ross,M.T., Searle,S.M., Sehra,H.K.,\\n            Sheridan,E., Skuce,C.D., Smith,S., Smith,M., Spraggon,L.,\\n            Squares,S.L., Steward,C.A., Sycamore,N., Tamlyn-Hall,G., Tester,J.,\\n            Theaker,A.J., Thomas,D.W., Thorpe,A., Tracey,A., Tromans,A.,\\n            Tubby,B., Wall,M., Wallis,J.M., West,A.P., White,S.S.,\\n            Whitehead,S.L., Whittaker,H., Wild,A., Willey,D.J., Wilmer,T.E.,\\n            Wood,J.M., Wray,P.W., Wyatt,J.C., Young,L., Younger,R.M.,\\n            Bentley,D.R., Coulson,A., Durbin,R., Hubbard,T., Sulston,J.E.,\\n            Dunham,I., Rogers,J. and Beck,S.\\n  TITLE     The DNA sequence and analysis of human chromosome 6\\n  JOURNAL   Nature 425 (6960), 805-811 (2003)\\n   PUBMED   14574404\\n  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA] (ALLELES DQA1*01:02;\\n            DQA1*03:01 AND DQA1*05:01).\\nREFERENCE   14 (residues 1 to 254)\\n  AUTHORS   Gerhard,D.S., Wagner,L., Feingold,E.A., Shenmen,C.M., Grouse,L.H.,\\n            Schuler,G., Klein,S.L., Old,S., Rasooly,R., Good,P., Guyer,M.,\\n            Peck,A.M., Derge,J.G., Lipman,D., Collins,F.S., Jang,W., Sherry,S.,\\n            Feolo,M., Misquitta,L., Lee,E., Rotmistrovsky,K., Greenhut,S.F.,\\n            Schaefer,C.F., Buetow,K., Bonner,T.I., Haussler,D., Kent,J.,\\n            Kiekhaus,M., Furey,T., Brent,M., Prange,C., Schreiber,K.,\\n            Shapiro,N., Bhat,N.K., Hopkins,R.F., Hsie,F., Driscoll,T.,\\n            Soares,M.B., Casavant,T.L., Scheetz,T.E., Brown-stein,M.J.,\\n            Usdin,T.B., Toshiyuki,S., Carninci,P., Piao,Y., Dudekula,D.B.,\\n            Ko,M.S., Kawakami,K., Suzuki,Y., Sugano,S., Gruber,C.E.,\\n            Smith,M.R., Simmons,B., Moore,T., Waterman,R., Johnson,S.L.,\\n            Ruan,Y., Wei,C.L., Mathavan,S., Gunaratne,P.H., Wu,J., Garcia,A.M.,\\n            Hulyk,S.W., Fuh,E., Yuan,Y., Sneed,A., Kowis,C., Hodgson,A.,\\n            Muzny,D.M., McPherson,J., Gibbs,R.A., Fahey,J., Helton,E.,\\n            Ketteman,M., Madan,A., Rodrigues,S., Sanchez,A., Whiting,M.,\\n            Madari,A., Young,A.C., Wetherby,K.D., Granite,S.J., Kwong,P.N.,\\n            Brinkley,C.P., Pearson,R.L., Bouffard,G.G., Blakesly,R.W.,\\n            Green,E.D., Dickson,M.C., Rodriguez,A.C., Grimwood,J., Schmutz,J.,\\n            Myers,R.M., Butterfield,Y.S., Griffith,M., Griffith,O.L.,\\n            Krzywinski,M.I., Liao,N., Morin,R., Palmquist,D., Petrescu,A.S.,\\n            Skalska,U., Smailus,D.E., Stott,J.M., Schnerch,A., Schein,J.E.,\\n            Jones,S.J., Holt,R.A., Baross,A., Marra,M.A., Clifton,S.,\\n            Makowski,K.A., Bosak,S. and Malek,J.\\n  CONSRTM   MGC Project Team\\n  TITLE     The status, quality, and expansion of the NIH full-length cDNA\\n            project: the Mammalian Gene Collection (MGC)\\n  JOURNAL   Genome Res 14 (10B), 2121-2127 (2004)\\n   PUBMED   15489334\\n  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ALLELES DQA1*03:01 AND\\n            DQA1*05:01).;\\n            TISSUE=Lymph\\n            Erratum:[Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to\\n            Morin, Ryan]]\\nREFERENCE   15 (residues 1 to 254)\\n  AUTHORS   Yasunaga,S., Kimura,A. and Sasazuki,T.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-JUL-1996) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 1-27 (ALLELES\\n            DQA1*01:01/DQA1*01:02/DQA1*01:03; DQA1*01:04; DQA1*01:05;\\n            DQA1*02:01/DQA1*03:01/DQA1*03:03; DQA1*04:01/DQA1*06:01 AND\\n            DQA1*05:01/DQA1*05:03/DQA1*05:05).;\\n            TISSUE=B-cell\\nREFERENCE   16 (residues 1 to 254)\\n  AUTHORS   Moriuchi,J., Moriuchi,T. and Silver,J.\\n  TITLE     Nucleotide sequence of an HLA-DQ alpha chain derived from a DRw9\\n            cell line: genetic and evolutionary implications\\n  JOURNAL   Proc Natl Acad Sci U S A 82 (10), 3420-3424 (1985)\\n   PUBMED   3879967\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 19-254 (ALLELE DQA1*03:02).\\nREFERENCE   17 (residues 1 to 254)\\n  AUTHORS   Lock,C.B., So,A.K., Welsh,K.I., Parkes,J.D. and Trowsdale,J.\\n  TITLE     MHC class II sequences of an HLA-DR2 narcoleptic\\n  JOURNAL   Immunogenetics 27 (6), 449-455 (1988)\\n   PUBMED   3259543\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 23-254 (ALLELE DQA1*01:02).\\nREFERENCE   18 (residues 1 to 254)\\n  AUTHORS   Yasunaga,S., Kimura,A., Hamaguchi,K., Ronningen,K.S. and\\n            Sasazuki,T.\\n  TITLE     Different contribution of HLA-DR and -DQ genes in susceptibility\\n            and resistance to insulin-dependent diabetes mellitus (IDDM)\\n  JOURNAL   Tissue Antigens 47 (1), 37-48 (1996)\\n   PUBMED   8929711\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-250 (ALLELE DQA1*03:02/DQA1*03:03)\\n            AND NUCLEOTIDE SEQUENCE [MRNA] OF 24-249 (ALLELES DQA1*01:01;\\n            DQA1*01:03; DQA1*01:04; DQA1*04:01; DQA1*05:03; DQA1*05:05 AND\\n            DQA1*06:01).\\nREFERENCE   19 (residues 1 to 254)\\n  AUTHORS   Yasunaga,S.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-JAN-1996) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-249 (ALLELE DQA1*01:05).\\nREFERENCE   20 (residues 1 to 254)\\n  AUTHORS   Goetz,H., Kratzin,H., Thinnes,F.P., Yang,C.-Y., Kruse,T., Pauly,E.,\\n            Koelbel,S., Egert,G., Wernet,P. and Hilschmann,N.\\n  TITLE     Primary structure of human class II histocompatibility antigens 3rd\\n            communication. Amino acid sequence comparison between DR and DC\\n            subclass antigens derived from a lymphoblastoid B cell line\\n            homozygous at the HLA loci (HLA-A3,3; B7,7; Dw2,2; DR2,2: MT1,1;\\n            Dc1,1: MB1,1)\\n  JOURNAL   Hoppe-Seyler\\'s Z. Physiol. Chem. 364, 749-755 (1983)\\n   PUBMED   6576979\\n  REMARK    PROTEIN SEQUENCE OF 24-89; 102-120 AND 164-212 (ALLELE DQA1*01:02).\\nREFERENCE   21 (residues 1 to 254)\\n  AUTHORS   Kao,H.T., Gregersen,P.K., Tang,J.C., Takahashi,T., Wang,C.Y. and\\n            Silver,J.\\n  TITLE     Molecular analysis of the HLA class II genes in two DRw6-related\\n            haplotypes, DRw13 DQw1 and DRw14 DQw3\\n  JOURNAL   J Immunol 142 (5), 1743-1747 (1989)\\n   PUBMED   2493052\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 24-209 (ALLELE DQA1*05:01).\\nREFERENCE   22 (residues 1 to 254)\\n  AUTHORS   Cordovado,S.K., Hancock,L.N., Simone,A.E., Hendrix,M. and\\n            Mueller,P.W.\\n  TITLE     High-resolution genotyping of HLA-DQA1 in the GoKinD study and\\n            identification of novel alleles HLA-DQA1*040102, HLA-DQA1*0402 and\\n            HLA-DQA1*0404\\n  JOURNAL   Tissue Antigens 65 (5), 448-458 (2005)\\n   PUBMED   15853899\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELES DQA1*04:02 AND\\n            DQA1*04:04).\\nREFERENCE   23 (residues 1 to 254)\\n  AUTHORS   Simone,A.E., Cordovado,S.K., Hendrix,M.M. and Mueller,P.W.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-DEC-2002) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELE DQA1*06:02).\\nREFERENCE   24 (residues 1 to 254)\\n  AUTHORS   Hancock,L.N., Cordovado,S.K. and Mueller,P.W.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-MAR-2004) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-203 (ALLELE DQA1*01:07).\\nREFERENCE   25 (residues 1 to 254)\\n  AUTHORS   Horn,G.T., Bugawan,T.L., Long,C.M., Manos,M.M. and Erlich,H.A.\\n  TITLE     Sequence analysis of HLA class II genes from insulin-dependent\\n            diabetic individuals\\n  JOURNAL   Hum Immunol 21 (4), 249-263 (1988)\\n   PUBMED   3372263\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-110 (ALLELE DQA1*01:02),\\n            AND NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-109 (ALLELE\\n            DQA1*05:01).\\nREFERENCE   26 (residues 1 to 254)\\n  AUTHORS   Trejaut,J.A., Greville,W.D. and Dunckley,H.\\n  TITLE     Direct Submission\\n  JOURNAL   Submitted (??-JAN-1997) to the EMBL/GenBank/DDBJ databases\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 29-109 (ALLELE DQA1*05:04).\\nREFERENCE   27 (residues 1 to 254)\\n  AUTHORS   Luo,M., Blanchard,J., Maclean,I. and Brunham,R.\\n  TITLE     Identification of a novel HLA-DQA1 allele (DQA1*0106) by\\n            sequence-based DQA1 typing\\n  JOURNAL   Tissue Antigens 53 (6), 595-596 (1999)\\n   PUBMED   10395113\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 31-109 (ALLELE DQA1*01:06).\\nREFERENCE   28 (residues 1 to 254)\\n  AUTHORS   Marsh,S.G. and Bodmer,J.G.\\n  TITLE     HLA class II nucleotide sequences, 1992\\n  JOURNAL   Tissue Antigens 40 (5), 229-243 (1992)\\n   PUBMED   1362295\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 35-74 (ALLELE DQA1*05:02).\\n            Correction to:[Tissue Antigens. 1992 Apr;39(4):161-73. PMID:\\n            1529427]\\nREFERENCE   29 (residues 1 to 254)\\n  AUTHORS   Liu,C.P., Bach,F.H. and Wu,S.K.\\n  TITLE     Molecular studies of a rare DR2/LD-5a/DQw3 HLA class II haplotype.\\n            Multiple genetic mechanisms in the generation of polymorphic HLA\\n            class II genes\\n  JOURNAL   J Immunol 140 (10), 3631-3639 (1988)\\n   PUBMED   3129499\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 36-254 (ALLELE DQA1*05:01).\\n            Erratum:[J Immunol 1990 Feb 15;144(4):1544]\\nREFERENCE   30 (residues 1 to 254)\\n  AUTHORS   Liu,C.P., Bach,F.H. and Wu,S.K.\\n  JOURNAL   J. Immunol. 144, 15441-15441 (1990)\\n  REMARK    ERRATUM OF PUBMED:3129499.\\nREFERENCE   31 (residues 1 to 254)\\n  AUTHORS   Auffray,C., Korman,A.J., Roux-Dosseto,M., Bono,R. and\\n            Strominger,J.L.\\n  TITLE     cDNA clone for the heavy chain of the human B cell alloantigen DC1:\\n            strong sequence homology to the HLA-DR heavy chain\\n  JOURNAL   Proc Natl Acad Sci U S A 79 (20), 6337-6341 (1982)\\n   PUBMED   6815651\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 40-254 (ALLELE DQA1*03:01).\\nREFERENCE   32 (residues 1 to 254)\\n  AUTHORS   Gyllensten,U.B. and Erlich,H.A.\\n  TITLE     Ancient roots for polymorphism at the HLA-DQ alpha locus in\\n            primates\\n  JOURNAL   Proc Natl Acad Sci U S A 86 (24), 9986-9990 (1989)\\n   PUBMED   2513578\\n  REMARK    NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 41-102 (ALLELE DQA1*05:01),\\n            AND NUCLEOTIDE SEQUENCE [GENOMIC DNA] OF 41-103 (ALLELE\\n            DQA1*01:02).\\nREFERENCE   33 (residues 1 to 254)\\n  AUTHORS   Turco,E., Care,A., Compagnone-Post,P., Robinson,C., Cascino,I. and\\n            Trucco,M.\\n  TITLE     Allelic forms of the alpha- and beta-chain genes encoding\\n            DQw1-positive heterodimers\\n  JOURNAL   Immunogenetics 26 (4-5), 282-290 (1987)\\n   PUBMED   2888727\\n  REMARK    NUCLEOTIDE SEQUENCE [MRNA] OF 119-254 (ALLELE DQA1*01:02).\\nREFERENCE   34 (residues 1 to 254)\\n  AUTHORS   Cresswell,P.\\n  TITLE     Invariant chain structure and MHC class II function\\n  JOURNAL   Cell 84 (4), 505-507 (1996)\\n   PUBMED   8598037\\n  REMARK    REVIEW.\\nREFERENCE   35 (residues 1 to 254)\\n  AUTHORS   Villadangos,J.A.\\n  TITLE     Presentation of antigens by MHC class II molecules: getting the\\n            most out of them\\n  JOURNAL   Mol Immunol 38 (5), 329-346 (2001)\\n   PUBMED   11684289\\n  REMARK    REVIEW.\\nREFERENCE   36 (residues 1 to 254)\\n  AUTHORS   Menendez-Benito,V. and Neefjes,J.\\n  TITLE     Autophagy in MHC class II presentation: sampling from within\\n  JOURNAL   Immunity 26 (1), 1-3 (2007)\\n   PUBMED   17241953\\n  REMARK    REVIEW.\\nREFERENCE   37 (residues 1 to 254)\\n  AUTHORS   Rocha,N. and Neefjes,J.\\n  TITLE     MHC class II molecules on the move for successful antigen\\n            presentation\\n  JOURNAL   EMBO J 27 (1), 1-5 (2008)\\n   PUBMED   18046453\\n  REMARK    REVIEW.\\nREFERENCE   38 (residues 1 to 254)\\n  AUTHORS   Berger,A.C. and Roche,P.A.\\n  TITLE     MHC class II transport at a glance\\n  JOURNAL   J Cell Sci 122 (Pt 1), 1-4 (2009)\\n   PUBMED   19092054\\n  REMARK    REVIEW.\\nREFERENCE   39 (residues 1 to 254)\\n  AUTHORS   Beswick,E.J. and Reyes,V.E.\\n  TITLE     CD74 in antigen presentation, inflammation, and cancers of the\\n            gastrointestinal tract\\n  JOURNAL   World J Gastroenterol 15 (23), 2855-2861 (2009)\\n   PUBMED   19533806\\n  REMARK    REVIEW.\\nREFERENCE   40 (residues 1 to 254)\\n  AUTHORS   Lee,K.H., Wucherpfennig,K.W. and Wiley,D.C.\\n  TITLE     Structure of a human insulin peptide-HLA-DQ8 complex and\\n            susceptibility to type 1 diabetes\\n  JOURNAL   Nat Immunol 2 (6), 501-507 (2001)\\n   PUBMED   11376336\\n  REMARK    X-RAY CRYSTALLOGRAPHY (2.4 ANGSTROMS) OF 27-207 OF\\n            HLA-DQA1/HLA-DQB1 HETERODIMER IN COMPLEX WITH INS PEPTIDE, SUBUNIT,\\n            GLYCOSYLATION AT ASN-103, AND DISULFIDE BOND.\\n            Erratum:[Nat Immunol 2001 Sep;2(9):889]\\nREFERENCE   41 (residues 1 to 254)\\n  AUTHORS   Siebold,C., Hansen,B.E., Wyer,J.R., Harlos,K., Esnouf,R.E.,\\n            Svejgaard,A., Bell,J.I., Strominger,J.L., Jones,E.Y. and Fugger,L.\\n  TITLE     Crystal structure of HLA-DQ0602 that protects against type 1\\n            diabetes and confers strong susceptibility to narcolepsy\\n  JOURNAL   Proc Natl Acad Sci U S A 101 (7), 1999-2004 (2004)\\n   PUBMED   14769912\\n  REMARK    X-RAY CRYSTALLOGRAPHY (1.8 ANGSTROMS) OF 24-219 OF\\n            HLA-DQA1/HLA-DQB1 HETERODIMER (HLA-DQ0602) IN COMPLEX WITH HCRT\\n            PEPTIDE, POLYMORPHISM, SUBUNIT, AND DISULFIDE BOND.\\nREFERENCE   42 (residues 1 to 254)\\n  AUTHORS   Kim,C.Y., Quarsten,H., Bergseng,E., Khosla,C. and Sollid,L.M.\\n  TITLE     Structural basis for HLA-DQ2-mediated presentation of gluten\\n            epitopes in celiac disease\\n  JOURNAL   Proc Natl Acad Sci U S A 101 (12), 4175-4179 (2004)\\n   PUBMED   15020763\\n  REMARK    X-RAY CRYSTALLOGRAPHY (2.22 ANGSTROMS) OF 24-216 OF\\n            HLA-DQA1/HLA-DQB1 HETERODIMER (DQ2) IN COMPLEX WITH TRITICUM\\n            AESTIVUM ALPHA/BETA-GLIADIN PEPTIDE, SUBUNIT, AND POLYMORPHISM.\\nREFERENCE   43 (residues 1 to 254)\\n  AUTHORS   Henderson,K.N., Tye-Din,J.A., Reid,H.H., Chen,Z., Borg,N.A.,\\n            Beissbarth,T., Tatham,A., Mannering,S.I., Purcell,A.W., Dudek,N.L.,\\n            van Heel,D.A., McCluskey,J., Rossjohn,J. and Anderson,R.P.\\n  TITLE     A structural and immunological basis for the role of human\\n            leukocyte antigen DQ8 in celiac disease\\n  JOURNAL   Immunity 27 (1), 23-34 (2007)\\n   PUBMED   17629515\\n  REMARK    X-RAY CRYSTALLOGRAPHY (2.1 ANGSTROMS) OF 24-206 OF\\n            HLA-DQA1/HLA-DQB1 HETERODIMER IN COMPLEX WITH TRITICUM AESTIVUM\\n            ALPHA/BETA-GLIADIN, SUBUNIT, AND DISULFIDE BOND.\\nCOMMENT     On or before Mar 30, 2010 this sequence version replaced P01908.2,\\n            P05536.1, gi:74721757, gi:74705624, P01907.2, P04225.1, P04226.1,\\n            gi:74706178, gi:74706836, gi:121945665, gi:121948343, gi:121945666,\\n            gi:121941983, gi:74706863, gi:74706866, gi:74706867, gi:74706869,\\n            gi:74706870, gi:74706871, gi:74706872, gi:74709986, gi:74727328,\\n            gi:74713787, gi:74727342, gi:74717483, gi:74720013, gi:70108.\\n            [FUNCTION] Binds peptides derived from antigens that access the\\n            endocytic route of antigen presenting cells (APC) and presents them\\n            on the cell surface for recognition by the CD4 T-cells. The peptide\\n            binding cleft accommodates peptides of 10-30 residues. The peptides\\n            presented by MHC class II molecules are generated mostly by\\n            degradation of proteins that access the endocytic route, where they\\n            are processed by lysosomal proteases and other hydrolases.\\n            Exogenous antigens that have been endocytosed by the APC are thus\\n            readily available for presentation via MHC II molecules, and for\\n            this reason this antigen presentation pathway is usually referred\\n            to as exogenous. As membrane proteins on their way to degradation\\n            in lysosomes as part of their normal turn-over are also contained\\n            in the endosomal/lysosomal compartments, exogenous antigens must\\n            compete with those derived from endogenous components. Autophagy is\\n            also a source of endogenous peptides, autophagosomes constitutively\\n            fuse with MHC class II loading compartments. In addition to APCs,\\n            other cells of the gastrointestinal tract, such as epithelial\\n            cells, express MHC class II molecules and CD74 and act as APCs,\\n            which is an unusual trait of the GI tract. To produce a MHC class\\n            II molecule that presents an antigen, three MHC class II molecules\\n            (heterodimers of an alpha and a beta chain) associate with a CD74\\n            trimer in the ER to form a heterononamer. Soon after the entry of\\n            this complex into the endosomal/lysosomal system where antigen\\n            processing occurs, CD74 undergoes a sequential degradation by\\n            various proteases, including CTSS and CTSL, leaving a small\\n            fragment termed CLIP (class-II-associated invariant chain peptide).\\n            The removal of CLIP is facilitated by HLA-DM via direct binding to\\n            the alpha-beta-CLIP complex so that CLIP is released. HLA-DM\\n            stabilizes MHC class II molecules until primary high affinity\\n            antigenic peptides are bound. The MHC II molecule bound to a\\n            peptide is then transported to the cell membrane surface. In\\n            B-cells, the interaction between HLA-DM and MHC class II molecules\\n            is regulated by HLA-DO. Primary dendritic cells (DCs) also to\\n            express HLA-DO. Lysosomal microenvironment has been implicated in\\n            the regulation of antigen loading into MHC II molecules, increased\\n            acidification produces increased proteolysis and efficient peptide\\n            loading.\\n            [SUBUNIT] Heterodimer of an alpha and a beta subunit; also referred\\n            as MHC class II molecule. In the endoplasmic reticulum (ER) it\\n            forms a heterononamer; 3 MHC class II molecules bind to a CD74\\n            homotrimer (also known as invariant chain or HLA class II\\n            histocompatibility antigen gamma chain). In the endosomal/lysosomal\\n            system; CD74 undergoes sequential degradation by various proteases;\\n            leaving a small fragment termed CLIP on each MHC class II molecule.\\n            MHC class II molecule interacts with HLA_DM, and HLA_DO in B-cells,\\n            in order to release CLIP and facilitate the binding of antigenic\\n            peptides. {ECO:0000269|PubMed:11376336,\\n            ECO:0000269|PubMed:14769912, ECO:0000269|PubMed:15020763,\\n            ECO:0000269|PubMed:17629515}.\\n            [INTERACTION] P01909; P01920: HLA-DQB1; NbExp=8; IntAct=EBI-713389,\\n            EBI-1038012.\\n            [SUBCELLULAR LOCATION] Cell membrane; Single-pass type I membrane\\n            protein. Endoplasmic reticulum membrane; Single-pass type I\\n            membrane protein. Golgi apparatus, trans-Golgi network membrane;\\n            Single-pass type I membrane protein. Endosome membrane; Single-pass\\n            type I membrane protein. Lysosome membrane; Single-pass type I\\n            membrane protein. Note=The MHC class II complex transits through a\\n            number of intracellular compartments in the endocytic pathway until\\n            it reaches the cell membrane for antigen presentation.\\n            [POLYMORPHISM] The following alleles of DQA1 are known: DQA1*01:01,\\n            DQA1*01:02, DQA1*01:03, DQA1*01:04, DQA1*01:05, DQA1*01:06,\\n            DQA1*01:07, DQA1*02:01, DQA1*03:01, DQA1*03:02, DQA1*03:03,\\n            DQA1*04:01, DQA1*04:02, DQA1*04:03, DQA1*04:04, DQA1*05:01,\\n            DQA1*05:02, DQA1*05:03, DQA1*05:04, DQA1*05:05, DQA1*05:06,\\n            DQA1*05:07, DQA1*05:08, DQA1*05:09, DQA1*06:01, DQA1*06:02. The\\n            sequence shown is that of DQA1*05:01.\\n            [POLYMORPHISM] DQ2 (heterodimer of DQA1*05:01/DQB1*02:01) is\\n            associated with more than 90% of celiac disease patients. A\\n            minority displays DQ8 (heterodimer of DQA1*03/DQB1*03:02). DQ0602\\n            (heterodimer of DQA1*01:02/DQB1*06:02) confers dominant protection\\n            against type 1 diabetes (T1D) and strong susceptibility to\\n            narcolepsy.\\n            [SIMILARITY] Belongs to the MHC class II family. {ECO:0000305}.\\n            [SEQUENCE CAUTION] Sequence=AAD56720.1; Type=Erroneous gene model\\n            prediction; Evidence={ECO:0000305}.\\nFEATURES             Location/Qualifiers\\n     source          1..254\\n                     /organism=\"Homo sapiens\"\\n                     /db_xref=\"taxon:9606\"\\n     gene            1..254\\n                     /gene=\"HLA-DQA1\"\\n     Protein         1..254\\n                     /product=\"HLA class II histocompatibility antigen, DQ\\n                     alpha 1 chain\"\\n                     /note=\"DC-1 alpha chain; DC-alpha; HLA-DCA; MHC class II\\n                     DQA1\"\\n                     /UniProtKB_evidence=\"Evidence at protein level\"\\n     Region          1..23\\n                     /region_name=\"Signal\"\\n                     /note=\"/evidence=ECO:0000269|PubMed:6576979.\"\\n     Region          8\\n                     /region_name=\"Variant\"\\n                     /note=\"M -> L (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*04:01 and allele DQA1*06:01; dbSNP:rs1047989).\\n                     /id=VAR_033399.\"\\n     Region          11\\n                     /region_name=\"Conflict\"\\n                     /note=\"A -> S (in Ref. 4; AAA59760).\\n                     /evidence=ECO:0000305.\"\\n     Region          11\\n                     /region_name=\"Variant\"\\n                     /note=\"A -> T (in allele DQA1*05:05, allele DQA1*05:08 and\\n                     allele DQA1*05:09; dbSNP:rs1047992). /id=VAR_033400.\"\\n     Region          17\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> M (in allele DQA1*01:04 and allele DQA1*01:05;\\n                     dbSNP:rs12722039). /id=VAR_050380.\"\\n     Region          18\\n                     /region_name=\"Variant\"\\n                     /note=\"M -> T (in allele DQA1*03:03; dbSNP:rs11545686).\\n                     /id=VAR_050381.\"\\n     Region          23..27\\n                     /region_name=\"Conflict\"\\n                     /note=\"Missing (in Ref. 16; AAA59754).\\n                     /evidence=ECO:0000305.\"\\n     Region          24..254\\n                     /region_name=\"Mature chain\"\\n                     /note=\"HLA class II histocompatibility antigen, DQ alpha 1\\n                     chain. /id=PRO_0000018970.\"\\n     Region          24..216\\n                     /region_name=\"Topological domain\"\\n                     /note=\"Extracellular. /evidence=ECO:0000255.\"\\n     Region          24..119\\n                     /region_name=\"Region of interest in the sequence\"\\n                     /note=\"Alpha-1.\"\\n     Region          24\\n                     /region_name=\"Variant\"\\n                     /note=\"E -> K (in allele DQA1*05:09; dbSNP:rs41545012).\\n                     /id=VAR_060493.\"\\n     Region          25\\n                     /region_name=\"Variant\"\\n                     /note=\"D -> G (in allele DQA1*01:04 and allele DQA1*01:05;\\n                     dbSNP:rs12722042). /id=VAR_050382.\"\\n     Region          29..108\\n                     /region_name=\"MHC_II_alpha\"\\n                     /note=\"Class II histocompatibility antigen, alpha domain;\\n                     pfam00993\"\\n                     /db_xref=\"CDD:425978\"\\n     Region          29..40\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          34\\n                     /region_name=\"Variant\"\\n                     /note=\"Y -> C (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1129740).\\n                     /id=VAR_060494.\"\\n     Region          41..44\\n                     /region_name=\"Hydrogen bonded turn\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          41\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> F (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1071630).\\n                     /id=VAR_033401.\"\\n     Region          44\\n                     /region_name=\"Variant\"\\n                     /note=\"P -> L (in allele DQA1*05:04; dbSNP:rs41549715).\\n                     /id=VAR_060495.\"\\n     Region          45..52\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          48\\n                     /region_name=\"Variant\"\\n                     /note=\"Y -> F (in allele DQA1*02:01, allele DQA1*01:03,\\n                     allele DQA1*06:01 and allele DQA1*06:02;\\n                     dbSNP:rs12722051). /id=VAR_033402.\"\\n     Region          49\\n                     /region_name=\"Variant\"\\n                     /note=\"T -> S (in allele DQA1*03:01, allele DQA1*03:02 and\\n                     allele DQA1*03:03; dbSNP:rs1048023). /id=VAR_033403.\"\\n     Region          55..61\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          57\\n                     /region_name=\"Variant\"\\n                     /note=\"Q -> E (in allele DQA1*01:01, allele DQA1*01:04,\\n                     allele DQA1*01:05, allele DQA1*01:07, allele DQA1*02:01,\\n                     allele DQA1*03:01, allele DQA1*03:02 and allele\\n                     DQA1*03:03; dbSNP:rs10093). /id=VAR_014604.\"\\n     Region          62..65\\n                     /region_name=\"Hydrogen bonded turn\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          63\\n                     /region_name=\"Variant\"\\n                     /note=\"G -> E (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\\n                     allele DQA1*03:01, allele DQA1*03:02 and allele\\n                     DQA1*03:03; dbSNP:rs1142323). /id=VAR_060496.\"\\n     Region          64\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> K (in allele DQA1*01:03; dbSNP:rs36219699).\\n                     /id=VAR_050383.\"\\n     Region          66..71\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          67\\n                     /region_name=\"Variant\"\\n                     /note=\"T -> A (in allele DQA1*01:06; dbSNP:rs41543221).\\n                     /id=VAR_060497.\"\\n     Region          68\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> A (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142324).\\n                     /id=VAR_060498.\"\\n     Region          70\\n                     /region_name=\"Variant\"\\n                     /note=\"C -> K (in allele DQA1*02:01; requires 2 nucleotide\\n                     substitutions). /id=VAR_060499.\"\\n     Region          70\\n                     /region_name=\"Variant\"\\n                     /note=\"C -> Q (in allele DQA1*03:01, allele DQA1*03:02 and\\n                     allele DQA1*03:03; requires 2 nucleotide substitutions).\\n                     /id=VAR_060500.\"\\n     Region          70\\n                     /region_name=\"Variant\"\\n                     /note=\"C -> R (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142326).\\n                     /id=VAR_060501.\"\\n     Region          70\\n                     /region_name=\"Variant\"\\n                     /note=\"C -> Y (in dbSNP:rs3207983). /id=VAR_033404.\"\\n     Region          71\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> W (in allele DQA1*01:01, allele\\n                     DQA1*01:02,allele DQA1*01:03, allele DQA1*01:04, allele\\n                     DQA1*01:05, allele DQA1*01:06 and allele DQA1*01:07;\\n                     dbSNP:rs1142328). /id=VAR_060502.\"\\n     Region          72..76\\n                     /region_name=\"Helical region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          73\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> D (in dbSNP:rs760671632). /id=VAR_033406.\"\\n     Region          73\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> E (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs3208105).\\n                     /id=VAR_060503.\"\\n     Region          73\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> L (in allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02 and allele DQA1*03:03;\\n                     dbSNP:rs12722061). /id=VAR_033405.\"\\n     Region          74\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> F (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\\n                     allele DQA1*03:01, allele DQA1*03:02 and allele\\n                     DQA1*03:03; dbSNP:rs9272698). /id=VAR_060504.\"\\n     Region          75\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> H (in allele DQA1*02:01; dbSNP:rs28383449).\\n                     /id=VAR_060505.\"\\n     Region          75\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> S (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs9272699).\\n                     /id=VAR_060506.\"\\n     Region          76\\n                     /region_name=\"Variant\"\\n                     /note=\"Q -> K (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1048052).\\n                     /id=VAR_060507.\"\\n     Region          76\\n                     /region_name=\"Variant\"\\n                     /note=\"Q -> R (in allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02 and allele DQA1*03:03;\\n                     dbSNP:rs12722069). /id=VAR_060508.\"\\n     Region          77\\n                     /region_name=\"Variant\"\\n                     /note=\"F -> L (in allele DQA1*02:01; dbSNP:rs3188043).\\n                     /id=VAR_060509.\"\\n     Region          78\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> GG (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele\\n                     DQA1*01:05,allele DQA1*01:06 and allele DQA1*01:07;\\n                     dbSNP:rs4193). /id=VAR_060510.\"\\n     Region          78\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> RR (in allele DQA1*03:01, allele DQA1*03:02\\n                     and allele DQA1*03:03). /id=VAR_060511.\"\\n     Region          81..101\\n                     /region_name=\"Helical region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          81\\n                     /region_name=\"Variant\"\\n                     /note=\"P -> R (in allele DQA1*05:02; dbSNP:rs41541412).\\n                     /id=VAR_060512.\"\\n     Region          83\\n                     /region_name=\"Variant\"\\n                     /note=\"F -> G (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; requires 2\\n                     nucleotide substitutions). /id=VAR_060513.\"\\n     Region          86\\n                     /region_name=\"Variant\"\\n                     /note=\"T -> R (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142333).\\n                     /id=VAR_033408.\"\\n     Region          88\\n                     /region_name=\"Variant\"\\n                     /note=\"I -> M (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1142334).\\n                     /id=VAR_033409.\"\\n     Region          91\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> A (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; requires 2\\n                     nucleotide substitutions). /id=VAR_060514.\"\\n     Region          91\\n                     /region_name=\"Conflict\"\\n                     /note=\"L -> H (in Ref. 4; AAA59760).\\n                     /evidence=ECO:0000305.\"\\n     Region          91\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> T (in allele DQA1*04:01, allele DQA1*04:02,\\n                     allele DQA1*04:04, allele DQA1*06:01 and allele\\n                     DQA1*06:02; requires 2 nucleotide substitutions).\\n                     /id=VAR_060515.\"\\n     Region          97\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> I (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06, allele DQA1*01:07, allele DQA1*02:01,\\n                     allele DQA1*03:01, allele DQA1*03:02, allele DQA1*03:03,\\n                     allele DQA1*04:01, allele DQA1*04:02, allele DQA1*04:04,\\n                     allele DQA1*06:01 and allele DQA1*06:02; dbSNP:rs9279910).\\n                     /id=VAR_060516.\"\\n     Region          98\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> M (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1064944).\\n                     /id=VAR_060517.\"\\n     Region          98\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> V (in allele DQA1*03:01, allele DQA1*03:02 and\\n                     allele DQA1*03:03; dbSNP:rs1064944). /id=VAR_060518.\"\\n     Region          101\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> C (in allele DQA1*01:07; dbSNP:rs41542116).\\n                     /id=VAR_060519.\"\\n     Region          102..104\\n                     /region_name=\"Hydrogen bonded turn\"\\n                     /note=\"/evidence=ECO:0007829|PDB:1JK8.\"\\n     Region          102\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> Y (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:06 and allele DQA1*01:07; dbSNP:rs1129808).\\n                     /id=VAR_050384.\"\\n     Site            103\\n                     /site_type=\"glycosylation\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255.\"\\n     Region          107\\n                     /region_name=\"Conflict\"\\n                     /note=\"A -> P (in Ref. 16; AAA59754).\\n                     /evidence=ECO:0000305.\"\\n     Region          110..204\\n                     /region_name=\"IgC1_MHC_II_alpha_HLA-DQ\"\\n                     /note=\"Class II major histocompatibility complex (MHC)\\n                     alpha chain immunoglobulin domain of histocompatibility\\n                     antigen (HLA) DQ and related proteins; member of the\\n                     C1-set of Ig superfamily (IgSF) domains; cd21008\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          111..119\\n                     /region_name=\"Ig strand A\"\\n                     /note=\"Ig strand A [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          112..204\\n                     /region_name=\"Domain\"\\n                     /note=\"Ig-like C1-type.\"\\n     Region          113..120\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Site            order(115,117..119,121,173,175)\\n                     /site_type=\"other\"\\n                     /note=\"heterodimer interface [polypeptide binding]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          120..203\\n                     /region_name=\"Region of interest in the sequence\"\\n                     /note=\"Alpha-2.\"\\n     Region          124..126\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:4OZF.\"\\n     Region          124\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> V (in allele DQA1*05:06; dbSNP:rs41555012).\\n                     /id=VAR_060520.\"\\n     Region          127..136\\n                     /region_name=\"Ig strand B\"\\n                     /note=\"Ig strand B [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          128..137\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          129\\n                     /region_name=\"Variant\"\\n                     /note=\"I -> T (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\\n                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\\n                     and allele DQA1*06:02; dbSNP:rs707952). /id=VAR_050385.\"\\n     Bond            bond(132,188)\\n                     /bond_type=\"disulfide\"\\n                     /note=\"/evidence=ECO:0000255|PROSITE-ProRule:PRU00114,\\n                     ECO:0000269|PubMed:11376336, ECO:0000269|PubMed:14769912,\\n                     ECO:0000269|PubMed:17629515.\"\\n     Site            order(139..141,160..163,169..170,175)\\n                     /site_type=\"other\"\\n                     /note=\"MHC binding domain interface [polypeptide binding]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          142..148\\n                     /region_name=\"Ig strand C\"\\n                     /note=\"Ig strand C [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          143..148\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Site            143\\n                     /site_type=\"glycosylation\"\\n                     /note=\"N-linked (GlcNAc...) asparagine.\\n                     /evidence=ECO:0000255.\"\\n     Region          150..153\\n                     /region_name=\"Ig strand C\\'\"\\n                     /note=\"Ig strand C\\' [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          151..153\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          151\\n                     /region_name=\"Variant\"\\n                     /note=\"H -> Q (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:04, allele DQA1*01:05 and allele\\n                     DQA1*01:07; dbSNP:rs707950). /id=VAR_050386.\"\\n     Region          152\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> A (in allele DQA1*01:03; dbSNP:rs41547417).\\n                     /id=VAR_060521.\"\\n     Region          156\\n                     /region_name=\"Conflict\"\\n                     /note=\"G -> D (in Ref. 3; CAA25141).\\n                     /evidence=ECO:0000305.\"\\n     Region          157..165\\n                     /region_name=\"Ig strand D\"\\n                     /note=\"Ig strand D [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          157..159\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          160\\n                     /region_name=\"Variant\"\\n                     /note=\"T -> I (in allele DQA1*04:02; dbSNP:rs41545514).\\n                     /id=VAR_060522.\"\\n     Region          161\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> I (in allele DQA1*05:08; dbSNP:rs41544114).\\n                     /id=VAR_060524.\"\\n     Region          161\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> R (in allele DQA1*06:02; dbSNP:rs41552014).\\n                     /id=VAR_060523.\"\\n     Region          168..178\\n                     /region_name=\"Ig strand E\"\\n                     /note=\"Ig strand E [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          170..178\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          175\\n                     /region_name=\"Variant\"\\n                     /note=\"Y -> H (in allele DQA1*04:04; dbSNP:rs41550317).\\n                     /id=VAR_060525.\"\\n     Region          178\\n                     /region_name=\"Variant\"\\n                     /note=\"L -> F (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\\n                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\\n                     and allele DQA1*06:02; dbSNP:rs707949). /id=VAR_060526.\"\\n     Region          182\\n                     /region_name=\"Variant\"\\n                     /note=\"A -> D (in allele DQA1*03:02 and allele DQA1*03:03;\\n                     dbSNP:rs7990). /id=VAR_060527.\"\\n     Region          182\\n                     /region_name=\"Variant\"\\n                     /note=\"A -> S (in allele DQA1*05:03, allele DQA1*05:06 and\\n                     allele DQA1*05:07; dbSNP:rs41561312). /id=VAR_060528.\"\\n     Region          183\\n                     /region_name=\"Variant\"\\n                     /note=\"E -> D (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\\n                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\\n                     and allele DQA1*06:02; dbSNP:rs707963). /id=VAR_060529.\"\\n     Region          185..192\\n                     /region_name=\"Ig strand F\"\\n                     /note=\"Ig strand F [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          185\\n                     /region_name=\"Variant\"\\n                     /note=\"S -> I (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05,\\n                     allele DQA1*01:07, allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\\n                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\\n                     and allele DQA1*06:02; dbSNP:rs707962). /id=VAR_060530.\"\\n     Region          186..191\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          195..197\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          197\\n                     /region_name=\"Variant\"\\n                     /note=\"K -> E (in allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02, allele DQA1*03:03, allele DQA1*04:01,\\n                     allele DQA1*04:02, allele DQA1*04:04, allele DQA1*06:01\\n                     and allele DQA1*06:02; dbSNP:rs2308891). /id=VAR_060531.\"\\n     Region          197\\n                     /region_name=\"Variant\"\\n                     /note=\"K -> Q (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04, allele DQA1*01:05\\n                     and allele DQA1*01:07; dbSNP:rs2308891). /id=VAR_060532.\"\\n     Region          198..203\\n                     /region_name=\"Ig strand G\"\\n                     /note=\"Ig strand G [structural motif]\"\\n                     /db_xref=\"CDD:409599\"\\n     Region          199..203\\n                     /region_name=\"Beta-strand region\"\\n                     /note=\"/evidence=ECO:0007829|PDB:6U3M.\"\\n     Region          204..216\\n                     /region_name=\"Region of interest in the sequence\"\\n                     /note=\"Connecting peptide.\"\\n     Region          209\\n                     /region_name=\"Variant\"\\n                     /note=\"A -> T (in allele DQA1*03:01, allele DQA1*03:02 and\\n                     allele DQA1*03:03; dbSNP:rs9272785). /id=VAR_050387.\"\\n     Region          213\\n                     /region_name=\"Conflict\"\\n                     /note=\"E -> D (in Ref. 16; AAA59754).\\n                     /evidence=ECO:0000305.\"\\n     Region          217..239\\n                     /region_name=\"Transmembrane region\"\\n                     /note=\"Helical. /evidence=ECO:0000255.\"\\n     Region          221\\n                     /region_name=\"Variant\"\\n                     /note=\"A -> T (in allele DQA1*01:04; dbSNP:rs35087390).\\n                     /id=VAR_050388.\"\\n     Region          229\\n                     /region_name=\"Variant\"\\n                     /note=\"V -> M (in allele DQA1*01:02; dbSNP:rs9260).\\n                     /id=VAR_033411.\"\\n     Region          230\\n                     /region_name=\"Variant\"\\n                     /note=\"G -> C (in allele DQA1*05:07; dbSNP:rs41545416).\\n                     /id=VAR_060533.\"\\n     Region          237\\n                     /region_name=\"Variant\"\\n                     /note=\"F -> L (in allele DQA1*02:01, allele DQA1*03:01,\\n                     allele DQA1*03:02 and allele DQA1*03:03; dbSNP:rs1048430).\\n                     /id=VAR_033412.\"\\n     Region          240..254\\n                     /region_name=\"Topological domain\"\\n                     /note=\"Cytoplasmic.\"\\n     Region          240\\n                     /region_name=\"Variant\"\\n                     /note=\"R -> Q (in allele DQA1*01:01, allele DQA1*01:02,\\n                     allele DQA1*01:03, allele DQA1*01:04 and allele\\n                     DQA1*01:05; dbSNP:rs1048439). /id=VAR_033413.\"\\nORIGIN      \\n        1 milnkalmlg alalttvmsp cggedivadh vasygvnlyq sygpsgqyth efdgdeqfyv\\n       61 dlgrketvwc lpvlrqfrfd pqfaltniav lkhnlnslik rsnstaatne vpevtvfsks\\n      121 pvtlgqpnil iclvdnifpp vvnitwlsng hsvtegvset sflsksdhsf fkisyltllp\\n      181 saeesydckv ehwgldkpll khwepeipap mseltetvvc alglsvglvg ivvgtvfiir\\n      241 glrsvgasrh qgpl\\n//\\n\\n'"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'DQA1_HUMAN',db='protein')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Como conseguimos obter o Accession Number da proteína codificada pelo gene de interesse, procedemos à gravação do ficheiro da mesma."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Encontrados 10 resultados em protein. Irá ser processado 1 resultado.\n",
      "\n",
      "Ficheiro já existe. Não será criado novo ficheiro.\n",
      "Ficheiro gravado.\n",
      "Concluído.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'>sp|P01909.1|DQA1_HUMAN RecName: Full=HLA class II histocompatibility antigen, DQ alpha 1 chain; AltName: Full=DC-1 alpha chain; AltName: Full=DC-alpha; AltName: Full=HLA-DCA; AltName: Full=MHC class II DQA1; Flags: Precursor\\nMILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYTHEFDGDEQFYVDLGRKETVWC\\nLPVLRQFRFDPQFALTNIAVLKHNLNSLIKRSNSTAATNEVPEVTVFSKSPVTLGQPNILICLVDNIFPP\\nVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTLLPSAEESYDCKVEHWGLDKPLLKHWEPEIPAP\\nMSELTETVVCALGLSVGLVGIVVGTVFIIRGLRSVGASRHQGPL\\n\\n'"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pesquisa_ncbi(email,'P01909',db='protein',retmax=1,rettype='fasta',save='Y',display='n')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
